<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="google-site-verification" content="d3RoBpynB-G21xjnmrqTE7iJfzqgS6DugRe7B6yUVb8" />
  
    
  
    <link rel="alternate" type="application/rss+xml" href="/rss.xml" />
    
    <link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/libra-css/images/site/fav/favicon.ico"/>
    <link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>
    <link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-iphone-114-smile.png"/>
    <link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/libra-css/images/site/touch-icon-ipad-144-smile.png"/>  
  
    <title>Registry of Open Data on AWS</title>
  
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://assets.opendata.aws/css/bootstrap/3.4.1/bootstrap.min.css">
  
    <!-- Our local CSS -->
    <link rel="stylesheet" href="/css/main.css">
  
  
  </head>
  <body>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://assets.opendata.aws/js/jquery/3.5.1/jquery.min.js"></script>
    <!-- Latest compiled and minified JavaScript -->
    <script src="https://assets.opendata.aws/js/bootstrap/3.4.1/bootstrap.min.js"></script>
    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="roda-header">
          <h1>Registry of Open Data on AWS</h1>
          <a href="http://aws.amazon.com/what-is-cloud-computing">
            <img src="https://assets.opendata.aws/img/AWS-Logo_White-Color_300x180.png" alt="Powered by AWS Cloud Computing" id="aws_header_logo">
          </a>
        </div>
      </div>
    </nav>
    <div id="adx-banner">
      <div class="adx-banner-container">
        <img src="/img/info.svg" class="icon-info">
        <div class="adx-banner-text">
          <b>The Registry of Open Data on AWS is now available on AWS Data Exchange</b>
          <div>
            <span>All datasets on the Registry of Open Data are now discoverable on AWS Data Exchange alongside 3,000+ existing data products from category-leading data providers across industries. Explore the catalog to find open, free, and commercial data sets.</span>
            <a href="https://docs.aws.amazon.com/data-exchange/" target="_blank">Learn more about AWS Data Exchange <img src="/img/external.svg" class="icon" /></a>
          </div>
        </div>
      </div>
      <div class="adx-banner-container">
        <button class="adx-banner-button">
          Explore the catalog
          <img src="/img/external_black.svg" class="icon">
        </button>
        <div class="adx-close-button"><img src="/img/close.svg" class="icon"></div>
      </div>
    </div>
    <div id="spacer"></div>
    <!-- JavaScript for ADX banner -->
    <script src="/js/banner.js"></script>  
    <div class="container" >

    <div class="about col-md-5">
      <div class="row aboutbox">
            <h3>About</h3>
            <p>This registry exists to help people discover and share datasets that are available via AWS resources. See <a href="/change-log">recent additions</a> and <a href="https://opendata.aws">learn more about sharing data on AWS</a>.</p>
    
            <p>Get started using data quickly by viewing <a href="/service/sagemaker-studio-lab/usage-examples">all tutorials with associated SageMaker Studio Lab notebooks</a>.</p>        
            <p>See <a href="/usage-examples">all usage examples for datasets listed in this registry</a>.</p>
            <p>See datasets from <a href="/collab/ai2/">Allen Institute for Artificial Intelligence (AI2)</a>, <a href="/collab/deafrica/">Digital Earth Africa</a>, <a href="/collab/meta/">Data for Good at Meta</a>, <a href="/collab/nasa/">NASA Space Act Agreement</a>, <a href="/collab/nih/">NIH STRIDES</a>, <a href="/collab/noaa/">NOAA Open Data Dissemination Program</a>, <a href="/collab/stsci/">Space Telescope Science Institute</a>, and <a href="/collab/asdi/">Amazon Sustainability Data Initiative</a>.
            </p>
    
        <hr>
    
        <h4>Search datasets (currently <span id="count-matching">13</span> matching <span id="count-matching-text">datasets</span>)</h4>
        <form>
          <div class="form-group">
            <input type="text" id="search-box" class="form-control" placeholder="Search datasets" spellcheck="false" autocorrect="off">
          </div>
        </form>
        
    
        <hr>
    
        <h4>Add to this registry</h4>
        <p>If you want to add a dataset or example of how to use a dataset to this registry, please follow the instructions on the <a href="https://github.com/awslabs/open-data-registry/">Registry of Open Data on AWS GitHub repository</a>.</p>
    
        <p>Unless specifically stated in the applicable dataset documentation, datasets available through the Registry of Open Data on AWS are not provided and maintained by AWS. Datasets are provided and maintained by a variety of third parties under a variety of licenses. Please check dataset licenses and related documentation to determine if a dataset may be used for your application.</p>        
        
        <hr>
    
        <h4>Tell us about your project</h4>
        <p>If you have a project using a listed dataset, please <a href="mailto:opendata@amazon.com">tell us about it</a>. We may work with you to feature your project in a <a href="https://opendata.aws/blog" target="_blank">blog post</a>.</p>
      </div>
    </div>
    <div class="col-md-6 col-md-offset-1 datasets">
      <div class="row">
        <div id="bdsp-hsp" class="dataset">
          <h3><a href="/bdsp-hsp/">The Human Sleep Project</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>The Human Sleep Project (HSP) sleep physiology dataset is a growing collection of clinical polysomnography (PSG) recordings. Beginning with PSG recordings from from ~15K patients evaluated at the Massachusetts General Hospital, the HSP will grow over the coming years to include data from &gt;200K patients, as well as people evaluated outside of the clinical setting. This data is being used to develop CAISR (Complete AI Sleep Report), a collection of deep neural networks,  rule-based algorithms, and signal processing approaches designed to provide better-than-human detection of conventional PSG...</p>
          <p><a href="/bdsp-hsp/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.60508/r0t9-5r24" class="" target="_none">The Human Sleep Project</a> by <a href="https://bdsp.io" target="_none">Westover MB, Moura Junior V, Thomas R, Cash S, Nasiri S, Sun H, et al.</a></li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/26518699/" class="" target="_none">Decision modeling in sleep apnea: the critical roles of pre-test probability, cost of untreated OSA, and time horizon.  Journal of Clinical Sleep Medicine. 2016 Mar;12(3):409-418.  PMCID: PMC4773629.</a> by Moro M, Westover MB,  Kelly J, Bianchi MT.</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/21151998/" class="" target="_none">Power Law versus Exponential State Transition Dynamics:  Application to Sleep-Wake Architecture. PLoSONE  5(12):  e14204. Dec. 2, 2010. PMCID: PMC2996311.</a> by Chu-Shore J, Westover MB, Bianchi MT</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/34984446/" class="" target="_none">Optimal Spindle Detection Parameters for Predicting Cognitive Performance. Sleep. 2022 Jan 4:zsac001. doi: 10.1093/sleep/zsac001. Epub ahead of print. PMCID: PMC8996023.</a> by Adra N, Sun H, Ganglberger W, Ye EM, Dümmer LW, Tesh RA, et al.</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/32022674/" class="" target="_none">Artificial Intelligence in Sleep Medicine: An American Academy of Sleep Medicine Position Statement. Journal of Clinical Sleep Medicine. 2020 Feb 5. doi: 10.5664/jcsm.8288. PMCID: PMC7161449.</a> by Goldstein CA, Berry RB, Kent DT, Kristo DA, Seixas AA, Redline S, et al.</li></a>
          </li>
        </ul>
        <p><a href="/bdsp-hsp/#usageexamples">See 37 usage examples &rarr;</a></p>
        </div>
        <div id="commoncrawl" class="dataset">
          <h3><a href="/commoncrawl/">Common Crawl</a></h3>
          <p><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">web archive</span></p>
          <p>A corpus of web crawl data composed of over 50 billion web pages.</p>
          <p><a href="/commoncrawl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.lrec-conf.org/proceedings/lrec2016/pdf/388_Paper.pdf" class="" target="_none">C4Corpus: Multilingual Web-Size Corpus with Free License</a> by <a href="https://dkpro.github.io/dkpro-c4corpus/" target="_none">Ivan Habernal, Omnia Zayed, Iryna Gurevych</a></li></a>
          </li>
          <li>
            <a href="https://psuter.net/2019/07/07/z-index" class="" target="_none">Index fun</a> by Philippe Suter</li></a>
          </li>
          <li>
            <a href="https://commoncrawl.org/2018/03/index-to-warc-files-and-urls-in-columnar-format/" class="" target="_none">Index to WARC Files and URLs in Columnar Format</a> by Sebastian Nagel</li></a>
          </li>
          <li>
            <a href="https://wwwdb.inf.tu-dresden.de/research-projects/dresden-web-table-corpus/" class="" target="_none">Dresden Web Table Corpus (DWTC)</a> by <a href="https://wwwdb.inf.tu-dresden.de/" target="_none">Database Systems Group Dresden</a></li></a>
          </li>
          <li>
            <a href="https://www.aclweb.org/anthology/2020.emnlp-main.480" class="" target="_none">CCAligned: A Massive collection of cross-lingual web-document pairs</a> by <a href="http://www.statmt.org/cc-aligned/" target="_none">Ahmed El-Kishky, Vishrav Chaudhary, Francisco Guzmán, Philipp Koehn</a></li></a>
          </li>
        </ul>
        <p><a href="/commoncrawl/#usageexamples">See 35 usage examples &rarr;</a></p>
        </div>
        <div id="tcga" class="dataset">
          <h3><a href="/tcga/">The Cancer Genome Atlas</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Genome Atlas (TCGA), a collaboration between the National Cancer Institute (NCI) and National Human Genome Research Institute (NHGRI), aims to generate comprehensive, multi-dimensional maps of the key genomic changes in major types and subtypes of cancer. TCGA has analyzed matched tumor and normal tissues from 11,000 patients, allowing for the comprehensive characterization of 33 cancer types and subtypes, including 10 rare cancers.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/tcga/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.cell.2018.03.027" class="" target="_none">A Pan-Cancer Analysis of Enhancer Expression in Nearly 9000 Patient Samples</a> by Han Chen, Chunyan Li, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.ccell.2018.03.010" class="" target="_none">Comparative Molecular Analysis of Gastrointestinal Adenocarcinomas</a> by Yang Liu, Nilay S. Sethi, et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.celrep.2018.03.076" class="" target="_none">Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome
Atlas
</a> by Theo A. Knijnenburg, Linghua Wang, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.ccell.2018.07.001" class="" target="_none">Comprehensive Analysis of Alternative Splicing Across Tumors from 8,705 Patients</a> by André Kahles, Kjong-Van Lehmann, et al.</li></a>
          </li>
        </ul>
        <p><a href="/tcga/#usageexamples">See 29 usage examples &rarr;</a></p>
        </div>
        <div id="foldingathome-covid19" class="dataset">
          <h3><a href="/foldingathome-covid19/">Foldingathome COVID-19 Datasets</a></h3>
          <p><span class="label label-info tag link-tag">alchemical free energy calculations</span><span class="label label-info tag link-tag">biomolecular modeling</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">foldingathome</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular dynamics</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">SARS-CoV-2</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">structural biology</span></p>
          <p><a href="http://foldingathome.org">Folding@home</a> is a massively distributed computing project that uses biomolecular simulations to investigate the <a href="https://foldingathome.org/diseases/">molecular origins of disease</a> and accelerate the discovery of new therapies. Run by the <a href="https://foldingathome.org/about/the-foldinghome-consortium/">Folding@home Consortium</a>, a worldwide network of research laboratories focusing on a variety of different diseases, Folding@home seeks to address problems in human health on a scale that is infeasible by another other means, sharing the results of these large-scale studies with the research community through <a href="https://foldingathome.org/papers-results/">peer-reviewed publications</a> and publicly shared datasets. During the <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 epidemic</a>, Folding@home focused its resources on understanding the vulnerabilities in <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>, the virus that causes COVID-19 disease, and working closely with a number of experimental collaborators to accelerate progress toward effective therapies for treating COVID-19 and ending the pandemic. In the process, it created the world&#39;s first <a href="https://doi.org/10.1101/2020.06.27.175430">exascale distributed computing resource</a>, enabling it to generate valuable scientific datasets of unprecedented size. More information about Folding@home&#39;s COVID-19 research activities at the <a href="https://foldingathome.org/diseases/infectious-diseases/covid-19/">Folding@home COVID-19 page</a>. In addition to working directly with experimental collaborators and rapidly sharing new research findings through preprint servers, Folding@home has joined other researchers in committing to <a href="https://doi.org/10.1021/acs.jcim.0c00319">rapidly share all COVID-19 research data</a>, and has joined forces with <a href="https://aws.amazon.com/">AWS</a> and the <a href="http://molssi.org">Molecular Sciences Software Institute (MolSSI)</a> to share datasets of unprecedented side through the <a href="https://registry.opendata.aws/">AWS Open Data Registry</a>, indexing these massive datasets via the <a href="https://covid.molssi.org/">MolSSI COVID-19 Molecular Structure and Therapeutics Hub</a>. The complete index of all Folding@home datasets can be found <a href="https://covid.molssi.org//org-contributions/#folding--home">here</a>. Th...</p>
          <p><a href="/foldingathome-covid19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp3-macrodomain-macrodomain-host-immune-response" class="" target="_none">SARS-CoV-2 nsp3 macrodomain dataset: An 11 ms dataset of the SARS-CoV-2 nsp3 macrodomain in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp5-3clpro-3clpro-protease-activity" class="" target="_none">SARS-CoV-2 main viral protease (Mpro, 3CLPro, nsp5) monomer simulations: A 6.4 ms dataset of the SARS-CoV-2 main viral protease (apo, monomer) in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-the-sars-cov-2-spike-rbd-bound-to-human-ace2" class="" target="_none">SARS-CoV-2 spike RBD bound to human ACE2 receptor (173.8 us): Wild-type and mutant simulations</a> by <a href="https://choderalab.org" target="_none">The Chodera lab at the Memorial Sloan Kettering Cancer Center</a></li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp9-nsp9-nil" class="" target="_none">SARS-CoV-2 nsp9 simulations: A 9 ms dataset of the SARS-CoV-2 nsp9 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
          <li>
            <a href="https://covid.molssi.org//simulations/#foldinghome-simulations-of-nsp12-rdrp-nil" class="" target="_none">SARS-CoV-2 RNA polymerase (nsp12, RdRP) dataset: A 3.4 ms dataset of the SARS-CoV-2 nsp12 protein in search of cryptic pockets</a> by <a href="https://bowmanlab.biochem.wustl.edu" target="_none">The Bowman lab at Washington University in St. Louis</a></li></a>
          </li>
        </ul>
        <p><a href="/foldingathome-covid19/#usageexamples">See 24 usage examples &rarr;</a></p>
        </div>
        <div id="target" class="dataset">
          <h3><a href="/target/">Therapeutically Applicable Research to Generate Effective Treatments (TARGET)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Therapeutically Applicable Research to Generate Effective Treatments (TARGET) is the collaborative effort of a large, diverse consortium of extramural and NCI investigators. The goal of the effort is to accelerate molecular discoveries that drive the initiation and progression of hard-to-treat childhood cancers and facilitate rapid translation of those findings into the clinic.
TARGET projects provide comprehensive molecular characterization to determine the genetic changes that drive the initiation and progression of childhood cancers.The dataset contains open Clinical Supplement, Biospecimen...</p>
          <p><a href="/target/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;29315296" class="" target="_none">Genomics in childhood acute myeloid leukemia comes of age</a> by Brunner AM, Graubert TA.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">GDC Legacy Archive</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;26121087" class="" target="_none">Relapsed neuroblastomas show frequent RAS-MAPK pathway mutations</a> by Eleveld TF, Oldridge DA, Bernard V, Koster J, et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;27702824" class="" target="_none">Significance of TP53 Mutation in Wilms Tumors with Diffuse Anaplasia: A Report from the
Children&#x27;s Oncology Group
</a> by Ooms AH, Gadd S, Gerhard DS, et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;23334666" class="" target="_none">The genetic landscape of high-risk neuroblastoma</a> by Pugh TJ, Morozova O, Attiyeh EF, Asgharzadeh S, Wei JS, Auclair D, Carter SL, Cibulskis K,
Hanna M, Kiezun A, Kim J, Lawrence MS, Lichenstein L, et al.
</li></a>
          </li>
        </ul>
        <p><a href="/target/#usageexamples">See 24 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel-2" class="dataset">
          <h3><a href="/sentinel-2/">Sentinel-2</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The <a href="https://sentinel.esa.int/web/sentinel/missions/sentinel-2">Sentinel-2 mission</a> is
a land monitoring constellation of two satellites that provide high resolution
optical imagery and provide continuity for the current SPOT and Landsat missions.
The mission provides a global coverage of the Earth&#39;s land surface every 5 days,
making the data of great use in on-going studies. L1C data are available from
June 2015 globally. L2A data are available from November 2016 over Europe
region and globally since January 2017.</p>
          <p><a href="/sentinel-2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.matecdev.com/posts/landsat-sentinel-aws-s3-python.html" class="" target="_none">How to Work with Landsat and Sentinel-2 on AWS with Python</a> by <a href="https://www.matecdev.com" target="_none">Martin D. Maas</a></li></a>
          </li>
          <li>
            <a href="https://map.onesoil.ai/" class="" target="_none">Interactive map with AI detected fields and crops based on Sentinel-2 imagery</a> by <a href="https://onesoil.ai/" target="_none">OneSoil</a></li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/9165649" class="" target="_none">Using Remote Sensing Images and Cloud Services on AWS to Improve Land Use and Cover Monitoring</a> by K. R. Ferreira, et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/aws-sentinel2-smsl-notebook" class="aws-link" target="_none">Vegetation Analysis Pre/Post Lava Fire Using SentinelHub and Sentinel-2 Imagery on AWS</a> by Mike Jeffe</li></a>
          </li>
          <li>
            <a href="https://www.gearthblog.com/blog/archives/2017/02/exploring-chile-wildfires-landsat-sentinel-2-imagery.html" class="" target="_none">Exploring the Chile wildfires with Landsat and Sentinel-2 imagery</a> by Timothy Whitehead</li></a>
          </li>
        </ul>
        <p><a href="/sentinel-2/#usageexamples">See 23 usage examples &rarr;</a></p>
        </div>
        <div id="usgs-landsat" class="dataset">
          <h3><a href="/usgs-landsat/">USGS Landsat</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>This joint NASA/USGS program provides the longest continuous space-based record of 
Earth’s land in existence. Every day, Landsat satellites provide essential information 
to help land managers and policy makers make wise decisions about our resources and our environment.
Data is provided for Landsats 1, 2, 3, 4, 5, 7, 8, and 9 (excludes Landsat 6).As of June 28, 2023 (<a href="https://www.usgs.gov/landsat-missions/news/changes-coming-landsats-simple-notification-service-amazon-web-service">announcement</a>),
the previous single SNS topic <code>arn:aws:sns:us-west-2:673253540267:public-c2-notify</code> was replaced with
three new SNS topics for different types of scenes.</p>
          <p><a href="/usgs-landsat/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.usgs.gov/media/files/landsat-cloud-direct-access-requester-pays-tutorial" class="" target="_none">Landsat Cloud Direct Access Requester Pays Tutorial</a> by <a href="https://www.usgs.gov" target="_none">USGS</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/sentinel-hub/clouds-segmentation-in-landsat-8-images-da370815235" class="" target="_none">Cloud Segmentation in Landsat-8 Images</a> by Ingrid Grenet</li></a>
          </li>
          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></li></a>
          </li>
          <li>
            <a href="https://docs.safe.com/fme/html/FME_Desktop_Documentation/FME_ReadersWriters/landsat8aws/landsat8aws.htm" class="" target="_none">FME Landsat-8 on AWS Reader</a> by <a href="https://www.safe.com/" target="_none">Safe Software</a></li></a>
          </li>
          <li>
            <a href="https://twitter.com/aggibot" class="" target="_none">Aggi: the fire detecting Twitter bot</a> by <a href="https://twitter.com/aggibot" target="_none">Aggi</a></li></a>
          </li>
        </ul>
        <p><a href="/usgs-landsat/#usageexamples">See 23 usage examples &rarr;</a></p>
        </div>
        <div id="ilmn-dragen-1kgp" class="dataset">
          <h3><a href="/ilmn-dragen-1kgp/">1000 Genomes Phase 3 Reanalysis with DRAGEN 3.5, 3.7, 4.0, and 4.2</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">genotyping</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">population genetics</span><span class="label label-info tag link-tag">short read sequencing</span><span class="label label-info tag link-tag">structural variation</span><span class="label label-info tag link-tag">tertiary analysis</span><span class="label label-info tag link-tag">variant annotation</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This dataset contains alignment files and short nucleotide, copy number (CNV), repeat expansion (STR), structural variant (SV) and other variant call files from the 1000 Genomes Project Phase 3 dataset (n=3202) using Illumina DRAGEN v3.5.7b, v3.7.6, v4.0.3, and v4.2.7 software. All DRAGEN analyses were performed in the cloud using the Illumina Connected Analytics bioinformatics platform powered by Amazon Web Services (see &#39;Data solution empowering population genomics&#39; for more information).  The v3.7.6 and v4.2.7 datasets include results from trio small variant, de novo structural vari...</p>
          <p><a href="/ilmn-dragen-1kgp/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.illumina.com/content/dam/illumina-support/documents/downloads/software/dragen/release-notes/Illumina-DRAGEN-Bio-IT-Platform-3.7-Release-Notes-1000000142362-v00.pdf" class="" target="_none">DRAGEN 3.7.5 Software Release Notes</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
          <li>
            <a href="https://www.illumina.com/science/genomics-research/articles/gVCF-Genotyper.html" class="" target="_none">Genotyping variants at population scale using DRAGEN gVCF Genotyper</a> by Illumina Inc. (2023)</li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.11.13.380741v1" class="" target="_none">precisionFDA Truth Challenge V2: Calling variants from short- and long-reads in difficult-to-map regions (Preprint)</a> by Olson et al (2020)</li></a>
          </li>
          <li>
            <a href="https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub.html#key-features" class="" target="_none">DRAGEN on BaseSpace Sequence Hub</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
          <li>
            <a href="https://support.illumina.com/content/dam/illumina-support/help/Illumina_DRAGEN_Bio_IT_Platform_v3_7_1000000141465/Content/SW/FrontPages/DRAGENBioITPlatform.htm" class="" target="_none">Illumina DRAGEN Bio-IT Platform v3.7 Online Help</a> by <a href="https://www.illumina.com/" target="_none">Illumina Inc.</a></li></a>
          </li>
        </ul>
        <p><a href="/ilmn-dragen-1kgp/#usageexamples">See 22 usage examples &rarr;</a></p>
        </div>
        <div id="allen-cell-imaging-collections" class="dataset">
          <h3><a href="/allen-cell-imaging-collections/">Allen Cell Imaging Collections</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">microscopy</span></p>
          <p>This bucket contains multiple datasets (as Quilt packages) created by the Allen Institute for Cell Science. The types of data included in this bucket are listed below:<ol>
<li>Field of view or cropped images of cells</li>
<li>Segmentations of structures in the images (e.g., boundaries of cells, DNA, other intracellular structures, etc.)</li>
<li>Processed versions of the above images and segmentations</li>
<li>Machine learning predictions and labels of the data listed above</li>
<li>Models trained on the previously listed data</li>
<li>Additional supporting non-image data related to the above listed data types (e.g., gene expression data, whole genome sequencing data, features derived from the images or model predictions, metadata)</li>
<li>Simulation, analysis, and visualization data of in silico cell structures, cells, and cell populations</li>
</ol>
Extern...</p>
          <p><a href="/allen-cell-imaging-collections/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1101/2024.08.12.607546" class="" target="_none">4D hybrid model interrogates agent-level rules and parameters driving hiPS cell colony dynamics</a> by <a href="https://alleninstitute.org/about/team/" target="_none">Jessica S. Yu, Blair Lyons, Susanne Rafelski, Julie A. Theriot, Neda Bagheri &amp; Graham T. Johnson</a></li></a>
          </li>
          <li>
            <a href="https://www.allencell.org/visual-guide-to-human-cells.html" class="" target="_none">Visual Guide to Human Cells</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Allen Institute for Cell Science</a></li></a>
          </li>
          <li>
            <a href="github.com/AllenCellModeling/pytorch_integrated_cell" class="" target="_none">Pytorch 3D Integrated Cell</a> by <a href="https://www.alleninstitute.org/what-we-do/cell-science/about/team/" target="_none">Gregory R. Johnson, Rory M. Donovan-Maiye, Mary M. Maleckar</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41592-018-0111-2" class="" target="_none">Label-free prediction of three-dimensional fluorescence images from transmitted-light microscopy</a> by <a href="https://alleninstitute.org/about/team/" target="_none">Chawin Ounkomol, Sharmishtaa Seshamani, Mary M. Maleckar, Forrest Collman &amp; Gregory R. Johnson</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1101/2024.06.28.601071" class="" target="_none">Colony context and size-dependent compensation mechanisms give rise to variations in nuclear growth trajectories</a> by <a href="https://alleninstitute.org/about/team/" target="_none">Julie C. Dixon*, Christopher L. Frick*, Chantelle L. Leveille*, Philip Garrison, Peyton A. Lee, Saurabh S. Mogre, Benjamin Morris, Nivedita Nivedita, Ritvik Vasan, Jianxu Chen, Cameron L. Fraser, Clare R. Gamlin, Leigh K. Harris, Melissa C. Hendershott, Graham T. Johnson, Kyle N. Klein, Sandra A. Oluoch, Derek J. Thirstrup, M. Filip Sluzewski, Lyndsay Wilhelm, Ruian Yang, Daniel M. Toloudis, Matheus P. Viana, Julie A. Theriot &amp; Susanne M. Rafelski</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-cell-imaging-collections/#usageexamples">See 20 usage examples &rarr;</a></p>
        </div>
        <div id="sudachi" class="dataset">
          <h3><a href="/sudachi/">Sudachi Language Resources</a></h3>
          <p><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Japanese dictionaries and pre-trained models (word embeddings and language models) for natural language processing.
<a href="https://github.com/WorksApplications/SudachiDict">SudachiDict</a> is the dictionary for a Japanese tokenizer (morphological analyzer) <a href="https://github.com/WorksApplications/Sudachi">Sudachi</a>.
<a href="https://github.com/WorksApplications/chiVe">chiVe</a> is Japanese pretrained word embeddings (word vectors), trained using the ultra-large-scale web corpus NWJC by National...</p>
          <p><a href="/sudachi/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/WorksApplications/Sudachi" class="" target="_none">SudachiPy: Python version of Sudachi</a> by Works Applications</li></a>
          </li>
          <li>
            <a href="https://www.anlp.jp/proceedings/annual_meeting/2020/pdf_dir/P6-16.pdf" class="" target="_none">chiVe 2.0: SudachiとNWJCを用いた実用的な日本語単語ベクトルの実現に向けて</a> by 河村宗一郎, 久本空海, 真鍋陽俊, 髙岡一馬, 内田佳孝, 岡照晃, 浅原正幸</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/SudachiDict-full/" class="" target="_none">sudachidict_full on pypi.python.org - a Python module to download and install SudachiDict for the python tokenizer</a> by Works Applications</li></a>
          </li>
          <li>
            <a href="http://www.lrec-conf.org/proceedings/lrec2018/summaries/8884.html" class="" target="_none">Sudachi: a Japanese Tokenizer for Business</a> by Kazuma Takaoka, Sorami Hisamoto, Noriko Kawahara, Miho Sakamoto, Yoshitaka Uchida, Yuji Matsumoto</li></a>
          </li>
          <li>
            <a href="https://github.com/WorksApplications/elasticsearch-sudachi/blob/develop/docs/tutorial.md" class="" target="_none">analysis-sudachi Tutorial</a> by Works Applications</li></a>
          </li>
        </ul>
        <p><a href="/sudachi/#usageexamples">See 20 usage examples &rarr;</a></p>
        </div>
        <div id="czi-cellxgene-census" class="dataset">
          <h3><a href="/czi-cellxgene-census/">CZ CELLxGENE Discover Census</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">single-cell transcriptomics</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>CZ CELLxGENE Discover (<a href="https://cellxgene.cziscience.com/">cellxgene.cziscience.com</a>) is a free-to-use platform for the exploration, analysis, and retrieval of single-cell data. CZ CELLxGENE Discover hosts the largest aggregation of standardized single-cell data from the major human and mouse tissues, with modalities that include gene expression, chromatin accessibility, DNA methylation, and spatial transcriptomics.
This year, CZ CELLxGENE Discover has made available all of its human and mouse RNA single-cell data through Census (<a href="https://chanzuckerberg.github.io/cellxgene-census/">https://chanzuckerberg.github.io/cellxgene-census/</a>) – a free-to-use service with an API and data that...</p>
          <p><a href="/czi-cellxgene-census/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://chanzuckerberg.github.io/cellxgene-census/notebooks/api_demo/census_gget_demo.html" class="" target="_none">Querying data using the gget cellxgene module</a> by <a href="https://github.com/chanzuckerberg/" target="_none">CZI</a></li></a>
          </li>
          <li>
            <a href="https://chanzuckerberg.github.io/cellxgene-census/notebooks/api_demo/census_query_extract.html" class="" target="_none">Querying and fetching the single-cell data and cell/gene metadata.</a> by <a href="https://github.com/chanzuckerberg/" target="_none">CZI</a></li></a>
          </li>
          <li>
            <a href="https://chanzuckerberg.github.io/cellxgene-census/notebooks/api_demo/census_compute_over_X.html" class="" target="_none">Computing on the cell by gene matrix using online (incremental) algorithms</a> by <a href="https://github.com/chanzuckerberg/" target="_none">CZI</a></li></a>
          </li>
          <li>
            <a href="https://chanzuckerberg.github.io/cellxgene-census/notebooks/api_demo/census_dataset_presence.html" class="" target="_none">Genes measured in each cell (dataset presence matrix)</a> by <a href="https://github.com/chanzuckerberg/" target="_none">CZI</a></li></a>
          </li>
          <li>
            <a href="https://chanzuckerberg.github.io/cellxgene-census/notebooks/experimental/highly_variable_genes.html" class="" target="_none">Experimental Highly Variable Genes API</a> by <a href="https://github.com/chanzuckerberg/" target="_none">CZI</a></li></a>
          </li>
        </ul>
        <p><a href="/czi-cellxgene-census/#usageexamples">See 19 usage examples &rarr;</a></p>
        </div>
        <div id="kids-first" class="dataset">
          <h3><a href="/kids-first/">Gabriella Miller Kids First Pediatric Research Program (Kids First)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">pediatric</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">structural birth defect</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The NIH Common Fund&#39;s Gabriella Miller Kids First Pediatric Research Program’s (“Kids First”) vision is to “alleviate suffering from childhood cancer and structural birth defects by fostering collaborative research to uncover the etiology of these diseases and by supporting data sharing within the pediatric research community.” The program continues to generate and share whole genome sequence data from thousands of children affected by these conditions, ranging from rare pediatric cancers, such as osteosarcoma, to more prevalent diagnoses, such as congenital heart defects. In 2018, Kids Fi...</p>
          <p><a href="/kids-first/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/30930166" class="" target="_none">The Pediatric Cell Atlas: Defining the Growth Phase of Human Development at Single-Cell Resolution.</a> by Deanne M Taylor, Bruce J Aronow, et al.</li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31211835" class="" target="_none">Decreased ACKR3 (CXCR7) function causes oculomotor synkinesis in mice and humans.</a> by Mary C Whitman, Noriko Miyake, et al.</li></a>
          </li>
          <li>
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/31562520" class="" target="_none">Germline microsatellite genotypes differentiate children with medulloblastoma.</a> by Samuel Rivero-Hinojosa, Nicholas Kinney, et al.</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/32574564/" class="" target="_none">Genome-wide Enrichment of De Novo Coding Mutations in Orofacial Cleft Trios.</a> by Madison R Bishop, Kimberly K Diaz Perez, et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/kids-first/" class="" target="_none">Kids First DRC Source Code</a> by <a href="https://kidsfirstdrc.org" target="_none">Kids First DRC</a></li></a>
          </li>
        </ul>
        <p><a href="/kids-first/#usageexamples">See 19 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-power" class="dataset">
          <h3><a href="/nasa-power/">NASA Prediction of Worldwide Energy Resources (POWER)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">analytics</span><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">data assimilation</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">forecast</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">history</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">industry</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">machine translation</span><span class="label label-info tag link-tag">metadata</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">opendap</span><span class="label label-info tag link-tag">radiation</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">solar</span><span class="label label-info tag link-tag">statistics</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">time series forecasting</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>NASA&#39;s goal in Earth science is to observe, understand, and model the Earth system to discover how it is changing, to better predict change, and to understand the consequences for life on Earth. The Applied Sciences Program, within the Earth Science Division of the NASA Science Mission Directorate, serves individuals and organizations around the globe by expanding and accelerating societal and economic benefits derived from Earth science, information, and technology research and development.<br/><br/>
The <a href="https://power.larc.nasa.gov/">Prediction Of Worldwide Energy Resources (POWER)</a> Project, funded through the Applied Sciences Program at NASA Langley Research Center, gathers NASA Earth observation data and parameters related to the fields of surface solar irradiance and meteorology to serve the public in several free, easy-to-access and easy-to-use methods. POWER helps communities become resilient amid observed climate variability by improving data accessibility, aiding research in energy development, building energy efficiency, and supporting agriculture projects.<br/><br/>
The POWER project contains over 380 satellite-derived meteorology and solar energy Analysis Ready Data (ARD) at four temporal levels: hourly, daily, monthly, and climatology. The POWER data archive provides data at the native resolution of the source products. The data is updated nightly to maintain near real time availability (2-3 days for meteorological parameters and 5-7 days for solar). The POWER services catalog consists of a series of RESTful Application Programming Interfaces, geospatial enabled image services, and web mapping Data Access Viewer. These three service offerings support data discovery, access, and distribution to the project’s user base as ARD and as direct application inputs to decision support tools.<br/><br/>
The latest data version update includes hourly...</p>
          <p><a href="/nasa-power/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.jpsychires.2019.03.001" class="" target="_none">Association between solar insolation and a history of suicide attempts in bipolar I disorder</a> by Bauer M, et al., Stackhouse PW Jr., et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/nclimate3234" class="" target="_none">The Contribution of Solar Brightening to the US Maize Yield Trend</a> by Tollenaar, T., J. Fridgen, P. Tyagi, P. W. Stackhouse Jr., and S. Kumudini</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.2134/agronj2011.0038" class="" target="_none">Evaluation of Satellite-Based, Modeled-Derived Daily Solar Radiation Data for the Continental United States</a> by White, J. W., G. Hoogenboom, P. W. Wilkens, P. W. Stackhouse, and J. M. Hoel</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1175/BAMS-D-12-00169.1" class="" target="_none">Enhancing Climate Resilience at NASA Centers: A Collaboration between Science and Stewardship</a> by Rosenzweig, C., and Coauthors</li></a>
          </li>
          <li>
            <a href="https://power.larc.nasa.gov/api/pages/" class="" target="_none">POWER Application Programming Interface (API)</a> by <a href="https://power.larc.nasa.gov/" target="_none">The POWER Project</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-power/#usageexamples">See 18 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-goes" class="dataset">
          <h3><a href="/noaa-goes/">NOAA Geostationary Operational Environmental Satellites (GOES) 16, 17, 18 &amp; 19</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">weather</span></p>
          <p><br/>
<br/> 
NEW GOES-19 Data!!! GOES-19 products at the Provisional maturity level have begun streaming. Data files will be available between Provisional and the Operational Declaration of the satellite, however, these data are subject to the caveat that they are "GOES-19 Preliminary, Non-Operational Data".
<br/>
<br/>
NEW GOES 16 Reprocess Data!! The reprocessed GOES-16 ABI L1b data mitigates systematic data issues (including data gaps and image artifacts) seen in the Operational products, and improves the stability of both the radiometric and geometric calibration over the course of the entire mission life. These data were produced by recomputing the L1b radiance products from input raw L0 data using improved calibration algorithms and look-up tables, derived from data analysis of the NIST-traceable, on-board sources. In addition, the reprocessed data products contain enhancements to the L1b file format, including limb pixels and pixel timestamps, while maintaining compatibility with the operational products. The datasets currently available span the operational life of GOES-16 ABI, from early 2018 through the end of 2024. The Reprocessed L1b dataset shows improvement over the Operational L1b products but may still contain data gaps or discrepancies. Please provide feedback to Dan Lindsey (dan.lindsey@noaa.gov) and Gary Lin (guoqing.lin-1@nasa.gov).
<br/>
<br/>  
NOTICE: As of January 10th 2023, GOES-18 assumed the GOES-West position and all data files are deemed both operational and provisional, so no ‘preliminary, non-operational’ caveat is needed. GOES-17 is now offline, shifted approximately 105 degree West, where it will be in on-orbit storage. GOES-17 data will no longer flow into the GOES-17 bucket. Operational GOES-West products can be found in the GOES-18 bucket. 
<br/>
<br/> 
GOES satellites (GOES-16, GOES-17, GOES-18 & GOES-19) provide continuous weather imagery and
monitoring of meteorological and space environment data across North America.
GOES satellites provide the kind of continuous monitoring necessary for
intensive data analysis. They hover continuously ove</br/>...</p>
          <p><a href="/noaa-goes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/embracing-the-cloud-for-climate-research/" class="aws-link" target="_none">Embracing the cloud for climate research</a> by North Carolina State University’s North Carolina Institute for Climate Studies</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/solcast-solar-irradiance-forecasting-for-the-solar-powered-future/" class="aws-link" target="_none">Solar irradiance forecasting for the solar powered future</a> by Solcast</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1175/WAF-D-19-0141.1" class="" target="_none">Comparison of Lightning Forecasts from the High-Resolution Rapid Refresh Model to Geostationary Lightning Mapper Observations</a> by Brian K. Blaylock, and John D. Horel</li></a>
          </li>
          <li>
            <a href="https://noaa-goes16.s3.amazonaws.com/Beginners_Guide_to_GOES-R_Series_Data.pdf" class="" target="_none">Beginner’s Guide to GOES-R Series Data</a> by Danielle Losos</li></a>
          </li>
          <li>
            <a href="https://github.com/M00NSH0T/Hurricanes/tree/master/2021%20update" class="" target="_none">Forecasting Hurricane Tracks with TensorFlow and data from AWS S3</a> by Kyle Archie</li></a>
          </li>
        </ul>
        <p><a href="/noaa-goes/#usageexamples">See 18 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel-2-l2a-cogs" class="dataset">
          <h3><a href="/sentinel-2-l2a-cogs/">Sentinel-2 Cloud-Optimized GeoTIFFs</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The <a href="https://sentinel.esa.int/web/sentinel/missions/sentinel-2">Sentinel-2 mission</a> is
a land monitoring constellation of two satellites that provide high resolution
optical imagery and provide continuity for the current SPOT and Landsat missions.
The mission provides a global coverage of the Earth&#39;s land surface every 5 days,
making the data of great use in ongoing studies.
This dataset is the same as the <a href="https://registry.opendata.aws/sentinel-2/">Sentinel-2</a>
dataset, except the JP2K files were converted into Cloud-Optimized GeoTIFFs (COGs).
Additionally, SpatioTemporal Asset Catalog metadata has were in a JSON file
alongside the data, and a STAC API called <a href="https://earth-search.aws.element84.com/v1">Earth-search</a>
is freely available t...</p>
          <p><a href="/sentinel-2-l2a-cogs/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/intake/intake-stac/blob/master/examples/aws-earth-search.ipynb" class="" target="_none">Intake-STAC with sat-search</a> by Scott Henderson</li></a>
          </li>
          <li>
            <a href="https://stacindex.org/" class="" target="_none">SpatioTemporal Asset Catalogs</a> by STAC Contriubutors</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/monitor-water-hyacinth-invasion" class="aws-link" target="_none">Using Sentinel-2 Data from the AWS Open Data Project to measure water hyacinth growth</a> by Louise Liddell</li></a>
          </li>
          <li>
            <a href="https://github.com/developmentseed/titiler/blob/master/docs/examples/notebooks/Create_CustomSentinel2Tiler.ipynb" class="" target="_none">Create a Dynamic Tiler with TiTiler</a> by Vincent Sarago</li></a>
          </li>
          <li>
            <a href="https://www.linkedin.com/pulse/how-sunny-my-city-better-process-stac-sentinel2-lambda-alvaro-huarte" class="" target="_none">How to process Sentinel-2 data in a serverless Lambda on AWS?</a> by Alvaro Huarte</li></a>
          </li>
        </ul>
        <p><a href="/sentinel-2-l2a-cogs/#usageexamples">See 18 usage examples &rarr;</a></p>
        </div>
        <div id="cellpainting-gallery" class="dataset">
          <h3><a href="/cellpainting-gallery/">Cell Painting Gallery</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">cell painting</span><span class="label label-info tag link-tag">chemical biology</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">high-throughput imaging</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">image-based profiling</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">microscopy</span><span class="label label-info tag link-tag">organelle</span></p>
          <p>The Cell Painting Gallery is a collection of image datasets created using the <a href="https://pubmed.ncbi.nlm.nih.gov/27560178/">Cell Painting</a> assay. 
The images of cells are captured by microscopy imaging, and reveal the response of various labeled cell components to whatever treatments are tested, which can include genetic perturbations, chemicals or drugs, or different cell types. 
The datasets can be used for diverse applications in basic biology and pharmaceutical research, such as identifying disease-associated phenotypes, understanding disease mechanisms, and predicting a drug’s activity, toxicity, or mechanism of action (<a href="https://carpenter-singh-lab.broadinstitute.org/files/anne/files/141_Chandrasekaran_NatRevDrugDiscov_2020.pdf">Chandrasekaran et al 2020</a>). 
This collection is maintained by the <a href="https://carpenter-singh-lab.broadinstitute.org/">Carpenter–Singh lab</a> and the <a href="https://cimini-lab.broadinstitute.org/">Cimini lab</a> at the <a href="https://www.broadinstitute.org/">Broad Institute</a>.
A human-friendly listing of datasets, instructions for accessing them, and other documentation is at the <a href="https://github.com/broadinstitute/cellpainting-gallery">corresponding GitHub page</a> abou...</p>
          <p><a href="/cellpainting-gallery/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://forum.image.sc" class="" target="_none">Scientific Community Image Forum - for software-oriented aspects of scientific imaging including analysis, processing, and acquisition</a> by <a href="https://openbioimageanalysis.org" target="_none">Multiple Authors</a></li></a>
          </li>
          <li>
            <a href="https://github.com/CellProfiler/Distributed-CellProfiler" class="" target="_none">Distributed CellProfiler - Run encapsulated docker containers with CellProfiler in the Amazon Web Services infrastructure</a> by <a href="https://github.com/CellProfiler/Distributed-CellProfiler/graphs/contributors" target="_none">Multiple Authors</a></li></a>
          </li>
          <li>
            <a href="https://www.youtube.com/c/COBACenterforOpenBioimageAnalysis/videos" class="" target="_none">Center for Open Bioimage Analysis (COBA) YouTube Channel - video tutorials of CellProfiler and other softwares</a> by <a href="https://openbioimageanalysis.org" target="_none">Multiple Authors</a></li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/27560178/" class="" target="_none">Cell Painting, a high-content image-based assay for morphological profiling using multiplexed fluorescent dyes</a> by Bray M-A, Singh S, Han H, Davis CT, Borgeson B, Hartland C, Kost-Alimova M, Gustafsdottir SM, Gibson CC, &amp; Carpenter AE</li></a>
          </li>
          <li>
            <a href="http://journals.plos.org/plosone/article?id&#x3D;10.1371/journal.pone.0131370" class="" target="_none">Morphological Profiles of RNAi-Induced Gene Knockdown Are Highly Reproducible but Dominated by Seed Effects</a> by Singh S, Wu X, Ljosa V, Bray M-A, Piccioni F, Root DE, Doench JG, Boehm JS, &amp; Carpenter AE</li></a>
          </li>
        </ul>
        <p><a href="/cellpainting-gallery/#usageexamples">See 17 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-nexrad" class="dataset">
          <h3><a href="/noaa-nexrad/">NEXRAD on AWS</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Real-time and archival data from the Next Generation Weather Radar (NEXRAD) network.</p>
          <p><a href="/noaa-nexrad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.pnas.org/content/117/6/2987" class="" target="_none">Declines in an abundant aquatic insect, the burrowing mayfly, across major North American waterways</a> by Phillip M. Stepanian, Sally A. Entrekin, Charlotte E. Wainwright, Djordje Mirkovic, Jennifer L. Tank, &amp; Jeffrey F. Kelly</li></a>
          </li>
          <li>
            <a href="http://nbviewer.jupyter.org/gist/dopplershift/356f2e14832e9b676207" class="" target="_none">Using Python to Access NCEI Archived NEXRAD Level 2 Data (Jupyter notebook)</a> by <a href="http://dopplershift.github.io" target="_none">Ryan May</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/aws-nexrad-smsl-notebook/" class="aws-link" target="_none">Updated introduction to S3, Boto, and NOAA Nexrad in SageMaker Studio Lab (SMSL)</a> by Chris Stoner</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/nexradaws/" class="" target="_none">nexradaws on pypi.python.org - python module to query and download Nexrad data from Amazon S3</a> by <a href="https://github.com/aarande" target="_none">Aaron Anderson</a></li></a>
          </li>
          <li>
            <a href="https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2020GL089001" class="" target="_none">Extreme Pyroconvective Updrafts During a Megafire</a> by B. Rodriguez, N. P. Lareau, D. E. Kingsmill, &amp; C. B. Clements</li></a>
          </li>
        </ul>
        <p><a href="/noaa-nexrad/#usageexamples">See 17 usage examples &rarr;</a></p>
        </div>
        <div id="terrain-tiles" class="dataset">
          <h3><a href="/terrain-tiles/">Terrain Tiles</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span></p>
          <p>A global dataset providing bare-earth terrain heights, tiled for easy usage and provided on S3.</p>
          <p><a href="/terrain-tiles/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://onthegomap.com/" class="" target="_none">On The Go Map</a> by <a href="https://onthegomap.com/" target="_none">On The Go Map</a></li></a>
          </li>
          <li>
            <a href="https://terradactile.sparkgeo.com/" class="" target="_none">Terradactile: Generate DEM and hillshade Cloud Optimized GeoTIFFs for visualization and download based on terrain tiles</a> by <a href="https://sparkgeo.com/" target="_none">Sparkgeo</a></li></a>
          </li>
          <li>
            <a href="https://shademap.app/" class="" target="_none">ShadeMap: Simulate sun shadows for any time and place on Earth</a> by <a href="https://github.com/ted-piotrowski" target="_none">Ted Piotrowski</a></li></a>
          </li>
          <li>
            <a href="https://github.com/interline-io/planetutils" class="" target="_none">PlanetUtils (GitHub): Scripts and a Docker container to download, merge, and resample terrain tiles</a> by <a href="https://www.interline.io/" target="_none">Interline Technologies</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41598-021-94045-3" class="" target="_none">Landscape transformations produce favorable roosting conditions for turkey vultures and black vultures</a> by Jacob E. Hill, Kenneth F. Kellner, Bryan M. Kluever, Michael L. Avery, John S. Humphrey, Eric A. Tillman, Travis L. DeVault &amp; Jerrold L. Belant</li></a>
          </li>
        </ul>
        <p><a href="/terrain-tiles/#usageexamples">See 17 usage examples &rarr;</a></p>
        </div>
        <div id="its-live-data" class="dataset">
          <h3><a href="/its-live-data/">Inter-mission Time Series of Land Ice Velocity and Elevation (ITS_LIVE)</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geophysics</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">ice</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>The Inter-mission Time Series of Land Ice Velocity and Elevation (ITS_LIVE) project has a singular mission: to accelerate ice sheet and glacier research by producing globally comprehensive, high resolution, low latency, temporally dense, multi-sensor records of land ice and ice shelf change while minimizing barriers between the data and the user.
ITS_LIVE data currently consists of NetCDF Level 2 scene-pair ice flow products posted to a standard 120 m grid derived from Landsat 4/5/7/8/9, Sentinel-2 optical scenes, and Sentinel-1 SAR scenes. We have processed all land-ice intersecting image pai...</p>
          <p><a href="/its-live-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/nasa-jpl/itslive-py" class="" target="_none">ITS_LIVEpy: A Python client for ITS_LIVE glacier velocity data</a> by <a href="https://its-live.jpl.nasa.gov/#people" target="_none">Luis A. López and Mark Fahnestock</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41586-022-05037-w" class="" target="_none">Antarctic calving loss rivals ice-shelf thinning</a> by Greene, C.A., A.S. Gardner, N.-J. Schlegel, and A.D. Fraser</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/essd-14-3573-2022" class="" target="_none">Elevation change of the Antarctic Ice Sheet: 1985 to 2020</a> by Nilsson, J., A.S. Gardner, and F.S. Paolo</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/tc-14-4365-2020" class="" target="_none">Detecting seasonal ice dynamics in satellite images</a> by Greene, C.A., A.S. Gardner, and L.C. Andrews</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/essd-14-5111-2022" class="" target="_none">Processing methodology for the ITS_LIVE Sentinel-1 ice velocity products</a> by Lei, Y., A.S. Gardner, and P. Agram</li></a>
          </li>
        </ul>
        <p><a href="/its-live-data/#usageexamples">See 16 usage examples &rarr;</a></p>
        </div>
        <div id="esa-worldcover-vito" class="dataset">
          <h3><a href="/esa-worldcover-vito/">ESA WorldCover</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">land use</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>The European Space Agency (ESA) WorldCover product provides global land cover maps for 2020 &amp; 2021 at 10 m resolution based on Copernicus Sentinel-1 and Sentinel-2 data. The WorldCover product comes with 11 land cover classes and has been generated in the framework of the ESA WorldCover project, part of the 5th Earth Observation Envelope Programme (EOEP-5) of the European Space Agency. A first version of the product (v100), containing the 2020 map was released in October 2021. The 2021 map was released in October 2022 using an improved algorithm (v200). The WorldCover 2020 and 2021 maps we...</p>
          <p><a href="/esa-worldcover-vito/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.5281/zenodo.5571936" class="" target="_none">ESA WorldCover 10 m 2020 v100</a> by Zanaga, D., Van De Kerchove, R., De Keersmaecker, W., Souverijns, N., Brockmann, C., Quast, R., Wevers, J., Grosu, A., Paccini, A., Vergnaud, S., Cartus, O., Santoro, M., Fritz, S., Georgieva, I., Lesiv, M., Carter, S., Herold, M., Li, Linlin, Tsendbazar, N.E., Ramoino, F., Arino, O., 2021.</li></a>
          </li>
          <li>
            <a href="https://blog.vito.be/remotesensing/release-of-the-10-m-worldcover-map" class="" target="_none">Release of the 10 m WorldCover map</a> by Ruben Van De Kerchove</li></a>
          </li>
          <li>
            <a href="https://esa-worldcover.s3.eu-central-1.amazonaws.com/v100/2020/docs/WorldCover_PUM_V1.0.pdf" class="" target="_none">ESA WorldCover 10 m 2020 v100 - Product User Manual</a> by VITO</li></a>
          </li>
          <li>
            <a href="https://github.com/SatelliteVu/SatelliteVu-AWS-Disaster-Response-Hackathon" class="" target="_none">Global Fire Spread Prediction System</a> by <a href="https://www.satellitevu.com/" target="_none">SatelliteVu</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.jag.2022.103108" class="" target="_none">Fusing GEDI with earth observation data for large area aboveground biomass mapping</a> by Yuri Shendryk</li></a>
          </li>
        </ul>
        <p><a href="/esa-worldcover-vito/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="broad-gnomad" class="dataset">
          <h3><a href="/broad-gnomad/">Genome Aggregation Database (gnomAD)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">population genetics</span><span class="label label-info tag link-tag">short read sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators that aggregates and harmonizes both exome and genome data from a wide range of large-scale human sequencing projects. The summary data provided here are released for the benefit of the wider scientific community without restriction on use.
The v4.1 data set (GRCh38) spans 730,947 exome sequences and 76,215 whole-genome sequences from unrelated individuals, of <a href="https://gnomad.broadinstitute.org/stats#diversity">diverse ancestries</a>, sequenced sequenced as part of various disease-specific and population genetic studies.
The gnomAD Principal Investigators and team can be found <a href="https://gnomad.broadinstitute.org/team">here</a>, and the groups that have contributed data to the current release are listed <a href="https://gnomad.broadinstitute.org/about">here</a>.
Sign up for the gnom...</p>
          <p><a href="/broad-gnomad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1101/784157" class="" target="_none">Technical artifact drives apparent deviation from Hardy-Weinberg equilibrium at CCR5-∆32 and other variants in gnomAD. bioRxiv (p. 784157)</a> by Karczewski, K. J., Gauthier, L. D., Daly, M. J.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41586-020-2287-8" class="" target="_none">A structural variation reference for medical and population genetics. Nature 581, 444–451 (2020)</a> by Collins, R. L., Brand, H., Karczewski, K. J., Zhao, X., Alföldi, J., Francioli, L. C., Khera, A. V., Lowther, C., Gauthier, L. D., Wang, H., Watts, N. A., Solomonson, M., O’Donnell-Luria, A., Baumann, A., Munshi, R., Walker, M., Whelan, C., Huang, Y., Brookings, T., ... Talkowski, M. E.</li></a>
          </li>
          <li>
            <a href="https://gnomad.broadinstitute.org/blog/2019-10-gnomad-v3-0/" class="" target="_none">gnomAD v3.0</a> by Laurent Francioli, Daniel MacArthur</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41586-020-2329-2" class="" target="_none">Transcript expression-aware annotation improves rare variant interpretation. Nature 581, 452–458 (2020)</a> by Cummings, B. B., Karczewski, K. J., Kosmicki, J. A., Seaby, E. G., Watts, N. A., Singer-Berk, M., Mudge, J. M., Karjalainen, J., Kyle Satterstrom, F., O’Donnell-Luria, A., Poterba, T., Seed, C., Solomonson, M., Alföldi, J., The Genome Aggregation Database Production Team, The Genome Aggregation Database Consortium, Daly, M. J., &amp; MacArthur, D. G.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41467-019-12438-5" class="" target="_none">Landscape of multi-nucleotide variants in 125,748 human exomes and 15,708 genomes. Nature Communications 11, 2539 (2020)</a> by Wang, Q., Pierce-Hoffman, E., Cummings, B. B., Karczewski, K. J., Alföldi, J., Francioli, L. C., Gauthier, L. D., Hill, A. J., O’Donnell-Luria, A. H., Genome Aggregation Database (gnomAD) Production Team, Genome Aggregation Database (gnomAD) Consortium, &amp; MacArthur, D. G.</li></a>
          </li>
        </ul>
        <p><a href="/broad-gnomad/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="geonet" class="dataset">
          <h3><a href="/geonet/">GeoNet Aotearoa New Zealand Data</a></h3>
          <p><span class="label label-info tag link-tag">broadband</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">Continuously Operating Reference Station (CORS)</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">geophysics</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">GNSS</span><span class="label label-info tag link-tag">GPS</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">RINEX</span><span class="label label-info tag link-tag">seismology</span></p>
          <p>GeoNet provides geological hazard information for Aotearoa New Zealand. This dataset contains data and products recorded by the GeoNet sensor network. <br /> <br /> GNSS (Global Navigation Satellite System) data include raw data in proprietary and Receiver Independent Exchange Format (RINEX) and local tie-in survey conducted during equipment changes, more details can be found on <a href="https://www.geonet.org.nz/data/types/geodetic">the GeoNet geodetic page</a> website. <br /> Coastal gauge data include relative measurement of sea level measured by tsunami monitoring gauges. Raw and quality control data are provided in CREX format (Character Form for the Representtion and eXchange of metereological data), more details can be found on <a href="https://www.geonet.org.nz/data/types/tidal_gauges">the GeoNet coastal tsunami monitoring gauges page</a>. <br /> Camera images data include webcam images from the GeoNet Volcano monitoring network and Built Environment Instrumentation Programme, more details can be found on <a href="https://www.geonet.org.nz/data/types/camera">the GeoNet camera page</a>. <br /> Waveform data include raw data from weak and strong motion instruments of the GeoNet seismic networks, more details can be found on <a href="https://www.geonet.org.nz/data/types/seismic_waveforms">the GeoNet seismic waveform page</a>. <br /> Seismic data products include strong motion derived data, more details can be found on <a href="https://strongmotion.geonet.org.nz">the GeoNet Strong Motion products page</a>. <br /> Time Series data products include derived time...</p>
          <p><a href="/geonet/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.geonet.org.nz/data/access/aws" class="" target="_none">GeoNet datasets available via AWS Open Data Programme</a> by GNS Science - Te Pū Ao</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1785/gssrl.82.1.9" class="" target="_none">The New Zealand National Seismograph Network</a> by T. Petersen, K. Gledhill, M. Chadwick, N. Gale, J. Ristau</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.21420/EN0F-XY29" class="" target="_none">GeoNet Automatically Collected Volcano Data</a> by GNS Science - Te Pū Ao</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.21420/NQYC-EG34" class="" target="_none">Available GeoNet GNSS event high rate data</a> by GNS Science - Te Pū Ao</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.21420/TMHV-0T46" class="" target="_none">GeoNet continuous GNSS network RAW files</a> by GNS Science - Te Pū Ao</li></a>
          </li>
        </ul>
        <p><a href="/geonet/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="spacenet" class="dataset">
          <h3><a href="/spacenet/">SpaceNet</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>SpaceNet, launched in August 2016 as an open innovation project offering a repository of freely available
imagery with co-registered map features. Before SpaceNet, computer vision researchers had minimal options
to obtain free, precision-labeled, and high-resolution satellite imagery. Today, SpaceNet hosts datasets
developed by its own team, along with data sets from projects like IARPA’s Functional Map of the World (fMoW).</p>
          <p><a href="/spacenet/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/the-downlinq/spacenet-winning-implementations-and-new-imagery-release-55f738e14253" class="" target="_none">SpaceNet: Winning Implementations and New Imagery Release</a> by <a href="https://medium.com/@toddstavish" target="_none">Todd Stavish</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/the-downlinq/deploying-the-spacenet-6-baseline-on-aws-c811ad82da1" class="" target="_none">Deploying the SpaceNet 6 Baseline on AWS</a> by <a href="https://medium.com/@avanetten" target="_none">Adam Van Etten and Nick Weir</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/the-downlinq/creating-training-datasets-for-the-spacenet-road-detection-and-routing-challenge-6f970d413e2f" class="" target="_none">Creating Training Datasets for the SpaceNet Road Detection and Routing Challenge</a> by <a href="https://medium.com/@avanetten" target="_none">Adam Van Etten and Jake Shermeyer</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/the-downlinq/introducing-the-spacenet-off-nadir-imagery-and-buildings-dataset-e4a3c1cb4ce3" class="" target="_none">Introducing the SpaceNet Off-Nadir Imagery Dataset</a> by <a href="https://medium.com/@david.lindenbaum" target="_none">David Lindenbaum</a></li></a>
          </li>
          <li>
            <a href="https://openaccess.thecvf.com/content/CVPR2022W/EarthVision/papers/Hansch_SpaceNet_8_-_The_Detection_of_Flooded_Roads_and_Buildings_CVPRW_2022_paper.pdf" class="" target="_none">SpaceNet 8 - The Detection of Flooded Roads and Buildings</a> by Ronny Hansch, Jacob Arndt, Dalton Lunga, Matthew Gibb, Tyler Pedelose, Arnold Boedihardjo, Desiree Petrie, Todd M. Bacastow</li></a>
          </li>
        </ul>
        <p><a href="/spacenet/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="sgnex" class="dataset">
          <h3><a href="/sgnex/">The Singapore Nanopore Expression Data Set</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fasta</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">long read sequencing</span><span class="label label-info tag link-tag">short read sequencing</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Singapore Nanopore Expression (SG-NEx) project is an international collaboration to generate reference transcriptomes and a comprehensive benchmark data set for long read Nanopore RNA-Seq. Transcriptome profiling is done using PCR-cDNA sequencing (PCR-cDNA), amplification-free cDNA sequencing (direct cDNA), direct sequencing of native RNA (direct RNA), and short read RNA-Seq. The SG-NEx core data includes 5 of the most commonly used cell lines and it is extended with additional cell lines and samples that cover a broad range of human tissues. All core samples are sequenced with at least 3 ...</p>
          <p><a href="/sgnex/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41587-021-00949-w" class="" target="_none">Identification of differential RNA modifications from nanopore direct RNA sequencing with xPore.</a> by Ploy N Pratanwanich et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/GoekeLab/xpore" class="" target="_none">xPore: Identification of differential RNA modification from direct RNA-Seq data</a> by Ploy Pratanwanich et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1101/2021.09.20.461055" class="" target="_none">Detection of m6A from direct RNA sequencing using a Multiple Instance Learning framework.</a> by Christopher Hendra et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/Oshlack/JAFFA/wiki" class="" target="_none">JAFFAL: Detection of fusion genes from long read RNA-Seq data</a> by Nadia M Davidson et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/GoekeLab/bambu" class="" target="_none">Bambu: Transcript discovery and quantification using long read RNA-Seq data</a> by Ying Chen, Andre Sim et al.</li></a>
          </li>
        </ul>
        <p><a href="/sgnex/#usageexamples">See 15 usage examples &rarr;</a></p>
        </div>
        <div id="amazon-last-mile-challenges" class="dataset">
          <h3><a href="/amazon-last-mile-challenges/">2021 Amazon Last Mile Routing Research Challenge Dataset</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">analytics</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">last mile</span><span class="label label-info tag link-tag">logistics</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">optimization</span><span class="label label-info tag link-tag">routing</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">urban</span></p>
          <p>The 2021 Amazon Last Mile Routing Research Challenge was an innovative research initiative led by Amazon.com and supported by the Massachusetts Institute of Technology’s Center for Transportation and Logistics. Over a period of 4 months, participants were challenged to develop innovative machine learning-based methods to enhance classic optimization-based approaches to solve the travelling salesperson problem, by learning from historical routes executed by Amazon delivery drivers. The primary goal of the Amazon Last Mile Routing Research Challenge was to foster innovative applied research in r...</p>
          <p><a href="/amazon-last-mile-challenges/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2307.07357" class="" target="_none">Inverse Optimization for Routing Problems</a> by Pedro Zattoni Scroccaro, Piet van Beek, Peyman Mohajerin Esfahani, Bilge Atasoy</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/amazon-sagemaker-amazon-routing-challenge-sol" class="aws-link" target="_none">AWS Last Mile Route Sequence Optimization</a> by Chen Wu, Yin Song, Verdi March, Eden Duthi</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.ejor.2023.02.031" class="" target="_none">Planning robust drone-truck delivery routes under road traffic uncertainty</a> by Yu Yang, Chiwei Yan, Yufeng Cao, Roberto Roberti</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1287/trsc.2022.0323" class="" target="_none">Crowdkeeping in Last-Mile Delivery</a> by Xin Wang , Okan Arslan , Erick Delage</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1287/trsc.2022.1185" class="" target="_none">Constrained Local Search for Last-Mile Routing</a> by William Cook, Stephan Held, Keld Helsgaun</li></a>
          </li>
        </ul>
        <p><a href="/amazon-last-mile-challenges/#usageexamples">See 17 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-jpss" class="dataset">
          <h3><a href="/noaa-jpss/">NOAA Joint Polar Satellite System (JPSS)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p><em>Near Real Time JPSS data is now flowing! See bucket information on the right side of this page to access products!</em> <br /> Satellites in the JPSS constellation gather global measurements of atmospheric, terrestrial and oceanic conditions, including sea and land surface temperatures, vegetation, clouds, rainfall, snow and ice cover, fire locations and smoke plumes, atmospheric temperature, water vapor and ozone. JPSS delivers key observations for the Nation&#39;s essential products and services, including forecasting severe weather like hurricanes, tornadoes and blizzards days in advance, and assessin...</p>
          <p><a href="/noaa-jpss/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.jpss.noaa.gov/training_resources.html" class="" target="_none">JPSS Training Resources</a> by NOAA</li></a>
          </li>
          <li>
            <a href="https://www.jpss.noaa.gov/assets/pdfs/science_publications/2020_science_seminar_digest.pdf" class="" target="_none">JPSS Science Seminar Annual Digest 2020</a> by NOAA</li></a>
          </li>
          <li>
            <a href="https://cimss.ssec.wisc.edu/satellite-blog/archives/59235" class="" target="_none">NUCAPS EDR files in the cloud</a> by Scott Lindstrom</li></a>
          </li>
          <li>
            <a href="https://gitlab.cicsnc.org/workshop-development/jpss_aws" class="" target="_none">JPSS_AWS Geolocate</a> by Mya Sears</li></a>
          </li>
          <li>
            <a href="http://rammb.cira.colostate.edu/training/visit/links_and_tutorials/ams_austin_tx_JPSS_workshop_2018.asp" class="" target="_none">JPSS Short Course from the 2018 Annual Meeting of American Meteorological Society</a> by Colorado State University</li></a>
          </li>
        </ul>
        <p><a href="/noaa-jpss/#usageexamples">See 14 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-mangrove" class="dataset">
          <h3><a href="/deafrica-mangrove/">Digital Earth Africa Global Mangrove Watch</a></h3>
          <p><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The Global Mangrove Watch (GMW) dataset is a result of the collaboration between Aberystwyth University (U.K.), solo Earth Observation (soloEO; Japan), Wetlands International the World Conservation Monitoring Centre (UNEP-WCMC) and the Japan Aerospace Exploration Agency (JAXA). The primary objective of producing this dataset is to provide countries lacking a national mangrove monitoring system with first cut mangrove extent and change maps, to help safeguard against further mangrove forest loss and degradation.
The Global Mangrove Watch dataset (version 2) consists of a global baseline map of ...</p>
          <p><a href="/deafrica-mangrove/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.aboutamazon.com/news/aws/climate-next-how-data-and-community-can-save-zanzibars-mangroves" class="" target="_none">Climate Next: How data and community can save Zanzibar’s mangroves</a> by Amazon Staff</li></a>
          </li>
          <li>
            <a href="https://maps.digitalearth.africa/" class="" target="_none">Digital Earth Africa Map</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://docs.digitalearthafrica.org/en/latest/sandbox/notebooks/Datasets/Global_Mangrove_Watch.html" class="" target="_none">Digital Earth Africa Global Mangrove Watch Notebook</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://docs.digitalearthafrica.org/en/latest/sandbox/notebooks/Real_world_examples/Mangrove_analysis.html" class="" target="_none">Digital Earth Africa Monitoring Mangrove Extents Notebook</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://sandbox.digitalearth.africa/" class="" target="_none">Digital Earth Africa Sandbox</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-mangrove/#usageexamples">See 13 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-landsat" class="dataset">
          <h3><a href="/deafrica-landsat/">Digital Earth Africa Landsat Collection 2 Level 2</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Digital Earth Africa (DE Africa) provides free and open access to a copy of Landsat Collection 2 Level-2 products over Africa. These products are produced and provided by the United States Geological Survey (USGS).
The Landsat series of Earth Observation satellites, jointly led by USGS and NASA, have been continuously acquiring images of the Earth’s land surface since 1972. DE Africa provides data from Landsat 5, 7 and 8 satellites, including historical observations dating back to late 1980s and regularly updated new acquisitions.
New Level-2 Landsat 7 and Landsat 8 data are available after 15...</p>
          <p><a href="/deafrica-landsat/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://explorer.digitalearth.africa/products/ls7_sr" class="" target="_none">Digital Earth Africa Explorer (LS7 Surface Reflectance)</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="http://learn.digitalearthafrica.org/" class="" target="_none">Digital Earth Africa Training</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/ls7_st" class="" target="_none">Digital Earth Africa Explorer (LS7 Surface Temperature)</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/ls8_sr" class="" target="_none">Digital Earth Africa Explorer (LS8 Surface Reflectance)</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/ls5_st" class="" target="_none">Digital Earth Africa Explorer (LS5 Surface Temperature)</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-landsat/#usageexamples">See 13 usage examples &rarr;</a></p>
        </div>
        <div id="janelia-flylight" class="dataset">
          <h3><a href="/janelia-flylight/">Fly Brain Anatomy: FlyLight Gen1 and Split-GAL4 Imagery</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This data set, made available by Janelia&#39;s FlyLight project, consists of fluorescence images 
of Drosophila melanogaster driver lines, aligned to standard templates, and stored in formats 
suitable for rapid searching in the cloud. Additional data will be added as it is published. </p>
          <p><a href="/janelia-flylight/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/JaneliaSciComp/open-data-flylight/blob/master/tutorials/File_operations_on_AWS_S3.ipynb" class="" target="_none">File Operations on AWS S3</a> by Rob Svirskas</li></a>
          </li>
          <li>
            <a href="http://natverse.org/neuronbridger/" class="" target="_none">Tutorial for neuronbridger (R API)</a> by Alexander Bates</li></a>
          </li>
          <li>
            <a href="https://neuronbridge.janelia.org" class="" target="_none">NeuronBridge</a> by Jody Clements, Rob Svirskas, Hideo Otsuna, Cristian Goina, Konrad Rokicki</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.7554/eLife.04577" class="" target="_none">The neuronal architecture of the mushroom body provides a logic for associative learning</a> by Yoshinori Aso, Daisuke Hattori, Yang Yu, Rebecca M Johnston, Nirmala A Iyer, Teri-TB Ngo, Heather Dionne, LF Abbott, Richard Axel, Hiromu Tanimoto, Gerald M Rubin</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1101/2020.05.29.080473" class="" target="_none">An image resource of subdivided Drosophila GAL4-driver expression patterns for neuron-level searches</a> by Geoffrey W Meissner, Zachary Dorman, Aljoscha Nern, Kaitlyn Forster, Theresa Gibney, Jennifer Jeter, Lauren Johnson, Yisheng He, Kelley Lee, Brian Melton, Brianna Yarbrough, Jody Clements, Cristian Goina, Hideo Otsuna, Konrad Rokicki, Robert R Svirskas, Yoshinori Aso, Gwyneth M Card, Barry J Dickson, Erica Ehrhardt, Jens Goldammer, Masayoshi Ito, Wyatt Korff, Ryo Minegishi, Shigehiro Namiki, Gerald M Rubin, Gabriella Sterne, Tanya Wolff, Oz Malkesman, FlyLight Project Team</li></a>
          </li>
        </ul>
        <p><a href="/janelia-flylight/#usageexamples">See 13 usage examples &rarr;</a></p>
        </div>
        <div id="radarsat-1" class="dataset">
          <h3><a href="/radarsat-1/">RADARSAT-1</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">ice</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>Developed and operated by the Canadian Space Agency, it is Canada&#39;s first commercial Earth observation satellite 
Développé et exploité par l&#39;Agence spatiale canadienne, il s&#39;agit du premier satellite commercial d&#39;observation de la Terre au Canada.</p>
          <p><a href="/radarsat-1/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://catalyst.earth/products/catalyst-pro/" class="" target="_none">Catalyst Professional</a> by <a href="https://catalyst.earth" target="_none">CATALYST</a></li></a>
          </li>
          <li>
            <a href="https://gdal.org" class="" target="_none">GDAL</a> by <a href="https://gdal.org" target="_none">GDAL</a></li></a>
          </li>
          <li>
            <a href="https://github.com/nrcan-eodms-sgdot-rncan/eodms-rapi-orderdownload" class="" target="_none">EODMS RAPI Client Python Script</a> by <a href="https://www.eodms-sgdot.nrcan-rncan.gc.ca/index-en.html" target="_none">Earth Observation Data Management System - Natural Resources Canada</a></li></a>
          </li>
          <li>
            <a href="https://www.asf.alaska.edu/data-tools/mapready" class="" target="_none">MapReady</a> by <a href="https://www.asf.alaska.edu" target="_none">NASA</a></li></a>
          </li>
          <li>
            <a href="https://www.gamma-rs.ch/" class="" target="_none">Gamma</a> by <a href="https://www.gamma-rs.ch/" target="_none">Gamme Remote Sensing</a></li></a>
          </li>
        </ul>
        <p><a href="/radarsat-1/#usageexamples">See 12 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-chirps" class="dataset">
          <h3><a href="/deafrica-chirps/">Digital Earth Africa CHIRPS Rainfall</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Digital Earth Africa (DE Africa) provides free and open access to a copy of the Climate Hazards Group InfraRed Precipitation with Station data (CHIRPS) monthly and daily products over Africa. The CHIRPS rainfall maps are produced and provided by the Climate Hazards Center in collaboration with the US Geological Survey, and use both rain gauge and satellite observations.
The CHIRPS-2.0 Africa Monthly dataset is regularly indexed to DE Africa from the CHIRPS monthly data. The CHIRPS-2.0 Africa Daily dataset is likewise indexed from the CHIRPS daily data. Both products have been converted to clou...</p>
          <p><a href="/deafrica-chirps/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://maps.digitalearth.africa/" class="" target="_none">Digital Earth Africa Map</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://ows.digitalearth.africa" class="" target="_none">Digital Earth Africa web services</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/rainfall_chirps_monthly" class="" target="_none">Digital Earth Africa Explorer (CHIRPS monthly rainfall)</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://github.com/digitalearthafrica/deafrica-sandbox-notebooks" class="" target="_none">Digital Earth Africa Notebook Repo</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://www.africageoportal.com/pages/digital-earth-africa" class="" target="_none">Digital Earth Africa Geoportal</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-chirps/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-coastlines" class="dataset">
          <h3><a href="/deafrica-coastlines/">Digital Earth Africa Coastlines</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Africa&#39;s long and dynamic coastline is subject to a wide range of pressures, including extreme weather and climate, sea level rise and human development. Understanding how the coastline responds to these pressures is crucial to managing this region, from social, environmental and economic perspectives.
The Digital Earth Africa Coastlines (provisional) is a continental dataset that includes annual shorelines and rates of coastal change along the entire African coastline from 2000 to the present.
The product combines satellite data from the Digital Earth Africa program with tidal modelling t...</p>
          <p><a href="/deafrica-coastlines/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://geoserver.digitalearth.africa/geoserver/ows?service&#x3D;wfs&amp;version&#x3D;2.0.0&amp;request&#x3D;GetCapabilities" class="" target="_none">Digital Earth Africa Web Feature Service</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://geoserver.digitalearth.africa/geoserver/ows?service&#x3D;wms&amp;version&#x3D;1.3.0&amp;request&#x3D;GetCapabilities" class="" target="_none">Digital Earth Africa Web Map Service</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://www.digitalearthafrica.org/media-center/blog/monitoring-coastal-erosion-saly-portudal-resort-mbour-senegal" class="" target="_none">Monitoring coastal erosion at Saly Portudal resort, Mbour-Senegal</a> by Ndeye Fatou SANE, Lissong DIOP, Mamadou Lamine NDIAYE, Amadou SALL, Dr. Moussa SALL</li></a>
          </li>
          <li>
            <a href="https://docs.digitalearthafrica.org/en/latest/sandbox/notebooks/Real_world_examples/Coastal_erosion.html" class="" target="_none">Monitoring coastal erosion along Africa’s coastline</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://maps.digitalearth.africa/" class="" target="_none">Digital Earth Africa Map</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-coastlines/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-geomad" class="dataset">
          <h3><a href="/deafrica-geomad/">Digital Earth Africa GeoMAD</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>GeoMAD is the Digital Earth Africa (DE Africa) surface reflectance geomedian and triple Median Absolute Deviation data service. It is a cloud-free composite of satellite data compiled over specific timeframes.
The geomedian component combines measurements collected over the specified timeframe to produce one representative, multispectral measurement for every pixel unit of the African continent. The end result is a comprehensive dataset that can be used to generate true-colour images for visual inspection of anthropogenic or natural landmarks. The full spectral dataset can be used to develop m...</p>
          <p><a href="/deafrica-geomad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://explorer.digitalearth.africa/products/gm_s2_annual" class="" target="_none">Digital Earth Africa Explorer (Sentinel-2 Annual GeoMAD)</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://youtu.be/Wkf7N6O9jJQ" class="" target="_none">Introduction to DE Africa</a> by Dr Fang Yuan</li></a>
          </li>
          <li>
            <a href="https://ows.digitalearth.africa" class="" target="_none">Digital Earth Africa web services</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/gm_ls8_annual" class="" target="_none">Digital Earth Africa Explorer (Landsat 8 GeoMAD)</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://github.com/digitalearthafrica/deafrica-sandbox-notebooks" class="" target="_none">Digital Earth Africa Notebook Repo</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-geomad/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-wofs" class="dataset">
          <h3><a href="/deafrica-wofs/">Digital Earth Africa Water Observations from Space</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info tag link-tag">water</span></p>
          <p>Water Observations from Space (WOfS) is a service that draws on satellite imagery to provide historical surface water observations of the whole African continent. WOfS allows users to understand the location and movement of inland and coastal water present in the African landscape. It shows where water is usually present; where it is seldom observed; and where inundation of the surface has been observed by satellite.
They are generated using the WOfS classification algorithm on Landsat satellite data. There are several WOfS products available for the African continent including scene-level dat...</p>
          <p><a href="/deafrica-wofs/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://maps.digitalearth.africa/" class="" target="_none">Digital Earth Africa Map</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://ows.digitalearth.africa" class="" target="_none">Digital Earth Africa web services</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://www.africageoportal.com/pages/digital-earth-africa" class="" target="_none">Digital Earth Africa Geoportal</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://sandbox.digitalearth.africa/" class="" target="_none">Digital Earth Africa Sandbox</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/wofs_ls" class="" target="_none">Digital Earth Africa Explorer (Water Observations from Space)</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-wofs/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="fcp-indi" class="dataset">
          <h3><a href="/fcp-indi/">International Neuroimaging Data-Sharing Initiative (INDI)</a></h3>
          <p><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This bucket contains multiple neuroimaging datasets that are part of the International Neuroimaging Data-Sharing Initiative. Raw human and non-human primate neuroimaging data include 1) Structural MRI; 2) Functional MRI; 3) Diffusion Tensor Imaging; 4) Electroencephalogram (EEG)
In addition to the raw data, preprocessed data is also included for some datasets.
A complete list of the available datasets can be seen in the documentation lonk provided below. </p>
          <p><a href="/fcp-indi/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S0896627318307682" class="" target="_none">An Open Resource for Non-human Primate Imaging</a> by M.P. Milham, L. Ai, ..., C.E. Schroeder</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41467-018-04976-1" class="" target="_none">Assessment of the impact of shared brain imaging data on the scientific literature</a> by M.P. Milham, R.C. Craddock, ..., A. Klein</li></a>
          </li>
          <li>
            <a href="https://fcon_1000.projects.nitrc.org/indi/s3/index.html" class="" target="_none">Downloading FCP-INDI Neuroimaging Data from Amazon S3</a> by INDI</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/mp201378" class="" target="_none">The autism brain imaging data exchange: towards a large-scale evaluation of the intrinsic brain architecture in autism.</a> by A. Di Martino, C-G Yan, ..., M.P. Milham</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/sdata201710" class="" target="_none">Enhancing studies of the connectome in autism using the autism brain imaging data exchange II.</a> by A. Di Martino, D. O&#x27;Connor, M.P. Milham</li></a>
          </li>
        </ul>
        <p><a href="/fcp-indi/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="ladi" class="dataset">
          <h3><a href="/ladi/">Low Altitude Disaster Imagery (LADI) Dataset</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">seismology</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">urban</span><span class="label label-info tag link-tag">water</span></p>
          <p>The Low Altitude Disaster Imagery (LADI) Dataset consists of human and machine annotated airborne images collected by the Civil Air Patrol in support of various disaster responses from 2015-2023. Two key distinctions are the low altitude, oblique perspective of the imagery and disaster-related features, which are rarely featured in computer vision benchmarks and datasets.</p>
          <p><a href="/ladi/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/bwsi-hadr" class="" target="_none">Remote Sensing for Disaster Response Course</a> by <a href="https://beaverworks.ll.mit.edu/CMS/bw/bwsi" target="_none">Beaver Works Summer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1109/HPEC.2019.8916437" class="" target="_none">Large Scale Organization and Inference of an Imagery Dataset for Public Safety</a> by Jeffrey Liu, David Strohschein, Siddharth Samsi, Andrew Weinert</li></a>
          </li>
          <li>
            <a href="https://github.com/LADI-Dataset/ladi-tutorial" class="" target="_none">LADI v1 Tutorials</a> by <a href="https://github.com/LADI-Dataset" target="_none">Andrew Weinert, Jianyu Mao, Kiana Harris, Nae-Rong Chang, Caleb Pennell, Yiming Ren, Ryan Earley, Nadia Dimitrova</a></li></a>
          </li>
          <li>
            <a href="https://www-nlpir.nist.gov/projects/tv2020/dsdi.html" class="" target="_none">NIST TRECVID 2020 - Disaster Scene Description and Indexing (DSDI)</a> by TREC Video Retrieval Evaluation (TRECVID)</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2005.05495" class="" target="_none">Train and Deploy an Image Classifier for Disaster Response</a> by Jianyu Mao, Kiana Harris, Nae-Rong Chang, Caleb Pennell, Yiming Ren</li></a>
          </li>
        </ul>
        <p><a href="/ladi/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="maxar-open-data" class="dataset">
          <h3><a href="/maxar-open-data/">Maxar Open Data Program</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Pre and post event high-resolution satellite imagery in support of emergency planning, risk assessment, 
monitoring of staging areas and emergency response, damage assessment, and recovery. These images are generated
using the <a href="https://ard.maxar.com/docs">Maxar ARD</a> pipeline, tiled on an organized grid in analysis-ready
cloud-optimized formats.</p>
          <p><a href="/maxar-open-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://eprints.lancs.ac.uk/id/eprint/167032/1/Simmons_et_al_2022_CSTP_PRN_Paper_Special_Issue_Accepted_Version.pdf" class="" target="_none">Disaster, Infrastructure and Participatory Knowledge The Planetary Response Network</a> by Brooke Simmons, Chris Lintott, Steven Reece, et al.</li></a>
          </li>
          <li>
            <a href="https://ard.maxar.com/docs/ard-order-delivery/about-ard-order-delivery/" class="" target="_none">ARD Deliverables and File Structure</a> by <a href="https://www.maxar.com/open-data" target="_none">Maxar Open Data</a></li></a>
          </li>
          <li>
            <a href="https://ard.maxar.com/docs/sdk/" class="" target="_none">Maxar ARD SDK (max-ard)</a> by <a href="https://www.maxar.com/open-data" target="_none">Maxar Open Data</a></li></a>
          </li>
          <li>
            <a href="https://leafmap.org/notebooks/67_maxar_open_data/" class="" target="_none">Maxar Open Data - Leafmap</a> by <a href="https://geography.utk.edu/about-us/faculty/dr-qiusheng-wu/" target="_none">Qiusheng Wu</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1080/19475705.2022.2147455" class="" target="_none"> Machine learning remote sensing using the random forest classifier to detect the building damage caused by the Anak Krakatau Volcano tsunami</a> by Riantini Virtriana et al.</li></a>
          </li>
        </ul>
        <p><a href="/maxar-open-data/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-ofs" class="dataset">
          <h3><a href="/noaa-ofs/">NOAA Operational Forecast System (OFS)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>ANNOUNCEMENTS: [NOS OFS Version Updates and Implementation of Upgraded Oceanographic Forecast Modeling Systems for Lakes Superior and Ontario; Effective October 25, 2022}(<a href="https://www.weather.gov/media/notification/pdf2/scn22-91_nos_loofs_lsofs_v3.pdf">https://www.weather.gov/media/notification/pdf2/scn22-91_nos_loofs_lsofs_v3.pdf</a>)
<br/>
<br/>
For decades, mariners in the United States have depended on NOAA&#39;s Tide Tables for the best estimate of expected water levels. These tables provide accurate predictions of the astronomical tide (i.e., the change in water level due to the gravitational effects of the moon and sun and the rotation of the Earth); however, they cannot predict water-level changes due to wind, atmospheric pressure, and river flow, which are often significant.
<br/>
<br/>
The National Ocean Service (NOS) has the mission and mandate to provide guidance and information to support navigation and coastal needs. To support this mission, NOS has been developing and implementing hydrodynamic model-based <a href="https://tidesandcurrents.noaa.gov/forecast_info.html">Operational Forecast Systems</a>. 
<br/>
<br/>
This forecast guidance provides oceanographic information that helps mariners safely navigate their local waters. This national network of hydrodynamic models provides users with operational nowcast and forecast guidance (out to 48 – 120 hours) on parameters such as water levels, water temperature, salinity, and currents. These forecast systems are implemented in critical ports, harbors, estuaries, Great Lakes and coastal waters of the United States, and form a national backbone of real-time data, tidal predictions, data management and operational modeling.
<br/>
<br/>
Nowcasts and forecasts are scientific predictions about the present and future states of water levels (and possibly currents and other relevant oceanographic variables, such as salinity and temperature) in a coastal area. These predictions rely on either observed data or forecasts from a numerical model. A nowcast incorporates recent (and often near real-time) observed meteorological, oceanographic, and/or river flow rate data. A nowcast covers the period from the recent past (e.g., the past few days) to the present, and it can make predictions for locations where observational data are not available. A forecast incorporates meteorological, oceanographic, and/or river flow rate forecasts and makes predictions for times where observational data will not be available. A forecast is usually initiated by the results of a nowcast.
<br/>
<br/>
OFS generally runs four times per day (every 6 hours) on NOAA&#39;s Weather and Climate Operational Supercomputing Systems (WCOSS) in a standard Coastal Ocean Modeling Framework (COMF) developed by the <a href="https://tidesandcurrents.noaa.gov/">Center for Operational Oceanographic Products and Services (CO-OPS)</a>. COMF is a set...</p>
          <p><a href="/noaa-ofs/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://tidesandcurrents.noaa.gov/ofs/publications/DBOFS_Onepager.pdf" class="" target="_none">Delaware Bay and River OFS Flyer</a> by NOAA</li></a>
          </li>
          <li>
            <a href="https://tidesandcurrents.noaa.gov/ofs/publications/CBOFS_TechReport.pdf" class="" target="_none">Chesapeake Bay OFS: Model Development and Skill Assessment</a> by NOAA</li></a>
          </li>
          <li>
            <a href="https://tidesandcurrents.noaa.gov/ofs/publications/CBOFS_Onepager.pdf" class="" target="_none">Chesapeake Bay OFS Flyer</a> by NOAA</li></a>
          </li>
          <li>
            <a href="https://tidesandcurrents.noaa.gov/ofs/publications/NOS-TIN-TBOFS.pdf" class="" target="_none">Technical Implementation Notice for Tampa Bay OFS</a> by NOAA</li></a>
          </li>
          <li>
            <a href="https://opendap.co-ops.nos.noaa.gov/ofsdataagg/index.html" class="" target="_none">OFS Data Aggregation and Sub-Setting</a> by NOAA</li></a>
          </li>
        </ul>
        <p><a href="/noaa-ofs/#usageexamples">See 11 usage examples &rarr;</a></p>
        </div>
        <div id="cbers" class="dataset">
          <h3><a href="/cbers/">CBERS on AWS</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Imagery acquired
by the China-Brazil Earth Resources Satellite (CBERS), 4 and 4A.
The
image files are recorded and processed by Instituto Nacional de Pesquisas
Espaciais (INPE) and are converted to Cloud Optimized Geotiff
format in order to optimize its use for cloud based applications.
Contains all CBERS-4 MUX, AWFI, PAN5M and
PAN10M scenes acquired since
the start of the satellite mission and is daily updated with
new scenes.
CBERS-4A MUX Level 4 (Orthorectified) scenes are being
ingested starting from 04-13-2021. CBERS-4A WFI Level 4 (Orthorectified)
scenes are being ingested starting from ...</p>
          <p><a href="/cbers/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/RemotePixel/aws-sat-api-py" class="" target="_none">aws-sat-api-py</a> by <a href="http://remotepixel.ca/" target="_none">Remote Pixel</a></li></a>
          </li>
          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></li></a>
          </li>
          <li>
            <a href="https://stac.amskepler.com/v100" class="" target="_none">STAC V1.0.0 endpoint</a> by <a href="https://github.com/fredliporace/cbers-2-stac" target="_none">AMS Kepler</a></li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/9165649" class="" target="_none">Using Remote Sensing Images and Cloud Services on AWS to Improve Land Use and Cover Monitoring</a> by K. R. Ferreira, et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/fredliporace/cbersgif" class="" target="_none">CBERS timelapse GIF generator</a> by <a href="https://github.com/fredliporace" target="_none">Frederico Liporace</a></li></a>
          </li>
        </ul>
        <p><a href="/cbers/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-sentinel-1" class="dataset">
          <h3><a href="/deafrica-sentinel-1/">Digital Earth Africa Sentinel-1 Radiometrically Terrain Corrected</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>DE Africa’s Sentinel-1 backscatter product is developed to be compliant with the CEOS Analysis Ready Data for Land (CARD4L) specifications.
The Sentinel-1 mission, composed of a constellation of two C-band Synthetic Aperture Radar (SAR) satellites, are operated by European Space Agency (ESA) as part of the Copernicus Programme. The mission currently collects data every 12 days over Africa at a spatial resolution of approximately 20 m.
Radar backscatter measures the amount of microwave radiation reflected back to the sensor from the ground surface. This measurement is sensitive to surface rough...</p>
          <p><a href="/deafrica-sentinel-1/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://youtu.be/Wkf7N6O9jJQ" class="" target="_none">Introduction to DE Africa</a> by Dr Fang Yuan</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.3390/rs14020351" class="" target="_none">An Operational Analysis Ready Radar Backscatter Dataset for the African Continent</a> by Fang Yuan, Marko Repse, Alex Leith, Ake Rosenqvist, Grega Milcinski, Negin F. Moghaddam, Tishampati Dhar, Chad Burton, Lisa Hall, Cedric Jorand and Adam Lewis</li></a>
          </li>
          <li>
            <a href="https://docs.digitalearthafrica.org/en/latest/sandbox/notebooks/Real_world_examples/Radar_water_detection.html" class="" target="_none">Water detection with Sentinel-1</a> by Madeleine Seehaber</li></a>
          </li>
          <li>
            <a href="https://maps.digitalearth.africa/" class="" target="_none">Digital Earth Africa Map</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/s1_rtc/extents" class="" target="_none">Digital Earth Africa Explorer</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-sentinel-1/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-sentinel-2" class="dataset">
          <h3><a href="/deafrica-sentinel-2/">Digital Earth Africa Sentinel-2 Level-2A</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The Sentinel-2 mission is part of the European Union Copernicus programme for Earth observations. Sentinel-2 consists of twin satellites, Sentinel-2A (launched 23 June 2015) and Sentinel-2B (launched 7 March 2017). The two satellites have the same orbit, but 180° apart for optimal coverage and data delivery. Their combined data is used in the Digital Earth Africa Sentinel-2 product.
Together, they cover all Earth’s land surfaces, large islands, inland and coastal waters every 3-5 days.
Sentinel-2 data is tiered by level of pre-processing. Level-0, Level-1A and Level-1B data contain raw data fr...</p>
          <p><a href="/deafrica-sentinel-2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://maps.digitalearth.africa/" class="" target="_none">Digital Earth Africa Map</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/s2_l2a/extents" class="" target="_none">Digital Earth Africa Explorer</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://github.com/opendatacube/cube-in-a-box" class="" target="_none">Use Sentinel-2 data in the Open Data Cube</a> by Alex Leith</li></a>
          </li>
          <li>
            <a href="https://ows.digitalearth.africa" class="" target="_none">Digital Earth Africa web services</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://docs.digitalearthafrica.org/en/latest/sandbox/notebooks/Frequently_used_code/Downloading_data_with_STAC.html" class="" target="_none">Downloading and streaming data using STAC metadata</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-sentinel-2/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="mur" class="dataset">
          <h3><a href="/mur/">Multi-Scale Ultra High Resolution (MUR) Sea Surface Temperature (SST)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>A global, gap-free, gridded, daily 1 km Sea Surface Temperature (SST) dataset created by merging multiple Level-2 satellite SST datasets. Those input datasets include the NASA Advanced Microwave Scanning Radiometer-EOS (AMSR-E), the JAXA Advanced Microwave Scanning Radiometer 2 (AMSR-2) on GCOM-W1, the Moderate Resolution Imaging Spectroradiometers (MODIS) on the NASA Aqua and Terra platforms, the US Navy microwave WindSat radiometer, the Advanced Very High Resolution Radiometer (AVHRR) on several NOAA satellites, and in situ SST observations from the NOAA iQuam project. Data are available fro...</p>
          <p><a href="/mur/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://thredds.jpl.nasa.gov/thredds/catalog_ghrsst_gds2.html?dataset&#x3D;MUR-JPL-L4-GLOB-v4.1" class="" target="_none">THREDDS server</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></li></a>
          </li>
          <li>
            <a href="https://podaac-tools.jpl.nasa.gov/drive/files/allData/ghrsst/data/GDS2/L4/GLOB/JPL/MUR/v4.1" class="" target="_none">HTTPS server</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></li></a>
          </li>
          <li>
            <a href="https://podaac-tools.jpl.nasa.gov/drive/files/allData/ghrsst/sw/generic_nc_readers/python/" class="" target="_none">Python Reader Software</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">PO.DAAC</a></li></a>
          </li>
          <li>
            <a href="https://opendap.jpl.nasa.gov/opendap/OceanTemperature/ghrsst/data/GDS2/L4/GLOB/JPL/MUR/v4.1/contents.html" class="" target="_none">OPeNDAP server</a> by <a href="https://podaac.jpl.nasa.gov" target="_none">Physical Oceanography Distributed Active Archive Center (PO.DAAC)</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/improving-knowledge-oceans-providing-cloud-based-access-large-datasets/" class="aws-link" target="_none">Improving our knowledge about the oceans by providing cloud-based access to large datasets</a> by Chelle Gentemann</li></a>
          </li>
        </ul>
        <p><a href="/mur/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="nz-imagery" class="dataset">
          <h3><a href="/nz-imagery/">New Zealand Imagery</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The New Zealand Imagery dataset consists of New Zealand&#39;s publicly owned aerial and satellite imagery, which is freely available to use under an open licence. The dataset ranges from the latest high-resolution aerial imagery down to 5cm in some urban areas to lower resolution satellite imagery that provides full coverage of mainland New Zealand, Chathams and other offshore islands. It also includes historical imagery that has been scanned from film, orthorectified (removing distortions) and georeferenced (correctly positioned) to create a unique and crucial record of changes to the New Zea...</p>
          <p><a href="/nz-imagery/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://basemaps.linz.govt.nz" class="" target="_none">LINZ Basemaps</a> by <a href="https://www.linz.govt.nz" target="_none">Toitū Te Whenua Land Information New Zealand</a></li></a>
          </li>
          <li>
            <a href="https://github.com/linz/imagery/blob/master/docs/naming.md" class="" target="_none">Imagery survey naming and path conventions</a> by <a href="https://www.linz.govt.nz" target="_none">Toitū Te Whenua Land Information New Zealand</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.3390/rs10081195" class="" target="_none">A Boundary Regulated Network for Accurate Roof Segmentation and Outline Extraction</a> by Guangming Wu, Zhiling Guo, Xiaodan Shi, Qi Chen, Yongwei Xu, Ryosuke Shibasaki and Xiaowei Shao</li></a>
          </li>
          <li>
            <a href="https://storymaps.arcgis.com/collections/e6d212054d9744f399fcbed00a75ee43" class="" target="_none">ArcGIS Workflows: NZ Imagery from a public AWS S3 bucket</a> by <a href="https://www.eagle.co.nz/" target="_none">Eagle Technology</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.3390/rs11151774" class="" target="_none">Semantic Segmentation of Urban Buildings from VHR Remote Sensing Imagery Using a Deep Convolutional Neural Network</a> by Yaning Yi, Zhijie Zhang, Wanchang Zhang, Chuanrong Zhang, Weidong Li and Tian Zhao</li></a>
          </li>
        </ul>
        <p><a href="/nz-imagery/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="sbn-css" class="dataset">
          <h3><a href="/sbn-css/">Catalina Sky Survey (CSS) subset data on AWS</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">object detection</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">survey</span></p>
          <p>Raw data that discovers Near Earth Objects (NEOs) which potentially could impact Earth</p>
          <p><a href="/sbn-css/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://catch.astro.umd.edu/about" class="" target="_none">CATCH (Comet-Asteroid Telescopic Catalog Hub) service: A moving-target search tool for large astronomical data sets, such as time-domain or near-Earth object surveys.</a> by CATCH service is built and operated by the Small Bodies Node of the NASA Planetary Data System group at the University of Maryland.</li></a>
          </li>
          <li>
            <a href="https://sbnarchive.psi.edu/pds4/surveys/gbo.ast.catalina.survey/document/CSS_history_v6.pdf" class="" target="_none">History of the Catalina Sky Survey</a> by S. Larson, edited by R. Seaman</li></a>
          </li>
          <li>
            <a href="https://sbnarchive.psi.edu/pds4/surveys/gbo.ast.catalina.survey/document/CSS_operations_v8.pdf" class="" target="_none">Catalina Sky Survey Operations and Processing</a> by R. Seaman, E. Christensen, A. Gibbs, S. Larson, F. Shelly</li></a>
          </li>
          <li>
            <a href="https://sbnarchive.psi.edu/pds4/surveys/gbo.ast.catalina.survey/document/CSS_telescopes_v6.pdf" class="" target="_none">Catalina Sky Survey Telescopes and Instrumentation</a> by R. Seaman, E. Christensen, A. Gibbs, S. Larson, F. Shelly</li></a>
          </li>
          <li>
            <a href="https://heasarc.gsfc.nasa.gov/fitsio/fpack/" class="" target="_none">FITS image compression programs (fpack &amp; funpack)</a> by Rob Seaman (NOAO), William Pence (NASA/GSFC), Rick White (STScI).</li></a>
          </li>
        </ul>
        <p><a href="/sbn-css/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="oedi-data-lake" class="dataset">
          <h3><a href="/oedi-data-lake/">Department of Energy&#x27;s Open Energy Data Initiative (OEDI)</a></h3>
          <p><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">solar</span></p>
          <p>Data released under the Department of Energy&#39;s (DOE) Open Energy Data Initiative
(OEDI). The Open Energy Data Initiative aims to improve and automate
access of high-value energy data sets across the U.S. Department of Energy’s
programs, offices, and national laboratories. OEDI aims to make data
actionable and discoverable by researchers and industry to accelerate
analysis and advance innovation.</p>
          <p><a href="/oedi-data-lake/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nrel.gov/docs/fy16osti/65298.pdf" class="" target="_none">Rooftop Solar Photovoltaic Technical Potential in the United States: A Detailed Assessment</a> by Pieter Gagnon, Robert Margolis, Jennifer Melius, Caleb Phillips, and Ryan Elmore</li></a>
          </li>
          <li>
            <a href="https://emp.lbl.gov/tracking-sun-tool" class="" target="_none">Tracking the Sun Tool</a> by <a href="https://emp.lbl.gov/tracking-the-sun" target="_none">Lawrence Berkeley National Laboratory (LBNL)</a></li></a>
          </li>
          <li>
            <a href="https://maps.nrel.gov/nsrdb-viewer/?aL&#x3D;0&amp;bL&#x3D;groad&amp;cE&#x3D;0&amp;lR&#x3D;0&amp;mC&#x3D;31.653381399664%2C-3.1640625&amp;zL&#x3D;2" class="" target="_none">NSRDB Viewer</a> by <a href="https://www.nrel.gov/" target="_none">National Renewable Energy Laboratory (NREL)</a></li></a>
          </li>
          <li>
            <a href="https://emp.lbl.gov/sites/default/files/tracking_the_sun_2019_report.pdf" class="" target="_none">Tracking the Sun Pricing and Design Trends for Distributed Photovoltaic Systems in the United States: 2019 Edition</a> by G. Barbose, N. Darghouth</li></a>
          </li>
          <li>
            <a href="https://www.nrel.gov/docs/fy16osti/65283.pdf" class="" target="_none">On the Use of Coupled Wind, Wave, and Current Fields in the Simulation of Loads on BottomSupported Offshore Wind Turbines during Hurricanes</a> by E. Kim, L. Manuel, M. Curcic, S. S. Chen, C. Phillips, P. Veers</li></a>
          </li>
        </ul>
        <p><a href="/oedi-data-lake/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-alos-jers" class="dataset">
          <h3><a href="/deafrica-alos-jers/">Digital Earth Africa ALOS PALSAR, ALOS-2 PALSAR-2 and JERS-1</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>The ALOS/PALSAR annual mosaic is a global 25 m resolution dataset that combines data from many images captured by JAXA’s PALSAR and PALSAR-2 sensors on ALOS-1 and ALOS-2 satellites respectively. This product contains radar measurement in L-band and in HH and HV polarizations. It has a spatial resolution of 25 m and is available annually for 2007 to 2010 (ALOS/PALSAR) and 2015 to 2020 (ALOS-2/PALSAR-2).
The JERS annual mosaic is generated from images acquired by the SAR sensor on the Japanese Earth Resources Satellite-1 (JERS-1) satellite. This product contains radar measurement in L-band and H...</p>
          <p><a href="/deafrica-alos-jers/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://learn.digitalearthafrica.org/" class="" target="_none">Digital Earth Africa Training</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://sandbox.digitalearth.africa/" class="" target="_none">Digital Earth Africa Sandbox</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://maps.digitalearth.africa/" class="" target="_none">Digital Earth Africa Map</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/alos_palsar_mosaic" class="" target="_none">Digital Earth Africa Explorer (ALOS PALSAR and ALOS-2 PALSAR-2)</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/jers_sar_mosaic" class="" target="_none">Digital Earth Africa Explorer (JERS)</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-alos-jers/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-crop-extent" class="dataset">
          <h3><a href="/deafrica-crop-extent/">Digital Earth Africa Cropland Extent Map (2019)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Digital Earth Africa&#39;s cropland extent map (2019) shows the estimated location of croplands in Africa for the period January to December 2019. Cropland is defined as: &quot;a piece of land of minimum 0.01 ha (a single 10m x 10m pixel) that is sowed/planted and harvest-able at least once within the 12 months after the sowing/planting date.&quot; This definition will exclude non-planted grazing lands and perennial crops which can be difficult for satellite imagery to differentiate from natural vegetation. 
This provisional cropland extent map has a resolution of 10m, and was built using Cope...</p>
          <p><a href="/deafrica-crop-extent/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.digitalearthafrica.org/media-center/blog/cropland-extent-now-available-entire-african-continent" class="" target="_none">Cropland Extent is now available for the entire African continent</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://www.africageoportal.com/pages/digital-earth-africa" class="" target="_none">Digital Earth Africa Geoportal</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://sandbox.digitalearth.africa/" class="" target="_none">Digital Earth Africa Sandbox</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="http://learn.digitalearthafrica.org/" class="" target="_none">Digital Earth Africa Training</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://agu2021fallmeeting-agu.ipostersessions.com/Default.aspx?s&#x3D;BA-E4-89-84-15-50-25-89-AE-A8-EF-C5-32-7D-19-EE" class="" target="_none">Co-Production of a 10-m Cropland Extent Map for Continental Africa using Sentinel-2, Cloud Computing, and the Open-Data-Cube</a> by Chad Burton, Fang Yuan, Chong Ee-Faye, Meghan Halabisky, David Ongo, Fatou Mar, Victor Addabor, Bako Mamane and Sena Adimou</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-crop-extent/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-fractional-cover" class="dataset">
          <h3><a href="/deafrica-fractional-cover/">Digital Earth Africa Fractional Cover</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Fractional cover (FC) describes the landscape in terms of coverage by green vegetation, non-green vegetation (including deciduous trees during autumn, dry grass, etc.) and bare soil. It provides insight into how areas of dry vegetation and/or bare soil and green vegetation are changing over time. The product is derived from Landsat satellite data, using an algorithm developed by the <a href="https://www.jrsrp.org.au/">Joint Remote Sensing Research Program</a>. 
Digital Earth Africa&#39;s FC service has two components. Fractional Cover is estimated from each Landsat scene, providing measurements from individual days. Fractional Cover...</p>
          <p><a href="/deafrica-fractional-cover/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://youtu.be/Wkf7N6O9jJQ" class="" target="_none">Introduction to DE Africa</a> by Dr Fang Yuan</li></a>
          </li>
          <li>
            <a href="https://sandbox.digitalearth.africa/" class="" target="_none">Digital Earth Africa Sandbox</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/fc_ls" class="" target="_none">Digital Earth Africa Explorer (Fractional Cover)</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://github.com/digitalearthafrica/deafrica-sandbox-notebooks" class="" target="_none">Digital Earth Africa Notebook Repo</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="http://learn.digitalearthafrica.org/" class="" target="_none">Digital Earth Africa Training</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-fractional-cover/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-ndvi_anomaly" class="dataset">
          <h3><a href="/deafrica-ndvi_anomaly/">Digital Earth Africa Monthly Normalised Difference Vegetation Index (NDVI) Anomaly</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Digital Earth Africa’s Monthly NDVI Anomaly service provides estimate of vegetation condition, for each caldendar month, against the long-term baseline condition measured for the month from 1984 to 2020 in the NDVI Climatology.  A standardised anomaly is calculated by subtracting the long-term mean from an observation of interest and then dividing the result by the long-term standard deviation. Positive NDVI anomaly values indicate vegetation is greener than average conditions, and are usually due to increased rainfall in a region. Negative values indicate additional plant stress relative to t...</p>
          <p><a href="/deafrica-ndvi_anomaly/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://maps.digitalearth.africa/" class="" target="_none">Digital Earth Africa Map</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://www.digitalearthafrica.org/platform-resources/services/mean-ndvi-and-anomalies" class="" target="_none">Mean NDVI and Anomalies</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://github.com/digitalearthafrica/deafrica-sandbox-notebooks" class="" target="_none">Digital Earth Africa Notebook Repo</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://www.africageoportal.com/pages/digital-earth-africa" class="" target="_none">Digital Earth Africa Geoportal</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/ndvi_anomaly" class="" target="_none">Digital Earth Africa Explorer (Monthly Normalised Difference Vegetation Index (NDVI) Anomaly)</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-ndvi_anomaly/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="gtgseq" class="dataset">
          <h3><a href="/gtgseq/">Garvan Institute Long Read Sequencing Benchmark Data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">long read sequencing</span></p>
          <p>The dataset contains reference samples that will be useful for benchmarking and comparing bioinformatics tools for genome analysis. Currently, there are two samples, which are NA12878 (HG001) and NA24385 (HG002), sequenced on an Oxford Nanopore Technologies (ONT) PromethION using the latest R10.4.1 flowcells. Raw signal data output by the sequencer is provided for these datasets in BLOW5 format, and can be rebasecalled when basecalling software updates bring accuracy and feature improvements over the years. Raw signal data is not only for rebasecalling, but also can be used for emerging bioinf...</p>
          <p><a href="/gtgseq/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1093/bioinformatics/btad352" class="" target="_none">Accelerated nanopore basecalling with SLOW5 data format.</a> by Samarakoon, H., Ferguson, J.M., Gamaarachchi H. et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1186/s13059-023-02910-3" class="" target="_none">Flexible and efficient handling of nanopore sequencing signal data with slow5tools.</a> by Samarakoon, H., Ferguson, J.M., Jenner, S.P. et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/Psy-Fer/buttery-eel" class="" target="_none">buttery-eel: The buttery eel - a slow5 guppy basecaller wrapper</a> by Samarakoon, H., Ferguson, J.M., Gamaarachchi H. et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/BonsonW/slow5curl" class="" target="_none">Slow5curl: library and tool for accessing remote BLOW5 files.</a> by Wong, B., et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41587-021-01147-4" class="" target="_none">Fast nanopore sequencing data analysis with SLOW5.</a> by Gamaarachchi, H., Samarakoon, H., Jenner, S.P. et al.</li></a>
          </li>
        </ul>
        <p><a href="/gtgseq/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-pds-wtk" class="dataset">
          <h3><a href="/nrel-pds-wtk/">NREL Wind Integration National Dataset</a></h3>
          <p><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span></p>
          <p>Released to the public as part of the Department of Energy&#39;s Open Energy Data Initiative,
the <a href="https://www.nrel.gov/grid/wind-toolkit.html">Wind Integration National Dataset (WIND)</a>
is an update and expansion of the Eastern Wind Integration Data Set and
Western Wind Integration Data Set. It supports the next generation of wind
integration studies.</p>
          <p><a href="/nrel-pds-wtk/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NREL/hsds-viz" class="" target="_none">Wind Visualization</a> by <a href="https://www.nrel.gov/research/jordan-perr-sauer.html" target="_none">Jordan Perr-Sauer</a></li></a>
          </li>
          <li>
            <a href="https://maps.nrel.gov/wind-prospector/" class="" target="_none">Wind Prospector</a> by Paul Edwards</li></a>
          </li>
          <li>
            <a href="https://github.com/NREL/hsds-examples" class="" target="_none">HSDS Examples</a> by <a href="https://www.nrel.gov/research/caleb-phillips.html" target="_none">Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer, Michael Rossol</a></li></a>
          </li>
          <li>
            <a href="https://www.osti.gov/servlets/purl/2473210/" class="" target="_none">WTK-LED: The WIND Toolkit Long-Term Ensemble Dataset</a> by Caroline Draxl, Jiali Wang, Lindsay Sheridan, et al.</li></a>
          </li>
          <li>
            <a href="https://www.nrel.gov/docs/fy14osti/61714.pdf" class="" target="_none">Validation of Power Output for the WIND Toolkit</a> by J. King, Andrew Clifton, Bri-Mathias Hodge</li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-wtk/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="open-neurodata" class="dataset">
          <h3><a href="/open-neurodata/">Open NeuroData</a></h3>
          <p><span class="label label-info tag link-tag">array tomography</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">electron microscopy</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">light-sheet microscopy</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This bucket contains multiple neuroimaging datasets (as Neuroglancer Precomputed Volumes) across multiple modalities and scales, ranging from nanoscale (electron microscopy), to microscale (cleared lightsheet microscopy and array tomography), and mesoscale (structural and functional magnetic resonance imaging). Additionally, many of the datasets include segmentations and meshes.</p>
          <p><a href="/open-neurodata/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/seung-lab/cloud-volume" class="" target="_none">CloudVolume</a> by <a href="https://github.com/william-silversmith" target="_none">William Silversmith</a></li></a>
          </li>
          <li>
            <a href="https://github.com/google/neuroglancer" class="" target="_none">Neuroglancer</a> by <a href="https://github.com/jbms" target="_none">Jeremy Maitin-Shepard</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.neuron.2016.10.033" class="" target="_none">To the Cloud! A Grassroots Proposal to Accelerate Brain Science Discovery</a> by J. T. Vogelstein, B. Mensh, M. Häusser, N. Spruston, A. C. Evans, K. Kording, K. Amunts, C. Ebell, J. Muller, M. Telefont, S. Hill, S. P. Koushika, C. Calì, P. A. Valdés-Sosa, P. B. Littlewood, C. Koch, S. Saalfeld, A. Kepecs, H. Peng, Y. O. Halchenko, G. Kiar, M. M. Poo, J. B. Poline, M. P. Milham, A. P. Schaffer, R. Gidron, H. Okano, V. D. Calhoun, M. Chun, D. M. Kleissas, R. J. Vogelstein, E. Perlman, R. Burns, R. Huganir, and M. I. Miller</li></a>
          </li>
          <li>
            <a href="https://github.com/seung-lab/igneous" class="" target="_none">Igneous</a> by <a href="https://github.com/william-silversmith" target="_none">William Silversmith</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/1306.3543" class="" target="_none">The Open Connectome Project Data Cluster: Scalable Analysis and Vision for High-Throughput Neuroscience</a> by R. Burns, W. G. Roncal, D. Kleissas, K. Lillaney, P. Manavalan, E. Perlman, D. R. Berger, D. D. Bock, K. Chung, L. Grosenick, N. Kasthuri, N. C. Weiler, K. Deisseroth, M. Kazhdan, J. Lichtman, R. C. Reid, S. J. Smith, A. S. Szalay, J. T. Vogelstein, and R. J. Vogelstein.</li></a>
          </li>
        </ul>
        <p><a href="/open-neurodata/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="pubseq" class="dataset">
          <h3><a href="/pubseq/">PubSeq - Public Sequence Resource</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fasta</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">json</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">long read sequencing</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">metadata</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">RDF</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">SARS-CoV-2</span><span class="label label-info tag link-tag">SPARQL</span></p>
          <p>COVID-19 PubSeq is a free and open online bioinformatics public sequence resource with on-the-fly analysis of sequenced SARS-CoV-2 samples that allows for a quick turnaround in identification of new virus strains. PubSeq allows anyone to upload sequence material in the form of FASTA or FASTQ files with accompanying metadata through the web interface or REST API.</p>
          <p><a href="/pubseq/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://wwwnc.cdc.gov/eid/article/27/5/21-0514_article" class="" target="_none">Transmission of Severe Acute Respiratory Syndrome Coronavirus 2 during Border Quarantine and Air Travel, New Zealand (Aotearoa)</a> by Eichler, N. et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/pubseq/bh20-seq-resource/tree/master/workflows" class="" target="_none">Source code for CWL workflows</a> by <a href="https://github.com/pubseq" target="_none">PubSeq development team</a></li></a>
          </li>
          <li>
            <a href="https://github.com/pubseq/bh20-seq-resource/tree/master/bh20sequploader" class="" target="_none">Source code for (meta)data uploader</a> by <a href="https://github.com/pubseq" target="_none">PubSeq development team</a></li></a>
          </li>
          <li>
            <a href="https://covid19.genenetwork.org/blog?id&#x3D;using-covid-19-pubseq-part3" class="" target="_none">Submitting a sequence</a> by <a href="https://github.com/pubseq" target="_none">PubSeq development team</a></li></a>
          </li>
          <li>
            <a href="https://github.com/pubseq/bh20-seq-resource/tree/master/bh20simplewebuploader" class="" target="_none">Source code for website</a> by <a href="https://github.com/pubseq" target="_none">PubSeq development team</a></li></a>
          </li>
        </ul>
        <p><a href="/pubseq/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="southern-california-earthquakes" class="dataset">
          <h3><a href="/southern-california-earthquakes/">Southern California Earthquake Data</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">seismology</span></p>
          <p>This dataset contains ground motion velocity and acceleration seismic waveforms recorded by the Southern California Seismic Network (SCSN) and archived at the Southern California Earthquake Data Center (SCEDC). A Distributed Acousting Sensing (DAS) dataset is included.</p>
          <p><a href="/southern-california-earthquakes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://scedc.caltech.edu/data/getstarted-pds.html" class="" target="_none">Getting Started with SCEDC AWS Public Dataset</a> by <a href="https://scedc.caltech.edu" target="_none">Ellen Yu</a></li></a>
          </li>
          <li>
            <a href="https://github.com/SCEDC/cloud/tree/master/pds-lambda-apis" class="" target="_none">Using Amazon API Gateway and Lambda to window and decimate waveforms</a> by <a href="https://scedc.caltech.edu" target="_none">Shang-Lin Chen</a></li></a>
          </li>
          <li>
            <a href="https://nextjournal.com/thclements/cactus-to-clouds/#accessing-the-scedc-dataset-on-aws" class="" target="_none">Cactus to Clouds: Processing The SCEDC Open Data Set on AWS</a> by <a href="https://tclements.github.io/" target="_none">Tim Clements</a></li></a>
          </li>
          <li>
            <a href="https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2022JB025553" class="" target="_none">The Seismic Signature of California&#x27;s Earthquakes, Droughts, and Floods.</a> by Tim Clements; Marine Denolle</li></a>
          </li>
          <li>
            <a href="https://github.com/SCEDC/cloud/tree/master/pds-fetch-continuous-data" class="" target="_none">Script to Download Seismic Waveforms from the SCEDC AWS Public Dataset</a> by <a href="https://scedc.caltech.edu" target="_none">Aparna Bhaskaran</a></li></a>
          </li>
        </ul>
        <p><a href="/southern-california-earthquakes/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="usgs-lidar" class="dataset">
          <h3><a href="/usgs-lidar/">USGS 3DEP LiDAR Point Clouds</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The goal of the <a href="https://www.usgs.gov/core-science-systems/ngp/3dep">USGS 3D Elevation Program </a> (3DEP) is to collect elevation data in the form of light detection and ranging (LiDAR) data over the conterminous United States, Hawaii, and the U.S. territories, with data acquired over an 8-year period. This dataset provides two realizations of the 3DEP point cloud data. The first resource is a public access organization provided in <a href="https://entwine.io/entwine-point-tile.html">Entwine Point Tiles</a> format, which a lossless, full-density, streamable octree based on <a href="https://laszip.org">LASzip</a> (LAZ) encoding. The second resource is a <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/RequesterPaysBuckets.html">Requester Pays</a> of the original, Raw LAZ (Compressed LAS) 1.4 3DEP format, and more co...</p>
          <p><a href="/usgs-lidar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.opentopography.org/datasets" class="" target="_none">OpenTopography access to 3DEP lidar point cloud data</a> by <a href="https://opentopography.org/" target="_none">OpenTopography</a></li></a>
          </li>
          <li>
            <a href="https://www.usgs.gov/news/usgs-3dep-lidar-point-cloud-now-available-amazon-public-dataset" class="" target="_none">USGS 3DEP Lidar Point Cloud Now Available as Amazon Public Dataset</a> by <a href="https://www.usgs.gov" target="_none">Department of the Interior, U.S. Geological Survey</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/machine-learning/extracting-buildings-and-roads-from-aws-open-data-using-amazon-sagemaker/" class="aws-link" target="_none">Extracting buildings and roads from AWS Open Data using Amazon SageMaker</a> by Yunzhi Shi, Tianyu Zhang, and Xin Chen</li></a>
          </li>
          <li>
            <a href="https://equatorstudios.com/lidar-viewer/" class="" target="_none">Equator - View, Process, and Download USGS 3DEP LiDAR data in-browser</a> by <a href="https://equatorstudios.com" target="_none">Equator Studios</a></li></a>
          </li>
          <li>
            <a href="https://github.com/OpenTopography/OT_3DEP_Workflows" class="" target="_none">Jupyter Notebooks to enable programmatic access to cloud-hosted USGS 3D Elevation Program (3DEP) lidar data</a> by <a href="https://opentopography.org/" target="_none">OpenTopography</a></li></a>
          </li>
        </ul>
        <p><a href="/usgs-lidar/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="wb-light-every-night" class="dataset">
          <h3><a href="/wb-light-every-night/">World Bank - Light Every Night</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Light Every Night - World Bank Nighttime Light Data – provides open access to all nightly imagery and data from the Visible Infrared Imaging Radiometer Suite Day-Night Band (VIIRS DNB) from 2012-2020 and the Defense Meteorological Satellite Program Operational Linescan System (DMSP-OLS) from 1992-2013. The underlying data are sourced from the NOAA National Centers for Environmental Information (NCEI) archive. Additional processing by the University of Michigan enables access in Cloud Optimized GeoTIFF format (COG) and search using the Spatial Temporal Asset Catalog (STAC) standard. The data is...</p>
          <p><a href="/wb-light-every-night/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.asprs.org/wp-content/uploads/pers/97journal/june/1997_jun_727-734.pdf" class="" target="_none">Mapping city lights with nighttime data from the DMSP Operational Linescan System. Photogrammetric Engineering and Remote Sensing, 63(6)727-734.</a> by Elvidge, C.D., Baugh, K.E., Kihn, E.A., Kroehl, H.W. and Davis, E.R.</li></a>
          </li>
          <li>
            <a href="https://journals.sfu.ca/apan/index.php/apan/article/view/8/0" class="" target="_none">Nighttime lights compositing using the VIIRS day-night band - Preliminary results. Proceedings of the Asia-Pacific Advanced Network 35 (2013)70-86.</a> by Kimberly Baugh, Feng-Chi Hsu, Christopher D. Elvidge, and Mikhail Zhizhin.</li></a>
          </li>
          <li>
            <a href="http://nightlights.io" class="" target="_none">Twenty Years of India Lights</a> by <a href="http://nightlights.io" target="_none">Kwawu Mensan Gaba, Brian Min, Anand Thakker, Christopher Elvidge</a></li></a>
          </li>
          <li>
            <a href="http://www-personal.umich.edu/~brianmin/HREA/" class="" target="_none">High Resolution Electricity Access Indicators (HREA) - Settlement-level measures of electricity access, reliability, and usage.</a> by <a href="http://www-personal.umich.edu/~brianmin/" target="_none">Brian Min, Zachary O&#x27;Keeffe</a></li></a>
          </li>
          <li>
            <a href="https://www.cambridge.org/core/books/power-and-the-vote/8091820171410B4FA557B6EBC3A5DD06" class="" target="_none">Power and the Vote - Elections and Electricity in the Developing World. Cambridge. 2015.</a> by Brian Min</li></a>
          </li>
        </ul>
        <p><a href="/wb-light-every-night/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="motional-nuscenes" class="dataset">
          <h3><a href="/motional-nuscenes/">nuScenes</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">robotics</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">urban</span></p>
          <p>Public large-scale dataset for autonomous driving. It enables researchers to study challenging urban driving situations using the full sensor suite of a real self-driving car.</p>
          <p><a href="/motional-nuscenes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://colab.research.google.com/github/nutonomy/nuscenes-devkit/" class="" target="_none">nuScenes Map Expansion Tutorial</a> by Motional</li></a>
          </li>
          <li>
            <a href="https://colab.research.google.com/github/nutonomy/nuscenes-devkit/" class="" target="_none">nuScenes CAN bus tutorial</a> by Motional</li></a>
          </li>
          <li>
            <a href="https://colab.research.google.com/github/nutonomy/nuscenes-devkit/" class="" target="_none">nuImages devkit tutorial</a> by Motional</li></a>
          </li>
          <li>
            <a href="https://colab.research.google.com/github/nutonomy/nuscenes-devkit/" class="" target="_none">nuScenes prediction tutorial</a> by Motional</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2109.03805" class="" target="_none">Panoptic nuScenes: A Large-Scale Benchmark for LiDAR Panoptic Segmentation and Tracking</a> by Whye Kit Fong, Rohit Mohan, Juana Valeria Hurtado, Lubing Zhou, Holger Caesar, Oscar Beijbom, Abhinav Valada</li></a>
          </li>
        </ul>
        <p><a href="/motional-nuscenes/#usageexamples">See 9 usage examples &rarr;</a></p>
        </div>
        <div id="boreas" class="dataset">
          <h3><a href="/boreas/">Boreas Autonomous Driving Dataset</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">robotics</span></p>
          <p>This autonomous driving dataset includes data from a 128-beam Velodyne Alpha-Prime lidar, a 5MP Blackfly camera, a 360-degree Navtech radar, and post-processed Applanix POS LV GNSS data. This dataset was collect in various weather conditions (sun, rain, snow) over the course of a year. The intended purpose of this dataset is to enable benchmarking of long-term all-weather odometry and metric localization across various sensor types. In the future, we hope to also support an object detection benchmark.</p>
          <p><a href="/boreas/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2209.03304" class="" target="_none">Picking up speed: Continuous-time Lidar-only odometry using doppler velocity measurements</a> by Y Wu, D J Yoon, K Burnett, S Kammel, Y Chen, H Vhavle, T D Barfoot</li></a>
          </li>
          <li>
            <a href="https://www.dynsyslab.org/wp-content/papercite-data/pdf/burnett-ral21.pdf" class="" target="_none">Do we need to compensate for motion distortion and doppler effects in spinning radar navigation?</a> by K Burnett, A P Schoellig, T D Barfoot</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2203.10168" class="" target="_none">Boreas: A multi-season autonomous driving dataset</a> by K Burnett, D J Yoon, Y Wu, A Z Li, H Zhang, S Lu, J Qian, W Tseng, A Lambert, K YK Leung, A P Schoellig, Timothy D Barfoot</li></a>
          </li>
          <li>
            <a href="https://github.com/utiasASRL/pyboreas/blob/master/pyboreas/tutorials/intro.ipynb" class="" target="_none">Introduction to Visualizing Sensor Types (Jupyter notebook)</a> by Keenan Burnett</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2203.10174" class="" target="_none">Are We Ready for Radar to Replace Lidar in All-Weather Mapping and Localization?</a> by K Burnett, Y Wu, D J Yoon, A P Schoellig, T D Barfoot</li></a>
          </li>
        </ul>
        <p><a href="/boreas/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="wrf-cmip6" class="dataset">
          <h3><a href="/wrf-cmip6/">CMIP6 GCMs downscaled using WRF</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">weather</span></p>
          <p>High-resolution historical and future climate simulations from 1980-2100</p>
          <p><a href="/wrf-cmip6/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://dev.cal-adapt.org/files/01_Memo_Evaluation_of_Downscaled_GCMs_Using_WRF_CEC_final.pdf" class="" target="_none">Memo on the Evaluation of Downscaled GCMs Using WRF</a> by Stefan Rahimi</li></a>
          </li>
          <li>
            <a href="https://www.researchgate.net/publication/374504614_Data_tier_descriptions_directory_structure_and_data_access_of_the_Western_US_Dynamically_Downscaled_Dataset_WUS-D3_version_1" class="" target="_none">Downscaling file descriptions, directory structure, and data access</a> by Stefan Rahimi &amp; Lei Huang</li></a>
          </li>
          <li>
            <a href="https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2023GL105979" class="" target="_none">Is Bias Correction in Dynamical Downscaling Defensible?</a> by Risser, M. D., Rahimi, S., Goldenson, N., Hall, A., Lebo, Z. J., and Feldman, D. R.</li></a>
          </li>
          <li>
            <a href="https://nbviewer.jupyter.org/urls/wrf-cmip6-noversioning.s3.amazonaws.com/Example_notebook/Notebook_example.ipynb" class="" target="_none">Jupyter Notebook Example</a> by Stefan Rahimi</li></a>
          </li>
          <li>
            <a href="https://gmd.copernicus.org/articles/17/2265/2024/gmd-17-2265-2024.html" class="" target="_none">An Overview of the Western United States Dynamically Downscaled Dataset (WUS-D3)</a> by Rahimi, S., Huang, L., Norris, J., Hall, A., Goldenson, N., Krantz, W., Bass, B., Thackeray, C., Lin, H., Chen, D., Dennis, E., Collins, E., Lebo, Z. J., Slinskey, E., Graves, S., Biyani, S., Wang, B., Cropper, S., and the UCLA Center for Climate Science Team</li></a>
          </li>
        </ul>
        <p><a href="/wrf-cmip6/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="ccle" class="dataset">
          <h3><a href="/ccle/">Cancer Cell Line Encyclopedia (CCLE)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Cancer Cell Line Encyclopedia (CCLE) project is an effort to conduct a detailed genetic
characterization of a large panel of human cancer cell lines. The CCLE provides public access to
genomic data, visualization and analysis for over 1100 cancer cell lines. This dataset contains
RNA-Seq Aligned Reads, WXS Aligned Reads, and WGS Aligned Reads data.</p>
          <p><a href="/ccle/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://console.cloud.google.com/bigquery?p&#x3D;isb-cgc&amp;d&#x3D;CCLE_bioclin_v0&amp;page&#x3D;dataset" class="" target="_none">ISB CGC BigQuery tables</a> by <a href="https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/data/CCLE_top.html
" target="_none">Institute for Systems Biology</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41586-019-1186-3" class="" target="_none">Next-generation characterization of the Cancer Cell Line Encyclopedia</a> by Ghandi, M., Huang F. et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="https://docs.sevenbridges.com/docs/ccle-data" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://portals.broadinstitute.org/ccle" class="" target="_none">Broad Institute Cancer Cell Line Encyclopedia</a> by <a href="https://www.broadinstitute.org/" target="_none">The Broad Institute of MIT &amp; Harvard</a></li></a>
          </li>
        </ul>
        <p><a href="/ccle/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="wpto-pds-us-wave" class="dataset">
          <h3><a href="/wpto-pds-us-wave/">DOE&#x27;s Water Power Technology Office&#x27;s (WPTO) US Wave dataset</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">water</span></p>
          <p>Released to the public as part of the Department of Energy&#39;s Open Energy Data Initiative,
this is the highest resolution publicly available long-term wave hindcast
dataset that – when complete – will cover the entire U.S. Exclusive Economic
Zone (EEZ).</p>
          <p><a href="/wpto-pds-us-wave/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.osti.gov/biblio/1599105" class="" target="_none">Development and validation of a high-resolution regional wave hindcast model for U.S. West Coast wave resource characterization</a> by Wu, Wei-Cheng; Wang, Taiping; Yang, Zhaoqing; Garcia Medina, Gabriel</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.renene.2019.01.020" class="" target="_none">Development and validation of a regional-scale high-resolution unstructured model for wave energy resource characterization along the US East Coast</a> by Allahdadi, M.N., Gunawan, J. Lai, R. He, V.S. Neary</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.36688/imej.3.65-71" class="" target="_none">High-resolution hindcasts for U.S. wave energy resource characterization</a> by Yang, Z. and V.S. Neary</li></a>
          </li>
          <li>
            <a href="https://www.osti.gov/biblio/1573061/" class="" target="_none">High-Resolution Regional Wave Hindcast for the U.S. West Coast</a> by Yang, Zhaoqing; Wu, Wei-Cheng; Wang, Taiping; Castrucci, Luca</li></a>
          </li>
          <li>
            <a href="http://swanmodel.sourceforge.net/download/zip/swanuse.pdf" class="" target="_none">SWAN Cycle III version 41.31A</a> by The SWAN team</li></a>
          </li>
        </ul>
        <p><a href="/wpto-pds-us-wave/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-ndvi_climatology_ls" class="dataset">
          <h3><a href="/deafrica-ndvi_climatology_ls/">Digital Earth Africa Normalised Difference Vegetation Index (NDVI) Climatology</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Digital Earth Africa’s NDVI climatology product represents the long-term average baseline condition of vegetation for every Landsat pixel over the African continent. Both mean and standard deviation NDVI climatologies are available for each calender month.Some key features of the product are:<ul>
<li>NDVI climatologies were developed using harmonized Landsat 5,7,and 8 satellite imagery.</li>
<li>Mean and standard deviation NDVI climatologies are produced for each calender month, using a temporal baseline period from 1984-2020 (inclusive)</li>
<li>Datasets have a spatial...</p>
          <p><a href="/deafrica-ndvi_climatology_ls/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.africageoportal.com/pages/digital-earth-africa" class="" target="_none">Digital Earth Africa Geoportal</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://ows.digitalearth.africa" class="" target="_none">Digital Earth Africa web services</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://maps.digitalearth.africa/" class="" target="_none">Digital Earth Africa Map</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="http://learn.digitalearthafrica.org/" class="" target="_none">Digital Earth Africa Training</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://explorer.digitalearth.africa/products/ndvi_climatology_ls" class="" target="_none">Digital Earth Africa Explorer(Normalised Difference Vegetation Index (NDVI) Climatology)</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-ndvi_climatology_ls/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="geoglows-v2" class="dataset">
          <h3><a href="/geoglows-v2/">GEOGLOWS Hydrological Model Version 2</a></h3>
          <p><span class="label label-info tag link-tag">geopackage</span><span class="label label-info tag link-tag">hydrography</span><span class="label label-info tag link-tag">hydrologic model</span><span class="label label-info tag link-tag">hydrology</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>GEOGLOWS is the Group on Earth Observation&#39;s Global Water Sustainability Program. It coordinates efforts from public 
and private entities to make application ready river data more accessible and sustainably available to underdeveloped 
regions. The GEOGLOWS Hydrological Model provides a retrospective and daily forecast of global river discharge at 7 
million river sub-basins. The stream network is a hydrologically conditioned subset of the TDX-Hydro streams and 
basins data produced by the United State&#39;s National Geospatial Intelligence Agency. The daily forecast provides 3 
hourly average discharge in a 51 member ensemble and 15 day lead time derived from the ECMWF Integrated Forecast 
System (IFS). The retrospective simulation is derived from ERA5 climate reanalysis data and provides daily average 
streamflow beginning on 1 January 1940. New forecasts are uploaded daily and the retrospective simulation is updated 
weekly on Sundays to keep the lag time between 5 and 12 days.<br><br>
The geoglows-v2 bucket contains: (1) model configuration files used to generate the simulations, (2) the GIS streams 
datasets used by the model, (3) the GIS streams datasets optimized for visualizations used by Esri&#39;s Living Atlas 
layer, (4) several supporting table of metadata including country names, river names, hydrological properties used for
modeling.<br><br>
The geoglows-v2-forecasts bucket contains: (1) daily 15 forecasts in zarr format optimized for time series queries of 
all ensemble members in the prediction, (2) CSV formatted summary files optimized for producing time series animated 
web maps for the entire global streams dataset.<br><br>
The geoglows-v2-retrospective bucket contains: (1) the model retrospective outputs in (1a) zarr format optimized for 
time series queries of up to a few hundred rivers on demand as well as (1b) in netCDF format best for bulk download</br>...</p>
          <p><a href="/geoglows-v2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://data.geoglows.org/tutorials/finding-river-numbers" class="" target="_none">Finding River Numbers</a> by <a href="https://hales.app" target="_none">Riley Hales</a></li></a>
          </li>
          <li>
            <a href="https://data.geoglows.org/tutorials/query-retro-data" class="" target="_none">Access GEOGLOWS retrospective simulation data from AWS S3</a> by <a href="https://hales.app" target="_none">Riley Hales</a></li></a>
          </li>
          <li>
            <a href="https://data.geoglows.org/tutorials/download-retro-data" class="" target="_none">Bulk Download and Query simulation netCDF Files</a> by <a href="https://hales.app" target="_none">Riley Hales</a></li></a>
          </li>
          <li>
            <a href="https://apps.geoglows.org/apps/geoglows-hydroviewer" class="" target="_none">GEOGLOWS Hydroviewer</a> by <a href="https://hales.app" target="_none">Riley Hales</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1111/jfr3.12859" class="" target="_none">Advancing global hydrologic modeling with the GEOGloWS ECMWF streamflow service</a> by Hales et al.</li></a>
          </li>
        </ul>
        <p><a href="/geoglows-v2/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="ibl-brain-wide-map" class="dataset">
          <h3><a href="/ibl-brain-wide-map/">IBL Neuropixels Brainwide Map on AWS</a></h3>
          <p><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span><span class="label label-info tag link-tag">open source software</span></p>
          <p>Electrophysiological recordings of mouse brain activity acquired using Neuropixels probes and accompanying behavioral data.</p>
          <p><a href="/ibl-brain-wide-map/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://int-brain-lab.github.io/iblenv/loading_examples.html#" class="" target="_none">Loading Data</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
          <li>
            <a href="https://colab.research.google.com/drive/1th3MRZGHMSaeAvGmKGJQ84rBk8eEI4Fu" class="" target="_none">Example analysis of the brain-wide map dataset</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
          <li>
            <a href="https://int-brain-lab.github.io/iblenv/notebooks_external/data_download.html" class="" target="_none">Download the public datasets</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
          <li>
            <a href="https://colab.research.google.com/drive/1Ua-NlpYYZCIOF56xbsT9YR71Enkotd-b#scrollTo&#x3D;7XzVVlhsVHMK" class="" target="_none">Introduction to IBL and the brain-wide map dataset</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
          <li>
            <a href="https://viz.internationalbrainlab.org/app" class="" target="_none">Data Explorer</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
        </ul>
        <p><a href="/ibl-brain-wide-map/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="nasanex" class="dataset">
          <h3><a href="/nasanex/">NASA Earth Exchange (NEX) Data Collection</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">CMIP5</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>A collection of downscaled climate change projections, derived from the
General Circulation Model (GCM) runs conducted under the Coupled Model
Intercomparison Project Phase 5 (CMIP5) [Taylor et al. 2012] and across
the four greenhouse gas emissions scenarios known as Representative
Concentration Pathways (RCPs) [Meinshausen et al. 2011]. The NASA Earth
Exchange group maintains the NEX-DCP30 (CMIP5), NEX-GDDP (CMIP5), and
LOCA (CMIP5).</p>
          <p><a href="/nasanex/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.kdd.org/kdd2020/accepted-papers/view/climate-downscaling-using-ynet-a-deep-convolutional-network-with-skip-conne" class="" target="_none">Climate Downscaling Using YNet: a Deep Convolutional Network with Skip Connections and Fusion</a> by Yumin Liu, Auroop Ganguly, and Jennifer Dy</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/ftp/arxiv/papers/2201/2201.02596.pdf" class="" target="_none">Explainable deep learning for insights in El Nino and river flows</a> by Yumin Liu, Kate Duffy, Jennifer G. Dy, and Auroop R. Ganguly</li></a>
          </li>
          <li>
            <a href="https://cloud-gc.readthedocs.io/en/latest/chapter06_appendix/plot_NASANEX.html" class="" target="_none">Sample Python code to analyze NASA-NEX data.</a> by <a href="https://github.com/JiaweiZhuang" target="_none">Jiawei Zhuang</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1002/2013EO370002" class="" target="_none">Downscaled Climate Projections Suitable for Resource Management, Eos Trans. AGU, 94(37), 321.</a> by Thrasher, B., J. Xiong, W. Wang, F. Melton, A. Michaelis and R. Nemani (2013)</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1175/JHM-D-14-0082.1" class="" target="_none">Statistical downscaling using Localized Constructed Analogs (LOCA).</a> by Pierce, D. W., D. R. Cayan, and B. L. Thrasher (2014)</li></a>
          </li>
        </ul>
        <p><a href="/nasanex/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="ncei-wcsd-archive" class="dataset">
          <h3><a href="/ncei-wcsd-archive/">NOAA Water-Column Sonar Data Archive</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>Water-column sonar data archived at the NOAA National Centers for Environmental Information.</p>
          <p><a href="/ncei-wcsd-archive/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ngdc.noaa.gov/maps/water_column_sonar/index.html" class="" target="_none">Water Column Sonar Data Explorer - Map viewer for exploring sonar water column data</a> by NCEI, NOAA</li></a>
          </li>
          <li>
            <a href="https://github.com/CI-CMG/pyEcholab/tree/master/examples/cloud%20tutorials/Reading_And_Plotting_Bottom_Data.ipynb" class="" target="_none">Reading and Plotting Bottom Data</a> by Carrie Wall</li></a>
          </li>
          <li>
            <a href="https://github.com/CI-CMG/pyEcholab/tree/master/examples/cloud%20tutorials/Reading_And_Plotting_Processed_CSV_Data.ipynb" class="" target="_none">Reading and Plotting Processed CSV Data</a> by Carrie Wall</li></a>
          </li>
          <li>
            <a href="https://eos.org/science-updates/building-an-accessible-archive-for-water-column-sonar-data" class="" target="_none">Building an Accessible Archive for Water Column Sonar Data</a> by Carrie Wall</li></a>
          </li>
          <li>
            <a href="https://github.com/CI-CMG/pyEcholab" class="" target="_none">pyEcholab - an open-source, python-based toolkit for reading, processing, plotting, and exporting fisheries acoustic echosounder data</a> by Rick Towler, Chuck Anderson, Veronica Martinez, Pamme Crandell</li></a>
          </li>
        </ul>
        <p><a href="/ncei-wcsd-archive/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="nz-elevation" class="dataset">
          <h3><a href="/nz-elevation/">New Zealand Elevation</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The New Zealand Elevation dataset consists of New Zealand&#39;s publicly owned digital elevation models and digital surface models, which are freely available to use under an open licence. The dataset contains 1m resolution grids derived from LiDAR data. Point clouds are not included in the initial release.All of the elevation files are <a href="https://www.cogeo.org/">Cloud Optimised GeoTIFFs</a> using LERC compression for the main grid and LERC compression with lower max_z_error for the overviews. These elevation files are accompanied by <a href="https://stacspec.org/"...</p>
          <p><a href="/nz-elevation/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.3390/rs12121906" class="" target="_none">Stress Detection in New Zealand Kauri Canopies with WorldView-2 Satellite and LiDAR Data</a> by Jane J. Meiforth, Henning Buddenbaum, Joachim Hill, James D. Shepherd and John R. Dymond</li></a>
          </li>
          <li>
            <a href="https://basemaps.linz.govt.nz" class="" target="_none">LINZ Basemaps</a> by <a href="https://www.linz.govt.nz" target="_none">Toitū Te Whenua Land Information New Zealand</a></li></a>
          </li>
          <li>
            <a href="https://data.linz.govt.nz" class="" target="_none">LINZ Data Service</a> by <a href="https://www.linz.govt.nz" target="_none">Toitū Te Whenua Land Information New Zealand</a></li></a>
          </li>
          <li>
            <a href="https://github.com/linz/topo-workflows" class="" target="_none">LINZ Topographic Workflows - Bulk elevation data processing with Kubernetes and Argo Workflows</a> by <a href="https://www.linz.govt.nz" target="_none">Toitū Te Whenua Land Information New Zealand</a></li></a>
          </li>
          <li>
            <a href="https://github.com/linz/elevation/blob/master/docs/naming.md" class="" target="_none">Elevation survey naming and path conventions</a> by <a href="https://www.linz.govt.nz" target="_none">Toitū Te Whenua Land Information New Zealand</a></li></a>
          </li>
        </ul>
        <p><a href="/nz-elevation/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="northern-california-earthquakes" class="dataset">
          <h3><a href="/northern-california-earthquakes/">Northern California Earthquake Data</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">seismology</span></p>
          <p>This dataset contains various types of digital data relating
to earthquakes in central and northern California. 
Time series data come from broadband, short period, and strong motion
seismic sensors, GPS, and other geophysical sensors.</p>
          <p><a href="/northern-california-earthquakes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.researchgate.net/publication/252418955_Collaborative_Projects_at_the_Northern_California_Earthquake_Data_Center_NCEDC" class="" target="_none">Collaborative Projects at the Northern California Earthquake Data Center (NCEDC)</a> by Doug Neuhauser; David Oppenheimer; Stephane Zuzlewski; Lind Gee; Mark Murray</li></a>
          </li>
          <li>
            <a href="https://ncedc.org/db/cloud/getstarted-pds.html" class="" target="_none">Getting Started with NCEDC AWS Public Dataset</a> by <a href="https://ncedc.org" target="_none">Ellen Yu, Stephane Zuzlewski</a></li></a>
          </li>
          <li>
            <a href="https://www.researchgate.net/publication/252252363_The_Northern_California_Earthquake_Management_System_A_Unified_System_From_Realtime_Monitoring_to_Data_Distribution" class="" target="_none">The Northern California Earthquake Management System: A Unified System From Realtime Monitoring to Data Distribution</a> by Doug Neuhauser; Fred Klein; Stephane Zuzlewski; Lynn Dietz; Peter Lombard</li></a>
          </li>
          <li>
            <a href="https://www.researchgate.net/publication/253823256_Improved_Data_Access_From_the_Northern_California_Earthquake_Data_Center" class="" target="_none">Improved Data Access From the Northern California Earthquake Data Center</a> by Doug Neuhauser; Fred Klein; Stephane Zuzlewski; David Oppenheimer; Gray Jensen</li></a>
          </li>
          <li>
            <a href="https://www.researchgate.net/publication/252836017_The_Northern_California_Earthquake_Data_Center_Seismic_and_Geophysical_Data_for_Northern_California_and_Beyond" class="" target="_none">The Northern California Earthquake Data Center: Seismic and Geophysical Data for Northern California and Beyond</a> by Doug Neuhauser; Fred Klein; Stephane Zuzlewski; Lind Gee; David Oppenheimer</li></a>
          </li>
        </ul>
        <p><a href="/northern-california-earthquakes/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="radiant-mlhub" class="dataset">
          <h3><a href="/radiant-mlhub/">Radiant MLHub</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Radiant MLHub is an open library for geospatial training data that hosts datasets generated by <a href="https://www.radiant.earth/">Radiant Earth Foundation</a>&#39;s team as well as other training data catalogs contributed by Radiant Earth’s partners. Radiant MLHub is open to anyone to access, store, register and/or share their training datasets for high-quality Earth observations. All of the training datasets are stored using a <a href="https://stacspec.org/">SpatioTemporal Asset Catalog (STAC)</a> compliant catalog and exposed through a common API. Training datasets include pairs of imagery and labels for different types of machine learning problems including image ...</p>
          <p><a href="/radiant-mlhub/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/radiant-earth-insights/creating-a-machine-learning-commons-for-global-development-256ef3dd46aa" class="" target="_none">Creating a Machine Learning Commons for Global Development</a> by <a href="https://www.linkedin.com/in/hamedalemohammad/" target="_none">Hamed Alemohammad</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/asdi-smsl-wind-speed-data/" class="aws-link" target="_none">Explore wind speed images from Amazon Sustainability Data Initiative (ASDI) hosted on S3 using SageMaker Studio Lab (SMSL)</a> by Frank Rubino</li></a>
          </li>
          <li>
            <a href="https://registry.mlhub.earth/" class="" target="_none">Radiant MLHub Dataset Registry</a> by <a href="https://www.radiant.earth/" target="_none">Radiant Earth</a></li></a>
          </li>
          <li>
            <a href="https://github.com/radiantearth/mlhub-tutorials/tree/master/notebooks" class="" target="_none">Radiant MLHub Tutorials with Jupyter Notebooks</a> by <a href="https://www.linkedin.com/in/kbgg/" target="_none">Kevin Booth</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/radiant-earth-insights/a-guide-for-collecting-and-sharing-ground-reference-data-for-machine-learning-applications-90664930925e" class="" target="_none">A Guide for Collecting and Sharing Ground Reference Data for Machine Learning Applications</a> by <a href="https://www.linkedin.com/in/yonahbg/" target="_none">Yonah Bromberg Gaber</a></li></a>
          </li>
        </ul>
        <p><a href="/radiant-mlhub/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="targetepigenomics" class="dataset">
          <h3><a href="/targetepigenomics/">Toxicant Exposures and Responses by Genomic and Epigenomic Regulators of Transcription (TaRGET)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">epigenomics</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The TaRGET (Toxicant Exposures and Responses by Genomic and Epigenomic Regulators of Transcription) Program is a research consortium funded by the National Institute of Environmental Health Sciences (NIEHS). The goal of the collaboration is to address the role of environmental exposures in disease pathogenesis as a function of epigenome perturbation, including understanding the environmental control of epigenetic mechanisms and assessing the utility of surrogate tissue analysis in mouse models of disease-relevant environmental exposures.</p>
          <p><a href="/targetepigenomics/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/31177562" class="" target="_none">The role of environmental exposures and the epigenome in health and disease.</a> by Perera BPU, Faulk C, Svoboda LK, Goodrich JM, Dolinoy DC.</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/32382149/" class="" target="_none">Environmental Determinants of cardiovasular disease: lessons learned from air pollution</a> by Al-Kindi SG, Brook RD, Biswal S, Rajagopalan S.</li></a>
          </li>
          <li>
            <a href="https://data.targetepigenomics.org/help/user-guide/download-files" class="" target="_none">Finding and Downloading TaRGET II Data files</a> by <a href="https://data.targetepigenomics.org" target="_none">TaRGET-DCC</a></li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/32576878/" class="" target="_none">Comparison of differential accessibility analysis strategies for ATAC-seq data</a> by Gontarz P, Fu S, Xing X, Liu S, Miao B et.al.</li></a>
          </li>
          <li>
            <a href="https://www.jci.org/articles/view/137315" class="" target="_none">Metabolic effects of air pollution exposure and reversibility</a> by Rajagopalan S, Park B, Palanivel R, et al.</li></a>
          </li>
        </ul>
        <p><a href="/targetepigenomics/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="aster-l1t" class="dataset">
          <h3><a href="/aster-l1t/">ASTER L1T Cloud-Optimized GeoTIFFs</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mining</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The Advanced Spaceborne Thermal Emission and Reflection Radiometer (ASTER) Level 1
Precision Terrain Corrected Registered At-Sensor Radiance (AST_L1T) data contains
calibrated at-sensor radiance, which corresponds with the ASTER Level 1B (AST_L1B),
that has been geometrically corrected, and rotated to a north-up UTM projection.
The AST_L1T is created from a single resampling of the corresponding ASTER L1A (AST_L1A) product.The precision terrain correction process incorporates GLS2000 digital elevation data with
derived ground control points (GCPs) to achieve topographic accuracy for all daytim...</p>
          <p><a href="/aster-l1t/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://lpdaac.usgs.gov/documents/1401/ASTER_L1T_Product_Specification_v1.pdf" class="" target="_none">ASTER L1T Product Specification</a> by USGS EROS Data Center</li></a>
          </li>
          <li>
            <a href="https://landsat.usgs.gov/landsat_acq#convertPathRow" class="" target="_none">Latitude-Longitude to Path-Row conversion</a> by <a href="https://landsat.usgs.gov" target="_none">USGS</a></li></a>
          </li>
          <li>
            <a href="https://descarteslabs.com/platform" class="" target="_none">Descartes Labs Platform</a> by <a href="https://descarteslabs.com" target="_none">Descartes Labs Inc.</a></li></a>
          </li>
          <li>
            <a href="https://lpdaac.usgs.gov/documents/1400/AST_L1T_ATBD_V1.pdf" class="" target="_none">ASTER L1T Algorithm Theoretical Basis</a> by David Meyer, Dawn Siemonsma, Barbara Brooks, and Lowell Johnson</li></a>
          </li>
          <li>
            <a href="https://lpdaac.usgs.gov/documents/128/ASTER_L1T_Tutorial.html" class="" target="_none">Working with ASTER L1T Visible and Near Infrared (VNIR) Data in R</a> by <a href="https://lpdaac.usgs.gov/" target="_none">Cole Krehbiel</a></li></a>
          </li>
        </ul>
        <p><a href="/aster-l1t/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="pgc-arcticdem" class="dataset">
          <h3><a href="/pgc-arcticdem/">ArcticDEM</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>ArcticDEM - 2m GSD Digital Elevation Models (DEMs) and mosaics from 2007 to the present. The ArcticDEM project seeks to fill the need for high-resolution time-series elevation data in the Arctic. The time-dependent nature of the strip DEM files allows users to perform change detection analysis and to compare observations of topography data acquired in different seasons or years. The mosaic DEM tiles are assembled from multiple strip DEMs with the intention of providing a more consistent and comprehensive product over large areas. ArcticDEM data is constructed from in-track and cross-track high...</p>
          <p><a href="/pgc-arcticdem/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.isprsjprs.2017.12.008" class="" target="_none">Automatic relative RPC image model bias compensation through hierarchical image matching for improving DEM quality</a> by Myoung-Jong Noh, Ian M. Howat</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5069/G96Q1VFK" class="" target="_none">OpenTopography access to ArcticDEM</a> by <a href="https://opentopography.org/" target="_none">OpenTopography</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1080/15481603.2015.1008621" class="" target="_none">Automated stereo-photogrammetric DEM generation at high latitudes: Surface Extraction with TIN-based Search-space Minimization (SETSM) validation and demonstration over glaciated regions</a> by Myoung-Jong Noh, Ian M. Howat</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1029/2018JF004873" class="" target="_none">Future Evolution of Greenland&#x27;s Marine-Terminating Outlet Glaciers</a> by Ginny A. Catania, Leigh A. Stearns, Twila A. Moon, Ellen M. Enderlin, R. H. Jackson</li></a>
          </li>
          <li>
            <a href="https://arcticdem.apps.pgc.umn.edu/" class="" target="_none">ArcticDEM Explorer</a> by Polar Geospatial Center &amp; ESRI</li></a>
          </li>
        </ul>
        <p><a href="/pgc-arcticdem/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-2" class="dataset">
          <h3><a href="/cptac-2/">Clinical Proteomic Tumor Analysis Consortium 2 (CPTAC-2)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-2 is the Phase II of the CPTAC Initiative (2011-2016).
Datasets contain open RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression
Quantification, and miRNA Expression Quantification data.</p>
          <p><a href="/cptac-2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CPTAC-2" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.03.030" class="" target="_none">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities</a> by Suhas Vasaikar, Chen Huang, Xiaojing Wang. Vladislav A. Petyuk, Sara R. Savage, Bo Wen,
Yongchao Dou, Yun Zhang, Zhiao Shi, Osama A. Arshad, Marina A. Gritsenko, Lisa J. Zimmerman,
Jason E. McDermott, Therese R. Clauss, Ronald J. Moore, Rui Zhao, Matthew E. Monroe, Yi-Ting
Wang, Matthew C. Chambers, Robbert J.C. Slebos, Ken S. Lau, Qianxing Mo, Li Ding, Matthew
Ellis, Mathangi Thiagarajan, Christopher R. Kinsinger, Henry Rodriguez, Richard D. Smith,
Karin D. Rodland, Daniel C. Liebler, Tao Liu, Bing Zhang, Clinical Proteomic Tumor Analysis
Consortium
</li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/sdata.2015.22" class="" target="_none">Proteomic analysis of colon and rectal carcinoma using standard and customized databases
</a> by Slebos RJ, Wang X, Wang X, Zhang B, Tabb DL, Liebler DC</li></a>
          </li>
        </ul>
        <p><a href="/cptac-2/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="cmip6" class="dataset">
          <h3><a href="/cmip6/">Coupled Model Intercomparison Project 6</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The sixth phase of global coupled ocean-atmosphere general circulation model ensemble. 
<br /><br /></p>
          <p><a href="/cmip6/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;C0UhiiGgbWA" class="" target="_none">CMIP6 Data Informational Session Video Recording</a> by ASDI</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/analyze-terabyte-scale-geospatial-datasets-with-dask-and-jupyter-on-aws/" class="aws-link" target="_none">Analyze terabyte-scale geospatial datasets with Dask and Jupyter on AWS</a> by Ethan Fahy and Zac Flamig</li></a>
          </li>
          <li>
            <a href="https://nbviewer.jupyter.org/github/aradhakrishnanGFDL/gfdl-aws-analysis/blob/master/examples/a1r-CompareAWS-NCvsZarr.ipynb" class="" target="_none">Comparing CMIP6 Zarr vs NetCDF Holdings</a> by Aparna Radhakrishnan</li></a>
          </li>
          <li>
            <a href="https://nbviewer.jupyter.org/github/awslabs/amazon-asdi/blob/main/examples/dask/notebooks/cmip6_zarr.ipynb" class="aws-link" target="_none">Processing CMIP6 data in Zarr format with Dask &amp; AWS Fargate</a> by Zac Flamig</li></a>
          </li>
          <li>
            <a href="https://github.com/aradhakrishnanGFDL/gfdl-aws-analysis/blob/community/doc/quickstartdoc.md" class="" target="_none">Getting Started with CMIP6 Data</a> by Aparna Radhakrishnan</li></a>
          </li>
        </ul>
        <p><a href="/cmip6/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="brazil-data-cubes" class="dataset">
          <h3><a href="/brazil-data-cubes/">Earth Observation Data Cubes for Brazil</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Earth observation (EO) data cubes produced from analysis-ready data (ARD) of CBERS-4, Sentinel-2 A/B and Landsat-8 satellite images for Brazil. The datacubes are regular in time and use a hierarchical tiling system. Further details are described in <a href="https://www.mdpi.com/2072-4292/12/24/4033">Ferreira et al. (2020)</a>.</p>
          <p><a href="/brazil-data-cubes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/brazil-data-cube/cube-builder-aws" class="" target="_none">cube-builder-aws - Application to generate data cubes on AWS environment.</a> by <a href="https://brazil-data-cube.github.io" target="_none">Brazil Data Cube</a></li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/9165649" class="" target="_none">Using Remote Sensing Images and Cloud Services on AWS to Improve Land Use and Cover Monitoring</a> by K. R. Ferreira, et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/brazil-data-cube/code-gallery/blob/master/jupyter/Python/stac/stac-aws-introduction.ipynb" class="" target="_none">Using Python to Access Image Collections Data (Jupyter notebook)</a> by <a href="https://brazil-data-cube.github.io/" target="_none">Brazil Data Cube</a></li></a>
          </li>
          <li>
            <a href="https://github.com/brazil-data-cube/code-gallery/blob/master/jupyter/Python/tiler/bdc-tiler_introduction.ipynb" class="" target="_none">Tile Map Service to view Image Collections from Brazil Data Cube Catalog</a> by <a href="https://brazil-data-cube.github.io" target="_none">Brazil Data Cube</a></li></a>
          </li>
          <li>
            <a href="https://www.mdpi.com/2072-4292/12/24/4033" class="" target="_none">Earth Observation Data Cubes for Brazil: Requirements, Methodology and Products.</a> by K. R. Ferreira, et al.</li></a>
          </li>
        </ul>
        <p><a href="/brazil-data-cubes/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="gdelt" class="dataset">
          <h3><a href="/gdelt/">Global Database of Events, Language and Tone (GDELT)</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">events</span></p>
          <p>This project monitors the world&#39;s broadcast, print,
and web news from nearly every corner of every country in
over 100 languages and identifies the people, locations,
organizations, counts, themes, sources, emotions,
quotes, images and events driving our global society every
second of every day.</p>
          <p><a href="/gdelt/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/running-r-on-amazon-athena/" class="aws-link" target="_none">Running R on Amazon Athena</a> by <a href="https://www.linkedin.com/in/gopal-wunnava-b11a77/" target="_none">Gopal Wunnava</a></li></a>
          </li>
          <li>
            <a href="https://www.linkedin.com/pulse/media-brexit-story-so-far-may-corbyn-less-shambles-than-mark-chopping/" class="" target="_none">Analysing Brexit Coverage In The Media Over Time</a> by <a href="https://www.kognitio.com" target="_none">Mark Chopping</a></li></a>
          </li>
          <li>
            <a href="http://www.geomesa.org/documentation/tutorials/geomesa-hbase-s3-on-aws.html" class="" target="_none">Bootstrapping GeoMesa HBase on AWS S3</a> by <a href="https://www.ccri.com" target="_none">Commonwealth Computer Research, Inc.</a></li></a>
          </li>
          <li>
            <a href="https://gist.github.com/jakechen/6955f2de51212163312b6430555b8e0b" class="" target="_none">Creating PySpark DataFrame from CSV in AWS S3 in EMR</a> by <a href="https://github.com/jakechen" target="_none">Jake Chen</a></li></a>
          </li>
          <li>
            <a href="http://blog.julien.org/2017/03/exploring-gdelt-data-set-with-amazon.html" class="" target="_none">Exploring GDELT with Athena</a> by <a href="https://twitter.com/julsimon" target="_none">Julien Simon</a></li></a>
          </li>
        </ul>
        <p><a href="/gdelt/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="icgc" class="dataset">
          <h3><a href="/icgc/">ICGC on AWS</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">vcf</span></p>
          <p>The International Cancer Genome Consortium (ICGC) coordinates projects with the common aim of accelerating research into the causes and control of cancer. The PanCancer Analysis of Whole Genomes (PCAWG) study is an international collaboration to identify common patterns of mutation in whole genomes from ICGC. More than 2,400 consistently analyzed genomes corresponding to over 1,100 unique ICGC donors are now freely available on Amazon S3 to credentialed researchers subject to ICGC data sharing policies.</p>
          <p><a href="/icgc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ICGC-TCGA-PanCancer/cli/wiki" class="" target="_none">AWS Pancancer Workflow Launcher</a> by <a href="https://github.com/solomonshorser" target="_none">Solomon Shorser</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1970-0" class="" target="_none">Genomic basis of RNA alterations in cancer</a> by <a href="https://icgc.org/committees-and-working-groups#71954" target="_none">PCAWG Transcriptome Core Group et al. (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-020-1943-3" class="" target="_none">The repertoire of mutational signatures in human cancer</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Ludmil%20B.%20Alexandrov" target="_none">Alexandrov et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-019-1907-7" class="" target="_none">The evolutionary history of 2,658 cancers</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Moritz%20Gerstung" target="_none">Gerstung et al (2020)</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-019-1913-9" class="" target="_none">Patterns of somatic structural variation in human cancer genomes</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;search&amp;term&#x3D;Yilong%20Li" target="_none">Li et al. (2020)</a></li></a>
          </li>
        </ul>
        <p><a href="/icgc/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="materials-project" class="dataset">
          <h3><a href="/materials-project/">Materials Project Data</a></h3>
          <p><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">cloud computing</span><span class="label label-info tag link-tag">data assimilation</span><span class="label label-info tag link-tag">digital assets</span><span class="label label-info tag link-tag">digital preservation</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">free software</span><span class="label label-info tag link-tag">genome</span><span class="label label-info tag link-tag">HPC</span><span class="label label-info tag link-tag">information retrieval</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">json</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">materials science</span><span class="label label-info tag link-tag">molecular dynamics</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">physics</span><span class="label label-info tag link-tag">post-processing</span><span class="label label-info tag link-tag">x-ray crystallography</span></p>
          <p>Materials Project is an open database of computed materials properties aiming to accelerate materials science research. The resources in this OpenData dataset contain the raw, parsed, and build data products.</p>
          <p><a href="/materials-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pymatgen.org" class="" target="_none">Pymatgen</a> by <a href="https://materialsproject.org" target="_none">Materials Project</a></li></a>
          </li>
          <li>
            <a href="https://materialsproject.github.io/atomate2" class="" target="_none">Atomate2</a> by <a href="https://materialsproject.org" target="_none">Materials Project</a></li></a>
          </li>
          <li>
            <a href="https://docs.materialsproject.org/downloading-data/using-the-api" class="" target="_none">Getting Started with the Materials API (MAPI)</a> by <a href="https://materialsproject.org" target="_none">Materials Project</a></li></a>
          </li>
          <li>
            <a href="https://pubs.aip.org/aip/apm/article/1/1/011002/119685/Commentary-The-Materials-Project-A-materials" class="" target="_none">The Materials Project: A materials genome approach to accelerating materials innovation</a> by Jain A., Ong S.P., Hautier G., et al.</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/mp-api/" class="" target="_none">MP API python client</a> by <a href="https://materialsproject.org" target="_none">Materials Project</a></li></a>
          </li>
        </ul>
        <p><a href="/materials-project/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="nwm-archive" class="dataset">
          <h3><a href="/nwm-archive/">NOAA National Water Model CONUS Retrospective Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The NOAA National Water Model Retrospective dataset contains input and output from multi-decade CONUS retrospective simulations. These simulations used meteorological input fields from meteorological retrospective datasets. The output frequency and fields available in this historical NWM dataset differ from those contained in the real-time operational NWM forecast model. Additionally, note that no streamflow or other data assimilation is performed within any of the NWM retrospective simulations
<br/>
<br/>
One application of this dataset is to provide historical context to current near real-time streamflow, soil moisture and snowpack conditions. The retrospective data can be used to infer flow frequencies and perform temporal analyses with hourly streamflow output and 3-hourly land surface output. This dataset can also be used in the development of end user applications which require a long baseline of data for system training or verification purposes. <br />
</br/>...</p>
          <p><a href="/nwm-archive/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/gist/rsignell-usgs/d3dfaf3cd3d8b39894a69b22127dfe38" class="" target="_none">Explore the National Water Model V2.0 Retrospective in Zarr</a> by <a href="https://www.usgs.gov/staff-profiles/richard-p-signell" target="_none">Rich Signell</a></li></a>
          </li>
          <li>
            <a href="https://github.com/NCAR/rechunk_retro_nwm_v21" class="" target="_none">Explore the National Water Model V2.1 Retrospective Dataset in Zarr</a> by <a href="https://github.com/jmccreight" target="_none">James McCreight, Ishita Srivastava, Rich Signell</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.31223/X5BH0P" class="" target="_none">On Strictly Enforced Mass Conservation Constraints for Modeling the Rainfall-Runoff Process</a> by Jonathan M. Frame, Frederik Kratzert, Hoshin V. Gupta, Paul Ullrich and Grey S. Nearing</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1111/1752-1688.13184" class="" target="_none">NOAA&#x27;s National Water Model: Advancing Operational Hydrology Through Continental-scale Modeling.</a> by Brian Cosgrove, David Gochis, Trey Flowers, Aubrey Dugger, Fred Ogden, Tom Graziano, Ed Clark, et al; 2024</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.ocemod.2019.101526" class="" target="_none">Simulating storm surge and compound flooding events with a creek-to-ocean model: Importance of baroclinic effects</a> by Fei Ye, et al.</li></a>
          </li>
        </ul>
        <p><a href="/nwm-archive/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="openaq" class="dataset">
          <h3><a href="/openaq/">OpenAQ</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span></p>
          <p>Global, aggregated physical air quality data from public data sources provided by government, research-grade and other sources. These awesome groups do the hard work of measuring these data and publicly sharing them, and our community makes them more universally-accessible to both humans and machines.</p>
          <p><a href="/openaq/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.openaq.org/docs/querying-with-aws-athena" class="" target="_none">Querying with AWS Athena</a> by <a href="https://openaq.org" target="_none">OpenAQ</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ksree/world-air-quality" class="" target="_none">OpenAQ Aggregator</a> by <a href="https://github.com/ksree" target="_none">Kapil Sreedharan</a></li></a>
          </li>
          <li>
            <a href="https://docs.openaq.org/docs/accessing-openaq-archive-data" class="" target="_none">Accessing data from the OpenAQ archive</a> by <a href="https://openaq.org" target="_none">OpenAQ</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ropensci/ropenaq" class="" target="_none">ropenaq R package</a> by <a href="https://github.com/maelle" target="_none">Maëlle Salmon</a></li></a>
          </li>
          <li>
            <a href="https://www.facebook.com/smokeybot/" class="" target="_none">Smokey: Air Quality Bot</a> by <a href="https://twitter.com/amrit_sharma" target="_none">Amrit Sharma</a></li></a>
          </li>
        </ul>
        <p><a href="/openaq/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="openaerialmap" class="dataset">
          <h3><a href="/openaerialmap/">OpenAerialMap on AWS</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>OpenAerialMap is a collection of high-resolution openly licensed satellite and aerial imagery.</p>
          <p><a href="/openaerialmap/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://new.openaerialmap.org/" class="" target="_none">New OpenAerialMap Browser</a> by <a href="https://www.kontur.io/" target="_none">Kontur</a></li></a>
          </li>
          <li>
            <a href="https://library.alnap.org/system/files/content/resource/files/main/alnap-hot-open-aerial-mapping-case-study-2016.pdf" class="" target="_none">Mapping a response: Using satellite images to aid humanitarian action</a> by ALNAP</li></a>
          </li>
          <li>
            <a href="https://medium.com/@anthropoco/how-to-segment-buildings-on-drone-imagery-with-fast-ai-cloud-native-geodata-tools-ae249612c321" class="" target="_none">How to Segment Buildings on Drone Imagery with Fast.ai &amp; Cloud-Native GeoData Tools</a> by <a href="https://medium.com/@anthropoco" target="_none">Dave Luo</a></li></a>
          </li>
          <li>
            <a href="https://learn.arcgis.com/en/projects/use-deep-learning-to-assess-palm-tree-health/" class="" target="_none">Use deep learning to assess palm tree health</a> by <a href="https://www.esri.com" target="_none">Julia Lenhardt</a></li></a>
          </li>
          <li>
            <a href="https://www.slideshare.net/slideshow/openaerialmap-a-distributed-commons-for-searching-and-hosting-free-imagery/52880462" class="" target="_none">OpenAerialMap: A Distributed Commons for Searching and Hosting Free Imagery</a> by Kate Chapman, Humanitarian OpenStreetMap Team</li></a>
          </li>
        </ul>
        <p><a href="/openaerialmap/#usageexamples">See 10 usage examples &rarr;</a></p>
        </div>
        <div id="pgc-rema" class="dataset">
          <h3><a href="/pgc-rema/">Reference Elevation Model of Antarctica (REMA)</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The Reference Elevation Model of Antarctica - 2m GSD Digital Elevation Models (DEMs) and mosaics from 2009 to the present. The REMA project seeks to fill the need for high-resolution time-series elevation data in the Antarctic. The time-dependent nature of the strip DEM files allows users to perform change detection analysis and to compare observations of topography data acquired in different seasons or years. The mosaic DEM tiles are assembled from multiple strip DEMs with the intention of providing a more consistent and comprehensive product over large areas. REMA data is constructed from in...</p>
          <p><a href="/pgc-rema/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.isprsjprs.2017.12.008" class="" target="_none">Automatic relative RPC image model bias compensation through hierarchical image matching for improving DEM quality</a> by Myoung-Jong Noh, Ian M. Howat</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5069/G9X63K5S" class="" target="_none">OpenTopography access to REMA</a> by <a href="https://opentopography.org/" target="_none">OpenTopography</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41561-019-0510-8" class="" target="_none">Deep glacial troughs and stabilizing ridges unveiled beneath the margins of the Antarctic ice sheet</a> by Morlighem, M., Rignot, E., Binder, T. et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/tc-13-665-2019" class="" target="_none">The Reference Elevation Model of Antarctica</a> by Ian M. Howat, Claire Porter, Benjanim E. Smith, Myoung-Jong Noh, Paul Morin</li></a>
          </li>
          <li>
            <a href="https://rema.apps.pgc.umn.edu/" class="" target="_none">REMA Explorer</a> by Polar Geospatial Center &amp; ESRI</li></a>
          </li>
        </ul>
        <p><a href="/pgc-rema/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="scottish-lidar" class="dataset">
          <h3><a href="/scottish-lidar/">Scottish Public Sector LiDAR Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">urban</span></p>
          <p>This dataset is Lidar data that has been collected by the Scottish public sector and made available under the Open Government Licence. The data are available as point cloud (LAS format or in LAZ compressed format), along with the derived Digital Terrain Model (DTM) and Digital Surface Model (DSM) products as Cloud optimized GeoTIFFs (COG) or standard GeoTIFF. The dataset contains multiple subsets of data which were each commissioned and flown in response to different organisational requirements. The details of each can be found at <a href="https://remotesensingdata.gov.scot/data#/list">https://remotesensingdata.gov.scot/data#/list</a></p>
          <p><a href="/scottish-lidar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://journal.caa-international.org/articles/10.5334/jcaa.49/" class="" target="_none">Making LiGHT Work of Large Area Survey? Developing Approaches to Rapid Archaeological Mapping and the Creation of Systematic National-scaled Heritage Data</a> by Dave Cowley, Łukasz Banaszek, George Geddes, Angela Gannon, Mike Middleton and Kirsty Millican</li></a>
          </li>
          <li>
            <a href="https://www.mdpi.com/2076-3263/8/8/272" class="" target="_none">Towards National Archaeological Mapping. Assessing Source Data and Methodology - A Case Study from Scotland</a> by Łukasz Banaszek, Dave Cowley and Mike Middleton</li></a>
          </li>
          <li>
            <a href="https://github.com/DataScienceScotland/LiDAR-Tutorial-QGIS/blob/main/resources/new_light_on_medieval_settlement_in_lowl.pdf" class="" target="_none">New light on medieval settlement in lowland Scotland</a> by Dave Cowley and Piers Dixon</li></a>
          </li>
          <li>
            <a href="https://github.com/DataScienceScotland/LiDAR-Tutorial-QGIS/blob/main/resources/expedite_and_upscale_large_area_archaeol.pdf" class="" target="_none">Large area archaeological survey in the age of big data and machine learning</a> by Dave Cowley, George Geddes, Łukasz Banaszek, Kirsty Millican and Iris Kramer</li></a>
          </li>
          <li>
            <a href="https://remotesensingdata.gov.scot/" class="" target="_none">Scottish Remote Sensing Portal</a> by <a href="https://remotesensingdata.gov.scot/about" target="_none">Scottish Government and Joint Nature Conservation Committee (JNCC)</a></li></a>
          </li>
        </ul>
        <p><a href="/scottish-lidar/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="motional-nuplan" class="dataset">
          <h3><a href="/motional-nuplan/">nuPlan</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">robotics</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">urban</span></p>
          <p>nuPlan is the world&#39;s first large-scale planning benchmark for autonomous driving.</p>
          <p><a href="/motional-nuplan/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://colab.research.google.com/github/motional/nuplan-devkit/" class="" target="_none">nuPlan Scenario Visualization</a> by Motional</li></a>
          </li>
          <li>
            <a href="https://colab.research.google.com/github/motional/nuplan-devkit/" class="" target="_none">nuPlan Sensor Data Tutorial</a> by Motional</li></a>
          </li>
          <li>
            <a href="https://colab.research.google.com/github/motional/nuplan-devkit/" class="" target="_none">nuPlan Framework</a> by Motional</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2106.11810" class="" target="_none">NuPlan: A closed-loop ML-based planning benchmark for autonomous vehicles</a> by Holger Caesar, Juraj Kabzan, Kok Seang Tan, Whye Kit Fong, Eric Wolff, Alex Lang, Luke Fletcher, Oscar Beijbom, Sammy Omari</li></a>
          </li>
          <li>
            <a href="https://colab.research.google.com/github/motional/nuplan-devkit/" class="" target="_none">nuPlan Advacned Model Training</a> by Motional</li></a>
          </li>
        </ul>
        <p><a href="/motional-nuplan/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="io-lulc" class="dataset">
          <h3><a href="/io-lulc/">10m Annual Land Use Land Cover (9-class)</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">land use</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>This dataset, produced by Impact Observatory, Microsoft, and Esri, displays a global map of land use and land cover (LULC)
derived from ESA Sentinel-2 imagery at 10 meter resolution for the years 2017 - 2023. Each map is a composite of LULC predictions for 9 classes throughout the year
in order to generate a representative snapshot of each year. This dataset was generated by Impact Observatory, which used billions of human-labeled pixels 
(curated by the National Geographic Society) to train a deep learning model for land classification. 
Each global map was produced by applying this model to ...</p>
          <p><a href="/io-lulc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.fastcompany.com/90729824/these-maps-from-satellite-data-show-how-much-earth-has-changed-in-only-five-years" class="" target="_none">These maps from satellite data show how much Earth has changed in only five years</a> by Fast Company</li></a>
          </li>
          <li>
            <a href="https://www.nytimes.com/2022/09/11/world/asia/pakistan-floods-food-crisis.html" class="" target="_none">‘Very Dire’: Devastated by Floods, Pakistan Faces Looming Food Crisis</a> by New York Times</li></a>
          </li>
          <li>
            <a href="https://map.unbiodiversitylab.org/earth?basemap&#x3D;grayscale&amp;coordinates&#x3D;20,0,2&amp;layers&#x3D;10m-annual-land-use-land-cover-9-class-01_100" class="" target="_none">View the dataset on UN Biodiversity Lab Map Viewer</a> by United Nations Development Programme</li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/document/9553499" class="" target="_none">Global land use / land cover with Sentinel 2 and deep learning</a> by K. Karra, C. Kontgis, Z. Statman-Weil, J. C. Mazzariello, M. Mathis and S. P. Brumby</li></a>
          </li>
          <li>
            <a href="https://medium.com/impactobservatoryinc/mapping-the-world-with-unmatched-frequency-54a83527f138" class="" target="_none">Mapping the world with unmatched frequency</a> by Mark Hannel</li></a>
          </li>
        </ul>
        <p><a href="/io-lulc/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="argoverse" class="dataset">
          <h3><a href="/argoverse/">Argoverse</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">robotics</span></p>
          <p>Home of the Argoverse datasets.Public datasets supported by detailed maps to test, experiment, and teach self-driving vehicles how to understand the world around them.This bucket includes the following datasets:<ol>
<li>Argoverse 1 (AV1)</li>
</ol>
<ul>
<li>Motion Forecasting</li>
<li>Tracking</li>
</ul>
<ol start="2">
<li>Argoverse 2 (AV2)</li>
</ol>
<ul>
<li>Motion Forecasting</li>
<li>Lidar</li>
<li>Sensor</li>
</ul>
<ol start="3">
<li>Trust, but Verify (TbV)</li>
</ol>
<ul>
<li>Map Change Detection</li>
</ul>
</p>
          <p><a href="/argoverse/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/argoverse/av2-api" class="" target="_none">Argoverse 2 API</a> by <a href="https://github.com/argoverse" target="_none">Argoverse Authors</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2301.00493" class="" target="_none">Argoverse 2: Next Generation Datasets for Self-Driving Perception and Forecasting</a> by Benjamin Wilson, William Qi, Tanmay Agarwal, John Lambert, Jagjeet Singh, Siddhesh Khandelwal, Bowen Pan, Ratnesh Kumar, Andrew Hartnett, Jhony Kaesemodel Pontes, Deva Ramanan, Peter Carr, James Hays</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/av2/" class="" target="_none">PyPi package for &#x60;av2&#x60;</a> by <a href="https://github.com/argoverse" target="_none">Argoverse Authors</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2212.07312" class="" target="_none">Trust, but Verify: Cross-Modality Fusion for HD Map Change Detection</a> by John Lambert, James Hays</li></a>
          </li>
          <li>
            <a href="https://anaconda.org/conda-forge/av2" class="" target="_none">conda-forge package for &#x60;av2&#x60;</a> by <a href="https://github.com/argoverse" target="_none">Argoverse Authors</a></li></a>
          </li>
        </ul>
        <p><a href="/argoverse/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="bossdb" class="dataset">
          <h3><a href="/bossdb/">BossDB Open Neuroimagery Datasets</a></h3>
          <p><span class="label label-info tag link-tag">calcium imaging</span><span class="label label-info tag link-tag">electron microscopy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">light-sheet microscopy</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span><span class="label label-info tag link-tag">volumetric imaging</span><span class="label label-info tag link-tag">x-ray</span><span class="label label-info tag link-tag">x-ray microtomography</span><span class="label label-info tag link-tag">x-ray tomography</span></p>
          <p>This data ecosystem, Brain Observatory Storage Service &amp; Database (BossDB), contains several neuro-imaging datasets across multiple modalities and scales, ranging from nanoscale (electron microscopy), to microscale (cleared lightsheet microscopy and array tomography), and mesoscale (structural and functional magnetic resonance imaging). Additionally, many of the datasets include dense segmentation and meshes.</p>
          <p><a href="/bossdb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1101/217745v2" class="" target="_none">The Block Object Storage Service (bossDB): A Cloud-Native Approach for Petascale Neuroscience Discovery</a> by <a href="https://bossdb.org/publications" target="_none">Robert Hider Jr., Dean M. Kleissas, Derek Pryor, Timothy Gion, Luis Rodriguez, Jordan Matelsky, William Gray-Roncal, Brock Wester</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41592-018-0181-1" class="" target="_none">A Community-Developed Open-Source Computational Ecosystem for Big Neuro Data</a> by J. T. Vogelstein, E. Perlman, B. Falk, A. Baden, W. Gray Roncal, V. Chandrashekhar, F. Collman, S. Seshamani, J. L. Patsolic, K. Lillaney, M. Kazhdan, R. Hider, D. Pryor, J. Matelsky, T. Gion, P. Manavalan, B. Wester, M. Chevillet, E. T. Trautman, K. Khairy, E. Bridgeford, D. M. Kleissas, D. J. Tward, A. K. Crow, B. Hsueh, M. A. Wright, M. I. Miller, S. J. Smith, R. J. Vogelstein, K. Deisseroth, and R. Burns</li></a>
          </li>
          <li>
            <a href="https://github.com/seung-lab/cloud-volume" class="" target="_none">CloudVolume</a> by Seung Lab</li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.05.15.098707v1" class="" target="_none">intern: Integrated Toolkit for Extensible and Reproducible Neuroscience</a> by <a href="https://bossdb.org/publications" target="_none">Jordan K Matelsky, Luis Rodriguez, Daniel Xenes, Timothy Gion, Robert Hider Jr., Brock Wester, William Gray-Roncal</a></li></a>
          </li>
          <li>
            <a href="https://bossdb.org" class="" target="_none">bossDB</a> by bossDB Team</li></a>
          </li>
        </ul>
        <p><a href="/bossdb/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="capella_opendata" class="dataset">
          <h3><a href="/capella_opendata/">Capella Space Synthetic Aperture Radar (SAR) Open Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>Open Synthetic Aperture Radar (SAR) data from Capella Space. Capella Space is an information services company
  that provides on-demand, industry-leading, high-resolution synthetic aperture radar (SAR) Earth observation
  imagery. Through a constellation of small satellites, Capella provides easy access to frequent, timely, and
  flexible information affecting dozens of industries worldwide. Capella&#39;s high-resolution SAR satellites are
  matched with unparalleled infrastructure to deliver reliable global insights that sharpen our understanding
  of the changing world – improving decisions ...</p>
          <p><a href="/capella_opendata/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://tiledb.com/blog/analyzing-lidar-and-sar-data-with-capella-space-and-tiledb" class="" target="_none">Analyzing LiDAR and SAR data with Capella Space and TileDB</a> by Stavros Papadopoulos</li></a>
          </li>
          <li>
            <a href="https://capella-console-client.readthedocs.io/en/main/" class="" target="_none">Python SDK for api.capellaspace.com</a> by <a href="https://www.capellaspace.com/" target="_none">Capella Space</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/tiledb/open-sar-data-and-scalable-analytics-bfd1a5257e9" class="" target="_none">Open SAR data and scalable analytics</a> by Norman Barker</li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/document/9764201/" class="" target="_none">Radar Generalized Image Quality Equation Applied to Capella Open Dataset</a> by Wade Schwartzkopf, Jason Brown, Gordon Farquharson, Craig Stringham, Michael Duersch, Jordan Heemskerk</li></a>
          </li>
          <li>
            <a href="https://github.com/capellaspace/tools/tree/master/capella-slc-reader" class="" target="_none">Single Look Complex data reader for Capella SLC images - python module to convert Capella SLC data into an amplitude image.</a> by <a href="https://www.capellaspace.com/" target="_none">Capella Space</a></li></a>
          </li>
        </ul>
        <p><a href="/capella_opendata/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="cptac-3" class="dataset">
          <h3><a href="/cptac-3/">Clinical Proteomic Tumor Analysis Consortium 3 (CPTAC-3)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the
understanding of the molecular basis of cancer through the application of large-scale proteome and
genome analysis, or proteogenomics. CPTAC-3 is the Phase III of the CPTAC Initiative. The dataset
contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/cptac-3/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29718670" class="" target="_none">Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics</a> by Clark DJ, Hu Y, Bocik W, Chen L, Schnaubelt M, Roberts R, Shah P, Whiteley G, Zhang H
</li></a>
          </li>
          <li>
            <a href="https://proteomics.cancer.gov/data-portal" class="" target="_none">CPTAC Data Portal</a> by <a href="https://proteomics.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.10.007" class="" target="_none">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</a> by Clark DJ, Dhanasekaran SM, Petralia F, Pan J, Song X, Hu Y, da Veiga Leprevost F, Reva B,
Lih TM, Chang HY, Ma W, Huang C, Ricketts CJ, Chen L1, Krek A, Li Y, Rykunov D, Li QK, Chen
LS, Ozbek U, Vasaikar S, Wu Y, Yoo S, Chowdhury S, Wyczalkowski MA, Ji J, Schnaubelt M, Kong
A, Sethuraman S, Avtonomov DM, Ao M, Colaprico A, Cao S, Cho KC, Kalayci S, Ma S, Liu W,
Ruggles K, Calinawan A, Gümüş ZH, Geizler D, Kawaler E, Teo GC, Wen B, Zhang Y, Keegan S, Li
K, Chen F, Edwards N, Pierorazio PM, Chen XS, Pavlovich CP, Hakimi AA, Brominski G, Hsieh
JJ, Antczak A, Omelchenko T, Lubinski J, Wiznerowicz M, Linehan WM, Kinsinger CR,
Thiagarajan M, Boja ES, Mesri M, Hiltke T, Robles AI, Rodriguez H, Qian J, Fenyö D, Zhang B,
Ding L, Schadt E, Chinnaiyan AM, Zhang Z, Omenn GS, Cieslik M, Chan DW, Nesvizhskii AI, Wang
P, Zhang H; Clinical Proteomic Tumor Analysis Consortium
</li></a>
          </li>
          <li>
            <a href="https://cgc-accounts.sbgenomics.com/" class="" target="_none">Cancer Genomics Cloud</a> by <a href="http://www.cancergenomicscloud.org/" target="_none">Seven Bridges</a></li></a>
          </li>
          <li>
            <a href="https://pdc.esacinc.com/pdc/browse" class="" target="_none">Proteomic Data Commons</a> by <a href="https://pdc.esacinc.com/pdc" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cptac-3/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="deafrica-waterbodies" class="dataset">
          <h3><a href="/deafrica-waterbodies/">DE Africa Waterbodies Monitoring Service</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info tag link-tag">water</span></p>
          <p>The Digital Earth Africa continental Waterbodies Monitoring Service identifies more than 700,000 water bodies from over three decades of satellite observations. This service maps persistent and seasonal water bodies and the change in their water surface area over time. Mapped water bodies may include, but are not limited to, lakes, ponds, man-made reservoirs, wetlands, and segments of some river systems.On a local, regional, and continental scale, this service helps improve our understanding of surface water dynamics and water availability and can be used for monitoring water bodies such as we...</p>
          <p><a href="/deafrica-waterbodies/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://maps.digitalearth.africa" class="" target="_none">Digital Earth Africa Map</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://sandbox.digitalearth.africa" class="" target="_none">Digital Earth Africa Sandbox</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://www.digitalearthafrica.org/node/601" class="" target="_none">Waterbodies Monitoring Service</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="https://github.com/digitalearthafrica/deafrica-sandbox-notebooks" class="" target="_none">Digital Earth Africa Notebook Repo</a> by Digital Earth Africa Contributors</li></a>
          </li>
          <li>
            <a href="http://learn.digitalearthafrica.org" class="" target="_none">Digital Earth Africa Training</a> by Digital Earth Africa Contributors</li></a>
          </li>
        </ul>
        <p><a href="/deafrica-waterbodies/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="geoschem-input-data" class="dataset">
          <h3><a href="/geoschem-input-data/">GEOS-Chem Input Data</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Input data for the GEOS-Chem Chemical Transport Model, includes NASA/GMAO MERRA-2 and GEOS-FP <a href="https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#met">meteorological products</a>, <a href="https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#chemistry-input-data">chemistry input data</a>, <a href="https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#emis-inputs">emissions input data</a>, and other smaller datasets such as model <a href="https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#initial-conditions-input-data%3E">initial conditions</a>.</p>
          <p><a href="/geoschem-input-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1175/BAMS-D-18-0243.1" class="" target="_none">Enabling Immediate Access to Earth Science Models through Cloud Computing: Application to the GEOS-Chem Model</a> by Jiawei Zhuang et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1029/2020MS002064" class="" target="_none">Enabling high-performance cloud computing for Earth science modeling on over a thousand cores: application to the GEOS-Chem atmospheric chemistry model</a> by Jiawei Zhuang et al.</li></a>
          </li>
          <li>
            <a href="https://gchp.readthedocs.io/en/latest/getting-started/quick-start.html" class="" target="_none">GCHP Quickstart Guide</a> by <a href="http://geos-chem.org/support-team" target="_none">GEOS-Chem Support Team</a></li></a>
          </li>
          <li>
            <a href="https://geos-chem.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/getting-started-aws.html" class="" target="_none">Getting started with GEOS-Chem Input Data</a> by <a href="http://geos-chem.org/support-team" target="_none">GEOS-Chem Support Team</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/gmd-15-8731-2022" class="" target="_none">Improved advection, resolution, performance, and community access in the new generation (version 13) of the high-performance GEOS-Chem global atmospheric chemistry model (GCHP)</a> by Randall V. Martin et al.</li></a>
          </li>
        </ul>
        <p><a href="/geoschem-input-data/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="geoschem-nested-input-data" class="dataset">
          <h3><a href="/geoschem-nested-input-data/">GEOS-Chem Nested Input Data</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Input data for nested-grid simulations using the GEOS-Chem Chemical Transport Model. This includes the NASA/GMAO MERRA-2 and GEOS-FP <a href="https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#met">meteorological products</a>, the <a href="https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#emis-inputs">HEMCO emission inventories</a>, and other small data such as <a href="https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/restart-files.html">model initial conditions</a>.</p>
          <p><a href="/geoschem-nested-input-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://imi.seas.harvard.edu/" class="" target="_none">Integrated Methane Inversion</a> by <a href="https://imi.seas.harvard.edu/people" target="_none">Daniel Varon et al.</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1029/2020MS002064" class="" target="_none">Enabling high-performance cloud computing for Earth science modeling on over a thousand cores: application to the GEOS-Chem atmospheric chemistry model</a> by Jiawei Zhuang et al.</li></a>
          </li>
          <li>
            <a href="https://gchp.readthedocs.io/en/latest/getting-started/quick-start.html" class="" target="_none">GCHP Quickstart Guide</a> by <a href="http://geos-chem.org/support-team" target="_none">GEOS-Chem Support Team</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1175/BAMS-D-18-0243.1" class="" target="_none">Enabling Immediate Access to Earth Science Models through Cloud Computing: Application to the GEOS-Chem Model</a> by Jiawei Zhuang et al.</li></a>
          </li>
          <li>
            <a href="https://geos-chem.readthedocs.io/en/latest/getting-started/quick-start.html" class="" target="_none">GEOS-Chem Classic Quickstart Guide</a> by <a href="http://geos-chem.org/support-team" target="_none">GEOS-Chem Support Team</a></li></a>
          </li>
        </ul>
        <p><a href="/geoschem-nested-input-data/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="hycom-global-drifters" class="dataset">
          <h3><a href="/hycom-global-drifters/">HYCOM-OceanTrack Integrated HYCOM Eulerian Fields and Lagrangian Trajectories Dataset</a></h3>
          <p><span class="label label-info tag link-tag">drifters</span><span class="label label-info tag link-tag">Eulerian</span><span class="label label-info tag link-tag">HYCOM</span><span class="label label-info tag link-tag">Lagrangian</span><span class="label label-info tag link-tag">numerical particle</span><span class="label label-info tag link-tag">ocean circulation</span><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean sea surface height</span><span class="label label-info tag link-tag">ocean simulation</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>A combined dataset of simulated ocean sea surface height, near-surface velocities, and particle trajectories from a global 1/25th degree HYbrid Coordinate Ocean Model (HYCOM) 1-year run.</p>
          <p><a href="/hycom-global-drifters/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.5281/zenodo.12796468" class="" target="_none">HYCOM-OceanTrack: HYCOM-OceanTrack: Integrated HYCOM Eulerian Fields and Lagrangian Trajectories Dataset (v1.0)</a> by Shane Elipot</li></a>
          </li>
          <li>
            <a href="https://github.com/selipot/hycom-oceantrack/tree/main/tutorials/" class="" target="_none">HYCOM Lagrangian tutorial</a> by <a href="https://selipot.github.io" target="_none">Shane Elipot</a></li></a>
          </li>
          <li>
            <a href="https://github.com/Cloud-Drift/clouddrift" class="" target="_none">clouddrift</a> by <a href="https://selipot.github.io" target="_none">Shane Elipot</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41597-024-03813-z" class="" target="_none">An integrated dataset of near-surface Eulerian fields and Lagrangian trajectories from an ocean model</a> by Shane Elipot</li></a>
          </li>
          <li>
            <a href="https://github.com/selipot/hycom-oceantrack/tree/main/tutorials/" class="" target="_none">HYCOM Eulerian tutorial</a> by <a href="https://selipot.github.io" target="_none">Shane Elipot</a></li></a>
          </li>
        </ul>
        <p><a href="/hycom-global-drifters/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="hycom-gofs-3pt1-reanalysis" class="dataset">
          <h3><a href="/hycom-gofs-3pt1-reanalysis/">HYbrid Coordinate Ocean Model Global Ocean Forecast System Reanalysis</a></h3>
          <p><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>Global Ocean Forecasting System (GOFS) 3.1 output on the GLBv0.08 grid. The resolution is 0.08° resolution between 40°S and 40°N, 0.04° poleward of these latitudes. The temportal frequenct is 3 hourly. This data was created by the Naval Research Laboratory: Ocean Dynamics and Prediction Branch.</p>
          <p><a href="/hycom-gofs-3pt1-reanalysis/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1007/978-3-642-35088-7_13" class="" target="_none">Variational Data Assimilation for the Global Ocean</a> by James A. Cummings, Ole Martin Smedstad</li></a>
          </li>
          <li>
            <a href="https://github.com/HYCOM/HYCOM-examples" class="" target="_none">HYCOM-examples repository</a> by <a href="https://www.coaps.fsu.edu/alexandra-bozec" target="_none">Dr. Alexandra Bozec</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1256/qj.05.105" class="" target="_none">Operational multivariate ocean data assimilation</a> by James A. Cummings</li></a>
          </li>
          <li>
            <a href="https://www7320.nrlssc.navy.mil/pubs/2017/metzger-2017.pdf" class="" target="_none">GOFS 3.1 Validation Test Report</a> by E. Joseph Metzger, Naval Research Laboratory</li></a>
          </li>
          <li>
            <a href="https://apps.dtic.mil/sti/pdfs/ADA585251.pdf" class="" target="_none">Validation Test Report for the Improved Synthetic Ocean Profile (ISOP) System, Part I: Synthetic Profile Methods and Algorithm</a> by NAVAL RESEARCH LAB STENNIS DETACHMENT STENNIS SPACE CENTER MS OCEANOGRAPHY DIV</li></a>
          </li>
        </ul>
        <p><a href="/hycom-gofs-3pt1-reanalysis/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="ibl-reproducible-ephys" class="dataset">
          <h3><a href="/ibl-reproducible-ephys/">IBL Neuropixels Reproducible Ephys Data on AWS</a></h3>
          <p><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span><span class="label label-info tag link-tag">open source software</span></p>
          <p>Electrophysiological recordings acquired using Neuropixels probes in different mice and labs, targeting the same brain locations (including posterior parietal cortex, hippocampus, and thalamus).</p>
          <p><a href="/ibl-reproducible-ephys/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://int-brain-lab.github.io/iblenv/loading_examples.html#" class="" target="_none">Loading Data</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1101/2022.05.09.491042" class="" target="_none">Reproducibility of in-vivo electrophysiological measurements in mice</a> by <a href="www.internationalbrainlab.com" target="_none">International Brain Laboratory et al.</a></li></a>
          </li>
          <li>
            <a href="https://viz.internationalbrainlab.org/app" class="" target="_none">Data Explorer</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
          <li>
            <a href="https://int-brain-lab.github.io/iblenv/notebooks_external/data_download.html" class="" target="_none">Download the public datasets</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/ONE-api/" class="" target="_none">ONE-api on pypi.python.org - python API to query and download neurophysiology data</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
        </ul>
        <p><a href="/ibl-reproducible-ephys/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-rrfs" class="dataset">
          <h3><a href="/noaa-rrfs/">NOAA Rapid Refresh Forecast System (RRFS) [Prototype]</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Rapid Refresh Forecast System (RRFS) is the National Oceanic and Atmospheric Administration’s (NOAA) next generation convection-allowing, rapidly-updated ensemble prediction system, currently scheduled for operational implementation in 2026. The operational configuration will feature a 3 km grid covering North America and include deterministic forecasts every hour out to 18 hours, with deterministic and ensemble forecasts to 60 hours four times per day at 00, 06, 12, and 18 UTC.The RRFS will provide guidance to support forecast interests including, but not limited to, aviation, severe convective weather, renewable energy, heavy precipitation, and winter weather on timescales where rapidly-updated guidance is particularly useful.<br/><br/>The RRFS is underpinned by the <a href="https://ufscommunity.org/">Unified Forecast System (UFS)</a>, a community-based Earth modeling initiative, and benefits from collaborative development efforts across NOAA, academia, and research institutions.<br/><br/>This bucket provides access to real time, experimental RRFS prototype output.  And will provide access to final retrospective output once completed.<br/><br/><hr/>

The real-time RRFS prototype is experimental and evolving. [<b> Real-time RRFS output will cease to be generated for several months beginning on ~3 December 2024.  This step is being taken to allow for final retrospective testing to be completed. </b>] It is not under 24x7 monitoring and is not operational. Output may be delayed or missing. Outputs will change. When significant changes to output take place, this description will be updated.<br/><br/>We currently provide hourly deterministic forecasts at 3 km grid spacing out to 60 hours at 00, 06, 12, and 18 UTC, and out to 18 hours for other cycles. Output is organized by cycle date and cycle hour.For example, <code>rrfs_a/rrfs_a.20241201/12/control</code> contains the deterministic forecast initialized at 12 UTC on 01 December 2024. Users will find two types of output in GRIB2 format. The first is:<br/><br/>
<code>rrfs.t12z.natlev.f018.grib2</code> <br/><br/>Meaning that this is the RRFS_A initialized at 12 UTC, covers the full North America domain, and is the native level post-processed gridded data at hour 18.  This output is on a rotated latitude-longitude grid at 3 km grid spacing.<br/><br/>The second output file in grib2 format is:<br/><br/><code>rrfs.t12z.prslev.f018.conus.grib2</code> <br/><br/>The “prslev” descriptor indicates that this post-processed gridded data is output on pressure levels. The “conus” descriptor indicates that it covers the contiguous United StatesFor users interested in other domains, output is provided on the full 3-km North American grid and also subset over Alaska, Hawaii, and Puerto Rico. The files are identified as follows: <br/><br/>North America: <code>rrfs.t00z.prslev.f002.grib2</code>
Alaska: <code>rrfs.t00z.prslev.f002.ak.grib2</code>
Hawaii: <code>rrfs.t00z.prslev.f002.hi.grib2</code>
Puerto Rico: <code>rrfs.t00z.prslev.f002.pr.grib2</code><br/><br/>Beginning on December 8th, 2023 we now provide prototype RRFSv1 ensemble output and products. Output is available for 00, 06, 12, and 18 UTC cycles, and is organized by cycle date and cycle hour. For example, <code>rrfs_a/rrfs_a.20231214/00/mem0001</code> contains the forecast from member 1, and <code>rrfs_a/rrfs_a.20231214/00/enspost_timelag</code> ...</p>
          <p><a href="/noaa-rrfs/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://epic.noaa.gov/wp-content/uploads/2022/07/1.-SRW-Carley.pdf" class="" target="_none">Status and Opportunities with the Rapid Refresh Forecast System</a> by Carley J. R. and C. R. Alexander</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1029/2021MS002483" class="" target="_none">A Limited Area Modeling Capability for the Finite-Volume Cubed-Sphere (FV3) Dynamical Core and Comparison With a Global Two-Way Nest</a> by Black, T. L., J. A. Abeles, B. T. Blake, D. Jovic, E. Rogers, X. Zhang, E. A. Aligo, L. C. Dawson, Y. Lin, E. Strobach, P. C. Shafran, and J. R. Carley</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/gmd-15-6891-2022" class="" target="_none">Assessment of the data assimilation framework for the Rapid Refresh Forecast System v0.1 and impacts on forecasts of a convective storm case study</a> by Banos, I. H., W. D. Mayfield, G. Ge, L. F. Sapucci, J. R. Carley, and L. Nance</li></a>
          </li>
          <li>
            <a href="https://ams.confex.com/ams/101ANNUAL/meetingapp.cgi/Paper/381608" class="" target="_none">Prototype UFS-Based Rapid Refresh Forecast System (RRFS) on the Cloud</a> by Holt, C., D. Abdi, J. A. Abeles, J. R. Carley, C. W. Harrop, R. Panda, S. Trahan, and C. R. Alexander</li></a>
          </li>
          <li>
            <a href="https://ams.confex.com/ams/102ANNUAL/meetingapp.cgi/Paper/391702" class="" target="_none">Highlights from a Year of Continued Development of the Rapid Refresh Forecast System (RRFS)</a> by Carley J. R. and C. R. Alexander</li></a>
          </li>
        </ul>
        <p><a href="/noaa-rrfs/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="nyu-fastmri" class="dataset">
          <h3><a href="/nyu-fastmri/">NYU Langone &amp; FAIR FastMRI Dataset</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span></p>
          <p>This dataset contains deidentified raw k-space data and DICOM image files of over 1,500 knees and 6,970 brains.</p>
          <p><a href="/nyu-fastmri/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/@khodges42/how-to-get-started-diy-neurohacking-with-the-fastmri-dataset-and-why-you-should-6a02a6bb896e" class="" target="_none">How to Get Started Working with the FastMRI Dataset (and why you should!)</a> by Kevin Hodges</li></a>
          </li>
          <li>
            <a href="https://github.com/facebookresearch/fastMRI/blob/master/fastMRI_tutorial.ipynb" class="" target="_none">FastMRI Tutorial (Jupyter Notebook)</a> by <a href="https://scholar.google.com/citations?user&#x3D;LjjLn2YAAAAJ&amp;hl&#x3D;en" target="_none">Tullie Murrell</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29774597" class="" target="_none">Assessment of the generalization of learned image reconstruction and the potential for transfer learning.</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2019)</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/1904.01112" class="" target="_none">Deep Learning Methods for Parallel Magnetic Resonance Image Reconstruction</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2019)</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2001.02518" class="" target="_none">Advancing machine learning for MR image reconstruction with an open competition:Overview of the 2019 fastMRI challenge</a> by <a href="https://arxiv.org/search/eess?searchtype&#x3D;author&amp;query&#x3D;Knoll%2C+F" target="_none">Knoll et al (2020)</a></li></a>
          </li>
        </ul>
        <p><a href="/nyu-fastmri/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="nyc-tlc-trip-records-pds" class="dataset">
          <h3><a href="/nyc-tlc-trip-records-pds/">New York City Taxi and Limousine Commission (TLC) Trip Record Data</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">urban</span></p>
          <p>Data of trips taken by taxis and for-hire vehicles in New York City. Note: access to this dataset is free, however direct S3 access does require an AWS account. Anonymous downloads are accessible from the dataset&#39;s documentation webpage listed below.</p>
          <p><a href="/nyc-tlc-trip-records-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/build-a-real-time-stream-processing-pipeline-with-apache-flink-on-aws/" class="aws-link" target="_none">Build a Real-time Stream Processing Pipeline with Apache Flink on AWS</a> by Steffen Hausmann</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/build-and-run-streaming-applications-with-apache-flink-and-amazon-kinesis-data-analytics-for-java-applications/" class="aws-link" target="_none">Build and run streaming applications with Apache Flink and Amazon Kinesis Data Analytics for Java Applications</a> by Steffen Hausmann</li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/cloud-experiments/tree/master/experiments/notebooks/exploring-data" class="aws-link" target="_none">Exploring data with Python and Amazon S3 Select</a> by Manav Sehgal</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/machine-learning/machine-learning-on-distributed-dask-using-amazon-sagemaker-and-aws-fargate/" class="aws-link" target="_none">Machine learning on distributed Dask using Amazon SageMaker and AWS Fargate</a> by Ram Vittal</li></a>
          </li>
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;MzTZp47Jy7E" class="" target="_none">Deep Dive on Flink &amp; Spark on Amazon EMR</a> by Keith Steward</li></a>
          </li>
        </ul>
        <p><a href="/nyc-tlc-trip-records-pds/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="open-bio-ref-data" class="dataset">
          <h3><a href="/open-bio-ref-data/">Open Bioinformatics Reference Data for Galaxy</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>This dataset provides genomic reference data and software packages for use with <a href="https://galaxyproject.org/">Galaxy</a> and <a href="https://bioconductor.org/">Bioconductor</a> applications. The reference data is available for hundreds of reference genomes and has been formatted for use with a variety of tools. The available configuration files make this data easily incorporable with a local Galaxy server without additional data preparation. Additionally, Bioconductor&#39;s AnnotationHub and ExperimentHub data are provided for use via R packag...</p>
          <p><a href="/open-bio-ref-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1093/bioinformatics/btu119" class="" target="_none">Wrangling Galaxy&#x27;s reference data</a> by Daniel Blankenberg, James E. Johnson, The Galaxy Team, James Taylor, Anton Nekrutenko</li></a>
          </li>
          <li>
            <a href="https://bioconductor.org/" class="" target="_none">Bioconductor</a> by Bioconductor Project</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5862039/" class="" target="_none">Accessible, curated metagenomic data through ExperimentHub</a> by Edoardo Pasolli, Lucas Schiffer, Paolo Manghi, Audrey Renson, Valerie Obenchain, Duy Tin Truong, Francesco Beghini, Faizan Malik, Marcel Ramos, Jennifer B Dowd, Curtis Huttenhower, Martin Morgan, Nicola Segata, and Levi Waldron</li></a>
          </li>
          <li>
            <a href="https://usegalaxy.org/" class="" target="_none">Galaxy</a> by Galaxy Project</li></a>
          </li>
          <li>
            <a href="https://github.com/openbiorefdata/tutorials/" class="" target="_none">Using Open Bio Ref Data with Galaxy and Bioconductor</a> by Enis Afgan, Alexandru Mahmoud, Nuwan Goonasekera</li></a>
          </li>
        </ul>
        <p><a href="/open-bio-ref-data/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="grillo-openeew" class="dataset">
          <h3><a href="/grillo-openeew/">OpenEEW</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span></p>
          <p>Grillo has developed an IoT-based earthquake early-warning system,
with sensors currently deployed in Mexico, Chile, Puerto Rico and Costa Rica,
and is now opening its entire archive of unprocessed accelerometer
data to the world to encourage the development of new algorithms
capable of rapidly detecting and characterizing earthquakes in
real time.</p>
          <p><a href="/grillo-openeew/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://blog.grillo.io/detect-earthquakes-in-chile-in-real-time-from-anywhere-in-the-world-ff7386368f41" class="" target="_none">Detect earthquakes in Chile in real time — from anywhere in the world</a> by <a href="https://blog.grillo.io/" target="_none">Michael Allman</a></li></a>
          </li>
          <li>
            <a href="https://blog.grillo.io/analyzing-a-magnitude-7-2-earthquake-in-mexico-using-python-6272a4ff63e3" class="" target="_none">Analyzing a magnitude 7.2 earthquake in Mexico using Python</a> by <a href="https://blog.grillo.io/" target="_none">Michael Allman</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.31223/X5XS47" class="" target="_none">Evaluation of the Grillo sensor, a low-cost accelerometer for IoT-based Real-time seismology</a> by Vaclav Matej Kuna, Diego Melgar, Andres Meira</li></a>
          </li>
          <li>
            <a href="https://library.seg.org/doi/epdfplus/10.1190/image2022-3751420.1" class="" target="_none">NepalEEW: Testing the feasibility of an Earthquake Early Warning System in Nepal</a> by Vaclav M. Kuna</li></a>
          </li>
          <li>
            <a href="https://github.com/openeew/openeew-python" class="" target="_none">OpenEEW library for Python</a> by <a href="https://grillo.io/" target="_none">Grillo</a></li></a>
          </li>
        </ul>
        <p><a href="/grillo-openeew/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="pacific-sound" class="dataset">
          <h3><a href="/pacific-sound/">Pacific Ocean Sound Recordings</a></h3>
          <p><span class="label label-info tag link-tag">acoustics</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">marine mammals</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">open source software</span></p>
          <p>This project offers passive acoustic data (sound recordings) from a deep-ocean environment off central California.  Recording began in July 2015, has been nearly continuous, and is ongoing.  These resources are intended for applications
in ocean soundscape research, education, and the arts.</p>
          <p><a href="/pacific-sound/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.mbari.org/pacific-sound/" class="" target="_none">Tutorials on machine learning and signal processing methods for anthropogenic and cetacean study using the pacific-sound archives</a> by <a href="https://www.mbari.org/" target="_none">MBARI</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.3389/fmars.2020.00142" class="" target="_none">Seal bomb noise as a potential threat to Monterey Bay harbor porpoise</a> by Simonis et al. (2020)</li></a>
          </li>
          <li>
            <a href="http://doi.org/10.1109/OCEANS.2016.7761363" class="" target="_none">New Passive Acoustic Monitoring in Monterey Bay National Marine Sanctuary</a> by Ryan et al. (2016)</li></a>
          </li>
          <li>
            <a href="https://www.frontiersin.org/articles/10.3389/fmars.2021.656566" class="" target="_none">Reduction of Low-Frequency Vessel Noise in Monterey Bay National Marine Sanctuary During the COVID-19 Pandemic</a> by Ryan et al. (2021)</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cub.2020.08.105" class="" target="_none">Animal-borne metrics enable acoustic detection of blue whale migration</a> by Oestreich et al. (2020)</li></a>
          </li>
        </ul>
        <p><a href="/pacific-sound/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-pds-porotomo" class="dataset">
          <h3><a href="/nrel-pds-porotomo/">PoroTomo</a></h3>
          <p><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">geothermal</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">seismology</span></p>
          <p>Released to the public as part of the Department of Energy&#39;s Open Energy Data
Initiative, these data represent vertical and horizontal distributed acoustic
sensing (DAS) data collected as part of the Poroelastic Tomography (PoroTomo)
project funded in part by the Office of Energy Efficiency and Renewable
Energy (EERE), U.S. Department of Energy.</p>
          <p><a href="/nrel-pds-porotomo/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/openEDI/documentation/blob/master/PoroTomo/PoroTomo_Distributed_Acoustic_Sensing_(DAS)_Data_hsds.ipynb" class="" target="_none">PoroTomo DAS Data Processing Tutorial for hdf5 Files via HSDS and h5pyd</a> by <a href="https://orcid.org/0000-0001-7776-2028" target="_none">Michael Rossol and Nicole Taverna</a></li></a>
          </li>
          <li>
            <a href="https://pangea.stanford.edu/ERE/pdf/IGAstandard/SGW/2018/Miller.pdf" class="" target="_none">DAS and DTS at Brady Hot Springs: Observations about Coupling and Coupled Interpretations</a> by Douglas E. Miller, Thomas Coleman, Xiangfang Zeng, Jeremy R. Patterson , Elena C. Reinnisch, Michael A. Cardiff, Herbert F. Wang, Dante Fratta, Whitney Trainor-Guitton, Clifford H. Thurber, Michelle ROBERTSON, Kurt FEIGL, and The PoroTomo Team</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1093/gji/ggy182" class="" target="_none">Ground motion response to an ML 4.3 earthquake using co-located distributed acoustic sensing and seismometer arrays</a> by Herbert F Wang, Xiangfang Zeng, Douglas E Miller, Dante Fratta, Kurt L Feigl, Clifford H Thurber, Robert J Mellors</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.2172/1499141" class="" target="_none">PoroTomo Final Technical Report: Poroelastic Tomography by Adjoint Inverse Modeling of Data from Seismology, Geodesy, and Hydrology</a> by Kurt L. Feigl, Lesley M. Parker, and the PoroTomo Team</li></a>
          </li>
          <li>
            <a href="https://github.com/openEDI/documentation/blob/master/PoroTomo/PoroTomo_Distributed_Acoustic_Sensing_(DAS)_Data_SEGY.ipynb" class="" target="_none">PoroTomo DAS Data Processing Tutorial for SEG-Y Files</a> by <a href="https://orcid.org/0000-0001-7776-2028" target="_none">Nicole Taverna and Ross Ring-Jarvi</a></li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-porotomo/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="rareplanes" class="dataset">
          <h3><a href="/rareplanes/">RarePlanes</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>RarePlanes is a unique open-source machine learning dataset from CosmiQ Works and AI.Reverie that incorporates both real and synthetically generated satellite imagery. The RarePlanes dataset specifically focuses on the value of AI.Reverie synthetic data to aid computer vision algorithms in their ability to automatically detect aircraft and their attributes in satellite imagery. Although other synthetic/real combination datasets exist, RarePlanes is the largest openly-available very high resolution dataset built to test the value of synthetic data from an overhead perspective. The real portion ...</p>
          <p><a href="/rareplanes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/avanetten/yoltv4/blob/master/notebooks/train_test_pipeline.ipynb" class="" target="_none">Notebook for training and testing YOLYv4</a> by Adam Van Etten</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2006.02963" class="" target="_none">RarePlanes: Synthetic Data Takes Flight</a> by Jacob Shermeyer, Thomas Hossler, Adam Van Etten, Daniel Hogan, Ryan Lewis, Daeil Kim</li></a>
          </li>
          <li>
            <a href="https://dev.to/spara_50/getting-started-with-yoltv4-for-object-detection-in-imagery-getting-training-data-2055" class="" target="_none">Getting Started with YOLTv4 for Object Detection in Imagery: Getting Training Data</a> by <a href="https://authory.com/sophiaparafina" target="_none">Sophia Parafina</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aireveries/RarePlanes" class="" target="_none">RarePlanes Codebase</a> by Thomas Hossler and Jacob Shermeyer</li></a>
          </li>
          <li>
            <a href="https://towardsdatascience.com/announcing-yoltv4-improved-satellite-imagery-object-detection-f5091e913fad" class="" target="_none">Announcing YOLTv4: Improved Satellite Imagery Object Detection</a> by <a href="https://medium.com/@avanetten" target="_none">Adam Van Etten</a></li></a>
          </li>
        </ul>
        <p><a href="/rareplanes/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="serratus-lovelywater" class="dataset">
          <h3><a href="/serratus-lovelywater/">Serratus: Ultra-deep Search for Novel Viruses - Versioned Data Release</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">SARS-CoV-2</span><span class="label label-info tag link-tag">virus</span></p>
          <p>Serratus is a collaborative open science project for ultra-rapid discovery of known and unknown coronaviruses in response to the COVID-19 pandemic through re-analysis of publicly available genomic data. Our resulting vertebrate viral alignment data is explorable via the <a href="https://serratus.io/explorer">Serratus Explorer</a> and directly accessible on Amazon S3.</p>
          <p><a href="/serratus-lovelywater/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.07.28.224584v2" class="" target="_none">coronaSPAdes. From biosynthetic gene clusters to RNA viral assemblies</a> by Meleshko D., Hajirasouliha I., and Korobeynikov A. (2021)</li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.08.07.241729v2" class="" target="_none">Petabase-scale sequence alignment catalyses viral discovery</a> by Edgar R., Taylor J., Lin V., et al (2021)</li></a>
          </li>
          <li>
            <a href="https://www.pnas.org/content/118/3/e2019907118" class="" target="_none">Diversification of mammalian deltaviruses by host shifting</a> by Bergner L.M., Orton R.J., et al (2021)</li></a>
          </li>
          <li>
            <a href="https://serratus.io/explorer" class="" target="_none">Serratus Explorer</a> by Serratus Team</li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2021.03.02.433648v1" class="" target="_none">Ribovirus classification by a polymerase barcode sequence</a> by Babaian A., and Edgar R. (2021)</li></a>
          </li>
        </ul>
        <p><a href="/serratus-lovelywater/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="sdoml-fdl" class="dataset">
          <h3><a href="/sdoml-fdl/">Solar Dynamics Observatory (SDO) Machine Learning Dataset</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">NASA SMD AI</span></p>
          <p>The v1 dataset includes AIA/HMI observations 2010-2018 and v2 includes AIA/HMI observations 2010-2020 in all 10 wavebands (94A, 131A, 171A, 193A, 211A, 304A, 335A, 1600A, 1700A, 4500A), with 512x512 resolution and 6 minutes cadence; HMI vector magnetic field observations in Bx, By, and Bz components, with 512x512 resolution and 12 minutes cadence; The EVE observations in 39 wavelengths from 2010-05-01 to 2014-05-26, with 10 seconds cadence.</p>
          <p><a href="/sdoml-fdl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://adsabs.harvard.edu/abs/2019ApJS..242....7G" class="" target="_none">A Machine-learning Data Set Prepared from the NASA Solar Dynamics Observatory Mission</a> by Galvez, Richard; Fouhey, David F.; Jin, Meng; Szenicer, Alexandre; et al</li></a>
          </li>
          <li>
            <a href="https://github.com/SDOML/SDOML" class="" target="_none">Scripts for generating the SDOMLv1 dataset</a> by Fouhey, David F.; Jin, Meng</li></a>
          </li>
          <li>
            <a href="https://github.com/spaceml-org/helionb-sdoml" class="" target="_none">ML applications based on the SDOMLv2 dataset</a> by Wright, Paul J.</li></a>
          </li>
          <li>
            <a href="https://github.com/SDOML/SDOMLv2" class="" target="_none">Scripts for generating the SDOMLv2 dataset</a> by Jin, Meng</li></a>
          </li>
          <li>
            <a href="https://gitlab.com/frontierdevelopmentlab/living-with-our-star/expanding-sdo-capabilities" class="" target="_none">ML applications based on the SDOMLv1 dataset</a> by Salvatelli, Valentina; dos Santos, Luiz F. G.; Bose, Souvik; Neuberg, Brad; Cheung, Mark C. M.; Janvier, Miho; Jin, Meng; Gal, Yarin; Boerner, Paul; Baydin, Atılım Güneş</li></a>
          </li>
        </ul>
        <p><a href="/sdoml-fdl/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="dcc" class="dataset">
          <h3><a href="/dcc/">The MIT Supercloud Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cloud computing</span><span class="label label-info tag link-tag">datacenter</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">HPC</span><span class="label label-info tag link-tag">workload analysis</span></p>
          <p>Collection of parsed datacenter logs and time series data of hardware utilization from the MIT Supercloud system.</p>
          <p><a href="/dcc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MIT-AI-Accelerator/workload-classification" class="" target="_none">The MIT Supercloud Workload Classification Challenge</a> by Benny J. Tang, Qiqi Chen, Matthew L. Weiss, et. al.</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2108.02037" class="" target="_none">The MIT Supercloud Dataset</a> by Siddharth Samsi, Matthew Weiss, David Bestor, et. al.</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2204.05839" class="" target="_none">The MIT Supercloud Workload Classification Challenge</a> by <a href="https://github.com/MIT-AI-Accelerator/workload-classification" target="_none">Benny J. Tang, Qiqi Chen, Matthew L. Weiss, et. al.</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/MIT-AI-Accelerator/MIT-Supercloud-Dataset/blob/main/notebooks/Intro-to-MIT-Supercloud-Dataset.ipynb" class="" target="_none">Introduction to the MIT Supercloud Dataset</a> by Matthew Weiss</li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/MIT-AI-Accelerator/MIT-Supercloud-Dataset/blob/main/notebooks/Intro-to-MIT-Supercloud-Dataset.ipynb" class="" target="_none">Generating the labelled dataset</a> by <a href="https://github.com/MIT-AI-Accelerator/workload-classification" target="_none">Matthew Weiss</a></li></a>
          </li>
        </ul>
        <p><a href="/dcc/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="census-2010-dhc-nmf" class="dataset">
          <h3><a href="/census-2010-dhc-nmf/">2010 Census Production Settings Demographic and Housing Characteristics (DHC) Demonstration Noisy Measurement File</a></h3>
          <p><span class="label label-info tag link-tag">census</span><span class="label label-info tag link-tag">differential privacy</span><span class="label label-info tag link-tag">disclosure avoidance</span><span class="label label-info tag link-tag">ethnicity</span><span class="label label-info tag link-tag">group quarters</span><span class="label label-info tag link-tag">hispanic</span><span class="label label-info tag link-tag">housing</span><span class="label label-info tag link-tag">housing units</span><span class="label label-info tag link-tag">latino</span><span class="label label-info tag link-tag">noisy measurements</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">race</span><span class="label label-info tag link-tag">redistricting</span><span class="label label-info tag link-tag">voting age</span></p>
          <p>The 2010 Census Production Settings Demographic and Housing Characteristics (DHC) Demonstration Noisy Measurement File (2023-06-30) is an intermediate output of the 2020 Census Disclosure Avoidance System (DAS) TopDown Algorithm (TDA) (as described in Abowd, J. et al [2022] <a href="https://doi.org/10.1162/99608f92.529e3cb9">https://doi.org/10.1162/99608f92.529e3cb9</a> , and implemented in <a href="https://github.com/uscensusbureau/DAS_2020_Redistricting_Production_Code">https://github.com/uscensusbureau/DAS_2020_Redistricting_Production_Code</a>). The NMF was produced using the official “production settings,” the final set of algorithmic parameters and privacy-loss budget allocations, that were used to produce the 2020 Census Redistricting Data (P.L. 94-171) Summary File and the 2020 Census Demographic and Housing Characteristics File.  The NMF consists of the full set of privacy-protected statistical queries (counts of individuals or housing units with particular combinations of characteristics) of confidential 2010 Census data relating to the 2010 Demonstration Data Products Suite – Redistricting (P.L. 94-171) and Demographic and Housing Characteristics File – Production Settings (2023-04-03). These statistical queries, called “noisy measurements” were produced under the zero-Concentrated Differential Privacy framework (Bun, M. and Steinke, T [2016] <a href="https://arxiv.org/abs/1605.02065">https://arxiv.org/abs/1605.02065</a>; see also Dwork C. and Roth, A. [2014] <a href="https://www.cis.upenn.edu/~aaroth/Papers/privacybook.pdf">https://www.cis.upenn.edu/~aaroth/Papers/privacybook.pdf</a>) implemented via the discrete Gaussian mechanism (Cannone C., et al., [2023] <a href="https://arxiv.org/abs/2004.00010">https://arxiv.org/abs/2004.00010</a>), which added positive or negative integer-valued noise to each of the resulting counts. The noisy measurements are an intermediate stage of the TDA prior to the post-processing the TDA then performs to ensure internal and hierarchical consistency within the resulting tables. The Census Bureau has released these 2010 Census demonstration data to enable data users to evaluate the expected impact of disclosure avoidance variability on 2020 Census data. The 2010 Census Production Settings Demographic and Housing Characteristics (DHC) Demonstration Noisy Measurement File (2023-04-03) has been cleared for public dissemination by the Census Bureau Disclosure Review Board (CBDRB-FY22-DSEP-004). <br/> <br/> The 2010 Census Production Settings Demographic and Housing Characteristics Demonstration Noisy Measurement File includes zero-Concentrated Differentially Private (zCDP) (Bun, M. and Steinke, T [2016]) noisy measurements, implemented via the discrete Gaussian mechanism. These are estimated counts of individuals and housing units included in the 2010 Census Edited File (CEF), which includes confidential data initially collected in the 2010 Census of Population and Housing. The noisy measurements included in this file were subsequently post-processed by the TopDown Algorithm (TDA) to produce the 2010 Census Production Settings Privacy-Protected Microdata File - Redistricting (P.L. 94-171) and Demographic and Housing Characteristics File (2023-04-03) (<a href="https://www2.census.gov/programs-surveys/decennial/2020/program-management/data-product-planning/2010-demonstration-data-products/04-Demonstration_Data_Products_Suite/2023-04-03/">https://www2.census.gov/programs-surveys/decennial/2020/program-management/data-product-planning/2010-demonstration-data-products/04-Demonstration_Data_Products_Suite/2023-04-03/</a>). As these 2010 Census demonstration data are intended to support study of the design and expected impacts of the 2020 Disclosure Avoidance System, the 2010 CEF records were pre-processed before application of the zCDP framework. This pre-processing converted the 2010 CEF records into the input-file format, response codes, and tabulation categories used for the 2020 Census, which differ in substantive ways from the format, response codes, and tabulation categories originally used for the 2010 Census. <br/> <br/> The NMF provides estimates ...</p>
          <p><a href="/census-2010-dhc-nmf/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1162/99608f92.529e3cb9" class="" target="_none">The 2020 Census Disclosure Avoidance System Topdown Algorithm</a> by Abowd, J., Ashmead, R., Cumings-Menon, R., Garfinkel, S., Heineck, M., Heiss, C., Johns, R., Kifer, D., Leclerc, P., Machanavajjhala, A., Moran, B., Sexton, W., Spence, M., Zhuravlev, P.</li></a>
          </li>
          <li>
            <a href="https://www2.census.gov/programs-surveys/decennial/2020/program-management/data-product-planning/2010-demonstration-data-products/04-Demonstration_Data_Products_Suite/2023-04-03/Computing_Confidence_Intervals_Using_the_2010_Census_Redistricting_NMF.html" class="" target="_none">Computing Confidence Intervals Using the 2010 Census Production Settings Redistricting Data (P.L. 94-171) Demonstration Noisy Measurement File (2023-04-03)</a> by Cumings-Menon, R., Hawes, M., and Spence, M. (2023) &quot;(Jupyter notebook explaining how to calculate estimates and confidence intervals from the noisy measurement files)&quot;</li></a>
          </li>
          <li>
            <a href="https://github.com/uscensusbureau/DAS_2020_Redistricting_Production_Code" class="" target="_none">DAS 2020 Redistricting Production Code Release</a> by U.S. Census Bureau (Public GitHub repository for the 2020 Census DAS, vintaged as of the commit used to produce the official production run of the Redistricting product. The zCDP framework NMFs were generated in a for-internal-use-only pickled (https://docs.python.org/3/library/pickle.html; https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.SparkContext.pickleFile.html) form as a byproduct of the use of this code. A stand-alone script was developed and used to convert these internal-use NMFs into the Parquet format used in this product (that script is not yet publicly available.)</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.48550/arXiv.2203.16654" class="" target="_none">Geographic Spines in the 2020 Census Disclosure Avoidance System</a> by Abowd, J., Ashmead, R., Cumings-Menon, R., Garfinkel, S., Heineck, M., Heiss, C., Johns, R., Kifer, D., Leclerc, P., Machanavajjhala, A., Moran, B., Sexton, W., Spence, M., Zhuravlev, P.</li></a>
          </li>
          <li>
            <a href="https://www2.census.gov/programs-surveys/decennial/2010/technical-documentation/complete-tech-docs/summary-file/sf1.pdf" class="" target="_none">2010 Census Summary File 1 Technical Documentation</a> by U.S. Census Bureau</li></a>
          </li>
        </ul>
        <p><a href="/census-2010-dhc-nmf/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="census-2020-dhc-nmf" class="dataset">
          <h3><a href="/census-2020-dhc-nmf/">2020 Census Demographic and Housing Characteristics (DHC) Noisy Measurement File</a></h3>
          <p><span class="label label-info tag link-tag">census</span><span class="label label-info tag link-tag">differential privacy</span><span class="label label-info tag link-tag">disclosure avoidance</span><span class="label label-info tag link-tag">ethnicity</span><span class="label label-info tag link-tag">group quarters</span><span class="label label-info tag link-tag">housing</span><span class="label label-info tag link-tag">housing units</span><span class="label label-info tag link-tag">noisy measurements</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">race</span><span class="label label-info tag link-tag">redistricting</span><span class="label label-info tag link-tag">voting age</span></p>
          <p>The 2020 Census Demographic and Housing Characteristics Noisy Measurement File is an intermediate output of the 2020 Census Disclosure Avoidance System (DAS) TopDown Algorithm (TDA) (as described in Abowd, J. et al [2022], and implemented in <a href="https://github.com/uscensusbureau/DAS_2020_DHC_Production_Code/blob/main/das_decennial/programs/engine/primitives.py">primitives.py</a>). The 2020 Census Demographic and Housing Characteristics Noisy Measurement File includes zero-Concentrated Differentially Private (zCDP) (Bun, M. and Steinke, T [2016]) noisy measurements, implemented via the discrete Gaussian mechanism (Cannone C., et al., [2023] ), which added positive or negative integer-valued noise to each of the resulting counts. These are estimated counts of individuals and housing units included in the 2020 Census Edited File (CEF), which includes confidential data collected in the 2020 Census of Population and Housing.
<br/>
<br/>
The noisy measurements included in this file were subsequently post-processed by the TopDown Algorithm (TDA) to produce the Census Demographic and Housing Characteristics Summary File. In addition to the noisy measurements, constraints based on invariant calculations --- counts computed without noise --- are also included (with the exception of the state-level total populations, which can be sourced separately from data.census.gov).
<br/>
<br/>
The Noisy Measurement File was produced using the official “production settings,” the final set of algorithmic parameters and privacy-loss budget allocations that were used to produce the 2020 Census Redistricting Data (P.L. 94-171) Summary File and the 2020 Census Demographic and Housing Characteristics File.
<br/>
<br/>
The noisy measurements are p...</p>
          <p><a href="/census-2020-dhc-nmf/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1162/99608f92.529e3cb9" class="" target="_none">The 2020 Census Disclosure Avoidance System Topdown Algorithm</a> by Abowd, J., Ashmead, R., Cumings-Menon, R., Garfinkel, S., Heineck, M., Heiss, C., Johns, R., Kifer, D., Leclerc, P., Machanavajjhala, A., Moran, B., Sexton, W., Spence, M., Zhuravlev, P.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.48550/arXiv.2203.16654" class="" target="_none">Geographic Spines in the 2020 Census Disclosure Avoidance System</a> by Abowd, J., Ashmead, R., Cumings-Menon, R., Garfinkel, S., Heineck, M., Heiss, C., Johns, R., Kifer, D., Leclerc, P., Machanavajjhala, A., Moran, B., Sexton, W., Spence, M., Zhuravlev, P.</li></a>
          </li>
          <li>
            <a href="https://www.census.gov/programs-surveys/decennial-census/technical-documentation/complete-technical-documents.html#dhc-and-dp" class="" target="_none">2020 Census Demographic and Housing Characteristics File Technical Documentation</a> by U.S. Census Bureau. Note that the zCDP framework NMFs were generated in a for-internal-use-only pickled (https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.SparkContext.pickleFile.html) form as a byproduct of the use of this code. A stand-alone script was developed and used to convert these internal-use NMFs into the Parquet format used in this product (that script is not yet publicly available).</li></a>
          </li>
          <li>
            <a href="https://www2.census.gov/programs-surveys/decennial/2020/program-management/data-product-planning/2010-demonstration-data-products/04-Demonstration_Data_Products_Suite/2023-04-03/Computing_Confidence_Intervals_Using_the_2010_Census_Redistricting_NMF.html" class="" target="_none">Computing Confidence Intervals Using the 2010 Census Production Settings Redistricting Data (P.L. 94-171) Demonstration Noisy Measurement File (2023-04-03) (Jupyter notebook explaining how to calculate estimates and confidence intervals from the noisy measurement files)</a> by Cumings-Menon, R., Hawes, M., and Spence, M. (2023)</li></a>
          </li>
          <li>
            <a href="https://github.com/uscensusbureau/DAS_2020_DHC_Production_Code/tree/main" class="" target="_none">DAS 2020 DHC Production Code Release</a> by U.S. Census Bureau</li></a>
          </li>
        </ul>
        <p><a href="/census-2020-dhc-nmf/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="3kricegenome" class="dataset">
          <h3><a href="/3kricegenome/">3000 Rice Genomes Project</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The 3000 Rice Genome Project is an international effort to sequence the genomes of 3,024 rice varieties from 89 countries.</p>
          <p><a href="/3kricegenome/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41467-019-08451-3" class="" target="_none">Tracking the origin of two genetic components associated with transposable element bursts in domesticated rice</a> by Chen J et al (2019)</li></a>
          </li>
          <li>
            <a href="https://www.mdpi.com/1422-0067/23/3/1065" class="" target="_none">Identification and Allele Combination Analysis of Rice Grain Shape-Related Genes by Genome-Wide Association Study</a> by Meng B et al (2022)</li></a>
          </li>
          <li>
            <a href="https://github.com/InternationalRiceResearchInstitute/RiceGalaxy" class="" target="_none">RiceGalaxy</a> by <a href="https://www.irri.org/" target="_none">International Rice Research Institute</a></li></a>
          </li>
          <li>
            <a href="https://genome.cshlp.org/content/29/5/870.short" class="" target="_none">Structural variants in 3000 rice genomes</a> by Fuentes RR et al (2019)</li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/gigascience/article/8/5/giz028/5492234" class="" target="_none">Rice Galaxy: an open resource for plant science</a> by Juanillas V et al (2019)</li></a>
          </li>
        </ul>
        <p><a href="/3kricegenome/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="amazonia" class="dataset">
          <h3><a href="/amazonia/">Amazonia EO satellite on AWS</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Imagery acquired
by Amazonia-1 satellite.
The
image files are recorded and processed by Instituto Nacional de Pesquisas
Espaciais (INPE) and are converted to Cloud Optimized Geotiff
format in order to optimize its use for cloud based applications.
WFI Level 4 (Orthorectified) scenes are being
ingested daily starting from 08-29-2022, the complete
Level 4 archive will be ingested by the end of October 2022.</p>
          <p><a href="/amazonia/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/stactools-packages/amazonia-1" class="" target="_none">Amazonia 1 stactools package</a> by <a href="https://github.com/fredliporace" target="_none">Frederico Liporace</a></li></a>
          </li>
          <li>
            <a href="https://developmentseed.org/blog/2023-10-20-asdi" class="" target="_none">The Evolution of ASDI&#x27;s Data Infrastructure</a> by Sean Harkins</li></a>
          </li>
          <li>
            <a href="https://stac.amskepler.com/v100" class="" target="_none">STAC V1.0.0 endpoint</a> by <a href="https://github.com/fredliporace" target="_none">AMS Kepler</a></li></a>
          </li>
          <li>
            <a href="https://github.com/developmentseed/stactools-pipelines/pull/33" class="" target="_none">Amazonia 1 stactools-pipeline</a> by <a href="https://github.com/fredliporace" target="_none">Frederico Liporace</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/keeping-a-spatiotemporal-asset-catalog-stac-up-to-date-with-sns-sqs/" class="aws-link" target="_none">Keeping a SpatioTemporal Asset Catalog (STAC) Up To Date with SNS/SQS</a> by Frederico Liporace</li></a>
          </li>
        </ul>
        <p><a href="/amazonia/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="argo-gdac-marinedata" class="dataset">
          <h3><a href="/argo-gdac-marinedata/">Argo marine floats data and metadata from Global Data Assembly Centre (Argo GDAC)</a></h3>
          <p><span class="label label-info tag link-tag">chemical biology</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">datacenter</span><span class="label label-info tag link-tag">digital assets</span><span class="label label-info tag link-tag">geochemistry</span><span class="label label-info tag link-tag">geophysics</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">marine</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>Argo is an international program to observe the interior of the ocean with a fleet of profiling floats drifting in the deep ocean currents (<a href="https://argo.ucsd.edu">https://argo.ucsd.edu</a>). Argo GDAC is a dataset of 5 billion in situ ocean observations from 18.000 profiling floats (4.000 active) which started 20 years ago. Argo GDAC dataset is a collection of 18.000 NetCDF files.   It is a major asset for ocean and climate science, a contributor to IOCCP reports.</p>
          <p><a href="/argo-gdac-marinedata/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://fleetmonitoring.euro-argo.eu" class="" target="_none">Argo floats dashboard</a> by <a href="https://www.euro-argo.eu" target="_none">Euro-Argo ERIC</a></li></a>
          </li>
          <li>
            <a href="https://dataselection.euro-argo.eu" class="" target="_none">Argo data selection</a> by <a href="https://www.euro-argo.eu" target="_none">Euro-Argo ERIC</a></li></a>
          </li>
          <li>
            <a href="https://argo.ucsd.edu/outreach/publications/bibliography/" class="" target="_none">Argo data bibliography</a> by <a href="https://argo.ucsd.edu" target="_none">Argo program</a></li></a>
          </li>
          <li>
            <a href="https://argo.ucsd.edu/outreach/" class="" target="_none">Outreach activities</a> by <a href="https://argo.ucsd.edu" target="_none">Argo program</a></li></a>
          </li>
          <li>
            <a href="https://www.euro-argo.eu/Outreach/Outreach-material/Outreach-resources-about-Argo-floats" class="" target="_none">Outreach ressources for Argo floats</a> by <a href="https://www.euro-argo.eu" target="_none">Euro-Argo ERIC</a></li></a>
          </li>
        </ul>
        <p><a href="/argo-gdac-marinedata/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="asem-project" class="dataset">
          <h3><a href="/asem-project/">Automated Segmentation of Intracellular Substructures in Electron Microscopy (ASEM) on AWS</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">electron microscopy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span><span class="label label-info tag link-tag">segmentation</span></p>
          <p>The Automated Segmentation of intracellular substructures in Electron Microscopy (ASEM) project provides deep learning models trained to segment structures in 3D images of cells acquired by Focused Ion Beam Scanning Electron Microscopy (FIB-SEM). Each model is trained to detect a single type of structure (mitochondria, endoplasmic reticulum, golgi apparatus, nuclear pores, clathrin-coated pits) in cells prepared via chemically-fixation (CF) or high-pressure freezing and freeze substitution (HPFS). You can use our open source pipeline to load a model and predict a class of sub-cellular structur...</p>
          <p><a href="/asem-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://colab.research.google.com/drive/1FgRKN2mM6x18S02Qe4XMyxFeFubZxaMw" class="" target="_none">ASEM Colab Notebook (Interactive Demo)</a> by <a href="https://github.com/patrickstock" target="_none">Patrick Stock</a></li></a>
          </li>
          <li>
            <a href="http://asem-viewer-env.eba-rrnvmfwa.us-east-1.elasticbeanstalk.com/" class="" target="_none">TK Lab Data Explorer</a> by <a href="https://github.com/patrickstock" target="_none">Patrick Stock</a></li></a>
          </li>
          <li>
            <a href="https://github.com/kirchhausenlab/incasem/blob/main/README.md" class="" target="_none">How to use models</a> by <a href="https://github.com/kirchhausenlab/incasem/blob/main/README.md" target="_none">kirchhausenlab</a></li></a>
          </li>
          <li>
            <a href="https://open.quiltdata.com/b/asem-project" class="" target="_none">Data layout and how to view</a> by <a href="https://github.com/kirchhausenlab" target="_none">kirchhausenlab</a></li></a>
          </li>
          <li>
            <a href="https://rupress.org/jcb/article-pdf/222/2/e202208005/1444783/jcb_202208005.pdf" class="" target="_none">Deep neural network automated segmentation of cellular structures in volume electron microscopy</a> by Benjamin Gallusser, Giorgio Maltese, Giuseppe Di Caprio, Tegy John Vadakkan, Anwesha Sanyal, Elliott Somerville, Mihir Sahasrabudhe, Justin O’Connor, Martin Weigert, Tom Kirchhausen</li></a>
          </li>
        </ul>
        <p><a href="/asem-project/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="ncar-dart-cam6" class="dataset">
          <h3><a href="/ncar-dart-cam6/">CAM6 Data Assimilation Research Testbed (DART) Reanalysis: Cloud-Optimized Dataset</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">data assimilation</span><span class="label label-info tag link-tag">forecast</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>This is a cloud-hosted subset of the CAM6+DART (Community Atmosphere Model version 6 Data Assimilation Research Testbed) Reanalysis dataset. These data products are designed to facilitate a broad variety of research using the NCAR CESM 2.1 (National Center for Atmospheric Research&#39;s Community Earth System Model version 2.1), including model evaluation, ensemble hindcasting, data assimilation experiments, and sensitivity studies. They come from an 80 member ensemble reanalysis of the global troposphere and stratosphere using DART and CAM6. The data products represent states of the atmospher...</p>
          <p><a href="/ncar-dart-cam6/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41598-021-92927-0" class="" target="_none">A new CAM6 + DART reanalysis with surface forcing from CAM6 to other CESM models</a> by Raeder, K., Hoar, T.J., El Gharamti, M. et al (2021)</li></a>
          </li>
          <li>
            <a href="https://medium.com/pangeo/cesm-lens-on-aws-4e2a996397a1" class="" target="_none">Analyzing large climate model ensembles in the cloud</a> by Joe Hamman, NCAR</li></a>
          </li>
          <li>
            <a href="https://ncar-dart-cam6.s3-us-west-2.amazonaws.com/examples/plot-ensemble-values.html" class="" target="_none">Rendered (static) version of Jupyter Notebook</a> by Brian Bonnlander, NCAR</li></a>
          </li>
          <li>
            <a href="https://github.com/NCAR/ncar-dart-cam6" class="" target="_none">Jupyter Notebook and other documentation and tools for DART Reanalysis on AWS</a> by NCAR Science at Scale team</li></a>
          </li>
          <li>
            <a href="https://ncar-dart-cam6.s3-us-west-2.amazonaws.com/catalogs/aws-dart-cam6.json" class="" target="_none">Intake-ESM Catalog</a> by Brian Bonnlander, NCAR</li></a>
          </li>
        </ul>
        <p><a href="/ncar-dart-cam6/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="camelyon" class="dataset">
          <h3><a href="/camelyon/">CAncer MEtastases in LYmph nOdes challeNge (CAMELYON) Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">computational pathology</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">grand-challenge.org</span><span class="label label-info tag link-tag">histopathology</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>&quot;This dataset contains the all data for the <a href="https://camelyon17.grand-challenge.org">CAncer MEtastases in LYmph nOdes challeNge or CAMELYON</a>. CAMELYON was the first challenge using whole-slide images in computational pathology and aimed to help pathologists identify breast cancer metastases in sentinel lymph nodes. Lymph node metastases are extremely important to find, as they indicate that the cancer is no longer localized and systemic treatment might be warranted. Searching for these metastases in H&amp;E-stained tissue is difficult and time-consuming and AI algorithms can play a role in helping make this faster and more accura...</p>
          <p><a href="/camelyon/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ieeexplore.ieee.org/document/8447230/" class="" target="_none">From Detection of Individual Metastases to Classification of Lymph Node Status at the Patient Level: The CAMELYON17 Challenge</a> by <a href="https://www.diagnijmegen.nl/people/peter-bandi/" target="_none">Peter Bandi, et al.</a></li></a>
          </li>
          <li>
            <a href="https://camelyon17.grand-challenge.org/" class="" target="_none">CAMELYON Challenge</a> by <a href="https://www.diagnijmegen.nl/" target="_none">Diagnostic Image Analysis Group, Radboudumc, Nijmegen</a></li></a>
          </li>
          <li>
            <a href="https://jamanetwork.com/journals/jama/fullarticle/2665774" class="" target="_none">Diagnostic Assessment of Deep Learning Algorithms for Detection of Lymph Node Metastases in Women With Breast Cancer</a> by <a href="https://www.linkedin.com/in/babakint" target="_none">Babak Ehteshami Bejnordi, et al.</a></li></a>
          </li>
          <li>
            <a href="https://computationalpathologygroup.github.io/ASAP/" class="" target="_none">ASAP Viewer</a> by <a href="https://www.diagnijmegen.nl/people/geert-litjens/" target="_none">Geert Litjens</a></li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/gigascience/article/7/6/giy065/5026175" class="" target="_none">1399 H&amp;E-stained sentinel lymph node sections of breast cancer patients: the CAMELYON dataset</a> by <a href="https://www.diagnijmegen.nl/people/geert-litjens/" target="_none">Geert Litjens, et al.</a></li></a>
          </li>
        </ul>
        <p><a href="/camelyon/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="cesm-hr" class="dataset">
          <h3><a href="/cesm-hr/">CESM-HR</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">climate projections</span><span class="label label-info tag link-tag">CMIP6</span><span class="label label-info tag link-tag">ocean circulation</span><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean sea surface height</span><span class="label label-info tag link-tag">ocean simulation</span><span class="label label-info tag link-tag">ocean velocity</span></p>
          <p>This dataset provides several global fields describing the state of atmosphere, ocean, land and ice from a high-resolution (0.1o for the ocean/ice models 0.25o for the land/atmosphere models) numerical earth system model, the Community Earth System Model (CESM, <a href="https://www.cesm.ucar.edu/">https://www.cesm.ucar.edu/</a>). Texas A&amp;M University (TAMU) and National Center for Atmospheric Research together with international partners collaboratively carried out a large set of high-resolution climate simulations, including a 500-year long preindustrial control simulation (PI-CTRL) described here. The CESM uses dynamic equation...</p>
          <p><a href="/cesm-hr/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s43247-023-00681-0" class="" target="_none">Uncertain future of sustainable fisheries environment in eastern boundary upwelling zones under climate change</a> by Chang et al. (2023)</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1029/2020MS002298" class="" target="_none">An Unprecedented Set of High‐Resolution Earth System Simulations for Understanding Multiscale Interactions in Climate Variability and Change</a> by Chang et al. (2020)</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41558-024-02008-z" class="" target="_none">High-resolution modelling identifies the Bering Strait’s role in amplified Arctic warming</a> by Xu et al. (2024)</li></a>
          </li>
          <li>
            <a href="https://github.com/MESACLIP/cesm-hr-pi-ctrl-aws/tree/main/Tutorials" class="" target="_none">CESM-HR Tools and Applications</a> by <a href="https://github.com/jaisonkn" target="_none">Jaison Kurian</a></li></a>
          </li>
          <li>
            <a href="https://github.com/MESACLIP/cesm-hr-pi-ctrl-aws/tree/main/Tutorials" class="" target="_none">CESM-HR Tutorial</a> by <a href="https://github.com/jaisonkn" target="_none">Jaison Kurian</a></li></a>
          </li>
        </ul>
        <p><a href="/cesm-hr/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="cmas-data-warehouse" class="dataset">
          <h3><a href="/cmas-data-warehouse/">CMAS Data Warehouse</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span></p>
          <p>CMAS Data Warehouse on AWS collects and disseminates meteorology, emissions and air quality model input and output for Community Multiscale Air Quality (CMAQ) Model Applications. This dataset is available as part of the AWS Open Data Program, therefore egress fees are not charged to either the host or the person downloading the data.  This S3 bucket is maintained as a public service by the University of North Carolina&#39;s CMAS Center, the US EPA’s Office of Research and Development, and the US EPA’s Office of Air and Radiation.  Metadata and DOIs for datasets included in the CMAS Data Wareho...</p>
          <p><a href="/cmas-data-warehouse/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pcluster-cmaq.readthedocs.io/en/latest/index.html#" class="" target="_none">Documentation for CMAQ on AWS ParallelCluster</a> by <a href="https://github.com/CMASCenter/pcluster-cmaq/" target="_none">CMAS Center</a></li></a>
          </li>
          <li>
            <a href="https://catalog.workshops.aws/cmaq-tutorial/en-US/" class="" target="_none">CMAQ on AWS HPC Workshop</a> by <a href="https://catalog.workshops.aws/" target="_none">CMAS Center, U.S. EPA Office of Research and Development and AWS Workshop Studio</a></li></a>
          </li>
          <li>
            <a href="https://github.com/lizadams/EQUATES_BENCHMARK" class="" target="_none">EQUATES Benchmark</a> by <a href="https://github.com/lizadams/" target="_none">Liz Adams</a></li></a>
          </li>
          <li>
            <a href="https://github.com/USEPA/CMAQ/blob/main/DOCS/Users_Guide/Tutorials/CMAQ_UG_tutorial_benchmark.md" class="" target="_none">CMAQ Installation and Benchmarking Tutorial</a> by <a href="https://github.com/USEPA/CMAQ" target="_none">USEPA</a></li></a>
          </li>
          <li>
            <a href="https://github.com/USEPA/AMET/blob/master/docs/README.md" class="" target="_none">AMET Documentation</a> by <a href="https://www.epa.gov/cmaq/forms/contact-us-about-community-multiscale-air-quality-modeling-system" target="_none">USEPA</a></li></a>
          </li>
        </ul>
        <p><a href="/cmas-data-warehouse/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="mmrf-commpass" class="dataset">
          <h3><a href="/mmrf-commpass/">CoMMpass from the Multiple Myeloma Research Foundation</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Relating Clinical Outcomes in Multiple Myeloma to Personal Assessment of Genetic Profile study is the Multiple Myeloma Research Foundation (MMRF)’s landmark personalized medicine initiative.  CoMMpass is a
longitudinal observation study of around 1000 newly diagnosed myeloma patients receiving various
standard approved treatments. The MMRF’s vision is to track the treatment and results for each
CoMMpass patient so that someday the information can be used to guide decisions for newly
diagnosed patients. CoMMpass checked on patients every 6 months for 8 years, collecting tissue
samples, gene...</p>
          <p><a href="/mmrf-commpass/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood.V122.21.532.532" class="" target="_none">&quot;Interim Analysis Of The MMRF CoMMpass Trial: a Longitudinal Study In Multiple Myeloma
Relating Clinical Outcomes To Genomic and Immunophenotypic Profiles&quot;
</a> by Keats JJ, Craig DW, Liang W, Venkata Y, Kurdoglu A, Aldrich J, Auclair D, Allen K, Harrison
B, Jewell S, Kidd PG, Correll M, Jagannath S, Siegel DS, Vij R, Orloff G, Zimmerman TM, MMRF
CoMMpass Network, Capone W, Carpten J, Lonial S.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/MMRF-COMMPASS" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V124.21.722.722" class="" target="_none">&quot;Interim Analysis of the Mmrf Commpass Trial: Identification of Novel Rearrangements
Potentially Associated with Disease Initiation and Progression&quot;
</a> by Sagar Lonial, MD, Venkata D Yellapantula, Winnie Liang, PhD, Ahmet Kurdoglu, BS, Jessica
Aldrich, MSc, Christophe M. Legendre, MD, Kristi Stephenson, Jonathan Adkins, Jackie
McDonald, Adrienne Helland, Megan Russell, Austin Christofferson, Lori Cuyugan, Dan Rohrer,
Alex Blanski, Meghan Hodges, Mmrf CoMMpass Network, Mary Derome, Daniel Auclair, PhD, Pamela
G. Kidd, MD, Scott Jewell, PhD, David Craig, PhD, John Carpten, PhD, Jonathan J. Keats, PhD
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V128.22.194.194" class="" target="_none">&quot;Molecular Predictors of Outcome and Drug Response in Multiple Myeloma: An Interim Analysis
of the Mmrf CoMMpass Study&quot;
</a> by Jonathan J Keats, PhD, Gil Speyer, Austin Christofferson, Christophe Legendre, PhD, Jessica
Aldrich, Megan Russell, Lori Cuyugan, Jonathan Adkins, Alex Blanski, Meghan Hodges, Dan
Rohrer, Sundar Jagannath, MD, Ravi Vij, MD, Gregory Orloff, MD, Todd Zimmerman, MD, Ruben
Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD, Robert M Rifkin, Norma C
Gutierrez, MD PhD, Mmrf CoMMpass Network, Jennifer Yesil, MS, Mary Derome, MS, Seungchan
Kim, PhD, Winnie Liang, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John David Carpten, PhD,
Daniel Auclair, PhD, Sagar Lonial, MD FACP
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1182/blood.V126.23.722.722" class="" target="_none">&quot;Identification of Initiating Trunk Mutations and Distinct Molecular Subtypes: An Interim
Analysis of the Mmrf Commpass Study&quot;
</a> by Jonathan J Keats, PhD, Gil Speyer, Legendre Christophe, Christofferson Austin, Kristi
Stephenson, BS, Ahmet Kurdoglu, Megan Russell, Aldrich Jessica, Cuyugan Lori, Jonathan
Adkins, Jackie McDonald, Adrienne Helland, Alex Blanski, Meghan Hodges, Dan Rohrer, Sundar
Jagannath, MD, David Siegel, MD PhD, Ravi Vij, MD MBA, Gregory Orloff, MD, Todd Zimmerman,
MD, Ruben Niesvizky, MD, Darla Liles, MD, Joseph W. Fay, Jeffrey L. Wolf, MD PhD, Robert M.
Rifkin, Norma C Gutierrez, The MMRF CoMMpass Network, Jen Toups, Mary Derome, MS, Winnie
Liang, PhD, Seunchan Kim, Daniel Auclair, PhD, Pamela G. Kidd, MD, Scott Jewell, PhD, John
David Carpten, PhD, Sagar Lonial, MD
</li></a>
          </li>
        </ul>
        <p><a href="/mmrf-commpass/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="ncar-cesm-lens" class="dataset">
          <h3><a href="/ncar-cesm-lens/">Community Earth System Model Large Ensemble (CESM LENS)</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">ice</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>The Community Earth System Model (CESM) Large Ensemble Numerical Simulation (LENS) dataset includes a 40-member ensemble of climate simulations for the period 1920-2100 using historical data (1920-2005) or assuming the RCP8.5 greenhouse gas concentration scenario (2006-2100), as well as longer control runs based on pre-industrial conditions. The data comprise both surface (2D) and volumetric (3D) variables in the atmosphere, ocean, land, and ice domains. The total data volume of the original dataset is ~500TB, which has traditionally been stored as ~150,000 individual CF/NetCDF files on disk o...</p>
          <p><a href="/ncar-cesm-lens/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NCAR/cesm-lens-aws" class="" target="_none">Jupyter Notebook and other documentation and tools for CESM LENS on AWS</a> by NCAR Science at Scale team</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5065/d6j101d1" class="" target="_none">The Community Earth System Model (CESM) Large Ensemble Project: A Community Resource for Studying Climate Change in the Presence of Internal Climate Variability</a> by Kay et al. (2015), Bull. AMS, 96, 1333-1349</li></a>
          </li>
          <li>
            <a href="http://ncar-aws-www.s3-website-us-west-2.amazonaws.com/kay-et-al-2015.v2.html" class="" target="_none">Rendered (static) version of Jupyter Notebook</a> by Anderson Banihirwe, NCAR</li></a>
          </li>
          <li>
            <a href="https://medium.com/pangeo/cesm-lens-on-aws-4e2a996397a1" class="" target="_none">Analyzing large climate model ensembles in the cloud</a> by Joe Hamman, NCAR</li></a>
          </li>
          <li>
            <a href="https://zhonghuazheng.com/UrbanClimateExplorer/pages/about.html" class="" target="_none">Urban Climate Explorer</a> by Zhonghua Zheng</li></a>
          </li>
        </ul>
        <p><a href="/ncar-cesm-lens/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="daylight-osm" class="dataset">
          <h3><a href="/daylight-osm/">Daylight Map Distribution of OpenStreetMap</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">osm</span></p>
          <p>Daylight is a complete distribution of global, open map data that’s freely available with support from community and professional mapmakers. Meta combines the work of global contributors to projects like OpenStreetMap with quality and consistency checks from Daylight mapping partners to create a free, stable, and easy-to-use street-scale global map. <br/><br/>
The Daylight Map Distribution contains a validated subset of the OpenStreetMap database. In addition to the standard OpenStreetMap PBF format, Daylight is available in two parquet formats that are optimized for AWS Athena including geometries (Points, LineStrings, Polygons, or MultiPolygons). First, <strong>Daylight OSM Features</strong> contains the nearly 1B renderable OSM features. Second, <strong>Daylight OSM Elements</strong> contains all of OSM, including all 7B nodes without attributes, and relations that do not contain geometries, such as turn restrictions. <br/><br/> 
Daylight ...</p>
          <p><a href="/daylight-osm/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://daylightmap.org/earth/" class="" target="_none">Daylight Earth Tables</a> by Daylight Contributors</li></a>
          </li>
          <li>
            <a href="https://daylightmap.org/assets/nacis_2022_daylight_earth_table.pdf" class="" target="_none">Daylight Earth Tables (NACIS 2022)</a> by Jennings Anderson, Jonah Adkins, Jacob Wasserman</li></a>
          </li>
          <li>
            <a href="https://gist.github.com/jenningsanderson/3e42a99dcb8f760038ad8aa47ea38ce8" class="" target="_none">Loading the Daylight Map Distribution OpenStreetMap Features into AWS Athena</a> by Jennings Anderson</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/querying-daylight-openstreetmap-distribution-amazon-athena/" class="aws-link" target="_none">Querying the Daylight OpenStreetMap Distribution with Amazon Athena</a> by <a href="http://jenningsanderson.com" target="_none">Jennings Anderson and Mike Jeffe</a></li></a>
          </li>
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;rVBGLCFKAtw&amp;t&#x3D;161s" class="" target="_none">Increasing OpenStreetMap Data Accessibility with the Analysis-Ready Daylight Distribution of OpenStreetMap: A Demonstration of Cloud-Based Assessments of Global Building Completeness</a> by Jennings Anderson and Timmera Whaley Omidire</li></a>
          </li>
        </ul>
        <p><a href="/daylight-osm/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="esa-worldcover-vito-composites" class="dataset">
          <h3><a href="/esa-worldcover-vito-composites/">ESA WorldCover Sentinel-1 and Sentinel-2 10m Annual Composites</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">land use</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>The WorldCover 10m Annual Composites were produced, as part of the European Space Agency (ESA) WorldCover project, from the yearly Copernicus Sentinel-1 and Sentinel-2 archives for both years 2020 and 2021. These global mosaics consists of four products composites. A Sentinel-2 RGBNIR yearly median composite for bands B02, B03, B04, B08. A Sentinel-2 SWIR yearly median composite for bands B11 and B12. A Sentinel-2 NDVI yearly percentiles composite (NDVI 90th, NDVI 50th NDVI 10th percentiles). A Sentinel-1 GAMMA0 yearly median composite for bands VV, VH and VH/VV (power scaled). Each product is...</p>
          <p><a href="/esa-worldcover-vito-composites/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://blog.vito.be/remotesensing/release-of-the-10-m-worldcover-map" class="" target="_none">Release of the 10 m WorldCover map</a> by Ruben Van De Kerchove</li></a>
          </li>
          <li>
            <a href="https://esa-worldcover.s3.eu-central-1.amazonaws.com/v200/2021/docs/WorldCover_PUM_V2.0.pdf" class="" target="_none">ESA WorldCover 10 m 2021 v200 - Product User Manual</a> by VITO</li></a>
          </li>
          <li>
            <a href="https://viewer.esa-worldcover.org/worldcover/" class="" target="_none">WorldCover Viewer</a> by <a href="https://esa-worldcover.org/en" target="_none">VITO</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ESA-WorldCover/esa-worldcover-datasets" class="" target="_none">Exploring the datasets</a> by <a href="https://esa-worldcover.org" target="_none">VITO</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5281/zenodo.7254221" class="" target="_none">ESA WorldCover 10 m 2021 v200</a> by Zanaga, D., Van De Kerchove, R.,Daems, D.,De Keersmaecker, W., Brockmann, C., Kirches, G., Wevers, J., Cartus, O., Santoro, M., Fritz, S., Lesiv, M., Herold, M., Tsendbazar, N.E., Xu, P., Ramoino, F., Arino, O.</li></a>
          </li>
        </ul>
        <p><a href="/esa-worldcover-vito-composites/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="pgc-earthdem" class="dataset">
          <h3><a href="/pgc-earthdem/">EarthDEM</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>EarthDEM - 2m GSD Digital Elevation Models (DEMs) and mosaics from 2002 to the present. The EarthDEM project seeks to fill the need for high-resolution time-series elevation data in non-polar regions. The time-dependent nature of the strip DEM files allows users to perform change detection analysis and to compare observations of topography data acquired in different seasons or years. The mosaic DEM tiles are assembled from multiple strip DEMs with the intention of providing a more consistent and comprehensive product over large areas. EarthDEM data is constructed from in-track and cross-track ...</p>
          <p><a href="/pgc-earthdem/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://csdap.earthdata.nasa.gov/" class="" target="_none">NASA CSDA SmallSat Data Explorer</a> by NASA Commercial Smallsat Acquisition Program</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.isprsjprs.2017.04.019" class="" target="_none">The surface extraction from TIN based search-space minimization (SETSM) algorithm</a> by Myoung-Jong Noh, Ian M. Howat</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.3390/rs13040599" class="" target="_none">Multi-Source EO for Dynamic Wetland Mapping and Monitoring in the Great Lakes Basin</a> by Michael J. Battaglia, Sarah Banks, Amir Behnamian, Laura Bourgeau-Chavez, Brian Brisco, Jennifer Corcoran, Zhaohua Chen, Brian Huberty, James Klassen, Joseph Knight, Paul Morin, Kevin Murnaghan, Keith Pelletier, Lori White</li></a>
          </li>
          <li>
            <a href="https://glars.org/data/view-and-download/" class="" target="_none">GLARS Data Viewer</a> by Great Lakes Alliance for Remote Sensing</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1080/15481603.2015.1008621" class="" target="_none">Automated stereo-photogrammetric DEM generation at high latitudes: Surface Extraction with TIN-based Search-space Minimization (SETSM) validation and demonstration over glaciated regions</a> by Myoung-Jong Noh, Ian M. Howat</li></a>
          </li>
        </ul>
        <p><a href="/pgc-earthdem/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-pds-building-stock" class="dataset">
          <h3><a href="/nrel-pds-building-stock/">End-Use Load Profiles for the U.S. Building Stock</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">energy modeling</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">metadata</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">utilities</span></p>
          <p>The U.S. Department of Energy (DOE) funded a three-year project, End-Use Load Profiles for the U.S. Building Stock, that culminated in this publicly
available dataset of calibrated and validated 15-minute resolution load profiles for all major residential and commercial building types and end uses,
across all climate regions in the United States. These EULPs were created by calibrating the ResStock and ComStock physics-based building stock models
using many different measured datasets, as described <a href="https://www.nrel.gov/docs/fy22osti/80889.pdf">here</a>.
This dataset includes load profiles for both the baseline building stock and the building ...</p>
          <p><a href="/nrel-pds-building-stock/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://comstock.nrel.gov/page/datasets" class="" target="_none">Web-Based Data Viewer for Commercial Load Profiles</a> by National Renewable Energy Laboratory (NREL)</li></a>
          </li>
          <li>
            <a href="https://resstock.nrel.gov/datasets" class="" target="_none">Web-Based Data Viewer for Residential Load Profiles</a> by National Renewable Energy Laboratory (NREL)</li></a>
          </li>
          <li>
            <a href="https://nrel.github.io/ComStock.github.io/docs/resources/resources.html" class="" target="_none">Tutorials, how-to guides, explanations, and reference material for users of the ComStock dataset.</a> by National Renewable Energy Laboratory (NREL)</li></a>
          </li>
          <li>
            <a href="https://github.com/NREL/buildstock-query" class="" target="_none">BuildStockQuery: a python library designed to simplify and streamline the process of querying datasets generated by ResStock and ComStock</a> by National Renewable Energy Laboratory (NREL)</li></a>
          </li>
          <li>
            <a href="https://www.nrel.gov/docs/fy22osti/80889.pdf" class="" target="_none">End-Use Load Profiles for the U.S. Building Stock</a> by E. Wilson, A. Parker, A. Fontanini, et al.</li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-building-stock/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="glo-30-hand" class="dataset">
          <h3><a href="/glo-30-hand/">Global 30m Height Above Nearest Drainage (HAND)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">hydrology</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Height Above Nearest Drainage (HAND) is a terrain model that normalizes topography to the relative heights along the drainage network and is used to describe the relative soil gravitational potentials or the local drainage potentials. Each pixel value represents the vertical distance to the nearest drainage. The HAND data provides near-worldwide land coverage at 30 meters and was produced from the 2021 release of the Copernicus GLO-30 Public DEM as distributed in the <a href="https://registry.opendata.aws/copernicus-dem/">Registry of Open Data on AWS</a>.</p>
          <p><a href="/glo-30-hand/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ASFHyP3/OpenData/blob/main/glo-30-hand/generate-hand-notebook.ipynb" class="" target="_none">Generate your own HAND data in Python</a> by <a href="https://asf.alaska.edu/" target="_none">Alaska Satellite Facility</a></li></a>
          </li>
          <li>
            <a href="https://docs.google.com/presentation/d/1GzHn4bQrqqKj6DFFXwQ9zIvN7-etSjyIpHz3fQmxO0s/edit#slide&#x3D;id.p" class="" target="_none">AGU 2022: A New Global 30m HAND Dataset to Support Hydrological Services and Applications</a> by <a href="https://jhkennedy.org/" target="_none">Joseph H. Kennedy, et al.</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ASFHyP3/OpenData/blob/main/glo-30-hand/search-hand-notebook.ipynb" class="" target="_none">Search the GLO-30 HAND Catalog in Python</a> by <a href="https://asf.alaska.edu/" target="_none">Alaska Satellite Facility</a></li></a>
          </li>
          <li>
            <a href="https://asf-daac.maps.arcgis.com/apps/mapviewer/index.html?webmap&#x3D;6b82a2e4ccd343d5ba73dc04d386e4ee" class="" target="_none">GLO-30 HAND Webmap</a> by <a href="https://asf.alaska.edu/" target="_none">Alaska Satellite Facility</a></li></a>
          </li>
          <li>
            <a href="https://gis.asf.alaska.edu/arcgis/rest/services/GlobalHAND/GLO30_HAND/ImageServer" class="" target="_none">GLO-30 HAND ImageServer</a> by <a href="https://asf.alaska.edu/" target="_none">Alaska Satellite Facility</a></li></a>
          </li>
        </ul>
        <p><a href="/glo-30-hand/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="ebd-sentinel-1-global-coherence-backscatter" class="dataset">
          <h3><a href="/ebd-sentinel-1-global-coherence-backscatter/">Global Seasonal Sentinel-1 Interferometric Coherence and Backscatter Data Set</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geology</span><span class="label label-info tag link-tag">geophysics</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">synthetic aperture radar</span><span class="label label-info tag link-tag">urban</span></p>
          <p>This data set is the first-of-its-kind spatial representation of multi-seasonal, global SAR repeat-pass interferometric coherence and backscatter signatures. Global coverage comprises all land masses and ice sheets from 82 degrees northern to 79 degrees southern latitude. The data set is derived from high-resolution multi-temporal repeat-pass interferometric processing of about 205,000 Sentinel-1 Single-Look-Complex data acquired in Interferometric Wide-Swath mode (Sentinel-1 IW mode) from 1-Dec-2019 to 30-Nov-2020. The data set was developed by <a href="https://earthbigdata.com">Earth Big Data LLC</a> and <a href="https://www.gamma-rs.ch">Gamma Remote Sensing AG</a>, under contract for <a href="https://jpl.nasa.gov">NASA&#39;s Jet Propulsion Laboratory</a>. ...</p>
          <p><a href="/ebd-sentinel-1-global-coherence-backscatter/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;ZR3L9l7WcHU" class="" target="_none">Webinar: The new era of SAR Time Series Analysis and Visualization: Cloud meets Big SAR Data. IEEE GRSS Bay Area Chapter (Dec. 3rd 2021)</a> by Josef Kellndorfer</li></a>
          </li>
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;U3wbC6UWqx4" class="" target="_none">Generating Global Temporal Coherence Maps from one year of Sentinel-1 C-band data, ESA Fringe 2021 Poster (Youtube)</a> by Oliver Cartus, Josef Kellndorfer, Shadi Oveisgharan, Batu Osmanoglu, Paul Rosen, Urs Wegmüller</li></a>
          </li>
          <li>
            <a href="https://github.com/EarthBigData/openSAR" class="" target="_none">Jupyter Notebook to access and visualize sub regions of the global data set</a> by <a href="https://earthbigdata.com" target="_none">Josef Kellndorfer</a></li></a>
          </li>
          <li>
            <a href="https://github.com/EarthBigData/openSAR" class="" target="_none">Jupyter Notebook to access and visualize global mosaics of the global data set</a> by <a href="https://earthbigdata.com" target="_none">Josef Kellndorfer</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41597-022-01189-6" class="" target="_none">Global seasonal Sentinel-1 interferometric coherence and backscatter data set</a> by Josef Kellndorfer, Oliver Cartus, Marco Lavalle, Christophe Magnard, Pietro Milillo, Shadi Oveisgharan, Batu Osmanoglu, Paul A. Rosen, Urs Wegmüller</li></a>
          </li>
        </ul>
        <p><a href="/ebd-sentinel-1-global-coherence-backscatter/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="mapping-africa" class="dataset">
          <h3><a href="/mapping-africa/">High resolution, annual cropland and landcover maps for selected African countries</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>High resolution, annual cropland and landcover maps for selected African countries developed by <a href="https://clarku.edu">Clark University</a>&#39;s <a href="https://agroimpacts.info/">Agricultural Impacts Research Group</a> using various machine learning approaches applied to Planet imagery, including field boundary and cultivated frequency maps, as well as multi-class land cover.</p>
          <p><a href="/mapping-africa/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/agroimpacts/mapping-africa" class="" target="_none">Accessing and downloading data</a> by <a href="https://github.com/Rahebe22" target="_none">Rahebe Abedi</a></li></a>
          </li>
          <li>
            <a href="https://cropanalytics.net/wp-content/uploads/2022/04/FarmerlineClark-Report-Feb-2022-002.pdf" class="" target="_none">Final report-Phase 1: Creating open agricultural maps and ground truth data to better deliver farm extension services</a> by Estes et al (2022)</li></a>
          </li>
          <li>
            <a href="https://cropanalytics.net/wp-content/uploads/2022/11/Farmerline-Clark-Round-2-Report-V2-Nov-8-2022.pdf" class="" target="_none">Final report-Phase 2: Creating next generation field boundary and crop type maps: Rigorous multi-scale groundtruth provides sustainable extension services for smallholders</a> by Wussah et al (2022)</li></a>
          </li>
          <li>
            <a href="https://www.frontiersin.org/article/10.3389/frai.2021.744863" class="" target="_none">High resolution, annual maps of field boundaries for smallholder-dominated croplands at national scales</a> by Estes et al. (2022)</li></a>
          </li>
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S1569843222003405" class="" target="_none">A super-ensemble approach to map land cover types with high resolution over data-sparse African savanna landscapes</a> by Song et al. (2023)</li></a>
          </li>
        </ul>
        <p><a href="/mapping-africa/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="ibl-behaviour" class="dataset">
          <h3><a href="/ibl-behaviour/">IBL Behavioral Data on AWS</a></h3>
          <p><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span><span class="label label-info tag link-tag">open source software</span></p>
          <p>Behavioral data of mice performing a decision-making task, associated with 2020 publication of the IBL.</p>
          <p><a href="/ibl-behaviour/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://int-brain-lab.github.io/iblenv/loading_examples.html#" class="" target="_none">Loading Data</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.7554/eLife.63711" class="" target="_none">Standardized and reproducible measurement of decision-making in mice</a> by <a href="www.internationalbrainlab.com" target="_none">International Brain Laboratory et al.</a></li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/ONE-api/" class="" target="_none">ONE-api on pypi.python.org - python API to query and download neurophysiology data</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
          <li>
            <a href="https://int-brain-lab.github.io/iblenv/notebooks_external/data_download.html" class="" target="_none">Download the public data via ONE</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
          <li>
            <a href="https://int-brain-lab.github.io/iblenv/notebooks_external/data_download.html#Find-data-associated-with-a-release-or-publication" class="" target="_none">Find data associated with a release or publication</a> by <a href="https://github.com/orgs/int-brain-lab/teams/data-architecture-wg/members" target="_none">IBL Data Architecture Working Group</a></li></a>
          </li>
        </ul>
        <p><a href="/ibl-behaviour/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-himawari" class="dataset">
          <h3><a href="/noaa-himawari/">JMA Himawari-8/9</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Himawari-9, stationed at 140.7E, owned and operated by the Japan Meteorological Agency (JMA), is a geostationary meteorological satellite, with Himawari-8 as on-orbit back-up, that provides constant and uniform coverage of east Asia, and the west and central Pacific regions from around 35,800 km above the equator with an orbit corresponding to the period of the earth’s rotation. This allows JMA weather offices to perform uninterrupted observation of environmental phenomena such as typhoons, volcanoes, and general weather systems. Archive data back to July 2015 is available for Full Disk (AHI-L...</p>
          <p><a href="/noaa-himawari/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/awslabs/open-data-docs/blob/main/docs/noaa/noaa-himawari/2020July08_Amazon_Himawari.pdf" class="aws-link" target="_none">Himawari-8 on AWS (pdf file)</a> by ASDI</li></a>
          </li>
          <li>
            <a href="https://github.com/awslabs/open-data-docs/blob/main/docs/noaa/noaa-himawari/2020July08_NOAA_Himawari.pdf" class="aws-link" target="_none">Himawari-8 Advanced Himawari Imager Data on AWS (pdf file)</a> by NOAA NESDIS</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/himawari-8-enabling-access-key-weather-data/" class="aws-link" target="_none">Himawari-8: Enabling access to key weather data</a> by Manan Dalal, Jena Kent</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/pdf/2211.08883.pdf" class="" target="_none">Identifying the Causes of Pyrocumulonimbus (PyroCb)</a> by Emiliano Díaz Salas-Porras, Kenza Tazi, Ashwin Braude, Daniel Okoh, Kara D. Lamb, Duncan Watson-Parris, Paula Harder, and Nis Meinert</li></a>
          </li>
          <li>
            <a href="https://github.com/awslabs/open-data-docs/blob/main/docs/noaa/noaa-himawari/2020July07_JMA_Himawari.pdf" class="aws-link" target="_none">Introduction of Himawari-8/9 (pdf file)</a> by JMA</li></a>
          </li>
        </ul>
        <p><a href="/noaa-himawari/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="kyfromabove" class="dataset">
          <h3><a href="/kyfromabove/">KyFromAbove on AWS</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span></p>
          <p>The KyFromAbove initiative is focused on building and maintaining a current basemap for Kentucky that can meet the needs of its users at the state, federal, local, and regional level. A common basemap, including current color leaf-off aerial photography and elevation data (LiDAR), reduces the cost of developing GIS applications, promotes data sharing, and add efficiencies to many business processes. All basemap data acquired through this effort is being made available in the public domain. KyFromAbove acquires aerial imagery and LiDAR during leaf-off conditions in the Commonwealth. The imagery...</p>
          <p><a href="/kyfromabove/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.geoweeknews.com/news/kyfromabove-aerial-imagery-lidar-data-topography-elevation" class="" target="_none">KyFromAbove is democratizing key data in Kentucky</a> by Matt Collins</li></a>
          </li>
          <li>
            <a href="https://www.mydigitalpublication.com/publication/?m&#x3D;16892&amp;i&#x3D;816848&amp;view&#x3D;articleBrowser&amp;article_id&#x3D;4737566&amp;ver&#x3D;html5" class="" target="_none">A New View of Kentucky&#x27;s Cities</a> by Kent Anness</li></a>
          </li>
          <li>
            <a href="https://github.com/ianhorn/kyfromabove-on-aws-examples/blob/main/examples/clip_tiles_to_boundary.ipynb" class="" target="_none">KyFromAbove Bulk Download Area of Interest from Tile Index</a> by Ian Horn</li></a>
          </li>
          <li>
            <a href="https://www.esri.com/about/newsroom/arcuser/kytopo-kentuckys-new-topographic-map-series/" class="" target="_none">KyTopo! Kentucky&#x27;s New Topographic Map Series</a> by Kent Anness</li></a>
          </li>
          <li>
            <a href="https://github.com/ianhorn/kyfromabove-on-aws-examples/blob/main/examples/bulk_download_county.ipynb" class="" target="_none">KyFromAbove County Bulk Download</a> by Ian Horn</li></a>
          </li>
        </ul>
        <p><a href="/kyfromabove/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="pasteur-logan" class="dataset">
          <h3><a href="/pasteur-logan/">Logan Unitigs and Contigs of the Sequence Read Archive (SRA) on AWS</a></h3>
          <p><span class="label label-info tag link-tag">fasta</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">metagenomics</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole exome sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This repository is a re-analysis of the NCBI Sequence Read Archive (SRA), December 2023 freeze, to make it more accessible. The SRA is an open access database of biological sequences, containing raw data from high-throughput DNA and RNA sequencing platforms. It is the largest database of public DNA sequences worldwide, containing a wealth of genomic diversity across all living organisms. This repository contains Logan, a set of compressed FASTA files for all individual SRA accessions, in the form of unitigs and contigs. Borrowing methods from the real of genome assembly, unitigs preserve nearl...</p>
          <p><a href="/pasteur-logan/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://openvirome.com/" class="" target="_none">Open Virome</a> by <a href="https://rrna.ca" target="_none">Artem Babaian</a></li></a>
          </li>
          <li>
            <a href="https://github.com/IndexThePlanet/Logan/blob/main/Chickens.md" class="" target="_none">Downloading, mapping many contigs to a gene of interest</a> by <a href="https://github.com/IndexThePlanet" target="_none">Rayan Chikhi</a></li></a>
          </li>
          <li>
            <a href="https://github.com/IndexThePlanet/Logan/blob/main/Accessions.md" class="" target="_none">Downloading unitigs of several accessions</a> by <a href="https://github.com/IndexThePlanet" target="_none">Rayan Chikhi</a></li></a>
          </li>
          <li>
            <a href="https://github.com/IndexThePlanet/Logan/blob/main/Kmer_search.md" class="" target="_none">Search for a k-mer of interest inside an unitigs accession</a> by <a href="https://github.com/IndexThePlanet" target="_none">Rayan Chikhi</a></li></a>
          </li>
          <li>
            <a href="https://github.com/GATB/minia" class="" target="_none">Minia 3</a> by <a href="https://github.com/GATB" target="_none">Rayan Chikhi</a></li></a>
          </li>
        </ul>
        <p><a href="/pasteur-logan/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="monkey" class="dataset">
          <h3><a href="/monkey/">MONKEY</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">classification</span><span class="label label-info tag link-tag">computational pathology</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">digital pathology</span><span class="label label-info tag link-tag">grand-challenge.org</span><span class="label label-info tag link-tag">histopathology</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medical image computing</span><span class="label label-info tag link-tag">medical imaging</span></p>
          <p>This dataset contains the training data for the <a href="https://monkey.grand-challenge.org/">Machine learning for Optimal detection of iNflammatory cells in the KidnEY or MONKEY</a> challenge. The MONKEY challenge focuses on the automated detection and classification of inflammatory cells, specifically monocytes and lymphocytes, in kidney transplant biopsies using Periodic acid-Schiff (PAS) stained whole-slide images (WSI). It contains 80 WSI, collected from 4 different pathology institutes, with annotated regions of interest. For each WSI up to 3 different PAS scans and one IHC slide scan are available. This dataset and challenge support th...</p>
          <p><a href="/monkey/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/computationalpathologygroup/monkey-challenge" class="" target="_none">MONKEY challenge tutorials</a> by <a href="https://www.computationalpathologygroup.eu/" target="_none">Computational Pathology Group Radboudumc, Nijmegen</a></li></a>
          </li>
          <li>
            <a href="https://github.com/DIAGNijmegen/pathology-whole-slide-data" class="" target="_none">Whole slide data - python package for working with wholeslide images and annotations files</a> by <a href="https://www.diagnijmegen.nl/" target="_none">Diagnostic Image Analysis Group, Radboudumc, Nijmegen</a></li></a>
          </li>
          <li>
            <a href="https://monkey.grand-challenge.org/" class="" target="_none">MONKEY Challenge</a> by <a href="https://www.computationalpathologygroup.eu/" target="_none">Computational Pathology Group Radboudumc, Nijmegen</a></li></a>
          </li>
          <li>
            <a href="https://computationalpathologygroup.github.io/ASAP/" class="" target="_none">ASAP Viewer</a> by <a href="https://www.diagnijmegen.nl/people/geert-litjens/" target="_none">Geert Litjens</a></li></a>
          </li>
          <li>
            <a href="https://zenodo.org/records/13794656" class="" target="_none">Structured description of the MONKEY Challenge</a> by <a href="https://www.computationalpathologygroup.eu/" target="_none">Linda Studer, Dominique van Midden, Fazael Ayatollahi, Jeroen van der Laak</a></li></a>
          </li>
        </ul>
        <p><a href="/monkey/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-usgs-lunar-orbiter-laser-altimeter" class="dataset">
          <h3><a href="/nasa-usgs-lunar-orbiter-laser-altimeter/">NASA / USGS Lunar Orbiter Laser Altimeter Cloud Optimized Point Cloud</a></h3>
          <p><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The lunar orbiter laser altimeter (LOLA) has collected and released almost 7 billion individual laser altimeter returns from the lunar surface. This dataset includes individual altimetry returns scraped from the Planetary Data System (PDS) LOLA Reduced Data Record (RDR) Query Tool, V2.0. Data are organized in 15˚ x 15˚ (longitude/latitude) sections, compressed and encoded into the Cloud Optimized Point Cloud (COPC) file format, and collected into a Spatio-Temporal Asset Catalog (STAC) collection for query and analysis. The data are in latitude, longitude, and radius (X, Y, Z) format with the p...</p>
          <p><a href="/nasa-usgs-lunar-orbiter-laser-altimeter/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/examples/to_qgis/" class="" target="_none">Querying for Data in an ROI and Loading it into QGIS</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/cli/" class="" target="_none">Discovering and Downloading Data via the Command Line</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://github.com/stac-utils/pystac-client" class="" target="_none">PySTAC Client</a> by <a href="https://github.com/stac-utils/pystac-client/graphs/contributors" target="_none">PySTAC-Client Contributors</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/basicpython/" class="" target="_none">Discovering and Downloading Data with Python</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://pdal.io/" class="" target="_none">Point Data Abstraction Library (PDAL)</a> by <a href="https://github.com/PDAL/PDAL" target="_none">PDAL Contributors</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-usgs-lunar-orbiter-laser-altimeter/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-sra" class="dataset">
          <h3><a href="/ncbi-sra/">NIH NCBI Sequence Read Archive (SRA) on AWS</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole exome sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Sequence Read Archive (SRA), produced by the <a href="https://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information (NCBI)</a> at the <a href="http://nlm.nih.gov/">National Library of Medicine (NLM)</a> at the <a href="http://www.nih.gov/">National Institutes of Health (NIH)</a>, stores raw DNA sequencing data and alignment information from high-throughput sequencing platforms. The SRA provides open access to these biological sequence data to support the research community&#39;s efforts to enhance reproducibility and make new discoveries by comparing data sets. Buckets in this registry contain public SRA data in the original (user submitted) format from select high value and newly-rel...</p>
          <p><a href="/ncbi-sra/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Access SRA data using Amazon Web Services (AWS)</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-cloud/" class="" target="_none">SRA in the Cloud</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-athena/" class="" target="_none">Get started with the SRA and Amazon Athena</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013647/" class="" target="_none">The Sequence Read Archive</a> by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term&#x3D;Leinonen%20R%5BAuthor%5D&amp;cauthor&#x3D;true&amp;cauthor_uid&#x3D;21062823" target="_none">Leinonen et al (2011)</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ncbi/sra-tools" class="" target="_none">SRA Toolkit</a> by <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_none">NCBI SRA</a></li></a>
          </li>
        </ul>
        <p><a href="/ncbi-sra/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-nws-naqfc-pds" class="dataset">
          <h3><a href="/noaa-nws-naqfc-pds/">NOAA National Air Quality Forecast Capability (NAQFC) Regional Model Guidance</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The National Air Quality Forecasting Capability (NAQFC) dataset contains model-generated Air-Quality (AQ) forecast guidance from three different prediction systems. The first system is a coupled weather and atmospheric chemistry numerical forecast model, known as the Air Quality Model (AQM). It is used to produce forecast guidance for ozone (O3) and particulate matter with diameter equal to or less than 2.5 micrometers (PM2.5) using meteorological forecasts based on NCEP’s operational weather forecast models such as North American Mesoscale Models (NAM) and Global Forecast System (GFS), and atmospheric chemistry based on the EPA’s Community Multiscale Air Quality (CMAQ) model. In addition, the modeling system incorporates information related to chemical emissions, including anthropogenic emissions provided by the EPA and fire emissions from NOAA/NESDIS. The NCEP NAQFC AQM output fields in this archive include 72-hr forecast products of model raw and bias-correction predictions, extending back to 1 January 2020. All of the output was generated by the contemporaneous operational AQM, beginning with AQMv5 in 2020, with upgrades to AQMv6 on 20 July 2021, and AQMv7 on 14 May 2024. The history of AQM upgrades is documented <a href="https://www.emc.ncep.noaa.gov/mmb/aq/AQChangelog.html">here</a>
<br/>
<br/>
The second prediction is known as the Hybrid Single-Particle Lagrangian Integrated Trajectory model (HYSPLIT). It is a widely used atmospheric transport and dispersion model containing an internal dust-generation module. It provides forecast guidance for atmospheric dust concentration and, prior to 28 June 2022, it also provided the NAQFC forecast guidance for smoke. Since that date, the third prediction system, a regional numerical weather prediction (NWP) model known as the Rapid Refresh (RAP) model, has subsumed HYSPLIT for operational smoke guidance, simulating the emission, transport, and deposition of smoke particles that originate from biomass burning (fires) and anthropogenic sources.
<br/>
<br/>
The output from each of these modeling systems is generated over three separate domains, one covering CONUS, one Alaska, and the other Hawaii. Currently, for this archive, the ozone, (PM2.5), and smoke output is available over all three domains, while dust products are available only over the CONUS domain. The predicted concentrations of all species in the lowest model layer (i.e., the layer in contact with the surface) are available, as are vertically integrated values of smoke and dust. The data is gridded horizontally within each domain, with a grid spacing of approximately 5 km over CONUS, 6 km over Alaska, and 2.5 km over Hawaii. Ozone concentrations are provided in parts per billion (PPB), while the concentrations of all other species are quantified in units of micrograms per cubic meter (ug/m3), except for the column-integrated smoke values which are expressed in units of mg/m2.
<br/>
<br/>
Temporally, O3 and PM2.5 are available as maximum and/or averaged values over various time periods. Specifically, O3 is available in both 1-hour and 8-hour (backward calculated) averages, as well as preceding 1-hour and 8-hour maximum values. Similarly, PM2.5 is available in 1-hour and 24-hour average values and 24-hour maximum values. In addition, all O3 and PM2.5 fields are available with bias-corrected magnitudes, based on derived model biases relative to observations.
<br/>
<br/>
The AQM produces hourly forecast guidance for O3 and PM2.5 out to 72 hours twice per day, starting at 0600 and 1...</p>
          <p><a href="/noaa-nws-naqfc-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1109/IGARSS.2017.8127581" class="" target="_none">Using VIIRS fire radiative power data to simulate biomass burning emissions, plume rise and smoke transport in a real-time air quality modeling system (Proc. 2017 IEEE Int. Geoscience and Remote Sensing Symp. (IGARSS0),Fort Worth, TX, IEEE, 2806–2808)</a> by Ahmadov, R., and Coauthors</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/gmd-15-3281-2022" class="" target="_none">Development and evaluation of an advanced National Air Quality Forecasting Capability using the NOAA Global Forecast System version 16 (2022, Geosci. Model Dev., 15, 3281–3313)</a> by Campbell, P.C., and Coauthors</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1175/WAF-D-16-0118.1" class="" target="_none">Improving NOAA NAQFC PM2.5 predictions with a bias correction approach (2017, Wea. and Forecasting, 32(2), 407–421)</a> by Huang, J., McQueen, J., Wilczak, J., Djalalova, I., Stajner, I., Shafran, P., Allured, D., Lee, P., Pan, L., Tong, D., Huang, H.-C., DiMego, G., Upadhayay, S., &amp; Delle Monache, L</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1029/2009JD013167" class="" target="_none">An empirically derived emission algorithm for wind-blown dust (J.Geophys. Res., 115, D16212)</a> by Draxler, R. R., P. Ginoux, and A. F. Stein</li></a>
          </li>
          <li>
            <a href="https://epic.noaa.gov/" class="" target="_none">Development of the next-generation air quality prediction system in the UFS framework: Enhancing predictability of wildfire air quality impacts (2024)(Bull. Amer. Meteor. Soc. In review)</a> by Huang, J.P., I. Stajner, R. Montuoro, F. Yang, K. Wang, H.-C. Huang, C.-H. Jeon, B. Curtis, J. McQueen, H. Liu, B. Baker, D. Tong , Y. Tang, P. Campbell, G. Grell, G. Frost, R. Schwantes, S. Wang, S. Kondragunta, F. Li, and Y. Jung</li></a>
          </li>
        </ul>
        <p><a href="/noaa-nws-naqfc-pds/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="ndui" class="dataset">
          <h3><a href="/ndui/">Normalized Difference Urban Index (NDUI)</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">urban</span></p>
          <p>NDUI is combined with cloud shadow-free Landsat Normalized Difference Vegetation Index (NDVI) composite and DMSP/OLS Night Time Light (NTL) to characterize global urban areas at a 30 m resolution,and it can greatly enhance urban areas, which can then be easily distinguished from bare lands including fallows and deserts. With the capability to delineate urban boundaries and, at the same time, to present sufficient spatial details within urban areas, the  NDUI has the potential for urbanization studies at regional and global scales.</p>
          <p><a href="/ndui/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/yifwahaha/aws-ndui" class="" target="_none">An example of using ndui data with AWS sagemaker tools</a> by <a href="https://github.com/yifwahaha" target="_none">Yifang Wang</a></li></a>
          </li>
          <li>
            <a href="https://code.earthengine.google.com/e0488afe77caa345c3217b0ff339ccca" class="" target="_none">Global DMSP images Correction</a> by <a href="https://github.com/yifwahaha" target="_none">Yifang Wang</a></li></a>
          </li>
          <li>
            <a href="https://github.com/yifwahaha/NDUI_work_flow" class="" target="_none">Automated extraction of urban built-up areas with NDUI using Python and Google Earth Engine</a> by <a href="https://github.com/yifwahaha" target="_none">Yifang Wang</a></li></a>
          </li>
          <li>
            <a href="https://www.mdpi.com/2072-4292/7/9/11887" class="" target="_none">Building a Better Urban Picture:Combining Day and Night Remote Sensing Imagery</a> by Qingling Zhang and Bin Li and David Thau &amp; Rebecca Moore</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1109/TGRS.2016.2572724" class="" target="_none">A Robust Method to Generate a Consistent Time Series From DMSP/OLS Nighttime Light Data</a> by Qingling Zhang and Bhartendu Pandey and Keren C.Seto</li></a>
          </li>
        </ul>
        <p><a href="/ndui/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="open-meteo" class="dataset">
          <h3><a href="/open-meteo/">Open-Meteo Weather API Database</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Open-Meteo integrates weather models from reputable national weather services, offering a swift and efficient weather API. Real-time weather forecasts are unified into a time-series database that provides historical and future weather data for any location worldwide.Through Open-Meteo on AWS Open Data, you can download the Open-Meteo weather database and analysis weather data locally. Docker images are provided to download data and to expose an HTTP API endpoint. Using Open-Meteo SDKs, you can seamlessly integrate weather data into your Python, Typescript, Swift, Kotlin, or Java applications.T...</p>
          <p><a href="/open-meteo/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/open-meteo/open-meteo" class="" target="_none">Open-Meteo API source-code</a> by <a href="https://www.open-meteo.com/" target="_none">Open-Meteo</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5281/ZENODO.7970649" class="" target="_none">Zippenfenig, P. (2023). Open-Meteo.com Weather API [Computer software]</a> by Patrick Zippenfenig</li></a>
          </li>
          <li>
            <a href="https://www.open-meteo.com/en/docs" class="" target="_none">Open-Meteo API documentation</a> by <a href="https://www.open-meteo.com/" target="_none">Open-Meteo</a></li></a>
          </li>
          <li>
            <a href="https://github.com/open-meteo/open-data/tree/main/tutorial_weather_api" class="" target="_none">Run Your Own Weather API</a> by <a href="https://www.open-meteo.com/" target="_none">Open-Meteo</a></li></a>
          </li>
          <li>
            <a href="https://github.com/open-meteo/open-data/tree/main/tutorial_download_era5" class="" target="_none">Accessing 80 Years Historical Weather Data from ERA5</a> by <a href="https://www.open-meteo.com/" target="_none">Open-Meteo</a></li></a>
          </li>
        </ul>
        <p><a href="/open-meteo/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="osm" class="dataset">
          <h3><a href="/osm/">OpenStreetMap on AWS</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">osm</span></p>
          <p>OSM is a free, editable map of the world, created and maintained by volunteers. Regular OSM data archives are made available in Amazon S3.</p>
          <p><a href="/osm/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/developmentseed/mrworldwide/" class="" target="_none">OSM+Athena (GitHub)</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/develop-and-extract-value-from-open-data/" class="aws-link" target="_none">Develop and Extract Value from Open Data</a> by Daniel Bernao</li></a>
          </li>
          <li>
            <a href="https://www.openstreetmap.org/user/Jennings%20Anderson/diary/394762" class="" target="_none">Querying OpenStreetMap Changesets with Amazon Athena</a> by <a href="https://www.openstreetmap.org/user/Jennings%20Anderson/diary" target="_none">Jennings Anderson</a></li></a>
          </li>
          <li>
            <a href="https://github.com/interline-io/planetutils" class="" target="_none">PlanetUtils (GitHub): Scripts and a Docker container to maintain your own OpenStreetMap planet</a> by <a href="https://www.interline.io/" target="_none">Interline Technologies</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/querying-openstreetmap-with-amazon-athena/" class="aws-link" target="_none">Querying OpenStreetMap with Amazon Athena</a> by <a href="https://github.com/mojodna" target="_none">Seth Fitzsimmons</a></li></a>
          </li>
        </ul>
        <p><a href="/osm/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="overture" class="dataset">
          <h3><a href="/overture/">Overture Maps Foundation Open Map Data</a></h3>
          <p><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">osm</span><span class="label label-info tag link-tag">parquet</span><span class="label label-info tag link-tag">transportation</span></p>
          <p>Overture is a collaboratively built, global, open map data project for developers who build map services or use geospatial data. Overture Open Map Data contains data that are standardized under the themes of Admins, Base, Buildings, Places, and Transportation. Overture also includes a Global Entity Reference System (GERS) which encodes map data to a shared universal reference. Beginning with the Overture 2023-11-14-alpha.0 release, the data is available as cloud-native GeoParquet files. </p>
          <p><a href="/overture/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.overturemaps.org/" class="" target="_none">Overture Data Schema</a> by Overture Maps Foundation</li></a>
          </li>
          <li>
            <a href="https://overturemaps.org/webinar-recap-working-with-overture-data-a-step-by-step-guide/" class="" target="_none">Working With Overture Data: A Step-by-Step Guide</a> by Jennings Anderson</li></a>
          </li>
          <li>
            <a href="https://docs.overturemaps.org/gers/" class="" target="_none">Global Entity Reference System</a> by Overture Maps Foundation</li></a>
          </li>
          <li>
            <a href="https://project.linuxfoundation.org/hubfs/Overture%20Maps/Building%20Heights%20Whitepaper_041423.pdf" class="" target="_none">Building Heights: From open USGS lidar to open Overture maps</a> by Overture Maps Foundation</li></a>
          </li>
          <li>
            <a href="https://github.com/OvertureMaps/data/blob/main/README.md" class="" target="_none">Accessing Overture Maps Data</a> by Overture Maps Foundation</li></a>
          </li>
        </ul>
        <p><a href="/overture/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="omi-no2-nasa" class="dataset">
          <h3><a href="/omi-no2-nasa/">Ozone Monitoring Instrument (OMI) / Aura NO2 Tropospheric Column Density</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>NO2 tropospheric column density, screened for CloudFraction &lt; 30% global daily
 composite at 0.25 degree resolution for the temporal range of 2004 to May
2020. Original archive data in HDF5 has been processed into a <a href="https://www.cogeo.org/">Cloud-Optimized
GeoTiff (COG)</a> format. Quality Assurance - This data
has been validated by the NASA Science Team at Goddard Space Flight Center.Cautionary Note: <a href="https://airquality.gsfc.nasa.gov/caution-interpretation">https://airquality.gsfc.nasa.gov/caution-interpretation</a>.</p>
          <p><a href="/omi-no2-nasa/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://rasterio.readthedocs.io/" class="" target="_none">Rasterio (Python Library) for access to geospatial raster data</a> by <a href="https://www.mapbox.com/" target="_none">Mapbox</a></li></a>
          </li>
          <li>
            <a href="https://cogeo.xyz/?url&#x3D;s3://omi-no2-nasa/OMI-Aura_L3-OMNO2d_2020m0101_v003-2020m0330t173100.tif" class="" target="_none">COG Viewer</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></li></a>
          </li>
          <li>
            <a href="https://pfg2k5mjei.execute-api.us-west-2.amazonaws.com/docs" class="" target="_none">COG Application Programming Interface (API)</a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/devseed/cog-talk-part-1-whats-new-941facbcd3d1" class="" target="_none">Medium Post: COG Talk — Part 1: What’s new?</a> by <a href="https://medium.com/@_VincentS_" target="_none">Vincent Sarago</a></li></a>
          </li>
          <li>
            <a href="https://pfg2k5mjei.execute-api.us-west-2.amazonaws.com/v1/metadata?url&#x3D;https%3A%2F%2Fomi-no2-nasa.s3-us-west-2.amazonaws.com%2FOMI-Aura_L3-OMNO2d_2018m0101_v003-2019m1123t055315.tif&amp;pmin&#x3D;2&amp;pmax&#x3D;98&amp;max_size&#x3D;1024&amp;histogram_bins&#x3D;20" class="" target="_none">COG metadata example </a> by <a href="https://developmentseed.org/" target="_none">Development Seed</a></li></a>
          </li>
        </ul>
        <p><a href="/omi-no2-nasa/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="pmsp-lidar" class="dataset">
          <h3><a href="/pmsp-lidar/">Prefeitura Municipal de São Paulo (PMSP) LiDAR Point Cloud</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">urban</span></p>
          <p>The objective of the Mapa 3D Digital da Cidade (M3DC) of the São Paulo City Hall is to publish LiDAR point cloud data. The initial data was acquired in 2017 by aerial surveying and future data will be added. This publicly accessible dataset is provided in the <a href="https://entwine.io/entwine-point-tile.html">Entwine Point Tiles</a> format as a lossless octree, full density, based on <a href="https://laszip.org">LASzip</a> (LAZ) encoding.</p>
          <p><a href="/pmsp-lidar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pdal.io/" class="" target="_none">PDAL - Point Data Abstraction Library</a> by <a href="https://doi.org/10.5281/zenodo.2556738" target="_none">PDAL Contributors</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/isprs-annals-IV-2-W5-437-2019" class="" target="_none">Describing the Vertical Structure of Informal Settlements on the Basis of LiDAR Data – A Case Study for Favelas (Slums) in Sao Paulo City</a> by S. C. L. Ribeiro, M. Jarzabek-Rychard, J. P. Cintra, H.-G. Maas</li></a>
          </li>
          <li>
            <a href="http://forsys.sefs.uw.edu/fusion/fusionlatest.html" class="" target="_none">Fusion</a> by <a href="https://www.usda.gov/" target="_none">US Department of Agriculture - Forest Service</a></li></a>
          </li>
          <li>
            <a href="https://rapidlasso.com/lastools/" class="" target="_none">LAStools</a> by <a href="https://rapidlasso.com" target="_none">rapidlasso GmbH, GERMANY</a></li></a>
          </li>
          <li>
            <a href="https://entwine.io/" class="" target="_none">Entwine</a> by <a href="https://hobu.co/" target="_none">Hobu, Inc.</a></li></a>
          </li>
        </ul>
        <p><a href="/pmsp-lidar/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="racecar-dataset" class="dataset">
          <h3><a href="/racecar-dataset/">RACECAR Dataset</a></h3>
          <p><span class="label label-info tag link-tag">autonomous racing</span><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">GNSS</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">localization</span><span class="label label-info tag link-tag">object detection</span><span class="label label-info tag link-tag">object tracking</span><span class="label label-info tag link-tag">perception</span><span class="label label-info tag link-tag">radar</span><span class="label label-info tag link-tag">robotics</span></p>
          <p>The RACECAR dataset is the first open dataset for full-scale and high-speed autonomous racing. Multi-modal sensor data has been collected from fully autonomous Indy race cars operating at speeds of up to 170 mph (273 kph). Six teams who raced in the Indy Autonomous Challenge during 2021-22 have contributed to this dataset. The dataset spans 11 interesting racing scenarios across two race tracks which include solo laps, multi-agent laps, overtaking situations, high-accelerations, banked tracks, obstacle avoidance, pit entry and exit at different speeds. The data is organized and released in bot...</p>
          <p><a href="/racecar-dataset/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/linklab-uva/RACECAR_DATA/tree/main#tutorial-2-ros2-localization" class="" target="_none">RACECAR Tutorials - ROS2 Localization</a> by Amar Kulkarni</li></a>
          </li>
          <li>
            <a href="https://github.com/linklab-uva/rosbag2nuscenes" class="" target="_none">rosbag2nuscenes conversion library</a> by <a href="https://github.com/linklab-uva/rosbag2nuscenes" target="_none">John Chrosniak, Emory Ducote, John Link, Madhur Behl</a></li></a>
          </li>
          <li>
            <a href="https://github.com/linklab-uva/RACECAR_DATA/tree/main#tutorial-1-ros2-visualization" class="" target="_none">RACECAR Tutorials - ROS2 Visualization</a> by Amar Kulkarni, Utkarsh Chirimar</li></a>
          </li>
          <li>
            <a href="https://github.com/linklab-uva/RACECAR_DATA/tree/main#tutorial-3-nuscenes-jupyter-notebook" class="" target="_none">RACECAR Tutorials - nuScenes</a> by John Chrosniak</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.48550/arXiv.2306.03252" class="" target="_none">RACECAR--The Dataset for High-Speed Autonomous Racing</a> by Amar Kulkarni, John Chrosniak, Emory Ducote, Florian Sauerbeck, Andrew Saba, Utkarsh Chirimar, John Link, Marcello Cellina, and Madhur Behl</li></a>
          </li>
        </ul>
        <p><a href="/racecar-dataset/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="spartan-cloud" class="dataset">
          <h3><a href="/spartan-cloud/">SPARTAN Data</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">environmental</span></p>
          <p>SPARTAN (Surface PARTiculate mAtter Network) measures and provides surface ambient particulate matter (PM2.5 and PM10) concentration and the chemical composition around the world, with the purpose of connecting ground-based PM2.5 and satellite remote sensing.</p>
          <p><a href="/spartan-cloud/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1029/2022JD036937" class="" target="_none">A Global-Scale Mineral Dust Equation</a> by Xuan Liu, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/amt-8-505-2015" class="" target="_none">SPARTAN: A global network to evaluate and enhance satellite-based estimates of ground-level particulate matter for global health applications</a> by Graydon Snider, et al.</li></a>
          </li>
          <li>
            <a href="https://pubs.acs.org/doi/10.1021/acsestair.3c00069" class="" target="_none">Elemental Characterization of Ambient Particulate Matter for a Globally Distributed Monitoring Network: Methodology and Implications</a> by Xuan Liu, et al.</li></a>
          </li>
          <li>
            <a href="https://spartan-cloud.readthedocs.io/en/latest/index.html" class="" target="_none">Tutorial on using SPARTAN data on AWS</a> by <a href="https://github.com/haihui-zhu" target="_none">Haihui Zhu</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/acp-16-9629-2016" class="" target="_none">Variation in global chemical composition of PM2.5: emerging results from SPARTAN</a> by Graydon Snider, et al.</li></a>
          </li>
        </ul>
        <p><a href="/spartan-cloud/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="sau-global-fisheries-catch-data" class="dataset">
          <h3><a href="/sau-global-fisheries-catch-data/">Sea Around Us Global Fisheries Catch Data</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">fisheries</span><span class="label label-info tag link-tag">marine</span></p>
          <p>The project presents Sea Around Us Global Fisheries Catch Data aggregated at EEZ level. The data are computed from reconstructed catches from various official fisheries statistics, scientific, technical and policy reports about the fisheries, and includes estimation of discards, unreported and illegal catch data from all maritime countries and major territories of the world.This project was the result of a work between Sea Around Us and the CIC programme, a collaborative programme between the University of British Columbia (UBC) and AWS.</p>
          <p><a href="/sau-global-fisheries-catch-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.seaaroundus.org/data/" class="" target="_none">Sea Around Us Tools &amp; Data</a> by <a href="https://www.seaaroundus.org/people-2/" target="_none">Sea Around Us</a></li></a>
          </li>
          <li>
            <a href="https://www.seaaroundus.org/articles/" class="" target="_none">List of the Sea Around Us publications</a> by <a href="https://www.seaaroundus.org/people-2/" target="_none">Sea Around Us</a></li></a>
          </li>
          <li>
            <a href="https://github.com/SeaAroundUs/sau-ubc-cic-catchdata" class="" target="_none">Sea Around Us Global Fisheries Catch Data</a> by Sea Around Us, CIC</li></a>
          </li>
          <li>
            <a href="https://docs.google.com/document/d/1PNcTBnQHE9imF0JjZCs3zWNv1kOXbUHl/edit?usp&#x3D;sharing&amp;ouid&#x3D;105083864237876642152&amp;rtpof&#x3D;true&amp;sd&#x3D;true" class="" target="_none">Tutorial for analyzing Sea Around Us Global Fisheries Catch Data using Jupyter Notebook</a> by <a href="http://www.seaaroundus.org/shelumiel-ortiz-website-manager/" target="_none">Shelumiel Ortiz</a></li></a>
          </li>
          <li>
            <a href="https://islandpress.org/books/global-atlas-marine-fisheries" class="" target="_none">Global Atlas of Marine Fisheries: A critical appraisal of catches and ecosystem impacts. Island Press, Washington D.C. (2016)</a> by <a href="https://www.seaaroundus.org/people-2/" target="_none">Pauly, D.;  Zeller, D. (eds)</a></li></a>
          </li>
        </ul>
        <p><a href="/sau-global-fisheries-catch-data/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="sofar-spotter-archive" class="dataset">
          <h3><a href="/sofar-spotter-archive/">Sofar Spotter Archive</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>This dataset includes archival hourly data from the [Sofar Spotter buoy global network] (<a href="https://weather.sofarocean.com/">https://weather.sofarocean.com/</a>) from 2019 to March 2022. <br /></p>
          <p><a href="/sofar-spotter-archive/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://journals.ametsoc.org/view/journals/atot/36/6/jtech-d-18-0151.1.xml" class="" target="_none">Performance Characteristics of “Spotter,” a Newly Developed Real-Time Wave Measurement Buoy (2019)</a> by K. Raghukumar, G. Chang, F. Spada, C. Jones, T. Janssen, A. Gans</li></a>
          </li>
          <li>
            <a href="https://github.com/sofarocean/spotter-data-archive/blob/main/examples/plot_spotter_bulk_data.ipynb" class="" target="_none">Exploring Bulk Variables from the Spotter Archive</a> by Isabel A. Houghton</li></a>
          </li>
          <li>
            <a href="https://journals.ametsoc.org/view/journals/atot/38/5/JTECH-D-20-0187.1.xml" class="" target="_none">Performance Statistics of a Real-Time Pacific Ocean Weather Sensor Network (2021)</a> by I. Houghton, P. Smit, D. Clark, C. Dunning, A. Fisher, N. Nidzieko, P. Chamberlain, T. Janssen</li></a>
          </li>
          <li>
            <a href="https://github.com/sofarocean/spotter-data-archive/blob/main/examples/spotter_data_with_clouddrift.ipynb" class="" target="_none">Analyzing Spotter data with CloudDrift</a> by Milan Curcic</li></a>
          </li>
          <li>
            <a href="https://github.com/sofarocean/spotter-data-archive/blob/main/examples/plot_spotter_spectra_data.ipynb" class="" target="_none">Exploring Wave Spectra Variables from the Spotter Archive</a> by Isabel A. Houghton</li></a>
          </li>
        </ul>
        <p><a href="/sofar-spotter-archive/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="sondehub-telemetry" class="dataset">
          <h3><a href="/sondehub-telemetry/">SondeHub Radiosonde Telemetry</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">GPS</span><span class="label label-info tag link-tag">weather</span></p>
          <p>SondeHub Radiosonde telemetry contains global radiosonde (weather balloon) data captured by SondeHub from our participating radiosonde_auto_rx receiving stations. radiosonde_auto_rx is a open source project aimed at receiving and decoding telemetry from airborne radiosondes using software-defined-radio techniques, enabling study of the telemetry and sometimes recovery of the radiosonde itself.
Currently 313 receiver stations are providing data for an average of 384 radiosondes a day.  The data within this repository contains received telemetry frames, including radiosonde type, gps position, a...</p>
          <p><a href="/sondehub-telemetry/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/projecthorus/pysondehub/blob/main/examples/sondehub.ipynb" class="" target="_none">Using pysondehub to read radiosonde data</a> by Michaela Wheeler</li></a>
          </li>
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;YBy-bXEWZeM" class="" target="_none">STM32 Development Boards (literally) Falling From The Sky (How to submit data)</a> by Mark Jessop &amp; Michaela Wheeler</li></a>
          </li>
          <li>
            <a href="https://github.com/projecthorus/pysondehub/blob/main/examples/sondehub-athena.ipynb" class="" target="_none">Using Athena to read radiosonde data</a> by Michaela Wheeler</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/sondehub/" class="" target="_none">pysondehub</a> by <a href="https://github.com/projecthorus/pysondehub" target="_none">Sondehub</a></li></a>
          </li>
          <li>
            <a href="https://github.com/projecthorus/pysondehub/blob/main/examples/README.md" class="" target="_none">Loading example notebooks into SageMaker</a> by Michaela Wheeler</li></a>
          </li>
        </ul>
        <p><a href="/sondehub-telemetry/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="hcp-openaccess" class="dataset">
          <h3><a href="/hcp-openaccess/">The Human Connectome Project</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>The Human Connectome Project (HCP Young Adult, HCP-YA) is mapping the healthy human connectome by collecting and freely distributing neuroimaging and behavioral data on 1,200 normal young adults, aged 22-35.</p>
          <p><a href="/hcp-openaccess/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3720813/" class="" target="_none">The minimal preprocessing pipelines for the Human Connectome Project</a> by Glasser MF, Sotiropoulos SN, Wilson JA, Coalson TS, Fischl B, Andersson JL, Xu J, Jbabdi S, et al.</li></a>
          </li>
          <li>
            <a href="https://store.humanconnectome.org/courses/2019/exploring-the-human-connectome.php" class="" target="_none">Exploring the Human Connectom</a> by The Human Connectome Project</li></a>
          </li>
          <li>
            <a href="https://www.humanconnectome.org/software/connectome-workbench" class="" target="_none">The Human Connectome Workbench</a> by The Human Connectome Project</li></a>
          </li>
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S1053811921008168" class="" target="_none">The Human Connectome Project: A retrospective</a> by Elam JS, Glasser MF, Harms MP, Sotiropoulos SN, Andersson JL, Burgess GC, Curtiss SW, et al.</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3724347/" class="" target="_none">The WU-Minn Human Connectome Project: an overview.</a> by Van Essen DC, Smith SM, Barch DM, Behrens TEJ, Yacoub E, Ugurbil, K, and the WU-Minn HCP Consortium.</li></a>
          </li>
        </ul>
        <p><a href="/hcp-openaccess/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="vt-opendata" class="dataset">
          <h3><a href="/vt-opendata/">Vermont Open Geospatial on AWS</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">lidar</span></p>
          <p>The State of Vermont has partnered with Amazon&#39;s Open Data Initative to make a wide range of geospatial data available in the public domain. Vermont acquires aerial imagery and LiDAR during leaf-off conditions. The imagery typically ranges from 30-centimeter to 15-centimeter in resolution and is available from Vermont&#39;s Amazon S3 bucket in a Cloud Optimized GeoTiff (COG) format. LiDAR data has been acquired and is available as USGS Quality Level-1 (QL1) and Level-2 (QL2) compliant datasets in COG format. Geospatial datasets derived from imagery and/or lidar are also available as COGs, ...</p>
          <p><a href="/vt-opendata/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://vtopendata-prd.s3-us-east-2.amazonaws.com/index.html#Elevation/" class="" target="_none">Vermont AWS S3 Open Data Bucket Browser - Elevation</a> by Vermont Center for Geographic Information</li></a>
          </li>
          <li>
            <a href="https://geodata.vermont.gov/pages/land-cover" class="" target="_none">Landcover Page - Vermont Open Geodata Portal</a> by Vermont Center for Geographic Information</li></a>
          </li>
          <li>
            <a href="https://vtopendata-prd.s3-us-east-2.amazonaws.com/index.html#Imagery/" class="" target="_none">Vermont AWS S3 Open Data Bucket Browser - Imagery</a> by Vermont Center for Geographic Information</li></a>
          </li>
          <li>
            <a href="https://vtopendata-prd.s3-us-east-2.amazonaws.com/index.html#Landcover/" class="" target="_none">Vermont AWS S3 Open Data Bucket Browser - Landcover</a> by Vermont Center for Geographic Information</li></a>
          </li>
          <li>
            <a href="https://geodata.vermont.gov/pages/imagery" class="" target="_none">Imagery Page - Vermont Open Geodata Portal</a> by Vermont Center for Geographic Information</li></a>
          </li>
        </ul>
        <p><a href="/vt-opendata/#usageexamples">See 8 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-blast-databases" class="dataset">
          <h3><a href="/ncbi-blast-databases/">Basic Local Alignment Sequences Tool (BLAST) Databases</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>A centralized repository of pre-formatted BLAST databases created by the National Center for Biotechnology Information (NCBI).</p>
          <p><a href="/ncbi-blast-databases/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="" target="_none">Gapped BLAST and PSI-BLAST: A New Generation of Protein Database Search Programs</a> by S F Altschul, T L Madden, A A Schäffer, J Zhang, Z Zhang, W Miller, D J Lipman</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/20003500" class="" target="_none">BLAST+: Architecture and Applications</a> by Christiam Camacho  1 , George Coulouris, Vahram Avagyan, Ning Ma, Jason Papadopoulos, Kevin Bealer, Thomas L Madden</li></a>
          </li>
          <li>
            <a href="https://medium.com/codex/blast-on-the-cloud-with-ncbis-elasticblast-3b35b29afde7" class="" target="_none">BLAST on the Cloud with NCBI’s ElasticBLAST</a> by <a href="https://dgg32.medium.com/" target="_none">Sixing Huang</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ncbi/blast_plus_docs" class="" target="_none">BLAST+ Docker</a> by NCBI BLAST</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-blast-databases/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ccic" class="dataset">
          <h3><a href="/ccic/">Chalmers Cloud Ice Climatology</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">exploration</span><span class="label label-info tag link-tag">geophysics</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">ice</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>The Chalmers Cloud Ice Climatology (CCIC) is a novel, deep-learning-based climate record of ice-particle concentrations in the atmosphere. CCIC results are available at high spatial and temporal resolution (0.07° / 3 h from 1983, 0.036° / 30 min from 2000) and thus ideally suited for evaluating high-resolution weather and climate models or studying individual weather systems.</p>
          <p><a href="/ccic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ccic.readthedocs.io/en/latest/applications/storm_tracking.html" class="" target="_none">Storm tracking with CCIC</a> by <a href="https://juliakukulies.github.io/" target="_none">Julia Kukulies</a></li></a>
          </li>
          <li>
            <a href="https://ccic.readthedocs.io/en/latest/getting_started.html#reading-ccic-data" class="" target="_none">Reading CCIC data</a> by Adrià Amell &amp; Simon Pfreundschuh</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/ccic/" class="" target="_none">CCIC on PyPI -- software for training and processing the CCIC retrievals and to read CCIC Zarr files</a> by <a href="https://github.com/SEE-GEO/ccic" target="_none">Simon Pfreundschuh</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/egusphere-2023-1953" class="" target="_none">The Chalmers Cloud Ice Climatology: Retrieval implementation and validation</a> by Adrià Amell, Simon Pfreundschuh, and Patrick Eriksson</li></a>
          </li>
        </ul>
        <p><a href="/ccic/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ncar-cesm2-lens" class="dataset">
          <h3><a href="/ncar-cesm2-lens/">Community Earth System Model v2 Large Ensemble (CESM2 LENS)</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">ice</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>The US National Center for Atmospheric Research partnered with the IBS Center for Climate Physics in South Korea to generate the CESM2 Large Ensemble which consists of 100 ensemble members at 1 degree spatial resolution covering the period 1850-2100 under CMIP6 historical and SSP370 future radiative forcing scenarios. Data sets from this ensemble were made downloadable via the Climate Data Gateway on June 14th, 2021.
NCAR has copied a subset (currently ~500 TB) of CESM2 LENS data to Amazon S3 as part of the AWS Public Datasets Program. To optimize for large-scale analytics we have represented ...</p>
          <p><a href="/ncar-cesm2-lens/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ncar.github.io/cesm2-le-aws/kay_et_al_lens2.html" class="" target="_none">Rendered (static) version of Jupyter Notebook</a> by Maxwell Grover, NCAR</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5194/esd-2021-50" class="" target="_none">Ubiquity of human-induced changes in climate variability</a> by Rodgers et al. (2021), Earth Syst. Dynam., 12, 1–19, 2021</li></a>
          </li>
          <li>
            <a href="https://github.com/NCAR/cesm2-le-aws" class="" target="_none">Jupyter Notebook and other documentation and tools for CESM2-LE on AWS</a> by NCAR Science at Scale team</li></a>
          </li>
          <li>
            <a href="https://zhonghuazheng.com/UrbanClimateExplorer/pages/about.html" class="" target="_none">Urban Climate Explorer</a> by Zhonghua Zheng</li></a>
          </li>
        </ul>
        <p><a href="/ncar-cesm2-lens/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-pds-dsgrid" class="dataset">
          <h3><a href="/nrel-pds-dsgrid/">Demand-Side Grid (dsgrid) Toolkit</a></h3>
          <p><span class="label label-info tag link-tag">data assimilation</span><span class="label label-info tag link-tag">electricity</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">energy modeling</span><span class="label label-info tag link-tag">industrial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">solar</span><span class="label label-info tag link-tag">transportation</span></p>
          <p>Projects that use the dsgrid toolkit assemble bottom-up descriptions of electricity 
demand and related data that are highly resolved geographically, temporally, and sectorally. 
Typically modelers describe multiple scenarios of future energy use at hourly resolution, 
suitable for inclusion in long-term power system planning models, i.e., capacity expansion and 
production cost models. </p>
          <p><a href="/nrel-pds-dsgrid/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/dsgrid/dsgrid-project-StandardScenarios/tree/main/tempo_project" class="" target="_none">dsgrid Project Standard Scenarios for the TEMPO Project</a> by Elaine Hale</li></a>
          </li>
          <li>
            <a href="https://dsgrid.github.io/dsgrid/" class="" target="_none">dsgrid Documentation</a> by Elaine Hale</li></a>
          </li>
          <li>
            <a href="https://github.com/dsgrid/dsgrid-legacy-efs-api" class="" target="_none">Python API for Accessing dsgrid Data for the Electrification Futures Study (EFS)</a> by Elaine Hale</li></a>
          </li>
          <li>
            <a href="https://github.com/dsgrid/dsgrid" class="" target="_none">GitHub Repository for Working with the dsgrid Projects</a> by Elaine Hale</li></a>
          </li>
          <li>
            <a href="https://www.nrel.gov/docs/fy23osti/83916.pdf" class="" target="_none">Highly Resolved Projections of Passenger Electric Vehicle Charging Loads for the Contiguous United States</a> by Arthur Yip, Christopher Hoehne, Paige Jadun, Catherine Ledna, Elaine Hale, and Matteo Muratori</li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-dsgrid/#usageexamples">See 7 usage examples &rarr;</a></p>
        </div>
        <div id="ecmwf-forecasts" class="dataset">
          <h3><a href="/ecmwf-forecasts/">ECMWF real-time forecasts</a></h3>
          <p><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">near-surface air temperature</span><span class="label label-info tag link-tag">near-surface relative humidity</span><span class="label label-info tag link-tag">near-surface specific humidity</span><span class="label label-info tag link-tag">precipitation</span><span class="label label-info tag link-tag">weather</span></p>
          <p>These products are a subset of the ECMWF real-time forecast data and are made available to the public free of charge. They are based on the medium-range (high-resolution and ensemble) and seasonal forecast models. Products are available at 0.4 degrees resolution in GRIB2 format unless stated otherwise. </p>
          <p><a href="/ecmwf-forecasts/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ecmwf/ecmwf-opendata" class="" target="_none">A package to simplify the download of ECMWF open data. It implements a request-based interface to the dataset using ECMWF&#x27;s MARS language to select meteorological fields, similar to the existing ecmwf-api-client Python package.</a> by <a href="https://www.ecmwf.int/" target="_none">ECMWF</a></li></a>
          </li>
          <li>
            <a href="https://www.ecmwf.int/en/about/media-centre/news/2022/ecmwf-makes-wide-range-data-openly-available" class="" target="_none">ECMWF makes wide range of data openly available</a> by <a href="https://www.ecmwf.int/" target="_none">ECMWF</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ecmwf/notebook-examples/tree/master/opencharts" class="" target="_none">A collection of Jupyter Notebooks that make use of the ecmwf-opendata package. This package simplifies the download of ECMWF open data.</a> by <a href="https://www.ecmwf.int/" target="_none">ECMWF</a></li></a>
          </li>
          <li>
            <a href="https://github.com/blaylockbk/Herbie" class="" target="_none">Herbie: A Python package to retrieve NWP data.</a> by <a href="https://github.com/blaylockbk" target="_none">Brian Blaylock</a></li></a>
          </li>
        </ul>
        <p><a href="/ecmwf-forecasts/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="gnss-ro-opendata" class="dataset">
          <h3><a href="/gnss-ro-opendata/">Earth Radio Occultation</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">signal processing</span><span class="label label-info tag link-tag">weather</span></p>
          <p>This is an updating archive of radio occultation (RO) data using the transmitters of the Global Navigation Satellite Systems (GNSS) as generated and processed by the COSMIC DAAC (ucar), the Jet Propulsion Laboratory (jpl) of the California Institute of Technology, and the Radio Occultation Meteorology Satellite Application Facility (romsaf). The contributions for ucar and romsaf are currently active. <p> This dataset is funded by the NASA Earth Science Data Systems and the Advancing Collaborative Connections for Earth System Science (ACCESS) 2019 program.</p>
          <p><a href="/gnss-ro-opendata/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="doi.org/10.1029/97JD01569" class="" target="_none">Observing Earth&#x27;s atmosphere with radio occultation measurements using the Global Positioning System</a> by Kursinski, E.R. et al.</li></a>
          </li>
          <li>
            <a href="http://github.com/gnss-ro/aws-opendata/tree/master/tutorials/" class="" target="_none">Tutorial Demonstrations for Working with GNSS Radio Occultation Data in AWS Open Data Registry archive</a> by Stephen Leroy, Amy McVey, Hailing Zhang, Chi Ao</li></a>
          </li>
          <li>
            <a href="http://github.com/gnss-ro/aws-opendata/tree/master/awsgnssroutils/" class="" target="_none">Utilities for Handling Data in the AWS Open Data Registry archive of GNSS Radio Occultation Data</a> by <a href="http://www.aer.com" target="_none">Stephen Leroy, Amy McVey</a></li></a>
          </li>
          <li>
            <a href="http://doi.org/10.1016/S1364-6826(01)00114-6" class="" target="_none">A technical description of atmospheric sounding by GPS occultation</a> by Hajj, G.A. et al.</li></a>
          </li>
        </ul>
        <p><a href="/gnss-ro-opendata/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="encode-project" class="dataset">
          <h3><a href="/encode-project/">Encyclopedia of DNA Elements (ENCODE)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Encyclopedia of DNA Elements (ENCODE) Consortium is an international collaboration of
research groups funded by the National Human Genome Research Institute (NHGRI). The goal
of ENCODE is to build a comprehensive parts list of functional elements in the human genome,
including elements that act at the protein and RNA levels, and regulatory elements that
control cells and circumstances in which a gene is active. ENCODE investigators employ a
variety of assays and methods to identify functional elements. The discovery and annotation
of gene elements is accomplished primarily by sequencing a ...</p>
          <p><a href="/encode-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb" class="" target="_none">Ingesting ENCODE data into TileDB with S3 backend</a> by <a href="https://github.com/ottojolanki" target="_none">Otto Jolanki</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb" class="" target="_none">Exploring ENCODE data from EC2 with Jupyter notebook</a> by <a href="https://github.com/keenangraham" target="_none">Keenan Graham</a></li></a>
          </li>
          <li>
            <a href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ctcf_chip_seq_cross_cell_type_correlation.ipynb" class="" target="_none">ENCODE CTCF ChIP-seq data correlation across different cell types</a> by <a href="https://github.com/p-sud" target="_none">Paul Sud</a></li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715" class="" target="_none">New developments on the Encyclopedia of DNA Elements (ENCODE) data portal</a> by <a href="http://orcid.org/0000-0001-6948-2042" target="_none">Luo et al 2020</a></li></a>
          </li>
        </ul>
        <p><a href="/encode-project/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="giab" class="dataset">
          <h3><a href="/giab/">Genome in a Bottle on AWS</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">vcf</span></p>
          <p>Several reference genomes to enable translation of whole human genome sequencing to clinical practice. On 11/12/2020 these data were updated to reflect the most <a href="https://www.nist.gov/programs-projects/genome-bottle">up to date GIAB release</a>.</p>
          <p><a href="/giab/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/sdata201625" class="" target="_none">Extensive sequencing of seven human genomes to characterize benchmark reference materials</a> by Zook J et al (2016)</li></a>
          </li>
          <li>
            <a href="https://github.com/ga4gh/benchmarking-tools" class="" target="_none">GA4GH Benchmarking Tools</a> by GA4GH Benchmarking Team</li></a>
          </li>
          <li>
            <a href="https://github.com/genome-in-a-bottle/about_GIAB" class="" target="_none">The Genome in a Bottle Github Project</a> by Genome In A Bottle Consortium</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41597-019-0098-2" class="" target="_none">High-coverage, long-read sequencing of Han Chinese trio reference samples</a> by Wang Y et al (2019)</li></a>
          </li>
        </ul>
        <p><a href="/giab/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="dataforgood-fb-forests" class="dataset">
          <h3><a href="/dataforgood-fb-forests/">High Resolution Canopy Height Maps by WRI and Meta</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>Global and regional Canopy Height Maps (CHM). Created using machine learning models on high-resolution worldwide Maxar satellite imagery. </p>
          <p><a href="/dataforgood-fb-forests/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://meta-forest-monitoring-okw37.projects.earthengine.app/view/canopyheight" class="" target="_none">Global Canopy Height on Earth Engine</a> by Meta and WRI</li></a>
          </li>
          <li>
            <a href="https://research.facebook.com/blog/2023/4/every-tree-counts-large-scale-mapping-of-canopy-height-at-the-resolution-of-individual-trees/" class="" target="_none">Every tree counts: Large-scale mapping of canopy height at the resolution of individual trees</a> by Jamie Tolan, Camille Couprie, and Tracy Johns</li></a>
          </li>
          <li>
            <a href="https://sustainability.fb.com/blog/2024/04/22/using-artificial-intelligence-to-map-the-earths-forests/" class="" target="_none">Using Artificial Intelligence to Map the Earth’s Forests</a> by Jamie Tolan, Camille Couprie, John Brandt, Justine Spore, Tobias Tiecke, Tracy Johns and Patrick Nease</li></a>
          </li>
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S003442572300439X" class="" target="_none">Sub-meter resolution canopy height maps using self-supervised learning and a vision transformer trained on Aerial and GEDI Lidar</a> by Jamie Tolan, Hung-I Yang, Ben Nosarzewski, Guillaume Couairon, Huy Vo, John Brandt, Justine Spore, Sayantan Majumdar, Daniel Haziza, Janaki Vamaraju, Theo Moutakanni, Piotr Bojanowski, Tracy Johns, Brian White, Tobias Tiecke, Camille Couprie</li></a>
          </li>
        </ul>
        <p><a href="/dataforgood-fb-forests/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ideam-radares" class="dataset">
          <h3><a href="/ideam-radares/">IDEAM - Colombian Radar Network</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Historical and one-day delay data from the IDEAM radar network.</p>
          <p><a href="/ideam-radares/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.openradarscience.org/projects/xradar/en/stable/notebooks/Read-plot-Sigmet-data-from-AWS.html" class="" target="_none">Read and plot Sigmet files available on AWS using Xradar</a> by Alfonso Ladino</li></a>
          </li>
          <li>
            <a href="https://projectpythia.org/AtmosCol-2023/notebooks/2.acceso-datos/2.2.Radares.html" class="" target="_none">Taller de datos científicos con Python y R - AtmosCol 2023</a> by Alfonso Ladino, Nicole Rivera, Max Grover</li></a>
          </li>
          <li>
            <a href="https://projectpythia.org/radar-cookbook/notebooks/example-workflows/kdp-comparison.html" class="" target="_none">Specific Differential Phase (KDP) retrieval methods comparison</a> by Alfonso Ladino, Max Grover</li></a>
          </li>
          <li>
            <a href="http://www.pronosticosyalertas.gov.co/documents/78690/112207077/AWS_RADARESCOL.pdf/aafc181d-0b6a-47e0-97e9-9b6290ac84d6" class="" target="_none">Guia de como explorar y plotear los archivos de radar utilizando el lenguaje de programación Python</a> by IDEAM</li></a>
          </li>
        </ul>
        <p><a href="/ideam-radares/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="jaxa-usgs-nasa-kaguya-tc-dtms" class="dataset">
          <h3><a href="/jaxa-usgs-nasa-kaguya-tc-dtms/">JAXA / USGS / NASA Kaguya/SELENE Terrain Camera Digital Terrain Models</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The Japan Aerospace EXploration Agency (JAXA) SELenological and ENgineering Explorer (SELENE) mission’s Kaguya spacecraft was launched on September 14, 2007 and science operations around the Moon started October 20, 2007. The primary mission in a circular polar orbit 100-km above the surface lasted from October 20, 2007 until October 31, 2008. An extended mission was then conducted in lower orbits (averaging 50km above the surface) from November 1, 2008 until the SELENE mission ended with Kaguya impacting the Moon on June 10, 2009. These data are digital terrain models derived using the NASA A...</p>
          <p><a href="/jaxa-usgs-nasa-kaguya-tc-dtms/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/examples/to_qgis/" class="" target="_none">Querying for Data in an ROI and Loading it into QGIS</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/cli/" class="" target="_none">Discovering and Downloading Data via the Command Line</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/basicpython/" class="" target="_none">Discovering and Downloading Data with Python</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://github.com/stac-utils/pystac-client" class="" target="_none">PySTAC Client</a> by <a href="https://github.com/stac-utils/pystac-client/graphs/contributors" target="_none">PySTAC-Client Contributors</a></li></a>
          </li>
        </ul>
        <p><a href="/jaxa-usgs-nasa-kaguya-tc-dtms/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="jaxa-usgs-nasa-kaguya-tc" class="dataset">
          <h3><a href="/jaxa-usgs-nasa-kaguya-tc/">JAXA / USGS / NASA Kaguya/SELENE Terrain Camera Observations</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The Japan Aerospace EXploration Agency (JAXA) SELenological and ENgineering Explorer (SELENE) mission’s Kaguya spacecraft was launched on September 14, 2007 and science operations around the Moon started October 20, 2007. The primary mission in a circular polar orbit 100-km above the surface lasted from October 20, 2007 until October 31, 2008. An extended mission was then conducted in lower orbits (averaging 50km above the surface) from November 1, 2008 until the SELENE mission ended with Kaguya impacting the Moon on June 10, 2009. These data were collected in monoscopic observing mode. To cre...</p>
          <p><a href="/jaxa-usgs-nasa-kaguya-tc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/stac-utils/pystac-client" class="" target="_none">PySTAC Client</a> by <a href="https://github.com/stac-utils/pystac-client/graphs/contributors" target="_none">PySTAC-Client Contributors</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/basicpython/" class="" target="_none">Discovering and Downloading Data with Python</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/cli/" class="" target="_none">Discovering and Downloading Data via the Command Line</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/examples/to_qgis/" class="" target="_none">Querying for Data in an ROI and Loading it into QGIS</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
        </ul>
        <p><a href="/jaxa-usgs-nasa-kaguya-tc/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="met-office-global-ensemble" class="dataset">
          <h3><a href="/met-office-global-ensemble/">Met Office Global Ensemble Prediction System (MOGREPS-G) on a 30-day rolling archive</a></h3>
          <p><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">forecast</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">near-surface air temperature</span><span class="label label-info tag link-tag">near-surface relative humidity</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The flagship Numerical Weather Prediction (NWP) model developed and used at the Met Office, is the Unified Model, the same model is used for both weather and climate prediction. For weather forecasting the Met Office runs several configurations of the Unified Model as part of its operational Numerical Weather Prediction suite. The global ensemble (MOGREPS-G) produces forecasts for the whole globe up to a week ahead. The projection used is the Equirectangular Latitude-Longitude and the grid resolution is 20km. The forecast is updated regularly with a 4-hour time delay and formatted via NetCDF. ...</p>
          <p><a href="/met-office-global-ensemble/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://scitools-iris.readthedocs.io/en/stable/userguide/index.html" class="" target="_none">Iris</a> by <a href="https://scitools-iris.readthedocs.io/en/stable/copyright.html" target="_none">Iris Contributors</a></li></a>
          </li>
          <li>
            <a href="https://www.metoffice.gov.uk/research/weather/ensemble-forecasting/what-is-an-ensemble-forecast" class="" target="_none">What is an ensemble forecast?</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
          <li>
            <a href="https://www.metoffice.gov.uk/research/approach/modelling-systems/unified-model" class="" target="_none">The Met Office Unified Model</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
          <li>
            <a href="https://www.metoffice.gov.uk/research/approach/modelling-systems/unified-model/weather-forecasting" class="" target="_none">Numerical weather prediction models</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
        </ul>
        <p><a href="/met-office-global-ensemble/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="mp2prt" class="dataset">
          <h3><a href="/mp2prt/">Molecular Profiling to Predict Response to Treatment (phs001965)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Molecular Profiling to Predict Response to Treatment (MP2PRT) program is part of the NCI&#39;s Cancer Moonshot Initiative. The aim of this program is the retrospective characterization and analysis of biospecimens collected from completed NCI-sponsored trials of the National Clinical Trials Network and the NCI Community Oncology Research Program. This study, titled &quot;Identification of Genetic Changes Associated with Relapse and/or Adaptive Resistance in Patients Registered as Favorable Histology Wilms Tumor on AREN03B2&quot;, performs genomic characterization (WGS 30X, Total RNAseq, mi...</p>
          <p><a href="/mp2prt/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.xcrm.2022.100667" class="" target="_none">Finding the way to Wilms tumor by comparing the primary and relapse tumor samples</a> by Filippo Spreafico, Sara Ciceri, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.xcrm.2022.100644" class="" target="_none">Genetic changes associated with relapse in favorable histology Wilms tumor: A Children&#x27;s Oncology Group AREN03B2 study</a> by Samantha Gadd, Vicki Huff, et al.</li></a>
          </li>
          <li>
            <a href="https://datacatalog.ccdi.cancer.gov/dataset/GDC-MP2PRT" class="" target="_none">Childhood Cancer Data Initiative Data Catalog</a> by <a href="https://datacatalog.ccdi.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/mp2prt/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="janelia-mouselight" class="dataset">
          <h3><a href="/janelia-mouselight/">Mouse Brain Anatomy: MouseLight Imagery</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>This data set, made available by Janelia&#39;s MouseLight project, consists of 
images and neuron annotations of the Mus musculus brain, stored in formats suitable
for viewing and annotation using the HortaCloud cloud-based annotation system.</p>
          <p><a href="/janelia-mouselight/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.janelia.org/project-team/mouselight" class="" target="_none">MouseLight Project Website</a> by Tiago A. Ferreira, Jayaram Chandrashekar</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.cell.2019.07.042" class="" target="_none">Reconstruction of 1,000 Projection Neurons Reveals New Cell Types and Organization of Long-Range Connectivity in the Mouse Brain</a> by Johan Winnubst, Erhan Bas, Tiago A. Ferreira, Zhuhao Wu, Michael N. Economo, Patrick Edson, Ben J. Arthur, Christopher Bruns, Konrad Rokicki, David Schauder, Donald J. Olbris, Sean D. Murphy, David G. Ackerman, Cameron Arshadi, Perry Baldwin, Regina Blake, Ahmad Elsayed, Mashtura Hasan, Daniel Ramirez, Bruno Dos Santos, Monet Weldon, Amina Zafar, Joshua T. Dudman, Charles R. Gerfen, Adam W. Hantman, Wyatt Korff, Scott M. Sternson, Nelson Spruston, Karel Svoboda, Jayaram Chandrashekar</li></a>
          </li>
          <li>
            <a href="http://ml-neuronbrowser.janelia.org/" class="" target="_none">MouseLight NeuronBrowser</a> by Tiago A. Ferreira, Jayaram Chandrashekar</li></a>
          </li>
          <li>
            <a href="https://hortacloud.janelia.org/" class="" target="_none">HortaCloud</a> by David Schauder, Donald J. Olbris, Jody Clements, Cristian Goina, Robert R. Svirskas, Konrad Rokicki</li></a>
          </li>
        </ul>
        <p><a href="/janelia-mouselight/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ncar-na-cordex" class="dataset">
          <h3><a href="/ncar-na-cordex/">NA-CORDEX - North American component of the Coordinated Regional Downscaling Experiment</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">model</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>The NA-CORDEX dataset contains regional climate change scenario data and guidance for North America, for use in impacts, decision-making, and climate science. The NA-CORDEX data archive contains output from regional climate models (RCMs) run over a domain covering most of North America using boundary conditions from global climate model (GCM) simulations in the CMIP5 archive. These simulations run from 1950–2100 with a spatial resolution of 0.22°/25km or 0.44°/50km.  This AWS S3 version of the data includes selected variables converted to Zarr format from the original NetCDF. Only daily data a...</p>
          <p><a href="/ncar-na-cordex/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ncar-na-cordex.s3-us-west-2.amazonaws.com/catalogs/aws-na-cordex.json" class="" target="_none">Intake-ESM Catalog</a> by Brian Bonnlander (NCAR)</li></a>
          </li>
          <li>
            <a href="http://ncar-aws-www.s3-website-us-west-2.amazonaws.com/plot-zarr-diagnostics.html" class="" target="_none">Rendered (static) version of Jupyter Notebook</a> by Brian Bonnlander (NCAR)</li></a>
          </li>
          <li>
            <a href="https://github.com/NCAR/na-cordex-aws" class="" target="_none">Jupyter Notebook and other documentation and tools</a> by Brian Bonnlander, Seth McGinnis (NCAR)</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.5065/D6SJ1JCH" class="" target="_none">The NA-CORDEX dataset, version 1.0. NCAR Climate Data Gateway, Boulder CO (2017)</a> by Mearns, Linda O., et al.</li></a>
          </li>
        </ul>
        <p><a href="/ncar-na-cordex/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="naip" class="dataset">
          <h3><a href="/naip/">NAIP on AWS</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">regulatory</span></p>
          <p>The National Agriculture Imagery Program (NAIP) acquires aerial imagery during the agricultural growing seasons in the continental U.S. This &quot;leaf-on&quot; imagery andtypically ranges from 30 centimeters to 100 centimeters in resolution and is available from the naip-analytic Amazon S3 bucket as 4-band (RGB + NIR) imagery in MRF format, on naip-source Amazon S3 bucket as 4-band (RGB + NIR) in uncompressed Raw GeoTiff format and naip-visualization as 3-band (RGB) Cloud Optimized GeoTiff format. More details on <a href="https://naip-usdaonline.hub.arcgis.com/">NAIP</a></p>
          <p><a href="/naip/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.48550/arXiv.2208.10607" class="" target="_none">Individual Tree Detection in Large-Scale Urban Environments using High-Resolution Multispectral Imagery</a> by Jonathan Ventura, Milo Honsberger, Cameron Gonsalves, Julian Rice, Camille Pawlak, Natalie L.R. Love, Skyler Han, Viet Nguyen, Keilana Sugano, Jacqueline Doremus, G. Andrew Fricker, Jenn Yost, Matt Ritter</li></a>
          </li>
          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></li></a>
          </li>
          <li>
            <a href="https://github.com/jonathanventura/urban-tree-detection" class="" target="_none">Urban Tree Detection</a> by Jonathan Ventura</li></a>
          </li>
          <li>
            <a href="https://www.voyagersearch.com/using-aws-batch-to-generate-image-thumbnails-for-voyager" class="" target="_none">VoyagerSearch showing off Batch + NAIP</a> by <a href="https://www.voyagersearch.com/" target="_none">Voyager</a></li></a>
          </li>
        </ul>
        <p><a href="/naip/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-usgs-europa-dtms" class="dataset">
          <h3><a href="/nasa-usgs-europa-dtms/">NASA / USGS Controlled Europa DTMs</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Knowledge of a planetary surface’s topography is necessary to understand its geology and enable landed mission operations. The Solid State Imager (SSI) on board NASA’s Galileo spacecraft acquired more than 700 images of Jupiter’s moon Europa. Although moderate- and high-resolution coverage is extremely limited, repeat coverage of a small number of sites enables the creation of digital terrain models (DTMs) via stereophotogrammetry. Here we provide stereo-derived DTMs of five sites on Europa. The sites are the bright band Agenor Linea, the crater Cilix, the crater Pwyll, pits and chaos adjacent...</p>
          <p><a href="/nasa-usgs-europa-dtms/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/basicpython/" class="" target="_none">Discovering and Downloading Data with Python</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://github.com/stac-utils/pystac-client" class="" target="_none">PySTAC Client</a> by <a href="https://github.com/stac-utils/pystac-client/graphs/contributors" target="_none">PySTAC-Client Contributors</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/cli/" class="" target="_none">Discovering and Downloading Data via the Command Line</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/examples/to_qgis/" class="" target="_none">Querying for Data in an ROI and Loading it into QGIS</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-usgs-europa-dtms/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-usgs-themis-mosaics" class="dataset">
          <h3><a href="/nasa-usgs-themis-mosaics/">NASA / USGS Controlled THEMIS Mosaics</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>These data are infrared image mosaics, tiled to the Mars quadrangle, generated using Thermal Emission Imaging System (THEMIS) images from the 2001 Mars Odyssey orbiter mission. The mosaic is generated at the full resolution of the THEMIS infrared dataset, which is approximately 100 meters/pixel. The mosaic was absolutely photogrammetrically controlled to an improved Viking MDIM network that was develop by the USGS Astrogeology processing group using the Integrated Software for Imagers and Spectrometers. Image-to-image alignment precision is subpixel (i.e., &lt;100m). These 8-bit, qualitative d...</p>
          <p><a href="/nasa-usgs-themis-mosaics/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/examples/to_qgis/" class="" target="_none">Querying for Data in an ROI and Loading it into QGIS</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://github.com/stac-utils/pystac-client" class="" target="_none">PySTAC Client</a> by <a href="https://github.com/stac-utils/pystac-client/graphs/contributors" target="_none">PySTAC-Client Contributors</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/cli/" class="" target="_none">Discovering and Downloading Data via the Command Line</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/basicpython/" class="" target="_none">Discovering and Downloading Data with Python</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-usgs-themis-mosaics/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-usgs-europa-mosaics" class="dataset">
          <h3><a href="/nasa-usgs-europa-mosaics/">NASA / USGS Europa Controlled Observation Mosaics</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The Solid State Imager (SSI) on NASA&#39;s Galileo spacecraft acquired more than 500 images of Jupiter&#39;s moon, Europa. These images vary from relatively low-resolution hemispherical imaging, to high-resolution targeted images that cover a small portion of the surface. Here we provide a set of 92 image mosaics generated from minimally processed, projected Galileo images with photogrammetrically improved locations on Europa&#39;s surface.<br/><br/>
These images provide users with nearly the entire Galileo Europa imaging dataset at its native resolution and with improved relative image locations. The S</br/>...</p>
          <p><a href="/nasa-usgs-europa-mosaics/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/examples/to_qgis/" class="" target="_none">Querying for Data in an ROI and Loading it into QGIS</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/cli/" class="" target="_none">Discovering and Downloading Data via the Command Line</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/basicpython/" class="" target="_none">Discovering and Downloading Data with Python</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://github.com/stac-utils/pystac-client" class="" target="_none">PySTAC Client</a> by <a href="https://github.com/stac-utils/pystac-client/graphs/contributors" target="_none">PySTAC-Client Contributors</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-usgs-europa-mosaics/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-usgs-europa-observations" class="dataset">
          <h3><a href="/nasa-usgs-europa-observations/">NASA / USGS Europa Controlled Observations</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The Solid State Imager (SSI) on NASA&#39;s Galileo spacecraft acquired more than 500 images of Jupiter&#39;s moon, Europa. These images vary from relatively low-resolution hemispherical imaging, to high-resolution targeted images that cover a small portion of the surface. Here we provide a set of 481 minimally processed, projected Galileo images with photogrammetrically improved locations on Europa&#39;s surface. These individual images were subsequently used as input into a set of 92 observation mosaics. <br/><br/>
These images provide users with nearly the entire Galileo Europa imaging dataset at its native resolution and with improved relative image locations. The Solid State Imager on NASA&#39;s Galileo spacecraft provided the only moderate- to high-resolution images of Jupiter&#39;s moon, Europa. Unfortunately, uncertainty in the position and pointing of the spacecraft, as well as the position and orientation of Europa, when the images were acquired resulted in significant errors in image locations on the surface. The result of these errors is that images acquired during different Galileo orbits, or even at different times during the same orbit, are significantly misaligned (errors of up to 100 km on the surface). <br/><br/>
The dataset provides a set of individual images that can be used for scientific analysis</br/>...</p>
          <p><a href="/nasa-usgs-europa-observations/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/cli/" class="" target="_none">Discovering and Downloading Data via the Command Line</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/basicpython/" class="" target="_none">Discovering and Downloading Data with Python</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://github.com/stac-utils/pystac-client" class="" target="_none">PySTAC Client</a> by <a href="https://github.com/stac-utils/pystac-client/graphs/contributors" target="_none">PySTAC-Client Contributors</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/examples/to_qgis/" class="" target="_none">Querying for Data in an ROI and Loading it into QGIS</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-usgs-europa-observations/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-usgs-controlled-mro-ctx-dtms" class="dataset">
          <h3><a href="/nasa-usgs-controlled-mro-ctx-dtms/">NASA / USGS Mars Reconnaissance Orbiter (MRO) Context Camera (CTX) Targeted DTMs</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>As of March, 2023 the Mars Reconnaissance Orbiter (MRO) High Resolution Science Experiment (HiRISE) sensor has collected more than 5000 targeted stereopairs. During HiRISE acquisition, the Context Camera (CTX) also collects lower resolution, higher spatial extent context images. These CTX acquisitions are also targeted stereopairs. This data set contains targeted CTX DTMs and orthoimages, created using the NASA Ames Stereopipeline. These data have been created using relatively controlled CTX images that have been globally bundle adjusted using the USGS Integrated System for Imagers and Spectro...</p>
          <p><a href="/nasa-usgs-controlled-mro-ctx-dtms/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/stac-utils/pystac-client" class="" target="_none">PySTAC Client</a> by <a href="https://github.com/stac-utils/pystac-client/graphs/contributors" target="_none">PySTAC-Client Contributors</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/cli/" class="" target="_none">Discovering and Downloading Data via the Command Line</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/examples/to_qgis/" class="" target="_none">Querying for Data in an ROI and Loading it into QGIS</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/basicpython/" class="" target="_none">Discovering and Downloading Data with Python</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-usgs-controlled-mro-ctx-dtms/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-usgs-mars-hirise-dtms" class="dataset">
          <h3><a href="/nasa-usgs-mars-hirise-dtms/">NASA / USGS Released HiRISE Digital Terrain Models</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>These data are digital terrain models (DTMs) created by multiple different institutions and released to the Planetary Data System (PDS) by the University of Arizona. The data are processed from the Planetary Data System (PDS) stored JP2 files, map projected, and converted to Cloud Optimized GeoTiffs (COGs) for efficient remote data access. These data are controlled to the Mars Orbiter Laser Altimeter (MOLA). Therefore, they are a proxy for the geodetic coordinate reference frame. These data are not guaranteed to co-register with an uncontrolled products (e.g., the uncontrolled High Resolution ...</p>
          <p><a href="/nasa-usgs-mars-hirise-dtms/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/examples/to_qgis/" class="" target="_none">Querying for Data in an ROI and Loading it into QGIS</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://github.com/stac-utils/pystac-client" class="" target="_none">PySTAC Client</a> by <a href="https://github.com/stac-utils/pystac-client/graphs/contributors" target="_none">PySTAC-Client Contributors</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/basicpython/" class="" target="_none">Discovering and Downloading Data with Python</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/cli/" class="" target="_none">Discovering and Downloading Data via the Command Line</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-usgs-mars-hirise-dtms/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-usgs-mars-hirise" class="dataset">
          <h3><a href="/nasa-usgs-mars-hirise/">NASA / USGS Uncontrolled HiRISE RDRs</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">planetary</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>These data are red and color Reduced Data Record (RDR) observations collected and originally processed by the High Resolution Imaging Science Experiment (HiRISE) team. The mdata are processed from the Planetary Data System (PDS) stored RDRs, map projected, and converted to Cloud Optimized GeoTiffs (COGs) for efficient remote data access. These data are not photogrammetrically controlled and use a priori NAIF SPICE pointing. Therefore, these data will not co-register with controlled data products. Data are released using simple cylindrical (planetocentric positive East, center longitude 0, -180...</p>
          <p><a href="/nasa-usgs-mars-hirise/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/cli/" class="" target="_none">Discovering and Downloading Data via the Command Line</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://github.com/stac-utils/pystac-client" class="" target="_none">PySTAC Client</a> by <a href="https://github.com/stac-utils/pystac-client/graphs/contributors" target="_none">PySTAC-Client Contributors</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/examples/to_qgis/" class="" target="_none">Querying for Data in an ROI and Loading it into QGIS</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
          <li>
            <a href="https://stac.astrogeology.usgs.gov/docs/tutorials/basicpython/" class="" target="_none">Discovering and Downloading Data with Python</a> by <a href="https://astrogeology.usgs.gov" target="_none">J. Laura</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-usgs-mars-hirise/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-ghcn" class="dataset">
          <h3><a href="/noaa-ghcn/">NOAA Global Historical Climatology Network Daily (GHCN-D)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p><br /> UPDATE TO GHCN PREFIXES - The NODD team is working on improving performance and access to the GHCNd data and will be implementing an updated prefix structure. For more information on the prefix changes, please see the &quot;<a href="https://github.com/NOAA-Big-Data-Program/bdp-data-docs/tree/main/GHCN-D">READ ME on the NODD Github</a>&quot;. If you have questions, comments, or feedback, please reach out to <a href="mailto:&#x6e;&#111;&#100;&#100;&#64;&#x6e;&#111;&#97;&#x61;&#46;&#103;&#111;&#x76;">&#x6e;&#111;&#100;&#100;&#64;&#x6e;&#111;&#97;&#x61;&#46;&#103;&#111;&#x76;</a> with GHCN in the subject line. <br /> <br >
Global Historical Climatology Network - Daily is a dataset from NOAA that contains daily observations over global land areas. It contains station-based measurements f...</p>
          <p><a href="/noaa-ghcn/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/industries/calculating-growing-degree-days-using-aws-registry-of-open-data/" class="aws-link" target="_none">Calculating growing degree days using AWS Registry of Open Data</a> by Karen Hildebrand and Zac Flamig</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/visualize-over-200-years-of-global-climate-data-using-amazon-athena-and-amazon-quicksight/" class="aws-link" target="_none">Visualize over 200 years of global climate data using Amazon Athena and Amazon QuickSight</a> by Conor Delaney</li></a>
          </li>
          <li>
            <a href="http://www.pjoes.com/Natural-and-Socioeconomic-Conditions-Influence-nTick-Borne-Encephalitis-Cases-in,153552,0,2.html" class="" target="_none">Natural and Socioeconomic Conditions Influence Tick-Borne Encephalitis Cases in Russia</a> by Lantian Zhang, Linsheng Yang, Li Wang, Lijuan Gu, Hairong Li, Svetlana Malkhazova</li></a>
          </li>
          <li>
            <a href="https://medium.com/swlh/explore-visualize-200-years-of-global-temperature-using-apache-spark-bigquery-and-google-data-699ed8b3c67?sk&#x3D;3721ca3220a75af80293ac3e4bf3f4d1" class="" target="_none">Explore &amp; Visualize 200+ Years of Global Temperature</a> by Kapil Sreedharan</li></a>
          </li>
        </ul>
        <p><a href="/noaa-ghcn/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-hrrr-pds" class="dataset">
          <h3><a href="/noaa-hrrr-pds/">NOAA High-Resolution Rapid Refresh (HRRR) Model</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The HRRR is a NOAA real-time 3-km resolution, hourly updated, cloud-resolving, convection-allowing atmospheric model, initialized by 3km grids with 3km radar assimilation. Radar data is assimilated in the HRRR every 15 min over a 1-h period adding further detail to that provided by the hourly data assimilation from the 13km radar-enhanced Rapid Refresh. <br /> <br /> The HRRR ZARR formatted data was originally generated by the University of Utah under a grant provided by NOAA. They are are continuing to publish ZARR versions of HRRR data. For information about data in the s3://hrrrzarr/ please contact <a href="mailto:&#9</b>...</p>
          <p><a href="/noaa-hrrr-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/blaylockbk/HRRR_archive_download" class="" target="_none">HRRR-B Python package: download and read HRRR grib2 files</a> by Brian Blaylock</li></a>
          </li>
          <li>
            <a href="https://journals.ametsoc.org/view/journals/wefo/aop/WAF-D-22-0213.1/WAF-D-22-0213.1.xml" class="" target="_none">Using the U.S. Climate Reference Network to Identify Biases in Near- and Sub-Surface Meteorological Fields in the High-Resolution Rapid Refresh (HRRR) Weather Prediction Model</a> by Temple R. Lee, Ronald D. Leeper, Tim Wilson, Howard Diamond, Tilden P. Meyers, and David D. Turner</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1175/JTECH-D-21-0106.1" class="" target="_none">Using Cloud Computing to Analyze Model Output Archived in Zarr Format</a> by Taylor A. Gowan, John D. Horel, Alexander A. Jacques, and Adair Kovac</li></a>
          </li>
          <li>
            <a href="https://mesowest.utah.edu/html/hrrr/" class="" target="_none">The HRRR Zarr Archive Managed by MesoWest</a> by Taylor Gowan</li></a>
          </li>
        </ul>
        <p><a href="/noaa-hrrr-pds/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-pds-nsrdb" class="dataset">
          <h3><a href="/nrel-pds-nsrdb/">NREL National Solar Radiation Database</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">solar</span></p>
          <p>Released to the public as part of the Department of Energy&#39;s Open Energy Data Initiative,
the <a href="https://nsrdb.nrel.gov/">National Solar Radiation Database (NSRDB)</a> is
a serially complete collection of hourly and half-hourly values of the three
most common measurements of solar radiation – global horizontal, direct
normal, and diffuse horizontal irradiance — and meteorological data. These
data have been collected at a sufficient number of locations and temporal and
spatial scales to accurately represent regional solar radiation climates.</p>
          <p><a href="/nrel-pds-nsrdb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nsrdb.nrel.gov/nsrdb-viewer" class="" target="_none">NSRDB Viewer</a> by Manajit Sengupta, Yu Xe, Anthony Lopez, Aron Habte, Galen Maclaurin, James Shelby, Paul Edwards</li></a>
          </li>
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S136403211830087X" class="" target="_none">The National Solar Radiation Data Base (NSRDB)</a> by Manajit Sengupta, Yu Xe, Anthony Lopez, Aron Habte, Galen Maclaurin, James Shelby</li></a>
          </li>
          <li>
            <a href="https://www.sciencedirect.com/science/article/abs/pii/S0038092X22000044" class="" target="_none">Physics-guided machine learning for improved accuracy of the National Solar Radiation Database</a> by Grant Buster, Mike Bannister, Aron Habte, Dylan Hettinger, Galen Maclaurin, Michael Rossol, Manajit Sengupta, Yu Xie</li></a>
          </li>
          <li>
            <a href="https://github.com/NREL/hsds-examples" class="" target="_none">HSDS Examples</a> by Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer, Michael Rossol</li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-nsrdb/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="openalex" class="dataset">
          <h3><a href="/openalex/">OpenAlex dataset</a></h3>
          <p><span class="label label-info tag link-tag">graph</span><span class="label label-info tag link-tag">json</span><span class="label label-info tag link-tag">metadata</span><span class="label label-info tag link-tag">scholarly communication</span></p>
          <p>An open, comprehensive index of scolarly papers, citations, authors, institutions, and journals.</p>
          <p><a href="/openalex/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.openalex.org/download-snapshot" class="" target="_none">Download snapshot</a> by <a href="https://ourresearch.org" target="_none">OurResearch</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/aws-openalex-knowledge-graph" class="aws-link" target="_none">Data Analysis and Knowledge Graph with OpenAlex dataset</a> by Gabriel Bella Martini</li></a>
          </li>
          <li>
            <a href="https://twitter.com/ipoga/status/1521604494592782336" class="" target="_none">Getting citation data from OpenAlex by DOI (Jupyter notebook)</a> by <a href="https://twitter.com/ipoga" target="_none">Jens Peter Anderson</a></li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2205.01833" class="" target="_none">OpenAlex: A fully-open index of scholarly works, authors, venues, institutions, and concepts</a> by Jason Priem, Heather Piwowar, Richard Orr</li></a>
          </li>
        </ul>
        <p><a href="/openalex/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="czb-opencell" class="dataset">
          <h3><a href="/czb-opencell/">OpenCell on AWS</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">microscopy</span></p>
          <p>The OpenCell project is a proteome-scale effort to measure the localization and interactions of human proteins
using high-throughput genome engineering to endogenously tag thousands of proteins in the human proteome.
This dataset consists of the raw confocal fluorescence microscopy images for all tagged cell lines in the OpenCell library. 
These images can be interpreted both individually, to determine the localization of particular proteins of interest,
and in aggregate, by training machine learning models to classify or quantify subcellular localization patterns. </p>
          <p><a href="/czb-opencell/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://opencell.czbiohub.org" class="" target="_none">OpenCell web portal</a> by <a href="https://www.czbiohub.org/manuel-leonetti-intracellular-architecture" target="_none">OpenCell team</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2021.03.29.437595v1" class="" target="_none">Self-Supervised Deep-Learning Encodes High-Resolution Features of Protein Subcellular Localization</a> by <a href="https://www.royerlab.org" target="_none">Hirofumi Kobayashi, Keith C. Cheveralls, Manuel D. Leonetti, Loic A. Royer</a></li></a>
          </li>
          <li>
            <a href="https://github.com/royerlab/cytoself" class="" target="_none">cytoself (an unsupervised ML model to quantify localization patterns)</a> by <a href="https://www.royerlab.org" target="_none">Hirofumi Kobayashi, Keith C. Cheveralls, Manuel D. Leonetti, Loic A. Royer</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2021.03.29.437450v1" class="" target="_none">OpenCell: proteome-scale endogenous tagging enables the cartography of human cellular organization</a> by <a href="https://www.czbiohub.org/manuel-leonetti-intracellular-architecture" target="_none">Nathan H. Cho, Keith C. Cheveralls, Andreas-David Brunner, Kibeom Kim, André C. Michaelis, Preethi Raghavan, et al.</a></li></a>
          </li>
        </ul>
        <p><a href="/czb-opencell/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="rcm-ceos-ard" class="dataset">
          <h3><a href="/rcm-ceos-ard/">RCM CEOS Analysis Ready Data | Données prêtes à l&#x27;analyse du CEOS pour le MCR</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">analysis ready data</span><span class="label label-info tag link-tag">ceos</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>The <a href="https://www.asc-csa.gc.ca/eng/satellites/radarsat/">RADARSAT Constellation Mission (RCM)</a> is Canada&#39;s third generation of Earth observation satellites. Launched on June 12, 2019, the three identical satellites work together to bring solutions to key challenges for Canadians. As part of ongoing <a href="https://open.canada.ca/en/about-open-government">Open Government</a> efforts, NRCan produces a CEOS analysis ready data (ARD) of Canada landmass using a 30M Compact-Polarization standard coverage, every 12 days. RCM CEOS-ARD (POL) is the first ever polarimetric dataset <a href="https://ceos.org/ard/index.html#datasets">approved by the CEOS committee</a>. Previously, users were stuck ordering, downloading and processing RCM images (level 1) on their own, often with expensive software. This new dataset aims to remove these burdens with a new STAC catalog for discovery and direct download links.
<br/> <br/>
La mission de la Constellation RADARSAT (MCR) est la troisième génération de satellites d&#39;observation de la Terre du Canada. Lancés le 12 juin 2019, les trois satellites identiques travaillent ensemble pour apporter des solutions aux principaux défis des Canadiens. Dans le cadre des efforts continus pour un gouvernement ouvert, RNCan produit des données prêtes à l&#39;analyse CEOS (ARD) de la masse terrestre du Canada en utilisant une couverture standard de 30 m en polarisation compacte, tous les 12 jours. Les CEOS-ARD (POL) du MCR constituent le premier ensemble de données polarimétriques jamais <a href="https://ceos.org/ard/index.html#datasets">approuvé par le comité CEOS</a>. Auparavant, les utilisateurs étaient obligés de commander, de télécharger...</p>
          <p><a href="/rcm-ceos-ard/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ceos.org/ard/" class="" target="_none">CEOS Analysis Ready Data</a> by <a href="https://ceos.org/" target="_none">Committee on Earth Observation Satellites (CEOS)</a></li></a>
          </li>
          <li>
            <a href="https://ceos.org/ard/files/PFS/SAR/v1.1/CEOS-ARD_PFS_Synthetic_Aperture_Radar_v1.1.pdf" class="" target="_none">Synthetic Aperture Radar (CEOS-ARD SAR)</a> by Committee on Earth Observation Satellites (CEOS) for developing the CEOS ARD Standards. Specific acknowledgement to François Charbonneau (NRCan) for contributions to the standard development through CEOS committee membership as well as application to Canadian RADARSAT data.</li></a>
          </li>
          <li>
            <a href="https://researchdata.tuwien.ac.at/records/n2d1v-gqb91" class="" target="_none">Sentinel-1 Global Backscatter Model (S1GBM)</a> by <a href="https://researchdata.tuwien.ac.at/" target="_none">TU Wien Research Data</a></li></a>
          </li>
          <li>
            <a href="https://dataspace.copernicus.eu/explore-data/data-collections/copernicus-contributing-missions/collections-description/COP-DEM" class="" target="_none">Copernicus Global Digital Elevation Model</a> by <a href="https://www.esa.int/" target="_none">European Space Agency (ESA)</a></li></a>
          </li>
        </ul>
        <p><a href="/rcm-ceos-ard/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="refgenie" class="dataset">
          <h3><a href="/refgenie/">Refgenie reference genome assets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">single-cell transcriptomics</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Pre-built refgenie reference genome data assets used for aligning and analyzing DNA sequence data.</p>
          <p><a href="/refgenie/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://refgenie.databio.org/en/latest/tutorial/" class="" target="_none">Basic Refgenie tutorial</a> by <a href="http://www.databio.org" target="_none">Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://pepatac.databio.org" class="" target="_none">Pipeline for ATAC-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="http://peppro.databio.org" class="" target="_none">Pipeline for PRO-seq data analysis</a> by <a href="http://databio.org" target="_none">Jason Smith and Nathan Sheffield</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1093/gigascience/giz149" class="" target="_none">Refgenie: a reference genome resource manager</a> by <a href="http://www.databio.org" target="_none">Michał Stolarczyk, Vincent P Reuter, Jason P Smith, Neal E Magee, Nathan C Sheffield</a></li></a>
          </li>
        </ul>
        <p><a href="/refgenie/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="silo" class="dataset">
          <h3><a href="/silo/">SILO climate data on AWS</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p><a href="https://www.longpaddock.qld.gov.au/silo">SILO</a> is a database of Australian <a href="https://www.longpaddock.qld.gov.au/silo/about/climate-variables">climate data</a> from 1889 to the present. It provides continuous, daily time-step <a href="https://www.longpaddock.qld.gov.au/silo/about/data-products">data products</a> in ready-to-use <a href="https://www.longpaddock.qld.gov.au/silo/about/file-formats-and-samples">formats</a> for research and operational applications.
SIL...</p>
          <p><a href="/silo/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/github/RichardScottOZ/Adelaide-Temperatue/blob/main/Humidity-grids.ipynb" class="" target="_none">Using relative humidity grids with xarray from s3</a> by Richard Scott</li></a>
          </li>
          <li>
            <a href="https://www.longpaddock.qld.gov.au/silo/gridded-data/python" class="" target="_none">Python script to calculate a regional mean</a> by SILO</li></a>
          </li>
          <li>
            <a href="https://www.longpaddock.qld.gov.au/silo/gridded-data/nco" class="" target="_none">NetCDF Operators to calculate seasonal means</a> by SILO</li></a>
          </li>
          <li>
            <a href="https://www.longpaddock.qld.gov.au/silo/gridded-data/conversion" class="" target="_none">Convert NetCDF to ESRI ArcASCII or GeoTIFF</a> by SILO</li></a>
          </li>
        </ul>
        <p><a href="/silo/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="surftemp-sst" class="dataset">
          <h3><a href="/surftemp-sst/">Sea Surface Temperature Daily Analysis: European Space Agency Climate Change Initiative product version 2.1</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>Global daily-mean sea surface temperatures, presented on a 0.05° latitude-longitude grid, with gaps between available daily observations filled by statistical means, spanning late 1981 to recent time.  Suitable for large-scale oceanographic meteorological and climatological applications, such as evaluating or constraining environmental models or case-studies of marine heat wave events. Includes temperature uncertainty information and auxiliary information about land-sea fraction and sea-ice coverage. For reference and citation see: <a href="http://www.nature.com/articles/s41597-019-0236-x">www.nature.com/articles/s41597-019-0236-x</a>.</p>
          <p><a href="/surftemp-sst/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41597-019-0236-x" class="" target="_none">Satellite-based time-series of sea-surface temperature since 1981 for climate applications (2019).</a> by Merchant, C.J., Embury, O., Bulgin, C.E., Block, T., Corlett, G.K., Fiedler, E., Good, S.A., Mittaz, J., Rayner, N.A., Berry, D., Eastwood, S., Taylor, M., Tsushima, Y., Waterfall, A., Wilson, R. and Donlon, C.</li></a>
          </li>
          <li>
            <a href="https://www.mdpi.com/2072-4292/12/16/2554" class="" target="_none">Adjusting for desert-dust-related biases in a climate data record of sea surface temperature (2020).</a> by Merchant, C.J. and Embury, O.</li></a>
          </li>
          <li>
            <a href="https://surftemp.github.io/sst-data-tutorials#tutorial2" class="" target="_none">Working with surftemp-sst data - Tutorial 2 - Analysing Marine Heatwaves</a> by Niall McCarroll</li></a>
          </li>
          <li>
            <a href="https://surftemp.github.io/sst-data-tutorials#tutorial1" class="" target="_none">Working with surftemp-sst data - Tutorial 1 - Getting started</a> by Niall McCarroll</li></a>
          </li>
        </ul>
        <p><a href="/surftemp-sst/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel-1" class="dataset">
          <h3><a href="/sentinel-1/">Sentinel-1</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p><a href="https://sentinel.esa.int/web/sentinel/missions/sentinel-1">Sentinel-1</a> is a pair of European radar imaging (SAR) satellites launched in 2014 and 2016. Its 6 days revisit cycle and ability to observe through clouds makes it perfect for sea and land monitoring, emergency response due to environmental disasters, and economic applications. This dataset represents the global Sentinel-1 GRD archive, from beginning to the present, converted to <a href="https://www.cogeo.org/">cloud-optimized GeoTIFF format</a>.</p>
          <p><a href="/sentinel-1/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://www.sentinel-hub.com/apps/wms" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service by Sinergise</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
          <li>
            <a href="https://apps.sentinel-hub.com/sentinel-playground/?baseWmsUrl&#x3D;services.sentinel-hub.com&amp;instanceID&#x3D;a7c04225-b861-452d-ac89-7765352cd855&amp;source&#x3D;S2&amp;lat&#x3D;40.98676619345964&amp;lng&#x3D;28.89301300048828&amp;zoom&#x3D;13&amp;preset&#x3D;IW_VV&amp;layers&#x3D;B04,B03,B02&amp;maxcc&#x3D;20&amp;gain&#x3D;1.0&amp;gamma&#x3D;1.0&amp;time&#x3D;2015-01-01%7C2018-01-16&amp;atmFilter&#x3D;&amp;showDates&#x3D;false" class="" target="_none">Sentinel Playground</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
          <li>
            <a href="https://eos.com/landviewer/" class="" target="_none">EOS Land Viewer</a> by <a href="https://eos.com/" target="_none">Earth Observing System</a></li></a>
          </li>
          <li>
            <a href="http://apps.sentinel-hub.com/eo-browser/" class="" target="_none">EO Browser</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
        </ul>
        <p><a href="/sentinel-1/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel-s2-l2a-mosaic-120" class="dataset">
          <h3><a href="/sentinel-s2-l2a-mosaic-120/">Sentinel-2 L2A 120m Mosaic</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>Sentinel-2 L2A 120m mosaic is a derived product, which contains best pixel values for 10-daily periods, modelled by removing the cloudy pixels and then performing interpolation among remaining values. As there are some parts of the world, which have lengthy cloudy periods, clouds might be remaining in some parts. The actual modelling script is available <a href="https://sentinel-hub.github.io/custom-scripts/sentinel-2/interpolated_time_series/">here</a>.</p>
          <p><a href="/sentinel-s2-l2a-mosaic-120/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://medium.com/sentinel-hub/how-to-make-the-perfect-time-lapse-of-the-earth-351f214527f6" class="" target="_none">How to Make the Perfect Time-Lapse of the Earth</a> by <a href="https://medium.com/@lubej.matic" target="_none">Matic Lubej</a></li></a>
          </li>
          <li>
            <a href="https://apps.sentinel-hub.com/digital_twin_sandbox/" class="" target="_none">Digital Twin Sandbox</a> by <a href="https://www.sentinel-hub.com/" target="_none">Sentinel Hub</a></li></a>
          </li>
          <li>
            <a href="https://github.com/sentinel-hub/public-collections/blob/main/collections/sentinel-s2-l2a-mosaic-120.md" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service and Process API</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
          <li>
            <a href="https://medium.com/sentinel-hub/digital-twin-sandbox-sentinel-2-collection-available-to-everyone-20f3b5de846e" class="" target="_none">Digital Twin Sandbox Sentinel-2 collection available to everyone</a> by <a href="http://www.sinergise.com/" target="_none">Grega Milcinski &amp; Matic Lubej</a></li></a>
          </li>
        </ul>
        <p><a href="/sentinel-s2-l2a-mosaic-120/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel-3" class="dataset">
          <h3><a href="/sentinel-3/">Sentinel-3</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>This data set consists of observations from the Sentinel-3 satellite of the European Commission’s Copernicus Earth Observation Programme. Sentinel-3 is a polar orbiting satellite that completes 14 orbits of the Earth a day. It carries the Ocean and Land Colour Instrument (OLCI) for medium resolution marine and terrestrial optical measurements, the Sea and Land Surface Temperature Radiometer (SLSTR), the SAR Radar Altimeter (SRAL), the MicroWave Radiometer (MWR) and the Precise Orbit Determination (POD) instruments. The satellite was launched in 2016 and entered routine operational phase in 201...</p>
          <p><a href="/sentinel-3/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://step.esa.int/main/toolboxes/sentinel-3-toolbox/" class="" target="_none">Sentinel-3 Toolbox</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></li></a>
          </li>
          <li>
            <a href="https://sentinel.esa.int/web/sentinel/user-guides" class="" target="_none">Sentinel-3 Document Library</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></li></a>
          </li>
          <li>
            <a href="https://github.com/Sentinel-5P/data-on-s3/blob/master/notebooks/Sentinel3_Tutorial.ipynb" class="" target="_none">Accessing Sentinel-3 Data on S3 by MEEO</a> by <a href="http://www.meeo.it/" target="_none">Meteorological Environmental Earth Observation</a></li></a>
          </li>
          <li>
            <a href="https://meeo-s3.s3.amazonaws.com/index.html#/?t&#x3D;catalogs" class="" target="_none">Catalogue of data set</a> by <a href="https://www.meeo.it/" target="_none">Meteorological Environmental Earth Observation</a></li></a>
          </li>
        </ul>
        <p><a href="/sentinel-3/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel5p" class="dataset">
          <h3><a href="/sentinel5p/">Sentinel-5P Level 2</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>This data set consists of observations from the Sentinel-5 Precursor (Sentinel-5P) satellite of the European Commission’s Copernicus Earth Observation Programme. Sentinel-5P is a polar orbiting satellite that completes 14 orbits of the Earth a day. It carries the TROPOspheric Monitoring Instrument (TROPOMI) which is a spectrometer that senses ultraviolet (UV), visible (VIS), near (NIR) and short wave infrared (SWIR) to monitor ozone, methane, formaldehyde, aerosol, carbon monoxide, nitrogen dioxide and sulphur dioxide in the atmosphere. The satellite was launched in October 2017 and entered ro...</p>
          <p><a href="/sentinel5p/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/Sentinel-5P/data-on-s3/blob/master/notebooks/Sentinel5P_Tutorial.ipynb" class="" target="_none">Accessing Sentinel-5P Data on S3 by MEEO</a> by <a href="http://www.meeo.it/" target="_none">Meteorological Environmental Earth Observation</a></li></a>
          </li>
          <li>
            <a href="https://sentinel.esa.int/web/sentinel/user-guides/sentinel-5p-tropomi/document-library" class="" target="_none">Sentinel-5P TROPOMI Document Library</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></li></a>
          </li>
          <li>
            <a href="https://meeo-s5p.s3.amazonaws.com/index.html#/?t&#x3D;catalogs" class="" target="_none">Catalogue of data set</a> by <a href="https://www.meeo.it/" target="_none">Meteorological Environmental Earth Observation</a></li></a>
          </li>
          <li>
            <a href="https://atmospherictoolbox.org/" class="" target="_none">The Atmospheric Toolbox</a> by <a href="https://www.esa.int/" target="_none">European Space Agency</a></li></a>
          </li>
        </ul>
        <p><a href="/sentinel5p/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="sipecam" class="dataset">
          <h3><a href="/sipecam/">SiPeCaM (Sitios Permanentes de la Calibración y Monitoreo de la Biodiversidad)</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">multimedia</span><span class="label label-info tag link-tag">wildlife</span></p>
          <p>The SiPeCaM goal is to create a data source that allows to evaluate changes in the biodiversity state, considering key aspect of how does the ecosystem behaves.</p>
          <p><a href="/sipecam/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://monitoreo.conabio.gob.mx/sipecam_files/Proyecto_SiPeCaM_2020_Fase_I.pdf" class="" target="_none">Sitios Permanente de la Calibración y Monitoreo de la Biodiversidad</a> by Michael Schmidt et. al.</li></a>
          </li>
          <li>
            <a href="https://github.com/CONABIO/sipecam-open-data/blob/main/tutorials/notebooks/audios/01_alfresco_consultation.ipynb" class="" target="_none">Sample search query on november audio files for cumulus 92, using Alfresco.</a> by Carolina Acosta</li></a>
          </li>
          <li>
            <a href="https://github.com/CONABIO/sipecam-open-data/blob/main/tutorials/notebooks/images/01_alfresco_consultation.ipynb" class="" target="_none">Sample search query on all images files for cumulus 92, using Alfresco.</a> by Carolina Acosta</li></a>
          </li>
          <li>
            <a href="https://github.com/CONABIO/sipecam-open-data/blob/main/tutorials/notebooks/videos/01_alfresco_consultation.ipynb" class="" target="_none">Sample search query on november video files for cumulus 92, using Alfresco.</a> by Carolina Acosta</li></a>
          </li>
        </ul>
        <p><a href="/sipecam/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="speedtest-global-performance" class="dataset">
          <h3><a href="/speedtest-global-performance/">Speedtest by Ookla Global Fixed and Mobile Network Performance Maps</a></h3>
          <p><span class="label label-info tag link-tag">analytics</span><span class="label label-info tag link-tag">broadband</span><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">civic</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">government spending</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">network traffic</span><span class="label label-info tag link-tag">parquet</span><span class="label label-info tag link-tag">regulatory</span><span class="label label-info tag link-tag">telecommunications</span><span class="label label-info tag link-tag">tiles</span></p>
          <p>Global fixed broadband and mobile (cellular) network performance, allocated to zoom level 16 web mercator tiles (approximately 610.8 meters by 610.8 meters at the equator). Data is provided in both Shapefile format as well as Apache Parquet with geometries represented in Well Known Text (WKT) projected in EPSG:4326. Download speed, upload speed, and latency are collected via the Speedtest by Ookla applications for Android and iOS and averaged for each tile. Measurements are filtered to results containing GPS-quality location accuracy.</p>
          <p><a href="/speedtest-global-performance/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://towardsdatascience.com/bootstrapping-dask-on-1000-cores-with-aws-fargate-1e1b7fd1a32c" class="" target="_none">Bootstrapping Dask on 1000 cores with AWS Fargate</a> by Imri Paran</li></a>
          </li>
          <li>
            <a href="https://developmentseed.org/blog/2023-10-23-lonboard" class="" target="_none">Launching Lonboard - A Python library for extremely fast geospatial vector data visualization in Jupyter</a> by <a href="https://developmentseed.org/team/kyle-barron" target="_none">Kyle Barron</a></li></a>
          </li>
          <li>
            <a href="https://www.speedtest.net/insights/blog/best-ookla-open-data-projects-2021/" class="" target="_none">New Year, Great Data: The Best Ookla Open Data Projects We’ve Seen So Far</a> by Katie Jolly</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/how-to-deliver-performant-gis-desktop-applications-amazon-appstream-2-0/" class="aws-link" target="_none">How to deliver performant GIS desktop applications with Amazon AppStream 2.0</a> by Ethan Fahy and Spencer DeBrosse</li></a>
          </li>
        </ul>
        <p><a href="/speedtest-global-performance/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="sevir" class="dataset">
          <h3><a href="/sevir/">Storm EVent ImageRy (SEVIR)</a></h3>
          <p><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Collection of spatially and temporally aligned GOES-16 ABI satellite imagery, NEXRAD radar mosaics, and GOES-16 GLM lightning detections.</p>
          <p><a href="/sevir/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.jupyter.org/github/MIT-AI-Accelerator/eie-sevir/blob/master/examples/Generators.ipynb" class="" target="_none">Using Generators for SEVIR data</a> by <a href="https://github.com/markveillette" target="_none">Mark Veillette</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.jupyter.org/github/MIT-AI-Accelerator/eie-sevir/blob/master/examples/SEVIR_Tutorial.ipynb" class="" target="_none">Introduction to SEVIR</a> by <a href="https://github.com/markveillette" target="_none">Mark Veillette</a></li></a>
          </li>
          <li>
            <a href="https://github.com/MIT-AI-Accelerator/eie-sevir" class="" target="_none">sevir -- python utilities for working with SEVIR dataset</a> by <a href="https://github.com/markveillette" target="_none">Mark Veillette</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.3390/rs14010024" class="" target="_none">Towards a More Realistic and Detailed Deep-Learning-Based Radar Echo Extrapolation Method</a> by Yuan Hu, Lei Chen, Zhibin Wang, Xiang Pan and Hao Li</li></a>
          </li>
        </ul>
        <p><a href="/sevir/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="synthea-omop" class="dataset">
          <h3><a href="/synthea-omop/">Synthea synthetic patient generator data in OMOP Common Data Model</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">us</span></p>
          <p>The Synthea generated data is provided here as a 1,000 person (1k), 100,000 person (100k), and 2,800,000 persom (2.8m) data sets in the <a href="https://www.ohdsi.org/data-standardization/">OMOP Common Data Model</a> format.  SyntheaTM is a synthetic patient generator that models the medical history of synthetic patients. Our mission is to output high-quality synthetic, realistic but not real, patient data and associated health records covering every aspect of healthcare. The resulting data is free from cost, privacy, and security restrictions. It can be used without restriction for a variety of secondary uses in academia, research, industry, and gov...</p>
          <p><a href="/synthea-omop/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/OHDSI/OHDSIonAWS" class="" target="_none">OHDSIonAWS</a> by James Wiggins</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/creating-data-science-environments-on-aws-for-health-analysis-using-ohdsi/" class="aws-link" target="_none">Create data science environments on AWS for health analysis using OHDSI</a> by James Wiggins</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/industries/predict-patient-health-outcomes-using-ohdsi-and-machine-learning-on-aws/" class="aws-link" target="_none">Predict patient health outcomes using OHDSI and machine learning on AWS</a> by James Wiggins</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/machine-learning/map-clinical-notes-to-the-omop-common-data-model-and-healthcare-ontologies-using-amazon-comprehend-medical/" class="aws-link" target="_none">Map clinical notes to the OMOP Common Data Model and healthcare ontologies using Amazon Comprehend Medical</a> by James Wiggins</li></a>
          </li>
        </ul>
        <p><a href="/synthea-omop/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ukbb-ld" class="dataset">
          <h3><a href="/ukbb-ld/">UK Biobank Linkage Disequilibrium Matrices</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genome wide association study</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population genetics</span></p>
          <p>Linkage disequilibrium (LD) matrices of UK Biobank participants of a British ancestry, based on imputed genotypes.</p>
          <p><a href="/ukbb-ld/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41588-020-00735-5" class="" target="_none">Functionally informed fine-mapping and polygenic localization of complex trait heritability</a> by Weissbrod et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/omerwe/polyfun" class="" target="_none">PolyFun and PolyPred software</a> by Omer Weissbrod</li></a>
          </li>
          <li>
            <a href="https://github.com/omerwe/polyfun/wiki" class="" target="_none">PolyFun Wiki</a> by Omer Weissbrod</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41588-022-01036-9" class="" target="_none">Leveraging fine-mapping and multipopulation training data to improve cross-population polygenic risk scores</a> by Weissbrod et al.</li></a>
          </li>
        </ul>
        <p><a href="/ukbb-ld/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="broad-pan-ukb" class="dataset">
          <h3><a href="/broad-pan-ukb/">UK Biobank Pan-Ancestry Summary Statistics</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genome wide association study</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population genetics</span></p>
          <p>A multi-ancestry analysis of 7,221 phenotypes using a generalized mixed model association testing framework, spanning 16,119 genome-wide association studies. We provide standard meta-analysis across all populations and with a leave-one-population-out approach for each trait. The data are provided in tsv format (per phenotype) and Hail MatrixTable (all phenotypes and variants). Metadata is provided in phenotype and variant manifests.</p>
          <p><a href="/broad-pan-ukb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/quickstart/architecture/hail/" class="aws-link" target="_none">Hail on AWS Quick Start</a> by Amazon Web Services and PrivoIT</li></a>
          </li>
          <li>
            <a href="https://pan.ukbb.broadinstitute.org/" class="" target="_none">Pan-ancestry genetic analysis of the UK Biobank</a> by Pan UKBB Team</li></a>
          </li>
          <li>
            <a href="https://hail.is/docs/0.2/tutorials-landing.html" class="" target="_none">Hail Tutorials</a> by <a href="https://hail.is/" target="_none">Hail Team</a></li></a>
          </li>
          <li>
            <a href="https://hail.is" class="" target="_none">Hail</a> by <a href="https://hail.is/" target="_none">Hail Team</a></li></a>
          </li>
        </ul>
        <p><a href="/broad-pan-ukb/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="vadcr-crmp-aws" class="dataset">
          <h3><a href="/vadcr-crmp-aws/">Virginia Coastal Resilience Master Plan, Phase 1 - December 2021</a></h3>
          <p><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">floods</span></p>
          <p>The Virginia Coastal Resilience Master Plan builds on the 2020 Virginia Coastal Resilience Master Planning Framework, which outlined the goals and principles of the Commonwealth’s statewide coastal resilience strategy. Recognizing the urgent challenge flooding already poses, the Commonwealth developed Phase One of the Master Plan on an accelerated timeline and focused this first assessment on the impacts of tidal and storm surge coastal flooding on coastal Virginia.  The Master Plan leveraged the combined efforts of more than two thousand stakeholders, subject matter experts, and government personnel. We centered the development of this plan around three core components:
<br/>
<br/>
A Technical Study compiled essential data, research, processes, products, and resilience efforts in the Coastal Resilience Database, which forms much of basis of this plan and the Coastal Resilience Web Explorer;
<br/>
<br/>
A Technical Advisory Committee supported coordination across key stakeholders and ensured the incorporation of the best available subject matter knowledge, data, and methods into this plan; and
<br/>
<br/>
Stakeholder Engagement captured diverse resilience perspectives from residents, local and regional officials, and other stakeholders across Virginia’s coastal communities to drive regionally specific resilience priorities.Data products used and generated for the Virginia Coastal Resilience.
<br/>
<br/>
This dataset represents the data that was developed for the technical study.  Appendix F - Data Product List provides a list of available data.  Other Appendix documents provide the inpu</br/>...</p>
          <p><a href="/vadcr-crmp-aws/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://vacrmp.dewberryanalytics.com/arcgis/rest/services" class="" target="_none">ArcGIS REST Services Directory</a> by <a href="https://www.dcr.virginia.gov/" target="_none">Virginia Department of Conservation and Recreation</a></li></a>
          </li>
          <li>
            <a href="https://www.dcr.virginia.gov/crmp/ResilienceExplorer" class="" target="_none">Virginia Coastal Resilience Web Explorer</a> by <a href="https://www.dcr.virginia.gov/" target="_none">Virginia Department of Conservation and Recreation</a></li></a>
          </li>
          <li>
            <a href="https://www.dcr.virginia.gov/crmp/document/Appendix-F-Data-Product-List.pdf" class="" target="_none">Appendix F Data Product List</a> by <a href="https://www.dcr.virginia.gov/" target="_none">Virginia Department of Conservation and Recreation</a></li></a>
          </li>
          <li>
            <a href="https://www.dcr.virginia.gov/crmp/plan" class="" target="_none">Virginia Coastal Resilience Master Plan, Phase One December 2021</a> by <a href="https://www.dcr.virginia.gov/" target="_none">Virginia Department of Conservation and Recreation</a></li></a>
          </li>
        </ul>
        <p><a href="/vadcr-crmp-aws/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ycb-benchmarks" class="dataset">
          <h3><a href="/ycb-benchmarks/">Yale-CMU-Berkeley (YCB) Object and Model Set</a></h3>
          <p><span class="label label-info tag link-tag">robotics</span></p>
          <p>This project primarily aims to facilitate performance benchmarking in robotics research. The dataset provides mesh models, RGB, RGB-D and point cloud images of over 80 objects. The physical objects are also available via the <a href="http://www.ycbbenchmarks.com/">YCB benchmarking project</a>. The data are collected by two state of the art systems: UC Berkley&#39;s scanning rig and the Google scanner. The UC Berkley&#39;s scanning rig data provide meshes generated with Poisson reconstruction, meshes generated with volumetric range image integration, textured versions of both meshes, Kinbody files for using the meshes with OpenRAVE, 600 ...</p>
          <p><a href="/ycb-benchmarks/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/8460950" class="" target="_none">Label Fusion: A Pipeline for Generating Ground Truth Labels for Real RGBD Data of Cluttered Scenes</a> by <a href="https://groups.csail.mit.edu/locomotion/russt.html" target="_none">Pat Marion, Peter R. Florence, Lucas Manuelli, Russ Tedrake</a></li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/8304743" class="" target="_none">The Closure Signature: A Functional Approach to Model Underactuated Compliant Robotic Hands</a> by <a href="http://www.dii.unisi.it/~domenico/" target="_none">Maria Pozzi, Gionata Salvietti, João Bimbo, Monica Malvezzi, Domenico Prattichizzo</a></li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/document/7254318" class="" target="_none">Benchmarking in Manipulation Research: Using the Yale-CMU-Berkeley Object and Model Set</a> by <a href="https://www.wpi.edu/people/faculty/bcalli" target="_none">Berk Calli, Aaron Walsman, Arjun Singh, Siddhartha Srinivasa, Pieter Abbeel, Aaron M Dollar</a></li></a>
          </li>
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/7989594" class="" target="_none">Pre-touch sensing for sequential manipulation</a> by <a href="https://sensor.cs.washington.edu/jrs.html" target="_none">Boling Yang, Patrick Lancaster, Joshua R. Smith</a></li></a>
          </li>
        </ul>
        <p><a href="/ycb-benchmarks/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="isdasoil" class="dataset">
          <h3><a href="/isdasoil/">iSDAsoil</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">analytics</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">conservation</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>iSDAsoil is a resource containing soil property predictions for the entire African continent, generated using machine learning. Maps for over 20 different soil properties have been created at 2 different depths (0-20 and 20-50cm).  Soil property predictions were made using machine learning coupled with remote sensing data and a training set of over 100,000 analyzed soil samples. Included in this dataset are images of predicted soil properties, model error and satellite covariates used in the mapping process.</p>
          <p><a href="/isdasoil/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://observablehq.com/d/1c48938b8a4f5da3" class="" target="_none">iSDAsoil liming demo app on Observable</a> by <a href="https://jamiecollinson.com/" target="_none">Jamie Collinson</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41598-021-85639-y" class="" target="_none">African soil properties and nutrients mapped at 30 m spatial resolution using two-scale ensemble machine learning</a> by Tomislav Hengl, Matthew A. E. Miller, Josip Križan, Keith D. Shepherd, Andrew Sila, Milan Kilibarda, Ognjen Antonijević, Luka Glušica, Achim Dobermann, Stephan M. Haefele, Steve P. McGrath, Gifty E. Acquah, Jamie Collinson, Leandro Parente, Mohammadreza Sheykhmousa, Kazuki Saito, Jean-Martial Johnson, Jordan Chamberlin, Francis B. T. Silatsa, Martin Yemefack, John Wendt, Robert A. MacMillan, Ichsani Wheeler &amp; Jonathan Crouch</li></a>
          </li>
          <li>
            <a href="https://www.isda-africa.com/isdasoil/" class="" target="_none">iSDAsoil homepage - view soil property maps online</a> by iSDA</li></a>
          </li>
          <li>
            <a href="https://github.com/iSDA-Africa/isdasoil-tutorial" class="" target="_none">iSDAsoil Python tutorial</a> by <a href="https://github.com/mmiller8878" target="_none">Matt Miller</a></li></a>
          </li>
        </ul>
        <p><a href="/isdasoil/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="real-changesets" class="dataset">
          <h3><a href="/real-changesets/">real-changesets</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">osm</span></p>
          <p>The real-changesets is an augmented representation of OpenStreetMap changesets in JSON format. It contains the current and the previous version of each feature in a changeset.
It&#39;s primary used by OSMCha, the main OpenStreetMap validation tool, to have a visualization of the changeset and provide to the user the understanding of what was changed on the map.
The real-changesets are created by combining the changeset metadata and the augmented diff generated by overpass.</p>
          <p><a href="/real-changesets/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://osmcha.org" class="" target="_none">OSMCha</a> by <a href="https://github.com/willemarcel/" target="_none">Wille Marcel, Mapbox et al.</a></li></a>
          </li>
          <li>
            <a href="https://osmlab.github.io/changeset-map/" class="" target="_none">changeset-map</a> by <a href="https://github.com/osmlab/" target="_none">OSM Lab</a></li></a>
          </li>
          <li>
            <a href="https://www.openstreetmap.org/user/geohacker/diary/40846" class="" target="_none">Preparing accurate history and caching changesets</a> by <a href="https://github.com/geohacker/" target="_none">Sajjad Anwar</a></li></a>
          </li>
          <li>
            <a href="https://www.npmjs.com/package/real-changesets-parser" class="" target="_none">real-changesets-parser</a> by <a href="https://www.mapbox.com/" target="_none">Mapbox</a></li></a>
          </li>
        </ul>
        <p><a href="/real-changesets/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="asf-event-data" class="dataset">
          <h3><a href="/asf-event-data/">ASF SAR Data Products for Disaster Events</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>synthetic Aperture Radar (SAR) data is a powerful tool for monitoring and assessing disaster events and can provide valuable insights for researchers, scientists, and emergency response teams. 
The Alaska Satellite Facility (ASF) curates this collection of (primarily) SAR and SAR-derived satellite data products from a variety of data sources for disaster events.</p>
          <p><a href="/asf-event-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://search.asf.alaska.edu/#/?searchType&#x3D;Event%20Search" class="" target="_none">ASF Event Search</a> by <a href="https://asf.alaska.edu/" target="_none">Alaska Satellite Facility</a></li></a>
          </li>
          <li>
            <a href="https://docs.asf.alaska.edu/datasets/events_about/" class="" target="_none">SARVIEWS Events Collection</a> by <a href="https://asf.alaska.edu/" target="_none">Alaska Satellite Facility</a></li></a>
          </li>
          <li>
            <a href="https://docs.asf.alaska.edu/vertex/events/" class="" target="_none">Event Search Manual</a> by <a href="https://asf.alaska.edu/" target="_none">Alaska Satellite Facility</a></li></a>
          </li>
        </ul>
        <p><a href="/asf-event-data/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="allen-ivy-glioblastoma-atlas" class="dataset">
          <h3><a href="/allen-ivy-glioblastoma-atlas/">Allen Ivy Glioblastoma Atlas</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">gene expression</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">glioblastoma</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">neurobiology</span></p>
          <p>This dataset consists of images of glioblastoma human brain tumor tissue sections that have been probed for expression of particular genes believed to play a role in development of the cancer.  Each tissue section is adjacent to another section that was stained with a reagent useful for identifying histological features of the tumor.  Each of these types of images has been completely annotated for tumor features by a machine learning process trained by expert medical doctors.</p>
          <p><a href="/allen-ivy-glioblastoma-atlas/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://science.sciencemag.org/content/360/6389/660" class="" target="_none">An anatomic transcriptional atlas of human glioblastoma</a> by <a href="www.alleninstitute.org" target="_none">Ralph Puchalski, et al.</a></li></a>
          </li>
          <li>
            <a href="https://glioblastoma.alleninstitute.org" class="" target="_none">Ivy Glioblastoma Atlas Project</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
          <li>
            <a href="https://github.com/AllenInstitute/open_dataset_tools/blob/main/example_notebooks/Accessing_Ivy_Glioblastoma_Atlas_Project_Data.ipynb" class="" target="_none">Accessing Ivy Glioblastoma Atlas Project data</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-ivy-glioblastoma-atlas/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="allen-mouse-brain-atlas" class="dataset">
          <h3><a href="/allen-mouse-brain-atlas/">Allen Mouse Brain Atlas</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">gene expression</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Allen Mouse Brain Atlas is a genome-scale collection of cellular resolution gene expression profiles using in situ hybridization (ISH). Highly methodical data production methods and comprehensive anatomical coverage via dense, uniformly spaced sampling facilitate data consistency and comparability across &gt;20,000 genes. The use of an inbred mouse strain with minimal animal-to-animal variance allows one to treat the brain essentially as a complex but highly reproducible three-dimensional tissue array. The entire Allen Mouse Brain Atlas dataset and associated tools are available through an...</p>
          <p><a href="/allen-mouse-brain-atlas/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://mouse.brain-map.org" class="" target="_none">Allen Mouse Brain Atlas</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
          <li>
            <a href="http://www.nature.com/articles/nature05453" class="" target="_none">Genome-wide atlas of gene expression in the adult mouse brain</a> by <a href="www.alleninstitute.org" target="_none">Ed Lein, et al.</a></li></a>
          </li>
          <li>
            <a href="https://github.com/AllenInstitute/open_dataset_tools/blob/main/example_notebooks/Accessing_Allen_Mouse_Brain_Atlas_Data.ipynb" class="" target="_none">Accessing Allen Mouse Brain Atlas data</a> by <a href="www.alleninstitute.org" target="_none">Allen Institute for Brain Science</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-mouse-brain-atlas/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="beataml" class="dataset">
          <h3><a href="/beataml/">Beat Acute Myeloid Leukemia (AML) 1.0</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span></p>
          <p>Beat AML 1.0 is a collaborative research program involving 11 academic medical centers who worked
collectively to better understand  drugs and drug combinations that should be prioritized for
further development within clinical and/or molecular subsets of acute myeloid leukemia (AML)
patients. Beat AML 1.0 provides the largest-to-date dataset on primary acute myeloid leukemia
samples offering genomic, clinical, and drug response.This dataset contains open Clinical Supplement and RNA-Seq Gene Expression Quantification data.This dataset also contains controlled Whole Exome Sequencing (WXS) and R...</p>
          <p><a href="/beataml/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30651561" class="" target="_none">Clinical resistance to crenolanib in acute myeloid leukemia due to diverse molecular
mechanisms
</a> by Zhang H, Savage S, Schultz AR, Bottomly D, White L, Segerdell E, et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30333627" class="" target="_none">Functional Genomic Landscape of Acute Myeloid Leukemia</a> by Jeffrey W. Tyner, Cristina E. Tognon, Dan Bottomly et al.</li></a>
          </li>
        </ul>
        <p><a href="/beataml/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="blended-tropomi-gosat-methane" class="dataset">
          <h3><a href="/blended-tropomi-gosat-methane/">Blended TROPOMI+GOSAT Satellite Data Product for Atmospheric Methane</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>A dataset of satellite retrievals of atmospheric methane that extends from 30 April 2018 to present.</p>
          <p><a href="/blended-tropomi-gosat-methane/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.5194/amt-16-3787-2023" class="" target="_none">A blended TROPOMI+GOSAT satellite data product for atmospheric methane using machine learning to correct retrieval biases</a> by N. Balasus, D.J. Jacob, A. Lorente, J.D. Maasakkers, R.J. Parker, H. Boesch, Z. Chen, M.M. Kelp, H. Nesser, D.J. Varon</li></a>
          </li>
          <li>
            <a href="https://github.com/nicholasbalasus/write_blended_files/blob/main/resources/aws_tutorial_1.ipynb" class="" target="_none">Plotting one month of blended TROPOMI+GOSAT methane retrievals</a> by <a href="https://nicholasbalasus.github.io" target="_none">Nicholas Balasus</a></li></a>
          </li>
          <li>
            <a href="https://github.com/nicholasbalasus/write_blended_files/blob/main/resources/aws_tutorial_2.ipynb" class="" target="_none">Downloading the full data record from AWS</a> by <a href="https://nicholasbalasus.github.io" target="_none">Nicholas Balasus</a></li></a>
          </li>
        </ul>
        <p><a href="/blended-tropomi-gosat-methane/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="broad-references" class="dataset">
          <h3><a href="/broad-references/">Broad Genome References</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Broad maintained human genome reference builds hg19/hg38 and decoy references.</p>
          <p><a href="/broad-references/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/storage/using-amazon-fsx-for-lustre-for-genomics-workflows-on-aws/" class="aws-link" target="_none">Using Amazon FSx for Lustre for Genomics Workflows on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
          <li>
            <a href="https://www.cell.com/cell-reports-methods/pdf/S2667-2375(21)00165-X.pdf" class="" target="_none">Advancing NGS quality control to enable measurement of actionable mutations in circulating tumor DNA</a> by Willey J. C., Morrison T. B., Austermiller B., Crawford E. E., et al (2021)</li></a>
          </li>
        </ul>
        <p><a href="/broad-references/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="cobra" class="dataset">
          <h3><a href="/cobra/">COBRA</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">computational pathology</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">histopathology</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>This page describes the COBRA (Classification Of Basal cell carcinoma, Risky skin cancers and Abnormalities) skin pathology dataset, which comprises over 7000 histopathology whole-slide-images related to the diagnosis of basal cell carcinoma skin cancer, the most commonly diagnosed cancer. The dataset includes biopsies and excisions and is divided into four groups. The first group contains about 2,500 BCC biopsies with subtype labels, while the second group includes 2,500 non-BCC biopsies with different types of skin dysplasia. The third group has 1,000 labelled risky cancer biopsies, includin...</p>
          <p><a href="/cobra/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/daangeijs/cobra" class="" target="_none">COBRA Tools &amp; Labels</a> by <a href="https://www.diagnijmegen.nl/" target="_none">Diagnostic Image Analysis Group, Radboudumc, Nijmegen</a></li></a>
          </li>
          <li>
            <a href="https://github.com/DIAGNijmegen/pathology-whole-slide-data" class="" target="_none">Whole slide data - python package for working with wholeslide images and annotations files</a> by <a href="https://www.diagnijmegen.nl/" target="_none">Diagnostic Image Analysis Group, Radboudumc, Nijmegen</a></li></a>
          </li>
          <li>
            <a href="https://computationalpathologygroup.github.io/ASAP/" class="" target="_none">ASAP Viewer</a> by <a href="https://www.diagnijmegen.nl/people/geert-litjens/" target="_none">Geert Litjens</a></li></a>
          </li>
        </ul>
        <p><a href="/cobra/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="talend-covid19" class="dataset">
          <h3><a href="/talend-covid19/">COVID-19 Harmonized Data</a></h3>
          <p><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>A harmonized collection of the core data pertaining to COVID-19 reported cases by geography, in a format prepared for analysis</p>
          <p><a href="/talend-covid19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.talend.com/blog/2020/04/05/talend-joins-fight-against-covid-19-unlocking-data-for-health-researchers/" class="" target="_none">How Talend is joining the fight against COVID-19</a> by <a href="https://www.linkedin.com/in/thomasbennettw/" target="_none">Thomas Bennett</a></li></a>
          </li>
          <li>
            <a href="https://www.stitchdata.com/integrations/covid-19/" class="" target="_none">Stitch COVID-19 Integration</a> by <a href="https://github.com/jeffhuth-bytecode" target="_none">Jeff Huth</a></li></a>
          </li>
          <li>
            <a href="https://github.com/singer-io/tap-covid-19" class="" target="_none">Tap COVID-19 Python</a> by <a href="https://github.com/jeffhuth-bytecode" target="_none">Jeff Huth</a></li></a>
          </li>
        </ul>
        <p><a href="/talend-covid19/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="janelia-cosem" class="dataset">
          <h3><a href="/janelia-cosem/">Cell Organelle Segmentation in Electron Microscopy (COSEM) on AWS</a></h3>
          <p><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">electron microscopy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">organelle</span></p>
          <p>High resolution images of subcellular structures.</p>
          <p><a href="/janelia-cosem/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://science.sciencemag.org/content/367/6475/eaaz5357" class="" target="_none">Correlative three-dimensional super-resolution and block-face electron microscopy of whole vitreously frozen cells.</a> by David P. Hoffman, Gleb Shtengel, C. Shan Xu, Kirby R. Campbell, Melanie Freeman, Lei Wang, Daniel E. Milkie, H. Amalia Pasolli, Nirmala Iyer, John A. Bogovic, Daniel R. Stabley, Abbas Shirinifard, Song Pang, David Peale, Kathy Schaefer, Wim Pomp, Chi-Lun Chang, Jennifer Lippincott-Schwartz, Tom Kirchhausen1, David J. Solecki, Eric Betzig, Harald F. Hess</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41586-021-03977-3" class="" target="_none">Whole-cell organelle segmentation in volume electron microscopy</a> by Lisa Heinrich, Davis Bennett, David Ackerman, Woohyun Park, Jon Bogovic, Nils Eckstein, et al.</li></a>
          </li>
          <li>
            <a href="https://elifesciences.org/articles/25916" class="" target="_none">Enhanced FIB-SEM systems for large-volume 3D imaging</a> by C. Shan Xu, Kenneth J. Hayworth, Zhiyuan Lu, Patricia Grob, Ahmed M. Hassan, José G. García-Cerdán, Krishna K. Niyogi, Eva Nogales, Richard J. Weinberg, Harald F. Hess.</li></a>
          </li>
        </ul>
        <p><a href="/janelia-cosem/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="citrus-farm" class="dataset">
          <h3><a href="/citrus-farm/">CitrusFarm Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">IMU</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">localization</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">robotics</span></p>
          <p>CitrusFarm is a multimodal agricultural robotics dataset that provides both multispectral images and navigational sensor data for localization, mapping and crop monitoring tasks. <ol>
<li>It was collected by a wheeled mobile robot in the Agricultural Experimental Station at the University of California Riverside in the summer of 2023. </li>
<li>It offers a total of nine sensing modalities, including stereo RGB, depth, monochrome, near-infrared and thermal images, as well as wheel odometry, LiDAR, IMU and GPS-RTK data. </li>
<li>It comprises seven sequences collected from three citrus tree fields, featuring various tree species at different growth stages, distinctive planting patterns, as well as varying daylight conditions. </li>
<li>It spans a total operation time of 1.7 hours, covers a total distance of 7.5 km, and consti...</p>
          <p><a href="/citrus-farm/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ucr-robotics.github.io/Citrus-Farm-Dataset/tools.html" class="" target="_none">Python scripts used in the data collection and post-processing</a> by Hanzhe Teng et al.</li></a>
          </li>
          <li>
            <a href="https://link.springer.com/chapter/10.1007/978-3-031-47969-4_44" class="" target="_none">Multimodal Dataset for Localization, Mapping and Crop Monitoring in Citrus Tree Farms</a> by Hanzhe Teng, Yipeng Wang, Xiaoao Song and Konstantinos Karydis</li></a>
          </li>
          <li>
            <a href="https://ucr-robotics.github.io/Citrus-Farm-Dataset/download.html" class="" target="_none">Python script to download this dataset</a> by Hanzhe Teng et al.</li></a>
          </li>
        </ul>
        <p><a href="/citrus-farm/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="ctsp-dlbcl" class="dataset">
          <h3><a href="/ctsp-dlbcl/">Clinical Trial Sequencing Project - Diffuse Large B-Cell Lymphoma</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The goal of the project is to identify recurrent genetic alterations (mutations, deletions,
amplifications, rearrangements) and/or gene expression signatures. National Cancer Institute (NCI)
utilized whole genome sequencing and/or whole exome sequencing in conjunction with transcriptome
sequencing. The samples were processed and submitted for genomic characterization using pipelines
and procedures established within The Cancer Genome Analysis (TCGA) project.</p>
          <p><a href="/ctsp-dlbcl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CTSP-DLBCL1" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29925955[PMID]" class="" target="_none">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</a> by Phelan JD, Young RM, Webster DE, Roulland S, Wright GW, Kasbekar M, Shaffer AL 3rd,
Ceribelli M, Wang JQ, Schmitz R, Nakagawa M, Bachy E, Huang DW, Ji Y, Chen L, Yang Y, Zhao
H, Yu X, Xu W, Palisoc MM, Valadez RR, Davies-Hill T, Wilson WH, Chan WC, Jaffe ES, Gascoyne
RD, Campo E, Rosenwald A, Ott G, Delabie J, Rimsza LM, Rodriguez FJ, Estephan F, Holdhoff M,
Kruhlak MJ, Hewitt SM, Thomas CJ, Pittaluga S, Oellerich T, Staudt LM
</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., Calvin A.  Johnson,
Ph.D., James D. Phelan, Ph.D., James Q. Wang, Ph.D., Sandrine Roulland, Ph.D., Monica
Kasbekar, Ph.D., Ryan M. Young, Ph.D., Arthur L. Shaffer, Ph.D., Daniel J. Hodson, M.D.,
Ph.D., Wenming Xiao, Ph.D., et al.
</li></a>
          </li>
        </ul>
        <p><a href="/ctsp-dlbcl/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="gdr-data-lake" class="dataset">
          <h3><a href="/gdr-data-lake/">Department of Energy’s Geothermal Data Repository (GDR) Data Lake</a></h3>
          <p><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">geothermal</span></p>
          <p>Data released from projects funded by the Department of Energy&#39;s Geothermal 
Technologies Office (DOE GTO) that are too large or complex to be conveniently 
accessed by traditional means. The GDR data lake aims to improve and automate 
access of high-value geothermal data sets, making data actionable and discoverable 
by researchers and industry to accelerate analysis and advance innovation. 
This data lake is a sister-data lake to the Department of Energy’s Open Energy Data 
Initiative (OEDI) Data Lake.</p>
          <p><a href="/gdr-data-lake/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/openEDI/documentation/blob/master/PoroTomo/PoroTomo_Distributed_Acoustic_Sensing_(DAS)_Data_SEGY.ipynb" class="" target="_none">Dark Fiber Jupyter Notebook Tutorial</a> by <a href="https://orcid.org/0000-0001-7913-7189" target="_none">A. Nayak, A. Lowney</a></li></a>
          </li>
          <li>
            <a href="https://www.nrel.gov/docs/fy22osti/82837.pdf" class="" target="_none">The Geothermal Data Repository: Ten Years of Supporting the Geothermal Industry with Open Access to Geothermal Data</a> by J. Weers, A. Anderson, N. Taverna</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1785/0220220072" class="" target="_none">The Imperial Valley Dark Fiber Project: Toward Seismic Studies Using DAS and Telecom Infrastructure for Geothermal Applications</a> by J. Ajo-Franklin, V. Rodrigues Tribaldos, A. Nayak, et al.</li></a>
          </li>
        </ul>
        <p><a href="/gdr-data-lake/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="dandiarchive" class="dataset">
          <h3><a href="/dandiarchive/">Distributed Archives for Neurophysiology Data Integration (DANDI)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>DANDI is a public archive of neurophysiology datasets, including raw and processed data,  and associated software containers. Datasets are shared according to a Creative Commons  CC0 or CC-BY licenses. The data archive provides a broad range of cellular neurophysiology data.  This includes electrode and optical recordings, and associated imaging data using a  set of community standards: <a href="https://www.nwb.org/nwb-neurophysiology/">NWB:N - NWB:Neurophysiology</a>, <a href="https://bids.neuroimaging.io/">BIDS - Brain Imaging Data Structure</a>, and  <a href="htt...</p>
          <p><a href="/dandiarchive/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://hub.dandiarchive.org/" class="" target="_none">DANDI JupyterHub Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
          <li>
            <a href="https://github.com/dandi/dandi-cli" class="" target="_none">DANDI Shell Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
          <li>
            <a href="https://dandiarchive.org/" class="" target="_none">DANDI Web Interface</a> by <a href="https://dandiarchive.org/" target="_none">DANDI Project</a></li></a>
          </li>
        </ul>
        <p><a href="/dandiarchive/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="euro-cordex" class="dataset">
          <h3><a href="/euro-cordex/">EURO-CORDEX - European component of the Coordinated Regional Downscaling Experiment</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>The EURO-CORDEX dataset contains regional climate model data for Europe, for use in impacts, decision-making, and climate science. 
Currently, the bucket contains monthly datasets of 2m air temperature downscaled from CMIP5 global model datasets using different
regional climate models.</p>
          <p><a href="/euro-cordex/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.5281/zenodo.8059419" class="" target="_none">Beyond ESGF - Regional climate model datasets in the cloud on AWS S3</a> by Lars Buntemeyer</li></a>
          </li>
          <li>
            <a href="https://wcrp-cordex.github.io/cordex-tutorials/aws-access.html" class="" target="_none">Jupyter Book</a> by Lars Buntemeyer</li></a>
          </li>
          <li>
            <a href="https://euro-cordex.s3.eu-central-1.amazonaws.com/catalog/CORDEX-CMIP5.json" class="" target="_none">Intake-ESM Catalog</a> by Lars Buntemeyer</li></a>
          </li>
        </ul>
        <p><a href="/euro-cordex/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="exceptional-responders" class="dataset">
          <h3><a href="/exceptional-responders/">Exceptional Responders Initiative</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">epigenomics</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole exome sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Exceptional Responders Initiative is a pilot study to investigate the underlying molecular factors driving exceptional treatment responses of cancer patients to drug therapies. Study researchers will examine molecular profiles of tumors from patients either enrolled in a clinical trial for an investigational drug(s) and who achieved an exceptional response relative to other trial participants, or who achieved an exceptional response to a non-investigational chemotherapy. An exceptional response is defined as achievement of either a complete response or a partial response for at least 6 mon...</p>
          <p><a href="/exceptional-responders/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1093/jnci/djaa061" class="" target="_none">The Exceptional Responders Initiative: Feasibility of a National Cancer Institute Pilot Study</a> by Barbara A. Conley, Lou Staudt, et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/legacy-archive/" class="" target="_none">GDC Legacy Archive</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/exceptional-responders/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="fmi-radar" class="dataset">
          <h3><a href="/fmi-radar/">Finnish Meteorological Institute Weather Radar Data</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The up-to-date weather radar from the FMI radar network is available as Open Data. The data contain both single radar data along with composites over Finland in GeoTIFF and HDF5-formats. Available composite parameters consist of radar reflectivity (DBZ), rainfall intensity (RR), and precipitation accumulation of 1, 12, and 24 hours. Single radar parameters consist of radar reflectivity (DBZ), radial velocity (VRAD), rain classification (HCLASS), and Cloud top height (ETOP 20). Raw volume data from singe radars are also provided in HDF5 format with ODIM 2.3 conventions. Radar data becomes avail...</p>
          <p><a href="/fmi-radar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/fmidev/opendata-resources/blob/master/examples/python/Accumulated_prec_amount_from_radar_GeoTiff.ipynb" class="" target="_none">Processing GeoTIFF data with python</a> by Roope Tervo</li></a>
          </li>
          <li>
            <a href="https://github.com/fmidev/opendata-resources/tree/master/examples/qgis/radar-composite" class="" target="_none">Handling data with QGIS</a> by Markus Peura</li></a>
          </li>
          <li>
            <a href="https://github.com/fmidev/opendata-resources/blob/master/examples/python/radar-rhi-from-hdf5.ipynb" class="" target="_none">Processing HDF5 data with python</a> by Roope Tervo</li></a>
          </li>
        </ul>
        <p><a href="/fmi-radar/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="fm-ad" class="dataset">
          <h3><a href="/fm-ad/">Foundation Medicine Adult Cancer Clinical Dataset (FM-AD)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Foundation Medicine Adult Cancer Clinical Dataset (FM-AD) is a study conducted by Foundation
Medicine Inc (FMI). Genomic profiling data for approximately 18,000 adult patients with a diverse
array of cancers was generated using FoundationeOne, FMI&#39;s commercially available, comprehensive
genomic profiling assay. This dataset contains open Clinical and Biospecimen data.</p>
          <p><a href="/fm-ad/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/FM-AD" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;22981675[PMID]" class="" target="_none">Targeted next-generation sequencing of advanced prostate cancer identifies potential
therapeutic targets and disease heterogeneity.
</a> by Beltran H, Yelensky R, Frampton GM, Park K, Downing SR, MacDonald TY, Jarosz M, Lipson D,
Tagawa ST, Nanus DM, Stephens PJ, Mosquera JM, Cronin MT, Rubin MA
</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;28235761[PMID]" class="" target="_none">High-Throughput Genomic Profiling of Adult Solid Tumors Reveals Novel Insights into Cancer
Pathogenesis
</a> by Ryan J. Hartmaier, Lee A. Albacker, Juliann Chmielecki, Mark Bailey, Jie He, Michael E.
Goldberg, Shakti Ramkissoon, James Suh, Julia A. Elvin, Samuel Chiacchia, Garrett M.
Frampton, Jeffrey S. Ross, Vincent Miller, Philip J. Stephens and Doron Lipson
</li></a>
          </li>
        </ul>
        <p><a href="/fm-ad/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="maf-genome" class="dataset">
          <h3><a href="/maf-genome/">Golden Retriever Lifetime Study: Whole genome genotyping of Golden Retrievers on Axiom HD Arrays</a></h3>
          <p><span class="label label-info tag link-tag">genome</span><span class="label label-info tag link-tag">genotyping</span><span class="label label-info tag link-tag">golden retriever lifetime study</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">morris animal foundation</span></p>
          <p>Morris Animal Foundation’s Golden Retriever Lifetime Study is a longitudinal, prospective study following 3044 golden retrievers. The Study’s purpose is to identify the nutritional, environmental, lifestyle and genetic risk factors for cancer and other diseases. The Golden Oldie’s study enrolled an additional cohort of golden retrievers that had reached the age of 12 years or older and had not yet been diagnosed with a malignant cancer. This population can be used as a control group for conditions with high mortality in younger age. This dataset contains the data for ~1.1 million genetic marke...</p>
          <p><a href="/maf-genome/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://journals.plos.org/plosone/article?id&#x3D;10.1371/journal.pone.0269425" class="" target="_none">Cohort profile: The Golden Retriever Lifetime Study (GRLS)</a> by Julia Labadie, Brenna Swafford, Mara DePena, Kathy Tietje, Rodney Page, Janet Patterson-Kane</li></a>
          </li>
          <li>
            <a href="https://morrisanimalfoundation.github.io/grGWAS/" class="" target="_none">GRLS GWAS Tutorial</a> by Tamer Mansour</li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4581032/" class="" target="_none">The Golden Retriever Lifetime Study: establishing an observational cohort study with translational relevance for human health</a> by Michael K. Guy, Rodney L. Page, Wayne A. Jensen, Patricia N. Olson, J. David Haworth, Erin E. Searfoss, and Diane E. Brown</li></a>
          </li>
        </ul>
        <p><a href="/maf-genome/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="bdsp-icare" class="dataset">
          <h3><a href="/bdsp-icare/">I-CARE:International Cardiac Arrest REsearch consortium Electroencephalography Database</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>The International Cardiac Arrest REsearch consortium (I-CARE) Database includes baseline clinical information and continuous electroencephalography (EEG) recordings from 1,020 comatose patients with a diagnosis of cardiac arrest who were admitted to an intensive care unit from seven academic hospitals in the U.S. and Europe. Patients were monitored with 18 bipolar EEG channels over hours to days for the diagnosis of seizures and for neurological prognostication. Long-term neurological function was determined using the Cerebral Performance Category scale.</p>
          <p><a href="/bdsp-icare/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.60508/xwye-v214" class="" target="_none">I-CARE:International Cardiac Arrest REsearch consortium Electroencephalography Database</a> by Amorim E, Zheng WL, Ghassemi MM, Aghaeeaval M, Kandhare P, Karukonda V, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1101/2023.08.28.23294672" class="" target="_none">The International Cardiac Arrest Research (I-CARE) Consortium Electroencephalography Database</a> by Amorim E, Zheng WL, Ghassemi MM, Aghaeeaval M, Kandhare P, Karukonda V, et al.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.13026/gjvw-1m31" class="" target="_none">WFDB Software Package</a> by <a href="https://www.physionet.org" target="_none">Moody, G., Pollard, T., &amp; Moody, B.</a></li></a>
          </li>
        </ul>
        <p><a href="/bdsp-icare/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="in-imagery" class="dataset">
          <h3><a href="/in-imagery/">Indiana Statewide Digital Aerial Imagery Catalog</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The State of Indiana Geographic Information Office and IOT Office of Technology manage a series of digital orthophotography dating back to 2005.  Every year&#39;s worth of imagery is available as Cloud Optimized GeoTIFF (COG) files, original GeoTIFF, and other compressed deliverables such as ECW and MrSID.  Additionally, each imagery year is organized into a tile grid scheme covering the entire geography of Indiana.  All years of imagery are tiled from a 5,000 ft grid or sub tiles depending upon the resolution of the imagery.  The naming of the tiles reflects the lower left coordinate from the...</p>
          <p><a href="/in-imagery/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://gisimageryingov.s3.amazonaws.com/index.html" class="" target="_none">IGIO Imagery Opendata S3 Browser</a> by Indiana Geographic Information Office (IGIO)</li></a>
          </li>
          <li>
            <a href="https://indianamap-inmap.hub.arcgis.com/datasets/61d4dc991c154af49ad7c1d675182a4f/explore" class="" target="_none">ArcGIS Online Indiana Orthoimagery Viewer</a> by Indiana Geographic Information Office (IGIO)</li></a>
          </li>
          <li>
            <a href="https://imagery.gio.in.gov/pages/resources" class="" target="_none">Recording of 2025 - 2028 Indiana Orthoimagery Program Presentation</a> by Indiana Geographic Information Office (IGIO)</li></a>
          </li>
        </ul>
        <p><a href="/in-imagery/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="in-elevation" class="dataset">
          <h3><a href="/in-elevation/">Indiana Statewide Elevation Catalog</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The State of Indiana Geographic Information Office and IOT Office of Technology manage a series of digital LiDAR LAS files stored in AWS, dating back to the 2011-2013 collection and including the NRCS-funded 2016-2020 collection. These LiDAR datasets are available as uncompressed LAS files, for cloud storage and access. Each year&#39;s data is organized into a tile grid scheme covering the entire geography of Indiana, ensuring easy access and efficient processing. The tiles&#39; naming reflects each tile&#39;s lower left coordinate, facilitating accurate data management and retrieval. The AWS ...</p>
          <p><a href="/in-elevation/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://indianamap-inmap.hub.arcgis.com/maps/ff98e3834d464619bd5c8974b0038a13/about" class="" target="_none">ArcGIS Online Indiana Lidar Viewer</a> by Indiana Geographic Information Office (IGIO)</li></a>
          </li>
          <li>
            <a href="https://elevation.gio.in.gov/pages/resources" class="" target="_none">Recording of 2025 - 2028 Indiana Imagery and Elevation Program Presentation</a> by Indiana Geographic Information Office (IGIO)</li></a>
          </li>
          <li>
            <a href="https://giselevationingov.s3.amazonaws.com/index.html" class="" target="_none">IGIO Elevation Opendata S3 Browser</a> by Indiana Geographic Information Office (IGIO)</li></a>
          </li>
        </ul>
        <p><a href="/in-elevation/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="cotonoha-dic" class="dataset">
          <h3><a href="/cotonoha-dic/">Japanese Tokenizer Dictionaries</a></h3>
          <p><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">japanese</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Japanese Tokenizer Dictionaries for use with MeCab.</p>
          <p><a href="/cotonoha-dic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/polm/fugashi-sagemaker-demo/blob/master/fugashi%20wordcount.ipynb" class="" target="_none">Fugashi Word Count Tutorial</a> by <a href="https://cotonoha.io" target="_none">Paul O&#x27;Leary McCann</a></li></a>
          </li>
          <li>
            <a href="https://www.dampfkraft.com/nlp/how-to-tokenize-japanese.html" class="" target="_none">How to Tokenize Japanese in Python</a> by <a href="https://dampfkraft.com" target="_none">Paul O&#x27;Leary McCann</a></li></a>
          </li>
          <li>
            <a href="https://github.com/polm/unidic-py" class="" target="_none">unidic-py</a> by <a href="https://cotonoha.io" target="_none">Paul O&#x27;Leary McCann</a></li></a>
          </li>
        </ul>
        <p><a href="/cotonoha-dic/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="kraken2-ncbi-refseq-complete-v205" class="dataset">
          <h3><a href="/kraken2-ncbi-refseq-complete-v205/">Kraken2 NCBI RefSeq Complete V205 database on AWS</a></h3>
          <p><span class="label label-info tag link-tag">benchmark</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">metagenomics</span><span class="label label-info tag link-tag">microbiome</span></p>
          <p>Database for use with Kraken2 (taxonomic annotation of metagenomic sequencing reads) including all NCBI RefSeq genomes available in release V205</p>
          <p><a href="/kraken2-ncbi-refseq-complete-v205/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1099/mgen.0.000949" class="" target="_none">From defaults to databases: parameter and database choice dramatically impact the performance of metagenomic taxonomic classification tools</a> by Robyn J. Wright, Andre M. Comeau and Morgan G.I. Langille</li></a>
          </li>
          <li>
            <a href="https://github.com/R-Wright-1/kraken_metaphlan_comparison/wiki/Using-an-Amazon-Machine-Image-for-analysing-samples-with-Kraken2" class="" target="_none">Using an Amazon Machine Image for analysing samples with Kraken2</a> by <a href="https://github.com/R-Wright-1" target="_none">Robyn Wright</a></li></a>
          </li>
          <li>
            <a href="https://github.com/DerrickWood/kraken2/wiki/Manual" class="" target="_none">Kraken2</a> by <a href="https://github.com/DerrickWood/" target="_none">Derrick Wood, Jennifer Lu and Ben Langmead</a></li></a>
          </li>
        </ul>
        <p><a href="/kraken2-ncbi-refseq-complete-v205/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="mimiciii" class="dataset">
          <h3><a href="/mimiciii/">MIMIC-III (‘Medical Information Mart for Intensive Care’)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">us</span></p>
          <p>MIMIC-III (‘Medical Information Mart for Intensive Care’) is a large, 
single-center database comprising information relating to patients 
admitted to critical care units at a large tertiary care hospital. 
Data includes vital signs, medications, laboratory measurements, 
observations and notes charted by care providers, fluid balance, 
procedure codes, diagnostic codes, imaging reports, hospital length 
of stay, survival data, and more. The database supports applications 
including academic and industrial research, quality improvement initiatives, 
and higher education coursework. The MIMIC-I...</p>
          <p><a href="/mimiciii/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/machine-learning/building-predictive-disease-models-using-amazon-sagemaker-with-amazon-healthlake-normalized-data/" class="aws-link" target="_none">Building predictive disease models using Amazon SageMaker with Amazon HealthLake normalized data</a> by Ujjwal Ratan, Nihir Chadderwala, and Parminder Bhatia</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/perform-biomedical-informatics-without-a-database-using-mimic-iii-data-and-amazon-athena/" class="aws-link" target="_none">Perform biomedical informatics without a database using MIMIC-III data and Amazon Athena</a> by James Wiggins, Alistair Johnson</li></a>
          </li>
          <li>
            <a href="https://github.com/MIT-LCP/mimic-code" class="" target="_none">MIMIC-code GitHub repository</a> by <a href="https://github.com/alistairewj" target="_none">Alistair Johnson</a></li></a>
          </li>
        </ul>
        <p><a href="/mimiciii/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="msd" class="dataset">
          <h3><a href="/msd/">Medical Segmentation Decathlon</a></h3>
          <p><span class="label label-info tag link-tag">computed tomography</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">nifti</span><span class="label label-info tag link-tag">segmentation</span></p>
          <p>With recent advances in machine learning, semantic segmentation algorithms are becoming increasingly general purpose and translatable to unseen tasks. Many key algorithmic advances in the field of medical imaging are commonly validated on a small number of tasks, limiting our understanding of the generalisability of the proposed contributions. A model which works out-of-the-box on many tasks, in the spirit of AutoML, would have a tremendous impact on healthcare. The field of medical imaging is also missing a fully open source and comprehensive benchmark for general purpose algorithmic validati...</p>
          <p><a href="/msd/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/pdf/1902.09063.pdf" class="" target="_none">A large annotated medical image dataset for the development and evaluation of segmentation algorithms</a> by Simpson A. L., Antonelli M., Bakas S., Bilello M., Farahana K., van Ginneken B., et al</li></a>
          </li>
          <li>
            <a href="https://monai.io/start.html" class="" target="_none">MONAI: Getting Started</a> by <a href="https://github.com/Project-MONAI" target="_none">MONAI Development Team</a></li></a>
          </li>
          <li>
            <a href="https://github.com/Project-MONAI/tutorials" class="" target="_none">Pytorch-Integrated MSD Data Loader</a> by <a href="https://github.com/Project-MONAI" target="_none">MONAI Development Team</a></li></a>
          </li>
        </ul>
        <p><a href="/msd/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="met-office-global-deterministic" class="dataset">
          <h3><a href="/met-office-global-deterministic/">Met Office Global Deterministic 10km on a 2-year rolling archive</a></h3>
          <p><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">forecast</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">near-surface air temperature</span><span class="label label-info tag link-tag">near-surface relative humidity</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The flagship Numerical Weather Prediction model developed and used at the Met Office, is the Unified Model, the same model is used for both weather and climate prediction. For weather forecasting the Met Office runs several configurations of the Unified Model as part of its operational Numerical Weather Prediction suite. Uncovering 2 years&#39; worth of historical data, updated regularly with a time delay. The Global deterministic model is a global configuration of the Met Office Unified Models providing the most accurate short range deterministic forecast by any national meteorological servic...</p>
          <p><a href="/met-office-global-deterministic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://scitools-iris.readthedocs.io/en/stable/userguide/index.html" class="" target="_none">Iris</a> by <a href="https://scitools-iris.readthedocs.io/en/stable/copyright.html" target="_none">Iris Contributors</a></li></a>
          </li>
          <li>
            <a href="https://www.metoffice.gov.uk/research/approach/modelling-systems/unified-model" class="" target="_none">The Met Office Unified Model</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
          <li>
            <a href="https://www.metoffice.gov.uk/research/approach/modelling-systems/unified-model/weather-forecasting" class="" target="_none">Numerical weather prediction models</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
        </ul>
        <p><a href="/met-office-global-deterministic/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="met-office-global-ocean" class="dataset">
          <h3><a href="/met-office-global-ocean/">Met Office Global Ocean model on a 2-year rolling archive</a></h3>
          <p><span class="label label-info tag link-tag">forecast</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">marine</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">ocean sea surface height</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Global Ocean component of the Met Office Global Coupled Atmosphere-Land-Ocean-Ice system which has been running in operations since May 2022. The system provides a global physical analysis and coupled forecast products providing 3D daily mean fields of temperature and salinity, zonal and meridional velocities; 2D daily mean fields of sea surface height, bottom temperature, mixed layer depth, sea ice fraction, sea ice thickness and sea ice zonal and meridional velocities; and instantaneous hourly fields for sea surface height, sea surface temperature and surface currents. The Met Office Glo...</p>
          <p><a href="/met-office-global-ocean/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.metoffice.gov.uk/services/government/defence-and-security/ocean-models" class="" target="_none">Ocean models</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
          <li>
            <a href="https://www.metoffice.gov.uk/research/weather/ocean-forecasting/coupled-development" class="" target="_none">Coupled forecasting development</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
          <li>
            <a href="https://scitools-iris.readthedocs.io/en/stable/userguide/index.html" class="" target="_none">Iris</a> by <a href="https://scitools-iris.readthedocs.io/en/stable/copyright.html" target="_none">Iris Contributors</a></li></a>
          </li>
        </ul>
        <p><a href="/met-office-global-ocean/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="met-office-global-wave" class="dataset">
          <h3><a href="/met-office-global-wave/">Met Office Global Wave model on a 2-year rolling archive</a></h3>
          <p><span class="label label-info tag link-tag">forecast</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">marine</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">ocean sea surface height</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Met Office runs global wave forecast models to support marine safety and operational decision making. Met Office configurations are developed to be run using the community wave model WAVEWATCH IIITM. The global wave configuration is designed to generate accurate forecasts for open waters of the world’s oceans and larger seas. The Met Office wave models are forced using wind data from the Met Office Global Atmospheric Hi-Res Model. The global wave model is run to provide a five day outlook for wave characteristics defining height, period and direction of waves within a given sea-state. The ...</p>
          <p><a href="/met-office-global-wave/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://scitools-iris.readthedocs.io/en/stable/userguide/index.html" class="" target="_none">Iris</a> by <a href="https://scitools-iris.readthedocs.io/en/stable/copyright.html" target="_none">Iris Contributors</a></li></a>
          </li>
          <li>
            <a href="https://www.metoffice.gov.uk/research/weather/ocean-forecasting/coupled-development" class="" target="_none">Coupled forecasting development</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
          <li>
            <a href="https://www.metoffice.gov.uk/services/government/defence-and-security/ocean-models" class="" target="_none">Ocean models</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
        </ul>
        <p><a href="/met-office-global-wave/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="met-office-uk-deterministic" class="dataset">
          <h3><a href="/met-office-uk-deterministic/">Met Office UK Deterministic (UKV)2km on a 2-year rolling archive</a></h3>
          <p><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">forecast</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">near-surface air temperature</span><span class="label label-info tag link-tag">near-surface relative humidity</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The flagship Numerical Weather Prediction model developed and used at the Met Office, is the Unified Model, the same model is used for both weather and climate prediction. For weather forecasting the Met Office runs several configurations of the Unified Model as part of its operational Numerical Weather Prediction suite. Uncovering 2 years&#39; worth of historical data, updated regularly with a time delay. The UK deterministic model is a post processed regional downscaled configuration of the Unified Model, covering the UK and Ireland, with a resolution of approximately 0.018 degrees. The Unit...</p>
          <p><a href="/met-office-uk-deterministic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.metoffice.gov.uk/research/approach/modelling-systems/unified-model" class="" target="_none">The Met Office Unified Model</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
          <li>
            <a href="https://www.metoffice.gov.uk/research/approach/modelling-systems/unified-model/weather-forecasting" class="" target="_none">Numerical weather prediction models</a> by <a href="https://www.metoffice.gov.uk/" target="_none">Met Office</a></li></a>
          </li>
          <li>
            <a href="https://scitools-iris.readthedocs.io/en/stable/userguide/index.html" class="" target="_none">Iris</a> by <a href="https://scitools-iris.readthedocs.io/en/stable/copyright.html" target="_none">Iris Contributors</a></li></a>
          </li>
        </ul>
        <p><a href="/met-office-uk-deterministic/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="mevadata" class="dataset">
          <h3><a href="/mevadata/">Multiview Extended Video with Activities (MEVA)</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">urban</span><span class="label label-info tag link-tag">us</span><span class="label label-info tag link-tag">video</span></p>
          <p>The Multiview Extended Video with Activities (MEVA) dataset consists
video data of human activity, both scripted and unscripted,
collected with roughly 100 actors over several weeks.  The data was
collected with 29 cameras with overlapping and non-overlapping
fields of view. The current release consists of about 328 hours
(516GB, 4259 clips) of video data, as well as 4.6 hours (26GB) of
UAV data. Other data includes GPS tracks of actors, camera models,
and a site map. We have also released annotations for roughly 184 hours of
data. Further updates are planned.</p>
          <p><a href="/mevadata/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2107.11494" class="" target="_none">TinyAction Challenge: Recognizing Real-world Low-resolution Activities in Videos</a> by Praveen Tirupattur, Aayush J Rana, Tushar Sangam, Shruti Vyas, Yogesh S Rawat, Mubarak Shah</li></a>
          </li>
          <li>
            <a href="https://openaccess.thecvf.com/content/WACV2021/html/Corona_MEVA_A_Large-Scale_Multiview_Multimodal_Video_Dataset_for_Activity_Detection_WACV_2021_paper.html" class="" target="_none">MEVA: A Large-Scale Multiview, Multimodal Video Dataset for Activity Detection</a> by Kellie Corona, Katie Osterdahl, Roderic Collins, Anthony Hoogs</li></a>
          </li>
          <li>
            <a href="https://actev.nist.gov/" class="" target="_none">ActEV: Activities in Extended Video</a> by National Institute of Standards and Technology (NIST)</li></a>
          </li>
        </ul>
        <p><a href="/mevadata/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nex-gddp-cmip6" class="dataset">
          <h3><a href="/nex-gddp-cmip6/">NASA Earth Exchange Global Daily Downscaled Projections (NEX-GDDP-CMIP6)</a></h3>
          <p><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">climate projections</span><span class="label label-info tag link-tag">CMIP6</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">NASA Center for Climate Simulation (NCCS)</span><span class="label label-info tag link-tag">near-surface relative humidity</span><span class="label label-info tag link-tag">near-surface specific humidity</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">precipitation</span></p>
          <p>The NEX-GDDP-CMIP6 dataset is comprised of global downscaled climate
scenarios derived from the General Circulation Model (GCM) runs
conducted under the Coupled Model Intercomparison Project Phase 6
(CMIP6) and across two of the four &quot;Tier 1&quot; greenhouse gas emissions
scenarios known as Shared Socioeconomic Pathways (SSPs). The CMIP6
GCM runs were developed in support of the Sixth Assessment Report
of the Intergovernmental Panel on Climate Change (IPCC AR6). This
dataset includes downscaled projections from ScenarioMIP model
runs for which daily scenarios were produced and distributed...</p>
          <p><a href="/nex-gddp-cmip6/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://cmip6.surge.sh/" class="" target="_none">NEX-GDDP-CMIP6 Dashboard</a> by <a href="https://www.earthdata.nasa.gov/" target="_none">NASA</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/s41597-022-01393-4" class="" target="_none">NASA Global Daily Downscaled Projections, CMIP6</a> by Thrasher, B., Wang, W., Michaelis, A., Melton, F., Lee, T. and Nemani, R.</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/nasa-asdi-announce-no-cost-access-important-climate-dataset-aws-cloud/" class="aws-link" target="_none">NASA and ASDI announce no-cost access to important climate dataset on the AWS Cloud</a> by Dr. Manil Maskey and Ana Pinheiro Privette</li></a>
          </li>
        </ul>
        <p><a href="/nex-gddp-cmip6/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-heasarc" class="dataset">
          <h3><a href="/nasa-heasarc/">NASA High Energy Astrophysics Mission Data</a></h3>
          <p><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">datacenter</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">x-ray</span></p>
          <p>NASA data for high energy astrophysics (generally x-ray and gamma-ray domains) is made available here by the High Energy Astrophysics Science Archive Research Center. The HEASARC hosts the full data archives of over 30 different missions spanning 50 years. The data archive for each mission will contain a range of data types from spacecraft housekeeping and raw photon event list data up to high level science-ready products such as images, light curves (time series), and energy spectra.
<br/><br/>
This is a relatively modest total data volume but contains significant complexity and heterogeneity among the different missions. Data provided here are stored in the Flexible Image Transport System (<a href="https://heasarc.gsfc.nasa.gov/docs/heasarc/fits.html">FITS</a>) format common in astronomy. Higher level products are further defined to be consistent between missions following <a href="https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/ofwg_recomm.html">data model standards</a> agreed by the community and maintained by the HEASARC. Analysis of these data may require software also provided by HEASARC, the <a href="https://heasarc.gsfc.nasa.gov/docs/software/lheasoft/">HEASoft package</a>, consisting of tools generic to all FITS data, generic to all HEASARC-compliant data, and/or specific to individual missions as appropriate. Some missions provide standard science-ready data products, while others provide low-level data types and software to generate science-ready products from them. See the links for each mission for more information on how to use the data.
<br/><br/>
The <a href="https://heasarc.gsfc.nasa.gov/">HEASARC Website</a> also has archive browsing tools where you can query for observations corresponding to temporal and spatial constraints among others. These tools will ultimately point to files located on the archive by giving a URL beginning with the path <a href="https://heasarc.gsfc.nasa.gov/FTP/">https://heasarc.gsfc.nasa.gov/FTP/</a>. The data that are provided in the ODR follow the same structure, so when our tools give an https access URL, a user can simply swap in s3://nasa-heasarc/ for the first part of that URL and get a cloud URI. Note also that some selections have been made to what has been copied to the ODR, while the HEASARC archive itself remains the definitive and legacy source for the complete datasets.
<br/><br/>
The HEASARC also...</p>
          <p><a href="/nasa-heasarc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://heasarc.gsfc.nasa.gov/docs/archive/cloud.html" class="" target="_none">HEASARC Cloud access page</a> by <a href="https://heasarc.gsfc.nasa.gov/" target="_none">NASA/HEASARC</a></li></a>
          </li>
          <li>
            <a href="https://heasarc.gsfc.nasa.gov/docs/heasarc/biblio/pubs/" class="" target="_none">HEASARC Bibliography</a> by <a href="https://heasarc.gsfc.nasa.gov/" target="_none">Various; curated by HEASARC</a></li></a>
          </li>
          <li>
            <a href="https://heasarc.gsfc.nasa.gov/docs/software/lheasoft/" class="" target="_none">HEASoft</a> by <a href="https://heasarc.gsfc.nasa.gov/" target="_none">NASA/HEASARC</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-heasarc/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-lambda" class="dataset">
          <h3><a href="/nasa-lambda/">NASA Legacy Archive for Microwave Background Data Analysis (LAMBDA)</a></h3>
          <p><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">datacenter</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>NASA data for cosmic microwave background (CMB) analysis is made available here by the Legacy Archive for Microwave Background Data Analysis (LAMBDA), which is a part of NASA&#39;s High Energy Astrophysics Science Archive Research Center (HEASARC). LAMBDA hosts the data archives of over 30 different CMB missions spanning 30+ years. The data archive for each mission may contain a range of data types from low-level time-ordered data to high level science-ready products such as sky maps and angular power spectra. Also provided in consistent formats are a variety of full sky maps in complementary ...</p>
          <p><a href="/nasa-lambda/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://lambda.gsfc.nasa.gov/product/aws/" class="" target="_none">LAMBDA cloud access</a> by <a href="https://lambda.gsfc.nasa.gov/" target="_none">NASA/LAMBDA</a></li></a>
          </li>
          <li>
            <a href="https://lambda.gsfc.nasa.gov/toolbox/" class="" target="_none">LAMBDA Tools</a> by <a href="https://lambda.gsfc.nasa.gov/" target="_none">NASA/LAMBDA</a></li></a>
          </li>
          <li>
            <a href="https://lambda.gsfc.nasa.gov/papers/" class="" target="_none">CMB Experiment Bibliography</a> by <a href="https://lambda.gsfc.nasa.gov/" target="_none">Various; curated by LAMBDA</a></li></a>
          </li>
        </ul>
        <p><a href="/nasa-lambda/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-soho-comet-challenge-on-aws" class="dataset">
          <h3><a href="/nasa-soho-comet-challenge-on-aws/">NASA SOHO/LASCO2 comet challenge on AWS</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">NASA SMD AI</span></p>
          <p>The SOHO/LASCO data set (prepared for the challenge hosted in Topcoder) provided here comes from the instrument’s C2 telescope and comprises approximately 36,000 images spread across 2,950 comet observations. The human eye is a very sensitive tool and it is the only tool currently used to reliably detect new comets in SOHO data - particularly comets that are very faint and embedded in the instrument background noise. Bright comets can be easily detected in the LASCO data by relatively simple automated algorithms, but the majority of comets observed by the instrument are extremely faint, noise-...</p>
          <p><a href="/nasa-soho-comet-challenge-on-aws/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;E4OxTaqTP6E" class="" target="_none">Topcoder NASA Comet Discovery: A Recap</a> by TopCoder</li></a>
          </li>
          <li>
            <a href="https://www.topcoder.com/blog/topcoder-challenge-finds-two-new-comets-for-nasa/" class="" target="_none">Topcoder Challenge Finds Two New Comets For NASA</a> by Annika Nagy</li></a>
          </li>
          <li>
            <a href="https://science.nasa.gov/science-news/winners-selected-for-the-nasa-soho-comet-search-with-artificial-intelligence-open-science-challenge" class="" target="_none">Winners Selected for the NASA SOHO Comet Search with Artificial Intelligence Open-Science Challenge</a> by Denise Hill</li></a>
          </li>
        </ul>
        <p><a href="/nasa-soho-comet-challenge-on-aws/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-osdr" class="dataset">
          <h3><a href="/nasa-osdr/">NASA Space Biology Open Science Data Repository (OSDR)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">GeneLab</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">space biology</span></p>
          <p>NASA’s Space Biology Open Science Data Repository (OSDR) introduces a one-stop site where users can explore and contribute a variety of NASA open science biological data. This site consolidates data from the Ames Life Sciences Data Archive (ALSDA) and GeneLab and includes information about the broader NASA Open Science and Open Data initiatives, all at one centralized location. Our mission is to maximize the utilization of the valuable biological research resources and enable new discoveries.<br/><br/>
OSDR introduces access to data generated from spaceflight and space relevant experiments that explore </br/>...</p>
          <p><a href="/nasa-osdr/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S2211124720314303" class="" target="_none">Advancing the Integration of Biosciences Data Sharing to Further Enable Space Exploration</a> by Ryan T. Scott, Kirill Grigorev, Graham Mackintosh, Samrawit G. Gebre, Christopher E. Mason, Martha E. Del Alto, Sylvain V. Costes</li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/nar/article/49/D1/D1515/5932845" class="" target="_none">NASA GeneLab: interfaces for the exploration of space omics data</a> by Daniel C Berrios, Jonathan Galazka, Kirill Grigorev, Samrawit Gebre, Sylvain V Costes</li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/bioinformatics/article/35/10/1753/513406" class="" target="_none">GeneLab: Omics database for spaceflight experiments</a> by Shayoni Ray, Samrawit Gebre, Homer Fogle, Daniel C Berrios, Peter B Tran, Jonathan M Galazka, Sylvain V Costes</li></a>
          </li>
        </ul>
        <p><a href="/nasa-osdr/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nifs-lhd" class="dataset">
          <h3><a href="/nifs-lhd/">NIFS Large Helical Device (LHD) Experiment</a></h3>
          <p><span class="label label-info tag link-tag">analytics</span><span class="label label-info tag link-tag">anomaly detection</span><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">computed tomography</span><span class="label label-info tag link-tag">datacenter</span><span class="label label-info tag link-tag">digital assets</span><span class="label label-info tag link-tag">electricity</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">fluid dynamics</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">physics</span><span class="label label-info tag link-tag">post-processing</span><span class="label label-info tag link-tag">radiation</span><span class="label label-info tag link-tag">signal processing</span><span class="label label-info tag link-tag">source code</span><span class="label label-info tag link-tag">turbulence</span><span class="label label-info tag link-tag">video</span><span class="label label-info tag link-tag">x-ray</span><span class="label label-info tag link-tag">x-ray tomography</span></p>
          <p>The Large Helical Device (LHD), owned and operated by the National Institute for Fusion Science (NIFS), is one of the world&#39;s largest plasma confinement device which employs a heliotron magnetic configuration generated by the superconducting coils. The objectives are to conduct academic research on the confinement of steady-state, high-temperature, high-density plasmas, core plasma physics, and fusion reactor engineering, which are necessary to develop future fusion reactors. All the archived data of the LHD plasma diagnostics are available since the beginning of the LHD experiment, starte...</p>
          <p><a href="/nifs-lhd/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www-lhd.nifs.ac.jp/pub/pdf/LHD_Guide/5_DataLHDv2.pdf" class="" target="_none">Data handling in the LHD experiments -- Guide for the collaborator (pdf)</a> by <a href="https://researchmap.jp/ohdachi?lang&#x3D;en" target="_none">Satoshi Ohdachi</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.13182/FST10-A10830" class="" target="_none">Data Acquisition and Management System of LHD</a> by <a href="https://orcid.org/0000-0001-6388-4489" target="_none">Hideya Nakanishi, et al.</a></li></a>
          </li>
          <li>
            <a href="https://w3.lhd.nifs.ac.jp/en/Service/DataAccess/RetrieveManual-e.htm" class="" target="_none">How to retrieve diagnostic raw data and/or primarily processed data using &quot;Retrieve&quot;</a> by <a href="https://w3.lhd.nifs.ac.jp/" target="_none">NIFS LABCOM</a></li></a>
          </li>
        </ul>
        <p><a href="/nifs-lhd/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-oar-hourly-gdp" class="dataset">
          <h3><a href="/noaa-oar-hourly-gdp/">NOAA - hourly position, current, and sea surface temperature from drifters</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>This dataset includes <em>hourly</em> sea surface temperature and current data collected by satellite-tracked surface drifting buoys (&quot;drifters&quot;) of the <a href="https://www.aoml.noaa.gov/phod/gdp">NOAA Global Drifter Program</a>. The <a href="https://www.aoml.noaa.gov/phod/gdp/data.php">Drifter Data Assembly Center (DAC) at NOAA’s Atlantic Oceanographic and Meteorological Laboratory (AOML)</a> has applied quality control procedures and processing to edit these observational data and obtain estimates at regular hourly intervals. The data include positions (latitude and longitude), sea surface temperatures (total, diurnal, and non-diurnal components) and velocities (eastward, northward) with accompanying uncertainty estimates. Metadata include identification numbers, experiment number, start location and time, end location and time, drogue loss date, death code, manufacturer, and drifter type. <br /> <br /> Please note that data from the Global Drifter Program are also available at <a href="https://doi.org/10.25921/7ntx-z961">6-hourly intervals</a> but derived via alternative methods. The 6-hourly dataset goes back further in time (1979) and may be more appropriate for studies of long-term, low frequency patterns of the oceanic circulation. Yet, the 6-hourly dataset does not resolve fully high-frequency processes such as tides and inertial oscillations as well as sea surface temperature diurnal variability. <br /> <br /> [CITING NOAA - hourly position, current, and sea surface temperature from drifters data. Citation for this dataset should include the following information below.] <br /> Elipot, Shane; Sykulski, Adam; Lumpkin, Rick; ...</p>
          <p><a href="/noaa-oar-hourly-gdp/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41597-022-01670-2" class="" target="_none">A Dataset of Hourly Sea Surface Temperature From Drifting Buoys (2022)</a> by Elipot, S., A. Sykulski, R. Lumpkin, L. Centurioni, and M. Pazos</li></a>
          </li>
          <li>
            <a href="https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2016JC011716" class="" target="_none">A global surface drifter dataset at hourly resolution (2016)</a> by Elipot, S., R. Lumpkin, R. C. Perez, J. M. Lilly, J. J. Early, and A. M. Sykulski</li></a>
          </li>
          <li>
            <a href="https://github.com/Cloud-Drift/gdp-get-started" class="" target="_none">Working with GDP hourly data using python and xarray, a CloudDrift notebook</a> by <a href="https://people.miami.edu/profile/2c5f7c9fe748639bf50c85194ee4b558" target="_none">Shane Elipot</a></li></a>
          </li>
        </ul>
        <p><a href="/noaa-oar-hourly-gdp/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-eri" class="dataset">
          <h3><a href="/noaa-eri/">NOAA Emergency Response Imagery</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">weather</span></p>
          <p>In order to support NOAA&#39;s homeland security and emergency response requirements, the National Geodetic Survey Remote Sensing Division (NGS/RSD) has the capability to acquire and rapidly disseminate a variety of spatially-referenced datasets to federal, state, and local government agencies, as well as the general public. Remote sensing technologies used for these projects have included lidar, high-resolution digital cameras, a film-based RC-30 aerial camera system, and hyperspectral imagers. Examples of rapid response initiatives include acquiring high resolution images with the Emerge/App...</p>
          <p><a href="/noaa-eri/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/open-data-helps-recovery-aftermath-devastating-weather-events/" class="aws-link" target="_none">Open data helps recovery in the aftermath of devastating weather events</a> by Jena Kent</li></a>
          </li>
          <li>
            <a href="https://github.com/NOAA-Big-Data-Program/bdp-data-docs/blob/main/NGS/NOAA_ER_Pre_Event_Data.pdf" class="" target="_none">Using Emergency and Pre-Event Imagery</a> by Jon Sellars</li></a>
          </li>
          <li>
            <a href="https://gitlab.cicsnc.org/workshop-development/eri_notebook" class="" target="_none">ERI notebook using SageMaker Studio Lab (SMSL)</a> by Mya Sears</li></a>
          </li>
        </ul>
        <p><a href="/noaa-eri/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-gefs-reforecast" class="dataset">
          <h3><a href="/noaa-gefs-reforecast/">NOAA Global Ensemble Forecast System (GEFS) Re-forecast</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOAA has generated a multi-decadal reanalysis and reforecast data set to accompany the next-generation version of its ensemble prediction system, the Global Ensemble Forecast System, version 12 (GEFSv12). Accompanying the real-time forecasts are “reforecasts” of the weather, that is, retrospective forecasts spanning the period 2000-2019. These reforecasts are not as numerous as the real-time data; they were generated only once per day, from 00 UTC initial conditions, and only 5 members were provided, with the following exception. Once weekly, an 11-member reforecast was generated, and these ex...</p>
          <p><a href="/noaa-gefs-reforecast/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/vgensini/gefs_v12_example/blob/master/GEFS_v12_eample.ipynb" class="" target="_none">Example using GEFS Reforecast Data from AWS</a> by <a href="https://atlas.niu.edu" target="_none">Victor Gensini</a></li></a>
          </li>
          <li>
            <a href="https://drive.google.com/file/d/1lCIB3qz9S1nu2qAaetnLq0N5ReyTXuHd/view" class="" target="_none">The GEFS v12 and its reanalyses and reforecasts (Slides)</a> by Tom Hamill</li></a>
          </li>
          <li>
            <a href="https://mogismog.github.io/retro-diss-part-3.html#retro-diss-part-3" class="" target="_none">Retrospective Dissertation - Part 3: Reforecast Data</a> by <a href="https://github.com/mogismog" target="_none">Francisco Alvarez</a></li></a>
          </li>
        </ul>
        <p><a href="/noaa-gefs-reforecast/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-gfs-bdp-pds" class="dataset">
          <h3><a href="/noaa-gfs-bdp-pds/">NOAA Global Forecast System (GFS)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOTE - Upgrade NCEP Global Forecast System to v16.3.0 - Effective November 29, 2022 See notification <a href="https://www.weather.gov/media/notification/pdf2/scn22-104_gfs.v16.3.0.pdf">HERE</a>
<br/>
<br/>
The Global Forecast System (GFS) is a weather forecast model produced
by the National Centers for Environmental Prediction (NCEP). Dozens of
atmospheric and land-soil variables are available through this dataset,
from temperatures, winds, and precipitation to soil moisture and
atmospheric ozone concentration. The entire globe is covered by the GFS
at a base horizontal resolution of 18 miles (28 kilometers) between grid
points, which is used by the operational forecasters who predict weather
out to 16 days in the future. Horizontal resolution drops to 44 miles
(70 kilometers) between grid point for forecasts between one week and two
weeks.
<br/>
<br/>
The NOAA Global Forecast Systems (GFS) Warm Start Initial Conditions are 
produced by the National Centers for Environmental Prediction Center (NCEP) 
to run operational deterministic medium-range numerical weather predictions.<br>The GFS is built with the GFDL Finite-Volume Cubed-Sphere Dynamical Core (FV3) 
and the Grid-Point Statistical Interpolation (GSI) data assimilation system.<br>Please visit the links below in the Documentation section to find more details 
about the model and the data...</p>
          <p><a href="/noaa-gfs-bdp-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.mdpi.com/2077-1312/12/2/220/pdf" class="" target="_none">Validation and Application of the Accu-Waves Operational Platform for Wave Forecasts at Ports</a> by Christos Makris et al.</li></a>
          </li>
          <li>
            <a href="https://noaa-gfs-warmstart-pds.s3.amazonaws.com/GFS+Warm+Restart+Files.v1.pdf" class="" target="_none">GFS Warm Restart Files Additional Information</a> by Fanglin Yang</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1109/JSTARS.2024.3356858" class="" target="_none">Toward the Next Generation of Microwave Sounders: Benefits of a Low-Earth Orbit Hyperspectral Microwave Instrument in All- Weather Conditions using AI</a> by Eric S. Maddy</li></a>
          </li>
        </ul>
        <p><a href="/noaa-gfs-bdp-pds/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="napierone" class="dataset">
          <h3><a href="/napierone/">NapierOne Mixed File Dataset</a></h3>
          <p><span class="label label-info tag link-tag">computer forensics</span><span class="label label-info tag link-tag">computer security</span><span class="label label-info tag link-tag">cyber security</span><span class="label label-info tag link-tag">digital forensics</span><span class="label label-info tag link-tag">malware</span><span class="label label-info tag link-tag">mixed file dataset</span><span class="label label-info tag link-tag">ransomware</span></p>
          <p>NapierOne is a modern cybersecurity mixed file data set, primarily aimed at, but not limited to, ransomware detection and forensic analysis. The dataset contains over 500,000 distinct files, representing 44 distinct popular file types. It was designed to address the known deficiency in research reproducibility and improve consistency by facilitating research replication and repeatability. The data set was inspired by the Govdocs1 data set and it is intended that ‘NapierOne’ be used as a complement to this original data set. An investigation was performed with the goal of determining the common...</p>
          <p><a href="/napierone/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.fsidi.2021.301330" class="" target="_none">NapierOne - A modern mixed file data set alternative to Govdocs1</a> by Simon R.Davies, Richard Macfarlane, William J.Buchanan</li></a>
          </li>
          <li>
            <a href="https://github.com/simonrdavies/NapierOne/blob/main/tutorials.md" class="" target="_none">napierOne use examples</a> by <a href="https://github.com/simonrdavies/NapierOne/" target="_none">s.davies</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1109/ICEET53442.2021.9659618" class="" target="_none">Exploring the Need For an Updated Mixed File Research Data Set</a> by Simon R.Davies, Richard Macfarlane, William J.Buchanan.</li></a>
          </li>
        </ul>
        <p><a href="/napierone/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nci-imaging-data-commons" class="dataset">
          <h3><a href="/nci-imaging-data-commons/">National Cancer Institute Imaging Data Commons (IDC) Collections</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">digital pathology</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">microscopy</span><span class="label label-info tag link-tag">radiology</span></p>
          <p><a href="https://imaging.datacommons.cancer.gov">Imaging Data Commons (IDC)</a>  is a repository within the 
<a href="https://datacommons.cancer.gov">Cancer Research Data Commons (CRDC)</a> that manages imaging data 
and enables its integration with the other components of CRDC. IDC hosts a growing number of imaging collections that are contributed 
by either funded US National Cancer Institute (NCI)  data collection 
activities, or by the individual researchers.Image data hosted by IDC is stored in DICOM  format. </p>
          <p><a href="/nci-imaging-data-commons/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ImagingDataCommons/IDC-Tutorials/tree/master/notebooks/getting_started" class="" target="_none">IDC &quot;Getting started&quot; tutorial series</a> by <a href="https://learn.canceridc.dev/idc-team" target="_none">Imaging Data Commons team</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1158/0008-5472.can-21-0950" class="" target="_none">NCI Imaging Data Commons</a> by <a href="https://learn.canceridc.dev/idc-team" target="_none">Fedorov A, Longabaugh W, Pot D, Clunie D, Pieper S, Aerts H, et al.</a></li></a>
          </li>
          <li>
            <a href="https://portal.imaging.datacommons.cancer.gov/" class="" target="_none">IDC Portal</a> by <a href="https://learn.canceridc.dev/idc-team" target="_none">Imaging Data Commons team</a></li></a>
          </li>
        </ul>
        <p><a href="/nci-imaging-data-commons/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-pds-ncdb" class="dataset">
          <h3><a href="/nrel-pds-ncdb/">National Climate Database (NCDB)</a></h3>
          <p><span class="label label-info tag link-tag">climate projections</span><span class="label label-info tag link-tag">CMIP5</span><span class="label label-info tag link-tag">CMIP6</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">solar</span></p>
          <p>The National Climate Database (NCDB) seeks to be the definitive source of climate 
data for energy applications. The goal of the NCDB is to provide unbiased high 
temporal and spatial resolution climate data needed for renewable energy modeling. 
The NCDB seeks to maintain the inherent relationship between the various parameters 
that are needed to model solar, wind, hydrology and load and provide data for multiple 
important climate scenarios.  </p>
          <p><a href="/nrel-pds-ncdb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NREL/hsds-examples" class="" target="_none">HSDS Examples</a> by Caleb Phillips, Caroline Draxl, John Readey, Jordan Perr-Sauer, Michael Rossol</li></a>
          </li>
          <li>
            <a href="https://ascmo.copernicus.org/articles/9/103/2023/" class="" target="_none">Regridding uncertainty for statistical downscaling of solar radiation</a> by Maggie D. Bailey, Douglas Nychka, Manajit Sengupta, Aron Habte, Yu Xie, Soutir Bandyopadhyay</li></a>
          </li>
          <li>
            <a href="https://climate.nrel.gov" class="" target="_none">NCDB Website</a> by NREL NCDB Team</li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-ncdb/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nsw-herbarium" class="dataset">
          <h3><a href="/nsw-herbarium/">National Herbarium of NSW</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">digital preservation</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span></p>
          <p>The National Herbarium of New South Wales is one of the most significant scientific, cultural and historical botanical resources in the Southern hemisphere. The 1.43 million preserved plant specimens have been captured as high-resolution images and the biodiversity metadata associated with each of the images captured in digital form. Botanical specimens date from year 1770 to today, and form voucher collections that document the distribution and diversity of the world&#39;s flora through time, particularly that of NSW, Austalia and the Pacific.The data is used in biodiversity assessment, syste...</p>
          <p><a href="/nsw-herbarium/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ala.org.au/" class="" target="_none">Atlas of Living Australia</a> by <a href="https://www.ala.org.au/about-ala/" target="_none">CSIRO</a></li></a>
          </li>
          <li>
            <a href="https://www.rbgsyd.nsw.gov.au/science/national-herbarium-of-new-south-wales" class="" target="_none">Accessing the National Herbarium of NSW on AWS</a> by <a href="https://www.rbgsyd.nsw.gov.au/Science/Our-Science-Staff" target="_none">Dr Shelley James</a></li></a>
          </li>
          <li>
            <a href="https://avh.ala.org.au/occurrences/search?q&#x3D;collection_uid%3Aco54#tab_mapView" class="" target="_none">The Australasian Virtual Herbarium</a> by <a href="https://avh.chah.org.au/index.php/about/" target="_none">avh@chah.org.au</a></li></a>
          </li>
        </ul>
        <p><a href="/nsw-herbarium/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="opencitymodel" class="dataset">
          <h3><a href="/opencitymodel/">Open City Model (OCM)</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">events</span><span class="label label-info tag link-tag">geospatial</span></p>
          <p>Open City Model is an initiative to provide cityGML data for all the buildings in the United States.
By using other open datasets in conjunction with our own code and algorithms it is our goal to provide 3D geometries for every US building.</p>
          <p><a href="/opencitymodel/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/opencitymodel/opencitymodel/blob/master/examples/Query-OpenCityModel-using-AWS-Athena.md" class="" target="_none">Running queries on Open City Model using AWS Athena</a> by <a href="https://github.com/agilliland" target="_none">Allen Gilliland</a></li></a>
          </li>
          <li>
            <a href="https://github.com/opencitymodel/opencitymodel/blob/master/examples/3dCityDB-to-GoogleEarth.md" class="" target="_none">Using Open City Model with the 3dCityDB</a> by <a href="https://github.com/agilliland" target="_none">Allen Gilliland</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/aws-asdi-cities-smsl-notebook" class="aws-link" target="_none">Investigating environmental characteristics of US cities using publicly available ASDI datasets</a> by Darren Ko</li></a>
          </li>
        </ul>
        <p><a href="/opencitymodel/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="craam-open-vlf" class="dataset">
          <h3><a href="/craam-open-vlf/">Open VLF: Scientific Open Data Initiative for CRAAM&#x27;s SAVNET and AWESOME VLF Data.</a></h3>
          <p><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>This platform is maintained by <a href="https://www.mackenzie.br/centro-de-radio-astronomia-e-astrofisica-mackenzie">CRAAM</a> (Mackenzie Radio Astronomy and Astrophysics Center), a research center operated by <a href="https://www.mackenzie.br/">UPM</a> (Mackenzie Presbyterian University) and <a href="https://www.gov.br/inpe">INPE</a> (National Institute for Space Research), to provide public and free access for researchers, students, and the interested public to VLF (Very Low Frequency) data from CRAAM&#39;s antenna systems. Amazon AWS supports all data stored through the <a href="https://aws.amazon.com/pt/opendata/">AWS Open Data Program</a>.
Very Low Frequency (VLF) signals can be used for navigation services, communication with submarines, and are a powerful tool to study the low-altitude Earth&#39;s io...</p>
          <p><a href="/craam-open-vlf/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://open-vlf.web.app" class="" target="_none">Open VLF platform</a> by <a href="https://www.mackenzie.br/centro-de-radio-astronomia-e-astrofisica-mackenzie" target="_none">CRAAM</a></li></a>
          </li>
          <li>
            <a href="https://github.com/open-vlf" class="" target="_none">All source code, program utilities, and file layouts.</a> by Kauffmann, DHV; Santiago, LS; Oliveira, R de.</li></a>
          </li>
          <li>
            <a href="https://open-vlf.web.app" class="" target="_none">Open VLF: Scientific Open Data Initiative for CRAAM&#x27;s SAVNET and AWESOME VLF Data (2023).</a> by Kauffmann, DHV; Santiago, LS; Raulin, JP; Correia, E; Oliveira, R de.</li></a>
          </li>
        </ul>
        <p><a href="/craam-open-vlf/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="openfold" class="dataset">
          <h3><a href="/openfold/">OpenProteinSet</a></h3>
          <p><span class="label label-info tag link-tag">alphafold</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">msa</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">openfold</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">protein folding</span><span class="label label-info tag link-tag">protein template</span></p>
          <p>Multiple sequence alignments (MSAs) for 140,000 unique <a href="https://www.rcsb.org/">Protein Data Bank</a> (PDB) chains and 16,000,000 <a href="https://uniclust.mmseqs.com/">UniClust30</a> clusters. Template hits are also provided for the PDB chains and 270,000 UniClust30 clusters chosen for maximal diversity and MSA depth. MSAs were generated with HHBlits (-n3) and JackHMMER against MGnify, BFD, UniRef90, and UniClust30 while templates were identified from PDB70 with HHSearch, all according to procedures outlined in the supplement to the AlphaFold 2 Nature paper, <a href="https://www.nature.com/articles/s41586-021-03819-2">Jumper et al. 2021</a>. We expect the database to be broadly useful to structural biologists training or valid...</p>
          <p><a href="/openfold/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/machine-learning/run-inference-at-scale-for-openfold-a-pytorch-based-protein-folding-ml-model-using-amazon-eks/" class="aws-link" target="_none">Run inference at scale for OpenFold, a PyTorch-based protein folding ML model, using Amazon EKS</a> by Shubha Kumbadakone, Ankur Srivastava, and Sachin Kadyan</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2308.05326" class="" target="_none">OpenProteinSet: Training data for structural biology at scale</a> by Ahdritz, Gustaf; Bouatta, Nazim; Kadyan, Sachin; Jarosch, Lukas; Berenberg, Daniel; Fisk, Ian, et al</li></a>
          </li>
          <li>
            <a href="https://biorxiv.org/content/10.1101/2022.11.20.517210" class="" target="_none">OpenFold: Retraining AlphaFold2 yields new insights into its learning mechanisms and capacity for generalization</a> by Ahdritz, Gustaf; Bouatta, Nazim; Kadyan, Sachin; Xia, Qinghui; Gerecke, William; O&#x27;Donnell, Timothy J, et al</li></a>
          </li>
        </ul>
        <p><a href="/openfold/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="pd12m" class="dataset">
          <h3><a href="/pd12m/">PD12M</a></h3>
          <p><span class="label label-info tag link-tag">art</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">media</span></p>
          <p>PD12M is a collection of 12.4 million CC0/PD image-caption pairs for the purpose of training generative image models.</p>
          <p><a href="/pd12m/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://huggingface.co/datasets/Spawning/PD12M" class="" target="_none">Hugging Face Dataset</a> by Spawning</li></a>
          </li>
          <li>
            <a href="https://source.plus/pd12m" class="" target="_none">Source.Plus</a> by Spawning</li></a>
          </li>
          <li>
            <a href="https://arxiv.org/abs/2410.23144" class="" target="_none">PD12M: A Large-Scale Image Captioning Dataset</a> by Jordan Meyer, Nick Padgett, Laura Exline, Cullen Miller</li></a>
          </li>
          <li>
            <a href="https://huggingface.co/datasets/Spawning/PD12M/blob/main/tutorials/images.md" class="" target="_none">Downloading Images</a> by Spawning</li></a>
          </li>
          <li>
            <a href="https://huggingface.co/datasets/Spawning/PD12M/blob/main/Datasheet.pdf" class="" target="_none">Datasheet</a> by Spawning</li></a>
          </li>
        </ul>
        <p><a href="/pd12m/#usageexamples">See 6 usage examples &rarr;</a></p>
        </div>
        <div id="pohang-canal-dataset" class="dataset">
          <h3><a href="/pohang-canal-dataset/">Pohang Canal Dataset: A Multimodal Maritime Dataset for Autonomous Navigation in Restricted Waters</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">marine navigation</span><span class="label label-info tag link-tag">robotics</span></p>
          <p>This dataset presents a multi-modal maritime dataset acquired in restricted waters in Pohang, South Korea. The sensor suite is composed of three LiDARs (one 64-channel LiDAR and two 32-channel LiDARs), a marine radar, two visual cameras used as a stereo camera, an infrared camera, an omnidirectional camera with 6 directions, an AHRS, and a GPS with RTK. The dataset includes the sensor calibration parameters and SLAM-based baseline trajectory. It was acquired while navigating a 7.5 km route that includes a narrow canal area, inner and outer port areas, and a near-coastal area. The aim of this d...</p>
          <p><a href="/pohang-canal-dataset/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/dhchung/rosmsg_player" class="" target="_none">Ros Message Player for Pohang Dataset</a> by Dongha Chung</li></a>
          </li>
          <li>
            <a href="https://github.com/dhchung/lidar_to_image" class="" target="_none">ROS package for LiDAR to image of Pohang Canal Dataset</a> by Dongha Chung</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1177/02783649231191145" class="" target="_none">Pohang Canal Dataset: A Multimodal Maritime Dataset for Autonomous Navigation in Restricted Waters</a> by Dongha Chung, Jonghwi Kim, Changyu Lee, Jinwhan Kim</li></a>
          </li>
        </ul>
        <p><a href="/pohang-canal-dataset/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="qiime2" class="dataset">
          <h3><a href="/qiime2/">QIIME 2 Tutorial Data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">metagenomics</span><span class="label label-info tag link-tag">microbiome</span></p>
          <p>QIIME 2 (pronounced “chime two”) is a microbiome multi-omics bioinformatics and data science platform that is trusted, free, open source, extensible, and community developed and supported.</p>
          <p><a href="/qiime2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://use.qiime2.org/" class="" target="_none">Using QIIME 2</a> by <a href="https://github.com/caporaso-lab/using-qiime2/graphs/contributors" target="_none">Caporaso Lab</a></li></a>
          </li>
          <li>
            <a href="https://docs.qiime2.org/" class="" target="_none">QIIME 2 User Documentation (currently being replaced by *Using QIIME 2*)</a> by <a href="https://github.com/qiime2/docs/graphs/contributors" target="_none">The QIIME 2 Development Team</a></li></a>
          </li>
          <li>
            <a href="https://develop.qiime2.org/" class="" target="_none">Developing with QIIME 2</a> by <a href="https://github.com/caporaso-lab/developing-with-qiime2/graphs/contributors" target="_none">Caporaso and Bolyen</a></li></a>
          </li>
        </ul>
        <p><a href="/qiime2/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="bdsp-sparcnet" class="dataset">
          <h3><a href="/bdsp-sparcnet/">SPaRCNet data:Seizures, Rhythmic and Periodic Patterns in ICU Electroencephalography</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>The IIIC dataset includes  50,697 labeled EEG samples from 2,711 patients&#39; and 6,095 EEGs that were annotated by physician experts from 18 institutions. These samples were used to train SPaRCNet (Seizures, Periodic and Rhythmic Continuum patterns Deep Neural Network), a computer program that classifies IIIC events with an accuracy matching clinical experts.</p>
          <p><a href="/bdsp-sparcnet/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.60508/cw6j-s785" class="" target="_none">SPaRCNet data:Seizures, Rhythmic and Periodic Patterns in ICU Electroencephalography</a> by Jing, J., Ge, W., Struck, A. F., Fernandes, M., Hong, S., An, S., et al.</li></a>
          </li>
          <li>
            <a href="https://github.com/bdsp-core/ICU_EEG_Neuro_Prognosis" class="" target="_none">IIIC-SPaRCNet Github Repository</a> by <a href="https://bdsp.io" target="_none">Brain Data Science Platform (BDSP)</a></li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/36878708/" class="" target="_none">Development of Expert-Level Classification of Seizures and Rhythmic and Periodic Patterns During EEG Interpretation</a> by Jing J, Ge W, Hong S, Fernandes MB, Lin Z, Yang C et al., et al.</li></a>
          </li>
        </ul>
        <p><a href="/bdsp-sparcnet/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="stoic2021-training" class="dataset">
          <h3><a href="/stoic2021-training/">STOIC2021 Training</a></h3>
          <p><span class="label label-info tag link-tag">computed tomography</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">grand-challenge.org</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">SARS-CoV-2</span></p>
          <p>The STOIC project collected Computed Tomography (CT) images of 10,735 individuals suspected of being infected with SARS-COV-2 during the first wave of the pandemic in France, from March to April 2020. For each patient in the training set, the dataset contains binary labels for COVID-19 presence, based on RT-PCR test results, and COVID-19 severity, defined as intubation or death within one month from the acquisition of the CT scan. This S3 bucket contains the training sample of the STOIC dataset as used in the STOIC2021 challenge on grand-challenge.org.</p>
          <p><a href="/stoic2021-training/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.mdpi.com/2313-433X/8/12/320" class="" target="_none">How Well Do Self-Supervised Models Transfer to Medical Imaging?</a> by Anton J, Castelli L, Chan MF, Outthers M, Tang WH, Cheung V, et al.</li></a>
          </li>
          <li>
            <a href="https://pubs.rsna.org/doi/full/10.1148/radiol.2021210384" class="" target="_none">Study of Thoracic CT in COVID-19: The STOIC Project</a> by <a href="https://www.aphp.fr/" target="_none">Revel MP, Boussouar S, de Margerie-Mellon C, Saab I, Lapotre T, Mompoint D, et al.</a></li></a>
          </li>
          <li>
            <a href="https://stoic2021.grand-challenge.org/" class="" target="_none">STOIC2021 Challenge</a> by <a href="https://www.diagnijmegen.nl/" target="_none">Diagnostic Image Analysis Group, Radboudumc, Nijmegen</a></li></a>
          </li>
        </ul>
        <p><a href="/stoic2021-training/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="usgs_aqr" class="dataset">
          <h3><a href="/usgs_aqr/">Sentinel-2 ACOLITE-DSF Aquatic Reflectance for the Conterminous United States</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">water</span></p>
          <p>Aquatic reflectance produced with the dark spectrum fitting (DSF) algorithm as implemented in the Atmospheric Correction for OLI “lite” (ACOLITE) software (version 20221114.0). Aquatic reflectance is defined here as unitless water-leaving radiance reflectance and represents the ratio of water-leaving radiance (units of watts per square meter per steradian per nanometer) to downwelling irradiance (units of watts per square meter per nanometer) multiplied by pi.</p>
          <p><a href="/usgs_aqr/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://app.globus.org/file-manager?origin_id&#x3D;8fd8727f-c464-4e86-a5ed-c6db72848c02&amp;origin_path&#x3D;%2F" class="" target="_none">GLOBUS Access Point</a> by T.V. King, et al.</li></a>
          </li>
          <li>
            <a href="https://www.sciencebase.gov/catalog/item/640f612dd34e254fd352e1ed" class="" target="_none">Sentinel-2 ACOLITE-DSF Aquatic Reflectance for the Conterminous United States</a> by T.V. King, et al.</li></a>
          </li>
          <li>
            <a href="https://www.sciencebase.gov/catalog/item/640f612dd34e254fd352e1ed" class="" target="_none">tutorial.zip</a> by S.D. Ducar</li></a>
          </li>
        </ul>
        <p><a href="/usgs_aqr/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="sorel-20m" class="dataset">
          <h3><a href="/sorel-20m/">Sophos/ReversingLabs 20 Million malware detection dataset</a></h3>
          <p><span class="label label-info tag link-tag">cyber security</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>A dataset intended to support research on machine learning
techniques for detecting malware.  It includes metadata and EMBER-v2
features for approximately 10 million benign and 10 million malicious
Portable Executable files, with disarmed but otherwise complete
files for all malware samples.  All samples are labeled using Sophos
in-house labeling methods, have features extracted using the
EMBER-v2 feature set, well as metadata obtained via the pefile
python library, detection counts obtained via ReversingLabs
telemetry, and additional behavioral tags that indicate the rough
behavior of the sam...</p>
          <p><a href="/sorel-20m/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2012.07634" class="" target="_none">SOREL-20M: A Large Scale Benchmark Dataset for Malicious PE Detection
</a> by Richard Harang and Ethan M Rudd</li></a>
          </li>
          <li>
            <a href="https://github.com/sophos-ai/SOREL-20M" class="" target="_none">SOREL-20M dataset interface code</a> by Richard Harang and Ethan M Rudd</li></a>
          </li>
          <li>
            <a href="https://github.com/sophos-ai/SOREL-20M#quickstart" class="" target="_none">SOREL-20M quickstart</a> by Richard Harang</li></a>
          </li>
        </ul>
        <p><a href="/sorel-20m/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="human-microbiome-project" class="dataset">
          <h3><a href="/human-microbiome-project/">The Human Microbiome Project</a></h3>
          <p><span class="label label-info tag link-tag">amino acid</span><span class="label label-info tag link-tag">fasta</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">metagenomics</span><span class="label label-info tag link-tag">microbiome</span></p>
          <p>The NIH-funded Human Microbiome Project (HMP) is a collaborative effort of over 300 scientists from more than 80 organizations to comprehensively characterize the microbial communities inhabiting the human body and elucidate their role in human health and disease. To accomplish this task, microbial community samples were isolated from a cohort of 300 healthy adult human subjects at 18 specific sites within five regions of the body (oral cavity, airways, urogenital track, skin, and gut). Targeted sequencing of the 16S bacterial marker gene and/or whole metagenome shotgun sequencing was performe...</p>
          <p><a href="/human-microbiome-project/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.sciencedirect.com/science/article/pii/S2452231718300289" class="" target="_none">New microbe genomic variants in patients fecal community following surgical disruption of the upper human gastrointestinal tract</a> by Ranjit Kumar, Jayleen Grams, Daniel I. Chu, David K.Crossman, Richard Stahl, Peter Eipers, et al</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/nature06244" class="" target="_none">The Human Microbiome Project</a> by Peter J. Turnbaugh, Ruth E. Ley, Micah Hamady, Claire M. Fraser-Liggett, Rob Knight &amp; Jeffrey I. Gordon</li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/nature23889" class="" target="_none">Strains, functions and dynamics in the expanded Human Microbiome Project</a> by Jason Lloyd-Price, Anup Mahurkar, Gholamali Rahnavard, Jonathan Crabtree, Joshua Orvis, A. Brantley Hall, et al.</li></a>
          </li>
        </ul>
        <p><a href="/human-microbiome-project/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="hail-vep-pipeline" class="dataset">
          <h3><a href="/hail-vep-pipeline/">Variant Effect Predictor (VEP) and the Loss-Of-Function Transcript Effect Estimator (LOFTEE) Plugin</a></h3>
          <p><span class="label label-info tag link-tag">genome wide association study</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">loftee</span><span class="label label-info tag link-tag">vep</span></p>
          <p>VEP determines the effect of genetic variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. The European Bioinformatics Institute produces the VEP tool/db and releases updates every 1 - 6 months. The latest release contains 267 genomes from 232 species containing 5567663 protein coding genes. This dataset hosts the last 5 releases for human, rat, and zebrafish. Also, it hosts the required reference files for the Loss-Of-Function Transcript Effect Estimator (LOFTEE) plugin as it is commonly used with VEP.</p>
          <p><a href="/hail-vep-pipeline/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://hail.is/" class="" target="_none">Hail</a> by <a href="https://www.nealelab.is/" target="_none">Neale Lab</a></li></a>
          </li>
          <li>
            <a href="https://github.com/konradjk/loftee" class="" target="_none">Loss-Of-Function Transcript Effect Estimator (LOFTEE)</a> by <a href="https://github.com/konradjk" target="_none">Konrad Karczewski</a></li></a>
          </li>
          <li>
            <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" class="" target="_none">Variant Effect Predictor (VEP)</a> by <a href="http://uswest.ensembl.org/info/docs/tools/vep/index.html" target="_none">Ensembl</a></li></a>
          </li>
        </ul>
        <p><a href="/hail-vep-pipeline/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="vf-libraries" class="dataset">
          <h3><a href="/vf-libraries/">VirtualFlow Ligand Libraries</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">pharmaceutical</span><span class="label label-info tag link-tag">structural biology</span></p>
          <p>VirtualFlow Versions of Ligand Libraries in Ready-To-Dock Format</p>
          <p><a href="/vf-libraries/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2023.04.25.537981v1.abstract" class="" target="_none">VirtualFlow 2.0 - The Next Generation Drug Discovery Platform Enabling Adaptive Screens of 69 Billion Molecules</a> by Christoph Gorgulla, AkshatKumar Nigam, Matt Koop, Süleyman Selim Çınaroğlu, Christopher Secker, Mohammad Haddadnia, Abhishek Kumar, Yehor Malets, Alexander Hasson, Roni Levin-Konigsberg, Dmitry Radchenko, Aditya Kumar, Minko Gehev, Pierre-Yves Aquilanti, Henry Gabb, Amr Alhossary, Gerhard Wagner, Al, Yurii S. Moroz, Konstantin Fackeldey, Haribabu Arthanari</li></a>
          </li>
          <li>
            <a href="https://github.com/VirtualFlow/VFVS" class="" target="_none">VirtualFlow for Virtual Screening (VFVS) Module on GitHub</a> by Christoph Gorgulla</li></a>
          </li>
          <li>
            <a href="https://virtual-flow.org/tutorials" class="" target="_none">VirtualFlow tutorial</a> by <a href="http://christophgorgulla.info" target="_none">Christoph Gorgulla</a></li></a>
          </li>
        </ul>
        <p><a href="/vf-libraries/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="wis2-global-cache" class="dataset">
          <h3><a href="/wis2-global-cache/">WIS2 Global Cache on AWS</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">forecast</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">hydrology</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Global real-time Earth system data deemed by the World Meteorological Organisation (WMO) as essential for provision of services for the protection of life and property and for the well-being of all nations. Data is sourced from all WMO Member countries / territories and retained for 24-hours. Met Office and NOAA operate this Global Cache service curating and publishing the dataset on behalf of WMO.</p>
          <p><a href="/wis2-global-cache/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://wmo-im.github.io/wis2-guide/guide/wis2-guide-DRAFT.html" class="" target="_none">Guide to the WMO Information System (WMO-No. 1061), Volume II, WMO Information System 2.0</a> by World Meteorological Organisation</li></a>
          </li>
          <li>
            <a href="https://library.wmo.int/idurl/4/68731" class="" target="_none">Manual on the WMO Information System (WMO-No. 1060), Volume II, WMO Information System 2.0</a> by World Meteorological Organisation</li></a>
          </li>
          <li>
            <a href="https://vimeo.com/761382308" class="" target="_none">WIS 2.0 video for 19th World Meterological Congress</a> by WMO Secretariat</li></a>
          </li>
        </ul>
        <p><a href="/wis2-global-cache/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nrel-pds-windai" class="dataset">
          <h3><a href="/nrel-pds-windai/">Wind AI Bench</a></h3>
          <p><span class="label label-info tag link-tag">benchmark</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>This data lake contains multiple datasets related to fundamental problems 
in wind energy research. This includes data for wind plant power production 
for various layouts/wind flow scenarios, data for two- and three-dimensional 
flow around different wind turbine airfoils/blades, wind turbine noise 
production, among others. The purpose of these datasets is to establish a 
standard benchmark against which new AI/ML methods can be tested, compared, 
and deployed. Details regarding the generation and formatting of the data for 
each dataset is included in the metadata as well as example noteboo...</p>
          <p><a href="/nrel-pds-windai/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NREL/windAI_bench/tree/main/FLORIS_PLayGen" class="" target="_none">Wind AI Bench FLORIS PlayGen</a> by Dakota Ramos, Andrew Glaws</li></a>
          </li>
          <li>
            <a href="https://github.com/NREL/windAI_bench/tree/main/airfoil_2k" class="" target="_none">Airfoil 2k</a> by Dakota Ramos, Andrew Glaws</li></a>
          </li>
          <li>
            <a href="https://github.com/NREL/windAI_bench/tree/main/airfoil_9k" class="" target="_none">Airfoil 9k</a> by Dakota Ramos, Andrew Glaws</li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-windai/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="nara-1940-census" class="dataset">
          <h3><a href="/nara-1940-census/">1940 Census Population Schedules, Enumeration District Maps, and Enumeration District Descriptions</a></h3>
          <p><span class="label label-info tag link-tag">1940 census</span><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">census</span><span class="label label-info tag link-tag">demography</span><span class="label label-info tag link-tag">nara</span></p>
          <p>The 1940 Census population schedules were created by the Bureau of the Census in an attempt to enumerate every person living in the United States on April 1, 1940, although some persons were missed. The 1940 census population schedules were digitized by the National Archives and Records Administration (NARA) and released publicly on April 2, 2012.
The 1940 Census enumeration district maps contain maps of counties, cities, and other minor civil divisions that show enumeration districts, census tracts, and related boundaries and numbers used for each census. The coverage is nation wide and inclu...</p>
          <p><a href="/nara-1940-census/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://1940census.archives.gov" class="" target="_none">National Archives 1940 Census</a> by <a href="https://www.archives.gov" target="_none">National Archives and Records Administration</a></li></a>
          </li>
          <li>
            <a href="https://www.archives.gov/developer/1940-census" class="" target="_none">1940 Census on the AWS Registry of Open Data</a> by National Archives and Records Administration</li></a>
          </li>
        </ul>
        <p><a href="/nara-1940-census/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="nara-1950-census" class="dataset">
          <h3><a href="/nara-1950-census/">1950 Census Population Schedules, Enumeration District Maps, and Enumeration District Descriptions</a></h3>
          <p><span class="label label-info tag link-tag">1950 census</span><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">census</span><span class="label label-info tag link-tag">demography</span><span class="label label-info tag link-tag">nara</span></p>
          <p>The 1950 Census population schedules were created by the Bureau of the Census in an attempt to enumerate every person living in the United States on April 1, 1950, although some persons were missed. The 1950 census population schedules were digitized by the National Archives and Records Administration (NARA) and released publicly on April 1, 2022.
The 1950 Census enumeration district maps contain maps of counties, cities, and other minor civil divisions that show enumeration districts, census tracts, and related boundaries and numbers used for each census. The coverage is nation wide and inclu...</p>
          <p><a href="/nara-1950-census/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.archives.gov/developer/1950-census" class="" target="_none">1950 Census on the AWS Registry of Open Data</a> by National Archives and Records Administration</li></a>
          </li>
          <li>
            <a href="https://1950census.archives.gov" class="" target="_none">National Archives 1950 Census</a> by <a href="https://www.archives.gov" target="_none">National Archives and Records Administration</a></li></a>
          </li>
        </ul>
        <p><a href="/nara-1950-census/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="census-2010-pl94-nmf" class="dataset">
          <h3><a href="/census-2010-pl94-nmf/">2010 Census Production Settings Redistricting Data (P.L. 94-171) Demonstration Noisy Measurement File</a></h3>
          <p><span class="label label-info tag link-tag">census</span><span class="label label-info tag link-tag">differential privacy</span><span class="label label-info tag link-tag">disclosure avoidance</span><span class="label label-info tag link-tag">ethnicity</span><span class="label label-info tag link-tag">group quarters</span><span class="label label-info tag link-tag">hispanic</span><span class="label label-info tag link-tag">housing</span><span class="label label-info tag link-tag">housing units</span><span class="label label-info tag link-tag">latino</span><span class="label label-info tag link-tag">noisy measurements</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">race</span><span class="label label-info tag link-tag">redistricting</span><span class="label label-info tag link-tag">voting age</span></p>
          <p>The 2010 Census Production Settings Redistricting Data (P.L. 94-171) Demonstration Noisy Measurement File (2023-04-03) is an intermediate output of the 2020 Census Disclosure Avoidance System (DAS) TopDown Algorithm (TDA) (as described in Abowd, J. et al [2022] <a href="https://doi.org/10.1162/99608f92.529e3cb9">https://doi.org/10.1162/99608f92.529e3cb9</a> , and implemented in <a href="https://github.com/uscensusbureau/DAS_2020_Redistricting_Production_Code">https://github.com/uscensusbureau/DAS_2020_Redistricting_Production_Code</a>). The NMF was produced using the official “production settings,” the final set of algorithmic parameters and privacy-loss budget allocations, that were used to produce the 2020 Census Redistricting Data (P.L. 94-171) Summary File and the 2020 Census Demographic and Housing Characteristics File.
<br/>
<br/>
The NMF consists of the full set of privacy-protected statistical queries (counts of individuals or housing units with particular combinations of characteristics) of confidential 2010 Census data relating to the redistricting data portion of the 2010 Demonstration Data Products Suite – Redistricting and Demographic and Housing Characteristics File – Production Settings (2023-04-03). These statistical queries, called “noisy measurements” were produced under the zero-Concentrated Differential Privacy framework (Bun, M. and Steinke, T [2016] <a href="https://arxiv.org/abs/1605.02065">https://arxiv.org/abs/1605.02065</a>; see also Dwork C. and Roth, A. [2014] <a href="https://www.cis.upenn.edu/~aaroth/Papers/privacybook.pdf">https://www.cis.upenn.edu/~aaroth/Papers/privacybook.pdf</a>) implemented via the discrete Gaussian mechanism (Cannone C., et al., [2023] <a href="https://arxiv.org/abs/2004.00010">https://arxiv.org/abs/2004.00010</a>), which added positive or negative integer-valued noise to each of the resulting counts. The noisy measurements are an intermediate stage of the TDA prior to the post-processing the TDA then performs to ensure internal and hierarchical consistency within the resulting tables. The Census Bureau has released these 2010 Census demonstration data to enable data users to evaluate the expected impact of disclosure avoidance variability on 2020 Census data. The 2010 Census Production Settings Redistricting Data (P.L.94-171) Demonstration Noisy Measurement File (2023-04-03) has been cleared for public dissemination by the Census Bureau Disclosure Review Board (CBDRB-FY22-DSEP-004).
<br/>
<br/>
The data includes zero-Concentrated Differentially Private (zCDP) (Bun, M. and Steinke, T [2016]) noisy measurements, implemented via the discrete Gaussian mechanism. These are estimated counts of individuals and housing units included in the 2010 Census Edited File (CEF), which includes confidential data initially collected in the 2010 Census of Population and Housing. The noisy measurements included in this file were subsequently post-processed by the TopDown Algorithm (TDA) to produce the 2010 Census Production Settings Privacy-Protected Microdata File - Redistricting (P.L. 94-171) and Demographic and Housing Characteristics File (2023-04-03) (<a href="https://www2.census.gov/programs-surveys/decennial/2020/program-management/data-product-planning/2010-demonstration-data-products/04-Demonstration_Data_Products_Suite/2023-04-03/">https://www2.census.gov/programs-surveys/decennial/2020/program-management/data-product-planning/2010-demonstration-data-products/04-Demonstration_Data_Products_Suite/2023-04-03/</a>). As these 2010 Census demonstration data are intended to support study of the design and expected impacts of the 2020 Disclosure Avoidance System, the 2010 CEF records were pre-processed before application of the zCDP framework. This pre-processing converted the 2010 CEF records into the input-file format, response codes, and tabulation categories used for the 2020 Census, which differ in substantive ways from the format, response codes, and tabulation categories originally used for the 2010 Census.
<br/>
<br/>
The NMF provides estimates of counts of...</p>
          <p><a href="/census-2010-pl94-nmf/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.48550/arXiv.2203.16654" class="" target="_none">Geographic Spines in the 2020 Census Disclosure Avoidance System</a> by Abowd, J., Ashmead, R., Cumings-Menon, R., Garfinkel, S., Heineck, M., Heiss, C., Johns, R., Kifer, D., Leclerc, P., Machanavajjhala, A., Moran, B., Sexton, W., Spence, M., Zhuravlev, P.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1162/99608f92.529e3cb9" class="" target="_none">The 2020 Census Disclosure Avoidance System Topdown Algorithm</a> by Abowd, J., Ashmead, R., Cumings-Menon, R., Garfinkel, S., Heineck, M., Heiss, C., Johns, R., Kifer, D., Leclerc, P., Machanavajjhala, A., Moran, B., Sexton, W., Spence, M., Zhuravlev, P.</li></a>
          </li>
        </ul>
        <p><a href="/census-2010-pl94-nmf/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="census-2020-pl94-nmf" class="dataset">
          <h3><a href="/census-2020-pl94-nmf/">2020 Census Redistricting Data (P.L. 94-171) Noisy Measurement File</a></h3>
          <p><span class="label label-info tag link-tag">census</span><span class="label label-info tag link-tag">differential privacy</span><span class="label label-info tag link-tag">disclosure avoidance</span><span class="label label-info tag link-tag">ethnicity</span><span class="label label-info tag link-tag">group quarters</span><span class="label label-info tag link-tag">housing</span><span class="label label-info tag link-tag">housing units</span><span class="label label-info tag link-tag">noisy measurements</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">race</span><span class="label label-info tag link-tag">redistricting</span><span class="label label-info tag link-tag">voting age</span></p>
          <p>The 2020 Census Redistricting Data (P.L. 94-171) Noisy Measurement File (NMF) is an intermediate output of the 2020 Census Disclosure Avoidance System (DAS) TopDown Algorithm (TDA) (as described in Abowd, J. et al [2022] <a href="https://doi.org/10.1162/99608f92.529e3cb9">https://doi.org/10.1162/99608f92.529e3cb9</a>, and implemented in the <a href="https://github.com/uscensusbureau/DAS_2020_Redistricting_Production_Code">DAS 2020 Redistricting Production Code</a>). The NMF was generated using <a href="https://github.com/uscensusbureau/DAS_2020_Redistricting_Production_Code/blob/289ee463936a6f0efcf2e378abe410ec01d0e140/source/programs/engine/primitives.py#L183">the Census Bureau&#39;s implementation</a> of the <a href="https://arxiv.org/abs/2004.00010">Discrete Gaussian Mechanism</a>, calibrated to satisfy <a href="https://arxiv.org/abs/1605.02065">zero-Concentrated Differential Privacy</a> with <a href="https://dl.acm.org/doi/10.1145/1989323.1989345">bounded neighbors</a>.
<br/>
  <br/>
The NMF values, called <strong>noisy measurements</strong> are the output of applying the Discrete Gaussian Mechanism to counts from the 2020 Census Edited File (CEF). They are generally inconsistent with one another (for example, in a county composed of two tracts, the noisy measurement for the county&#39;s total population may not equal the sum of the noisy measurements of the two tracts&#39; total population), and frequently negative (especially when the population being measured was small), but are integer-valued. The NMF was later post-processed as part of the DAS code to take the form of microdata and to satisfy various constraints. The NMF documented here contains both the noisy measurements themselves as well as the data needed to represent the DAS constraints; thus, the NMF could be used to reproduce the steps taken by the DAS code to produce microdata from the noisy measurements by applying <a href="https://github.com/uscensusbureau/DAS_2020_Redistricting_Production_Code/blob/289ee463936a6f0efcf2e378abe410ec01d0e140/source/">the production code base</a>.
<br/>
<br/>
The 2020 Census Redistricting Data (P.L. 94-171) Noisy Measurement File includes zero-Concentrated Differentially Private (zCDP) (Bun, M. and Steinke, T [2016]) noisy measurements, implemented via the discrete Gaussian mechanism. These are estimated counts of individuals and housing units included in the 2020 Census Edited File (CEF), which includes confidential data initially collected in the 2020 Census of Population and Housing. The noisy measurements included in this file were subsequently post-processed by the TopDown Algorithm (TDA) to produce the <a href="https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html">2020 Census Redistricting Data (P.L. 94-171) Summary File</a>.
<br/>
<br/>
The NMF ...</p>
          <p><a href="/census-2020-pl94-nmf/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1162/99608f92.529e3cb9" class="" target="_none">The 2020 Census Disclosure Avoidance System Topdown Algorithm</a> by Abowd, J., Ashmead, R., Cumings-Menon, R., Garfinkel, S., Heineck, M., Heiss, C., Johns, R., Kifer, D., Leclerc, P., Machanavajjhala, A., Moran, B., Sexton, W., Spence, M., Zhuravlev, P.</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.48550/arXiv.2203.16654" class="" target="_none">Geographic Spines in the 2020 Census Disclosure Avoidance System</a> by Abowd, J., Ashmead, R., Cumings-Menon, R., Garfinkel, S., Heineck, M., Heiss, C., Johns, R., Kifer, D., Leclerc, P., Machanavajjhala, A., Moran, B., Sexton, W., Spence, M., Zhuravlev, P.</li></a>
          </li>
        </ul>
        <p><a href="/census-2020-pl94-nmf/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="4dnucleome" class="dataset">
          <h3><a href="/4dnucleome/">4D Nucleome (4DN)</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The goal of the National Institutes of Health (NIH) Common Fund’s 4D Nucleome (4DN) program
is to study the three-dimensional organization of the nucleus in space and time (the 4th dimension).
The nucleus of a cell contains DNA, the genetic “blueprint” that encodes all of the genes a living
organism uses to produce proteins needed to carry out life-sustaining cellular functions. Understanding
the conformation of the nuclear DNA and how it is maintained or changes in response to environmental
and cellular cues over time will provide insights into basic biology as well as aspects of human
health...</p>
          <p><a href="/4dnucleome/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://data.4dnucleome.org/help/user-guide/downloading-files" class="" target="_none">Finding and Downloading 4DN Data files</a> by <a href="data.4dnucleome.org" target="_none">4DN-DCIC</a></li></a>
          </li>
          <li>
            <a href="https://data.4dnucleome.org/tools/jupyterhub" class="" target="_none">Using jupyterhub on the 4DN data portal</a> by <a href="data.4dnucleome.org" target="_none">4DN-DCIC</a></li></a>
          </li>
        </ul>
        <p><a href="/4dnucleome/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="global-drought-flood-catalogue" class="dataset">
          <h3><a href="/global-drought-flood-catalogue/">A Global Drought and Flood Catalogue from 1950 to 2016</a></h3>
          <p><span class="label label-info tag link-tag">floods</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">near-surface air temperature</span><span class="label label-info tag link-tag">near-surface specific humidity</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">precipitation</span></p>
          <p>Hydrological extremes, in the form of droughts and floods, have impacts on a wide range of sectors including water availability, food security, and energy production, among others. Given continuing large impacts of droughts and floods and the expectation for significant regional changes projected in the future, there is an urgent need to provide estimates of past events and their future risk, globally. However, current estimates of hydrological extremes are not robust and accurate enough, due to lack of long-term data records, standardized methods for event identification, geographical inconsi...</p>
          <p><a href="/global-drought-flood-catalogue/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://prep-next.github.io/data/GDFC/products.html" class="" target="_none">Data introduction included in the GDFC</a> by <a href="http://hydro.iis.u-tokyo.ac.jp/~hexg/cv.html" target="_none">Xiaogang He</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1175/BAMS-D-18-0269.1" class="" target="_none">A Global Drought and Flood Catalogue from 1950 to 2016</a> by He, X., M. Pan, Z. Wei, E. F. Wood, and J. Sheffield</li></a>
          </li>
        </ul>
        <p><a href="/global-drought-flood-catalogue/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="africa-field-boundary-labels" class="dataset">
          <h3><a href="/africa-field-boundary-labels/">A region-wide, multi-year set of crop field boundary labels for Africa</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>Crop field boundaries digitized in Planet imagery collected across Africa between 2017 and 2023, developed by <a href="https://farmerline.co/">Farmerline</a>, <a href="https://spatialcollective.com/">Spatial Collective</a>, and the <a href="https://agroimpacts.info/">Agricultural Impacts Research Group</a> at <a href="https://www.clarku.edu/">Clark University</a>, with support from the <a href="https://lacunafund.org/">Lacuna Fund</a></p>
          <p><a href="/africa-field-boundary-labels/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/agroimpacts/lacunalabels/blob/main/docs/report/technical-report.pdf" class="" target="_none">Technical report on label develop and processing</a> by Wussah et al. (2023)</li></a>
          </li>
          <li>
            <a href="https://www.frontiersin.org/article/10.3389/frai.2021.744863" class="" target="_none">High resolution, annual maps of field boundaries for smallholder-dominated croplands at national scales</a> by Estes et al. (2022)</li></a>
          </li>
          <li>
            <a href="https://zenodo.org/records/11060871" class="" target="_none">A region-wide, multi-year set of crop field boundary labels for Africa</a> by Wussah et al. (2023)</li></a>
          </li>
          <li>
            <a href="https://github.com/agroimpacts/lacunalabels" class="" target="_none">Instructions on data access and label-making demonstration notebook</a> by <a href="https://github.com/ldemaz" target="_none">Lyndon Estes</a></li></a>
          </li>
          <li>
            <a href="http://www.sciencedirect.com/science/article/pii/S136481521630010X" class="" target="_none">A platform for crowdsourcing the creation of representative, accurate landcover maps</a> by Estes et al. (2016)</li></a>
          </li>
        </ul>
        <p><a href="/africa-field-boundary-labels/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="afsis" class="dataset">
          <h3><a href="/afsis/">Africa Soil Information Service (AfSIS) Soil Chemistry</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>This dataset contains soil infrared spectral data and paired soil property
reference measurements for georeferenced soil samples that were collected
through the Africa Soil Information Service (AfSIS) project, which lasted
from 2009 through 2018. In this release, we include data collected during
Phase I (2009-2013.) Georeferenced samples were collected from 19 countries
in Sub-Saharan African using a statistically sound sampling scheme,
and their soil properties were analyzed using <em>both</em> conventional soil
testing methods and spectral methods (infrared diffuse reflectance
spectroscopy). The two ...</p>
          <p><a href="/afsis/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/qedsoftware/afsis-soil-chem-tutorial/" class="" target="_none">AfSIS Soil Chemistry - Usage Tutorial</a> by <a href="https://qed.ai" target="_none">QED</a></li></a>
          </li>
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;Fb9R0CnPMkc" class="" target="_none">Goalkeepers 2018, Soil - The Big Data Beneath Your Feet</a> by <a href="https://qed.ai" target="_none">QED</a></li></a>
          </li>
        </ul>
        <p><a href="/afsis/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="intelinair_agriculture_vision" class="dataset">
          <h3><a href="/intelinair_agriculture_vision/">AgricultureVision</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>Agriculture-Vision aims to be a publicly available large-scale aerial agricultural image dataset that is high-resolution, multi-band, and with multiple types of patterns annotated by agronomy experts.  The original dataset affiliated with the 2020 CVPR paper includes 94,986 512x512images sampled from 3,432 farmlands with nine types of annotations: double plant, drydown, endrow, nutrient deficiency, planter skip, storm damage, water, waterway and weed cluster.  All of these patterns have substantial impacts on field conditions and the final yield. These farmland images were captured between 201...</p>
          <p><a href="/intelinair_agriculture_vision/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2001.01306" class="" target="_none">Agriculture-Vision: A Large Aerial Image Database for Agricultural Pattern Analysis</a> by Mang Tik Chiu, Xingqian Xu, Yunchao Wei, Zilong Huang, Alexander Schwing, Robert Brunner, Hrant Khachatrian, Hovnatan Karapetyan, Ivan Dozier, Greg Rose, David Wilson, Adrian Tudor, Naira Hovakimyan, Thomas S. Huang, Honghui Shi</li></a>
          </li>
          <li>
            <a href="https://www.agriculture-vision.com/" class="" target="_none">The 2nd International Workshop and Prize Challenge on Agriculture-Vision, Challenges &amp; Opportunities for Computer Vision in Agricutlure</a> by Humphrey Shi, Naira Hovakimyan, Jennifer Hobbs, Ed Delp, Melba Crawford, Zhen Li, David Clifford, Jim Yuan, Mang Tik Chiu, Xingqian Xu</li></a>
          </li>
        </ul>
        <p><a href="/intelinair_agriculture_vision/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="allen-synphys" class="dataset">
          <h3><a href="/allen-synphys/">Allen Institute for Brain Science - Synaptic Physiology Public Data Set</a></h3>
          <p><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>This is a large-scale survey that describes the physiology (strength, kinetics, and short term plasticity) of thousands of synapses from patch clamp experiments in mouse visual cortex and human middle temporal gyrus.  </p>
          <p><a href="/allen-synphys/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/AllenInstitute/aisynphys" class="" target="_none">aisynphys python package for accessing synaptic physiology data</a> by Campagnola L., Seeman S., et al.</li></a>
          </li>
          <li>
            <a href="https://pubmed.ncbi.nlm.nih.gov/35271334/" class="" target="_none">Local connectivity and synaptic dynamics in mouse and human neocortex</a> by Campagnola L., Seeman S., et al.</li></a>
          </li>
        </ul>
        <p><a href="/allen-synphys/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="allen-nd-ephys-compression" class="dataset">
          <h3><a href="/allen-nd-ephys-compression/">Allen Institute for Neural Dynamics - Extracellular Electrophysiology Compression Benchmark</a></h3>
          <p><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>Extracellular electrophysiology data is growing at a remarkable pace. This data, collected neuropixels probes by the Allen Institute and the International Brain Lab can be used to benchmark throughput rates and storage ratios of various data compression algorithms.</p>
          <p><a href="/allen-nd-ephys-compression/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://allenneuraldynamics.github.io/data.html#aind-benchmark-dataephys-compression" class="" target="_none">AIND ephys compression benchmark data</a> by <a href="https://twitter.com/AllenInstitute" target="_none">David Feng, Alessio Buccino</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2023.05.22.541700v2" class="" target="_none">Compression strategies for large-scale electrophysiology data</a> by <a href="http://neuraldynamics.alleninstitute.org/" target="_none">Alessio P. Buccino, Olivier Winter, David Bryant, David Feng, Karel Svoboda, Joshua H. Siegle.</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-nd-ephys-compression/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="apd_galaxysegmentation" class="dataset">
          <h3><a href="/apd_galaxysegmentation/">Astrophysics Division Galaxy Segmentation Benchmark Dataset</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">NASA SMD AI</span><span class="label label-info tag link-tag">segmentation</span></p>
          <p>Pan-STARSS imaging data and associated labels for galaxy segmentation into galactic centers, galactic bars, spiral arms and foreground stars derived from citizen scientist labels from the Galaxy Zoo: 3D project.</p>
          <p><a href="/apd_galaxysegmentation/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://iopscience.iop.org/article/10.3847/1538-4365/abb82b" class="" target="_none">Pan-STARRS Pixel Processing: Detrending, Warping, Stacking</a> by C. Z. Waters, E. A. Magnier, P. A. Price, K. C. Chambers, W. S. Burgett, P. W. Draper, H. A. Flewelling, K. W. Hodapp, M. E. Huber, R. Jedicke, N. Kaiser, R.-P. Kudritzki, R. H. Lupton, N. Metcalfe, A. Rest, W. E. Sweeney, J. L. Tonry, R. J. Wainscoat, and W. M. Wood-Vase</li></a>
          </li>
          <li>
            <a href="https://academic.oup.com/mnras/article/507/3/3923/6347355" class="" target="_none">Galaxy Zoo: 3D – crowdsourced bar, spiral, and foreground star masks for MaNGA target galaxies</a> by Karen L Masters, Coleman Krawczyk, Shoaib Shamsi, Alexander Todd, Daniel Finnegan, Matthew Bershady, Kevin Bundy, Brian Cherinka, Amelia Fraser-McKelvie, Dhanesh Krishnarao, Sandor Kruk, Richard R Lane, David Law, Chris Lintott, Michael Merrifield, Brooke Simmons, Anne-Marie Weijmans, Renbin Yan</li></a>
          </li>
        </ul>
        <p><a href="/apd_galaxysegmentation/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="meteo-france-models" class="dataset">
          <h3><a href="/meteo-france-models/">Atmospheric Models from Météo-France</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Global and high-resolution regional atmospheric models from Météo-France.<ul>
<li>ARPEGE World covers the entire world at a base horizontal resolution of 0.5° (~55km) between grid points, it predicts weather out up to 114 hours in the future.</li>
<li>ARPEGE Europe covers Europe and North-Africa at a base horizontal resolution of 0.1° (~11km) between grid points, it predicts weather out up to 114 hours in the future.</li>
<li>AROME France covers France at a base horizontal resolution of 0.025° (~2.5km) between grid points, it predicts weather out up to 42 hours in the future.</li>
<li>AROME France HD covers France and neighborhood at a base horizontal resolution of 0.01° (~1.5km) between grid points, it predicts weather out up to 42 hours in the future.</li>
</ul>
Dozens of atmospheric variables are avail...</p>
          <p><a href="/meteo-france-models/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://windy.com" class="" target="_none">Windy.com</a> by <a href="https://windy.com" target="_none">Windy</a></li></a>
          </li>
          <li>
            <a href="https://windguru.cz" class="" target="_none">Windguru.cz</a> by <a href="https://windguru.cz" target="_none">Windguru</a></li></a>
          </li>
        </ul>
        <p><a href="/meteo-france-models/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aurora_msds" class="dataset">
          <h3><a href="/aurora_msds/">Aurora Multi-Sensor Dataset</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">robotics</span><span class="label label-info tag link-tag">traffic</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">urban</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Aurora Multi-Sensor Dataset is an open, large-scale multi-sensor dataset with highly accurate localization ground truth, captured between January 2017 and February 2018 in the metropolitan area of Pittsburgh, PA, USA by Aurora (via Uber ATG) in collaboration with the University of Toronto. The de-identified dataset contains rich metadata, such as weather and semantic segmentation, and spans all four seasons, rain, snow, overcast and sunny days, different times of day, and a variety of traffic conditions.
<br/>
The Aurora Multi-Sensor Dataset contains data from a 64-beam Velodyne HDL-64E LiDAR sensor and seven 1920x1200-pixel resolution cameras including a forward-facing stereo pair and five wide-angle lenses covering a 360-degree view around the vehicle.
<br/>
This data can be used to develop and evaluate large-scale long-term approaches to autonomous vehicle localization. Its size and diversity make it suitable for a wide range of research areas such as 3D reconstruction, virtual tourism, HD map construction, and map compression, among others.
<br/>
The data was first presented at the International Conference on Intelligent Robots an</br/>...</p>
          <p><a href="/aurora_msds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ras.papercept.net/images/temp/IROS/files/0132.pdf" class="" target="_none">&quot;Pit30M: A benchmark for global localization in the age of self-driving cars&quot;, in 2020 IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS) (pp. 4477-4484)</a> by Martinez, J., Doubov, S., Fan, J., Bârsan, I. A., Wang, S., Máttyus, G., Urtasun, R.</li></a>
          </li>
          <li>
            <a href="https://studiolab.sagemaker.aws/import/github/pit30m/pit30m/blob/main/examples/tutorial_00_introduction.ipynb" class="" target="_none">Introduction to Visualizing Sensor Types (Jupyter notebook)</a> by Andrei Bârsan (note: Aurora makes no representations as to the accuracy or functionality of the tutorial)</li></a>
          </li>
        </ul>
        <p><a href="/aurora_msds/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="bhl-open-data" class="dataset">
          <h3><a href="/bhl-open-data/">Biodiversity Heritage Library Metadata and Page Images</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Biodiversity Heritage Library (BHL) is the world’s largest open access digital library for biodiversity literature and archives. BHL operates as a worldwide consortium of natural history, botanical, research, and national libraries working together to digitize the natural history literature held in their collections and make it freely available for open access.</p>
          <p><a href="/bhl-open-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.3897/biss.7.112436" class="" target="_none">Unearthing the Past for a Sustainable Future: Extracting and transforming data in the Biodiversity Heritage Library for climate action</a> by <a href="https://biss.pensoft.net/article/112436/list/9/" target="_none">Dearborn J, Lichtenberg M, Richard J, deVeer J, Trizna M, Mika K</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.25573/data.23523495.v1" class="" target="_none">AI models are getting better and better at reading handwriting, but how can we find handwritten text to begin with?</a> by <a href="https://datascience.si.edu/people/mike-trizna" target="_none">Mike Trizna, JJ Dearborn</a></li></a>
          </li>
          <li>
            <a href="https://github.com/gnames/bhlindex" class="" target="_none">BHLIndex</a> by <a href="https://globalnames.org/" target="_none">Global Names</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29853643[PMID]" class="" target="_none">Understanding BHL Through Metadata: Patterns of Bio-Diverse Knowledge Production</a> by <a href="https://lidiapv.com/" target="_none">Lidia Ponce de la Vega</a></li></a>
          </li>
          <li>
            <a href="https://biostor.org/" class="" target="_none">BioStor</a> by <a href="https://scholar.google.com/citations?user&#x3D;4Z5WABAAAAAJ&amp;hl&#x3D;en" target="_none">Roderic Page</a></li></a>
          </li>
        </ul>
        <p><a href="/bhl-open-data/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="bps_microscopy" class="dataset">
          <h3><a href="/bps_microscopy/">Biological and Physical Sciences (BPS) Microscopy Benchmark Training Dataset</a></h3>
          <p><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">GeneLab</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genetic maps</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span><span class="label label-info tag link-tag">NASA SMD AI</span></p>
          <p>Fluorescence microscopy images of individual nuclei from mouse fibroblast cells, irradiated with Fe particles or X-rays with fluorescent foci indicating 53BP1 positivity, a marker of DNA damage. These are maximum intensity projections of 9-layer microscopy Z-stacks.</p>
          <p><a href="/bps_microscopy/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://science.nasa.gov/files/atoms/files/NASA%20SMD%20AI%20Workshop%2021%20(spreads)%20(1).pdf" class="" target="_none">NASA SMD AI Workshop Report</a> by SMD Artificial Intelligence (AI) Initiative</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1667/RR15338.1" class="" target="_none">Dose, LET and Strain Dependence of Radiation-Induced 53BP1 Foci in 15 Mouse Strains Ex Vivo Introducing Novel DNA Damage Metrics</a> by Sébastien Penninckx, Egle Cekanaviciute, Charlotte Degorre, Elodie Guiet, Louise Viger, Stéphane Lucasb, Sylvain V. Costes</li></a>
          </li>
        </ul>
        <p><a href="/bps_microscopy/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="bps_rnaseq" class="dataset">
          <h3><a href="/bps_rnaseq/">Biological and Physical Sciences (BPS) RNA Sequencing Benchmark Training Dataset</a></h3>
          <p><span class="label label-info tag link-tag">gene expression</span><span class="label label-info tag link-tag">GeneLab</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genetic maps</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">NASA SMD AI</span><span class="label label-info tag link-tag">space biology</span></p>
          <p>RNA sequencing data from spaceflown and control mouse liver samples, sourced from NASA GeneLab and augmented with generative adversarial network.</p>
          <p><a href="/bps_rnaseq/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://science.nasa.gov/files/atoms/files/NASA%20SMD%20AI%20Workshop%2021%20(spreads)%20(1).pdf" class="" target="_none">NASA SMD AI Workshop Report</a> by SMD Artificial Intelligence (AI) Initiative</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1093/bioinformatics/btab035" class="" target="_none">Adversarial generation of gene expression data</a> by Ramon Viñas, Helena Andrés-Terré, Pietro Liò, Kevin Bryson</li></a>
          </li>
        </ul>
        <p><a href="/bps_rnaseq/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aws-covid19-lake" class="dataset">
          <h3><a href="/aws-covid19-lake/">COVID-19 Data Lake</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span></p>
          <p>A centralized repository of up-to-date and curated datasets on or related to the spread and characteristics of the novel corona virus (SARS-CoV-2) and its associated illness, COVID-19. Globally, there are several efforts underway to gather this data, and we are working with partners to make this crucial data freely available and keep it up-to-date. Hosted on the AWS cloud, we have seeded our curated data lake with COVID-19 case tracking data from Johns Hopkins and The New York Times, hospital bed availability from Definitive Healthcare, and over 45,000 research articles about COVID-19 and rela...</p>
          <p><a href="/aws-covid19-lake/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploring-the-public-aws-covid-19-data-lake/" class="aws-link" target="_none">Exploring the public AWS COVID-19 data lake</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://github.com/aws-samples/query-data-in-s3-with-amazon-athena-and-aws-sdk-for-dotnet" class="aws-link" target="_none">How to use SQL to query data in S3 Bucket with Amazon Athena and AWS SDK for .NET</a> by <a href="https://github.com/orgs/aws-samples/teams/aws-proserve-us-west-applications" target="_none">AWS ProServe US West Applications Team</a></li></a>
          </li>
          <li>
            <a href="https://us-east-2.console.aws.amazon.com/cloudformation/home?region&#x3D;us-east-2#/stacks/create/review?templateURL&#x3D;https://covid19-lake.s3.us-east-2.amazonaws.com/cfn/CovidLakeStack.template.json&amp;stackName&#x3D;CovidLakeStack" class="" target="_none">CloudFormation template for Glue Catalog table definitions</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://go.aws/covid-19-data-lake" class="aws-link" target="_none">Explore the COVID-19 data lake public S3 bucket</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/a-public-data-lake-for-analysis-of-covid-19-data/" class="aws-link" target="_none">A public data lake for analysis of COVID-19 data</a> by <a href="https://aws.amazon.com/big-data/datalakes-and-analytics/" target="_none">AWS Data Lake Team</a></li></a>
          </li>
        </ul>
        <p><a href="/aws-covid19-lake/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="cgci" class="dataset">
          <h3><a href="/cgci/">Cancer Genome Characterization Initiatives - Burkitt Lymphoma, HIV+ Cervical Cancer</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Cancer Genome Characterization Initiatives (CGCI) program supports cutting-edge genomics research of adult and pediatric cancers. CGCI investigators develop and apply advanced sequencing methods that examine genomes, exomes, and transcriptomes within various types of tumors. The program includes Burkitt Lymphoma Genome Sequencing Project (BLGSP) project and HIV+ Tumor Molecular Characterization Project - Cervical Cancer (HTMCP-CC) project.
The dataset contains open Clinical Supplement, Biospecimen Supplement, RNA-Seq Gene Expression Quantification, miRNA-Seq Isoform Expression Quantificati...</p>
          <p><a href="/cgci/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1182/blood-2018-09-871418" class="" target="_none">Genome-wide discovery of somatic coding and noncoding mutations in pediatric endemic and
sporadic Burkitt lymphoma
</a> by Grande B. M., Gerhard D. S., Jiang A., Griner N. B., Abramson J. S., Alexander T. B., et al.
</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/CGCI-BLGSP" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/cgci/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="cell-painting-image-collection" class="dataset">
          <h3><a href="/cell-painting-image-collection/">Cell Painting Image Collection</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">cell painting</span><span class="label label-info tag link-tag">fluorescence imaging</span><span class="label label-info tag link-tag">high-throughput imaging</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microscopy</span></p>
          <p>The Cell Painting Image Collection is a collection of freely
downloadable microscopy image sets. Cell Painting is an
unbiased high throughput imaging assay used to analyze
perturbations in cell models. In addition to the images
themselves, each set includes a description of the biological
application and some type of &quot;ground truth&quot; (expected results).
Researchers are encouraged to use these image sets as reference
points when developing, testing, and publishing new image
analysis algorithms for the life sciences. We hope that the
this data set will lead to a better understanding of w...</p>
          <p><a href="/cell-painting-image-collection/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/cytodata/cytodata-hackathon-2018/tree/master/cytodata-toolkit/" class="" target="_none">Example submission for the 2018 CytoData Hackathon (in R and Python)</a> by <a href="broadinstitute.org" target="_none">Juan Caicedo, Tim Becker</a></li></a>
          </li>
          <li>
            <a href="https://lifesciences-resources.awscloud.com/healthcare-life-sciences-aws-for-industries/accelerating-drug-discovery-with-high-throughput-cell-painting-on-aws" class="" target="_none">Accelerating Drug Discovery with high-throughput Cell Painting on AWS</a> by Chris Kaspar</li></a>
          </li>
        </ul>
        <p><a href="/cell-painting-image-collection/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="jhu-indexes" class="dataset">
          <h3><a href="/jhu-indexes/">Cloud Indexes for Bowtie, Kraken, HISAT, and Centrifuge</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Genomic tools use reference databases as indexes to operate quickly and efficiently, analogous to how web search engines use indexes for fast querying. Here, we aggregate genomic, pan-genomic and metagenomic indexes for analysis of sequencing data.</p>
          <p><a href="/jhu-indexes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.03.03.975219v4.full.pdf" class="" target="_none">Reducing reference bias using multiple population reference genomes</a> by Chen et al (2020)</li></a>
          </li>
          <li>
            <a href="https://benlangmead.github.io/aws-indexes/tutorial" class="" target="_none">Table of contents for tutorials for constituent tools</a> by <a href="https://benlangmead.github.io/aws-indexes" target="_none">Ben Langmead</a></li></a>
          </li>
        </ul>
        <p><a href="/jhu-indexes/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="open-lidar-data" class="dataset">
          <h3><a href="/open-lidar-data/">Collection of open nation-scale LiDAR datasets</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">survey</span></p>
          <p>The goal of this project is to collect all publicly available large scale LiDAR datasets and archive them in an
uniform fashion for easy access and use. Initial efforts to collect the datasets are concentrated on Europe and will
be in future expanded to USA and other regions, striving for global coverage. Every dataset includes files in original
data format and translated to COPC format. For faster browsing, we include an overview file that includes a small
subset of data points from every dataset file in a single COPC file.</p>
          <p><a href="/open-lidar-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://hub.flai.ai/" class="" target="_none">FlaiHub - web application for accessing, browsing and processing open LiDAR datasets</a> by <a href="https://flai.ai/" target="_none">Flai</a></li></a>
          </li>
          <li>
            <a href="https://github.com/flai-ai/open-lidar-data/blob/main/tutorials%2Fhow-to-download-data-with-flai-hub.md" class="" target="_none">How to download data with FlaiHub</a> by <a href="https://flai.ai/" target="_none">Flai</a></li></a>
          </li>
        </ul>
        <p><a href="/open-lidar-data/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="visym-cap" class="dataset">
          <h3><a href="/visym-cap/">Consented Activities of People</a></h3>
          <p><span class="label label-info tag link-tag">activity detection</span><span class="label label-info tag link-tag">activity recognition</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">privacy</span><span class="label label-info tag link-tag">video</span></p>
          <p>The Consented Activities of People (CAP) dataset is a fine grained activity dataset for visual AI research curated using the <a href="https://www.visym.com/collector/">Visym Collector</a> platform.</p>
          <p><a href="/visym-cap/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/visym/collector" class="" target="_none">Visym Collector</a> by Visym Labs &amp; Systems &amp; Technology Research</li></a>
          </li>
          <li>
            <a href="https://openfad.nist.gov" class="" target="_none">OpenFAD - Open Fine Grained Activity Detection Challenge</a> by Visym Labs &amp; NIST</li></a>
          </li>
        </ul>
        <p><a href="/visym-cap/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="copernicus-dem" class="dataset">
          <h3><a href="/copernicus-dem/">Copernicus Digital Elevation Model (DEM)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Copernicus DEM is a Digital Surface Model (DSM) which represents the surface of the Earth including buildings, infrastructure and vegetation. We provide two instances of Copernicus DEM named GLO-30 Public and GLO-90. GLO-90 provides worldwide coverage at 90 meters. GLO-30 Public provides limited worldwide coverage at 30 meters because a small subset of tiles covering specific countries are not yet released to the public by the Copernicus Programme. Note that in both cases ocean areas do not have tiles, there one can assume height values equal to zero. Data is provided as Cloud Optimized Ge...</p>
          <p><a href="/copernicus-dem/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://apps.sentinel-hub.com/eo-browser/" class="" target="_none">EO Browser</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
          <li>
            <a href="http://www.sentinel-hub.com/" class="" target="_none">Sentinel Hub WMS/WMTS/WCS Service and Process API</a> by <a href="http://www.sinergise.com/" target="_none">Sinergise</a></li></a>
          </li>
        </ul>
        <p><a href="/copernicus-dem/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="covers-br" class="dataset">
          <h3><a href="/covers-br/">CoversBR</a></h3>
          <p><span class="label label-info tag link-tag">copyright monitoring</span><span class="label label-info tag link-tag">cover song identification</span><span class="label label-info tag link-tag">live song identification</span><span class="label label-info tag link-tag">music</span><span class="label label-info tag link-tag">music features dataset</span><span class="label label-info tag link-tag">music information retrieval</span><span class="label label-info tag link-tag">music recognition</span></p>
          <p>CoversBR is the first large audio database with, predominantly, Brazilian music for the tasks of Covers Song
Identification (CSI) and Live Song Identifications (LSI). Due to copyright restrictions audios of
the songs cannot be made available, however metadata and files of features have public access. Audio
streamings captured from radio and TV channels for the live song identification task will be made public.
CoversBR is composed of metadata and features extracted from 102298 songs, distributed in 26366
groups of covers/versions, with an average of 3.88 versions per group. The entire collecti...</p>
          <p><a href="/covers-br/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/aws-music-genre-classification" class="aws-link" target="_none">Getting started with Cover Song Identification and Genre classification</a> by Gabriel Bella Martini</li></a>
          </li>
          <li>
            <a href="https://github.com/SPLab-IT/CoversBR#using-the-dataset---tutorial" class="" target="_none">Using the (CoversBR) dataset</a> by <a href="https://github.com/SPLab-IT/CoversBR" target="_none">Dirceu Silva, Atila Xavier, Edgard Moraes, Marco Grivet and Fernando Perdigão</a></li></a>
          </li>
        </ul>
        <p><a href="/covers-br/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="us-hiring-rates-pandemic" class="dataset">
          <h3><a href="/us-hiring-rates-pandemic/">Covid Job Impacts - US Hiring Data Since March 1 2020</a></h3>
          <p><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">economics</span><span class="label label-info tag link-tag">financial markets</span><span class="label label-info tag link-tag">hiring</span><span class="label label-info tag link-tag">market data</span></p>
          <p>This dataset provides daily updates on the volume of US job listings filtered by geography industry job family and role; normalized to pre-covid levels.These data files feed the business intelligence visuals at covidjobimpacts.greenwich.hr, a public-facing site hosted by Greenwich.HR and OneModel Inc.
Data is derived from online job listings tracked continuously, calculated daily and published nightly.  On average data from 70% of all new US jobs are captured,
and the dataset currently contains data from 3.3 million hiring organizations.Data for each filter segment is represented as the 7-day ...</p>
          <p><a href="/us-hiring-rates-pandemic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://greenwichhr-covidjobimpacts.s3.us-east-2.amazonaws.com/ghr_data_specs_covid_public.pdf" class="" target="_none">Documentation of dataset schemas</a> by <a href="https://www.greenwich.hr" target="_none">Greenwich.HR</a></li></a>
          </li>
          <li>
            <a href="https://covidjobimpacts.greenwich.hr" class="" target="_none">CovidJobImpats.Greenwich.HR - online visualization of daily hiring data and weekly unemployment data including links to recorded discussions using the data</a> by Greenwich.HR and OneModel Inc.</li></a>
          </li>
        </ul>
        <p><a href="/us-hiring-rates-pandemic/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="cryoet-data-portal" class="dataset">
          <h3><a href="/cryoet-data-portal/">CryoET Data Portal</a></h3>
          <p><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">cryo electron tomography</span><span class="label label-info tag link-tag">czi</span><span class="label label-info tag link-tag">electron tomography</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">segmentation</span><span class="label label-info tag link-tag">structural biology</span></p>
          <p>Cryo-electron tomography (cryoET) is a powerful technique for visualizing 3D structures of cellular macromolecules at near atomic resolution in their native environment. Observing the inner workings of cells in context enables better understanding about the function of healthy cells and the changes associated with disease. However, the analysis of cryoET data remains a significant bottleneck, particularly the annotation of macromolecules within a set of tomograms, which often requires a laborious and time-consuming process of manual labelling that can take months to complete. Given the current...</p>
          <p><a href="/cryoet-data-portal/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://chanzuckerberg.github.io/cryoet-data-portal/python-api.html" class="" target="_none">Python API for CryoET Data Portal</a> by <a href="https://github.com/chanzuckerberg/" target="_none">CZI</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2024.07.13.603322v1" class="" target="_none">Cryo-ET reveals the in situ architecture of the polar tube invasion apparatus from microsporidian parasites</a> by <a href="https://scholar.google.com/scholar?q&#x3D;Mahrukh+Usmani" target="_none">Mahrukh Usmani, et al.</a></li></a>
          </li>
          <li>
            <a href="https://chanzuckerberg.github.io/cryoet-data-portal/cryoet_data_portal_docsite_examples.html" class="" target="_none">Examples of working with the portal API</a> by <a href="https://github.com/chanzuckerberg/" target="_none">CZI</a></li></a>
          </li>
          <li>
            <a href="https://chanzuckerberg.github.io/cryoet-data-portal/cryoet_data_portal_docsite_aws.html" class="" target="_none">Downloading data via the AWS CLI</a> by <a href="https://github.com/chanzuckerberg/" target="_none">CZI</a></li></a>
          </li>
          <li>
            <a href="https://chanzuckerberg.github.io/cryoet-data-portal/tutorial_sample_boundaries.html" class="" target="_none">Predicting sample boundaries</a> by <a href="https://github.com/chanzuckerberg/" target="_none">CZI</a></li></a>
          </li>
        </ul>
        <p><a href="/cryoet-data-portal/#usageexamples">See 5 usage examples &rarr;</a></p>
        </div>
        <div id="dnastack-covid-19-sra-data" class="dataset">
          <h3><a href="/dnastack-covid-19-sra-data/">DNAStack COVID19 SRA Data</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">fasta</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">long read sequencing</span><span class="label label-info tag link-tag">SARS-CoV-2</span><span class="label label-info tag link-tag">vcf</span><span class="label label-info tag link-tag">virus</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The <a href="https://www.ncbi.nlm.nih.gov/sra/">Sequence Read Archive (SRA)</a> is the primary archive of high-throughput sequencing data, hosted by the National Institutes of Health (NIH). The SRA represents the largest publicly available repository of SARS-CoV-2 sequencing data. This dataset was created by DNAstack using SARS-CoV-2 sequencing data sourced from the SRA. Where possible, raw sequence data were processed by DNAstack through a unified bioinformatics pipeline to produce genome assemblies and variant calls. The use of a standardized workflow to produce this harmonized dataset allows public data generated using different methodol...</p>
          <p><a href="/dnastack-covid-19-sra-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/DNAstack/dnastack-open-data/tree/master/tutorials/assign_lineage" class="" target="_none">Viral lineage assignment</a> by Heather Ward</li></a>
          </li>
          <li>
            <a href="https://viral.ai/collections/ncbi-sra" class="" target="_none">Viral AI</a> by DNAstack</li></a>
          </li>
        </ul>
        <p><a href="/dnastack-covid-19-sra-data/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="digitalcorpora" class="dataset">
          <h3><a href="/digitalcorpora/">DigitalCorpora</a></h3>
          <p><span class="label label-info tag link-tag">computer forensics</span><span class="label label-info tag link-tag">computer security</span><span class="label label-info tag link-tag">CSI</span><span class="label label-info tag link-tag">cyber security</span><span class="label label-info tag link-tag">digital forensics</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">information retrieval</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">intrusion detection</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">machine translation</span><span class="label label-info tag link-tag">text analysis</span></p>
          <p>Disk images, memory dumps, network packet captures, and files for use in digital forensics research and education. All of this information is accessible through the digitalcorpora.org website, and made available at s3://digitalcorpora/. Some of these datasets implement scenarios that were performed by students, faculty, and others acting <em>in persona</em>. As such, the information is synthetic and may be used without prior authorization or IRB approval. Details of these datasets can be found at <a href="http://www.simson.net/clips/academic/2009...</p>
          <p><a href="/digitalcorpora/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://simson.net/clips/academic/2011.ADFSL.Corpora.pdf" class="" target="_none">Creating Realistic Corpora for Forensic and Security Education</a> by <a href="https://simson.net/" target="_none">Woods, K., Christopher Lee, Simson Garfinkel, David Dittrich, Adam Russel, Kris Kearton</a></li></a>
          </li>
          <li>
            <a href="http://www.simson.net/clips/academic/2009.DFRWS.Corpora.pdf" class="" target="_none">Bringing Science to Digital Forensics with Standardized Forensic Corpora</a> by <a href="https://simson.net/" target="_none">Garfinkel, Farrell, Roussev and Dinolt</a></li></a>
          </li>
        </ul>
        <p><a href="/digitalcorpora/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="wrf-alaska-snap" class="dataset">
          <h3><a href="/wrf-alaska-snap/">Downscaled Climate Data for Alaska (v1.1, August 2023)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>This dataset contains historical and projected dynamically downscaled climate data for the State of Alaska and surrounding regions at 20km spatial resolution and hourly temporal resolution. Select variables are also summarized into daily resolutions. This data was produced using the Weather Research and Forecasting (WRF) model (Version 3.5). We downscaled both ERA-Interim historical reanalysis data (1979-2015) and both historical and projected runs from 2 GCM’s from the Coupled Model Inter-comparison Project 5 (CMIP5): GFDL-CM3 and NCAR-CCSM4 (historical run: 1970-2005 and RCP 8.5: 2006-2100)....</p>
          <p><a href="/wrf-alaska-snap/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1175/JAMC-D-15-0153.1" class="" target="_none">Dynamical Downscaling of ERA-Interim Temperature and Precipitation for Alaska</a> by Peter A. Bieniek, Uma S. Bhatt, John E. Walsh, T. Scott Rupp, Jing Zhang, Jeremy R. Krieger, and Rick Lader</li></a>
          </li>
          <li>
            <a href="https://www.mdpi.com/2073-4433/13/11/1791/pdf" class="" target="_none">Atmospheric Circulation Drivers of Extreme High Water Level Events at Foggy Island Bay, Alaska</a> by Peter A. Bieniek, Li Erikson, and Jeremy Kasper</li></a>
          </li>
        </ul>
        <p><a href="/wrf-alaska-snap/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="emory-breast-imaging-dataset-embed" class="dataset">
          <h3><a href="/emory-breast-imaging-dataset-embed/">EMory BrEast Imaging Dataset (EMBED)</a></h3>
          <p><span class="label label-info tag link-tag">bias</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">mammography</span><span class="label label-info tag link-tag">x-ray</span></p>
          <p>EMBED is a racially diverse mammography dataset containing 3.4M screening and diagnostic images from 110,000 patients collected from 2013-2020, with an
equal representation of black and white women. The dataset is comprised of 2D, synthetic 2D (C-view), and 3D (digital breast tomosynthesis, i.e. DBT)
images. It contains 60,000 annotated lesions linked to structured imaging descriptors and ground truth pathologic outcomes grouped into six severity
classes. This release represents 20% of the total 2D and C-view dataset and is available for research use. DBT, US, and MRI exams will be added at a
...</p>
          <p><a href="/emory-breast-imaging-dataset-embed/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubs.rsna.org/doi/epdf/10.1148/ryai.220047" class="" target="_none">The EMory BrEast imaging Dataset (EMBED): A Racially Diverse, Granular Dataset of 3.4M Screening and Diagnostic Mammograms</a> by <a href="https://scholar.google.com/citations?hl&#x3D;en&amp;user&#x3D;_VYSm7YAAAAJ" target="_none">Jiwoong J. Jeong, Brianna L. Vey, Ananth Reddy, Thomas Kim, Thiago Santos, Ramon Correa, Raman Dutt, Marina Mosunjac, Gabriela Oprea-Ilies, Geoffrey Smith, Minjae Woo, Christopher R. McAdams, Mary S. Newell, Imon Banerjee, Judy Gichoya, Hari Trivedi</a></li></a>
          </li>
          <li>
            <a href="https://github.com/Emory-HITI/EMBED_Open_Data/blob/main/Sample_Notebook.ipynb" class="" target="_none">Sample Notebook</a> by Emory-HITI</li></a>
          </li>
        </ul>
        <p><a href="/emory-breast-imaging-dataset-embed/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="mrkr" class="dataset">
          <h3><a href="/mrkr/">Emory Knee Radiograph (MRKR) dataset</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medical image computing</span><span class="label label-info tag link-tag">medical imaging</span><span class="label label-info tag link-tag">radiology</span><span class="label label-info tag link-tag">x-ray</span></p>
          <p>The Emory Knee Radiograph (MRKR) dataset is a large, demographically diverse collection of
503,261 knee radiographs from 83,011 patients, 40% of which are African American. This dataset
provides imaging data in DICOM format along with detailed clinical information, including patient-
reported pain scores, diagnostic codes, and procedural codes, which are not commonly available in
similar datasets. The MRKR dataset also features imaging metadata such as image laterality, view type,
and presence of hardware, enhancing its value for research and model development. MRKR addresses
significant gaps ...</p>
          <p><a href="/mrkr/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2411.00866" class="" target="_none">Emory Knee Radiograph Dataset</a> by Brandon Price, Jason Adleberg, Kaesha Thomas, Zach Zaiman, Aawez Mansuri, Beatrice Brown-Mulry, Chima Okecheukwu, Judy Gichoya, Hari Trivedi.</li></a>
          </li>
          <li>
            <a href="https://github.com/Emory-HITI/MRKR/blob/main/example_notebook.ipynb" class="" target="_none">Example Notebook</a> by Emory-HITI</li></a>
          </li>
        </ul>
        <p><a href="/mrkr/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ford-multi-av-seasonal" class="dataset">
          <h3><a href="/ford-multi-av-seasonal/">Ford Multi-AV Seasonal Dataset</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">robotics</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">urban</span><span class="label label-info tag link-tag">weather</span></p>
          <p>This research presents a challenging multi-agent seasonal dataset collected by a fleet of Ford autonomous vehicles at different days and times during 2017-18. The vehicles The vehicles were manually driven on an average route of 66 km in Michigan that included a mix of driving scenarios like the Detroit Airport, freeways, city-centres, university campus and suburban neighbourhood, etc. Each vehicle used in this data collection  is a Ford Fusion outfitted with an Applanix POS-LV inertial measurement unit (IMU), four HDL-32E Velodyne 3D-lidar scanners, 6 Point Grey 1.3 MP Cameras arranged on the...</p>
          <p><a href="/ford-multi-av-seasonal/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/amazon-eks-autonomous-driving-data-service" class="aws-link" target="_none">Autonomous Driving Data Service (ADDS)</a> by Ajay Vohra, Amazon</li></a>
          </li>
          <li>
            <a href="https://github.com/Ford/AVData" class="" target="_none">Ford AV Dataset Tutorial</a> by Ford Motor Company</li></a>
          </li>
        </ul>
        <p><a href="/ford-multi-av-seasonal/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="gatk-sv-data" class="dataset">
          <h3><a href="/gatk-sv-data/">GATK Structural Variation (SV) Data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cromwell</span><span class="label label-info tag link-tag">gatk-sv</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">structural variation</span></p>
          <p>This dataset holds the data needed to run a <a href="https://github.com/broadinstitute/gatk-sv">structural variation discovery pipeline</a>
for Illumina short-read whole-genome sequencing (WGS) data in AWS.</p>
          <p><a href="/gatk-sv-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/industries/using-structural-variant-analysis-on-aws-with-amazon-fsx-for-lustre-in-novel-therapeutic-discovery/" class="aws-link" target="_none">Structural Variant Analysis on AWS with Amazon FSx for Lustre</a> by Goldfinch Bio and Loka Inc.</li></a>
          </li>
          <li>
            <a href="https://github.com/goldfinchbio/aws-gatk-sv" class="" target="_none">AWS Setup &amp; Execution</a> by Goldfinch Bio and Loka Inc.</li></a>
          </li>
        </ul>
        <p><a href="/gatk-sv-data/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="mcrpc" class="dataset">
          <h3><a href="/mcrpc/">Genomic Characterization of Metastatic Castration Resistant Prostate Cancer</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Biopsies of castration resistant prostate cancer metastases were subjected to whole genome sequencing (WGS), along with RNA-sequencing (RNA-Seq). The overarching goal of the study is to illuminate molecular mechanisms of acquired resistance to therapeutic agents, and particularly androgen signaling inhibitors, in the treatment of metastatic castration resistant prostate cancer (mCRPC). This study is made available on AWS via the NIH STRIDES Initiative.</p>
          <p><a href="/mcrpc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1002/pros.24450" class="" target="_none">Genomic characterization of metastatic castration-resistant prostate cancer patients undergoing PSMA radioligand therapy: A single-center experience</a> by Swayamjeet Satapathy, Chandan K Das, et al.</li></a>
          </li>
        </ul>
        <p><a href="/mcrpc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="bdsp-harvard-eeg" class="dataset">
          <h3><a href="/bdsp-harvard-eeg/">Harvard Electroencephalography Database</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>The Harvard EEG Database will encompass data gathered from four hospitals affiliated with Harvard University:Massachusetts General Hospital (MGH), Brigham and Women&#39;s Hospital (BWH), Beth Israel Deaconess Medical Center (BIDMC), and Boston Children&#39;s Hospital (BCH).</p>
          <p><a href="/bdsp-harvard-eeg/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/bdsp-core/Harvard-EEG-Database-Tools" class="" target="_none">Harvard-EEG-Database-Tools</a> by <a href="https://bdsp.io" target="_none">Brain Data Science Platform (BDSP)</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.60508/7651-3337" class="" target="_none">Harvard Electroencephalography Database</a> by Zafar, S., Loddenkemper, T., Lee, J. W., Cole, A., Goldenholz, D., Peters, J., et al.</li></a>
          </li>
        </ul>
        <p><a href="/bdsp-harvard-eeg/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="bdsp-heedb" class="dataset">
          <h3><a href="/bdsp-heedb/">Harvard-Emory ECG Database</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">neurophysiology</span><span class="label label-info tag link-tag">neuroscience</span></p>
          <p>The Harvard-Emory ECG database (HEEDB) is a large collection of 12-lead electrocardiography (ECG) recordings, prepared through a collaboration between Harvard University and Emory University investigators.</p>
          <p><a href="/bdsp-heedb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.13026/gjvw-1m31" class="" target="_none">WFDB Software Package</a> by <a href="https://www.physionet.org" target="_none">Moody, G., Pollard, T., &amp; Moody, B.</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.60508/g072-7n95" class="" target="_none">Harvard Electroencephalography Database</a> by Moura Junior, V.; Reyna, M.; Hong, S.; Gupta, A.; Ghanta, M.; Sameni, R., et al.</li></a>
          </li>
        </ul>
        <p><a href="/bdsp-heedb/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="hecatomb" class="dataset">
          <h3><a href="/hecatomb/">Hecatomb Databases</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">metagenomics</span><span class="label label-info tag link-tag">virus</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Preprocessed databases for use with the Hecatomb pipeline for viral and phage sequence annotation.</p>
          <p><a href="/hecatomb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.3390/pathogens10070787" class="" target="_none">No Evidence Known Viruses Play a Role in the Pathogenesis of Onchocerciasis-Associated Epilepsy. An Explorative Metagenomic Case-Control Study</a> by <a href="https://bioinf.cc" target="_none">Michael Roach,Adrian Cantu,Melissa Krizia Vieri,Matthew Cotten,Paul Kellam,My Phan,Lia van der Hoek,Michel Mandro,Floribert Tepage,Germain Mambandu,Gisele Musinya,Anne Laudisoit,Robert Colebunders,Robert Edwards, John L. Mokili</a></li></a>
          </li>
          <li>
            <a href="https://hecatomb.readthedocs.io/en/latest/tutorialPt1/" class="" target="_none">The Hecatomb Tutorial</a> by <a href="https://bioinf.cc" target="_none">Michael Roach</a></li></a>
          </li>
        </ul>
        <p><a href="/hecatomb/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="kaiju-indexes" class="dataset">
          <h3><a href="/kaiju-indexes/">Indexes for Kaiju</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">metagenomics</span><span class="label label-info tag link-tag">microbiome</span><span class="label label-info tag link-tag">reference index</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This dataset comprises pre-built indexes for the bioinformatics software Kaiju, which is used for taxonomic classification of metagenomic sequencing data. Various indexes for different source reference databases are available.</p>
          <p><a href="/kaiju-indexes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/bioinformatics-centre/kaiju/blob/master/Quickstart.md" class="" target="_none">Quickstart Tutorial for downloading the index files and running Kaiju.</a> by <a href="https://github.com/pmenzel" target="_none">Peter Menzel</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/ncomms11257" class="" target="_none">Fast and sensitive taxonomic classification for metagenomics with Kaiju</a> by Peter Menzel et al (2016)</li></a>
          </li>
        </ul>
        <p><a href="/kaiju-indexes/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="luad-eagle" class="dataset">
          <h3><a href="/luad-eagle/">Integrative Analysis of Lung Adenocarcinoma in Environment and Genetics Lung cancer Etiology (Phase 2)</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">epigenomics</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole exome sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>We performed whole genome sequencing and whole exome sequencing of 31 lung adenocarcinoma (LUAD) samples from the Environment And Genetics in Lung cancer Etiology (EAGLE) study. The EAGLE study is made available on AWS via the NIH STRIDES Initiative (<a href="https://aws.amazon.com/blogs/publicsector/aws-and-national-institutes-of-health-collaborate-to-accelerate-discoveries-with-strides-initiative/">https://aws.amazon.com/blogs/publicsector/aws-and-national-institutes-of-health-collaborate-to-accelerate-discoveries-with-strides-initiative/</a>).</p>
          <p><a href="/luad-eagle/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://dceg.cancer.gov/research/who-we-study/cancer-cases-controls/eagle-study#:~:text&#x3D;Environment%20And%20Genetics%20in%20Lung%20cancer%20Etiology%20(EAGLE)%20was%20developed,%2C%20clinical%2C%20and%20behavioral%20data." class="" target="_none">EAGLE - Environment And Genetics in Lung cancer Etiology</a> by <a href="https://dceg.cancer.gov/" target="_none">National Cancer Institute Division of Cancer Epidemiology and Genetics</a></li></a>
          </li>
        </ul>
        <p><a href="/luad-eagle/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="jwst" class="dataset">
          <h3><a href="/jwst/">James Webb Space Telescope (JWST)</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The James Webb Space Telescope (JWST) is the world&#39;s next flagship infrared observatory led by NASA with its partners, ESA (European Space Agency), and CSA (Canadian Space Agency). JWST offers scientists the opportunity to observe galaxy evolution, the formation of stars and planets, exoplanetary systems, and our own solar system, in ways never before possible.</p>
          <p><a href="/jwst/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://outerspace.stsci.edu/display/MASTDOCS/JWST+Archive+Manual" class="" target="_none">MAST Archive Manual for JWST Data</a> by MAST, STScI</li></a>
          </li>
          <li>
            <a href="https://jwst-docs.stsci.edu/getting-started-with-jwst-data" class="" target="_none">Getting Started with JWST Data (JDox)</a> by STScI</li></a>
          </li>
        </ul>
        <p><a href="/jwst/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="leopard" class="dataset">
          <h3><a href="/leopard/">LEarning biOchemical Prostate cAncer Recurrence from histopathology sliDes challenge (LEOPARD) Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">computational pathology</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">grand-challenge.org</span><span class="label label-info tag link-tag">histopathology</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>&quot;This dataset contains the all data for the <a href="https://leopard.grand-challenge.org/">LEarning biOchemical Prostate cAncer Recurrence from histopathology sliDes challenge or LEOPARD</a>.Prostate cancer, impacting 1.4 million men annually, is a prevalent malignancy (<a href="https://acsjournals.onlinelibrary.wiley.com/doi/10.3322/caac.21660">H. Sung et al., 2021</a>). A substantial number of these individuals undergo prostatectomy as the primary curative treatment. The efficacy of this surgery is assessed, in part, by monitoring the concentration of prostate-specific antigen (PSA) in the bloodstream. While the role of PSA in prostate cancer screening is debatable (<a href="https://jamanetwork.com/journals/jama/fullarticle/2680553">W. F. Clark et al., 2018;</a> <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ijc.31102">E. A. M. Heijnsdijk et al., 2018</a>), it serves as a valuable biomarker for postprostatectomy follow-up in patients. Following successful surgery, PSA concentration is typically undetectable (&lt;0.1 ng/mL) within 4-6 weeks (<a href="https://wchh.onlinelibrary.wiley.com/doi/abs/10.1002/tre.407">S. S. Goonewardene et al., 2014</a>). However, approximately 30% of patients experience biochemical recurrence, signifying the resurgence of prostate cancer cells. This recurrence serves as a prognostic indicator for progression to clinical metastases and eventual prostate cancer-related mortality (<a href="https://pubmed.ncbi.nlm.nih.gov/10840432/">C. L. Amling, 2014;</a> <a href="https://jamanetwork.com/journals/jama/fullarticle/201291">S. J. Freedland et al., 2005;</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S0094014305701634?via%3Dihub">M. Han et al., 2001;</a> <a href="https://www.sciencedirect.com/science/article/pii/S0302283818307528?via%3Dihub">T. Van den Broeck et al., 2001</a>. Current clinical practices gauge the risk of biochemical recurrence by considering the International Society of Urological Pathology (ISUP) grade, PSA value at diagnosis, and TNM staging criteria (<a href="https://journals.lww.com/ajsp/abstract/2016/02000/the_2014_international_society_of_urological.10.aspx">J. I. Epstein et al., 2016</a>). A recent European consensus guideline suggests categorizing patients into low-risk, intermediate-risk, and high-risk groups based on these factors (<a href="https://www.sciencedirect.com/science/article/abs/pii/S0302283820307697?via%3Dihub">N. Mottet et al., 2021</a>). Notably, a high ISUP grade independently assigns a patient to the intermediate (grade 2/3) or high-risk group (grade 4/5). The Gleason growth patterns, representing morphological patterns of prostate cancer, are used to categorize cancerous tissue into ISUP grade groups (<a href="https://www.auajournals.org/doi/10.1016/j.juro.2009.10.046">J. I. Epstein, 2010;</a> <a href="https://bjui-journals.onlinelibrary.wiley.com/doi/10.1111/j.1464-410X.2012.11611.x">P. M. Pierorazio et al., 2013;</a> <a href="https://journals.lww.com/ajsp/fulltext/2020/08000/the_2019_international_society_of_urological.1.aspx">G. J. L. H. van Leenders et al., 2020;</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S0302283815005576?via%3Dihub">J. I. Epstein et al., 2016</a>). However, the ISUP grade has limitations, such as grading disagreement among pathologists (<a href="https://www.sciencedirect.com/science/article/abs/pii/S0302283815005576?via%3Dihub">J. I. Epstein et al., 2016</a>) and coarse descriptors of tissue morphology. Recently, deep learning was shown (<a href="https://www.nature.com/articles/s43856-022-00126-3">H. Pinckaers et al., 2022</a>; <a href="https://www.nature.com/articles/s44303-023-00005-z">O. Eminaga et. al., 2024</a>)...</p>
          <p><a href="/leopard/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://leopard.grand-challenge.org/" class="" target="_none">LEOPARD Challenge</a> by <a href="https://www.diagnijmegen.nl/" target="_none">Diagnostic Image Analysis Group, Radboudumc, Nijmegen</a></li></a>
          </li>
          <li>
            <a href="https://computationalpathologygroup.github.io/ASAP/" class="" target="_none">ASAP Viewer</a> by <a href="https://www.diagnijmegen.nl/people/geert-litjens/" target="_none">Geert Litjens</a></li></a>
          </li>
        </ul>
        <p><a href="/leopard/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-psi" class="dataset">
          <h3><a href="/nasa-psi/">NASA Physical Sciences Informatics (PSI)</a></h3>
          <p><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">fluid dynamics</span><span class="label label-info tag link-tag">materials science</span><span class="label label-info tag link-tag">physics</span><span class="label label-info tag link-tag">space biology</span></p>
          <p>NASA&#39;s Physical Sciences Research Program, along with its predecessors, has conducted significant fundamental and applied research in the physical sciences. The International Space Station (ISS) is an orbiting laboratory that provides an ideal facility to conduct long-duration experiments in the near absence of gravity and allows continuous and interactive research similar to Earth-based laboratories. This enables scientists to pursue innovations and discoveries not currently achievable by other means. NASA&#39;s Physical Sciences Research Program also benefits from collaborations with several of the ISS international partners—Europe, Russia, Japan, and Canada—and foreign governments with space programs, such as France, Germany and Italy. <br/><br/> In fulfillment of the Open Science model, NASA&#39;s Physical Sciences Research Program is pleased to offer the PSI data repository for physical science experiments performed in reduced-gravity environments such as the ISS, Space Shuttle flights, and Free-flyers. PSI also includes data from some related ground-based studies. The PSI system is accessible and open to the public. This provides the opportunity for researchers to data mine results from prior flight investigations, expanding on the research performed. This approach will allow numerous ground-based investigations to be conducted fro</br/>...</p>
          <p><a href="/nasa-psi/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://youtu.be/9uXgDUuWxTo" class="" target="_none">PSI System Demonstration</a> by Cheryl Payne</li></a>
          </li>
          <li>
            <a href="https://www.nasa.gov/sites/default/files/atoms/files/psi_researchers_guide-tagged.pdf" class="" target="_none">Researcher&#x27;s Guide to Physical Sciences Informatics System</a> by NASA ISS Program Science Office</li></a>
          </li>
        </ul>
        <p><a href="/nasa-psi/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-pmc" class="dataset">
          <h3><a href="/ncbi-pmc/">NIH NCBI PubMed Central (PMC) Article Datasets - Full-Text Biomedical and Life Sciences Journal Articles on AWS</a></h3>
          <p><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">txt</span><span class="label label-info tag link-tag">xml</span></p>
          <p>PubMed Central® (PMC) is a free full-text archive of biomedical and life sciences journal article at the U.S. National Institutes of Health&#39;s National Library of Medicine (NIH/NLM). The PubMed Central (PMC) Article Datasets include full-text articles archived in PMC and made available under license terms that allow for text mining and other types of secondary analysis and reuse. The articles are organized on AWS based on general license type:<br/><br/>
The PMC Open Access (OA) Subset, which includes all articles in PMC with a machine-readable Creative Commons license<br/><br/> The Author Manuscript Dataset, which includes all articles collected under a funder policy in PMC and made available in machine-readable formats for text mining<br/><br/>
These datasets collectively span</br/>...</p>
          <p><a href="/ncbi-pmc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/tools/pmcaws/" class="" target="_none">Accessing PMC Article Datasets Using Amazon Web Services</a> by <a href="https://www.ncbi.nlm.nih.gov/pmc/" target="_none">NCBI PMC</a></li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/extracting-insights-from-pubmed-articles-using-amazon-q-business/" class="aws-link" target="_none">Extracting insights from PubMed articles using Amazon Q Business</a> by Bharath Gunapati and Stefan Mationg</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-pmc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-nws-aorc" class="dataset">
          <h3><a href="/noaa-nws-aorc/">NOAA Analysis of Record for Calibration (AORC) Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Analysis Of Record for Calibration (AORC) is a gridded record of near-surface  weather conditions covering the continental United States and Alaska and their  hydrologically contributing areas. It is defined on a latitude/longitude spatial grid with a mesh length of 30 arc seconds (~800 m), and a temporal resolution of one hour.  Elements include hourly total precipitation, temperature, specific humidity, terrain-level  pressure, downward longwave and shortwave radiation, and west-east and south-north wind components. It spans the period from 1979 across the Continental U.S. (CONUS) and from 1981 across Alaska, to the near-present (at all locations). This suite of eight variables is sufficient to drive most land-surface and hydrologic models and is used as input to the National Water Model (NWM) retrospective simulation. While the native AORC process generates netCDF output, the data is post-processed to create a cloud optimized Zarr formatted equivalent for dissemination using cloud technology and infrastructure.
<br/>
<br/>
<strong>AORC Version 1.1 dataset creation</strong>
<br/>
The AORC dataset was created after reviewing, identifying, and processing multiple large-scale, observation, and analysis datasets. There are two versions of The Analysis Of Record for Calibration (AORC) data.
<br/>
<br/>
The initial AORC Version 1.0 dataset was completed in November 2019 and consisted of a grid with 8 elements at a resolution of 30 arc seconds.  The AORC version 1.1 dataset was created to address issues &quot;<a href="https://onlinelibrary.wiley.com/doi/10.1111/1752-1688.13143">see Table 1 in Fall et al., 2023</a>&quot; in the version 1.0 CONUS dataset. Full documentation on version 1.1 of the AORC data and the related journal publication are provided below.
<br/>
<br/>
The native AORC version 1.1 process creates a dataset that consists of netCDF files with the following dimensions: 1 hour, 4201 latitude values (ranging from 25.0 to 53.0), and 8401 longitude values (ranging from -125.0 to -67).
<br/>
<br/>
The data creation runs with a 10-day lag to ensure the inclusion of any corrections to the input Stage IV and NLDAS data.
<br/>
<br/>
<strong>Note</strong> - The full extent of the AORC grid as defined in its data files exceed those cited above; those outermost rows and columns of data grids are filled with missing values and are the remnant of an early set of required AORC extents that have since been adjusted inward.
<br/>
<br/>
<strong>AORC Version 1.1 Zarr Conversion</strong>
<br/>
<br/>
The goal for converting the AORC data from netCDF to Zarr was to allow users to quickly and efficiently load/use the data.  For example, one year of data takes 28 mins to load via NetCDF while only taking 3.2 seconds to load via Zarr (resulting in a substantial increase in speed).  For longer periods of time, the percentage increase in speed using Zarr (vs NetCDF) is even higher.  Using Zarr also leads to less memory and CPU utilization.
<br/>
<br/>
It was determined that the optimal conversion for the data was 1 year worth of Zarr files with a chunk size of 18MB. The chunking was completed across all 8 variables.  The chunks consist of the following dimensions: 144 time, 128 latitude, and 256 longitude. To create the files in the Zarr format, the NetCDF files were rechunked using chunk() and &quot;<a href="https://docs.xarray.dev/en/stable/">Xarray</a>&quot;.  After chunking the files, they were converted to a monthly Zarr file. Then, each monthly Zarr file was combined using &quot;<a href="https://docs.xarray.dev/en/latest/generated/xarray.Dataset.to_zarr.html">to_zarr</a>&quot; to create a Zarr file that represents a full year
<br/>
<br/>
Users wanting more than 1 year of data will be able to utilize Zarr utilities/libraries to combine multiple years up to the span of the full data set.
<br/>
<br/>
There are eight variables representing the meteorological conditions
<br/>
<strong>Total Precipitaion (APCP_surface)</strong>
<br/><ol>
<li>Hourly total precipitation (kgm-2 or mm)  for Calibration (AORC) dataset<br/></li>
</ol>
<strong>Air Temperature (TMP_2maboveground)</strong><ol>
<li>Temperature (at 2 m above-ground-level (AGL)) (K)<br/></li>
</ol>
<strong>Specific Humidity (SPFH_2maboveground)</strong>
<br/><ol>
<li>Specific humidity (at 2 m AGL) (g g-1) <br/></li>
</ol>
<strong>Downward Long-Wave Radiation Flux (DLWRF_surface)</strong>
<br/><ol>
<li>longwave (infrared)</li>
<li>radiation flux (at the  surface) (W m-2) <br/></li>
</ol>
<strong>Downward Short-Wave Radiation Flux (DSWRF_surface)</strong>
<br/><ol>
<li>Downward shortwave (solar)</li>
<li>radiation flux (at the  surface) (W m-2)<br/></li>
</ol>
<strong>Pressure (PRES_surface)</strong>
<br/><ol>
<li>Air pressure (at the surface) (Pa)<br/></li>
</ol>
**U-Component of Wind (UGRD_10maboveground)&quot;
<br/>
1)U (west-east) - components of the wind (at 10 m AGL) (m s-1)
<br/>
**V-Component of Wind (VGRD_10maboveground)&quot;
<br/><ol>
<li>V (south-north) - components of the wind (at 10 m AGL) (m s-1)<br/>
<br/></li>
</ol>
<strong>Precipitation and Temperature</strong>
<br/>
<br/>
The gridded AORC precipitation dataset contains one-hour Accumulated Surface Precipitation (APCP) ending at the “top” of each hour, in liquid water-equivalent units  (kg m-2 to the nearest 0.1 kg m-2), while the gridded AORC temperature dataset is comprised of instantaneous, 2 m above-ground-level (AGL) temperatures at the top of each hour (in Kelvin, to the nearest 0.1).
<br/>
<br/>
<strong>Specific Humidity, Pressure, Downward Radiation, Wind</strong>
<br/>
<br/>
...</p>
          <p><a href="/noaa-nws-aorc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1111/1752-1688.13143" class="" target="_none">The Office of Water Prediction&#x27;s Analysis of Record for Calibration, version 1.1: Dataset description and precipitation evaluation (09 July 2023). J. Am. Water Resour. Assoc., 59 (6). 1246-1272.</a> by Greg Fall, David Kitzmiller, Sandra Pavlovic, Ziya Zhang, Nathan Patrick, Michael St. Laurent, Carl Trypaluk, Wanru Wu, and Dennis Miller</li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/NOAA-OWP/AORC-jupyter-notebooks/blob/master/jupyter_notebooks/AORC_Zarr_notebook.ipynb" class="" target="_none">Explore the AORC 1.1 dataset in Zarr</a> by Michael AuCoin</li></a>
          </li>
        </ul>
        <p><a href="/noaa-nws-aorc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-cfs" class="dataset">
          <h3><a href="/noaa-cfs/">NOAA Climate Forecast System (CFS)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Climate Forecast System (CFS) is a model representing the global interaction between Earth&#39;s oceans, land, and atmosphere. Produced by several dozen scientists under guidance from the National Centers for Environmental Prediction (NCEP), this model offers hourly data with a horizontal resolution down to one-half of a degree (approximately 56 km) around Earth for many variables. CFS uses the latest scientific approaches for taking in, or assimilating, observations from data sources including surface observations, upper air balloon observations, aircraft observations, and satellite obser...</p>
          <p><a href="/noaa-cfs/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://journals.ametsoc.org/view/journals/bams/91/8/2010bams3001_1.xml" class="" target="_none">The NCEP Climate Forecast System Reanalysis</a> by Saha, Suranjana, and Coauthors</li></a>
          </li>
          <li>
            <a href="https://journals.ametsoc.org/view/journals/clim/27/6/jcli-d-12-00823.1.xml" class="" target="_none">The NCEP Climate Forecast System Version 2</a> by Saha, Suranjana, and Coauthors</li></a>
          </li>
        </ul>
        <p><a href="/noaa-cfs/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-mrms-pds" class="dataset">
          <h3><a href="/noaa-mrms-pds/">NOAA Multi-Radar/Multi-Sensor System (MRMS)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The MRMS system was developed to produce severe weather, transportation, and precipitation products for improved decision-making capability to improve hazardous weather forecasts and warnings, along with hydrology, aviation, and numerical weather prediction.
<br/>
<br/>
MRMS is a system with fully-automated algorithms that quickly and intelligently integrate data streams from multiple radars, surface and upper air observations, lightning detection systems, satellite observations, and forecast models. Numerous two-dimensional multiple-sensor products offer assistance for hail, wind, tornado, quantitative precipitation estimations, convection, icing, and turbulence diagnosis.
<br/>
<br/>
MRMS is being used to develop and test new Federal Aviation Administration (FAA) NextGen products in addition to advancing techniques in quality control, icing detection, and turbulence in collaboration with the National Center for Atmospheric Research, the University Corporation for Atmospheric Research, and Lincoln Laboratories.
<br/>
<br/>
MRMS was deployed operationally in 2014 at the National Center for Environmental Prediction (NCEP). All of the 100+ products it produces are available via NCEP to all of the WFOs, RFCs, CWSUs and NCEP service centers. In addition, the MRMS product suite is publicly available to any other entity who wishes to access and use the data. Other federal agencies that use MRMS include FEMA, DOD, FAA, and USDA.
<br/>
<br/><br>MRMS is the proposed operational version of the WDSS-II and NMQ research systems.
<br/>
<br/> 
</br/>...</p>
          <p><a href="/noaa-mrms-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://journals.ametsoc.org/view/journals/bams/97/4/bams-d-14-00174.1.xml" class="" target="_none">Multi-Radar Multi-Sensor (MRMS) Quantitative Precipitation Estimation: Initial Operating Capabilities</a> by Jian Zhang, Kenneth Howard, Carrie Langston, Brian Kaney, Youcun Qi, Lin Tang, Heather Grams, Yadong Wang, Stephen Cocks, Steven Martinaitis, Ami Arthur, Karen Cooper, Jeff Brogden, David Kitzmiller</li></a>
          </li>
          <li>
            <a href="https://journals.ametsoc.org/view/journals/bams/97/9/bams-d-14-00173.1.xml" class="" target="_none">Multi-Radar Multi-Sensor (MRMS) Severe Weather and Aviation Products: Initial Operating Capabilities</a> by Travis M. Smith, Valliappa Lakshmanan, Gregory J. Stumpf, Kiel L. Ortega, Kurt Hondl, Karen Cooper, Kristin M. Calhoun, Darrel M. Kingfield, Kevin L. Manross, Robert Toomey, Jeff Brogden</li></a>
          </li>
        </ul>
        <p><a href="/noaa-mrms-pds/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-ufs-s2s" class="dataset">
          <h3><a href="/noaa-ufs-s2s/">NOAA Unified Forecast System Subseasonal to Seasonal Prototypes</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Unified Forecast System Subseasonal to Seasonal prototypes consist of reforecast data from the UFS atmosphere-ocean coupled model experimental prototype version 5, 6, 7, and 8 produced by the Medium Range and Subseasonal to Seasonal Application team of the UFS-R2O project. The UFS prototypes are the first dataset released to the broader weather community for analysis and feedback as part of the development of the next generation operational numerical weather prediction system from NWS. The datasets includes all the major weather variables for atmosphere, land, ocean, sea ice, and ocean wav...</p>
          <p><a href="/noaa-ufs-s2s/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2022GL102392" class="" target="_none">Advances in Seasonal Predictions of Arctic Sea Ice With NOAA UFS</a> by Jieshun Zhu, Wanqiu Wang, Yanyun Liu, Arun Kumar, and David DeWitt</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1175/WAF-D-22-0162.1" class="" target="_none">The impact of tropical SST biases on the S2S precipitation forecast skill over the Contiguous United States in the UFS global coupled model</a> by Hedanqiu Bai, Bin Li, Avichal Mehra, Jessica Meixner, Shrinivas Moorthi, Sulagna Ray, Lydia Stefanova, Jiande Wang, Jun Wang, Denise Worthen, Fanglin Yang, and Cristiana Stan</li></a>
          </li>
        </ul>
        <p><a href="/noaa-ufs-s2s/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="noaa-wod" class="dataset">
          <h3><a href="/noaa-wod/">NOAA World Ocean Database (WOD)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The World Ocean Database (WOD) is the largest uniformly formatted, quality-controlled, publicly available historical subsurface ocean profile database. From Captain Cook&#39;s second voyage in 1772 to today&#39;s automated Argo floats, global aggregation of ocean variable information including temperature, salinity, oxygen, nutrients, and others vs. depth allow for study and understanding of the changing physical, chemical, and to some extent biological state of the World&#39;s Oceans. Browse the bucket via the AWS S3 explorer: <a href="https://noaa-wod-pds.s3.amazonaws.com/index.html">https://noaa-wod-pds.s3.amazonaws.com/index.html</a></p>
          <p><a href="/noaa-wod/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://data.nodc.noaa.gov/woa/WOD/DOC/wodreadme.pdf" class="" target="_none">The World Ocean Database User&#x27;s Manual</a> by Hernan E. Garcia, Tim P. Boyer, Ricardo A. Locarnini, Olga K. Baranova, Melissa M. Zweng</li></a>
          </li>
          <li>
            <a href="https://data.nodc.noaa.gov/woa/WOD/DOC/wod_intro.pdf" class="" target="_none">The World Ocean Database Introduction</a> by Tim P. Boyer, Olga K. Baranova, Carla Coleman, Hernan E. Garcia, Alexandra Grodsky, Ricardo A. Locarnini, Alexey V. Mishonov, Christopher R. Paver, James R. Reagan, Dan Seidov, Igor V. Smolyar, Katharine W. Weathers, Melissa M. Zweng</li></a>
          </li>
        </ul>
        <p><a href="/noaa-wod/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="nara-national-archives-catalog" class="dataset">
          <h3><a href="/nara-national-archives-catalog/">National Archives Catalog</a></h3>
          <p><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">government records</span><span class="label label-info tag link-tag">nara</span><span class="label label-info tag link-tag">national archives catalog</span></p>
          <p>The National Archives Catalog dataset contains all of the descriptions; authority records; digitized and electronic records; and tags, transcriptions and comments for NARA’s archival holdings available in the Catalog.</p>
          <p><a href="/nara-national-archives-catalog/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://catalog.archives.gov" class="" target="_none">National Archives Catalog</a> by <a href="https://www.archives.gov" target="_none">National Archives and Records Administration</a></li></a>
          </li>
          <li>
            <a href="https://www.archives.gov/developer/national-archives-catalog-dataset" class="" target="_none">National Archives Catalog on the AWS Registry of Open Data</a> by National Archives and Records Administration</li></a>
          </li>
        </ul>
        <p><a href="/nara-national-archives-catalog/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="nciccr-dlbcl" class="dataset">
          <h3><a href="/nciccr-dlbcl/">National Cancer Institute Center for Cancer Research - Diffuse Large B Cell Lymphoma (DLBCL) Genomics and Expression</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The study describes integrative analysis of genetic lesions in 574 diffuse large B cell lymphomas
(DLBCL) involving exome and transcriptome sequencing, array-based DNA copy number analysis and
targeted amplicon resequencing. The dataset contains open RNA-Seq Gene Expression Quantification
data.</p>
          <p><a href="/nciccr-dlbcl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/NCICCR-DLBCL" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1056/NEJMoa1801445" class="" target="_none">Genetics and Pathogenesis of Diffuse Large B Cell Lymphoma</a> by Roland Schmitz, Ph.D., George W. Wright, Ph.D., Da Wei Huang, M.D., et al.</li></a>
          </li>
        </ul>
        <p><a href="/nciccr-dlbcl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="black_marble_combustion" class="dataset">
          <h3><a href="/black_marble_combustion/">Nighttime-Fire-Flare</a></h3>
          <p><span class="label label-info tag link-tag">anomaly detection</span><span class="label label-info tag link-tag">classification</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">NASA SMD AI</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">socioeconomic</span><span class="label label-info tag link-tag">urban</span></p>
          <p>Detection of nighttime combustion (fire and gas flaring) from daily top of atmosphere data from NASA&#39;s Black Marble VNP46A1 product using VIIRS Day/Night Band and VIIRS thermal bands.</p>
          <p><a href="/black_marble_combustion/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1088/1748-9326/acb6a7" class="" target="_none">Potentially underestimated gas flaring activities—a new approach to detect combustion using machine learning and NASA&#x27;s Black Marble product suite</a> by Srija Chakraborty, Tomohiro Oda, Virginia Kalb, Zhuosen Wang, Miguel O Román</li></a>
          </li>
          <li>
            <a href="https://science.nasa.gov/files/atoms/files/NASA%20SMD%20AI%20Workshop%2021%20(spreads)%20(1).pdf" class="" target="_none">NASA SMD AI Workshop Report</a> by SMD Artificial Intelligence Machine Learning (AI/ML) Working Group</li></a>
          </li>
        </ul>
        <p><a href="/black_marble_combustion/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="open-cravat" class="dataset">
          <h3><a href="/open-cravat/">OpenCRAVAT</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">sqlite</span><span class="label label-info tag link-tag">tertiary analysis</span><span class="label label-info tag link-tag">variant annotation</span></p>
          <p>OpenCRAVAT is a module variant annotation tool developed by KarchinLab at Johns Hopkins.
This dataset is a mirror of the OpenCRAVAT store available at <a href="https://store.opencravat.org">https://store.opencravat.org</a>.
You can configure OpenCRAVAT to use this mirror by editing the &quot;cravat-system.yml&quot; file.
The path to this file is in the first output line of the command &quot;oc config system&quot;. In that file,
change the value of &quot;store_url&quot; to &quot;<a href="https://opencravat-store-aws.s3.amazonaws.com&quot;">https://opencravat-store-aws.s3.amazonaws.com&quot;</a>.</p>
          <p><a href="/open-cravat/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://opencravat.org" class="" target="_none">OpenCRAVAT</a> by <a href="https://karchinlab.org" target="_none">Karchinlab</a></li></a>
          </li>
          <li>
            <a href="https://open-cravat.readthedocs.io/en/latest/AWS-OpenData.html" class="" target="_none">Changing the OpenCRAVAT store url</a> by Kyle Moad</li></a>
          </li>
        </ul>
        <p><a href="/open-cravat/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ohsu-cnl" class="dataset">
          <h3><a href="/ohsu-cnl/">Oregon Health &amp; Science University Chronic Neutrophilic Leukemia Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The OHSU-CNL study offers the whole exome and RNA-sequencing on a cohort of 100 cases with rare
hematologic malignancies such as Chronic neutrophilic leukemia (CNL), atypical chronic myeloid
leukemia (aCML), and unclassified myelodysplastic syndrome/myeloproliferative neoplasms
(MDS/MPN-U). This dataset contains open RNA-Seq Gene Expression Quantification data.</p>
          <p><a href="/ohsu-cnl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/31366621" class="" target="_none">Genomic landscape of neutrophilic leukemias of ambiguous diagnosis</a> by Zhang H, Wilmot B, Bottomly D et al.</li></a>
          </li>
        </ul>
        <p><a href="/ohsu-cnl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="palsar2-scansar-turkey-syria" class="dataset">
          <h3><a href="/palsar2-scansar-turkey-syria/">PALSAR-2 ScanSAR Turkey &amp; Syria Earthquake (L2.1 &amp; L1.1)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>JAXA has responded to the Earthquake events in Turkey and Syria by conducting emergency disaster observations and providing data as requested by the Disaster and Emergency Management Authority (AFAD), Ministry of Interior in Turkey, through Sentinel Asia and the International Disaster Charter. Additional information on the event and dataset can be found <a href="https://earth.jaxa.jp/en/earthview/2023/02/14/7381/index.html">here</a>. The 25 m PALSAR-2 ScanSAR is normalized backscatter data of PALSAR-2 broad area observation mode with observation width of 350 km. Polarization data are stored as 16-bit digital numbers (DN). The DN values can be converted to gamma naught...</p>
          <p><a href="/palsar2-scansar-turkey-syria/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.eorc.jaxa.jp/ALOS/en/dataset/alos_open_and_free_e.htm" class="" target="_none">ALOS series Open and Free Data</a> by JAXA EORC</li></a>
          </li>
          <li>
            <a href="https://earth.jaxa.jp/en/earthview/2023/02/14/7381/index.html" class="" target="_none">ALOS-2 observations of earthquakes in southeastern Turkey in 2023</a> by JAXA EORC</li></a>
          </li>
        </ul>
        <p><a href="/palsar2-scansar-turkey-syria/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="organoid-pancreatic" class="dataset">
          <h3><a href="/organoid-pancreatic/">Pancreatic Cancer Organoid Profiling</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This study generated a collection of patient-derived pancreatic normal and cancer organoids and it was sequenced using Whole Genome Sequencing (WGS), Whole Exome Sequencing (WXS) and RNA-Seq as well as matched tumor and normal tissue if available. The study provides a valuable resource for pancreatic cancer researchers.
The dataset contains open RNA-Seq Gene Expression Quantification data and controlled WGS/WXS/RNA-Seq Aligned Reads, WXS Annotated Somatic Mutation, WXS Raw Somatic Mutation, and RNA-Seq Splice Junction Quantification.</p>
          <p><a href="/organoid-pancreatic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed?cmd&#x3D;DetailsSearch&amp;term&#x3D;29853643[PMID]" class="" target="_none">Organoid Profiling Identifies Common Responders to Chemotherapy in Pancreatic Cancer</a> by Tiriac H, Belleau P, Engle DD, Plenker D, Deschênes A, Somerville TD, et al.</li></a>
          </li>
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/ORGANOID-PANCREATIC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/organoid-pancreatic/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="pdb-3d-structural-biology-data" class="dataset">
          <h3><a href="/pdb-3d-structural-biology-data/">Protein Data Bank 3D Structural Biology Data</a></h3>
          <p><span class="label label-info tag link-tag">amino acid</span><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biomolecular modeling</span><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">chemical biology</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">electron microscopy</span><span class="label label-info tag link-tag">electron tomography</span><span class="label label-info tag link-tag">enzyme</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">nuclear magnetic resonance</span><span class="label label-info tag link-tag">pharmaceutical</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">protein template</span><span class="label label-info tag link-tag">SARS-CoV-2</span><span class="label label-info tag link-tag">structural biology</span><span class="label label-info tag link-tag">x-ray crystallography</span></p>
          <p>The &quot;Protein Data Bank (PDB) archive&quot; was established in 1971 as the first open-access digital data archive in biology. It is a collection of three-dimensional (3D) atomic-level structures of biological macromolecules (i.e., proteins, DNA, and RNA) and their complexes with one another and various small-molecule ligands (e.g., US FDA approved drugs, enzyme co-factors). For each PDB entry (unique identifier: 1abc or PDB_0000001abc) multiple data files contain information about the 3D atomic coordinates, sequences of biological macromolecules, information about any small molecules/ligan...</p>
          <p><a href="/pdb-3d-structural-biology-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1093/nar/gky949" class="" target="_none">Protein Data Bank: the single global archive for 3D macromolecular structure data</a> by wwPDB consortium</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1038/nsb1203-980" class="" target="_none">Announcing the worldwide Protein Data Bank</a> by Berman, H., Henrick, K. &amp; Nakamura, H.</li></a>
          </li>
        </ul>
        <p><a href="/pdb-3d-structural-biology-data/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="catalyst-cooperative-pudl" class="dataset">
          <h3><a href="/catalyst-cooperative-pudl/">Public Utility Data Liberation Project</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">electricity</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">energy modeling</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">government records</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">utilities</span></p>
          <p>The Public Utility Data Liberation Project (PUDL) provides analysis-ready energy system data to climate advocates,
researchers, policymakers, and journalists.
<br/>
<br/>
<a href="https://catalyst.coop/pudl/">PUDL</a> is an <a href="https://github.com/catalyst-cooperative/pudl">open source data processing pipeline</a>
that makes US energy data easier to access and use programmatically. Hundreds of gigabytes of valuable data
are published by US government agencies, but it&#39;s often difficult to work with.
PUDL takes the original spreadsheets, CSV files, and databases and turns them into a unified resource. This allows users to
spend more time on novel analysis and less time on data preparation.
<br/>
<br/>
This...</p>
          <p><a href="/catalyst-cooperative-pudl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.kaggle.com/code/catalystcooperative/02-state-hourly-electricity-demand" class="" target="_none">State Hourly Electricity Demand</a> by <a href="https://catalyst.coop/" target="_none">Catalyst Cooperative</a></li></a>
          </li>
          <li>
            <a href="https://www.kaggle.com/code/catalystcooperative/01-pudl-data-access" class="" target="_none">PUDL Data Access</a> by <a href="https://catalyst.coop/" target="_none">Catalyst Cooperative</a></li></a>
          </li>
        </ul>
        <p><a href="/catalyst-cooperative-pudl/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="rapid-nrt-flood-maps" class="dataset">
          <h3><a href="/rapid-nrt-flood-maps/">RAPID NRT Flood Maps</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">water</span></p>
          <p>Near Real-time and archival data of High-resolution (10 m) flood inundation dataset over the Contiguous United States, developed based on the Sentinel-1 SAR imagery (2016-current) archive, using an automated Radar Produced Inundation Diary (RAPID) algorithm.</p>
          <p><a href="/rapid-nrt-flood-maps/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.rse.2018.11.008" class="" target="_none">Near Real-Time Nonobstructed Flood Inundation Mapping by Synthetic Aperture Radar</a> by Xinyi Shen, Emmanouil N. Anagnostou, George H. Allen, G. Robert Brakenridge, Albert J. Kettner</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.3390/rs11070879" class="" target="_none">Inundation Extent Mapping by Synthetic Aperture Radar: A Review</a> by Xinyi Shen, Dacheng Wang, Kebiao Mao, Emmanouil Anagnostou, and Yang Hong</li></a>
          </li>
        </ul>
        <p><a href="/rapid-nrt-flood-maps/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="redasa-covid-data" class="dataset">
          <h3><a href="/redasa-covid-data/">REDASA COVID-19 Open Data</a></h3>
          <p><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">information retrieval</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">text analysis</span></p>
          <p>The REaltime DAta Synthesis and Analysis (REDASA) COVID-19 snapshot contains the output of the curation protocol produced by our curator community. A detailed description can be found in <a href="https://www.jmir.org/2021/5/e25714">our paper</a>. The first S3 bucket listed in Resources contains a large collection of medical documents in text format extracted from the <a href="https://registry.opendata.aws/cord-19/">CORD-19 dataset</a>, plus other sources deemed relevant by the REDASA consortium.  The second S3 bucket contains a series of documents surfaced by <a href="https://aws.amazon.com/kendra/">Amazon Kendra</a> that were considered relevant for each medical question asked. The final S3 bucket contains the GroundTruth annotations cr...</p>
          <p><a href="/redasa-covid-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.jmir.org/2021/5/e25714" class="" target="_none">Using a Secure, Continually Updating, Web Source Processing Pipeline to Support the Real-Time Data Synthesis and Analysis of Scientific Literature: Development and Validation Study</a> by Uddhav Vaghela, Simon Rabinowicz, Paris Bratsos, Guy Martin, Epameinondas Fritzilas, et al.</li></a>
          </li>
          <li>
            <a href="https://curadr.com/" class="" target="_none">Curadr - Curation Platform</a> by <a href="https://www.pansurg.org/redasa/" target="_none">REDASA Consortium, Imperial College London</a></li></a>
          </li>
        </ul>
        <p><a href="/redasa-covid-data/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="pacbio-human-wgs-reference" class="dataset">
          <h3><a href="/pacbio-human-wgs-reference/">Reference data for HiFi human WGS</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">long read sequencing</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">variant annotation</span><span class="label label-info tag link-tag">vcf</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Reference data bundle for analyzing HiFi human whole genome
sequencing data</p>
          <p><a href="/pacbio-human-wgs-reference/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/PacificBiosciences/HiFi-human-WGS-WDL" class="" target="_none">PacBio WGS Variant Pipeline README</a> by <a href="https://www.pacb.com" target="_none">Pacific Biosciences of California, Inc</a></li></a>
          </li>
          <li>
            <a href="https://www.pacb.com/wp-content/uploads/Application-note-Consolidated-analysis-tools-with-the-PacBio-WGS-Variant-Pipeline.pdf" class="" target="_none">Consolidated analysis with the PacBio WGS Variant Pipeline</a> by <a href="https://www.pacb.com" target="_none">Pacific Biosciences of California, Inc</a></li></a>
          </li>
        </ul>
        <p><a href="/pacbio-human-wgs-reference/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="satellogic-earthview" class="dataset">
          <h3><a href="/satellogic-earthview/">Satellogic EarthView dataset</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Satellogic EarthView dataset includes high-resolution satellite images captured over all continents. The dataset is organized in Hive partition format and hosted by AWS. The dataset can be accessed via STAC browser or aws cli. Each item of the dataset corresponds to a specific region and date, with some of the regions revisited for additional data. The dataset provides Top-of-Atmosphere (TOA) reflectance values across four spectral bands (Red, Green, Blue, Near-Infrared) at a Ground Sample Distance (GSD) of 1 meter, accompanied by comprehensive metadata such as off-nadir angles, sun elevation,...</p>
          <p><a href="/satellogic-earthview/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/satellogic/satellogic-earthview/" class="" target="_none">Explore Satellogic EarthView in SageMaker Studio Lab (SMSL)</a> by Javier Marin</li></a>
          </li>
          <li>
            <a href="https://satellogic-earthview.s3.us-west-2.amazonaws.com/index.html" class="" target="_none">EarthView: A Large Scale Remote Sensing Dataset for Self-Supervision</a> by Velázquez, Diego and Rodríguez, Pau and Alonso, Sergio and Gonfaus, Josep M. and González, Jordi and, Richarte, Gerardo and Marín, Javier and Bengio, Yoshua and Lacoste, Alexandre</li></a>
          </li>
        </ul>
        <p><a href="/satellogic-earthview/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="seefar" class="dataset">
          <h3><a href="/seefar/">SeeFar V0</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>A collection of multi-resolution satellite images from both public and commercial satellites. The dataset is specifically curated for training geospatial foundation models.</p>
          <p><a href="/seefar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2406.06776" class="" target="_none">SeeFar: Satellite Agnostic Multi-Resolution Dataset for Geospatial Foundation Models</a> by Lowman, J., Zheng, K. L., Fraser, R., The, J. V. G., &amp; Valipour, M.</li></a>
          </li>
          <li>
            <a href="https://coastalcarbon.ai/seefar" class="" target="_none">Getting Started with SeeFar for Multi-Resolution Geospatial Analysis</a> by <a href="https://www.coastalcarbon.ai/team" target="_none">James Lowman - Coastal Carbon</a></li></a>
          </li>
        </ul>
        <p><a href="/seefar/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="sentinel1-slc-seasia-pds" class="dataset">
          <h3><a href="/sentinel1-slc-seasia-pds/">Sentinel-1 SLC dataset for South and Southeast Asia, Taiwan, Korea and Japan</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>The S1 Single Look Complex (SLC) dataset contains Synthetic Aperture Radar (SAR) data in the C-Band wavelength. The SAR sensors are installed on a two-satellite (Sentinel-1A and Sentinel-1B) constellation orbiting the Earth with a combined revisit time of six days, operated by the European Space Agency. The S1 SLC data are a Level-1 product that collects radar amplitude and phase information in all-weather, day or night conditions, which is ideal for studying natural hazards and emergency response, land applications, oil spill monitoring, sea-ice conditions, and associated climate change effec...</p>
          <p><a href="/sentinel1-slc-seasia-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/earthobservatory/sentinel1-opds/wiki" class="" target="_none">Sentinel-1 Opendataset Wiki and Tutorials</a> by <a href="https://earthobservatory.sg/research-group/geodesy-group-emma-hill/people" target="_none">Earth Observatory of Singapore</a></li></a>
          </li>
          <li>
            <a href="https://www.nature.com/articles/s41597-020-0443-5" class="" target="_none">Rapid flood and damage mapping using synthetic aperture radar in response to Typhoon Hagibis, Japan</a> by Cheryl W. J. Tay, Sang-Ho Yun, Shi Tong Chin, Alok Bhardwaj, Jungkyo Jung &amp; Emma M. Hill</li></a>
          </li>
        </ul>
        <p><a href="/sentinel1-slc-seasia-pds/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="elp-nouabale-landscape" class="dataset">
          <h3><a href="/elp-nouabale-landscape/">Sounds of Central African landscapes</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">survey</span></p>
          <p>Archival soundscapes recorded in the rainforest landscapes of
Central Africa, with a focus on the vocalizations of African forest
elephants (Loxodonta cyclotis).</p>
          <p><a href="/elp-nouabale-landscape/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.aboutamazon.com/news/sustainability/listen-to-the-rainforest-chorus-thats-helping-scientists-protect-african-elephants" class="" target="_none">Listen to the rainforest chorus that&#x27;s helping scientists protect African elephants</a> by Amazon Staff</li></a>
          </li>
          <li>
            <a href="https://www.nationalgeographic.com/animals/article/researchers-share-more-than-a-million-hours-of-rainforest-recordings" class="" target="_none">You can now hear rainforest sounds worldwide-here&#x27;s why that matters</a> by Rachel Fobar</li></a>
          </li>
        </ul>
        <p><a href="/elp-nouabale-landscape/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ctrees-california-vhr-tree-height" class="dataset">
          <h3><a href="/ctrees-california-vhr-tree-height/">Sub-Meter Canopy Tree Height of California in 2020 by CTrees.org</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">conservation</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">land cover</span></p>
          <p>Canopy Tree Height maps for California in 2020. Created using a deep learning model on very-high-resolution airborne imagery from the National Agriculture Imagery Program (NAIP) by United States Department of Agriculture (USDA). </p>
          <p><a href="/ctrees-california-vhr-tree-height/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ctrees.org/news/california-tree-height-map-34" class="" target="_none">Canopy Height Unlocked: California Forest Resources Detailed in New Tree-level Map</a> by Daniel Melling</li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/j.rse.2024.114099" class="" target="_none">Sub-Meter Tree Height Mapping of California using Aerial Images and LiDAR-Informed U-Net Model</a> by Fabien H Wagner, Sophia Roberts, Alison L Ritz, Griffin Carter, Ricardo Dalagnol, Samuel Favrichon, Mayumi CM Hirye, Martin Brandt, Philippe Ciais and Sassan Saatchi</li></a>
          </li>
        </ul>
        <p><a href="/ctrees-california-vhr-tree-height/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="tiger" class="dataset">
          <h3><a href="/tiger/">TIGER Training</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">computational pathology</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">grand-challenge.org</span><span class="label label-info tag link-tag">histopathology</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>&quot;This dataset contains the training data for the <a href="https://tiger.grand-challenge.org">Tumor InfiltratinG lymphocytes in breast cancER or TIGER</a> challenge. TIGER is the first challenge on fully automated assessment of tumor-infiltrating lymphocytes (TILs) in breast cancer histopathology slides. TILs are proving to be an important biomarker in cancer patients as they can play a part in killing tumor cells, particularly in some types of breast cancer. Identifying and measuring TILs can help to better target treatments, particularly immunotherapy, and may result in lower levels of other more aggressive treatments, including chemo...</p>
          <p><a href="/tiger/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/DIAGNijmegen/pathology-whole-slide-data" class="" target="_none">Whole slide data - python package for working with wholeslide images and annotations files</a> by <a href="https://www.diagnijmegen.nl/" target="_none">Diagnostic Image Analysis Group, Radboudumc, Nijmegen</a></li></a>
          </li>
          <li>
            <a href="https://tiger.grand-challenge.org/" class="" target="_none">TIGER Challenge</a> by <a href="https://www.diagnijmegen.nl/" target="_none">Diagnostic Image Analysis Group, Radboudumc, Nijmegen</a></li></a>
          </li>
        </ul>
        <p><a href="/tiger/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="terrafusion" class="dataset">
          <h3><a href="/terrafusion/">Terra Fusion Data Sampler</a></h3>
          <p><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Terra Basic Fusion dataset is a fused dataset of the original Level 1 radiances
from the five Terra instruments. They have been fully validate to contain the original
Terra instrument Level 1 data. Each Level 1 Terra Basic Fusion file contains one full
Terra orbit of data and is typically 15 – 40 GB in size, depending on how much data was
collected for that orbit. It contains instrument radiance in physical units; radiance
quality indicator; geolocation for each IFOV at its native resolution; sun-view geometry;
bservation time; and other attributes/metadata. It is stored in HDF5, conformed to CF
conventions, and accessible by netCDF-4 enhanced models. It’s naming convention
follows: TERRA_BF_L1B_OXXXX_YYYYMMDDHHMMSS_F000_V000.h5. A concise description of the
dataset, along with links to complete documentation and available software tools, can
be found on the Terra Fusion project page: <a href="https://terrafusion.web.illinois.edu">https://terrafusion.web.illinois.edu</a>.</br></br>Terra is the flagship satellite of NASA’s Earth Observing System (EOS). It was launched
into orbit on December 18, 1999 and carries five instruments. These are the
Moderate-resolution Imaging Spectroradiometer (MODIS), the Multi-angle Imaging
SpectroRadiometer (MISR), the Advanced Spaceborne Thermal Emission and Reflection
Radiometer (ASTER), the Clouds and Earth’s Radiant Energy System (CERES), and the
Measurements of Pollution in the Troposphere (MOPITT).</br></br>The Terra Basic Fusion dataset is an easy-to-access record of the Level 1 radiances
for instruments on...</p>
          <p><a href="/terrafusion/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://go.illinois.edu/terra-fusion-doc" class="" target="_none">Basic Terra fusion product algorithm theoretical basis and data specifications</a> by <a href="http://hdl.handle.net/2142/105967" target="_none">Zhao, Guangu; Yang, Muqun; Clipp, Landon; Gao, Yizhao; Lee, Joe H.</a></li></a>
          </li>
          <li>
            <a href="https://github.com/TerraFusion" class="" target="_none">TerraFusion GitHub</a> by University of Illinois</li></a>
          </li>
        </ul>
        <p><a href="/terrafusion/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="tess" class="dataset">
          <h3><a href="/tess/">Transiting Exoplanet Survey Satellite (TESS)</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The Transiting Exoplanet Survey Satellite (TESS) is a multi-year survey that has discovered exoplanets in orbit around bright stars across the entire sky using high-precision photometry. The survey also enables a wide variety of stellar astrophysics, solar system science, and extragalactic variability studies. More information about TESS is available at <a href="https://archive.stsci.edu/missions-and-data/tess">MAST</a> and the <a href="https://heasarc.gsfc.nasa.gov/docs/tess/">TESS Science Support Center</a>.</p>
          <p><a href="/tess/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://outerspace.stsci.edu/display/TESS" class="" target="_none">TESS Archive Manual</a> by MAST Staff</li></a>
          </li>
          <li>
            <a href="https://timeseries.science.stsci.edu" class="" target="_none">TIKE - a free, Jupyter-based cloud platform to access and analyze MAST&#x27;s AWS TESS data</a> by MAST Staff</li></a>
          </li>
        </ul>
        <p><a href="/tess/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="coawst" class="dataset">
          <h3><a href="/coawst/">USGS COAWST (Coupled Ocean Atmosphere Wave and Sediment Transport) Forecast Model Archive, US East and Gulf Coasts</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span></p>
          <p>The COAWST modeling system has been used to simulate ocean, wave and sediment transport processes along the of US East Coast and Gulf of Mexico. The grid has a horizontal resolution of approximately 5km and is resolved with 16 vertical terrain following levels. The model has been executed on a daily basis since August 2009 with outputs written every hour. This archive contains model output from 2009-08-21 to 2022-06-17.</p>
          <p><a href="/coawst/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.5066/P9NQUAOW" class="" target="_none">Coupled-Ocean-Atmosphere-Wave-Sediment Transport (COAWST) Modeling System, U.S. Geological Survey Software Release, 23 April 2019</a> by Warner, J.C., Ganju, N.K., Sherwood, C.R., Kalra, T.S., Aretxabaleta, A., He, R., Zambon, J., and Kumar, N.</li></a>
          </li>
          <li>
            <a href="https://github.com/fs-jbzambon/opendata-coawst" class="" target="_none">COAWST Explorer Notebook</a> by <a href="https://about.me/rich.signell" target="_none">Rich Signell</a></li></a>
          </li>
        </ul>
        <p><a href="/coawst/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="uniprot" class="dataset">
          <h3><a href="/uniprot/">UniProt</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">enzyme</span><span class="label label-info tag link-tag">graph</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecule</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">RDF</span><span class="label label-info tag link-tag">SPARQL</span></p>
          <p>The Universal Protein Resource (UniProt) is a comprehensive resource for protein sequence and annotation data. The UniProt databases are the UniProt Knowledgebase (UniProtKB), the UniProt Reference Clusters (UniRef), and the UniProt Archive (UniParc). The UniProt consortium and host institutions <a href="https://www.ebi.ac.uk">EMBL-EBI</a>, <a href="https://www.sib.swiss">SIB Swiss Institute of Bioinformatics</a> and <a href="https://proteininformationresource.org/">PIR</a> are committed to the long-term preservation of the <a href="https://www.uniprot.org">UniProt</a> databases.</p>
          <p><a href="/uniprot/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/industries/exploring-the-uniprot-protein-knowledgebase-with-aws-open-data-and-amazon-neptune/" class="aws-link" target="_none">Exploring the UniProt protein knowledgebase with AWS Open Data and Amazon Neptune</a> by Eric Greene, Rafa Xu, Yuan Shi (AWS)</li></a>
          </li>
          <li>
            <a href="https://github.com/sib-swiss/sparql-training/tree/master/uniprot" class="" target="_none">UniProt SPARQL</a> by Swiss-Prot Group at SIB Swiss Institute of Bioinformatics</li></a>
          </li>
        </ul>
        <p><a href="/uniprot/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="whiffle-wins50" class="dataset">
          <h3><a href="/whiffle-wins50/">Whiffle WINS50 Open Data on AWS</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">electricity</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">turbulence</span><span class="label label-info tag link-tag">weather</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>Large Eddy Simulation (LES) data of the Winds of the North Sea in 2050 (WINS50) project.</p>
          <p><a href="/whiffle-wins50/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://wes.copernicus.org/articles/8/787/2023" class="" target="_none">Investigating energy production and wake losses of multi-gigawatt offshore wind farms with atmospheric large-eddy simulation</a> by Baas, P., Verzijlbergh, R., van Dorp, P., and Jonker, H</li></a>
          </li>
          <li>
            <a href="https://gitlab.com/whiffle-public/whiffle-open-data" class="" target="_none">Whiffle WINS50 Tutorials</a> by <a href="https://whiffle.nl" target="_none">Whiffle</a></li></a>
          </li>
        </ul>
        <p><a href="/whiffle-wins50/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="1000-genomes" class="dataset">
          <h3><a href="/1000-genomes/">1000 Genomes</a></h3>
          <p><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The 1000 Genomes Project is an international collaboration which has established the most detailed catalogue of human genetic variation, including SNPs, structural variants, and their haplotype context. The final phase of the project sequenced more than 2500 individuals from 26 different populations around the world and produced an integrated set of phased haplotypes with more than 80 million variants for these individuals.</p>
          <p><a href="/1000-genomes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/exploratory-data-analysis-of-genomic-datasets-using-adam-and-mango-with-apache-spark-on-amazon-emr/" class="aws-link" target="_none">Exploratory data analysis of genomic datasets using ADAM and Mango with Apache Spark on Amazon EMR</a> by <a href="https://research.eecs.berkeley.edu/~akmorrow/" target="_none">Alyssa Marrow</a></li></a>
          </li>
        </ul>
        <p><a href="/1000-genomes/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="wise-cryo-3band" class="dataset">
          <h3><a href="/wise-cryo-3band/">3-Band Cryo Data | Wide-field Infrared Survey Explorer (WISE)</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">survey</span></p>
          <p>The Wide-field Infrared Survey Explorer (WISE) was a NASA Medium Explorer satellite in low-Earth orbit that conducted an all-sky astronomical imaging survey over four infrared bands from 2010-2011. The 3-Band Cryo Data Release contains 3.4, 4.6 and 12 micron (W1, W2, W3) imaging data that were acquired between 6 Aug and 29 Sept 2010 while the detectors were cooled by the inner cryogen tank following the exhaustion of the outer tank.</p>
          <p><a href="/wise-cryo-3band/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://irsa.ipac.caltech.edu/docs/notebooks/" class="" target="_none">Notebook Tutorials</a> by <a href="https://irsa.ipac.caltech.edu" target="_none">Caltech/IPAC-IRSA</a></li></a>
          </li>
        </ul>
        <p><a href="/wise-cryo-3band/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="3dcompat" class="dataset">
          <h3><a href="/3dcompat/">3DCoMPaT: Composition of Materials on Parts of 3D Things</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>3D CoMPaT is a richly annotated large-scale dataset of rendered compositions of Materials on Parts of thousands of unique 3D Models.
This dataset primarily focuses on stylizing 3D shapes at part-level with compatible materials.
Each object with the applied part-material compositions is rendered from four equally spaced views as well as four randomized views.
We introduce a new task, called Grounded CoMPaT Recognition (GCR), to collectively recognize and ground compositions of materials on parts of 3D objects.
We present two variations of this task and adapt state-of-art 2D/3D deep learning met...</p>
          <p><a href="/3dcompat/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://3dcompat-dataset.org/pdf/paper.pdf" class="" target="_none">3DCoMPaT: Composition of Materials on Parts of 3D Things</a> by Yuchen Li, Ujjwal Upadhyay, Habib Slim, Ahmed Abdelreheem, Arpit Prajapati, Suhail Pothigara, Peter Wonka &amp; Mohamed Elhoseiny</li></a>
          </li>
        </ul>
        <p><a href="/3dcompat/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="aev-a2d2" class="dataset">
          <h3><a href="/aev-a2d2/">A2D2: Audi Autonomous Driving Dataset</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">robotics</span></p>
          <p>An open multi-sensor dataset for autonomous driving research. This dataset comprises semantically segmented images, semantic point clouds, and 3D bounding boxes. In addition, it contains unlabelled 360 degree camera images, lidar, and bus data for three sequences. We hope this dataset will further facilitate active research and development in AI, computer vision, and robotics for autonomous driving.</p>
          <p><a href="/aev-a2d2/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/amazon-eks-autonomous-driving-data-service" class="aws-link" target="_none">Autonomous Driving Data Service (ADDS)</a> by Ajay Vohra, Amazon</li></a>
          </li>
        </ul>
        <p><a href="/aev-a2d2/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allenai-diagrams" class="dataset">
          <h3><a href="/allenai-diagrams/">AI2 Diagram Dataset (AI2D)</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span></p>
          <p>4,817 illustrative diagrams for research on diagram understanding and associated question answering.</p>
          <p><a href="/allenai-diagrams/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.semanticscholar.org/paper/A-Diagram-is-Worth-a-Dozen-Images-Kembhavi-Salvato/e18ec2c9f0b4a817b8cf0435822bbc879d7db698" class="" target="_none">A Diagram is Worth a Dozen Images</a> by Aniruddha Kembhavi, Michael Salvato, Eric Kolve, Minjoon Seo, Hannaneh Hajishirzi, Ali Farhadi</li></a>
          </li>
        </ul>
        <p><a href="/allenai-diagrams/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allenai-meaningful-citations" class="dataset">
          <h3><a href="/allenai-meaningful-citations/">AI2 Meaningful Citations Data Set</a></h3>
          <p><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>630 paper annotations</p>
          <p><a href="/allenai-meaningful-citations/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.semanticscholar.org/paper/Identifying-Meaningful-Citations-Valenzuela-Ha/1c7be3fc28296a97607d426f9168ad4836407e4b#citing-papers" class="" target="_none">Identifying Meaningful Citations</a> by Marco Valenzuela, Vu A. Ha, Oren Etzioni</li></a>
          </li>
        </ul>
        <p><a href="/allenai-meaningful-citations/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allenai-arc" class="dataset">
          <h3><a href="/allenai-arc/">AI2 Reasoning Challenge (ARC) 2018</a></h3>
          <p><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">json</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>7,787 multiple choice science questions and associated corpora</p>
          <p><a href="/allenai-arc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.semanticscholar.org/paper/Think-you-have-Solved-Question-Answering-Try-ARC%2C-Clark-Cowhey/88bb0a28bb58d847183ec505dda89b63771bb495" class="" target="_none">Think you have Solved Question Answering? Try ARC, the AI2 Reasoning Challengg</a> by Peter Clark, Isaac Cowhey, Oren Etzioni, Tushar Khot, Ashish Sabharwal, Carissa Schoenick, Oyvind Tafjord</li></a>
          </li>
        </ul>
        <p><a href="/allenai-arc/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="arpa-e-perform" class="dataset">
          <h3><a href="/arpa-e-perform/">ARPA-E PERFORM Forecast data</a></h3>
          <p><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">solar</span></p>
          <p>The ARPA-E PERFORM Program is an ARPA-E funded program that aim to use
time-coincident power and load seeks to develop innovative management systems
that represent the relative delivery risk of each asset and balance the
collective risk of all assets across the grid. A risk-driven paradigm allows
operators to: (i) fully understand the true likelihood of maintaining a
supply-demand balance and system reliability, (ii) optimally manage the system,
and (iii) assess the true value of essential reliability services. This
paradigm shift is critical for all power systems and is essential for grids
wi...</p>
          <p><a href="/arpa-e-perform/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arpa-e.energy.gov/technologies/programs/perform" class="" target="_none">ARPA-E PERFORM</a> by <a href="https://arpa-e.energy.gov" target="_none">ARPA-E</a></li></a>
          </li>
        </ul>
        <p><a href="/arpa-e-perform/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="wise-allsky" class="dataset">
          <h3><a href="/wise-allsky/">All-Sky Data | Wide-field Infrared Survey Explorer (WISE)</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">survey</span></p>
          <p>The Wide-field Infrared Survey Explorer (WISE) was a NASA Medium Explorer satellite in low-Earth orbit that conducted an all-sky astronomical imaging survey over four infrared bands from 2010-2011. The All-Sky Release includes all data taken during the WISE full cryogenic mission phase, 7 January 2010 to 6 August 2010, in the 3.4, 4.6, 12, and 22 micron bands (i.e., W1, W2, W3, W4) that were processed with improved calibrations and reduction algorithms.</p>
          <p><a href="/wise-allsky/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://irsa.ipac.caltech.edu/docs/notebooks/" class="" target="_none">Notebook Tutorials</a> by <a href="https://irsa.ipac.caltech.edu" target="_none">Caltech/IPAC-IRSA</a></li></a>
          </li>
        </ul>
        <p><a href="/wise-allsky/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="wise-allwise" class="dataset">
          <h3><a href="/wise-allwise/">AllWISE Data | Wide-field Infrared Survey Explorer (WISE)</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">object detection</span><span class="label label-info tag link-tag">parquet</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">survey</span></p>
          <p>The Wide-field Infrared Survey Explorer (WISE) was a NASA Medium Explorer satellite in low-Earth orbit that conducted an all-sky astronomical imaging survey over four infrared bands from 2010-2011. The AllWISE Data Release combines data from all cryogenic and post-cryogenic survey phases and provides a comprehensive view of the mid-infrared sky. The Images Atlas includes 18,240 FITS image sets at 3.4, 4.6, 12 and 22 microns. The Source Catalog contains position, apparent motion, and flux information for over 747 million objects detected on the Atlas Images.</p>
          <p><a href="/wise-allwise/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://irsa.ipac.caltech.edu/docs/notebooks/" class="" target="_none">Notebook Tutorials</a> by <a href="https://irsa.ipac.caltech.edu" target="_none">Caltech/IPAC-IRSA</a></li></a>
          </li>
        </ul>
        <p><a href="/wise-allwise/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allen-brain-observatory" class="dataset">
          <h3><a href="/allen-brain-observatory/">Allen Brain Observatory - Visual Coding AWS Public Data Set</a></h3>
          <p><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>The Allen Brain Observatory – Visual Coding is a large-scale, standardized survey of physiological activity across the mouse visual cortex, hippocampus, and thalamus. It includes datasets collected with both two-photon imaging and Neuropixels probes, two complementary techniques for measuring the activity of neurons in vivo. The two-photon imaging dataset features visually evoked calcium responses from GCaMP6-expressing neurons in a range of cortical layers, visual areas, and Cre lines. The Neuropixels dataset features spiking activity from distributed cortical and subcortical brain regions, c...</p>
          <p><a href="/allen-brain-observatory/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/AllenInstitute/AllenSDK/wiki/Use-the-Allen-Brain-Observatory-%E2%80%93-Visual-Coding-on-AWS" class="" target="_none">Use the Allen Brain Observatory – Visual Coding on AWS</a> by <a href="https://twitter.com/AllenInstitute" target="_none">Nika Keller, David Feng</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-brain-observatory/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allen-nd-open-data" class="dataset">
          <h3><a href="/allen-nd-open-data/">Allen Institute for Neural Dynamics - Mouse Neuroanatomy and Physiology Data</a></h3>
          <p><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>The Allen Institute for Neural Dynamics (AIND) is committed to FAIR, Open, and Reproducible science. We therefore share all of the raw and derived data we collect publicly with rich metadata, including preliminary data collected during methods development, as near to the time of collection as possible.</p>
          <p><a href="/allen-nd-open-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://allenneuraldynamics.github.io/data.html#aind-open-data" class="" target="_none">AIND Open Data Access</a> by <a href="https://twitter.com/AllenInstitute" target="_none">David Feng, Saskia de Vries</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-nd-open-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="sentinel-1-rtc-indigo" class="dataset">
          <h3><a href="/sentinel-1-rtc-indigo/">Analysis Ready Sentinel-1 Backscatter Imagery</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>The <a href="https://sentinel.esa.int/web/sentinel/missions/sentinel-1">Sentinel-1 mission</a> is a constellation of
C-band Synthetic Aperature Radar (SAR) satellites from the European Space Agency launched since 2014.
These satellites collect observations of radar backscatter intensity day or night, regardless of the
weather conditions, making them enormously valuable for environmental monitoring.
These radar data have been processed from original Ground Range Detected (GRD) scenes into a Radiometrically
Terrain Corrected, tiled product suitable for analysis. This product is available over the Contiguous United States (CONUS)
since 2017 when Sentinel-1 data becam...</p>
          <p><a href="/sentinel-1-rtc-indigo/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/asdi-smsl-demo-delta" class="aws-link" target="_none">Compare Cloud-Optimized Geotiffs from Amazon Sustainability Data Initiative (ASDI) hosted on S3 using SageMaker Studio Lab (SMSL)</a> by Gianfranco Rapino</li></a>
          </li>
        </ul>
        <p><a href="/sentinel-1-rtc-indigo/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="apd_galaxymorph" class="dataset">
          <h3><a href="/apd_galaxymorph/">Astrophysics Division Galaxy Morphology Benchmark Dataset</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">NASA SMD AI</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>Hubble Space Telescope imaging data and associated identification labels for galaxy morphology derived from citizen scientist labels from the Galaxy Zoo: Hubble project.</p>
          <p><a href="/apd_galaxymorph/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://academic.oup.com/mnras/article/464/4/4176/2527878" class="" target="_none">Galaxy Zoo: morphological classifications for 120 000 galaxies in HST legacy imaging</a> by Kyle W. Willett, Melanie A. Galloway, Steven P. Bamford, Chris J. Lintott, Karen L. Masters, Claudia Scarlata, B. D. Simmons, Melanie Beck, Carolin N. Cardamone, Edmond Cheung, Edward M. Edmondson, Lucy F. Fortson, Roger L. Griffith, Boris Häußler, Anna Han, Ross Hart, Thomas Melvin, Michael Parrish, Kevin Schawinski, R. J. Smethurst, Arfon M. Smith</li></a>
          </li>
        </ul>
        <p><a href="/apd_galaxymorph/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="civic" class="dataset">
          <h3><a href="/civic/">CIViC (Clinical Interpretation of Variants in Cancer)</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">vcf</span></p>
          <p>Precision medicine refers to the use of prevention and treatment strategies that are tailored to the unique features of each individual and their disease. In the context of cancer this might involve the identification of specific mutations shown to predict response to a targeted therapy. The biomedical literature describing these associations is large and growing rapidly. Currently these interpretations exist largely in private or encumbered databases resulting in extensive repetition of effort. Realizing precision medicine will require this information to be centralized, debated and interpret...</p>
          <p><a href="/civic/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/ng.3774" class="" target="_none">CIViC is a community knowledgebase for expert crowdsourcing the clinical interpretation of variants in cancer</a> by <a href="https://griffithlab.org/" target="_none">Malachi Griffith</a></li></a>
          </li>
        </ul>
        <p><a href="/civic/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="cmsdesynpuf-omop" class="dataset">
          <h3><a href="/cmsdesynpuf-omop/">CMS 2008-2010 Data Entrepreneurs’ Synthetic Public Use File (DE-SynPUF) in OMOP Common Data Model</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">us</span></p>
          <p>DE-SynPUF is provided here as a 1,000 person (1k), 100,000 person (100k), and 2,300,000 persom (2.3m) data sets in the <a href="https://www.ohdsi.org/data-standardization/">OMOP Common Data Model</a> format.  The DE-SynPUF was created with the goal of providing a realistic set of claims data in the public domain while providing the very highest degree of protection to the Medicare beneficiaries’ protected health information.  The purposes of the DE-SynPUF are to: <ol>
<li>allow data entrepreneurs to develop and create software and applications that may eventually be applied to actual CMS claims data;</li>
<li>train researchers on the use and complexity of conducting analyses with CMS claims data prior to initiating the process to obtain access to actual CMS data; and,</li>
<li>support safe data mining innovations that may reveal unan...</p>
          <p><a href="/cmsdesynpuf-omop/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/OHDSI/OHDSIonAWS" class="" target="_none">OHDSIonAWS</a> by James Wiggins</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/big-data/creating-data-science-environments-on-aws-for-health-analysis-using-ohdsi/" class="aws-link" target="_none">Create data science environments on AWS for health analysis using OHDSI</a> by James Wiggins</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/industries/predict-patient-health-outcomes-using-ohdsi-and-machine-learning-on-aws/" class="aws-link" target="_none">Predict patient health outcomes using OHDSI and machine learning on AWS</a> by James Wiggins</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/machine-learning/map-clinical-notes-to-the-omop-common-data-model-and-healthcare-ontologies-using-amazon-comprehend-medical/" class="aws-link" target="_none">Map clinical notes to the OMOP Common Data Model and healthcare ontologies using Amazon Comprehend Medical</a> by James Wiggins</li></a>
          </li>
        </ul>
        <p><a href="/cmsdesynpuf-omop/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="ncbi-covid-19" class="dataset">
          <h3><a href="/ncbi-covid-19/">COVID-19 Genome Sequence Dataset</a></h3>
          <p><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">virus</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This repository within the <a href="https://www.nih.gov/research-training/medical-research-initiatives/activ/tracking-resistance-coronavirus-evolution-trace">ACTIV TRACE initiative</a> houses a comprehensive collection of datasets related to SARS-CoV-2. The processing of SARS-CoV-2 Sequence Read Archive (SRA) files has been optimized to identify genetic variations in viral samples. This information is then presented in the Variant Call Format (VCF). Each VCF file corresponds to the SRA parent-run&#39;s accession ID. Additionally, the data is available in the parquet format, making it easier to search and filter using the Amazon Athena Service. The SARS-CoV-2 Variant Calling Pipeline is designed to handle new data every six ho...</p>
          <p><a href="/ncbi-covid-19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sra-aws-download/" class="" target="_none">Download SRA sequence data using Amazon Web Services (AWS)</a> by NCBI SRA</li></a>
          </li>
        </ul>
        <p><a href="/ncbi-covid-19/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="cord-19" class="dataset">
          <h3><a href="/cord-19/">COVID-19 Open Research Dataset (CORD-19)</a></h3>
          <p><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">MERS</span><span class="label label-info tag link-tag">SARS</span></p>
          <p>Full-text and metadata dataset of COVID-19 and coronavirus-related research articles optimized for machine readability.</p>
          <p><a href="/cord-19/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge" class="" target="_none">COVID-19 Open Research Dataset Challenge (CORD-19)</a> by Kaggle</li></a>
          </li>
        </ul>
        <p><a href="/cord-19/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="caladapt-coproduced-climate-data" class="dataset">
          <h3><a href="/caladapt-coproduced-climate-data/">Co-Produced Climate Data to Support California&#x27;s Resilience Investments</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">weather</span><span class="label label-info tag link-tag">zarr</span></p>
          <p>Downscaled future and historical climate projections for California and her environs in support of California&#39;s Fifth Climate Assessment</p>
          <p><a href="/caladapt-coproduced-climate-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://analytics.cal-adapt.org/docs/use-cases/" class="" target="_none">Example Use Cases of Data Via Cal-Adapt Analytics Engine</a> by <a href="https://github.com/cal-adapt" target="_none">Cal-Adapt Analytics Engine Team</a></li></a>
          </li>
        </ul>
        <p><a href="/caladapt-coproduced-climate-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="comonscreens" class="dataset">
          <h3><a href="/comonscreens/">Common Screens</a></h3>
          <p><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>A corpus of web screenshot and metadata data composed of over 70 million websites.</p>
          <p><a href="/comonscreens/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://commonscreens.com/?page_id&#x3D;2512" class="" target="_none">IAB Text Classification</a> by Common Screens</li></a>
          </li>
        </ul>
        <p><a href="/comonscreens/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ncar-cesm2-arise" class="dataset">
          <h3><a href="/ncar-cesm2-arise/">Community Earth System Model v2 ARISE (CESM2 ARISE)</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">ice</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Data from ARISE-SAI Experiments with CESM2</p>
          <p><a href="/ncar-cesm2-arise/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NCAR/CESM2-ARISE" class="" target="_none">Coming Soon</a> by NCAR</li></a>
          </li>
        </ul>
        <p><a href="/ncar-cesm2-arise/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="short_peptides" class="dataset">
          <h3><a href="/short_peptides/">Conformational Space of Short Peptides</a></h3>
          <p><span class="label label-info tag link-tag">amino acid</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biomolecular modeling</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular dynamics</span><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">structural biology</span></p>
          <p>Co-managed by <a href="toyoko.io">Toyoko</a> and the <a href="http://ufq.unq.edu.ar/sbg/">Structural Biology Group at the Universidad Nacional de Quilmes</a>, this dataset allows us to explore the conformational space of all possible peptides using the 20 common amino acids. It consists of a collection of exhaustive molecular dynamics simulations of tripeptides and pentapeptides.</p>
          <p><a href="/short_peptides/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.toyoko.io/pepdataset/" class="" target="_none">Intro to Conformational Space of Short Peptides</a> by <a href="https://about.me/bassi" target="_none">Sebastian Bassi and Virginia Gonzalez</a></li></a>
          </li>
        </ul>
        <p><a href="/short_peptides/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="intelinair_corn_kernel_counting" class="dataset">
          <h3><a href="/intelinair_corn_kernel_counting/">Corn Kernel Counting Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>Dataset associated with the March 2021 Frontiers in Robotics and AI paper &quot;Broad Dataset and Methods for Counting and Localization of On-Ear Corn Kernels&quot;, DOI: 10.3389/frobt.2021.627009</p>
          <p><a href="/intelinair_corn_kernel_counting/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.frontiersin.org/articles/10.3389/frobt.2021.627009/abstract" class="" target="_none">Broad Dataset and Methods for Counting and Localization of On-Ear Corn Kernels</a> by Jennifer Hobbs, Vachik Khachatryan, Barathwaj Anandan, Harutyun Hovhannisyan, David Wilson</li></a>
          </li>
        </ul>
        <p><a href="/intelinair_corn_kernel_counting/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-uwpd-cmip5" class="dataset">
          <h3><a href="/noaa-uwpd-cmip5/">Coupled Model Intercomparison Project Phase 5 (CMIP5) University of Wisconsin-Madison Probabilistic Downscaling Dataset</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The University of Wisconsin Probabilistic Downscaling (UWPD) is a statistically downscaled dataset based on the Coupled Model Intercomparison Project Phase 5 (CMIP5) climate models. UWPD consists of three variables, daily precipitation and maximum and minimum temperature. The spatial resolution is 0.1<span>&#176;</span>x0.1<span>&#176;</span> degree resolution for the United States and southern Canada east of the Rocky Mountains.
<br/>
<br/>
The downscaling methodology is not deterministic. Instead, to properly capture unexplained variability and extreme events, the methodology predicts a spatially and temporally varying Probability Density Function (PDF) for each variable. Statistics such as the mean, mean PDF and annual maximum statistics can be calculated directly from the daily PDF and these statistics are included in the dataset. In addition, “standard”, “raw” data is created by randomly sampling from the PDFs to create a “realization” of the local scale given the large-scale from the climate model. There are 3 realizations for temperature and 14 realizations for precipitation.
...</p>
          <p><a href="/noaa-uwpd-cmip5/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://hdsc.nws.noaa.gov/hdsc/files25/NA14_Assessment_report_202201v1.pdf" class="" target="_none">Assessment Report: Analysis of Impact of Nonstationary Climate on NOAA Atlas 14 Estimates</a> by NOAA</li></a>
          </li>
        </ul>
        <p><a href="/noaa-uwpd-cmip5/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-dcdb-bathymetry-pds" class="dataset">
          <h3><a href="/noaa-dcdb-bathymetry-pds/">Crowdsourced Bathymetry</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>Community provided bathymetry data collected in collaboration with the International Hydrographic Organization.</p>
          <p><a href="/noaa-dcdb-bathymetry-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/dneufeldcu/notebooks/blob/main/esipCSBfinal.ipynb" class="" target="_none">Crowdsourced Bathymetry Data (CSB) Visualization</a> by David Neufeld</li></a>
          </li>
        </ul>
        <p><a href="/noaa-dcdb-bathymetry-pds/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="dmspssj" class="dataset">
          <h3><a href="/dmspssj/">Defense Meteorology Satellite Program (DMSP) Auroral Particle Flux</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">solar</span><span class="label label-info tag link-tag">space weather</span></p>
          <p>The United States Air Force (USAF) Defense Meteorological Satellite Program (DMSP) SSJ precipitating particle instrument measures in-situ total flux and energy distribution of electrons and ions at low earth orbit. These precipitating particles are of interest for space weather operations and research, in part because they produce aurora during normal and very strong geomagnetic storms. This dataset contains both sensor-level raw data (as detailed in Redmon et al. 2017) and a high-level machine-learning-ready data product.</p>
          <p><a href="/dmspssj/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/lkilcommons/ssj_latbin" class="" target="_none">ssj_latbin: Create and read ML-ready DMSP particle precipitation data</a> by <a href="https://www.linkedin.com/in/liam-kilcommons-69187449/" target="_none">Liam Kilcommons</a></li></a>
          </li>
        </ul>
        <p><a href="/dmspssj/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allenai-drop" class="dataset">
          <h3><a href="/allenai-drop/">Discrete Reasoning Over the content of Paragraphs (DROP)</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>The DROP dataset contains 96k Question and Answer pairs (QAs) over 6.7K paragraphs, split between train (77k QAs), development (9.5k QAs) and a hidden test partition (9.5k QAs).</p>
          <p><a href="/allenai-drop/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.semanticscholar.org/paper/DROP%3A-A-Reading-Comprehension-Benchmark-Requiring-Dua-Wang/dda6fb309f62e2557a071522354d8c2c897a2805" class="" target="_none">DROP: A Reading Comprehension Benchmark Requiring Discrete Reasoning Over Paragraphs</a> by Dheeru Dua, Yizhong Wang, Pradeep Dasigi, Gabriel Stanovsky, Sameer Singh, Matt Gardner</li></a>
          </li>
        </ul>
        <p><a href="/allenai-drop/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="era5-for-wrf" class="dataset">
          <h3><a href="/era5-for-wrf/">ERA5-for-WRF Open Data on AWS</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">electricity</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>ERA5 reanalysis data on AWS, preprocessed for use with the Weather Research and Forecasting (WRF) model.</p>
          <p><a href="/era5-for-wrf/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/moptis/era5-for-wrf/" class="" target="_none">ERA5-for-WRF Tutorials</a> by <a href="https://veer.eco" target="_none">Veer Renewables</a></li></a>
          </li>
        </ul>
        <p><a href="/era5-for-wrf/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="eot-web-archive" class="dataset">
          <h3><a href="/eot-web-archive/">End of Term Web Archive Dataset</a></h3>
          <p><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">web archive</span></p>
          <p>The End of Term Web Archive (EOT) captures and saves U.S. Government websites at the end of presidential administrations.  The EOT has thus far preserved websites from administration changes in 2008, 2012, 2016, and 2020.  Data from these web crawls have been made openly available in several formats in this dataset.</p>
          <p><a href="/eot-web-archive/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://digital.library.unt.edu/ark:/67531/metadc1998717/" class="" target="_none">Moving the End of Term Web Archive to the Cloud to Encourage Research Use and Reuse</a> by Mark Phillips and Sawood Alam</li></a>
          </li>
        </ul>
        <p><a href="/eot-web-archive/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="emearth" class="dataset">
          <h3><a href="/emearth/">Ensemble Meteorological Dataset for Planet Earth, EM-Earth</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">near-surface air temperature</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">precipitation</span></p>
          <p>EM-Earth provides data for precipitation, mean air temperature, air temperature range, and dew-point temperature at 0.1° spatial resolution over global land areas from 1950 to 2019. EM-Earth provides hourly/daily deterministic estimates, and daily probabilistic estimates (25 ensemble members), to meet the diverse requirements of hydrometeorological applications.</p>
          <p><a href="/emearth/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1175/BAMS-D-21-0106.1" class="" target="_none">EM-Earth: The Ensemble Meteorological Dataset for Planet Earth</a> by Guoqiang Tang, Martyn P. Clark &amp; Simon Michael Papalexiou</li></a>
          </li>
        </ul>
        <p><a href="/emearth/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="gatk-test-data" class="dataset">
          <h3><a href="/gatk-test-data/">GATK Test Data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The GATK test data resource bundle is a collection of files for resequencing human genomic data with the
Broad Institute&#39;s <a href="https://software.broadinstitute.org/gatk/">Genome Analysis Toolkit (GATK)</a>.</p>
          <p><a href="/gatk-test-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.opendata.aws/genomics-workflows/orchestration/cromwell/cromwell-examples/#real-world-example-haplotypecaller" class="aws-link" target="_none">Genomics Workflows on AWS - Cromwell on AWS</a> by <a href="https://www.linkedin.com/in/lee-pang-a039a26/" target="_none">W. Lee Pang</a></li></a>
          </li>
        </ul>
        <p><a href="/gatk-test-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="spatial-ucr" class="dataset">
          <h3><a href="/spatial-ucr/">Geosnap Data, Center for Geospatial Sciences</a></h3>
          <p><span class="label label-info tag link-tag">demographics</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">urban</span></p>
          <p>This bucket contains multiple datasets (as Quilt packages) created by the
Center for Geospatial Sciences (CGS) at the University of California-Riverside.
The data in this bucket contains the following:<ol>
<li>Tabular and geographic data from the US Census</li>
<li>Land Cover imagery collected from <a href="https://www.mrlc.gov/">Multi-Resolution Land Characteristics Consortium</a></li>
<li>Road network data processed from OpenStreetMap</li>
</ol>
</p>
          <p><a href="/spatial-ucr/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://spatialucr.github.io/geosnap-guide/content/home" class="" target="_none">Geosnap User Guide</a> by <a href="https://knaaptime.com" target="_none">Eli Knaap</a></li></a>
          </li>
        </ul>
        <p><a href="/spatial-ucr/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="gbif" class="dataset">
          <h3><a href="/gbif/">Global Biodiversity Information Facility (GBIF) Species Occurrences</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">conservation</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Global Biodiversity Information Facility (GBIF) is an international network and data infrastructure funded by the world&#39;s governments providing global data that document the occurrence of species. GBIF currently integrates datasets documenting over 1.6 billion species occurrences, growing daily. The GBIF occurrence dataset combines data from a wide array of sources including specimen-related data from natural history museums, observations from citizen science networks and environment recording schemes. While these data are constantly changing at GBIF.org, periodic snapshots are taken a...</p>
          <p><a href="/gbif/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://data-blog.gbif.org/post/aws-and-gbif/" class="" target="_none">GBIF and Apache-Spark on AWS tutorial</a> by John Waller</li></a>
          </li>
        </ul>
        <p><a href="/gbif/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="dataforgood-fb-hrsl" class="dataset">
          <h3><a href="/dataforgood-fb-hrsl/">High Resolution Population Density Maps + Demographic Estimates by CIESIN and Meta</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">demographics</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>Population data for a selection of countries, allocated to 1 arcsecond blocks and provided in a combination of CSV
and Cloud-optimized GeoTIFF files. This refines <a href="https://sedac.ciesin.columbia.edu/data/collection/gpw-v4">CIESIN’s Gridded Population of the World</a>
using machine learning models on high-resolution worldwide Maxar satellite imagery. CIESIN population counts aggregated from worldwide census
data are allocated to blocks where imagery appears to contain buildings.</p>
          <p><a href="/dataforgood-fb-hrsl/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/aws-asdi-cities-smsl-notebook" class="aws-link" target="_none">Investigating environmental characteristics of US cities using publicly available ASDI datasets</a> by Darren Ko</li></a>
          </li>
        </ul>
        <p><a href="/dataforgood-fb-hrsl/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="pyfr-mtu-t161-dns-data" class="dataset">
          <h3><a href="/pyfr-mtu-t161-dns-data/">High-Order Accurate Direct Numerical Simulation of Flow over a MTU-T161 Low Pressure Turbine Blade</a></h3>
          <p><span class="label label-info tag link-tag">computational fluid dynamics</span><span class="label label-info tag link-tag">green aviation</span><span class="label label-info tag link-tag">low-pressure turbine</span><span class="label label-info tag link-tag">turbulence</span></p>
          <p>The archive comprises snapshot, point-probe, and time-average data produced via a high-fidelity computational simulation of turbulent air flow over a low pressure turbine blade, which is an important component in a jet engine. The simulation was undertaken using the open source PyFR flow solver on over 5000 Nvidia K20X GPUs of the Titan supercomputer at Oak Ridge National Laboratory under an INCITE award from the US DOE. The data can be used to develop an enhanced understanding of the complex three-dimensional unsteady air flow patterns over turbine blades in jet engines. This could in turn le...</p>
          <p><a href="/pyfr-mtu-t161-dns-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.compfluid.2021.104989" class="" target="_none">High-Order Accurate Direct Numerical Simulation of Flow over a MTU-T161 Low Pressure Turbine Blade</a> by A. S. Iyer, Y. Abe, B. C. Vermeire, P. Bechlars, R. D. Baier, A. Jameson, F. D. Witherden, and P. E. Vincent</li></a>
          </li>
        </ul>
        <p><a href="/pyfr-mtu-t161-dns-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="hcmi-cmdc" class="dataset">
          <h3><a href="/hcmi-cmdc/">Human Cancer Models Initiative (HCMI) Cancer Model Development Center</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">STRIDES</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Human Cancer Models Initiative (HCMI) is an international consortium that is generating novel,
next-generation, tumor-derived culture models annotated with genomic and clinical data.
HCMI-developed models and related data are available as a community resource. The NCI is
contributing to the initiative by supporting four Cancer Model Development Centers (CMDCs).  CMDCs
are tasked with producing next-generation cancer models from clinical samples. The cancer models
include tumor types that are rare, originate from patients from underrepresented populations, lack
precision therapy, or lack ca...</p>
          <p><a href="/hcmi-cmdc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.gdc.cancer.gov/projects/HCMI-CMDC" class="" target="_none">Genomic Data Commons</a> by <a href="https://gdc.cancer.gov/" target="_none">National Cancer Institute</a></li></a>
          </li>
        </ul>
        <p><a href="/hcmi-cmdc/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="hpgp-data" class="dataset">
          <h3><a href="/hpgp-data/">Human PanGenomics Project</a></h3>
          <p><span class="label label-info tag link-tag">cram</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>This dataset includes sequencing data, assemblies, and analyses for the offspring of ten parent-offspring trios.</p>
          <p><a href="/hpgp-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41587-020-0503-6" class="" target="_none">Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes</a> by Shafin et al (2020)</li></a>
          </li>
        </ul>
        <p><a href="/hpgp-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="cropland_partitioining" class="dataset">
          <h3><a href="/cropland_partitioining/">IWMI DIWASA Rainfed and Irrigated Cropland Map for Africa</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cropland partitioning</span><span class="label label-info tag link-tag">irrigated cropland</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">land use</span><span class="label label-info tag link-tag">rainfed cropland</span></p>
          <p>A framework integrating the Budyko model has been developed to distinguish between rainfed and irrigated cropland areas across Africa. This expands on remote sensing land cover products available for agricultural water studies in Africa and thereby helps address the need for deeper insights into cropland patterns. Validation against an independent dataset revealed an overall accuracy of 73% with high precision and specificity scores. These results validate the framework’s effectiveness in identifying irrigated areas while minimizing errors in misclassifying rainfed areas as irrigated.</p>
          <p><a href="/cropland_partitioining/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.5066/P9N4R7SF" class="" target="_none">Rainfed and Irrigated Cropland Areas for Africa</a> by Owusu, A., Kagone, S., Leh, M., and Velpuri, N.M.</li></a>
          </li>
          <li>
            <a href="https://github.com/iwmiwaplus/ODR/blob/master/Partitioned%20Croplands/Applications/Awash_cropland%20partitioning.pdf" class="" target="_none">Water use in Awash basin</a> by <a href="https://github.com/iwmiwaplus" target="_none">A. Owusu, K. Akpoti, M. Leh, N. Velpuri</a></li></a>
          </li>
          <li>
            <a href="https://github.com/iwmiwaplus/ODR/tree/master/Partitioned%20Croplands/Tutorials" class="" target="_none">Cropland percentage</a> by <a href="https://github.com/iwmiwaplus" target="_none">iwmiwaplus</a></li></a>
          </li>
          <li>
            <a href="https://doi.org/10.1016/J.JAG.2023.103607" class="" target="_none">A framework for disaggregating remote-sensing cropland into rainfed and irrigated classes at continental scale</a> by Owusu, A., Kagone, S., Leh, M., Velpuri, N. M., Gumma, M. K., Ghansah, B., Thilina-Prabhath, P., Akpoti, K., Mekonnen, K., Tinonetsana, P., &amp; Mohammed, I.</li></a>
          </li>
        </ul>
        <p><a href="/cropland_partitioining/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="fast-ai-imageclas" class="dataset">
          <h3><a href="/fast-ai-imageclas/">Image classification - fast.ai datasets</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>Some of the most important datasets for image classification research, including
CIFAR 10 and 100, Caltech 101, MNIST, Food-101, Oxford-102-Flowers, Oxford-IIIT-Pets,
and Stanford-Cars.  This is part of the fast.ai datasets collection hosted by
AWS for convenience of fast.ai students. See documentation link for citation and
license details for each dataset.</p>
          <p><a href="/fast-ai-imageclas/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws/amazon-sagemaker-examples/blob/master/advanced_functionality/fastai_oxford_pets/fastai_lesson1_sagemaker_example.ipynb
" class="" target="_none">Oxford-IIIT Pet Image Classification on Amazon SageMaker</a> by <a href="https://aws.amazon.com/sagemaker/" target="_none">AWS</a></li></a>
          </li>
        </ul>
        <p><a href="/fast-ai-imageclas/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-gk2a-pds" class="dataset">
          <h3><a href="/noaa-gk2a-pds/">Korea Meteorological Administration (KMA) GK-2A Satellite Data</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Geo-KOMPSAT-2A (GK2A) is the new generation geostationary meteorological satellite (located in 128.2°E) of the Korea Meteorological Administration (KMA). The main mission of the GK2A is to observe the atmospheric phenomena over the Asia-Pacific region. The Advance Meteorological Imager (AMI) on GK2A scan the Earth full disk every 10 minutes and the Korean Peninsula area every 2 minutes with a high spatial resolution of 4 visible channels and 12 infrared channels. In addition, the AMI has an ability of flexible target area scanning useful for monitoring severe weather events such as typhoon...</p>
          <p><a href="/noaa-gk2a-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nmsc.kma.go.kr/enhome/html/base/cmm/selectPage.do?page&#x3D;satellite.gk2a.fact" class="" target="_none">GK2A Full Fact Sheet</a> by KMA</li></a>
          </li>
        </ul>
        <p><a href="/noaa-gk2a-pds/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="lofar-elais-n1" class="dataset">
          <h3><a href="/lofar-elais-n1/">LOFAR ELAIS-N1 cycle 2 observations on AWS</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">survey</span></p>
          <p>These data correspond to the <a href="http://www.lofar.org/">International LOFAR Telescope</a> observations of the sky field ELAIS-N1 (16:10:01 +54:30:36) during the cycle 2 of observations. There are 11 runs of about 8 hours each plus the corresponding observation of the calibration targets before and after the target field. The data are measurement sets (<a href="https://casa.nrao.edu/Memos/229.html">MS</a>) containing the cross-correlated data and metadata divided in 371 frequency sub-bands per target centred at ~150 MHz.</p>
          <p><a href="/lofar-elais-n1/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.lofarcloud.uk" class="" target="_none">Calibration of LOFAR ELAIS-N1 data in the Amazon cloud</a> by <a href="https://github.com/nudomarinero" target="_none">J. Sabater</a></li></a>
          </li>
        </ul>
        <p><a href="/lofar-elais-n1/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="lei" class="dataset">
          <h3><a href="/lei/">Legal Entity Identifier (LEI) and Legal Entity Reference Data (LE-RD)</a></h3>
          <p><span class="label label-info tag link-tag">analytics</span><span class="label label-info tag link-tag">blockchain</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">commerce</span><span class="label label-info tag link-tag">copyright monitoring</span><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">financial markets</span><span class="label label-info tag link-tag">governance</span><span class="label label-info tag link-tag">government spending</span><span class="label label-info tag link-tag">json</span><span class="label label-info tag link-tag">market data</span><span class="label label-info tag link-tag">socioeconomic</span><span class="label label-info tag link-tag">statistics</span><span class="label label-info tag link-tag">transparency</span><span class="label label-info tag link-tag">xml</span></p>
          <p>The Legal Entity Identifier (LEI) is a 20-character, alpha-numeric code based on the ISO 17442 standard developed by the International Organization for Standardization (ISO). It connects to key reference information that enables clear and unique identification of legal entities participating in financial transactions. Each LEI contains information about an entity’s ownership structure and thus answers the questions of &#39;who is who’ and ‘who owns whom’. Simply put, the publicly available LEI data pool can be regarded as a global directory, which greatly enhances transparency in the global ma...</p>
          <p><a href="/lei/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/publicsector/aws-hosts-new-open-dataset-help-businesses-identify-climate-finance-risks-investments/" class="aws-link" target="_none">AWS hosts new open dataset to help businesses identify climate finance risks and investments</a> by AWS Public Sector Blog Team</li></a>
          </li>
        </ul>
        <p><a href="/lei/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="intelinair_longitudinal_nutrient_deficiency" class="dataset">
          <h3><a href="/intelinair_longitudinal_nutrient_deficiency/">Longitudinal Nutrient Deficiency</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>Dataset associated with the 2021 AAAI Paper- Detection and Prediction of Nutrient Deficiency Stress using Longitudinal Aerial Imagery.  The dataset contains 3 image sequences of aerial imagery from 386 farm parcels which have been annotated for nutrient deficiency stress.</p>
          <p><a href="/intelinair_longitudinal_nutrient_deficiency/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2012.09654" class="" target="_none">Detection and Prediction of Nutrient Deficiency Stress using Longitudinal Aerial Imagery</a> by Saba Dadsetan, Gisele Rose, Naira Hovakimyan, Jennifer Hobbs</li></a>
          </li>
        </ul>
        <p><a href="/intelinair_longitudinal_nutrient_deficiency/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="man-truckscenes" class="dataset">
          <h3><a href="/man-truckscenes/">MAN TruckScenes</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">GPS</span><span class="label label-info tag link-tag">IMU</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">logistics</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">object detection</span><span class="label label-info tag link-tag">object tracking</span><span class="label label-info tag link-tag">perception</span><span class="label label-info tag link-tag">radar</span><span class="label label-info tag link-tag">robotics</span><span class="label label-info tag link-tag">transportation</span></p>
          <p>A large scale multimodal dataset for Autonomous Trucking. Sensor data was recorded with a heavy truck from MAN equipped with 6 lidars, 6 radars, 4 cameras and a high-precision GNSS. MAN TruckScenes allows the research community to come into contact with truck-specific challenges, such as trailer occlusions, novel sensor perspectives, and terminal environments for the first time.
It comprises more than 740 scenes of 20s each within a multitude of different environmental conditions. Bounding boxes are available for 27 object classes, 15 attributes, and a range of more than 230m. The scenes are t...</p>
          <p><a href="/man-truckscenes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/pdf/2407.07462" class="" target="_none">MANTruckScenes: A multimodal dataset for autonomous trucking in diverse conditions</a> by Felix Fent, Fabian Kuttenreich, Florian Ruch, Farija Rizwin, et al</li></a>
          </li>
          <li>
            <a href="https://github.com/TUMFTM/truckscenes-devkit/tree/main" class="" target="_none">TruckScenes devkit tutorial</a> by Felix Fent, Fabian Kuttenreich, Florian Ruch, Farija Rizwin</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/truckscenes-devkit/" class="" target="_none">PyPi package</a> by Felix Fent, Fabian Kuttenreich, Florian Ruch, Farija Rizwin</li></a>
          </li>
          <li>
            <a href="https://github.com/TUMFTM/truckscenes-devkit" class="" target="_none">TruckScenes devkit</a> by Felix Fent, Fabian Kuttenreich, Florian Ruch, Farija Rizwin</li></a>
          </li>
        </ul>
        <p><a href="/man-truckscenes/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="modis-astraea" class="dataset">
          <h3><a href="/modis-astraea/">MODIS MYD13A1, MOD13A1, MYD11A1, MOD11A1, MCD43A4</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>Data from the Moderate Resolution Imaging Spectroradiometer (MODIS), managed by
the U.S. Geological Survey and NASA. Five products are included:
MCD43A4 (MODIS/Terra and Aqua Nadir BRDF-Adjusted Reflectance Daily L3 Global 500 m SIN Grid),
MOD11A1 (MODIS/Terra Land Surface Temperature/Emissivity Daily L3 Global 1 km SIN Grid),
MYD11A1 (MODIS/Aqua Land Surface Temperature/Emissivity Daily L3 Global 1 km SIN Grid),
MOD13A1 (MODIS/Terra Vegetation Indices 16-Day L3 Global 500 m SIN Grid),
and MYD13A1 (MODIS/Aqua Vegetation Indices 16-Day L3 Global 500 m SIN Grid).
MCD43A4 has global coverage, all...</p>
          <p><a href="/modis-astraea/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://astraea.earth/earth-ondemand/" class="" target="_none">Astraea Earth OnDemand</a> by <a href="https://astraea.earth" target="_none">Astraea, Inc.</a></li></a>
          </li>
        </ul>
        <p><a href="/modis-astraea/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="nasa-gcms" class="dataset">
          <h3><a href="/nasa-gcms/">Mars Spectrometry 2: Gas Chromatography for the Sample Analysis at Mars Data (SAM) Instrument</a></h3>
          <p><span class="label label-info tag link-tag">analytics</span><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">NASA SMD AI</span><span class="label label-info tag link-tag">planetary</span></p>
          <p>NASA missions like the Curiosity and Perseverance rovers carry a rich array of instruments suited to collect data and build evidence towards answering if Mars ever had livable environmental conditions. These rovers can collect rock and soil samples and can take measurements that can be used to determine their chemical makeup. <br/><br/>
Because communication between rovers and Earth is severely constrained, with limited transfer rates and short daily communication windows, scientists have a limited time to analyze the data and make difficult inferences about the chemistry in order to prioritize the next operations and send those instructions back to the rover. <br/><br/>
This project aimed at building a model to automatically analyze gas chromatography mass spectrometry (GCMS) data collected for Mars exploration in order to help the scientists in their analysis of understanding the past habitability of Mars. <br/><br/>
More information are available at <a href="https://mars.nasa.gov/msl/spacecraft/instruments/sam/">https://mars.nasa.gov/msl/spacecraft/instruments/sam/</a> and the data from Mars are available and described at <a href="https://pds-geosciences.wustl.edu/missions/msl/sam.htm">https://pds-geosciences.wustl.edu/missions/msl/sam.htm</a>. <br/><br/>We request that you cite th...</p>
          <p><a href="/nasa-gcms/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.drivendata.org/competitions/97/nasa-mars-gcms/" class="" target="_none">Mars Spectrometry: Mars Spectrometry 2 (Challenge Results)</a> by DrivenData team and NASA partners</li></a>
          </li>
        </ul>
        <p><a href="/nasa-gcms/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="nasa-ega" class="dataset">
          <h3><a href="/nasa-ega/">Mars Spectrometry: Detect Evidence for Past Habitability</a></h3>
          <p><span class="label label-info tag link-tag">analytics</span><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">NASA SMD AI</span><span class="label label-info tag link-tag">planetary</span></p>
          <p>NASA missions like the Curiosity and Perseverance rovers carry a rich array of instruments suited to collect data and build evidence towards answering if Mars ever had livable environmental conditions. These rovers can collect rock and soil samples and can take measurements that can be used to determine their chemical makeup. <br/><br/>
Because communication between rovers and Earth is severely constrained, with limited transfer rates and short daily communication windows, scientists have a limited time to analyze the data and make difficult inferences about the chemistry in order to prioritize the next operations and send those instructions back to the rover. <br/><br/>
This project aimed at building a model to automatically analyze evolved gas analysis mass spectrometry (EGA-MS) data collected for Mars exploration in order to help the scientists in their analysis of understanding the past habitability of Mars. <br/><br/>
More information are available at <a href="https://mars.nasa.gov/msl/spacecraft/instruments/sam/">https://mars.nasa.gov/msl/spacecraft/instruments/sam/</a> and the data from Mars are available and described at <a href="https://pds-geosciences.wustl.edu/missions/msl/sam.htm">https://pds-geosciences.wustl.edu/missions/msl/sam.htm</a>. <br/><br/>
We request that you ci...</p>
          <p><a href="/nasa-ega/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.drivendata.org/competitions/93/nasa-mars-spectrometry/page/575/" class="" target="_none">Mars Spectrometry: Detect Evidence for Past Habitability (Challenge Results)</a> by DrivenData team and NASA partners</li></a>
          </li>
        </ul>
        <p><a href="/nasa-ega/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="met-office-ukesm1-arise" class="dataset">
          <h3><a href="/met-office-ukesm1-arise/">Met Office UK Earth System Model (UKESM1) ARISE-SAI geoengineering experiment data</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">CMIP6</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">ice</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>Data from the UK Earth System Model (UKESM1) ARISE-SAI experiment. The UKESM1 ARISE-SAI experiment explores the impacts of geoengineering via the injection of sulphur dioxide (SO2) into the stratosphere in order to keep global mean surface air temperature near 1.5 C above the pre-industrial climate. Data includes a five member ensemble of simulations with SO2 injection plus a five member ensemble of SSP2-4.5 simulations from CMIP6 to serve as a reference data set</p>
          <p><a href="/met-office-ukesm1-arise/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MetOffice/arise-cmor-tables/tree/master/examples" class="" target="_none">Examples coming soon</a> by MetOffice</li></a>
          </li>
        </ul>
        <p><a href="/met-office-ukesm1-arise/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="m3ed" class="dataset">
          <h3><a href="/m3ed/">Multi-robot, Multi-Sensor, Multi-Environment Event Dataset (M3ED)</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">event camera</span><span class="label label-info tag link-tag">global shutter camera</span><span class="label label-info tag link-tag">GNSS</span><span class="label label-info tag link-tag">GPS</span><span class="label label-info tag link-tag">h5</span><span class="label label-info tag link-tag">hdf5</span><span class="label label-info tag link-tag">IMU</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">perception</span><span class="label label-info tag link-tag">robotics</span><span class="label label-info tag link-tag">RTK</span></p>
          <p>M3ED is the first multi-sensor event camera (EC) dataset focused on high-speed dynamic motions in robotics applications. M3ED provides high-quality synchronized data from multiple platforms (car, legged robot, UAV), operating in challenging conditions such as off-road trails, dense forests, and performing aggressive flight maneuvers. M3ED also covers demanding operational scenarios for EC, such as high egomotion and multiple independently moving objects. M3ED includes high-resolution stereo EC (1280×720), grayscale and RGB cameras, a high-quality IMU, a 64-beam LiDAR, and RTK localization.</p>
          <p><a href="/m3ed/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://openaccess.thecvf.com/content/CVPR2023W/EventVision/papers/Chaney_M3ED_Multi-Robot_Multi-Sensor_Multi-Environment_Event_Dataset_CVPRW_2023_paper.pdf" class="" target="_none">M3ED: Multi-Robot, Multi-Sensor, Multi-Environment Event Dataset</a> by Chaney K, Cladera F, et al.</li></a>
          </li>
        </ul>
        <p><a href="/m3ed/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="multiconer" class="dataset">
          <h3><a href="/multiconer/">MultiCoNER Datasets</a></h3>
          <p><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>MultiCoNER 1 is a large multilingual dataset (11 languages) for Named Entity Recognition. It is designed to represent some of the contemporary challenges in NER, including low-context scenarios (short and uncased text), syntactically complex entities such as movie titles, and long-tail entity distributions. MultiCoNER 2 is a large multilingual dataset (12 languages) for fine grained Named Entity Recognition. Its fine-grained taxonomy contains 36 NE classes, representing real-world challenges for NER, where named entities, apart from the surface form, context represents a critical role in disti...</p>
          <p><a href="/multiconer/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://assets.amazon.science/95/4c/47c2afd44236ac1a05d184e7813e/dynamic-gazetteer-integration-in-multilingual-models-for-cross-lingual-and-cross-domain-named-entity-recognition.pdf" class="" target="_none">Dynamic Gazetteer Integration in Multilingual Models for Cross-Lingual and Cross-Domain Named Entity Recognition</a> by Besnik Fetahu, Anjie Fang, Oleg Rokhlenko and Shervin Malmasi</li></a>
          </li>
          <li>
            <a href="https://www.amazon.science/publications/gazetteer-enhanced-named-entity-recognition-for-code-mixed-web-queries" class="" target="_none">Gazetteer Enhanced Named Entity Recognition for Code-Mixed Web Queries</a> by Besnik Fetahu, Anjie Fang, Oleg Rokhlenko and Shervin Malmasi</li></a>
          </li>
          <li>
            <a href="https://www.amazon.science/publications/gemnet-effective-gated-gazetteer-representations-for-recognizing-complex-entities-in-low-context-input" class="" target="_none">GEMNET: Effective Gated Gazetteer Representations for Recognizing Complex Entities in Low-context Input</a> by Tao Meng, Anjie Fang, Oleg Rokhlenko and Shervin Malmasi</li></a>
          </li>
          <li>
            <a href="https://aclanthology.org/2022.coling-1.334.pdf" class="" target="_none">MultiCoNER: A Large-scale Multilingual Dataset for Complex Named Entity Recognition</a> by Shervin Malmasi, Anjie Fang, Besnik Fetahu, Sudipta Kar, Oleg Rokhlenko</li></a>
          </li>
        </ul>
        <p><a href="/multiconer/#usageexamples">See 4 usage examples &rarr;</a></p>
        </div>
        <div id="sdgstoday-mst" class="dataset">
          <h3><a href="/sdgstoday-mst/">My School Today</a></h3>
          <p><span class="label label-info tag link-tag">education</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">schools</span></p>
          <p>This database provides estimates of walking travel time of school-aged populations to schools recorded in OpenStreetMap. Population counts of male and female students are sorted into 3 groups of travel time - under 30 minutes, 30-60 minutes, and over 60 minutes. It covers the African continent and is aggregated by first-level administrative divisions.</p>
          <p><a href="/sdgstoday-mst/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://sdgstoday.org/myschooltoday" class="" target="_none">Map a School</a> by <a href="https://sdgstoday.org/" target="_none">SDGs Today</a></li></a>
          </li>
        </ul>
        <p><a href="/sdgstoday-mst/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="wise-postcryo" class="dataset">
          <h3><a href="/wise-postcryo/">NEOWISE Post-Cryo Data | Wide-field Infrared Survey Explorer (WISE)</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">survey</span></p>
          <p>The Wide-field Infrared Survey Explorer (WISE) was a NASA Medium Explorer satellite in low-Earth orbit that conducted an all-sky astronomical imaging survey over four infrared bands from 2010-2011. The NEOWISE Post-Cryo Data Release contains 3.4 and 4.6 micron (W1 and W2) imaging data that were acquired between 29 September 2010 and 1 February 2011 following the exhaustion of the inner and outer cryogen tanks.</p>
          <p><a href="/wise-postcryo/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://irsa.ipac.caltech.edu/docs/notebooks/" class="" target="_none">Notebook Tutorials</a> by <a href="https://irsa.ipac.caltech.edu" target="_none">Caltech/IPAC-IRSA</a></li></a>
          </li>
        </ul>
        <p><a href="/wise-postcryo/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="wise-neowiser" class="dataset">
          <h3><a href="/wise-neowiser/">NEOWISE Reactivation Data | Near-Earth Object Wide-field Infrared Survey Explorer (NEOWISE)</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">object detection</span><span class="label label-info tag link-tag">parquet</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">survey</span></p>
          <p>The Near-Earth Object Wide-field Infrared Survey Explorer (NEOWISE) is a NASA Medium-class Explorer satellite in low-Earth orbit conducting an all-sky astronomical imaging survey over two infrared bands. The NEOWISE Reactivation mission began in 2013 when the original WISE satellite was brought out of hibernation to learn more about the population of near-Earth objects and comets that could pose an impact hazard to the Earth. The data is also used to study a wide range of astrophysical phenomena in the time domain including brown dwarfs, supernovae and active galactic nuclei.</p>
          <p><a href="/wise-neowiser/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://irsa.ipac.caltech.edu/docs/notebooks/" class="" target="_none">Notebook Tutorials</a> by <a href="https://irsa.ipac.caltech.edu" target="_none">Caltech/IPAC-IRSA</a></li></a>
          </li>
        </ul>
        <p><a href="/wise-neowiser/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-coastal-lidar" class="dataset">
          <h3><a href="/noaa-coastal-lidar/">NOAA Coastal Lidar Data</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">stac</span></p>
          <p>Lidar (light detection and ranging) is a technology that can measure the 3-dimentional location of objects, including the solid earth surface. The data consists of a point cloud of the positions of solid objects that reflected a laser pulse, typically from an airborne platform. In addition to the position, each point may also be attributed by the type of object it reflected from, the intensity of the reflection, and other system dependent metadata. The NOAA Coastal Lidar Data is a collection of lidar projects from many different sources and agencies, geographically focused on the coastal areas of the United States of America. The data is provided in Entwine Point Tiles (EPT; <a href="https://entwine.io">https://entwine.io</a>) format, which is a lossless streamable octree of the point cloud, and in LAZ format. Datasets are maintained in their original projects and care should be taken when merging projects. The coordinate reference system for the data is The NAD83(2011) UTM zone appropriate for the center of each data set for EPT and geographic coordinates for LAZ. Vertically they are in the orthometric datum appropriate for that area (for example, NAVD88 in the mainland United States, PRVD02 in Puerto Rico, or GUVD03 in Guam). The geoid model used is reflected in the data set resource name. <br>The ...</p>
          <p><a href="/noaa-coastal-lidar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://portal.opentopography.org/datasets" class="" target="_none">OpenTopography access and processing of NOAA Coastal Lidar Data</a> by <a href="https://opentopography.org/" target="_none">OpenTopography</a></li></a>
          </li>
        </ul>
        <p><a href="/noaa-coastal-lidar/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-gsod" class="dataset">
          <h3><a href="/noaa-gsod/">NOAA Global Surface Summary of Day</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">regulatory</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Global Surface Summary of the Day is derived from The Integrated Surface Hourly (ISH) dataset. The ISH dataset includes global data obtained from the USAF Climatology Center, located in the Federal Climate Complex with NCDC. The latest daily summary data are normally available 1-2 days after the date-time of the observations used in the daily summaries. The online data files begin with 1929 and are at the time of this writing at the Version 8 software level. Over 9000 stations&#39; data are typically available. The daily elements included in the dataset (as available from each station) are: <br/>
Mean temperature (.1 Fahrenheit) <br/>
Mean dew point (.1 Fahrenheit) <br/>
Mean sea level pressure (.1 mb) <br/>
Mean station pressure (.1 mb) <br/>
Mean visibility (.1 miles) <br/>
Mean wind speed (.1 knots) <br/>
Maximum sustained wind speed (.1 knots) <br/>
Maximum wind gust (.1 knots) <br/>
Maximum temperature (.1 Fahrenheit) <br/>
Minimum temperature (.1 Fahrenheit) <br/>
Precipitation amount (.01 inches) <br/>
Snow depth (.1 inches) <br/>
Indicator for occurrence of: Fog, Rain or Drizzle, Snow or Ice Pellets, Hail, Thunder, Tornado/Funnel Cloud.<br/><br/>
G</br/>...</p>
          <p><a href="/noaa-gsod/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/aws-smsl-predict-airquality-via-weather/" class="aws-link" target="_none">ML Demo: Predicting Air Quality w/ ASDI NOAA + OpenAQ Datasets in SageMaker Studio Lab (SMSL)</a> by Aaron Soto</li></a>
          </li>
        </ul>
        <p><a href="/noaa-gsod/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-isd" class="dataset">
          <h3><a href="/noaa-isd/">NOAA Integrated Surface Database (ISD)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Integrated Surface Database (ISD) consists
of global hourly and synoptic observations
compiled from numerous sources into a gzipped
fixed width format. ISD was developed as a joint
activity within Asheville&#39;s Federal Climate
Complex. The database includes over 35,000 stations
worldwide, with some having data as far back
as 1901, though the data show a substantial
increase in volume in the 1940s and again in
the early 1970s. Currently, there are over
14,000 &quot;active&quot; stations updated daily in the
database. The total uncompressed data volume is
around 600 gigabytes; however, it ...</p>
          <p><a href="/noaa-isd/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/awslabs/amazon-asdi/tree/main/examples/noaa-isd" class="aws-link" target="_none">NOAA Integrated Surface Database (ISD) Example Notebook</a> by Zac Flamig</li></a>
          </li>
        </ul>
        <p><a href="/noaa-isd/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-oar-myrorss-pds" class="dataset">
          <h3><a href="/noaa-oar-myrorss-pds/">NOAA Multi-Year Reanalysis of Remotely Sensed Storms (MYRORSS)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Multi-Year Reanalysis of Remotely Sensed Storms (MYRORSS) consists of radar reflectivity data run through the Multi-Radar, Multi-Sensor (MRMS) framework to create a three-dimensional radar volume on a quasi-Cartesian latitude-longitude grid across the entire contiguous United States. The radar reflectivity grid is also combined with hourly forecast model analyses to produce derived products such as echo top heights and hail size estimates. Radar Doppler velocity data was also processed into two azimuthal shear layer products. The source radar data was from the <a href="https://registry.opendata.aws/noaa-nexrad/">NEXRAD Level-II archive</a> and the model analyses came from <a href="https://www.ncei.noaa.gov/has/HAS.FileAppRouter?datasetname=RUCANL130&amp;subqueryby=STATION&amp;applname=&amp;outdest=FILE">NOAA&#39;s Rapid Update Cycle model</a>. Radar reflectivity was quality controlled to remove non-weather echoes and the data set was manually quality contolled to remove errors as revealed through inspection of daily accumulations of the hail size product and the azimuthal shear products. MYRORSS contains data from April 1998 through December 2011. The horizontal resolution is 0.01<span>&#176;</span> by 0.01<span>&#176;</span> and t...</p>
          <p><a href="/noaa-oar-myrorss-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.15763/DBS.CIMMS.MYRORSS.DATA" class="" target="_none">Comprehensive radar data for the contiguous United States: Multi-year reanalysis of remotely sensed storms.</a> by Williams, S. S., K. L. Ortega, T. M. Smith, and A. E. Reinhart</li></a>
          </li>
        </ul>
        <p><a href="/noaa-oar-myrorss-pds/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-ndfd" class="dataset">
          <h3><a href="/noaa-ndfd/">NOAA National Digital Forecast Database (NDFD)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Please note NWS is Soliciting Comments until April 30, 2024 on Availability of Probabilistic Snow Grids for Select Weather Forecast Offices (WFOs) as an Experimental Element in the National Digital Forecast Database (NDFD) for the Contiguous United States (CONUS). A PDF version of the Public Notice can be found &quot;<a href="https://www.weather.gov/media/notification/pdf2/pns19-02_prob_snow_aad.pdf">HERE</a>&quot;
<br />
<br />
The National Digital Forecast Database (NDFD) is a suite of gridded forecasts of sensible weather elements (e.g., cloud cover, maximum temperature).  Forecasts prepared by NWS field offices working in collaboration with the National Centers for Environmental Predictio...</p>
          <p><a href="/noaa-ndfd/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://noaa-ndfd-pds.s3.amazonaws.com/NDFDelem_fullres_202206.xls" class="" target="_none">NDFD Product Spreadsheet (excel file)</a> by NOAA MDL</li></a>
          </li>
        </ul>
        <p><a href="/noaa-ndfd/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-nwm-pds" class="dataset">
          <h3><a href="/noaa-nwm-pds/">NOAA National Water Model Short-Range Forecast</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The National Water Model (NWM) is a water resources model that simulates and forecasts water
budget variables, including snowpack, evapotranspiration, soil moisture and streamflow, over
the entire continental United States (CONUS). The model, launched in August 2016, is designed
to improve the ability of NOAA to meet the needs of its stakeholders (forecasters, emergency
managers, reservoir operators, first responders, recreationists, farmers, barge operators, and
ecosystem and floodplain managers) by providing expanded accuracy, detail, and frequency of water
information. It is operated by NOA...</p>
          <p><a href="/noaa-nwm-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.jhydrol.2023.129514" class="" target="_none">Harmonic Oscillator Seasonal Trend (HOST) Model for Hydrological Drought Pattern Identification and Analysis</a> by K. Raczyński, J. Dyer</li></a>
          </li>
        </ul>
        <p><a href="/noaa-nwm-pds/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-s111" class="dataset">
          <h3><a href="/noaa-s111/">NOAA S-111 Surface Water Currents Data</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">water</span></p>
          <p>S-111 is a data and metadata encoding specification that is part of the <a href="https://iho.int/en/s100-project">S-100 Universal Hydrographic Data Model</a>, an international standard for hydrographic data. This collection of data contains surface water currents forecast guidance from <a href="https://tidesandcurrents.noaa.gov/models.html">NOAA/NOS Operational Forecast Systems</a>, a set of operational hydrodynamic nowcast and forecast modeling systems, for various U.S. coastal waters and the great lakes. The collection also contains surface current forecast guidance output from the <a href="https://polar.ncep.noaa.gov/global/">NCEP Global Real-Time Ocean Forecast System (GRTOFS)</a> for some offshore areas. These datasets are encoded as HDF-5 f...</p>
          <p><a href="/noaa-s111/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.youtube.com/watch?v&#x3D;laC0Du6-x3k" class="" target="_none">NOAA Precision Marine Navigation Program: Developing Next-Gen Data Svcs for the Maritime Community</a> by NOAA</li></a>
          </li>
        </ul>
        <p><a href="/noaa-s111/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-climate-normals" class="dataset">
          <h3><a href="/noaa-climate-normals/">NOAA U.S. Climate Normals</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The U.S. Climate Normals are a large suite of data products that provide information about typical climate conditions for thousands of locations across the United States. Normals act both as a ruler to compare today’s weather and tomorrow’s forecast, and as a predictor of conditions in the near future. The official normals are calculated for a uniform 30 year period, and consist of annual/seasonal, monthly, daily, and hourly averages and statistics of temperature, precipitation, and other climatological variables from almost 15,000 U.S. weather stations. <br/><br/>
NCEI generates the official U.S. norma</br/>...</p>
          <p><a href="/noaa-climate-normals/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/aws-asdi-cities-smsl-notebook/" class="aws-link" target="_none">Investigating environmental characteristics of US cities using publicly available ASDI datasets using SageMaker Studio Lab (SMSL)</a> by Darren Ko</li></a>
          </li>
        </ul>
        <p><a href="/noaa-climate-normals/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-wave-ensemble-reforecast" class="dataset">
          <h3><a href="/noaa-wave-ensemble-reforecast/">NOAA Wave Ensemble Reforecast</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>This is a 20-year global wave reforecast generated by WAVEWATCH III model (<a href="https://github.com/NOAA-EMC/WW3">https://github.com/NOAA-EMC/WW3</a>) forced by GEFSv12 winds (<a href="https://noaa-gefs-retrospective.s3.amazonaws.com/index.html">https://noaa-gefs-retrospective.s3.amazonaws.com/index.html</a>). The wave ensemble was run with one cycle per day (at 03Z), spatial resolution of 0.25°X0.25° and temporal resolution of 3 hours. There are five ensemble members (control plus four perturbed members) and, once a week (Wednesdays), the ensemble is expanded to eleven members. The forecast range is 16 days and, once a week (Wednesdays), it extends to 35 days. More information about the wave modeling, wave grids and calibration can be found in the WAVEWATCH III regtest ww3_ufs1.3 (<a href="https://github.com/NOAA-EMC/WW3/tree/develop/regtests/ww3_ufs1.3">https://github.com/NOAA-EMC/WW3/tree/develop/regtests/ww3_ufs1.3</a>).
...</p>
          <p><a href="/noaa-wave-ensemble-reforecast/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/ricampos/gefswaves_reforecast" class="" target="_none">Examples and scripts to support users to download, visualize, and process the WAVEWATCH III output files</a> by <a href="https://github.com/ricampos" target="_none">Ricardo Campos</a></li></a>
          </li>
        </ul>
        <p><a href="/noaa-wave-ensemble-reforecast/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-ocean-climate-stations" class="dataset">
          <h3><a href="/noaa-ocean-climate-stations/">NOAA/PMEL Ocean Climate Stations Moorings</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The mission of the Ocean Climate Stations (OCS) Project is to make meteorological and 
oceanic measurements from autonomous platforms.  Calibrated, quality-controlled, and well-documented 
climatological measurements are available on the OCS webpage and the OceanSITES Global Data
Assembly Centers (GDACs), with near-realtime data available prior to release of the complete, 
downloaded datasets.<br/><br/>OCS measurements served through the Big Data Program come from OCS high-latitude moored buoys located in the Kuroshio 
Extension (32°N 145°E) and the Gulf of Alaska (50°N 145°W).  Initiated in 2004 and 20</br/>...</p>
          <p><a href="/noaa-ocean-climate-stations/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.pmel.noaa.gov/ocs/publications" class="" target="_none">OCS publications - All OCS-relevant publications are updated at the URL below.</a> by PMEL</li></a>
          </li>
        </ul>
        <p><a href="/noaa-ocean-climate-stations/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="nsf-ncar-era5" class="dataset">
          <h3><a href="/nsf-ncar-era5/">NSF NCAR Curated ECMWF Reanalysis 5 (ERA5)</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">data assimilation</span><span class="label label-info tag link-tag">forecast</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NSF NCAR is providing a NetCDF-4 structured version of the 0.25 degree atmospheric ECMWF Reanalysis 5 (ERA5) to the AWS ODSP. ERA5 is produced using high-resolution forecasts (HRES) at 31 kilometer resolution (one fourth the spatial resolution of the operational model) and a 62 kilometer resolution ten member 4D-Var ensemble of data assimilation (EDA) in CY41r2 of ECMWF&#39;s Integrated Forecast System (IFS) with 137 hybrid sigma-pressure (model) levels in the vertical, up to a top level of 0.01 hPa. Atmospheric data on these levels are interpolated to 37 pressure levels (the same levels as in...</p>
          <p><a href="/nsf-ncar-era5/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://rmets.onlinelibrary.wiley.com/doi/10.1002/qj.3803" class="" target="_none">The ERA5 global reanalysis</a> by <a href="https://orcid.org/0000-0001-5330-7071" target="_none">Hersbach et al 2020</a></li></a>
          </li>
        </ul>
        <p><a href="/nsf-ncar-era5/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="nyumets-brain" class="dataset">
          <h3><a href="/nyumets-brain/">NYUMets Brain Dataset</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">medical imaging</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">segmentation</span></p>
          <p>This dataset contains 8,000+ brain MRIs of 2,000+ patients with brain metastases.</p>
          <p><a href="/nyumets-brain/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.researchsquare.com/article/rs-2444113/v1" class="" target="_none">Longitudinal deep neural networks for assessing metastatic brain cancer on a massive open benchmark.</a> by <a href="https://scholar.google.com/citations?user&#x3D;bIAmWgUAAAAJ&amp;hl&#x3D;en" target="_none">Link et al (2023)</a></li></a>
          </li>
        </ul>
        <p><a href="/nyumets-brain/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="naturalearth" class="dataset">
          <h3><a href="/naturalearth/">Natural Earth</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">tiles</span></p>
          <p>Natural Earth is a public domain map dataset available at 1:10m, 1:50m, and 1:110 million scales. Featuring tightly integrated vector and raster data, with Natural Earth you can make a variety of visually pleasing, well-crafted maps with cartography or GIS software.</p>
          <p><a href="/naturalearth/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://shadedrelief.com/NEV/natural_earth_vector_cp64.pdf" class="" target="_none">Natural Earth Vector (2009)</a> by <a href="http://shadedrelief.com/" target="_none">Nathaniel Vaughn Kelso, Tom Patterson</a></li></a>
          </li>
        </ul>
        <p><a href="/naturalearth/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="nj-imagery" class="dataset">
          <h3><a href="/nj-imagery/">New Jersey Statewide Digital Aerial Imagery Catalog</a></h3>
          <p><span class="label label-info tag link-tag">aerial imagery</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">mapping</span></p>
          <p>The New Jersey Office of GIS, NJ Office of Information Technology manages a series of 11 digital orthophotography and scanned aerial photo maps collected at various years ranging from 1930 to 2017. Each year’s worth of imagery are available as Cloud Optimized GeoTIFF (COG) files and some years are available as compressed MrSID and/or JP2 files.  Additionally, each year of imagery is organized into a tile grid scheme covering the entire geography of New Jersey.  Many years share the same tiling grid while others have unique grids as defined by the project at the time.</p>
          <p><a href="/nj-imagery/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://njogis-imagery.s3-us-west-2.amazonaws.com/NJ+Imagery+Change.pdf" class="" target="_none">Visualize Imagery Changes</a> by <a href="https://njgin.nj.gov/njgin/edata/imagery/index.html" target="_none">stephanie.bosits@tech.nj.gov</a></li></a>
          </li>
        </ul>
        <p><a href="/nj-imagery/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="nj-lidar" class="dataset">
          <h3><a href="/nj-lidar/">New Jersey Statewide LiDAR</a></h3>
          <p><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">mapping</span></p>
          <p>Elevation datasets in New Jersey have been collected over several years as several
discrete projects.  Each project covers a geographic area, which is a subsection of
the entire state, and has differing specifications based on the available technology
at the time and project budget.  The geographic extent of one project may overlap that
of a neighboring project. Each of the 18 projects contains deliverable products such
as LAS (Lidar point cloud) files, unclassified/classified, tiled to cover project area;
relevant metadata records or documents, most adhering to the Federal Geographic Data
Com...</p>
          <p><a href="/nj-lidar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://njogis-elevation.s3-us-west-2.amazonaws.com/NJ+3D+Visualization.pdf" class="" target="_none">3D Visualization</a> by <a href="https://njgin.nj.gov/njgin/edata/elevation/index.html" target="_none">stephanie.bosits@tech.nj.us</a></li></a>
          </li>
        </ul>
        <p><a href="/nj-lidar/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ocmr_data" class="dataset">
          <h3><a href="/ocmr_data/">Ohio State Cardiac MRI Raw Data (OCMR)</a></h3>
          <p><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>OCMR is an open-access repository that provides multi-coil k-space data for cardiac cine.  The fully sampled MRI datasets are intended for quantitative comparison and evaluation of image reconstruction methods. The free-breathing, prospectively undersampled datasets are intended to evaluate their performance and generalizability qualitatively.</p>
          <p><a href="/ocmr_data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MRIOSU/OCMR/blob/master/Python/example_ocmr.ipynb" class="" target="_none">OCMR Tutorial</a> by <a href="https://cmr.engineering.osu.edu/people/trainees" target="_none">Chong Chen</a></li></a>
          </li>
        </ul>
        <p><a href="/ocmr_data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="opensurfaces" class="dataset">
          <h3><a href="/opensurfaces/">OpenSurfaces</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span></p>
          <p>A large database of annotated surfaces created from real-world consumer photographs.</p>
          <p><a href="/opensurfaces/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://opensurfaces.cs.cornell.edu/publications/opensurfaces/" class="" target="_none">OpenSurfaces: A Richly Annotated Catalog of Surface Appearance</a> by Sean Bell, Paul Upchurch, Noah Snavely, Kavita Bala</li></a>
          </li>
        </ul>
        <p><a href="/opensurfaces/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="openuniverse2024" class="dataset">
          <h3><a href="/openuniverse2024/">OpenUniverse 2024 Simulated Roman &amp; Rubin Images</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">object detection</span><span class="label label-info tag link-tag">parquet</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">survey</span></p>
          <p>This release consists of simulated data products designed to mimic observations of the same region of the sky as seen by two astronomical facilities: the Nancy Grace Roman Telescope and the Vera C. Rubin Observatory.</p>
          <p><a href="/openuniverse2024/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://irsa.ipac.caltech.edu/docs/notebooks/" class="" target="_none">Notebook Tutorials</a> by <a href="https://irsa.ipac.caltech.edu" target="_none">Caltech/IPAC-IRSA</a></li></a>
          </li>
        </ul>
        <p><a href="/openuniverse2024/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="orcasound" class="dataset">
          <h3><a href="/orcasound/">Orcasound - bioacoustic data for marine conservation</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">conservation</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>Live-streamed and archived audio data (~2018-present) from underwater microphones (hydrophones) containing marine biological signals as well as ambient ocean noise. Hydrophone placement and passive acoustic monitoring effort prioritizes detection of orca sounds (calls, clicks, whistles) and potentially harmful noise. Geographic focus is on the US/Canada critical habitat of Southern Resident killer whales (northern CA to central BC) with initial focus on inland waters of WA. In addition to the raw lossy or lossless compressed data, we provide a growing archive of annotated bioacoustic bouts.</p>
          <p><a href="/orcasound/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.github.com/orcasound" class="" target="_none">Github for our open source projects</a> by <a href="https://www.orcasound.net/support/#hack" target="_none">Orcasound open source community</a></li></a>
          </li>
        </ul>
        <p><a href="/orcasound/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ont-open-data" class="dataset">
          <h3><a href="/ont-open-data/">Oxford Nanopore Technologies Benchmark Datasets</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">fast5</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The ont-open-data registry provides reference sequencing data from Oxford Nanopore Technologies to support, 1) Exploration of the characteristics of nanopore sequence data. 2) Assessment and reproduction of performance benchmarks 3) Development of tools and methods. The data deposited showcases DNA sequences from a representative subset of sequencing chemistries. The datasets correspond to publicly-available reference samples (e.g. Genome In A Bottle reference cell lines). Raw data are provided with metadata and scripts to describe sample and data provenance.</p>
          <p><a href="/ont-open-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://labs.epi2me.io/tutorials/" class="" target="_none">ONT Dataset Tutorials</a> by <a href="https://labs.epi2me.io/" target="_none">EPI2MELabs</a></li></a>
          </li>
        </ul>
        <p><a href="/ont-open-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="jaxa-alos-palsar2-scansar" class="dataset">
          <h3><a href="/jaxa-alos-palsar2-scansar/">PALSAR-2 ScanSAR CARD4L (L2.2)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>The 25 m PALSAR-2 ScanSAR is normalized backscatter data of PALSAR-2 broad area observation mode with observation width of 350 km.
  The SAR imagery was ortho-rectificatied and slope corrected using the ALOS World 3D - 30 m (AW3D30) Digital Surface Model.
  Polarization data are stored as 16-bit digital numbers (DN).
  The DN values can be converted to gamma naught values in decibel unit (dB) using the following equation:
  γ0 = 10*log10(DN2) - 83.0 dB
  CARD4L stands for CEOS Analysis Ready Data for Land (Level 2.2) data are ortho-rectified and radiometrically terrain-corrected.
  This datase...</p>
          <p><a href="/jaxa-alos-palsar2-scansar/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.eorc.jaxa.jp/ALOS/en/dataset/alos_open_and_free_e.htm" class="" target="_none">ALOS series Open and Free Data</a> by JAXA EORC</li></a>
          </li>
        </ul>
        <p><a href="/jaxa-alos-palsar2-scansar/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="palsar-2-scansar-flooding-in-rwanda" class="dataset">
          <h3><a href="/palsar-2-scansar-flooding-in-rwanda/">PALSAR-2 ScanSAR Flooding in Rwanda (L2.1)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">deafrica</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>Torrential rainfall triggered flooding and landslides in many parts of Rwanda. The hardest-hit districts were Ngororero, Rubavu, Nyabihu, Rutsiro and Karongi. According to reports, 14 people have died in Karongi, 26 in Rutsiro, 18 in Rubavu, 19 in Nyabihu and 18 in Ngororero.Rwanda National Police reported that the Mukamira-Ngororero and Rubavu-Rutsiro roads are impassable due to flooding and landslide debris. UNITAR on behalf of United Nations Office for the Coordination of Humanitarian Affairs (OCHA) / Regional Office for Southern &amp; Eastern Africa in cooperation with Rwanda Space Agency ...</p>
          <p><a href="/palsar-2-scansar-flooding-in-rwanda/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.eorc.jaxa.jp/ALOS/en/dataset/alos_open_and_free_e.htm" class="" target="_none">ALOS series Open and Free Data by JAXA EORC</a> by JAXA EORC</li></a>
          </li>
        </ul>
        <p><a href="/palsar-2-scansar-flooding-in-rwanda/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="palsar-2-scansar-flooding-in-bangladesh" class="dataset">
          <h3><a href="/palsar-2-scansar-flooding-in-bangladesh/">PALSAR-2 ScanSAR Tropical Cycolne Mocha (L2.1)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>Tropical Cyclone Mocha began to form in the Bay of Bengal on 11 May 2023 and continues to intensify as it moves towards Myanmar and Bangladesh.Cyclone Mocha is the first storm to form in the Bay of Bengal this year and is expected to hit several coastal areas in Bangladesh on 14 May with wind speeds of up to 175 km/h.After made its landfall in the coast between Cox’s Bazar (Bangladesh) and Kyaukphyu (Myanmar) near Sittwe (Myanmar). At most, Catastrophic Damage-causing winds was possible especially in the areas of Rakhine State and Chin State, and Severe Damage-causing winds is possible in the ...</p>
          <p><a href="/palsar-2-scansar-flooding-in-bangladesh/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.eorc.jaxa.jp/ALOS/en/dataset/alos_open_and_free_e.htm" class="" target="_none">ALOS series Open and Free Data by JAXA EORC</a> by JAXA EORC</li></a>
          </li>
        </ul>
        <p><a href="/palsar-2-scansar-flooding-in-bangladesh/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allenai-quoref" class="dataset">
          <h3><a href="/allenai-quoref/">Quoref</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>24K Question/Answer (QA) pairs over 4.7K paragraphs, split between train (19K QAs), development (2.4K QAs) and a hidden test partition (2.5K QAs).</p>
          <p><a href="/allenai-quoref/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.semanticscholar.org/paper/Quoref%3A-A-Reading-Comprehension-Dataset-with-Dasigi-Liu/3838387ea8dd1bb8c2306be5a63c1c120075c5a2" class="" target="_none">Quoref: A Reading Comprehension Dataset with Questions Requiring Coreferential Reasoning</a> by Pradeep Dasigi, Nelson F. Liu, Ana Marasović, Noah A. Smith, Matt Gardner</li></a>
          </li>
        </ul>
        <p><a href="/allenai-quoref/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="rsna-abdominal-trauma-detection" class="dataset">
          <h3><a href="/rsna-abdominal-trauma-detection/">RSNA Abdominal Trauma Detection (RSNA-ABT)</a></h3>
          <p><span class="label label-info tag link-tag">computed tomography</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medical image computing</span><span class="label label-info tag link-tag">medical imaging</span><span class="label label-info tag link-tag">radiology</span><span class="label label-info tag link-tag">x-ray tomography</span></p>
          <p>Blunt force abdominal trauma is among the most common types of traumatic injury, with the most frequent cause being motor vehicle accidents. Abdominal trauma may result in damage and internal bleeding of the internal organs, including the liver, spleen, kidneys, and bowel. Detection and classification of injuries are key to effective treatment and favorable outcomes. A large proportion of patients with abdominal trauma require urgent surgery. Abdominal trauma often cannot be diagnosed clinically by physical exam, patient symptoms, or laboratory tests. Prompt diagnosis of abdominal trauma using...</p>
          <p><a href="/rsna-abdominal-trauma-detection/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.48550/arXiv.2405.19595" class="" target="_none">The RSNA Abdominal Traumatic Injury CT (RATIC) Dataset</a> by Rudie, Jeffrey D.</li></a>
          </li>
        </ul>
        <p><a href="/rsna-abdominal-trauma-detection/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="rsna-cervical-spine-fracture-detection" class="dataset">
          <h3><a href="/rsna-cervical-spine-fracture-detection/">RSNA Cervical Spine Fracture Detection (RSNA-CSF) Dataset</a></h3>
          <p><span class="label label-info tag link-tag">computed tomography</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medical image computing</span><span class="label label-info tag link-tag">medical imaging</span><span class="label label-info tag link-tag">radiology</span><span class="label label-info tag link-tag">x-ray tomography</span></p>
          <p>Over 1.5 million spine fractures occur annually in the United States alone resulting in over 17,730 spinal cord injuries annually. The most common site of spine fracture is the cervical spine. There has been a rise in the incidence of spinal fractures in the elderly and in this population, fractures can be more difficult to detect on imaging due to degenerative disease and osteoporosis. Imaging diagnosis of adult spine fractures is now almost exclusively performed with computed tomography (CT). Quickly detecting and determining the location of any vertebral fractures is essential to prevent ne...</p>
          <p><a href="/rsna-cervical-spine-fracture-detection/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubs.rsna.org/doi/full/10.1148/ryai.230034" class="" target="_none">The RSNA Cervical Spine Fracture CT Dataset</a> by Ming, Hui Lin</li></a>
          </li>
        </ul>
        <p><a href="/rsna-cervical-spine-fracture-detection/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="rsna-intracranial-hemorrhage-detection" class="dataset">
          <h3><a href="/rsna-intracranial-hemorrhage-detection/">RSNA Intracranial Hemorrhage Detection</a></h3>
          <p><span class="label label-info tag link-tag">computed tomography</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medical image computing</span><span class="label label-info tag link-tag">medical imaging</span><span class="label label-info tag link-tag">radiology</span><span class="label label-info tag link-tag">x-ray tomography</span></p>
          <p>RSNA assembled this dataset in 2019 for the RSNA Intracranial Hemorrhage Detection AI Challenge (<a href="https://www.kaggle.com/c/rsna-intracranial-hemorrhage-detection/">https://www.kaggle.com/c/rsna-intracranial-hemorrhage-detection/</a>). De-identified head CT studies were provided by four research institutions. A group of over 60 volunteer expert radiologists recruited by RSNA and the American Society of Neuroradiology labeled over 25,000 exams for the presence and subtype classification of acute intracranial hemorrhage.</p>
          <p><a href="/rsna-intracranial-hemorrhage-detection/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubs.rsna.org/doi/10.1148/ryai.2020190211" class="" target="_none">Construction of a Machine Learning Dataset through Collaboration: The RSNA 2019 Brain CT Hemorrhage Challenge</a> by Rudie, Jeffrey D.</li></a>
          </li>
        </ul>
        <p><a href="/rsna-intracranial-hemorrhage-detection/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="rsna-pulmonary-embolism-detection" class="dataset">
          <h3><a href="/rsna-pulmonary-embolism-detection/">RSNA Pulmonary Embolism Detection</a></h3>
          <p><span class="label label-info tag link-tag">computed tomography</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">medical image computing</span><span class="label label-info tag link-tag">medical imaging</span><span class="label label-info tag link-tag">radiology</span><span class="label label-info tag link-tag">x-ray tomography</span></p>
          <p>RSNA assembled this dataset in 2020 for the RSNA STR Pulmonary Embolism Detection AI Challenge (<a href="https://www.kaggle.com/c/rsna-str-pulmonary-embolism-detection/">https://www.kaggle.com/c/rsna-str-pulmonary-embolism-detection/</a>). With more than 12,000 CT pulmonary angiography (CTPA) studies contributed by five international research centers, it is the largest publicly available annotated PE dataset. RSNA collaborated with the Society of Thoracic Radiology to recruit more than 80 expert thoracic radiologists who labeled the dataset with detailed clinical annotations.</p>
          <p><a href="/rsna-pulmonary-embolism-detection/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubs.rsna.org/doi/full/10.1148/ryai.2021200254" class="" target="_none">The RSNA Pulmonary Embolism CT Dataset</a> by Colak, Errol</li></a>
          </li>
        </ul>
        <p><a href="/rsna-pulmonary-embolism-detection/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allenai-ropes" class="dataset">
          <h3><a href="/allenai-ropes/">Reasoning Over Paragraph Effects in Situations (ROPES)</a></h3>
          <p><span class="label label-info tag link-tag">json</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>14k QA pairs over 1.7K paragraphs, split between train (10k QAs), development (1.6k QAs) and a hidden test partition (1.7k QAs).</p>
          <p><a href="/allenai-ropes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.semanticscholar.org/paper/Reasoning-Over-Paragraph-Effects-in-Situations-Lin-Tafjord/2ebb01d08022a52c1025302379873dedb5100035" class="" target="_none">Reasoning Over Paragraph Effects in Situations</a> by Kevin Lin, Oyvind Tafjord, Peter Clark, Matt Gardner</li></a>
          </li>
        </ul>
        <p><a href="/allenai-ropes/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="silam" class="dataset">
          <h3><a href="/silam/">SILAM Air Quality</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Air Quality is a global SILAM atmospheric composition and air quality forecast performed on a daily basis for &gt; 100 species and covering the troposphere and the stratosphere. The output produces 3D concentration fields and aerosol optical thickness. The data are unique: 20km resolution for global AQ models is unseen worldwide.</p>
          <p><a href="/silam/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/fmidev/opendata-resources/tree/master/examples/python" class="" target="_none">Simple examples</a> by Roope Tervo</li></a>
          </li>
        </ul>
        <p><a href="/silam/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="safecast" class="dataset">
          <h3><a href="/safecast/">Safecast</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">radiation</span></p>
          <p>An ongoing collection of radiation and air quality measurements taken by devices involved in the Safecast project.</p>
          <p><a href="/safecast/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://safecast.org/tilemap/" class="" target="_none">Safecast Map</a> by <a href="https://github.com/frangible" target="_none">Nick Dolezal</a></li></a>
          </li>
        </ul>
        <p><a href="/safecast/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="allen-sea-ad-atlas" class="dataset">
          <h3><a href="/allen-sea-ad-atlas/">Seattle Alzheimer&#x27;s Disease Brain Cell Atlas (SEA-AD)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cell biology</span><span class="label label-info tag link-tag">cell imaging</span><span class="label label-info tag link-tag">epigenomics</span><span class="label label-info tag link-tag">gene expression</span><span class="label label-info tag link-tag">histopathology</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">microscopy</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroscience</span><span class="label label-info tag link-tag">single-cell transcriptomics</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>The Seattle Alzheimer&#39;s Disease Brain Cell Atlas (SEA-AD) consortium strives to gain a deep molecular and cellular understanding of the early pathogenesis of Alzheimer&#39;s disease and is funded by the National Institutes on Aging (NIA U19AG060909). The SEA-AD datasets available here comprise single cell profiling (transcriptomics and epigenomics) and quantitative neuropathology. To explore gene expression and chromatin accessibility information, the single-cell profiling data includes: snRNAseq and snATAC-seq data from the SEA-AD donor cohort (aged brains which span the spectrum of Alzhe...</p>
          <p><a href="/allen-sea-ad-atlas/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://sea-ad.org" class="" target="_none">Seattle Alzheimer’s Disease Brain Cell Atlas</a> by <a href="www.alleninstitute.org" target="_none">Lein, E et al.</a></li></a>
          </li>
        </ul>
        <p><a href="/allen-sea-ad-atlas/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="sentinel1-slc" class="dataset">
          <h3><a href="/sentinel1-slc/">Sentinel-1 SLC dataset for Germany</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>The Sentinel1 Single Look Complex (SLC) unzipped dataset contains Synthetic Aperture Radar (SAR) data from the European Space Agency’s Sentinel-1 mission. Different from the zipped data provided by ESA, this dataset allows direct access to individual swaths required for a given study area, thus drastically minimizing the storage and downloading time requirements of a project. Since the data is stored on S3, users can utilize the boto3 library and s3 get_object method to read the entire content of the object into the memory for processing, without actually having to download it. The Sentinel-1 ...</p>
          <p><a href="/sentinel1-slc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/live-eo/sentinel1-slc/blob/main/docs/tutorial_InSAR.md" class="" target="_none">Interferometric Synthetic Aperture Radar Tutorial</a> by <a href="https://live-eo.com/" target="_none">LiveEO</a></li></a>
          </li>
        </ul>
        <p><a href="/sentinel1-slc/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="singlecellhumanbloodatlas" class="dataset">
          <h3><a href="/singlecellhumanbloodatlas/">Single-Cell Atlas of Human Blood During Healthy Aging</a></h3>
          <p><span class="label label-info tag link-tag">protein</span><span class="label label-info tag link-tag">single-cell transcriptomics</span></p>
          <p>Comprehensive, large-scale single-cell profiling of healthy human blood at different ages is one of the critical pending tasks required to establish a framework for systematic understanding of human aging. Here, using single-cell RNA/TCR/BCR-seq with protein feature barcoding (20 antibodies), we profiled 317 samples from 166 healthy individuals aged 25 to 85 years old drawn over 3-year period. Dataset spanning ~2 million cells describes 50 subpopulations of blood immune cells, with 14 subpopulations changing with age, including a novel NKG2C+ CD8 Tcm population that decreases with age. We desc...</p>
          <p><a href="/singlecellhumanbloodatlas/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1016/j.immuni.2023.10.013" class="" target="_none">Single-cell atlas of healthy human blood unveils age-related loss of NKG2C+GZMB−CD8+ memory T cells and accumulation of type 2 memory T cells</a> by Terekhova M, Swain A, Bohacova P, Aladyeva E, Arthur L, et al</li></a>
          </li>
        </ul>
        <p><a href="/singlecellhumanbloodatlas/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="spitzer-seip" class="dataset">
          <h3><a href="/spitzer-seip/">Spitzer Enhanced Imaging Products (SEIP) Super Mosaics</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">survey</span></p>
          <p>Spitzer was an infrared astronomy space telescope with imaging from 3 to 160 microns and spectroscopy from 5 to 37 microns, launched into an Earth-trailing solar orbit as the last of NASA&#39;s Great Observatories. The SEIP Super Mosaics include data from the four channels of IRAC (3.6, 4.5, 5.8, 8 microns) and the 24 micron channel of MIPS. Data from multiple programs are combined where appropriate. Cryogenic Release v3.0 includes Spitzer data taken during commissioning and cryogenic operations, including calibration data.</p>
          <p><a href="/spitzer-seip/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://irsa.ipac.caltech.edu/docs/notebooks/" class="" target="_none">Notebook Tutorials</a> by <a href="https://irsa.ipac.caltech.edu" target="_none">Caltech/IPAC-IRSA</a></li></a>
          </li>
        </ul>
        <p><a href="/spitzer-seip/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="nrel-pds-sup3rcc" class="dataset">
          <h3><a href="/nrel-pds-sup3rcc/">Sup3rCC</a></h3>
          <p><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">solar</span></p>
          <p>Released to the public as part of the Department of Energy&#39;s Open Energy Data Initiative, 
these data represent a serially complete collection of hourly 4km wind, solar, temperature, 
humidity, and pressure fields for the Continental United States under climate change scenarios.Sup3rCC is downscaled Global Climate Model (GCM) data. For example, the initial file set tagged
&quot;sup3rcc_conus_mriesm20_ssp585_r1i1p1f1&quot; is downscaled from MRI ESM 2.0 for climate change 
scenario SSP5 8.5 and variant label r1i1p1f1. The downscaling process is performed using a 
generative machine learning...</p>
          <p><a href="/nrel-pds-sup3rcc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/NREL/sup3r/blob/main/examples/sup3rcc/using_the_data.ipynb" class="" target="_none">Using the Sup3rCC Data</a> by <a href="https://orcid.org/0000-0001-8616-8100" target="_none">Grant Buster</a></li></a>
          </li>
        </ul>
        <p><a href="/nrel-pds-sup3rcc/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="schweizer-haltestellen-oev" class="dataset">
          <h3><a href="/schweizer-haltestellen-oev/">Swiss Public Transport Stops</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">traffic</span><span class="label label-info tag link-tag">transportation</span></p>
          <p>The basic geo-data set for public transport stops comprises public transport stops in Switzerland and additional selected geo-referenced public transport locations that are of operational or structural importance (operating points).</p>
          <p><a href="/schweizer-haltestellen-oev/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://map.geo.admin.ch/?layers&#x3D;ch.bav.haltestellen-oev" class="" target="_none">Map Viewer</a> by <a href="https://twitter.com/swiss_geoportal" target="_none">Swiss Geoportal</a></li></a>
          </li>
        </ul>
        <p><a href="/schweizer-haltestellen-oev/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="synthea-coherent-data" class="dataset">
          <h3><a href="/synthea-coherent-data/">Synthea Coherent Data Set</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">dicom</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span></p>
          <p>This is a synthetic data set that includes FHIR resources, DICOM images, genomic data, physiological data (i.e., ECGs), and simple clinical notes. FHIR links all the data types together.</p>
          <p><a href="/synthea-coherent-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.3390/electronics11081199" class="" target="_none">The “Coherent Data Set”: Combining Patient Data and Imaging in a Comprehensive Synthetic Health Record.</a> by Walonoski J, Hall D, Bates KM, Farris MH, Dagher J, Downs ME, Sivek RT, Wellner B, Gregorowicz A, Hadley M, Campion FX, Levine L, Wacome K, Emmer G, Kemmer A, Malik M, Hughes J, Granger E, Russell S.</li></a>
          </li>
        </ul>
        <p><a href="/synthea-coherent-data/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="tabula-muris" class="dataset">
          <h3><a href="/tabula-muris/">Tabula Muris</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span></p>
          <p>Tabula Muris is a compendium of single cell transcriptomic data from the model organism <em>Mus musculus</em> comprising more than 100,000 cells from 20 organs and tissues. These data represent a new resource for cell biology, reveal gene expression in poorly characterized cell populations, and allow for direct and controlled comparison of gene expression in cell types shared between tissues, such as T-lymphocytes and endothelial cells from different anatomical locations. Two distinct technical approaches were used for most organs: one approach, microfluidic droplet-based 3’-end counting, enabled the s...</p>
          <p><a href="/tabula-muris/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.ncbi.nlm.nih.gov/pubmed/30283141" class="" target="_none">Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris.</a> by <a href="https://tabula-muris.ds.czbiohub.org/" target="_none">Tabula Muris Consortium (2019)</a></li></a>
          </li>
        </ul>
        <p><a href="/tabula-muris/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="tabula-muris-senis" class="dataset">
          <h3><a href="/tabula-muris-senis/">Tabula Muris Senis</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">single-cell transcriptomics</span></p>
          <p>Tabula Muris Senis is a comprehensive compendium of single cell transcriptomic data from the model organism <em>Mus musculus</em> comprising more than 500,000 cells from 18 organs and tissues across the mouse lifespan. We discovered cell-specific changes occurring across multiple cell types and organs, as well as age related changes in the cellular composition of different organs. Using single-cell transcriptomic data we were able to assess cell type specific manifestations of different hallmarks of aging, such as senescence, changes in the activity of metabolic pathways, depletion of stem-cell populat...</p>
          <p><a href="/tabula-muris-senis/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aws.amazon.com/blogs/industries/fast-queries-of-scrnaseq-datasets-with-amazon-athena/" class="aws-link" target="_none">Fast queries of scRNAseq datasets with Amazon Athena</a> by Andrew Ang, James Golden, Lisa McFerrin, and Lee Pang</li></a>
          </li>
        </ul>
        <p><a href="/tabula-muris-senis/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="tabula-sapiens" class="dataset">
          <h3><a href="/tabula-sapiens/">Tabula Sapiens</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">single-cell transcriptomics</span></p>
          <p>Tabula Sapiens will be a benchmark, first-draft human cell atlas of two million cells from 25 organs of eight normal human subjects. 
Taking the organs from the same individual controls for genetic background, age, environment, and epigenetic effects, and allows detailed analysis and comparison of cell types that are shared between tissues. 
Our work creates a detailed portrait of cell types as well as their distribution and variation in gene expression across tissues and within the endothelial, epithelial, stromal and immune compartments. 
A critical factor in the Tabula projects is our large collaborative network of PI’s with deep expertise at preparation of diverse organs, enabling all organs from a subject to be successfully processed within a single day. 
Tabula Sapiens leverages our network of human tissue experts and a close collaboration with a Donor Network West, a not-for-profit organ procurement organization. 
We use their experience to balance and assign cell types from each tissue compartment and optimally mix high-quality plate-seq data and high-volume droplet-based data to provide a broad and deep benchmark atlas. 
Our goal is to make sequence data rapidly and broadly available to the scientific community as a community resource. Before you use our data, please take note of our Data Release Policy below.</br></br>Data Release Policy</br></br>Our goal is to make sequence data rapidly and broadly available to the scientific community as a community resource. It is our intention to publish the work of this project in a timely fashion, and we welcome collaborative interaction on the project and analyses. 
However, considerable investment was made in generating these data and we ask that you respect rights of first publication and acknowledgment as outlined in the <a href="https://www.nature.com/articles/461168a">Toronto agreement</a>. 
By accessing these data, you agree not to publish any articles containing analyses of genes, cell types or transcriptomic data on a who...</p>
          <p><a href="/tabula-sapiens/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.science.org/doi/10.1126/science.abl4896" class="" target="_none">The Tabula Sapiens: a multiple organ single cell transcriptomic atlas of humans</a> by The Tabula Sapiens Consortium, Stephen R Quake</li></a>
          </li>
        </ul>
        <p><a href="/tabula-sapiens/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-nesdis-tcprimed-pds" class="dataset">
          <h3><a href="/noaa-nesdis-tcprimed-pds/">Tropical Cyclone Precipitation, Infrared, Microwave, and Environmental Dataset (TC PRIMED)</a></h3>
          <p><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geophysics</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">precipitation</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Tropical Cyclone Precipitation, Infrared, Microwave and Environmental Dataset (TC PRIMED) is a dataset centered around passive microwave observations of global tropical cyclones from low-Earth-orbiting satellites. TC PRIMED is a compilation of tropical cyclone data from various sources, including 1) tropical cyclone information from the National Oceanic and Atmospheric Administration (NOAA) National Weather Service National Hurricane Center (NHC) and Central Pacific Hurricane Center (CPHC) and the U.S. Department of Defense Joint Typhoon Warning Center, 2) low-Earth-orbiting satellite obse...</p>
          <p><a href="/noaa-nesdis-tcprimed-pds/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://rammb-data.cira.colostate.edu/tcprimed/products.html" class="" target="_none">NOAA Center for Artificial Intelligence (NCAI) TC PRIMED Learning Journey Jupyter Notebooks</a> by Naufal Razin (CSU/CIRA), Kathy Haynes (CSU/CIRA), Chris Slocum (NOAA/STAR)</li></a>
          </li>
        </ul>
        <p><a href="/noaa-nesdis-tcprimed-pds/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="census-dataworld-pums" class="dataset">
          <h3><a href="/census-dataworld-pums/">U.S. Census ACS PUMS</a></h3>
          <p><span class="label label-info tag link-tag">census</span><span class="label label-info tag link-tag">statistics</span><span class="label label-info tag link-tag">survey</span></p>
          <p>U.S. Census Bureau American Community Survey (ACS) Public Use Microdata Sample (PUMS) available in a linked data format using the Resource Description Framework (RDF) data model.</p>
          <p><a href="/census-dataworld-pums/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://docs.data.world/uscensus/#50---getting-started" class="" target="_none">Setting up Blazegraph on EC2</a> by <a href="https://data.world/" target="_none">data.world</a></li></a>
          </li>
        </ul>
        <p><a href="/census-dataworld-pums/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ukbppp" class="dataset">
          <h3><a href="/ukbppp/">UK Biobank Pharma Proteomics Project (UKB-PPP)</a></h3>
          <p><span class="label label-info tag link-tag">genome wide association study</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population genetics</span></p>
          <p>The UKB-PPP is a collaboration between the UK Biobank (UKB) and thirteen biopharmaceutical companies characterising the plasma proteomic profiles of 54,219 UKB participants. As part of a collaborative analysis across the thirteen UKB-PPP partners, we conducted comprehensive protein quantitative trait loci (pQTL) mapping of 2,923 proteins that identifies 14,287 primary genetic associations, of which 85% are newly discovered, in addition to ancestry-specific pQTL mapping in non-Europeans. We identify independent secondary associations in 87% of cis and 30% of trans loci, expanding the catalogue ...</p>
          <p><a href="/ukbppp/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://doi.org/10.1038/s41586-023-06592-6" class="" target="_none">Plasma proteomic associations with genetics and health in the UK Biobank</a> by Sun B, Chiou J, Traylor M, Benner C, Hsu Y, Richardson T, et al</li></a>
          </li>
        </ul>
        <p><a href="/ukbppp/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="wise-unwise" class="dataset">
          <h3><a href="/wise-unwise/">Unblurred Coadds of the Wide-field Infrared Survey Explorer (unWISE)</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span><span class="label label-info tag link-tag">object detection</span><span class="label label-info tag link-tag">parquet</span><span class="label label-info tag link-tag">survey</span></p>
          <p>unWISE is a reprocessing of Wide-field Infrared Survey Explorer (WISE) data which preserves the native angular resolution and is optimized for forced photometry. WISE was a NASA satellite producing all-sky imaging in four infrared bands centered at 3.4, 4.6, 12 and 22 microns (W1, W2, W3, and W4) starting in 2010 until the coolant was exhausted in 2011. It was reactivated in 2013 as NEOWISE and continued imaging in W1 and W2 until 2024.</p>
          <p><a href="/wise-unwise/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://irsa.ipac.caltech.edu/docs/notebooks/" class="" target="_none">Notebook Tutorials</a> by <a href="https://irsa.ipac.caltech.edu" target="_none">Caltech/IPAC-IRSA</a></li></a>
          </li>
        </ul>
        <p><a href="/wise-unwise/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="virtual_shizuoka" class="dataset">
          <h3><a href="/virtual_shizuoka/">Virtual Shizuoka, 3D Point Cloud Data</a></h3>
          <p><span class="label label-info tag link-tag">bathymetry</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">japanese</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">mapping</span></p>
          <p>This dataset comprises high-precision 3D point cloud data that encompasses the entire Shizuoka prefecture in Japan, covering 7,200 out of its 7,777 square kilometers. The data is produced through aerial laser survey, airborne laser bathymetry and mobile mapping systems, the culmination of many years of dedicated effort.This data will be visualized and analyzed for use in infrastructure maintenance, disaster prevention measures and autonomous vehicle driving.</p>
          <p><a href="/virtual_shizuoka/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aigid.jp/opendata_virutalshizuoka/tutorials.md" class="" target="_none">Tutorial of handling LAS format point cloud data</a> by AIGID</li></a>
          </li>
        </ul>
        <p><a href="/virtual_shizuoka/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="vitaldb" class="dataset">
          <h3><a href="/vitaldb/">VitalDB</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">signal processing</span></p>
          <p>VitalDB, a high-fidelity multi-parameter vital signs database in surgical patients.</p>
          <p><a href="/vitaldb/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.nature.com/articles/s41597-022-01411-5" class="" target="_none">VitalDB, a high-fidelity multi-parameter vital signs database in surgical patients</a> by Hyung-Chul Lee, Yoonsang Park, Soo Bin Yoon, Seong Mi Yang, Dongnyeok Park, and Chul-Woo Jung</li></a>
          </li>
        </ul>
        <p><a href="/vitaldb/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="lab41-sri-voices" class="dataset">
          <h3><a href="/lab41-sri-voices/">Voices Obscured in Complex Environmental Settings (VOiCES)</a></h3>
          <p><span class="label label-info tag link-tag">automatic speech recognition</span><span class="label label-info tag link-tag">denoising</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">speaker identification</span><span class="label label-info tag link-tag">speech processing</span></p>
          <p>VOiCES is a speech corpus recorded in acoustically challenging settings,
using distant microphone recording. Speech was recorded in real rooms with various
acoustic features (reverb, echo, HVAC systems, outside noise, etc.). Adversarial noise,
either television, music, or babble, was concurrently played with clean speech.
Data was recorded using multiple microphones strategically placed
throughout the room. The corpus includes audio recordings, orthographic transcriptions,
and speaker labels.</p>
          <p><a href="/lab41-sri-voices/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/voices18/utilities/blob/master/Using%20VOiCES%20corpus%20tutorial.ipynb" class="" target="_none">Getting started with VOiCES data</a> by M.A. Barrios</li></a>
          </li>
        </ul>
        <p><a href="/lab41-sri-voices/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="wbg-cckp" class="dataset">
          <h3><a href="/wbg-cckp/">World Bank Climate Change Knowledge Portal (CCKP)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">climate projections</span><span class="label label-info tag link-tag">CMIP6</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">netcdf</span></p>
          <p>CCKP provides open access to a comprehensive suite of climate and climate change resources derived from the latest generation of climate data archives. Products are based on a consistent and transparent approach with a systematic way of pre-processing the raw observed and model-based projection data to enable inter-comparable use across a broad range of applications. Climate products consist of basic climate variables as well as a large collection (70+) of more specialized, application-orientated variables and indices across different scenarios. Precomputed data can be extracted per specified ...</p>
          <p><a href="/wbg-cckp/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://worldbank.github.io/climateknowledgeportal/" class="" target="_none">World Bank Climate Change Knowledge Portal observed and projected climate datasets</a> by <a href="https://climateknowledgeportal.worldbank.org" target="_none">World Bank</a></li></a>
          </li>
        </ul>
        <p><a href="/wbg-cckp/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="xiph-media" class="dataset">
          <h3><a href="/xiph-media/">Xiph.Org Test Media</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">media</span><span class="label label-info tag link-tag">movies</span><span class="label label-info tag link-tag">multimedia</span><span class="label label-info tag link-tag">video</span></p>
          <p>Uncompressed video used for video compression and video processing research.</p>
          <p><a href="/xiph-media/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://media.xiph.org/aws.html" class="" target="_none">Encoding video with AV1 on EC2</a> by <a href="https://www.xiph.org/" target="_none">Thomas Daede</a></li></a>
          </li>
        </ul>
        <p><a href="/xiph-media/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="zinc15" class="dataset">
          <h3><a href="/zinc15/">ZINC Database</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">chemical biology</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular docking</span><span class="label label-info tag link-tag">pharmaceutical</span><span class="label label-info tag link-tag">protein</span></p>
          <p>3D models for molecular docking screens.</p>
          <p><a href="/zinc15/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://zinc15.docking.org" class="" target="_none">ZINC Database</a> by <a href="http://irwinlab.compbio.ucsf.edu" target="_none">John Irwin</a></li></a>
          </li>
        </ul>
        <p><a href="/zinc15/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ihart" class="dataset">
          <h3><a href="/ihart/">iHART Whole Genome Sequencing Data Set</a></h3>
          <p><span class="label label-info tag link-tag">autism spectrum disorder</span><span class="label label-info tag link-tag">bam</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">vcf</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>iHART is the <a href="http://www.thehartwellfoundation.org/">Hartwell Foundation</a>’s Autism Research and Technology Initiative. This release contains whole genome data from over 1000 families with 2 or more children with autism, of which biomaterials were provided by the Autism Genetic Resource Exchange (<a href="http://research.agre.org/">AGRE</a>).</p>
          <p><a href="/ihart/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)30780-9" class="" target="_none">Inherited and De Novo Genetic Risk for Autism Impacts Shared Networks</a> by Ruzzo et al. (2020)</li></a>
          </li>
        </ul>
        <p><a href="/ihart/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="recount" class="dataset">
          <h3><a href="/recount/">recount3</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">gene expression</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neuroscience</span><span class="label label-info tag link-tag">transcriptomics</span></p>
          <p>recount3 is an online resource consisting of RNA-seq gene, exon, and exon-exon junction counts as well as coverage bigWig files for 8,679 and 10,088 different studies for human and mouse respectively. It is the third generation of the ReCount project and part of recount.bio. recount2 is also included for historical purposes. The pipeline used to generate the data in recount3 (but not recount2) is available <a href="https://github.com/langmead-lab/monorail-external">here</a>.</p>
          <p><a href="/recount/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.bioconductor.org/packages/devel/bioc/vignettes/recount3/inst/doc/recount3-quickstart.html" class="" target="_none">recount3 quick start guide</a> by <a href="https://lcolladotor.github.io/" target="_none">Leonardo Collado-Torres</a></li></a>
          </li>
        </ul>
        <p><a href="/recount/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="noaa-nws-graphcastgfs-pds" class="dataset">
          <h3><a href="/noaa-nws-graphcastgfs-pds/">(EXPERIMENTAL) NOAA GraphCast Global Forecast System (GFS) (EXPERIMENTAL)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The GraphCast Global Forecast System (GraphCastGFS) is an experimental system set up by the National Centers for Environmental Prediction (NCEP) to produce medium range global forecasts. The horizontal resolution is a 0.25 degree latitude-longitude grid (about 28 km). The model runs 4 times a day at 00Z, 06Z, 12Z and 18Z cycles. Major atmospheric and surface fields including temperature, wind components, geopotential height, specific humidity, and vertical velocity, are available. The products are 6 hourly forecasts up to 10 days. The data format is GRIB2.
<br/>
<br/>
The GraphCastGFS system is an experimental weather forecast model built upon the pre-trained Google DeepMind’s GraphCast Machine Learning Weather Prediction (MLWP) model. The GraphCast model is implemented as a message-passing graph neural network (GNN) architecture with “encoder-processor-decoder” configuration. It uses an icosahedron grid with multiscale edges and has around 37 million parameters. This model is pre-trained with ECMWF’s ERA5 reanalysis data. The GraphCastGFSl takes two model states as initial conditions (current and 6-hr previous states) from NCEP 0.25 degree GDAS analysis data and runs GraphCast (37 levels) and GraphCast_operational (13 levels) with a pre-trained model provided by GraphCast. Unit conversion to the GDAS data is conducted to match the input data required by GraphCast and to generate forecast products consistent with GFS from GraphCastGFS’ native forecast data.
<br/>
<br/>
<strong>The GraphCastGFS version 2 made the following changes from the GraphcastCastGFS version 1.</strong>
<br/><ol>
<li>The 37 vertical levels model is removed due to the storage restriction and limited accuracy.<br/></li>
<li>The 13 levels graphcast ML model was fine-tuned with NCEP’s GDAS data as inputs and ECMWF ERA5 data as ground truth from 20210323 to 20220901, validated from 20220901 to 20230101. Evaluation is done with forecasts from 20230101-20240101. The new weights created from the training are used to create global forecasts. It is important to note that the GraphCastGFS v1 model weights obtained from Google’s DeepMInd were provided based on 12 timesteps training with ERA5 data, while the GraphCastGFS v2 model weights resulted from training with 14 timesteps with GDAS and ERA5 data that significantly increased the accuracy of the forecasts compared with GraphCastGFS V1.<br/>
<br/>
The input data generated from the GDAS data as GraphCast input is provided under input/ directory. An example of file names is shown below
<br/>
<br/>
source-gdas_date-2024022000_res-0.25_levels-13_steps-2.nc
<br/>
<br/>
The files are under forecasts_13_levels/. There are 40 files under each directory covering a 10 day forecast. An example of file name is listed below
<br/>
<br/>
graphcastgfs.t00z.pgrb2.0p25.f006
<br/>
<br/></li>
</ol>
<em>The GraphCastGFS version 2.1 change log:</em>
<br/>
<br/><ol>
<li>Starting from 06 cycle on 20240710, the forecast length is increased from 10 days to 16 days.<br/>
<br/>
Please note that th...</p>
          <p><a href="/noaa-nws-graphcastgfs-pds/">Details &rarr;</a></p>
        </div>
        <div id="cse-cic-ids2018" class="dataset">
          <h3><a href="/cse-cic-ids2018/">A Realistic Cyber Defense Dataset (CSE-CIC-IDS2018)</a></h3>
          <p><span class="label label-info tag link-tag">cyber security</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">intrusion detection</span><span class="label label-info tag link-tag">network traffic</span></p>
          <p>This dataset is the result of a collaborative project between the Communications Security Establishment (CSE) and The Canadian Institute for Cybersecurity (CIC) that use the notion of profiles to generate cybersecurity dataset in a systematic manner. It incluides a detailed description of intrusions along with abstract distribution models for applications, protocols, or lower level network entities. The dataset includes seven different attack scenarios, namely Brute-force, Heartbleed, Botnet, DoS, DDoS, Web attacks, and infiltration of the network from inside. The attacking infrastructure incl...</p>
          <p><a href="/cse-cic-ids2018/">Details &rarr;</a></p>
        </div>
        <div id="allenai-tablestore-questions" class="dataset">
          <h3><a href="/allenai-tablestore-questions/">AI2 TabMCQ: Multiple Choice Questions aligned with the Aristo Tablestore</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>9092 crowd-sourced science questions and 68 tables of curated facts</p>
          <p><a href="/allenai-tablestore-questions/">Details &rarr;</a></p>
        </div>
        <div id="allenai-tablestore" class="dataset">
          <h3><a href="/allenai-tablestore/">AI2 Tablestore (November 2015 Snapshot)</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>68 tables of curated facts</p>
          <p><a href="/allenai-tablestore/">Details &rarr;</a></p>
        </div>
        <div id="allenai-aristo-mini" class="dataset">
          <h3><a href="/allenai-aristo-mini/">Aristo Mini Corpus</a></h3>
          <p><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">json</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>1,197,377 science-relevant sentences</p>
          <p><a href="/allenai-aristo-mini/">Details &rarr;</a></p>
        </div>
        <div id="allenai-tuple-kb" class="dataset">
          <h3><a href="/allenai-tuple-kb/">Aristo Tuple KB</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>294,000 science-relevant tuples</p>
          <p><a href="/allenai-tuple-kb/">Details &rarr;</a></p>
        </div>
        <div id="australasian-genomics" class="dataset">
          <h3><a href="/australasian-genomics/">Australasian Genomes</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">conservation</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">transcriptomics</span><span class="label label-info tag link-tag">wildlife</span></p>
          <p>Australasian Genomes is the genomic data repository for the Threatened Species Initiative (TSI) and the ARC Centre for Innovations in Peptide and Protein Science (CIPPS). This repository contains reference genomes, transcriptomes, resequenced genomes and reduced representation sequencing data from Australasian species. Australasian Genomes is managed by the Australasian Wildlife Genomics Group (AWGG) at the University of Sydney on behalf of our collaborators within TSI and CIPPS.</p>
          <p><a href="/australasian-genomics/">Details &rarr;</a></p>
        </div>
        <div id="bobsrepository" class="dataset">
          <h3><a href="/bobsrepository/">Baby Open Brains (BOBs) Repository on AWS</a></h3>
          <p><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span><span class="label label-info tag link-tag">nifti</span><span class="label label-info tag link-tag">pediatric</span><span class="label label-info tag link-tag">segmentation</span></p>
          <p>Manually curated and reviewed infant brain segmentations and accompanying T1w and T2w images for a range of 1-9 month old participants from the Baby Connectome Project (BCP)</p>
          <p><a href="/bobsrepository/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://bobsrepository.readthedocs.io/en/latest/data_access/#how-to-download" class="" target="_none">Downloading BOBs Repository Data from Amazon S3 Bucket</a> by <a href="https://github.com/LuciMoore" target="_none">Lucille A. Moore</a></li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2023.03.22.533696v2" class="" target="_none">BIBSNet: A Deep Learning Baby Image Brain Segmentation Network for MRI Scans</a> by <a href="https://orcid.org/0000-0001-6862-6526" target="_none">Timothy J. Hendrickson et al.</a></li></a>
          </li>
          <li>
            <a href="https://zenodo.org/records/10679463" class="" target="_none">BIBsnet</a> by Timothy J. Hendrickson et al.</li></a>
          </li>
        </ul>
        <p><a href="/bobsrepository/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="csiro-cafe60" class="dataset">
          <h3><a href="/csiro-cafe60/">CAFE60 reanalysis</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The CSIRO Climate retrospective Analysis and Forecast Ensemble system: version 1 (CAFE60v1) provides a large ensemble retrospective analysis of the global climate system from 1960 to present with sufficiently many realizations and at spatio-temporal resolutions suitable to enable probabilistic climate studies. Using a variant of the ensemble Kalman filter, 96 climate state estimates are generated over the most recent six decades. These state estimates are constrained by monthly mean ocean, atmosphere and sea ice observations such that their trajectories track the observed state while enabling ...</p>
          <p><a href="/csiro-cafe60/">Details &rarr;</a></p>
        </div>
        <div id="cgiardata" class="dataset">
          <h3><a href="/cgiardata/">CCAFS-Climate Data</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">food security</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>High resolution climate data to help assess the impacts of climate change primarily on agriculture. These open access datasets of climate projections will help researchers make climate change impact assessments.</p>
          <p><a href="/cgiardata/">Details &rarr;</a></p>
        </div>
        <div id="fast-ai-coco" class="dataset">
          <h3><a href="/fast-ai-coco/">COCO - Common Objects in Context - fast.ai datasets</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>COCO is a large-scale object detection, segmentation, and captioning dataset.
This is part of the fast.ai datasets collection hosted by AWS for convenience
of fast.ai students. If you use this dataset in your research please cite
arXiv:1405.0312 [cs.CV].</p>
          <p><a href="/fast-ai-coco/">Details &rarr;</a></p>
        </div>
        <div id="molssi-covid19-hub" class="dataset">
          <h3><a href="/molssi-covid19-hub/">COVID-19 Molecular Structure and Therapeutics Hub</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular docking</span><span class="label label-info tag link-tag">pharmaceutical</span></p>
          <p>Aggregating critical information to accelerate drug discovery for the molecular modeling and simulation community.
A community-driven data repository and curation service for molecular structures, models, therapeutics, and
simulations related to computational research related to therapeutic opportunities for COVID-19
(caused by the SARS-CoV-2 coronavirus).</p>
          <p><a href="/molssi-covid19-hub/">Details &rarr;</a></p>
        </div>
        <div id="sissa-forecast-database-dataset" class="dataset">
          <h3><a href="/sissa-forecast-database-dataset/">CRC-SAS/SISSA historical seasonal and subseasonal forecast database</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">forecast</span><span class="label label-info tag link-tag">hydrology</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">weather</span></p>
          <p>En el marco del Sistema de Información de Sequías del Sur de Sudamérica (SISSA) se ha desarrollado una base de predicciones en escala subestacional y estacional con datos corregidos y sin corregir, con el propósito que permita estudiar predictibilidad en distintas escalas y también que sirva para alimentar modelos de sectores como agricultura e hidrología. 
<br/>
<br/>
La base contiene datos en escala diaria entre 2000-2019 (sin corregir) y 2010-2019 (corregidos) para diversas variables incluyendo: temperatura media, máxima y mínima, así como también lluvia, viento medio y otras variables pensadas para alimentar modelos hidrológicos y de cultivo.
<br/>
<br/>
La base de datos abarca toda el área del Centro Regional del Clima para el sur de sudamérica (CRC-SAS), abarcando desde Bolivia y centro-sur de Brasil hasta la Patagonia incluyendo los países miembros como Chile, Argentina, Brasil, Paraguay, Uruguay y Bolivia.
<br/>
<br/>
La base fue generada a partir de datos de GEFSv12 para escala subestacional (<a href="https://www.ncei.noaa.gov/products/weather-climate-models/global-ensemble-forecast">GEFS</a>) y CFS2 para escala estacional (<a href="https://www.ncei.noaa.gov/access/metadata/landing-page/bin/iso?id=gov.noaa.ncdc:C00878">CFS2</a>). Para la generación de los datos corregidos se utilizaron los datos del reanálisis de ERA5 (<a href="https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels?tab=overview">ERA5</a>).
<br/>
<br/>
<br/>
Within the framework of the Southern South American Drought Information System (SISSA), a base of sub-seasonal and seasonal scale predictions has been developed with corrected and uncorrected data, with the purpose of studying predictability at different scales and also to be used to feed models for sectors such as agriculture and hydrology.
<br/>
<br/>
The database contains daily scale data between 2000-2019 (uncorrected) and 2010-2019 (corrected) for several variables including: mean, maximum and minimum temperature, as well as rainfall, mean wind and other variables intended to feed hydrological and crop models.
<br/>
<br/>
The database covers the entire area of the Regional Climate Center for Southern South America (CRC-SAS), from Bolivia and south-central Brazil to Patagonia, including member countries such as Chile, Argentina, Brazil, Paraguay, Uruguay and Bolivia.
<br/>
<br/>
The base was generated from GEFSv12 data for subseasonal scale (<a href="https://www.ncei.noaa.gov/products/weather-climate-models/global-ensemble-forecast">GEFS</a>) and CFS2 for seasonal scale (<a href="https://www.ncei.noaa.gov/access/metadata/landing-page/bin/iso?id=gov.noaa.ncdc:C00878">CFS2</a>). Data from the ERA5 reanalysis (<a href="https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels?tab=overview">ERA5</a>) we...</p>
          <p><a href="/sissa-forecast-database-dataset/">Details &rarr;</a></p>
        </div>
        <div id="cwa_opendata" class="dataset">
          <h3><a href="/cwa_opendata/">Central Weather Administration OpenData</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Various kinds of weather raw data and charts from Central Weather Administration.</p>
          <p><a href="/cwa_opendata/">Details &rarr;</a></p>
        </div>
        <div id="cwb_opendata" class="dataset">
          <h3><a href="/cwb_opendata/">Central Weather Bureau OpenData</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Various kinds of weather raw data and charts from Central Weather Bureau.</p>
          <p><a href="/cwb_opendata/">Details &rarr;</a></p>
        </div>
        <div id="c2smsfloods" class="dataset">
          <h3><a href="/c2smsfloods/">Cloud to Street - Microsoft Flood and Clouds Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">floods</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>This dataset consists of chips of Sentinel-1 and Sentinel-2 satellite data. Each Sentinel-1 chip contains a corresponding label for water and each Sentinel-2 chip contains a corresponding label for water and clouds. Data is stored in folders by a unique event identifier as the folder name. Within each event folder there are subfolders for Sentinel-1 (s1) and Sentinel-2 (s2) data. Each chip is contained in its own sub-folder with the folder name being the source image id, followed by a unique chip identifier consisting of a hyphenated set of 5 numbers. All bands of the satellite data, as well a...</p>
          <p><a href="/c2smsfloods/">Details &rarr;</a></p>
        </div>
        <div id="epa-equates-v1" class="dataset">
          <h3><a href="/epa-equates-v1/">Community Multiscale Air Quality (CMAQ) 2019 3D Gridded and Column data from the EPA&#x27;s Air Quality Time Series (EQUATES) Project</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">model</span></p>
          <p>The data are part of EPA’s Air Quality Time Series (EQUATES) Project.  The data consist of hourly gridded pollutant concentrations estimates by the Community Multiscale Air Quality (CMAQ) model version 5.3.2 (<a href="https://doi.org/10.15139/S3/F2KJSK">https://doi.org/10.15139/S3/F2KJSK</a>) for January 1 – December 31, 2019.  Model data is provided for two spatial domains :  the Northern Hemisphere (108 km x 108km horizontal grid spacing) and the Contiguous United States including parts of Canada and Mexico (12km x 12km horizontal grid spacing).  Two types of hourly data are provided: three-dimensional air pollutant concentrations and vert...</p>
          <p><a href="/epa-equates-v1/">Details &rarr;</a></p>
        </div>
        <div id="darpa-invisible-headlights" class="dataset">
          <h3><a href="/darpa-invisible-headlights/">DARPA Invisible Headlights Dataset</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">broadband</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">lidar</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">segmentation</span><span class="label label-info tag link-tag">us</span></p>
          <p>&quot;The DARPA Invisible Headlights Dataset is a large-scale multi-sensor dataset annotated for autonomous, off-road navigation in challenging off-road environments. It features simultaneously collected off-road imagery from multispectral, hyperspectral, polarimetric, and broadband sensors spanning wave-lengths from the visible spectrum to long-wave infrared and provides aligned LIDAR data for ground-truth shape. Camera calibrations, LiDAR registrations, and traversability annotations for a subset of the data are available.&quot;</p>
          <p><a href="/darpa-invisible-headlights/">Details &rarr;</a></p>
        </div>
        <div id="marine-energy-data" class="dataset">
          <h3><a href="/marine-energy-data/">Department of Energy&#x27;s Marine Energy Data Lake</a></h3>
          <p><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">marine</span><span class="label label-info tag link-tag">water</span></p>
          <p>Data released from projects funded by the Department of Energy&#39;s Water Power Technologies Office (DOE WPTO) 
that are too large or complex to be conveniently accessed by traditional means. The Marine Energy data lake 
aims to improve and automate access of high-value MHK data sets, making data actionable and discoverable by 
researchers and industry to accelerate analysis and advance innovation. 
This data lake is a sister-data lake to the Department of Energy’s Open Energy Data Initiative (OEDI) data lake. </p>
          <p><a href="/marine-energy-data/">Details &rarr;</a></p>
        </div>
        <div id="dc-lidar" class="dataset">
          <h3><a href="/dc-lidar/">District of Columbia - Classified Point Cloud LiDAR</a></h3>
          <p><span class="label label-info tag link-tag">cities</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">us-dc</span></p>
          <p>LiDAR point cloud data for Washington, DC is available for anyone to use on Amazon S3.
This dataset, managed by the Office of the Chief Technology Officer (OCTO), through the
direction of the District of Columbia GIS program, contains tiled point cloud data for
the entire District along with associated metadata.</p>
          <p><a href="/dc-lidar/">Details &rarr;</a></p>
        </div>
        <div id="epa-edde-v1" class="dataset">
          <h3><a href="/epa-edde-v1/">EPA Dynamically Downscaled Ensemble (EDDE) Version 1</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">climate projections</span><span class="label label-info tag link-tag">CMIP5</span><span class="label label-info tag link-tag">CMIP6</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">Eulerian</span><span class="label label-info tag link-tag">events</span><span class="label label-info tag link-tag">floods</span><span class="label label-info tag link-tag">fluid dynamics</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">hdf5</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">HPC</span><span class="label label-info tag link-tag">hydrology</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">land use</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">near-surface air temperature</span><span class="label label-info tag link-tag">near-surface relative humidity</span><span class="label label-info tag link-tag">near-surface specific humidity</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">physics</span><span class="label label-info tag link-tag">post-processing</span><span class="label label-info tag link-tag">precipitation</span><span class="label label-info tag link-tag">radiation</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">us</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The data are a subset of the EPA Dynamically Downscaled Ensemble (EDDE),  Version 1. EDDE is a collection of physics-based modeled data that represent 3D atmospheric conditions for historical and future periods under different scenarios. The EDDE Version 1 datasets cover the contiguous United States at a horizontal grid spacing of 36 kilometers at hourly increments. EDDE Version 1 includes simulations that have been dynamically downscaled from multiple global climate models (GCMs) under both mid- and high-emission scenarios from the Fifth Coupled Model Intercomparison Project (CMIP5) using the...</p>
          <p><a href="/epa-edde-v1/">Details &rarr;</a></p>
        </div>
        <div id="epa-edde-v2" class="dataset">
          <h3><a href="/epa-edde-v2/">EPA Dynamically Downscaled Ensemble (EDDE) Version 2</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">climate model</span><span class="label label-info tag link-tag">climate projections</span><span class="label label-info tag link-tag">CMIP5</span><span class="label label-info tag link-tag">CMIP6</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">Eulerian</span><span class="label label-info tag link-tag">events</span><span class="label label-info tag link-tag">floods</span><span class="label label-info tag link-tag">fluid dynamics</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">hdf5</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">HPC</span><span class="label label-info tag link-tag">hydrology</span><span class="label label-info tag link-tag">infrastructure</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">land use</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">near-surface air temperature</span><span class="label label-info tag link-tag">near-surface relative humidity</span><span class="label label-info tag link-tag">near-surface specific humidity</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">physics</span><span class="label label-info tag link-tag">post-processing</span><span class="label label-info tag link-tag">precipitation</span><span class="label label-info tag link-tag">radiation</span><span class="label label-info tag link-tag">simulations</span><span class="label label-info tag link-tag">us</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The data are a subset of the EPA Dynamically Downscaled Ensemble (EDDE),  Version 2. EDDE is a collection of physics-based modeled data that represent 3D atmospheric conditions for historical and future periods under different scenarios. The EDDE Version 2 datasets cover the contiguous United States at a horizontal grid spacing of 12 kilometers at hourly increments. EDDE Version 2 will include simulations that have been dynamically downscaled from multiple global climate models (GCMs) under multiple emission scenarios from the Sixth Coupled Model Intercomparison Project (CMIP6) using the Weath...</p>
          <p><a href="/epa-edde-v2/">Details &rarr;</a></p>
        </div>
        <div id="epa-rsei-pds" class="dataset">
          <h3><a href="/epa-rsei-pds/">EPA Risk-Screening Environmental Indicators</a></h3>
          <p><span class="label label-info tag link-tag">environmental</span></p>
          <p>Detailed air model results from EPA’s Risk-Screening Environmental Indicators (RSEI) model.</p>
          <p><a href="/epa-rsei-pds/">Details &rarr;</a></p>
        </div>
        <div id="epoch-of-reionization" class="dataset">
          <h3><a href="/epoch-of-reionization/">Epoch of Reionization Dataset</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The data are from observations with the Murchison Widefield Array (MWA) which is a
Square Kilometer Array (SKA) precursor in Western Australia.  This particular
dataset is from the Epoch of Reionization project which is a key science driver
of the SKA. Nearly 2PB of such observations have been recorded to date, this is
a small subset of that which has been exported from the MWA data archive in
Perth and made available to the public on AWS.  The data were taken to detect
signatures of the first stars and galaxies forming and the effect of these early
stars and galaxies on the evolution of the u...</p>
          <p><a href="/epoch-of-reionization/">Details &rarr;</a></p>
        </div>
        <div id="ncbi-fcs-gx" class="dataset">
          <h3><a href="/ncbi-fcs-gx/">GX database for NCBI Foreign Contamination Screen (FCS) Tool Suite</a></h3>
          <p><span class="label label-info tag link-tag">assembly</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">contamination</span><span class="label label-info tag link-tag">fasta</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genome</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>Sequence database used by FCS-GX (Foreign Contamination Screen - Genome Cross-species aligner) to detect contamination from foreign organisms in genome sequences.</p>
          <p><a href="/ncbi-fcs-gx/">Details &rarr;</a></p>
        </div>
        <div id="galex" class="dataset">
          <h3><a href="/galex/">Galaxy Evolution Explorer Satellite (GALEX)</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The Galaxy Evolution Explorer Satellite (GALEX) was a NASA mission led by the California Institute of Technology, whose primary goal was to investigate how star formation in galaxies evolved from the early universe up to the present. GALEX used microchannel plate detectors to obtain direct images in the near-UV (NUV) and far-UV (FUV), and a grism to disperse light for low resolution spectroscopy.</p>
          <p><a href="/galex/">Details &rarr;</a></p>
        </div>
        <div id="genomeark" class="dataset">
          <h3><a href="/genomeark/">Genome Ark</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">conservation</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Genome Ark hosts genomic information for the Vertebrate Genomes Project (VGP) and other related projects. The VGP is an international collaboration that aims to generate complete and near error-free reference genomes for all extant vertebrate species. These genomes will be used to address fundamental questions in biology and disease, to identify species most genetically at risk for extinction, and to preserve genetic information of life.</p>
          <p><a href="/genomeark/">Details &rarr;</a></p>
        </div>
        <div id="gretel-synthetic-safety-alignment-en-v1" class="dataset">
          <h3><a href="/gretel-synthetic-safety-alignment-en-v1/">Gretel Synthetic Safety Alignment Dataset</a></h3>
          <p><span class="label label-info tag link-tag">ai safety</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">synthetic data</span></p>
          <p>A comprehensive dataset designed for aligning language models with safety and ethical guidelines. Contains 8,361 curated triplets of prompts, responses, and safe responses across various risk categories. Each entry includes safety scores, judge reasoning, and harm probability assessments, making it valuable for model alignment, testing, and benchmarking.</p>
          <p><a href="/gretel-synthetic-safety-alignment-en-v1/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://gretel.ai/blog/gretel-open-synthetic-safety-dataset" class="" target="_none">Building Datasets to Enable Safer AI Responses</a> by <a href="https://gretel.ai" target="_none">Gretel.ai</a></li></a>
          </li>
          <li>
            <a href="https://huggingface.co/datasets/gretelai/gretel-safety-alignment-en-v1" class="" target="_none">Getting Started with the Gretel Safety Alignment Dataset</a> by <a href="https://gretel.ai" target="_none">Gretel.ai</a></li></a>
          </li>
          <li>
            <a href="https://gretel.ai/blog/build-high-quality-datasets-for-ai-using-gretel-navigator" class="" target="_none">Gretel Navigator Data Designer</a> by <a href="https://gretel.ai" target="_none">Gretel.ai</a></li></a>
          </li>
        </ul>
        <p><a href="/gretel-synthetic-safety-alignment-en-v1/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="gadal" class="dataset">
          <h3><a href="/gadal/">Grid Algorithms and Data Analytics Library (GADAL)</a></h3>
          <p><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">model</span><span class="label label-info-sustainability tag link-tag">sustainability</span></p>
          <p>The aim of this project is to create an easy-to-use platform where various types of analytics can be performed on a wide range of electrical grid datasets. The aim is to establish an open-source library of algorithms that universities, national labs and other developers can contribute to which can be used on both open-source and proprietary grid data to improve the analysis of electrical distribution systems for the grid modeling community. OEDI Systems Integration (SI) is a grid algorithms and data analytics API created to standardize how data is sent between different modules that are run as...</p>
          <p><a href="/gadal/">Details &rarr;</a></p>
        </div>
        <div id="gulfwide-avian-monitoring" class="dataset">
          <h3><a href="/gulfwide-avian-monitoring/">Gulfwide Avian Colony Monitoring Survey Photos</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">conservation</span><span class="label label-info tag link-tag">ecosystems</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">object detection</span></p>
          <p>For this project, The Water Institute (the Institute) and subcontractor Colibri Ecological Consulting, LLC (Colibri) utilized established methods and protocols capable of assessing changes of colonial waterbird populations and their important habitats within individual states and the broader northern Gulf of Mexico region.  Data collection activities included:
    Aerial Photographic Nest Surveys: Implementation of fixed-wing aircraft surveys intended to assess waterbird colonies and document associated nesting within select portions of the northern Gulf of Mexico. Additional detail is provide...</p>
          <p><a href="/gulfwide-avian-monitoring/">Details &rarr;</a></p>
        </div>
        <div id="guys-breast-cancer-lymph-nodes" class="dataset">
          <h3><a href="/guys-breast-cancer-lymph-nodes/">Guy&#x27;s Breast Cancer Lymph Nodes (GRAPE)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">breast cancer</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">computational pathology</span><span class="label label-info tag link-tag">histopathology</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>This is a retrospective dataset of 1523 H&amp;E-stained whole slide images (WSI) of lymph nodes from breast cancer patients. The cohort consisted of 177 patients (122 LN-positive - metastasis was reported in at least 1 LN - and 55 LN-negative patients) with invasive breast carcinoma treated between 1984 and 2002 at Guy’s Hospital London, UK. Slides were scanned and digitised at 40x magnification (0.23 µm/pixel), NanoZoomer H.T2.0 2.0-HT (Hamamatsu Photonics UK, Ltd, Welwyn Garden City, UK). WSIs are in .ndpi format.</p>
          <p><a href="/guys-breast-cancer-lymph-nodes/">Details &rarr;</a></p>
        </div>
        <div id="hirlam" class="dataset">
          <h3><a href="/hirlam/">HIRLAM Weather Model</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>HIRLAM (High Resolution Limited Area Model) is an operational synoptic and mesoscale weather prediction model managed by the Finnish Meteorological Institute.</p>
          <p><a href="/hirlam/">Details &rarr;</a></p>
        </div>
        <div id="wrf-se-alaska-snap" class="dataset">
          <h3><a href="/wrf-se-alaska-snap/">High Resolution Downscaled Climate Data for Southeast Alaska</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>This dataset contains historical and projected dynamically downscaled climate data for the Southeast region of the State of Alaska at 1 and 4km spatial resolution and hourly temporal resolution. Select variables are also summarized into daily resolutions. This data was produced using the Weather Research and Forecasting (WRF) model (Version 4.0). We downscaled both Climate Forecast System Reanalysis (CFSR) historical reanalysis data (1980-2019) and both historical and projected runs from two GCM’s from the Coupled Model Inter-comparison Project 5 (CMIP5): GFDL-CM3 and NCAR-CCSM4 (historical ru...</p>
          <p><a href="/wrf-se-alaska-snap/">Details &rarr;</a></p>
        </div>
        <div id="hsip-lidar-us-cities" class="dataset">
          <h3><a href="/hsip-lidar-us-cities/">Homeland Security and Infrastructure US Cities</a></h3>
          <p><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">elevation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">lidar</span></p>
          <p>The U.S. Cities elevation data collection program supported the US Department of Homeland Security Homeland Security and Infrastructure Program (HSIP). As part of the HSIP Program, there were 133+ U.S. cities that had imagery and LiDAR collected to provide the Homeland Security, Homeland Defense, and Emergency Preparedness, Response and Recovery (EPR&amp;R) community with common operational, geospatially enabled baseline data needed to analyze threat, support critical infrastructure protection and expedite readiness, response and recovery in the event of a man-made or natural disaster. As a pa...</p>
          <p><a href="/hsip-lidar-us-cities/">Details &rarr;</a></p>
        </div>
        <div id="hst" class="dataset">
          <h3><a href="/hst/">Hubble Space Telescope</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The Hubble Space Telescope (HST) is one of the most productive scientific instruments ever created. This dataset contains calibrated and raw data for all currently active instruments on HST: ACS, COS, STIS, WFC3, and FGS.</p>
          <p><a href="/hst/">Details &rarr;</a></p>
        </div>
        <div id="cmip6-era5-hybrid-southeast-asia" class="dataset">
          <h3><a href="/cmip6-era5-hybrid-southeast-asia/">Hybrid statistical-dynamic downscaling based on multi-model ensembles in Southeast Asia</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">netcdf</span><span class="label label-info tag link-tag">precipitation</span></p>
          <p>GCMs under CMIP6 have been widely used to investigate climate change impacts and put forward associated adaptation and mitigation strategies. However, the relatively coarse spatial resolutions (usually 100~300km) preclude their direct applications at regional scales, which are exactly where the analysis (e.g., hydrological model simulation) is performed. To bridge this gap, a typical approach is to ‘refine’ the information from GCMs through regional climate downscaling experiments, which can be conducted statistically, dynamically, or a combination thereof. Statistical downscaling establishes ...</p>
          <p><a href="/cmip6-era5-hybrid-southeast-asia/">Details &rarr;</a></p>
        </div>
        <div id="iserv" class="dataset">
          <h3><a href="/iserv/">ISERV</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>ISS SERVIR Environmental Research and Visualization System (ISERV) was a fully-automated prototype camera aboard the International Space Station that was tasked to capture high-resolution Earth imagery of specific locations at 3-7 frames per second. In the course of its regular operations during 2013 and 2014, ISERV&#39;s camera acquired images that can be used primaliry in use is environmental and disaster management.</p>
          <p><a href="/iserv/">Details &rarr;</a></p>
        </div>
        <div id="fast-ai-imagelocal" class="dataset">
          <h3><a href="/fast-ai-imagelocal/">Image localization  - fast.ai datasets</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>Some of the most important datasets for image localization  research, including
Camvid and PASCAL VOC (2007 and 2012). This is part of the fast.ai datasets
collection hosted by AWS for convenience of fast.ai students. See
documentation link for citation and license details for each dataset.</p>
          <p><a href="/fast-ai-imagelocal/">Details &rarr;</a></p>
        </div>
        <div id="inlab-covid-19-images-dataset" class="dataset">
          <h3><a href="/inlab-covid-19-images-dataset/">InRad COVID-19 X-Ray and CT Scans</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">coronavirus</span><span class="label label-info tag link-tag">COVID-19</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">SARS</span></p>
          <p>This dataset is a collection of anonymized thoracic radiographs (X-Rays) and computed tomography (CT) scans of patients with suspected COVID-19. Images are acommpanied by a positive or negative diagnosis for SARS-CoV2 infection via RT-PCR. These images were provided by Hospital das Clínicas da Universidade de São Paulo, Hospital Sirio-Libanes, and by Laboratory Fleury.</p>
          <p><a href="/inlab-covid-19-images-dataset/">Details &rarr;</a></p>
        </div>
        <div id="k2" class="dataset">
          <h3><a href="/k2/">K2 Mission Data</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The K2 mission observed 100 square degrees for 80 days each across 20 different pointings along the ecliptic, collecting high-precision photometry for a selection of targets within each field. The mission began when the original Kepler mission ended due to loss of the second reaction wheel in 2013.</p>
          <p><a href="/k2/">Details &rarr;</a></p>
        </div>
        <div id="kitti" class="dataset">
          <h3><a href="/kitti/">KITTI Vision Benchmark Suite</a></h3>
          <p><span class="label label-info tag link-tag">autonomous vehicles</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">robotics</span></p>
          <p>Dataset and benchmarks for computer vision research in the context of autonomous driving. The dataset has been recorded in and around the city of Karlsruhe, Germany using the mobile platform AnnieWay (VW station wagon) which has been equipped with several RGB and monochrome cameras, a Velodyne HDL 64 laser scanner as well as an accurate RTK corrected GPS/IMU localization unit. The dataset has been created for computer vision and machine learning research on stereo, optical flow, visual odometry, semantic segmentation, semantic instance segmentation, road segmentation, single image depth predic...</p>
          <p><a href="/kitti/">Details &rarr;</a></p>
        </div>
        <div id="kepler" class="dataset">
          <h3><a href="/kepler/">Kepler Mission Data</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The Kepler mission observed the brightness of more than 180,000 stars near the Cygnus constellation at a 30 minute cadence for 4 years in order to find transiting exoplanets, study variable stars, and find eclipsing binaries.</p>
          <p><a href="/kepler/">Details &rarr;</a></p>
        </div>
        <div id="mimic-iv-demo" class="dataset">
          <h3><a href="/mimic-iv-demo/">MIMIC-IV Clinical Database Demo</a></h3>
          <p></p>
          <p>The Medical Information Mart for Intensive Care (MIMIC)-IV&nbsp;database is comprised&nbsp;of&nbsp;deidentified electronic health records for patients admitted to the Beth Israel Deaconess Medical Center. Access to MIMIC-IV is limited to credentialed users. Here, we have provided an openly-available demo of MIMIC-IV containing&nbsp;a subset of&nbsp;100 patients. The dataset includes similar content to MIMIC-IV, but excludes free-text clinical notes. The demo may be useful for running workshops&nbsp;and for&nbsp;assessing whether the MIMIC-IV is appropriate for a study before making an access r...</p>
          <p><a href="/mimic-iv-demo/">Details &rarr;</a></p>
        </div>
        <div id="mimic-iv-ecg" class="dataset">
          <h3><a href="/mimic-iv-ecg/">MIMIC-IV-ECG: Diagnostic Electrocardiogram Matched Subset</a></h3>
          <p></p>
          <p>The MIMIC-IV-ECG module contains approximately 800,000 diagnostic electrocardiograms across nearly 160,000 unique patients. These diagnostic ECGs use 12 leads and are 10 seconds in length. They are sampled at 500 Hz. This subset contains all of the ECGs for patients who appear in the MIMIC-IV Clinical Database. When a cardiologist report is available for a given ECG, we provide the needed information to link the waveform to the report. The patients in MIMIC-IV-ECG have been matched against the MIMIC-IV Clinical Database, making it possible to link to information across the MIMIC-IV modules.</p>
          <p><a href="/mimic-iv-ecg/">Details &rarr;</a></p>
        </div>
        <div id="megascenes" class="dataset">
          <h3><a href="/megascenes/">MegaScenes</a></h3>
          <p><span class="label label-info tag link-tag">benchmark</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">internet</span></p>
          <p>The MegaScenes Dataset is an extensive collection of around 430k scenes, featuring over 100k structure-from-motion reconstructions and over 2 million registered images. MegaScenes includes a diverse array of scenes, such as minarets, building interiors, statues, bridges, towers, religious buildings, and natural landscapes. The images of these scenes are captured under varying conditions, including different times of day, various weather and illumination, and from different devices with distinct camera intrinsics.</p>
          <p><a href="/megascenes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://megascenes.github.io" class="" target="_none">MegaScenes: Scene-Level View Synthesis at Scale</a> by Tung J., Chou G., Cai R., Yang, G., Zhang K., Wetzstein G., et al.</li></a>
          </li>
          <li>
            <a href="https://megascenes.github.io" class="" target="_none">MegaScenes: Scene-Level View Synthesis at Scale</a> by Tung J., Chou G., Cai R., Yang, G., Zhang K., Wetzstein G., et al.</li></a>
          </li>
          <li>
            <a href="https://megascenes.github.io" class="" target="_none">MegaScenes: Scene-Level View Synthesis at Scale</a> by Tung J., Chou G., Cai R., Yang, G., Zhang K., Wetzstein G., et al.</li></a>
          </li>
        </ul>
        <p><a href="/megascenes/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="multimedia-commons" class="dataset">
          <h3><a href="/multimedia-commons/">Multimedia Commons</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">multimedia</span><span class="label label-info tag link-tag">video</span></p>
          <p>The Multimedia Commons is a collection of audio and visual features computed for the nearly 100 million Creative Commons-licensed Flickr images and videos in the YFCC100M dataset from Yahoo! Labs, along with ground-truth annotations for selected subsets. The International Computer Science Institute (ICSI) and Lawrence Livermore National Laboratory are producing and distributing a core set of derived feature sets and annotations as part of an effort to enable large-scale video search capabilities. They have released this feature corpus into the public domain, under Creative Commons License 0, s...</p>
          <p><a href="/multimedia-commons/">Details &rarr;</a></p>
        </div>
        <div id="fast-ai-nlp" class="dataset">
          <h3><a href="/fast-ai-nlp/">NLP - fast.ai datasets</a></h3>
          <p><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Some of the most important datasets for NLP, with a focus on classification, including
IMDb, AG-News, Amazon Reviews (polarity and full), Yelp Reviews (polarity and
full), Dbpedia, Sogou News (Pinyin), Yahoo Answers, Wikitext 2 and Wikitext
103, and ACL-2010 French-English 10^9 corpus.  This is part of the
fast.ai datasets collection hosted by AWS for convenience of fast.ai
students. See documentation link for citation and license details for each
dataset.</p>
          <p><a href="/fast-ai-nlp/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nos-scuba-icesat2-pds" class="dataset">
          <h3><a href="/noaa-nos-scuba-icesat2-pds/">NOAA / NGA Satellite Computed Bathymetry Assessment-SCuBA</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">bathymetry</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">weather</span></p>
          <p>One of the National Geospatial-Intelligence Agency’s (NGA) and the National Oceanic and Atmospheric Administration’s (NOAA) missions is to ensure the safety of navigation on the seas by 
maintaining the most current information and the highest quality services for U.S. and global transport networks. To achieve this mission, we need accurate coastal bathymetry over diverse 
environmental conditions. The SCuBA program focused on providing critical information to improve existing bathymetry resources and techniques with two specific objectives. The first objective 
was to validate National Aeronautics and Space Administration’s (NASA) Ice, Cloud and land Elevation SATellite-2 (ICESat-2), an Earth observing, space-based light detection and ranging (LiDAR) 
capability, as a useful bathymetry tool for nearshore bathymetry information in differing environmental conditions. Upon validating the ICESat-2 bathymetry retrievals relative to sea floor 
type, water clarity, and water surface dynamics, the next objective is to use ICESat-2 as a calibration tool to improve existing Satellite Derived Bathymetry (SDB) coastal bathymetry products 
with poor coastal depth information but superior spatial coverage. Current resources that monitor coastal bathymetry can have large vertical depth errors (up to 50 percent) in the nearshore 
region; however, derived results from ICESat-2 shows promising results for improving the accuracy of the bathymetry information in the nearshore region.
<br/>
<br/>
<strong>Project Overview</strong>
<br/>
One of NGA’s and NOAA’s primary missions is to provide safety of navigation information.  However, coastal depth information is still lacking in some regions—specifically, remote regions. In fact, it has been reported that 80 percent of the entire seafloor has not been mapped. Traditionally, airborne LiDARs and survey boats are used to map the seafloor, but in remote areas, we have to rely on satellite capabilities, which currently lack the vertical accuracy desired to support safety of navigation in shallow water. In 2018, NASA launched a space-based LiDAR system called ICESat-2 that has global coverage and a polar orbit originally designed to monitor the ice elevation in polar regions. Remarkably, because it has a green laser beam, ICESat-2 also happens to collect bathymetry information <a href="nasa.gov">ICESat-2</a>. With algorithm development provided by University of Texas (UT) Austin, NGA Research and Development (R&amp;D) leveraged the ICESat-2 platform to generate SCuBA, an automated depth retrieval algorithm for accurate, global, refraction-corrected underwater depths from 0 m to 30 m, detailed in Figure 1 of the documentation. The key benefit of this product is the vertical depth accuracy of depth retrievals, which is ideal for a calibration tool.  NGA and NOAA National Geodetic Survey (NGS), partnered to make this product available to the public for all US territories.
...</p>
          <p><a href="/noaa-nos-scuba-icesat2-pds/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nos-stofs3d" class="dataset">
          <h3><a href="/noaa-nos-stofs3d/">NOAA 3-D Surge and Tide Operational Forecast System for the Atlantic Basin (STOFS-3D-Atlantic)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">marine navigation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOTICE - The Coast Survey Development Laboratory (CSDL) in NOAA/National Ocean Service (NOS)/Office of Coast Survey is upgrading the Surge and Tide Operational Forecast System (STOFS, formerly ESTOFS) to Version 2.1. A Service Change Notice (SCN) has been issued and can be found &quot;<a href="https://www.weather.gov/media/notification/pdf_2023_24/scn23-124_stofs_upgrade_v2.1.4_aaf.pdf">HERE</a>&quot;<br /><br /> NOAA&#39;s Surge and Tide Operational Forecast System: Three-Dimensional Component for the Atlantic Basin (STOFS-3D-Atlantic). STOFS-3D-Atlantic runs daily (at 12 UTC)  to provide users with 24-hour nowcasts (analyses of near present conditions) and up to 96-hour forecast guidance of water level conditions, and 2- and 3-dimensional fields of water temperature, salinity, and currents. The water level outputs represent the combined tidal and subtidal water surface elevations and are referenced to xGEOID20B <br /><br />
 STOFS-3D-Atlantic has been developed to serve the marine navigation, weather forecasting, and disaster mitigation user communities. It is developed in a collaborative effort between the NOAA/National Ocean Service (NOS)/Office of Coast Survey, the NOAA/National Weather Service (NWS)/National Centers for Environmental Prediction (NCEP) Central Operations (NCO), and the Virginia Institute of Marine Science. <br /><br /> STOFS-3D-Atlantic employs the Semi-implicit Cross-scale Hydroscience Integrated System Model (SCHISM) as the hydrodynamic model core. Its unstructured grid consists of 2,926,236 nodes and 5,654,157 triangular or quadrilateral elements. Grid resolution is 1.5-2 km near the shoreline, ~600 m for the floodplain, down to 8 m for watershed rivers (at least 3 nodes across each river cross-section), and around 2-10 m for levees. Along the U.S. coastline, the land boundary of the domain aligns with the 10-m contour above xGEOID20B, encompassing the coastal transitional zone most vulnerable to coastal and inland flooding. <br /> <br /> STOFS-3D-Atlantic makes uses of outputs from the National Water Model (NWM) to include inland hydrology and extreme precipitation effects on coastal flooding; forecast guidance from the NCEP Global Forecast System (GFS) and High-Resolution Rapid Refresh (HRRR) model as the surface meteorological forci...</p>
          <p><a href="/noaa-nos-stofs3d/">Details &rarr;</a></p>
        </div>
        <div id="noaa-cdr-atmospheric" class="dataset">
          <h3><a href="/noaa-cdr-atmospheric/">NOAA Atmospheric Climate Data Records</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOAA&#39;s Climate Data Records (CDRs) are robust, sustainable, and scientifically sound climate records that provide trustworthy information on how, where, and to what extent the land, oceans, atmosphere and ice sheets are changing. These datasets are thoroughly vetted time series measurements with the longevity, consistency, and continuity to assess and measure climate variability and change. NOAA CDRs are vetted using standards established by the National Research Council (NRC).<br/><br/>
Climate Data Records are created by merging data from surface, atmosphere, and space-based systems across decades. NOAA’s Climate Data Records provides authoritative and traceable long-term climate records. NOAA developed CDRs by applying modern data analysis methods to historical global satellite data. This process can clarify the underlying climate trends within the data and allows researchers and other users to identify economic and scientific value in these records. NCEI maintains and extends CDRs by applying the same methods to present-day and future satellite measurements.<br/><br/>
Atmospheric Climate Data Records are measurements of several global variables to help characterize the atmosphere</br/>...</p>
          <p><a href="/noaa-cdr-atmospheric/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nodd-kerchunk" class="dataset">
          <h3><a href="/noaa-nodd-kerchunk/">NOAA Cloud Optimized Zarr Reference Files (Kerchunk)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>This repository contains references to datasets published to the NOAA Open Data Dissemination Program. These reference datasets serve as index files to the original data by mapping to the <a href="https://zarr-specs.readthedocs.io/en/latest/v2/v2.0.html">Zarr V2 specification</a>. When multidimensional model output is read through zarr, data can be lazily loaded (i.e. retrieving only the data chunks needed for processing) and data reads can be scaled horizontally to optimize object storage read performance. 
<br/>
<br/>
The process used to optimize the data is called <a href="https://fsspec.github.io/kerchunk/">kerchunk</a>. RPS runs <a href="https://asascience-open.github.io/nextgen-dmac/ingest/ingest-prototype.html">the workflow</a> in their AWS cloud environment every time a new data notification is received from a relevant source data bucket.
<br/>
<br/>
These are the current datasets being cloud-optimized. Refer to those pages for file naming conventions and other information regarding the specific model implementations:
<br/>
<a href="https://registry.opendata.aws/noaa-ofs/">NOAA Operational Forecast System (OFS)</a>
<br/><br><a href="https://registry.opendata.aws/noaa-rtofs/">NOAA Global Real-Time Ocean Forecast System (Global RTOFS)</a>
<br/><br><a href="https://registry.opendata.aws/noaa-nwm-pds/">NOAA National Water Model Short-Range Forecast</a>
<br/>
<br/>
Filenames follow the source dataset’s conventions. For example, if the source file is
<br/>
nos.dbofs.fields.f024.20240527.t00z.nc
<br/>
<br/>
Then the cloud-optimized filename is the same, with “.zarr” appended
<br/>
nos.dbofs.fields.f024.20240527.t00z.nc.zarr
<br/>
<br/>
<strong>Data Aggregations</strong>
<br/>
We also produce virtual aggregations to group an entire forecast model run, and the “best” available forecast.
<br/>
Best Forecast (continuously updated) -  nos.dbofs.fields.best.nc.zarr
Full Model Run -  nos.dbofs.fields.forecast.[YYYYMMDD].t[CC]z.nc.zarr
<br/><ul>
<li>CC is the model run cycles, 00, 06, 12, 18 , or 03, 09, 15, 21 for nowcast and forecast runs<br/></li>
<li>YYYY = year, MM = month, DD = day<br/>
<br/>
Cloud o...</p>
          <p><a href="/noaa-nodd-kerchunk/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ncn" class="dataset">
          <h3><a href="/noaa-ncn/">NOAA Continuously Operating Reference Stations (CORS) Network (NCN)</a></h3>
          <p><span class="label label-info tag link-tag">broadcast ephemeris</span><span class="label label-info tag link-tag">Continuously Operating Reference Station (CORS)</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">GNSS</span><span class="label label-info tag link-tag">GPS</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">NOAA CORS Network (NCN)</span><span class="label label-info tag link-tag">post-processing</span><span class="label label-info tag link-tag">RINEX</span><span class="label label-info tag link-tag">survey</span></p>
          <p>The <a href="https://geodesy.noaa.gov/CORS/">NOAA Continuously Operating Reference Stations (CORS) Network (NCN)</a>, managed by NOAA/National Geodetic Survey (<a href="https://geodesy.noaa.gov/">NGS</a>), provide Global Navigation Satellite System (GNSS) data, supporting three dimensional positioning, meteorology, space weather, and geophysical applications throughout the United States. The NCN is a multi-purpose, multi-agency cooperative endeavor, combining the efforts of hundreds of government, academic, and private organizations. The stations are independently owned and operated. Each agency shares their GNSS/GPS carrier phase and code range measurements and station metadata with NGS, which are analyzed and distributed free of charge.
...</p>
          <p><a href="/noaa-ncn/">Details &rarr;</a></p>
        </div>
        <div id="noaa-cdr-fundamental" class="dataset">
          <h3><a href="/noaa-cdr-fundamental/">NOAA Fundamental Climate Data Records (FCDR)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOAA&#39;s Climate Data Records (CDRs) are robust, sustainable, and scientifically sound climate records that provide trustworthy information on how, where, and to what extent the land, oceans, atmosphere and ice sheets are changing. These datasets are thoroughly vetted time series measurements with the longevity, consistency, and continuity to assess and measure climate variability and change. NOAA CDRs are vetted using standards established by the National Research Council (NRC).<br/><br/>
Climate Data Records are created by merging data from surface, atmosphere, and space-based systems across decades. NOAA’s Climate Data Records provides authoritative and traceable long-term climate records. NOAA developed CDRs by applying modern data analysis methods to historical global satellite data. This process can clarify the underlying climate trends within the data and allows researchers and other users to identify economic and scientific value in these records. NCEI maintains and extends CDRs by applying the same methods to present-day and future satellite measurements.<br/><br/>
Fundamental CDRs are composed of sensor data (e.g. calibrated radiances, brightness temperatures) that have been </br/>...</p>
          <p><a href="/noaa-cdr-fundamental/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ufs-gdas-pds" class="dataset">
          <h3><a href="/noaa-ufs-gdas-pds/">NOAA Global Data Assimilation (DA) Test Data</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The <a href="https://ufs.epic.noaa.gov/">Unified Forecast System (UFS)</a> is a community-based, coupled, comprehensive Earth Modeling System. It supports <a href="https://ufs.epic.noaa.gov/applications/">multiple applications</a> with different forecast durations and spatial domains. The <a href="https://github.com/NOAA-EMC/GDASApp">Global Data Assimilation System (GDAS)</a> Application (App) is being used as the basis for uniting the <a href="http://NOAA-EMC/global-workflow">Global Workflow</a> and Global Forecast System (GFS) model with <a href="https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/">Joint Effort for Data assimilation Integration (JEDI)</a> capabilities.
<br/>
<br/>
The National Centers for Environmental Prediction (NCEP) use GDAS to interpolate data from various observing systems and instruments onto a three-dimensional grid. GDAS obtain...</p>
          <p><a href="/noaa-ufs-gdas-pds/">Details &rarr;</a></p>
        </div>
        <div id="noaa-gefs" class="dataset">
          <h3><a href="/noaa-gefs/">NOAA Global Ensemble Forecast System (GEFS)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Global Ensemble Forecast System (GEFS), previously known as the GFS Global ENSemble (GENS), is a weather forecast model made up of 21 separate forecasts, or ensemble members.  The National Centers for Environmental Prediction (NCEP) started the GEFS to address the nature of uncertainty in weather observations, which is used to initialize weather forecast models. The GEFS attempts to quantify the amount of uncertainty in a forecast by generating an ensemble of multiple forecasts, each minutely different, or perturbed, from the original observations. With global coverage, GEFS is produced fo...</p>
          <p><a href="/noaa-gefs/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ghe" class="dataset">
          <h3><a href="/noaa-ghe/">NOAA Global Hydro Estimator (GHE)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Global Hydro-Estimator provides a global
mosaic imagery of rainfall estimates from
multi-geostationary satellites, which
currently includes GOES-16, GOES-15,
Meteosat-8, Meteosat-11 and Himawari-8.
The GHE products include: Instantaneous
rain rate, 1 hour, 3 hour, 6 hour, 24 hour
and also multi-day rainfall accumulation.</p>
          <p><a href="/noaa-ghe/">Details &rarr;</a></p>
        </div>
        <div id="noaa-gmgsi" class="dataset">
          <h3><a href="/noaa-gmgsi/">NOAA Global Mosaic of Geostationary Satellite Imagery (GMGSI)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOAA/NESDIS Global Mosaic of Geostationary Satellite Imagery (GMGSI) visible (VIS), shortwave infrared (SIR), longwave infrared (LIR) imagery, and water vapor imagery (WV) are composited from data from several geostationary satellites orbiting the globe, including the GOES-East and GOES-West Satellites operated by U.S. NOAA/NESDIS, the Meteosat-10 and Meteosat-9 satellites from theMeteosat Second Generation (MSG) series of satellites operated by European Organization for the Exploitation of Meteorological Satellites (EUMETSAT), and the Himawari-9 satellite operated by the Japan Meteorological ...</p>
          <p><a href="/noaa-gmgsi/">Details &rarr;</a></p>
        </div>
        <div id="noaa-rtofs" class="dataset">
          <h3><a href="/noaa-rtofs/">NOAA Global Real-Time Ocean Forecast System (Global RTOFS)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOAA is soliciting public comment on petential changes to the Real Time Ocean Forecast System (RTOFS) through March 27, 2024. Please see Public Notice at (<a href="https://www.weather.gov/media/notification/pdf_2023_24/pns24-12_rtofs_v2.4.0.pdf">https://www.weather.gov/media/notification/pdf_2023_24/pns24-12_rtofs_v2.4.0.pdf</a>)<br /><br />NOAA&#39;s Global Real-Time Ocean Forecast System (Global RTOFS) provides users with nowcasts (analyses of near present conditions) and forecast guidance up to eight days of ocean temperature and salinity, water velocity, sea surface elevation, sea ice coverage and sea ice thickness. <br /><br />The Global Operational Real-Time Ocean Forecast System (Global RTOFS) is based on an eddy resolving 1/12° global HYCOM (HYbrid Coordinates Ocean Model) (<a href="https://www.hycom.org/">https://www.hycom.org/</a>), which is coupled to the Community Ice CodE (CICE) Version 4 (<a href="https://www.arcus.org/witness-the-arctic/2018/5/highlight/1">https://www.arcus.org/witness-the-arctic/2018/5/highlight/1</a>). The RTOFS grid has a 1/12 degree horizontal resolution and 41 hybrid vertical levels on a global tripolar grid. <br /><br />Since 2020, the RTOFS system implements a multivariate, multi-scale 3DVar data assimilation algorithm (Cummings and Smedstad, 2014) using a 24-hour update cycle. The data types presently assimilated include <br /><br />(1) satellite Sea Surface Temperature (SST) from METOP-B, JPSS-VIIRS, and  in-Situ SST, from ships, fixed and drifting buoys <br />
(2) Sea Surface Salinity (SSS) from SMAP, SMOS, and buoys <br />
(3) profiles of Temperature and Salinity from Animal-borne, Alamo floats, Argo floats, CTD, fixed buoys, gliders, TESAC, and XBT <br />
(4) Absolute Dynamic Topography (ADT) from Altika, Cryosat, Jason-3, Sentinel 3a, 3b, 6a <br />
(5) sea ice concentration from SSMI/S, AMSR2 <br /> <br />The system is designed to incorporate new observing systems as the data becomes available. <br /> <br />Once the observations go through a fully automated quality control and thinning process, the increments, or corrections, are obtained by executing the 3D variational algorithm. The increments are then added to the 24-hours forecast fields using a 6-hourly incremental analysis update. An earlier version of the system is described in Garraffo et al (2020). <br /> <br />Garraffo, Z.D., J.A. Cummings, S. Paturi, Y. Hao, D. Iredell, T. Spindler, B. Balasubramanian, I. Rivin, H-C. Kim, A. Mehra, 2020. Real Time Ocean-Sea Ice Coupled Three Dimensional Variational Global Data Assimilative Ocean Forecast System. In Research Activities in Earth System Modeling, edited by E. Astakhova, WMO, World Climate Research Program Report No.6, July 2020. <br /> <br />Cummings, J. A. and O. M. Smedstad. 2013. Variational Data Assimilation for the Global Ocean.
Data Assimilation for Atmospheric, Oceanic and Hydrologic Applications (Vol II)
S. Park and L. Xu (eds), Springer, Chapter 13, 303-343. <br /> <br />Global...</p>
          <p><a href="/noaa-rtofs/">Details &rarr;</a></p>
        </div>
        <div id="noaa-gestofs" class="dataset">
          <h3><a href="/noaa-gestofs/">NOAA Global Surge and Tide Operational Forecast System 2-D (STOFS-2D-Global)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">global</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">water</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOTICE - The Coast Survey Development Laboratory (CSDL) in NOAA/National Ocean Service (NOS)/Office of Coast Survey has upgraded the Surge and Tide Operational Forecast System (STOFS, formerly ESTOFS) to Version 2.1. A Service Change Notice (SCN) has been issued and can be found &quot;<a href="https://www.weather.gov/media/notification/pdf_2023_24/scn23-124_stofs_upgrade_v2.1.4_aaf.pdf">HERE</a>&quot;<br /><br />NOAA&#39;s Global Surge and Tide Operational Forecast System 2-D (STOFS-2D-Global) provides users with nowcasts (analyses of near present conditions) and forecast guidance of water level conditions for the entire globe. STOFS-2D-Global has been developed to serve the marine navigation, weather forecasting, and disaster mitigation user communities. STOFS-2D-Global was developed in a collaborative effort between the NOAA/National Ocean Service (NOS)/Office of Coast Survey, the NOAA/National Weather Service (NWS)/National Centers for Environmental Prediction (NCEP) Central Operations (NCO), the University of Notre Dame, the University of North Carolina, and The Water Institute of the Gulf. The model generates forecasts out to 180 hours four times per day; forecast output includes water levels caused by the combined effects of storm surge and tides, by astronomical tides alone, and by sub-tidal water levels (isolated storm surge).  <br /><br />The hydrodynamic model employed by STOFS-2D-Global is the ADvanced CIRCulation (ADCIRC) finite element model. The model is forced by GFS winds, mean sea level pressure, and sea ice. The unstructured grid used by STOFS-2D-Global consists of 12,785,004 nodes and 24,875,336 triangular elements. Coastal res...</p>
          <p><a href="/noaa-gestofs/">Details &rarr;</a></p>
        </div>
        <div id="noaa-historicalcharts" class="dataset">
          <h3><a href="/noaa-historicalcharts/">NOAA Historical Maps and Charts</a></h3>
          <p><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">history</span><span class="label label-info tag link-tag">mapping</span><span class="label label-info tag link-tag">survey</span></p>
          <p>Historical Charts are not for Navigation. The collection primarily consists of historic charts and maps produced by NOAA&#39;s Coast Survey and its predecessors, especially the U.S. Coast and Geodetic Survey and the U.S. Lake Survey (previously under the Department of War). The collection also includes bathymetric maps, land sketches, Civil War battle maps, aeronautical charting from the 1930s to the 1950s, and other drawings and photographs.</p>
          <p><a href="/noaa-historicalcharts/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nws-hafs" class="dataset">
          <h3><a href="/noaa-nws-hafs/">NOAA Hurricane Analysis and Forecast System (HAFS)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The last several hurricane seasons have been active with records being set for the number of tropical storms and hurricanes in the Atlantic basin.  These record-breaking seasons underscore the importance of accurate hurricane forecasting. Imperative to increased forecasting skill for hurricanes is the development of the Hurricane Forecast Analysis System or HAFS. To accelerate improvements in hurricane forecasting, this project has the following goals:
<br /> 
<br /> <ol>
<li>To improve the HAFS. The HAFS is NOAA’s next-generation multi-scale numerical model, with data assimilation package and ocean coupling, which will provide an operational analysis and forecast out to seven days, with reliable and skillful guidance on hurricane track and intensity (including rapid intensification), storm size, genesis, storm surge, rainfall and tornadoes associated with hurricanes.<br /> 
<br /> </li>
<li>To integrate into the Unified Forecasting System(UFS). The UFS is a community-based, coupled comprehensive Earth system modeling system whose numerical applications span local to global domains and predictive time scales from sub-hourly analyses to seasonal predictions. It is designed to support the Weather Enterprise and to be the source system for NOAA’s operational numerical weather prediction applications. The HAFS will be a part of UFS geared for hurricane model applications. HAFS comprises five major components; (a) High-resolution moving nest (b) High-resolution physics (c) Multi-scale data assimilation (DA) (d) 3D ocean coupling, and (e) Observations to support the DA.<br /> 
<br /> 
[Read about how the storm-following model improves intensity forecasts](https://www.aoml.noaa.gov/hurricane-model-that-follows-mult...</p>
          <p><a href="/noaa-nws-hafs/">Details &rarr;</a></p>
        </div>
        <div id="noaa-reanalyses-pds" class="dataset">
          <h3><a href="/noaa-reanalyses-pds/">NOAA NASA Joint Archive (NNJA) of Observations for Earth System Reanalysis</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The NOAA NASA Joint Archive (NNJA) of Observations for Earth System Reanalysis is a curated joint observation archive containing Earth system data from 1979 to present prepared by teams at NOAA&#39;s Physical Sciences Laboratory and NASA&#39;s Global Modeling and Assimilation Office. The goal is to foster collaboration across organizations and develop the ability for direct comparison of Earth System reanalysis results. Providing a singular dataset for observation input use will allow reanalyses to be compared on their unique development qualities by removing the variation from using different...</p>
          <p><a href="/noaa-reanalyses-pds/">Details &rarr;</a></p>
        </div>
        <div id="noaa-bathymetry" class="dataset">
          <h3><a href="/noaa-bathymetry/">NOAA National Bathymetric Source Data</a></h3>
          <p><span class="label label-info tag link-tag">bathymetry</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">marine navigation</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The National Bathymetric Source (NBS) project creates and maintains 
high-resolution bathymetry composed of the best available data. 
This project enables the creation of next-generation nautical charts 
while also providing support for modeling, industry, science, 
regulation, and public curiosity. Primary sources of bathymetry include 
NOAA and U.S. Army Corps of Engineers hydrographic surveys and 
topographic bathymetric (topo-bathy) lidar (light detection and ranging) 
data.  Data submitted through the NOAA Office of Coast Survey’s external 
source data process are also included, with gaps...</p>
          <p><a href="/noaa-bathymetry/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nbm" class="dataset">
          <h3><a href="/noaa-nbm/">NOAA National Blend of Models (NBM)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The National Blend of Models (NBM) is a nationally consistent and skillful suite of calibrated forecast guidance based on a blend of both NWS and non-NWS numerical weather prediction model data and post-processed model guidance. The goal of the NBM is to create a highly accurate, skillful and consistent starting point for the gridded forecast. </p>
          <p><a href="/noaa-nbm/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nam" class="dataset">
          <h3><a href="/noaa-nam/">NOAA North American Mesoscale Forecast System (NAM)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The North American Mesoscale Forecast System (NAM) is one of the National Centers For Environmental Prediction’s (NCEP) major models for producing weather forecasts. NAM generates multiple grids (or domains) of weather forecasts over the North American continent at various horizontal resolutions. Each grid contains data for dozens of weather parameters, including temperature, precipitation, lightning, and turbulent kinetic energy. NAM uses additional numerical weather models to generate high-resolution forecasts over fixed regions, and occasionally to follow significant weather events like hur...</p>
          <p><a href="/noaa-nam/">Details &rarr;</a></p>
        </div>
        <div id="noaa-cdr-oceanic" class="dataset">
          <h3><a href="/noaa-cdr-oceanic/">NOAA Oceanic Climate Data Records</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOAA&#39;s Climate Data Records (CDRs) are robust, sustainable, and scientifically sound climate records that provide trustworthy information on how, where, and to what extent the land, oceans, atmosphere and ice sheets are changing. These datasets are thoroughly vetted time series measurements with the longevity, consistency, and continuity to assess and measure climate variability and change. NOAA CDRs are vetted using standards established by the National Research Council (NRC).<br/><br/>
Climate Data Records are created by merging data from surface, atmosphere, and space-based systems across decades. NOAA’s Climate Data Records provides authoritative and traceable long-term climate records. NOAA developed CDRs by applying modern data analysis methods to historical global satellite data. This process can clarify the underlying climate trends within the data and allows researchers and other users to identify economic and scientific value in these records. NCEI maintains and extends CDRs by applying the same methods to present-day and future satellite measurements.<br/><br/>
Oceanic Climate Data Records are measurements of oceans and seas both surface and subsurface as well as frozen st</br/>...</p>
          <p><a href="/noaa-cdr-oceanic/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ocs-hydrodata" class="dataset">
          <h3><a href="/noaa-ocs-hydrodata/">NOAA Office of Coast Survey - Hydrographic Survey Data</a></h3>
          <p><span class="label label-info tag link-tag">bathymetry</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">marine navigation</span><span class="label label-info tag link-tag">model</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>Founded in 1807, NOAA’s Office of Coast Survey is the nation’s first scientific agency and today is responsible for supporting nearly $5.4 trillion in economic activity through providing advanced marine navigation services. The Office of Coast Survey collects and qualifies hydrographic, bathymetric, and topographic data, from NOAA platforms and many other data providers. These data and associated deliverables are posted here for various users to access, including but not limited to the &quot;<a href="https://registry.opendata.aws/noaa-bathymetry/">National Bathymetric Source Program</a>&quot; for incorporation into compilations of the best available bat...</p>
          <p><a href="/noaa-ocs-hydrodata/">Details &rarr;</a></p>
        </div>
        <div id="noaa-rap" class="dataset">
          <h3><a href="/noaa-rap/">NOAA Rapid Refresh (RAP)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Rapid Refresh (RAP) is a NOAA/NCEP operational weather prediction system comprised primarily of a numerical forecast model and analysis/assimilation system to initialize that model. It covers North America and is run with a horizontal resolution of 13 km and 50 vertical layers. The RAP was developed to serve users needing frequently updated short-range weather forecasts, including those in the US aviation community and US severe weather forecasting community. The model is run for every hour of the day; it is integrated to 51 hours for the 03/09/15/21 UTC cycles and to 21 hours for every ot...</p>
          <p><a href="/noaa-rap/">Details &rarr;</a></p>
        </div>
        <div id="noaa-rtma" class="dataset">
          <h3><a href="/noaa-rtma/">NOAA Real-Time Mesoscale Analysis (RTMA) / Unrestricted Mesoscale Analysis (URMA)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Real-Time Mesoscale Analysis (RTMA) is a NOAA National Centers For Environmental Prediction (NCEP) high-spatial and temporal resolution analysis/assimilation system for near-surf ace weather conditions. Its main component is the NCEP/EMC Gridpoint Statistical Interpolation (GSI) system applied in two-dimensional variational mode to assimilate conventional and satellite-derived observations.<br/><br/>
The RTMA was developed to support NDFD operations and provide field forecasters with high quality analyses for nowcasting, situational awareness, and forecast verification purposes. The system produces </br/>...</p>
          <p><a href="/noaa-rtma/">Details &rarr;</a></p>
        </div>
        <div id="noaa-s102" class="dataset">
          <h3><a href="/noaa-s102/">NOAA S-102 Bathymetric Surface Data</a></h3>
          <p><span class="label label-info tag link-tag">bathymetry</span><span class="label label-info tag link-tag">hydrography</span><span class="label label-info tag link-tag">marine navigation</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">seafloor</span><span class="label label-info tag link-tag">water</span></p>
          <p>S-102 is a data and metadata encoding specification that is part of the <a href="https://iho.int/en/s100-project">S-100 Universal Hydrographic Data Model</a>, an international standard for hydrographic data exchange. This collection of data contains bathymetric surfaces from <a href="https://nauticalcharts.noaa.gov/updates/building-the-national-bathymetry/">NOAA/NOS/OCS National Bathymetric Source</a>, for various U.S. coastal and offshore waters and the great lakes. These datasets are encoded as HDF5 files conforming to the S-102 specification.</p>
          <p><a href="/noaa-s102/">Details &rarr;</a></p>
        </div>
        <div id="noaa-swdi" class="dataset">
          <h3><a href="/noaa-swdi/">NOAA Severe Weather Data Inventory (SWDI)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Storm Events Database is an integrated database of severe weather events across the United States from 1950 to this year, with information about a storm event&#39;s location, azimuth, distance, impact, and severity, including the cost of damages to property and crops. It contains data documenting: The occurrence of storms and other significant weather phenomena having sufficient intensity to cause loss of life, injuries, significant property damage, and/or disruption to commerce. Rare, unusual, weather phenomena that generate media attention, such as snow flurries in South Florida or the S...</p>
          <p><a href="/noaa-swdi/">Details &rarr;</a></p>
        </div>
        <div id="noaa-space-weather" class="dataset">
          <h3><a href="/noaa-space-weather/">NOAA Space Weather Forecast and Observation Data</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">solar</span><span class="label label-info tag link-tag">weather</span></p>
          <p>Space weather forecast and observation data is collected and disseminated by NOAA’s Space Weather Prediction Center (SWPC) in Boulder, CO. SWPC produces forecasts for multiple space weather phenomenon types and the resulting impacts to Earth and human activities.  A variety of products are available that provide these forecast expectations, and their respective measurements, in formats that range from detailed technical forecast discussions to NOAA Scale values to simple bulletins that give information in laymen&#39;s terms. Forecasting is the prediction of future events, based on analysis and...</p>
          <p><a href="/noaa-space-weather/">Details &rarr;</a></p>
        </div>
        <div id="noaa-cdr-terrestrial" class="dataset">
          <h3><a href="/noaa-cdr-terrestrial/">NOAA Terrestrial Climate Data Records</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOAA&#39;s Climate Data Records (CDRs) are robust, sustainable, and scientifically sound climate records that provide trustworthy information on how, where, and to what extent the land, oceans, atmosphere and ice sheets are changing. These datasets are thoroughly vetted time series measurements with the longevity, consistency, and continuity to assess and measure climate variability and change. NOAA CDRs are vetted using standards established by the National Research Council (NRC).<br/><br/>
Climate Data Records are created by merging data from surface, atmosphere, and space-based systems across decades. NOAA’s Climate Data Records provides authoritative and traceable long-term climate records. NOAA developed CDRs by applying modern data analysis methods to historical global satellite data. This process can clarify the underlying climate trends within the data and allows researchers and other users to identify economic and scientific value in these records. NCEI maintains and extends CDRs by applying the same methods to present-day and future satellite measurements.<br/><br/>
Terrestrial CDRs are composed of sensor data that have been improved and quality controlled over time, together w</br/>...</p>
          <p><a href="/noaa-cdr-terrestrial/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nclimgrid" class="dataset">
          <h3><a href="/noaa-nclimgrid/">NOAA U.S. Climate Gridded Dataset (NClimGrid)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The NOAA Monthly U.S. Climate Gridded Dataset (NClimGrid) consists of four climate variables derived from the GHCN-D dataset: maximum temperature, minimum temperature, average temperature and precipitation. Each file provides monthly values in a 5x5 lat/lon grid for the Continental United States. Data is available from 1895 to the present. On an annual basis, approximately one year of &quot;final&quot; nClimGrid will be submitted to replace the initially supplied &quot;preliminary&quot; data for the same time period. Users should be sure to ascertain which level of data is required for their research.
<br/>
<br/>
EpiNOAA is an analysis ready dataset that consists of a daily time-series of nClimGrid measures (maximum temperature, minimum temperature, average temperature, and precipitation) at the county scale.  Each file provides daily values for the Continental United States. Data are available from 1951 to the present. Daily data are updated every 3 days with a preliminary data file and replaced with the scaled (i.e., quality controlled) data file every three months. This derivative data product is an enhancement from the original daily nClimGrid dataset in that all four weather parameters are now p</br/>...</p>
          <p><a href="/noaa-nclimgrid/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ufs-gefsv13replay-pds" class="dataset">
          <h3><a href="/noaa-ufs-gefsv13replay-pds/">NOAA Unified Forecast System (UFS) Global Ensemble Forecast System (GEFS) Version 13 Replay</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The NOAA Unified Forecast System (UFS) / Global Ensemble Forecast System version 13 (GEFSv13) Replay dataset supports the retrospective forecast archive in preparation for GEFSv13 / GFSv17. It includes a range of atmospheric and oceanic variables—such as temperature, humidity, winds, salinity, and currents—covering global conditions at a nominal horizontal resolution of ¼ degree, enabling detailed weather analysis.
<br/>
<br/>
The dataset was generated by replaying the coupled UFS model against pre-existing external reanalyses; ERA5 for atmospheric data and ORAS5 for ocean and ice dynamics. Each simulation stream was initialized from these reanalyses, which were pre-processed for the UFS model components, including the GFDL Finite-Volume Cubed-Sphere Dynamical Core (FV3; 25 km, 127 vertical levels) and the Modular Ocean Model (MOM6; ¼ degree tri-polar grid, 72 vertical levels). This replay methodology enforces a predetermined model state while allowing cross-component fluxes and unconstrained processes to be computed.
<br/>
<br/>
For the land surface, NOAA’s JEDI-based land data assimilation system incorporated snow depth observations from the NCEI Global Historical Climatology Network (GHCN) and satellite-derived snow cover from the U.S. National Ice Center. The JEDI Sea-ice Ocean and Coupled Analysis system (SOCA) adjusted sea-ice thickness and concentration for consistency with ORAS5.
<br/>
<br/>
The original dataset spanned January 1994 to October 2023, with plans for ongoing updates and a 1-degree version covering 1958 to 2023. The dataset is hosted on AWS and GCP clou</br/>...</p>
          <p><a href="/noaa-ufs-gefsv13replay-pds/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ufs-htf-pds" class="dataset">
          <h3><a href="/noaa-ufs-htf-pds/">NOAA Unified Forecast System (UFS) Hierarchical Testing Framework (HTF)</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The &quot;<a href="https://ufs.epic.noaa.gov/">Unified Forecast System</a>&quot; (UFS) is a community-based, coupled, comprehensive Earth Modeling System. The Hierarchical Testing Framework (HTF) serves as a comprehensive toolkit designed to enhance the testing capabilities within UFS &quot;<a href="https://github.com/ufs-community">repositories</a>&quot;. It aims to standardize and simplify the testing process across various &quot;<a href="https://github.com/ufs-community/ufs-weather-model">UFS Weather Model</a>&quot; (WM) components and associated modules, aligning with the Hierarchical System Development (HSD) approach and NOAA baseline operational metrics.
<br/>
<br/>
The HTF provides a structured methodology for test case design and execution, which enhances code management practices, fosters user accessibility, and promotes adherence to established testing protocols. It enables developers to conduct testing efficiently and consistently, ensuring code integrity and reliability through the use of established technologies such as CMake and CTest. When integrated with containerization techniques, the HTF facilitates portability of test cases and promotes reproducibility across different computing environments. This approach reduces the computational overhead and enhances collaboration within the UFS community by providing a unified testing framework. 
<br/>
<br/>
Acknowledgment - The Unified Forecast System (UFS) atmosphere-ocean coupled model...</p>
          <p><a href="/noaa-ufs-htf-pds/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ufs-land-da" class="dataset">
          <h3><a href="/noaa-ufs-land-da/">NOAA Unified Forecast System (UFS) Land Data Assimilation (DA) System</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Unified Forecast System (UFS) is a community-based, coupled, comprehensive Earth modeling system. It supports &quot;<a href="https://ufscommunity.org/science/aboutapps/">multiple applications</a>&quot; covering different forecast durations and spatial domains. The Land Data Assimilation (DA) System is an offline version of the Noah Multi-Physics (Noah-MP) land surface model (LSM) used in the UFS Weather Model (WM). Its data assimilation framework uses &quot;[Joint Effort for Data assimilation Integration - JEDI] (<a href="https://www.jcsda.org/jcsda-project-jedi)&quot;">https://www.jcsda.org/jcsda-project-jedi)&quot;</a> software. The offline Noah-MP LSM is a stand-alone, uncoupled model used to execute land surface simulations. In this traditional uncoupled mode, near-surface atmospheric forcing data is required as input. Sample forcing and restart data are provided in this data bucket. <br/> <br/> The Noah-MP LSM has evolved through community efforts to pursue and refine a modern-era LSM suitable for use in the National Centers for Environmental Prediction (NCEP) operational weather and climate prediction models. This collaborative effort continues with participation from entities such as NCAR, NCEP, NASA, and university groups. <br/> <br/> For details regarding the physical parameterizations used in Noah-MP, see &quot;[Niu, et al. (2011)] (<a href="https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2010JD015139)&quot;">https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2010JD015139)&quot;</a>. The &quot;[Land DA User’s Guide] (<a href="https://land-da.readthedocs.io/en/latest/)&quot;">https://land-da.readthedocs.io/en/latest/)&quot;</a> provides information on building and running the Land DA System in offline mode. Users can access additional technical support via the &quot;[UFS GitHub Discussions] (<a href="https://github.com/NOAA-EPIC/land-offline_workflow/discussions)&quot;">https://github.com/NOAA-EPIC/land-offline_workflow/discussions)&quot;</a> for the L...</p>
          <p><a href="/noaa-ufs-land-da/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ufs-marinereanalysis" class="dataset">
          <h3><a href="/noaa-ufs-marinereanalysis/">NOAA Unified Forecast System (UFS) Marine Reanalysis: 1979-2019</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The NOAA UFS Marine Reanalysis is a global sea ice ocean coupled reanalysis product produced by the marine data assimilation team of the UFS Research-to-Operation (R2O) project. Underlying forecast and data assimilation systems are based on the UFS model prototype version-6 and the Next Generation Global Ocean Data Assimilation System (NG-GODAS) release of the Joint Effort for Data assimilation Integration (JEDI) Sea Ice Ocean Coupled Assimilation (SOCA). Covering the 40 year reanalysis time period from 1979 to 2019, the data atmosphere option of the UFS coupled global atmosphere ocean sea ice (DATM-MOM6-CICE6) model was applied with two atmospheric forcing data sets: CFSR from 1979 to 1999 and GEFS from 2000 to 2019. Assimilated observation data sets include extensive space-based marine observations and conventional direct measurements of in situ profile data sets.
<br/><br/>
This first UFS-marine interim reanalysis product is released to the broader weather and earth system modeling and analysis communities to obtain scientific feedback and applications for the development of the next generation operational numerical weather prediction system at the National Weather Service(NWS). The released file sets include two parts 1.) 1979 - 2019 UFS-DATM-MOM6-CICE6 model free runs and 2) 1979-2019 reanalysis cycle outputs (see descriptions embedded in each file set). Analyzed sea ice and ocean variables are ocean temperature, salinity, sea surface height, and sea ice conce</br/>...</p>
          <p><a href="/noaa-ufs-marinereanalysis/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ufs-shortrangeweather" class="dataset">
          <h3><a href="/noaa-ufs-shortrangeweather/">NOAA Unified Forecast System Short-Range Weather (UFS SRW) Application</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The &quot;<a href="https://ufscommunity.org/">Unified Forecast System (UFS)</a>&quot; is a community-based, coupled, comprehensive Earth Modeling System. It supports &quot; <a href="https://ufscommunity.org/science/aboutapps/">multiple applications</a>&quot; with different forecast durations and spatial domains. The UFS Short-Range Weather (SRW) Application figures among these applications. It targets predictions of atmospheric behavior on a limited spatial domain and on time scales from minutes to several days. The SRW Application includes a prognostic atmospheric model, pre-processor, post-processor, and community workflow for running the system end-to-end. The &quot;<a href="https://ufs-srweather-app.readthedocs.io/en/develop/">SRW Application Users&#39;s Guide</a>&quot; includes information on these components and provides detailed instructions on how to build and run the SRW Application. Users can access additional technical support via the &quot;<a href="https://github.com/ufs-community/ufs-srweather-app/discussions/categories/q-a">UFS GitHub Discussions</a>&quot; <br/> <br/> This data registry contains the data required to run the “out-of-the-box” SRW Application case. The SRW App requires numerous input files to run, including static datasets (fix files containing climatological information, terrain and land use data), initial condition data files, lateral boundary condition data files, and model configuration files (such as namelists). The SRW App experiment generation system also contains a set of workflow end-to-end (WE2E) tests that exercise various configurations of the system (e.g., different grids, physics suites). Data for running a subset of these WE2E tests are also included within this registry. <br/> <br/> Users can generate forecasts for dates not included in this data registry by downloading and manually adding raw model files for the desired dates. Many of these model files are publicly available and can be accessed via links on the &quot;<a href="https://dtcenter.org/nwp-containers-online-tutorial/publicly-available-data-sets">Developmental Testbed Center</a>&quot; webs...</p>
          <p><a href="/noaa-ufs-shortrangeweather/">Details &rarr;</a></p>
        </div>
        <div id="noaa-ufs-regtests" class="dataset">
          <h3><a href="/noaa-ufs-regtests/">NOAA Unified Forecast System Weather Model (UFS-WM) Regression Tests</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Unified Forecast System (UFS) is a community-based, coupled, comprehensive Earth Modeling System. The ufs-weather-model (UFS-WM) is the model source    of the UFS for NOAA’s operational numerical weather prediction applications. The UFS-WM Regression Test (RT) is the testing software to ensure that previously developed and tested capabilities in UFS-WM still work after code changes are integrated into the system. It is required that UFS-WM RTs are performed successfully on the required Tier-1 platforms whenever code changes are made to the UFS-WM.  The results of the UFS-WM RTs are summarized in log files and these files will be committed to the UFS-WM repository along with the code changes. Currently, the UFS-WM RTs have been developed to support several applications targeted for operational implementations including the global weather forecast, subseasonal to seasonal forecasts, hurricane forecast, regional rapid refresh forecast, and ocean analysis. <br/> <br/> At this time, there are 123 regression tests to support the UFS applications. The tests are evolving along with the development merged to the UFS-WM  code repository. The regression test framework has been developed in the UFS-WM to run these tests on tier-1 supported systems. Each of the regression tests require a set of input data files and configuration files. The configuration files include namelist and model configuration files residing within the UFS-WM code repository. The input data includes initial conditions, climatology data, and fixed data sets such as orographic data and grid sp</br/>...</p>
          <p><a href="/noaa-ufs-regtests/">Details &rarr;</a></p>
        </div>
        <div id="noaa-wsa-enlil" class="dataset">
          <h3><a href="/noaa-wsa-enlil/">NOAA Wang Sheeley Arge (WSA) Enlil</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">solar</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The WSA-Enlil heliospheric model provides critical information regarding the propagation of solar Coronal Mass Ejections (CMEs) and transient structures within the heliosphere. Two distinct models comprise the WSA-Enlil modeling system; 1) the Wang-Sheeley-Arge (WSA) semi-empirical solar coronal model, and 2) the Enlil magnetohydrodynamic (MHD) heliospheric model. MHD modeling of the full domain (solar photosphere to Earth) is extremely computationally demanding due to the large parameter space and resulting characteristic speeds within the system. To reduce the computational burden and improve the timeliness (and hence the utility in forecasting space weather disturbances) of model results, the domain of the MHD model (Enlil) is limited from 21.5 Solar Radii (R_s) to just beyond the orbit of Earth, while the inner portion, spanning from the solar photosphere to 21.5R_s, is characterized by the WSA model. This coupled modeling system is driven by solar synoptic maps composed of numerous magnetogram observations from the National Solar Observatory’s (NSO) Global Oscillation Network Group (GONG). Such maps provide a full surface description of solar photospheric magnetic flux density, while not accounting for the evolution of surface features for regions outside the view of the observatories. <br/> <br/> In its current configuration (NOAA WSA-Enlil V3.0), the modeling system consists of WSA V5.4 and Enlil V2.9e. The system relies upon the zero point corrected GONG synoptic maps (mrzqs) to define the inner photospheric boundary. <br/> <br/> The operational data files provided in this bucket include NetCDF files containing 3-dimensional gridded neutral density from 100 to 1000 km, Total Electron Content (TEC), and Maximum Usable Frequency (MUF). <br/> <br/> The full 3D datasets from the operational model are provided here as compressed tar files with naming convention wsa_enlil.mrid########.full3d.tgz. These files consist of the full set of 3D datacubes (tim.<em>.nc), all time series results stored at predefined observation points (evo.</em>.nc), and supplemented by the operational CME fits (conefiles) and the operationally...</p>
          <p><a href="/noaa-wsa-enlil/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nws-wam-ipe" class="dataset">
          <h3><a href="/noaa-nws-wam-ipe/">NOAA Whole Atmosphere Model-Ionosphere Plasmasphere Electrodynamics (WAM-IPE) Forecast System (WFS)</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">solar</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The coupled Whole Atmosphere Model-Ionosphere Plasmasphere Electrodynamics (WAM-IPE) Forecast System (WFS) is developed and maintained by the NOAA Space Weather Prediction Center (SWPC). The WAM-IPE model provides a specification of ionosphere and thermosphere conditions with real-time nowcasts and forecasts up to two days in advance in response to solar, geomagnetic, and lower atmospheric forcing. The WAM is an extension of the Global Forecast System (GFS) with a spectral hydrostatic dynamical core utilizing an enthalpy thermodynamic variable to 150 vertical levels on a hybrid pressure-sigma grid, with a model top of approximately 3 x 10-7 Pa (typically 400-600km depending on levels of solar activity). Additional upper atmospheric physics and chemistry, including electrodynamics and plasma processes, are included. The IPE model provides the plasma component of the atmosphere. It is a time-dependent, global 3D model of the ionosphere and plasmasphere from 90 km to approximately 10,000 km. WAM fields of winds, temperature, and molecular and atomic atmospheric composition are coupled to IPE to enable the plasma to respond to changes driven by the neutral atmosphere. <br/> <br/>
The operational WAM-IPE is currently running in two different Concepts of Operation (CONOPS) to produce results of Nowcast and Forecast. The WAM-IPE real-time nowcast system (WRS) ingests real-time solar wind parameters every 5 minutes from NOAA’s spacecrafts located at Lagrange point 1 (L1) between the Sun and Earth in order to capture rapid changes in the ionosphere and thermosphere due to the sudden onset of geomagnetic storms. The nowcast is reinitialized every six hours to include the latest forcing from the lower atmosphere. The forecast system (WFS) runs four times daily (0, 6, 12, 18 UT), providing two-day forecasts. Observed solar wind parameters are used whenever observational values are available, for the forecast portion, the forecasted 3-hour Kp and daily F10.7 issued by SWFO are ingested into the model to estimate solar wind parameters. Lower atmospheric data assimilation only carries out twice daily at 0 and 12 UT cycles to maintain the stability of the coupled model. Model version v1.2 became available in July 2023, featuring the implementation of the WRS into operations, as well as improvements to the Kp-derived solar wind parameters utilized by WFS forecasts. <br/> <br/> The data files within this bucket are provided strictly on a non-operational basis, with no guarantee of timely delivery or availability. There may exist temporal gaps in coverage. <br/> <br/> The top-level versioned directories (v1.x) include NetCDF files from operational runs providing 3-dimensional gridded neutral density every 10 minutes with an altitude range from 100 to 1000 km. wfs.YYYYMMDD subdirectories contain two-day forecasts, updated four times daily (cycle initialization 00, 06, 12, 18 UT). wrs.YYYYMMDD subdirectories contain real-time nowcast neutral density outputs, reinitialized ever</br/>...</p>
          <p><a href="/noaa-nws-wam-ipe/">Details &rarr;</a></p>
        </div>
        <div id="noaa-nos-cora" class="dataset">
          <h3><a href="/noaa-nos-cora/">NOAA&#x27;s Coastal Ocean Reanalysis (CORA) Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">weather</span></p>
          <p>NOAA&#39;s Coastal Ocean Reanalysis (CORA) for the Gulf of Mexico and East Coast (GEC) is produced using verified hourly water levels from the Center of Operational Oceanographic Products &amp; Services (CO-OPS), through hydrodynamic modeling from Advanced Circulation &quot;<a href="https://adcirc.org/">ADCIRC</a>&quot; and Simulating WAves Nearshore &quot;<a href="https://swanmodel.sourceforge.io/">SWAN</a>&quot; models. Data are assimilated, processed, corrected, and processed again before quality assurance and skill assessment with additional verified tide station-based observations.
<br/>
<br/>
Details for CORA Dataset
<br/>
<br/>
<strong>Timeseries</strong> - 1979 to 2022
<br/>
<strong>Size</strong> - Approx. 20.5TB
<br/>
<strong>Domain</strong> - Lat 5.8 to 45.8 ; Long -98.0 to -53.8
<br/>
<strong>Nodes</strong> - 1813443 centroids, 3564104 elements 
<br/>
<strong>Grid cells</strong> - Currently apporximately 505
<br/>
<strong>Spatial Resolution</strong> ...</p>
          <p><a href="/noaa-nos-cora/">Details &rarr;</a></p>
        </div>
        <div id="nanopore" class="dataset">
          <h3><a href="/nanopore/">Nanopore Reference Human Genome</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This dataset includes the sequencing and assembly of a reference standard human genome (GM12878) using the MinION nanopore sequencing instrument with the R9.4 1D chemistry.</p>
          <p><a href="/nanopore/">Details &rarr;</a></p>
        </div>
        <div id="nsd" class="dataset">
          <h3><a href="/nsd/">Natural Scenes Dataset</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">neuroscience</span><span class="label label-info tag link-tag">nifti</span></p>
          <p>Here, we collected and pre-processed a massive, high-quality 7T fMRI dataset that can be used to advance our understanding of how the brain works. A unique feature of this dataset is the massive amount of data available per individual subject. The data were acquired using ultra-high-field fMRI (7T, whole-brain, 1.8-mm resolution, 1.6-s TR). We measured fMRI responses while each of 8 participants viewed 9,000–10,000 distinct, color natural scenes (22,500–30,000 trials) in 30–40 weekly scan sessions over the course of a year. Additional measures were collected including resting-state data, retin...</p>
          <p><a href="/nsd/">Details &rarr;</a></p>
        </div>
        <div id="epa-2022-modeling-platform" class="dataset">
          <h3><a href="/epa-2022-modeling-platform/">OAQPS 2022 Modeling Platform </a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">regulatory</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The data are part of the 2022 Modeling Platform used to support regulatory actions and technical analyses conducted by the EPA&#39;s Office of Air Quality Planning and Standards. Specifically, this data includes Weather Research and Forecasting Model (v4.4.2) conducted at a 12-km resolution over the Continental United States (12US). MCIP-processed files and wrfcamx-processed (12US1 domain) are also available as part of this dataset  to assist in the use of emissions processing and photochemical modeling. These files  may be used in downstream applications to generate emissions, photochemical  ...</p>
          <p><a href="/epa-2022-modeling-platform/">Details &rarr;</a></p>
        </div>
        <div id="ons-opendata-portal" class="dataset">
          <h3><a href="/ons-opendata-portal/">ONS Open Data Portal</a></h3>
          <p><span class="label label-info tag link-tag">electricity</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">hydrography</span></p>
          <p>The ONS Open Data Portal, produced by the National Operator of the Electric System (ONS), gathers historical data from the Brazilian electricity sector in an easy and democratic way with the main objective to facilitate and improve the access and consumption of this type of content by all users and audiences. The Portal gathers consolidated data on energy generation (hydraulic, thermal, solar and wind), areas of national and international energy exchange and energy load, information about equipments such as transmission lines, generating units, converters and others; and hydrological data obta...</p>
          <p><a href="/ons-opendata-portal/">Details &rarr;</a></p>
        </div>
        <div id="openfoodfacts-images" class="dataset">
          <h3><a href="/openfoodfacts-images/">Open Food Facts Images</a></h3>
          <p><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>A dataset of all images of Open Food Facts, the biggest open dataset of food products in the world.</p>
          <p><a href="/openfoodfacts-images/">Details &rarr;</a></p>
        </div>
        <div id="ooni" class="dataset">
          <h3><a href="/ooni/">Open Observatory of Network Interference (OONI)</a></h3>
          <p><span class="label label-info tag link-tag">internet</span></p>
          <p>A free software, global observation network for detecting censorship, surveillance and traffic manipulation on the internet.</p>
          <p><a href="/ooni/">Details &rarr;</a></p>
        </div>
        <div id="openneuro" class="dataset">
          <h3><a href="/openneuro/">OpenNeuro</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">imaging</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neurobiology</span><span class="label label-info tag link-tag">neuroimaging</span></p>
          <p>OpenNeuro is a database of openly-available brain imaging data. The data are shared according to a Creative Commons CC0 license, providing a broad range of brain imaging data to researchers and citizen scientists alike. The database primarily focuses on functional magnetic resonance imaging (fMRI) data, but also includes other imaging modalities including structural and diffusion MRI, electroencephalography (EEG), and magnetoencephalograpy (MEG). OpenfMRI is a project of the <a href="http://reproducibility.stanford.edu">Center for Reproducible Neuroscience at Stanford University</a>. Development of the OpenNeuro resource has been funded by th...</p>
          <p><a href="/openneuro/">Details &rarr;</a></p>
        </div>
        <div id="oida" class="dataset">
          <h3><a href="/oida/">Opioid Industry Documents Archive (OIDA) Data on AWS</a></h3>
          <p><span class="label label-info tag link-tag">archives</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">pharmaceutical</span><span class="label label-info tag link-tag">text analysis</span><span class="label label-info tag link-tag">txt</span></p>
          <p>The OIDA Data on AWS contain the metadata, documents, and extracted text for all of the documents in the UCSF-JHU Opioid Industry Documents Archive, a growing corpus of internal corporate records and other documents arising from the opioid industry.</p>
          <p><a href="/oida/">Details &rarr;</a></p>
        </div>
        <div id="proj-datum-grids" class="dataset">
          <h3><a href="/proj-datum-grids/">PROJ datum grids</a></h3>
          <p><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">mapping</span></p>
          <p>Horizontal and vertical adjustment datasets for coordinate transformation to be used by PROJ 7 or later. PROJ is a generic coordinate transformation software that transforms geospatial coordinates from one coordinate reference system (CRS) to another. This includes cartographic projections as well as geodetic transformations.</p>
          <p><a href="/proj-datum-grids/">Details &rarr;</a></p>
        </div>
        <div id="panstarrs" class="dataset">
          <h3><a href="/panstarrs/">Pan-STARRS PS1 Survey</a></h3>
          <p><span class="label label-info tag link-tag">astronomy</span></p>
          <p>The PS1 surveys used a 1.8 meter telescope and its 1.4 Gigapixel camera to image the sky in five broadband filters. The largest of these surveys provides coverage of the entire sky north of -30 degrees declination, with approximately 10 observation epochs across 3 years in each filter.</p>
          <p><a href="/panstarrs/">Details &rarr;</a></p>
        </div>
        <div id="physionet" class="dataset">
          <h3><a href="/physionet/">Physionet</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>PhysioNet offers free web access to large collections of recorded physiologic signals (PhysioBank) and related open-source software (PhysioToolkit).</p>
          <p><a href="/physionet/">Details &rarr;</a></p>
        </div>
        <div id="paracrawl" class="dataset">
          <h3><a href="/paracrawl/">Provision of Web-Scale Parallel Corpora for Official European Languages (ParaCrawl)</a></h3>
          <p><span class="label label-info tag link-tag">machine translation</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>ParaCrawl is a set of large parallel corpora to/from English for all official EU languages by a broad web crawling effort. State-of-the-art methods are applied for the entire processing chain from identifying web sites with translated text all the way to collecting, cleaning and delivering parallel corpora that are ready as training data for CEF.AT and translation memories for DG Translation.</p>
          <p><a href="/paracrawl/">Details &rarr;</a></p>
        </div>
        <div id="rsna-screening-mammography-breast-cancer-detection" class="dataset">
          <h3><a href="/rsna-screening-mammography-breast-cancer-detection/">RSNA Screening Mammography Breast Cancer Detection (RSNA-SMBC) Dataset</a></h3>
          <p><span class="label label-info tag link-tag">breast cancer</span><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">csv</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">mammography</span><span class="label label-info tag link-tag">medical image computing</span><span class="label label-info tag link-tag">medical imaging</span><span class="label label-info tag link-tag">radiology</span></p>
          <p>According to the WHO, breast cancer is the most commonly occurring cancer worldwide. In 2020 alone, there were 2.3 million new breast cancer diagnoses and 685,000 deaths. Yet breast cancer mortality in high-income countries has dropped by 40% since the 1980s when health authorities implemented regular mammography screening in age groups considered at risk. Early detection and treatment are critical to reducing cancer fatalities, and your machine learning skills could help streamline the process radiologists use to evaluate screening mammograms. Currently, early detection of breast cancer requi...</p>
          <p><a href="/rsna-screening-mammography-breast-cancer-detection/">Details &rarr;</a></p>
        </div>
        <div id="smn-ar-wrf-dataset" class="dataset">
          <h3><a href="/smn-ar-wrf-dataset/">SMN Hi-Res Weather Forecast over Argentina</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">weather</span></p>
          <p>The Servicio Meteorológico Nacional de Argentina (SMN-Arg), the National Meteorological Service of Argentina, shares its deterministic forecasts generated with WRF 4.0 (Weather and Research Forecasting) initialized at 00 and 12 UTC every day.
<br/>
<br/>
This forecast includes some key hourly surface variables –2 m temperature, 2 m relative humidity, 10 m wind magnitude and direction, and precipitation–, along with other daily variables, minimum and maximum temperature.
<br/>
<br/>
The forecast covers Argentina, Chile, Uruguay, Paraguay and parts of Bolivia and Brazil  in a Lambert conformal projection, with 4 km</br/>...</p>
          <p><a href="/smn-ar-wrf-dataset/">Details &rarr;</a></p>
        </div>
        <div id="sucho" class="dataset">
          <h3><a href="/sucho/">SUCHO Ukrainian Cultural Heritage Web Archives</a></h3>
          <p><span class="label label-info tag link-tag">cultural preservation</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">ukraine</span></p>
          <p>The dataset contains web archives of Open Access collections of digitised cultural heritage from more than 3,000+ websites of Ukrainian cultural institutions, such as museums, libraries or archives. The web archives have been produced by SUCHO, which is a volunteer group of more than 1,300 international cultural heritage professionals – librarians, archivists, researchers, programmers - who have joined forces to save as much digitised cultural heritage during the 2022 invasion of Ukraine before the servers hosting them get destroyed, damaged or go offline for any other reason. The web archives...</p>
          <p><a href="/sucho/">Details &rarr;</a></p>
        </div>
        <div id="sentinel-products-ca-mirror" class="dataset">
          <h3><a href="/sentinel-products-ca-mirror/">Sentinel Near Real-time Canada Mirror | Miroir Sentinel temps quasi réel du Canada</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info-sustainability tag link-tag">sustainability</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>The official Government of Canada (GC) 🍁 Near Real-time (NRT) Sentinel Mirror connected to the <a href="https://www.copernicus.eu">EU Copernicus programme</a>, focused on Canadian coverage. In 2015, <a href="https://www.esa.int/Applications/Observing_the_Earth/Copernicus/Canada_joins_Sentinel_collaborative_ground_segment">Canada joined the Sentinel collaborative ground segment</a> which introduced an NRT Sentinel mirror site for users and programs inside the Government of Canada (GC). In 2022, the <a href="https://defence-industry-space.ec.europa.eu/signature-copernicus-arrangement-between-canadian-space-agency-and-european-commission-2022-05-16_en">Commission signed a Copernicus Arrangement with the Canadian Space Agency</a> with the aim to share each other’s satellite Earth Observation data on the basis of reciprocity. Further to this arrangement as well as ongoing <a href="https://open.canada.ca/en/about-open-government">Open Government</a> efforts, the private mirror was made open to the public, here on the AWS Open Dataset Registry. Additional Sentinel-2 and Sentinel-6 products over Canada are also retrieved from <a href="https://eoportal.eumetsat.int/cas/login">EUMET...</p>
          <p><a href="/sentinel-products-ca-mirror/">Details &rarr;</a></p>
        </div>
        <div id="smithsonian-open-access" class="dataset">
          <h3><a href="/smithsonian-open-access/">Smithsonian Open Access</a></h3>
          <p><span class="label label-info tag link-tag">art</span><span class="label label-info tag link-tag">culture</span><span class="label label-info tag link-tag">encyclopedic</span><span class="label label-info tag link-tag">history</span><span class="label label-info tag link-tag">museum</span></p>
          <p>The Smithsonian’s mission is the &quot;increase and diffusion of knowledge&quot; and has been collecting since 1846. The Smithsonian, through its efforts to digitize its multidisciplinary collections, has created millions of digital assets and related metadata describing the collection objects. On February 25th, 2020, the Smithsonian released over 2.8 million CC0 interdisciplinary 2-D and 3-D images, related metadata, and additionally, research data from researches across the Smithsonian. The 2.8 million &quot;open access&quot; collections are a subset of the Smithsonian’s 155 million objects,...</p>
          <p><a href="/smithsonian-open-access/">Details &rarr;</a></p>
        </div>
        <div id="socialgene" class="dataset">
          <h3><a href="/socialgene/">SocialGene RefSeq Databases</a></h3>
          <p><span class="label label-info tag link-tag">amino acid</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">chemical biology</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">graph</span><span class="label label-info tag link-tag">metagenomics</span><span class="label label-info tag link-tag">microbiome</span><span class="label label-info tag link-tag">pharmaceutical</span><span class="label label-info tag link-tag">protein</span></p>
          <p>Precomputed SocialGene Neo4j graph databases of various sizes built from RefSeq genomes and MIBiG BGCs.</p>
          <p><a href="/socialgene/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2024.08.16.608329v1" class="" target="_none">Creating and leveraging bespoke large-scale knowledge graphs for comparative genomics and multi-omics drug discovery with SocialGene.</a> by C. M. Clark, J. C. Kwan</li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/socialgene/" class="" target="_none">socialgene on pypi.python.org - python module for creating and interacting with SocialGene Neo4j databases</a> by <a href="https://github.com/chasemc" target="_none">Chase M. Clark</a></li></a>
          </li>
          <li>
            <a href="http://socialgene.github.io/precomputed_databases/2023_v0.4.1/aws/aws_instructions/" class="" target="_none">How to use SocialGene on AWS</a> by <a href="chasemc.github.io" target="_none">Chase M. Clark</a></li></a>
          </li>
        </ul>
        <p><a href="/socialgene/#usageexamples">See 3 usage examples &rarr;</a></p>
        </div>
        <div id="software-heritage" class="dataset">
          <h3><a href="/software-heritage/">Software Heritage Graph Dataset</a></h3>
          <p><span class="label label-info tag link-tag">digital preservation</span><span class="label label-info tag link-tag">free software</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">source code</span></p>
          <p><a href="https://www.softwareheritage.org/">Software Heritage</a> is the largest
existing public archive of software source code and accompanying
development history. The Software Heritage Graph Dataset is a fully
deduplicated Merkle DAG representation of the Software Heritage archive.The dataset links together file content identifiers, source code
directories, Version Control System (VCS) commits tracking evolution over
time, up to the full states of VCS repositories as observed by Software
Heritage during periodic crawls. The dataset’s contents come from major
development forges (including GitHub and GitLab), FOSS distributions (e.g.,
Deb...</p>
          <p><a href="/software-heritage/">Details &rarr;</a></p>
        </div>
        <div id="allenai-tqa" class="dataset">
          <h3><a href="/allenai-tqa/">Textbook Question Answering (TQA)</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span></p>
          <p>1,076 textbook lessons, 26,260 questions, 6229 images</p>
          <p><a href="/allenai-tqa/">Details &rarr;</a></p>
        </div>
        <div id="gmsdata" class="dataset">
          <h3><a href="/gmsdata/">The Genome Modeling System</a></h3>
          <p><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The Genome Institute at Washington University has developed a high-throughput, fault-tolerant analysis information management system called the Genome Modeling System (GMS), capable of executing complex, interdependent, and automated genome analysis pipelines at a massive scale. The GMS framework provides detailed tracking of samples and data coupled with reliable and repeatable analysis pipelines. GMS includes a full system image with software and services, expandable from one workstation to a large compute cluster.</p>
          <p><a href="/gmsdata/">Details &rarr;</a></p>
        </div>
        <div id="klarna_productpage_dataset" class="dataset">
          <h3><a href="/klarna_productpage_dataset/">The Klarna Product-Page Dataset</a></h3>
          <p><span class="label label-info tag link-tag">commerce</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">graph</span><span class="label label-info tag link-tag">information retrieval</span><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>A collection of 51,701 product pages from 8175 e-commerce websites across 8 markets (US, GB, SE, NL, FI, NO, DE, AT) with 5 manually labelled elements, specifically, the product price, name and image, add-to-cart and go-to-cart buttons.
The dataset was collected between 2018 and 2019 and is made available has MHTML and as WebTraversalLibrary-format snapshots.</p>
          <p><a href="/klarna_productpage_dataset/">Details &rarr;</a></p>
        </div>
        <div id="mmid" class="dataset">
          <h3><a href="/mmid/">The Massively Multilingual Image Dataset (MMID)</a></h3>
          <p><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">machine translation</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>MMID is a large-scale, massively multilingual dataset of images paired with the words they represent collected at the <a href="https://upenn.edu">University of Pennsylvania</a>.
The dataset is doubly parallel: for each language, words are stored parallel to images that represent the word, <em>and</em> parallel to the word&#39;s translation into English (and corresponding images.)</p>
          <p><a href="/mmid/">Details &rarr;</a></p>
        </div>
        <div id="ucsf-bmsr" class="dataset">
          <h3><a href="/ucsf-bmsr/">The University of California San Francisco Brain Metastases Stereotactic Radiosurgery (UCSF-BMSR) MRI Dataset</a></h3>
          <p><span class="label label-info tag link-tag">cancer</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">magnetic resonance imaging</span><span class="label label-info tag link-tag">medical imaging</span><span class="label label-info tag link-tag">medicine</span><span class="label label-info tag link-tag">radiology</span></p>
          <p>The University of California San Francisco Brain Metastases Stereotactic Radiosurgery (UCSF-BMSR) dataset is a public, clinical, multimodal brain MRI dataset consisting of 560 brain MRIs from 412 patients with expert annotations of 5136 brain metastases. Data consists of registered and skull stripped T1 post-contrast, T1 pre-contrast, FLAIR and subtraction (T1 pre-contrast - T1 post-contrast) images and voxelwise segmentations of enhancing brain metastases in NifTI format. </p>
          <p><a href="/ucsf-bmsr/">Details &rarr;</a></p>
        </div>
        <div id="ucsc-genome-browser" class="dataset">
          <h3><a href="/ucsc-genome-browser/">UCSC Genome Browser Sequence and Annotations</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>The UCSC Genome Browser is an online graphical viewer for genomes, a genome browser, hosted by the University of California, Santa Cruz (UCSC). The interactive website offers access to genome sequence data from a variety of vertebrate and invertebrate species and major model organisms, integrated with a large collection of aligned annotations. This dataset is a copy of the MySQL tables in MyISAM binary and tab-sep format and all binary files in custom formats, sometimes referred as &#39;gbdb&#39;-files. Data from the UCSC Genome Browser is free and open for use by anyone. However, every genome...</p>
          <p><a href="/ucsc-genome-browser/">Details &rarr;</a></p>
        </div>
        <div id="usearch-molecules" class="dataset">
          <h3><a href="/usearch-molecules/">USearch Molecules</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">chemical biology</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">pharmaceutical</span></p>
          <p>Collection of 7 billion small molecules in SMILES notation with 28 billion fingerprints, including MACCS, ECFP4, FCFP4, and PubChem, with pre-constructed USearch indexes over them.</p>
          <p><a href="/usearch-molecules/">Details &rarr;</a></p>
        </div>
        <div id="umbra-open-data" class="dataset">
          <h3><a href="/umbra-open-data/">Umbra Synthetic Aperture Radar (SAR) Open Data</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>Umbra satellites generate the highest resolution Synthetic Aperture Radar (SAR) imagery ever offered from space, up to 16-cm resolution. SAR can capture images at night, through cloud cover, smoke and rain. SAR is unique in its abilities to monitor changes. The Open Data Program (ODP) features over twenty diverse time-series locations that are updated frequently, allowing users to experiment with SAR&#39;s capabilities. We offer single-looked spotlight mode in either 16cm, 25cm, 35cm, 50cm, or 1m resolution, and multi-looked spotlight mode. The ODP also features an assorted collection of over ...</p>
          <p><a href="/umbra-open-data/">Details &rarr;</a></p>
        </div>
        <div id="ubc-sunflower-genome" class="dataset">
          <h3><a href="/ubc-sunflower-genome/">University of British Columbia Sunflower Genome Dataset</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">food security</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>This dataset captures Sunflower&#39;s genetic diversity originating
from thousands of wild, cultivated, and landrace sunflower
individuals distributed across North America.The data consists of raw sequences and associated botanical metadata,
aligned sequences (to three different reference genomes), and sets of
SNPs computed across several cohorts.</p>
          <p><a href="/ubc-sunflower-genome/">Details &rarr;</a></p>
        </div>
        <div id="venus-l2a-cogs" class="dataset">
          <h3><a href="/venus-l2a-cogs/">VENUS L2A Cloud-Optimized GeoTIFFs</a></h3>
          <p><span class="label label-info tag link-tag">activity detection</span><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">cog</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">image processing</span><span class="label label-info tag link-tag">land cover</span><span class="label label-info tag link-tag">natural resource</span><span class="label label-info tag link-tag">satellite imagery</span><span class="label label-info tag link-tag">stac</span></p>
          <p>The <a href="https://www.theia-land.fr/en/product/venus/">Venµs science mission</a> is a joint research mission undertaken by CNES and ISA, 
the Israel Space Agency. It aims to demonstrate the effectiveness of high-resolution multi-temporal observation optimised through 
Copernicus, the global environmental and security monitoring programme. Venµs was launched from the Centre Spatial Guyanais by a 
VEGA rocket, during the night from 2017, August 1st to 2nd. Thanks to its multispectral camera (12 spectral bands in the visible 
and near-infrared ranges, with spectral characteristics provided <a href="https://labo.obs-mip.fr/multitemp/?page_id=14229">here</a>), it 
acquires imagery every 1-2 days over 100+ areas at...</p>
          <p><a href="/venus-l2a-cogs/">Details &rarr;</a></p>
        </div>
        <div id="challenge-2021" class="dataset">
          <h3><a href="/challenge-2021/">Will Two Do? Varying Dimensions in Electrocardiography: The PhysioNet/Computing in Cardiology Challenge 2021</a></h3>
          <p></p>
          <p>The electrocardiogram (ECG) is a non-invasive representation of the electrical activity of the heart. Although the twelve-lead ECG is the standard diagnostic screening system for many cardiological issues, the limited accessibility of twelve-lead ECG devices provides a rationale for smaller, lower-cost, and easier to use devices. While single-lead ECGs are limiting [1], reduced-lead ECG systems hold promise, with evidence that subsets of the standard twelve leads can capture useful information [2], [3], [4] and even be comparable to twelve-lead ECGs in some limited contexts. In 2017 we challen...</p>
          <p><a href="/challenge-2021/">Details &rarr;</a></p>
        </div>
        <div id="allenai-zest" class="dataset">
          <h3><a href="/allenai-zest/">ZEST: ZEroShot  learning  from Task descriptions</a></h3>
          <p><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>ZEST is a benchmark for zero-shot generalization to unseen NLP tasks, with 25K labeled instances across 1,251 different tasks.</p>
          <p><a href="/allenai-zest/">Details &rarr;</a></p>
        </div>
        <div id="inaturalist-open-data" class="dataset">
          <h3><a href="/inaturalist-open-data/">iNaturalist Licensed Observation Images</a></h3>
          <p><span class="label label-info tag link-tag">biodiversity</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">conservation</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">life sciences</span></p>
          <p>iNaturalist is a community science effort in which participants share observations of living organisms that they encounter and document with photographic evidence, location, and date. The community works together reviewing these images to identify these observations to species. This collection represents the licensed images accompanying iNaturalist observations.</p>
          <p><a href="/inaturalist-open-data/">Details &rarr;</a></p>
        </div>
        <div id="stdpopsim_kern" class="dataset">
          <h3><a href="/stdpopsim_kern/">stdpopsim species resources</a></h3>
          <p><span class="label label-info tag link-tag">genetic maps</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">population genetics</span><span class="label label-info tag link-tag">recombination maps</span><span class="label label-info tag link-tag">simulations</span></p>
          <p>Contains all resources (genome specifications, recombination maps, etc.) required for species specific simulation with the stdpopsim package. These resources are originally from a variety of other consortium and published work but are consolidated here for ease of access and use. If you are interested in adding a new species to the stdpopsim resource please raise an issue on the stdpopsim GitHub page to have the necessary files added here.</p>
          <p><a href="/stdpopsim_kern/">Details &rarr;</a></p>
        </div>
        <div id="abeja-cc-ja" class="dataset">
          <h3><a href="/abeja-cc-ja/">ABEJA CC JA</a></h3>
          <p><span class="label label-info tag link-tag">internet</span><span class="label label-info tag link-tag">japanese</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">web archive</span></p>
          <p>A large Japanese language corpus created through preprocessing Common Crawl data</p>
          <p><a href="/abeja-cc-ja/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://tech-blog.abeja.asia/entry/abeja-nedo-project-part2-202405" class="" target="_none">Building a Large-Scale Japanese Corpus from Common Crawl and Its Preprocessing</a> by Kyo Hattori</li></a>
          </li>
          <li>
            <a href="https://github.com/abeja-inc/Megatron-LM/blob/main/docs/dataset/tutorials.md" class="" target="_none">Tutorial of ABEJA CC JA dataset</a> by Kyo Hattori</li></a>
          </li>
        </ul>
        <p><a href="/abeja-cc-ja/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aws-public-blockchain" class="dataset">
          <h3><a href="/aws-public-blockchain/">AWS Public Blockchain Data</a></h3>
          <p><span class="label label-info tag link-tag">blockchain</span><span class="label label-info tag link-tag">web3</span></p>
          <p>The AWS Public Blockchain Data provides free access to blockchain datasets. Data is transformed into multiple  tables as compressed Parquet files, partitioned by date, to allow efficient access for most common analytics queries.
</br></br>
<strong>Datasets</strong></br></br> <table width="100%"> <thead>
  <tr><th>Blockchain dataset</th><th>Maintained by</th><th>Path</th></tr></thead> <tbody>
  <tr><td>Bitcoin</td>     <td>AWS</td>    <td><code>s3://aws-public-blockchain/v1.0/btc/</code></td></tr>
  <tr><td>Ethereum</td>    <td>AWS</td>    <td><code>s3://aws-public-blockchain/v1.0/eth/</code></td></tr>
  <tr><td>Arbitrum</td>    <td>SonarX</td> <td><code>s3://aws-public-blockchain/v1.1/sonarx/arbitrum/</code></td></tr>
  <tr><td>Aptos</td>       <td>SonarX</td> <td><code>s3://aws-public-blockchain/v1.1/sonarx/aptos/</code></td></tr>
  <tr><td>Base</td>        <td>SonarX</td> <td><code>s3://aws-public-blockchain/v1.1/sonarx/base/</code></td></tr>
  <tr><td>Provenance</td>  <td>SonarX</td> <td><code>s3://aws-public-blockchain/v1.1/sonarx/provenance/</code></td></tr>
  <tr><td>XRP Ledger</td>  <td>SonarX</td> <td><code>s3://aws-public-blockchain/v1.1/sonarx/xrp/</code></td></tr>
</tbody> </table> </br> For full datasets, with support and real-time updates, please visit <a href="https://sonarx.com">SonarX</a>....</p>
          <p><a href="/aws-public-blockchain/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://repost.aws/articles/AR3gztQGeSS8CfaKNNeyYwsQ" class="" target="_none">New datasets added to the AWS Public Blockchain Datasets — available for analytics and research</a> by Everton Fraga, Simon Goldberg</li></a>
          </li>
          <li>
            <a href="https://aws.amazon.com/blogs/database/access-bitcoin-and-ethereum-open-datasets-for-cross-chain-analytics/" class="aws-link" target="_none">Access Bitcoin and Ethereum open datasets for cross-chain analytics</a> by Oliver Steffmann, Bhaskar Ravat, Sreeji Gopal, and Stefan Dicker</li></a>
          </li>
        </ul>
        <p><a href="/aws-public-blockchain/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="amazon-bin-imagery" class="dataset">
          <h3><a href="/amazon-bin-imagery/">Amazon Bin Image Dataset</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>The Amazon Bin Image Dataset contains over 500,000 images and metadata from bins of a pod in an operating Amazon Fulfillment Center. The bin images in this dataset are captured as robot units carry pods as part of normal Amazon Fulfillment Center operations.</p>
          <p><a href="/amazon-bin-imagery/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/OneNow/AI-Inventory-Reconciliation" class="" target="_none">Amazon Inventory Reconciliation using AI</a> by <a href="https://github.com/pablo-tech" target="_none">Pablo Rodriguez Bertorello, Sravan Sripada, Nutchapol Dendumrongsup</a></li></a>
          </li>
          <li>
            <a href="https://github.com/silverbottlep/abid_challenge" class="" target="_none">Amazon Bin Image Dataset Challenge</a> by <a href="https://github.com/silverbottlep" target="_none">silverbottlep</a></li></a>
          </li>
        </ul>
        <p><a href="/amazon-bin-imagery/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_animal_acoustic_tracking_delayed_qc" class="dataset">
          <h3><a href="/aodn_animal_acoustic_tracking_delayed_qc/">Animal Tracking - Acoustic Telemetry - Quality controlled detections</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">marine mammals</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>Since 2007, the Integrated Marine Observing System’s Animal Tracking Facility (formerly known as the Australian Animal Tracking And Monitoring System (AATAMS)) has established a permanent array of acoustic receivers around Australia to detect the movements of tagged marine animals in coastal waters. Simultaneously, the Animal Tracking Facility developed a centralised national database (<a href="https://animaltracking.aodn.org.au/">https://animaltracking.aodn.org.au/</a>) to encourage collaborative research across the Australian research community and provide unprecedented opportunities to monitor broad-scale animal movements. The resulting da...</p>
          <p><a href="/aodn_animal_acoustic_tracking_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/animal_acoustic_tracking_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - Animal Tracking Facility - Acoustic Tracking - Quality Controlled Detections (2007 - ongoing)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_animal_acoustic_tracking_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="dep-mangroves" class="dataset">
          <h3><a href="/dep-mangroves/">Digital Earth Pacific Mangroves Extent and Density</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span></p>
          <p>Pacific Mangroves beta version product is an extension of
the Global Mangrove Watch (GMV v3, 2020). which shows the
extent of mangrove ecosystems across Pacific Island Countries
and Territories (PICTs). The changes in mangroves extent was
further classified into three categories of closed (high-density),
open (lower density) and non-mangrove. This was used as the
baseline training layer where mangrove categories between
2016 and 2022 were analysed.</p>
          <p><a href="/dep-mangroves/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://maps.digitalearthpacific.org/#share&#x3D;s-ux1MvvWAfZueW286xCPv" class="" target="_none">Digital Earth Pacific Map</a> by Digital Earth Pacific Contributors</li></a>
          </li>
          <li>
            <a href="https://digitalearthpacific.github.io/data-access/" class="" target="_none">Digital Earth Pacific Open Data Access Documentation</a> by <a href="https://github.com/digitalearthpacific" target="_none">digitalearthpacific</a></li></a>
          </li>
        </ul>
        <p><a href="/dep-mangroves/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="dep-wofs" class="dataset">
          <h3><a href="/dep-wofs/">Digital Earth Pacific Water Observatins from Space (WOfS)</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span><span class="label label-info tag link-tag">water</span></p>
          <p>Water Observations from Space (WOfS) beta version product for Water
Observations from Space (WOfS) is an annual summary of the temporal
and spatial extent of surface water over landscapes. In essence, this
highlights where water is usually or where it is rarely. The results
are visualised to compare points in time spanning over a year, a season
or multiple years. The dataset extends back historically to 2013.</p>
          <p><a href="/dep-wofs/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://digitalearthpacific.github.io/data-access/" class="" target="_none">Digital Earth Pacific Open Data Access Documentation</a> by <a href="https://github.com/digitalearthpacific" target="_none">digitalearthpacific</a></li></a>
          </li>
          <li>
            <a href="https://maps.digitalearthpacific.org/#share&#x3D;s-azdGYqKlUn1BWnljBhtB" class="" target="_none">Digital Earth Pacific Map</a> by Digital Earth Pacific Contributors</li></a>
          </li>
        </ul>
        <p><a href="/dep-wofs/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="census-2010-amc-mdf-replicates" class="dataset">
          <h3><a href="/census-2010-amc-mdf-replicates/">Estimating Confidence Intervals for 2020 Census Statistics Using Approximate Monte Carlo Simulation (2010 Census Proof of Concept)</a></h3>
          <p><span class="label label-info tag link-tag">age</span><span class="label label-info tag link-tag">approximate monte carlo</span><span class="label label-info tag link-tag">approximate monte carlo replicates</span><span class="label label-info tag link-tag">census</span><span class="label label-info tag link-tag">demographic and housing characteristics file</span><span class="label label-info tag link-tag">dhc</span><span class="label label-info tag link-tag">differential privacy</span><span class="label label-info tag link-tag">disclosure avoidance</span><span class="label label-info tag link-tag">ethnicity</span><span class="label label-info tag link-tag">group quarters</span><span class="label label-info tag link-tag">hispanic</span><span class="label label-info tag link-tag">household type</span><span class="label label-info tag link-tag">housing</span><span class="label label-info tag link-tag">housing units</span><span class="label label-info tag link-tag">latino</span><span class="label label-info tag link-tag">microdata</span><span class="label label-info tag link-tag">noisy measurements</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">race</span><span class="label label-info tag link-tag">redistricting</span><span class="label label-info tag link-tag">relation-to-householder</span><span class="label label-info tag link-tag">single year of age</span><span class="label label-info tag link-tag">voting age</span></p>
          <p>The 2010 Census Production Settings Demographic and Housing Characteristics (DHC) Approximate Monte Carlo (AMC) method seed Privacy Protected Microdata File (PPMF0) and PPMF replicates (PPMF1, PPMF2, ..., PPMF25) are a set of microdata files intended for use in estimating the magnitude of error(s) introduced by the 2020  Decennial Census Disclosure Avoidance System (DAS) into the Redistricting and DHC products. The PPMF0 was created by executing the 2020 DAS TopDown Algorithm (TDA) using the confidential  2010 Census Edited File (CEF) as the initial input; the replicates were then created by executing the 2020 DAS TDA repeatedly with the PPMF0 as its initial input.  Inspired by analogy to the use of bootstrap methods in non-private contexts, U.S. Census Bureau (USCB) researchers explored whether simple calculations  based on comparing each PPMFi to the PPMF0 could be used to reliably estimate the scale of errors introduced by the 2020 DAS, and generally found this approach  worked well. <br/> <br/> The PPMF0 and PPMFi files contained here are provided so that external researchers can estimate properties of DAS-introduced error without privileged  access to internal USCB-curated data sets; further information on the estimation methodology can be found in <a href="https://github.com/uscensusbureau/AMC_Confidence_Intervals/blob/main/Approx_Monte_Carlo_confidence_interval_paper.pdf">Ashmead et. al 2024</a>.  <br/> <br/> The 2010 DHC AMC seed PPMF0 and PPMF replicates have been cleared for public dissemination by the USCB Disclosure Review Board (CBDRB-FY24-DSEP-0002).  The 2010 PPMF0 included in these files was produced using the same parameters and settings as were used to produce the 2010 Demonstration Data Product  Suite (202...</p>
          <p><a href="/census-2010-amc-mdf-replicates/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/uscensusbureau/AMC_Confidence_Intervals/blob/main/Approx_Monte_Carlo_confidence_interval_paper.pdf" class="" target="_none">An Approximate Monte Carlo Simulation Method for Estimating Uncertainty and Constructing Confidence Intervals for 2020 Census Statistics</a> by Ashmead, R., Hawes, M. B., Pritts, M., Zhuravlev, P., Keller, S. A.</li></a>
          </li>
          <li>
            <a href="https://github.com/uscensusbureau/AMC_Confidence_Intervals/tree/main" class="" target="_none">Estimating Confidence Intervals Using Approximate Monte Carlo Simulation Iterations (Jupyter Notebook)</a> by Ashmead, R., Hawes, M. B., Pritts, M., Zhuravlev, P., Keller, S. A.</li></a>
          </li>
        </ul>
        <p><a href="/census-2010-amc-mdf-replicates/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="census-2020-amc-mdf-replicates" class="dataset">
          <h3><a href="/census-2020-amc-mdf-replicates/">Estimating Confidence Intervals for 2020 Census Statistics Using Approximate Monte Carlo Simulation (2020 Census Production Run)</a></h3>
          <p><span class="label label-info tag link-tag">2020 census</span><span class="label label-info tag link-tag">age</span><span class="label label-info tag link-tag">approximate monte carlo</span><span class="label label-info tag link-tag">approximate monte carlo replicates</span><span class="label label-info tag link-tag">census</span><span class="label label-info tag link-tag">decennial census</span><span class="label label-info tag link-tag">demographic and housing characteristics file</span><span class="label label-info tag link-tag">dhc</span><span class="label label-info tag link-tag">differential privacy</span><span class="label label-info tag link-tag">disclosure avoidance</span><span class="label label-info tag link-tag">ethnicity</span><span class="label label-info tag link-tag">group quarters</span><span class="label label-info tag link-tag">hispanic</span><span class="label label-info tag link-tag">household type</span><span class="label label-info tag link-tag">housing</span><span class="label label-info tag link-tag">housing units</span><span class="label label-info tag link-tag">latino</span><span class="label label-info tag link-tag">microdata</span><span class="label label-info tag link-tag">noisy measurements</span><span class="label label-info tag link-tag">population</span><span class="label label-info tag link-tag">race</span><span class="label label-info tag link-tag">redistricting</span><span class="label label-info tag link-tag">relation-to-householder</span><span class="label label-info tag link-tag">single year of age</span><span class="label label-info tag link-tag">voting age</span></p>
          <p>The 2020 Census Production Settings Demographic and Housing Characteristics (DHC) Approximate Monte Carlo (AMC) method seed Privacy Protected Microdata File (PPMF0) and PPMF replicates (PPMF1, PPMF2, ..., PPMF50) are a set of microdata files intended for use in estimating the magnitude of error(s) introduced by the 2020  Census Disclosure Avoidance System (DAS) into the 2020 Census Redistricting Data Summary File (P.L. 94-171), the Demographic and Housing Characteristics File, and the Demographic Profile.  <br/> <br/> The PPMF0 was the source of the publicly released, official 2020 Census data products referenced above, and was created by executing the 2020 DAS TopDown Algorithm (TDA) using the confidential  2020 Census Edited File (CEF) as the initial input; the official location for the PPMF0 is <a href="https://www2.census.gov/programs-surveys/decennial/2020/data/privacy-protected-microdata-file/">on the United States Census Bureau FTP server</a>, but we also include a copy of it here for convenience. The replicates were then created by executing the 2020 DAS TDA repeatedly with the PPMF0 as its initial input.  <br/> <br/> Inspired by analogy to the use of bootstrap methods in non-private contexts, U.S. Census Bureau (USCB) researchers explored whether simple calculations  based on comparing each PPMFi to the PPMF0 could be used to reliably estimate the scale of errors introduced by the 2020 DAS, and generally found this approach  worked well. <br/> <br/> The PPMF0 and PPMFi files contained here are provided so that external researchers can estimate properties of DAS-introduced error without privileged  access to internal USCB-curated data sets; further information on the estimation methodology can be found in <a href="https://github.com/uscensusbureau/AMC_Confidence_Intervals/blob/main/Approx_Monte_Carlo_confidence_interval_paper.pdf">Ashmead et. al 2024</a>.  <br/> <br/> The 2020 DHC AMC...</p>
          <p><a href="/census-2020-amc-mdf-replicates/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/uscensusbureau/AMC_Confidence_Intervals/tree/main" class="" target="_none">Estimating Confidence Intervals Using Approximate Monte Carlo Simulation Iterations (Jupyter Notebook)</a> by Ashmead, R., Hawes, M. B., Pritts, M., Zhuravlev, P., Keller, S. A.</li></a>
          </li>
          <li>
            <a href="https://github.com/uscensusbureau/AMC_Confidence_Intervals/blob/main/Approx_Monte_Carlo_confidence_interval_paper.pdf" class="" target="_none">An Approximate Monte Carlo Simulation Method for Estimating Uncertainty and Constructing Confidence Intervals for 2020 Census Statistics</a> by Ashmead, R., Hawes, M. B., Pritts, M., Zhuravlev, P., Keller, S. A.</li></a>
          </li>
        </ul>
        <p><a href="/census-2020-amc-mdf-replicates/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="genomekit" class="dataset">
          <h3><a href="/genomekit/">GenomeKit genomic data</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">genome</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">non-human primate</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">Rattus norvegicus</span><span class="label label-info tag link-tag">variant annotation</span></p>
          <p>GenomeKit is Deep Genomics’ Python library for fast and easy access to genomic resources such as sequence, data tracks, and annotations. The goal is to let machine learning researchers build data sets easily, and to be creative about how those data sets are designed. Out of the box, GenomeKit provides access to pre-built optimized genomic data files that are required for its operation.</p>
          <p><a href="/genomekit/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2023.09.20.558508v1" class="" target="_none">An RNA foundation model enables discovery of disease mechanisms and candidate therapeutics</a> by <a href="https://www.biorxiv.org/content/10.1101/2023.09.20.558508v1.article-info" target="_none">Albi Celaj, Alice Jiexin Gao, Tammy T.Y. Lau, Erle M. Holgersen, Alston Lo, Varun Lodaya, Christopher B. Cole, Robert E. Denroche, Carl Spickett, Omar Wagih, Pedro O. Pinheiro, Parth Vora, Pedrum Mohammadi-Shemirani, Steve Chan, Zach Nussbaum, Xi Zhang, Helen Zhu, Easwaran Ramamurthy, Bhargav Kanuparthi, Michael Iacocca, Diane Ly, Ken Kron, Marta Verby, Kahlin Cheung-Ong, Zvi Shalev, Brandon Vaz, Sakshi Bhargava, Farhan Yusuf, Sharon Samuel, Sabriyeh Alibai, Zahra Baghestani, Xinwen He, Kirsten Krastel, Oladipo Oladapo, Amrudha Mohan, Arathi Shanavas, Magdalena Bugno, Jovanka Bogojeski, Frank Schmitges, Carolyn Kim, Solomon Grant, Rachana Jayaraman, Tehmina Masud, Amit Deshwar, Shreshth Gandhi, Brendan J. Frey</a></li></a>
          </li>
          <li>
            <a href="https://deepgenomics.github.io/GenomeKit/quickstart.html" class="" target="_none">Quickstart</a> by Deep Genomics</li></a>
          </li>
        </ul>
        <p><a href="/genomekit/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="dep-ls-geomads" class="dataset">
          <h3><a href="/dep-ls-geomads/">Landsat Geometric Median and Absolute Deviations (GeoMAD) over the Pacific.</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span></p>
          <p>The GeoMAD is derived from Landsat surface reflectance data.
The data are masked for cloud, shadows and other image artefacts
using the associated pixel quality product to help provide as
clear a set of observations as possible from which to calculate the medians.</p>
          <p><a href="/dep-ls-geomads/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://digitalearthpacific.github.io/data-access/" class="" target="_none">Digital Earth Pacific Open Data Access Documentation</a> by <a href="https://github.com/digitalearthpacific" target="_none">digitalearthpacific</a></li></a>
          </li>
          <li>
            <a href="https://maps.digitalearthpacific.org/" class="" target="_none">Digital Earth Pacific Map</a> by Digital Earth Pacific Contributors</li></a>
          </li>
        </ul>
        <p><a href="/dep-ls-geomads/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_animal_ctd_satellite_relay_tagging_delayed_qc" class="dataset">
          <h3><a href="/aodn_animal_ctd_satellite_relay_tagging_delayed_qc/">Marine Animal - Satellite Relay Tagging -  Quality controlled profiles</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">chemical biology</span><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">marine mammals</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>CTD (Conductivity-Temperature_Depth)-Satellite Relay Data Loggers (CTD-SRDLs) are used to explore how marine animal behaviour relates to their oceanic environment. Loggers developed at the University of St Andrews Sea Mammal Research Unit transmit data in near real-time via the Argo satellite system. Data represented here was collected in the Southern Ocean, from elephant, fur and Weddell Seals. In 2024 data was added from flatback and olive ridley turtles, from a pilot study co-funded by the Royal Australian Navy in collaboration with the Australian Institute of Marine Science and Indigenous ...</p>
          <p><a href="/aodn_animal_ctd_satellite_relay_tagging_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/animal_ctd_satellite_relay_tagging_delayed_qc.ipynb" class="" target="_none">Accessing Satellite Relay Tagging Program - Southern Ocean - MEOP Quality Controlled CTD Profiles</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_animal_ctd_satellite_relay_tagging_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_mooring_hourly_timeseries_delayed_qc" class="dataset">
          <h3><a href="/aodn_mooring_hourly_timeseries_delayed_qc/">Moorings - Hourly time-series product</a></h3>
          <p><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>Integrated Marine Observing System (IMOS) have moorings across both it&#39;s National Mooring Network and Deep Water Moorings facilities. The National Mooring Network facility comprises a series of national reference stations and regional moorings designed to monitor particular oceanographic phenomena in Australian coastal ocean waters. The Deep Water Moorings facility (formerly known as the Australian Bluewater Observing System) provides the coordination of national efforts in the sustained observation of open ocean properties with particular emphasis on observations important to climate and ...</p>
          <p><a href="/aodn_mooring_hourly_timeseries_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/mooring_hourly_timeseries_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - Moorings - Hourly time-series product</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_mooring_hourly_timeseries_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="nasa-soteria-data" class="dataset">
          <h3><a href="/nasa-soteria-data/">NASA SOTERIA Simulation Testbed Data</a></h3>
          <p><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">neuroimaging</span><span class="label label-info tag link-tag">transportation</span><span class="label label-info tag link-tag">workload analysis</span></p>
          <p>Commercial pilot simulation data during safety-of-flight scenarios.</p>
          <p><a href="/nasa-soteria-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ntrs.nasa.gov/citations/20230008151/" class="" target="_none">SOTERIA Simulation - Experimental Methods, Data Processing, and Data Quality</a> by Tyler Fettrow, Chad Stephens, Lance Prinzel, Jon Holbrook, Sepher Bastami, Michael Stewart, Kathryn Ballard, Daniel Kiggins</li></a>
          </li>
          <li>
            <a href="https://github.com/tylerfettrow/soteria-processing-python/" class="" target="_none">Python Processing Code</a> by Tyler Fettrow</li></a>
          </li>
        </ul>
        <p><a href="/nasa-soteria-data/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_mooring_ctd_delayed_qc" class="dataset">
          <h3><a href="/aodn_mooring_ctd_delayed_qc/">National Mooring Network - CTD profiles</a></h3>
          <p><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>This collection includes conductivity-temperature-depth (CTD) profiles obtained at the National Reference Stations (NRS) as part of the water sampling program. The instruments used also measure dissolved oxygen, fluorescence, and turbidity. The collection also includes practical salinity, water density and artificial chlorophyll concentration, as computed from the measured properties. The data are processed in delayed mode, with automated quality control applied. The National Reference Station network is designed to provide baseline information, at timescales relevant to human response, that i...</p>
          <p><a href="/aodn_mooring_ctd_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/mooring_ctd_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - Australian National Mooring Network (ANMN) - CTD Profiles</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_mooring_ctd_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_slocum_glider_delayed_qc" class="dataset">
          <h3><a href="/aodn_slocum_glider_delayed_qc/">Ocean Gliders - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Australian National Facility for Ocean Gliders (ANFOG), with IMOS/NCRIS funding, deploys a fleet of eight gliders around Australia. The data represented by this record, are presented in delayed mode. The underwater ocean glider represents a technological revolution for oceanography. Autonomous ocean gliders can be built relatively cheaply, are controlled remotely and are reusable allowing them to make repeated subsurface ocean observations at a fraction of the cost of conventional methods. The data retrieved from the glider fleet will contribute to the study of the major boundary current s...</p>
          <p><a href="/aodn_slocum_glider_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/slocum_glider_delayed_qc.ipynb" class="" target="_none">Accessing Ocean Gliders - delayed mode glider deployments</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_slocum_glider_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_radar_bonneycoast_velocity_hourly_averaged_delayed_qc" class="dataset">
          <h3><a href="/aodn_radar_bonneycoast_velocity_hourly_averaged_delayed_qc/">Ocean Radar - Bonney coast site - Sea water velocity - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Bonney Coast (BONC) HF ocean radar system covers an area of the Bonney Coast, South Australia, which has a recurring annual upwelling feature near to the coast that significantly changes the ecosystem from one of warm water originating in Western Australia, to one dominated by cold upwelling water from off the continental shelf.  The dynamics of this area and the relationship between ocean circulation, chemistry and sediments control the larval species and the higher marine species and ecosystems in which they forage.  The data from this site provide linking observations between the Southe...</p>
          <p><a href="/aodn_radar_bonneycoast_velocity_hourly_averaged_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/radar_BonneyCoast_velocity_hourly_averaged_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - ACORN - Bonney Coast HF ocean radar site (South Australia, Australia) - Delayed mode sea water velocity</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_radar_bonneycoast_velocity_hourly_averaged_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_radar_capricornbunkergroup_velocity_hourly_averaged_delayed_qc" class="dataset">
          <h3><a href="/aodn_radar_capricornbunkergroup_velocity_hourly_averaged_delayed_qc/">Ocean Radar - Capricorn bunker group site - Sea water velocity - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Capricorn Bunker Group site is in the southern region  of the Great Barrier Reef Marine Park World Heritage Area (GBR).  The HF ocean radar coverage is from the coast to beyond the edge of the continental shelf.  This is an area where the East Australian Current (EAC) meanders as it moves south from the Swain Reefs and loses touch with the western land boundary.  The area is dynamic with warm EAC water recirculating and being wind-driven northwards along the coast inside the GBR lagoon.  The recirculating warm water contrasts with the upwelling tendency of the parts of the EAC which contin...</p>
          <p><a href="/aodn_radar_capricornbunkergroup_velocity_hourly_averaged_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/radar_CapricornBunkerGroup_velocity_hourly_averaged_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - ACORN - Capricorn Bunker Group HF ocean radar site (Great Barrier Reef, Queensland, Australia) - Delayed mode sea water velocity</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_radar_capricornbunkergroup_velocity_hourly_averaged_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_radar_coffsharbour_velocity_hourly_averaged_delayed_qc" class="dataset">
          <h3><a href="/aodn_radar_coffsharbour_velocity_hourly_averaged_delayed_qc/">Ocean Radar - Coffs Harbour site - Sea water velocity - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Coffs Harbour (COF) HF ocean radar site is located near the point at which the East Australian Current (EAC) begins to separate from the coast.  Here the EAC is at its narrowest and swiftest: to the north it is forming from the westwards subtropical jet, and to the south it forms eddies and eventually the warm water moves eastwards across the Tasman Sea, forming a front with the cold water of the Southern Ocean.  The connection between coastal and continental shelf waters is fundamental to the understanding of the anthropogenic impact on the coastal ocean and the role of the ocean in mitig...</p>
          <p><a href="/aodn_radar_coffsharbour_velocity_hourly_averaged_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/radar_CoffsHarbour_velocity_hourly_averaged_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - ACORN - Coffs Harbour HF ocean radar site (New South Wales, Australia) - Delayed mode sea water velocity</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_radar_coffsharbour_velocity_hourly_averaged_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_radar_coralcoast_velocity_hourly_averaged_delayed_qc" class="dataset">
          <h3><a href="/aodn_radar_coralcoast_velocity_hourly_averaged_delayed_qc/">Ocean Radar - Coral coast site - Sea water velocity - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Coral Coast (CORL) HF ocean radar system covers an area of the Western Australia Coast, Western Australia, an area subject to the variability of the Leeuwin Current (LC) and its coupling with coastal winds, tides, and waves. In this area the LC generates several eddies which control the larval species and the higher marine species and ecosystems in which they forage.The CORL HF ocean radar system consists of two SeaSonde crossed loop direction finding stations located at Dongara (29.283 S 114.920E) and Green Head (114.967 E 30.073 S).  These radars operate at a frequency of 4.463 MHz, with...</p>
          <p><a href="/aodn_radar_coralcoast_velocity_hourly_averaged_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/radar_CoralCoast_velocity_hourly_averaged_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - ACORN - Coral Coast HF ocean radar site (Western Australia, Australia) - Delayed mode sea water velocity</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_radar_coralcoast_velocity_hourly_averaged_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_radar_newcastle_velocity_hourly_averaged_delayed_qc" class="dataset">
          <h3><a href="/aodn_radar_newcastle_velocity_hourly_averaged_delayed_qc/">Ocean Radar - Newcastle site - Sea water velocity - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Newcastle (NEWC) HF ocean radar system covers an area of the Central Coast, New South Wales, an area subject to the variability of the East Australian Current (EAC) and its coupling with coastal winds, tides, and waves. In this area the EAC separates from the coast and generates several eddies which control the larval species and the higher marine species and ecosystems in which they forage.The NEWC HF ocean radar system consists of two SeaSonde crossed loop direction finding stations located at Sea Rocks (32.441575 S 152.539022 E) and Red Head (33.010245 S 151.727059 E).  These radars ope...</p>
          <p><a href="/aodn_radar_newcastle_velocity_hourly_averaged_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/radar_Newcastle_velocity_hourly_averaged_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - ACORN - Newcastle HF ocean radar site (New South Wales, Australia) - Delayed mode sea water velocity</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_radar_newcastle_velocity_hourly_averaged_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_radar_northwestshelf_velocity_hourly_averaged_delayed_qc" class="dataset">
          <h3><a href="/aodn_radar_northwestshelf_velocity_hourly_averaged_delayed_qc/">Ocean Radar - Northwest shelf site - Sea water velocity - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Northwest Shelf (NWA) HF ocean radar system covers an area which includes the Ningaloo Peninsula and the Ningaloo Reef to the west.  The Ningaloo Reef is one of the longest and most pristine reefs in the world. The reef is rich in marine biodiversity, with shark whales, turtles and fish aggregations, and high primary and secondary productions which are controlled by the physical oceanographic processes.  The NWA HF ocean radar is a WERA phased array system with 12-element receive arrays located at the Jurabi Turtle Centre (21.8068 S, 114.1015 E) and Point Billie (22.5432 S, 113.690 E).  Th...</p>
          <p><a href="/aodn_radar_northwestshelf_velocity_hourly_averaged_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/radar_NorthWestShelf_velocity_hourly_averaged_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - ACORN - Northwest Shelf HF ocean radar site (Western Australia, Australia) - Delayed mode sea water velocity</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_radar_northwestshelf_velocity_hourly_averaged_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_radar_rottnestshelf_velocity_hourly_averaged_delayed_qc" class="dataset">
          <h3><a href="/aodn_radar_rottnestshelf_velocity_hourly_averaged_delayed_qc/">Ocean Radar - Rottnest shelf site - Sea water velocity - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Rottnest Shelf (ROT) HF ocean radar system covers an area which includes Rottnest Island and the Perth Canyon to the north-west.  The Perth Canyon has the highest marine biodiversity in the region with whale and fish aggregations, and high primary and secondary productions which are controlled by the physical oceanographic processes.  Combined with the dynamics of the Perth Canyon is the dominant Leeuwin Current which produces a wake on the leeward side of Rottnest Island.   This is a topographically induced up-welling and associated primary and secondary productivity.  The region is influ...</p>
          <p><a href="/aodn_radar_rottnestshelf_velocity_hourly_averaged_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/radar_RottnestShelf_velocity_hourly_averaged_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - ACORN - Rottnest Shelf HF ocean radar site (Western Australia, Australia) - Delayed mode sea water velocity</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_radar_rottnestshelf_velocity_hourly_averaged_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_radar_southaustraliagulfs_velocity_hourly_averaged_delayed_qc_test" class="dataset">
          <h3><a href="/aodn_radar_southaustraliagulfs_velocity_hourly_averaged_delayed_qc_test/">Ocean Radar - South Australia Gulfs site (South Australia, Australia) - Sea water velocity - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The South Australia Gulfs (SAG) HF ocean radar system covers the area of about 40,000 square kilometres bounded by Kangaroo Island to the east and the Eyre Peninsula to the north.  This is a dynamic region where warm water from the remnants of the Leeuwin current is moving from the west, and water with varying density is exchanging with Spencer Gulf and the Gulf of St Vincent.  Upwelling events occur from the deep ocean on the south side of the observation area.  This is a key ocean area for aquaculture and fishing, and is a major shipping thoroughfare.  The data from this HF ocean radar syste...</p>
          <p><a href="/aodn_radar_southaustraliagulfs_velocity_hourly_averaged_delayed_qc_test/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/radar_SouthAustraliaGulfs_velocity_hourly_averaged_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - ACORN - South Australia Gulfs HF ocean radar site (South Australia, Australia) - Delayed mode sea water velocity</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_radar_southaustraliagulfs_velocity_hourly_averaged_delayed_qc_test/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_radar_turquoisecoast_velocity_hourly_averaged_delayed_qc" class="dataset">
          <h3><a href="/aodn_radar_turquoisecoast_velocity_hourly_averaged_delayed_qc/">Ocean Radar - Turquoise coast site - Sea water velocity - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Turquoise Coast (TURQ) HF ocean radar system covers the area of shelf between Seabird and Jurien Bay and is the logical continuation of major research efforts to understand the role of the Leeuwin Current System (Leeuwin Current, the Leeuwin Undercurrent and Capes Current) in controlling not only the physical system but also its links to both pelagic and benthic ecosystems.  In contrast to eastern ocean basins, which are highly productive, Western Australia experiences an oligotrophic environment. The Leeuwin Current is a shallow (&lt;300 m deep), narrow band (&lt; 100 km wide) of warm, lo...</p>
          <p><a href="/aodn_radar_turquoisecoast_velocity_hourly_averaged_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/radar_TurquoiseCoast_velocity_hourly_averaged_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - ACORN - Turquoise Coast HF ocean radar site (Western Australia, Australia) - Delayed mode sea water velocity</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_radar_turquoisecoast_velocity_hourly_averaged_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_model_sea_level_anomaly_gridded_realtime" class="dataset">
          <h3><a href="/aodn_model_sea_level_anomaly_gridded_realtime/">OceanCurrent - Gridded sea level anomaly - Near real time</a></h3>
          <p><span class="label label-info tag link-tag">ocean sea surface height</span><span class="label label-info tag link-tag">ocean velocity</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>Gridded (adjusted) sea level anomaly (GSLA), gridded sea level (GSL) and surface geostrophic velocity (UCUR,VCUR) for the Australasian region. GSLA is mapped using optimal interpolation of detided, de-meaned, inverse-barometer-adjusted altimeter and tidegauge estimates of sea level. GSL is GSLA plus an estimate of the departure of mean sea level from the geoid – mean sea level (over 18 years of model time) of Ocean Forecasting Australia Model version 3 (OFAM3). The geostrophic velocities are derived from GSLA and the mean surface velocity from OFAM3. The altimeter data window for input to the ...</p>
          <p><a href="/aodn_model_sea_level_anomaly_gridded_realtime/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/model_sea_level_anomaly_gridded_realtime.ipynb" class="" target="_none">Accessing IMOS - OceanCurrent - Gridded sea level anomaly - Near real time</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_model_sea_level_anomaly_gridded_realtime/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="dep-coastlines" class="dataset">
          <h3><a href="/dep-coastlines/">Pacific Coastlines Change</a></h3>
          <p><span class="label label-info tag link-tag">coastal</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span></p>
          <p>Pacific Coastlines beta version product includes coastline change detection
since the year 2000 for Pacific Island Country and Territories (PICTs).
This product will provide ongoing monitoring of coastline change detection.
This provides insights into processes including erosion (where landmass
area decreases) and accretion or deposition (where landmass area increases).</p>
          <p><a href="/dep-coastlines/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://digitalearthpacific.github.io/data-access/" class="" target="_none">Digital Earth Pacific Open Data Access Documentation</a> by <a href="https://github.com/digitalearthpacific" target="_none">digitalearthpacific</a></li></a>
          </li>
          <li>
            <a href="https://maps.digitalearthpacific.org/#share&#x3D;s-pzzMOVCCoPWNeWFs33Sg" class="" target="_none">Digital Earth Pacific Map</a> by Digital Earth Pacific Contributors</li></a>
          </li>
        </ul>
        <p><a href="/dep-coastlines/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="platinum-pedigree" class="dataset">
          <h3><a href="/platinum-pedigree/">Platinum Pedigree</a></h3>
          <p><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">genotyping</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">long read sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>The Platinum Pedigree Consortium (PCC) is a collaborative project to create a comprehensive reference for human genetic variation using a four-generation, 28-member family (CEPH-1463). We employed five different short and long-read sequencing technologies to generate phased assemblies and characterize both inherited and de novo variation, including at some of the most difficult to genotype genomic regions such as tandem repeats, centromeres, and the Y chromosome. This extensive &quot;truth set&quot; is publicly available and can be used to test and benchmark new algorithms and technologies to ...</p>
          <p><a href="/platinum-pedigree/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2024.10.02.616333v1" class="" target="_none">The Platinum Pedigree: A long-read benchmark for genetic variants</a> by Kronenberg et. al.</li></a>
          </li>
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2024.08.05.606142v1" class="" target="_none">A familial, telomere-to-telomere reference for human de novo mutation and recombination from a four-generation pedigree</a> by <a href="https://eichlerlab.gs.washington.edu/porubsky.html" target="_none">Porubsky et. al.</a></li></a>
          </li>
        </ul>
        <p><a href="/platinum-pedigree/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="ssl4eo-multi-product-data" class="dataset">
          <h3><a href="/ssl4eo-multi-product-data/">SSL4EO S12 Landsat Multi Product Dataset</a></h3>
          <p><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>This dataset combines SSL4EO-S12 and SSL4EO-L to create a multi-view dataset for multi-modal fusion using self-supervised learning for earth observation.</p>
          <p><a href="/ssl4eo-multi-product-data/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ieeexplore.ieee.org/abstract/document/7989594" class="" target="_none">SSL4EO-S12 - A Large-Scale Multi-Modal, Multi-Temporal Dataset for Self-Supervised Learning in Earth Observation</a> by <a href="https://scholar.google.de/citations?user&#x3D;rEADuMEAAAAJ&amp;hl&#x3D;en" target="_none">Yi Wang, Nassim Ait Ali Braham, Zhitong Xiong, Chenying Liu, Conrad M Albrecht, Xiao Xiang Zhu</a></li></a>
          </li>
          <li>
            <a href="https://proceedings.neurips.cc/paper_files/paper/2023/file/bbf7ee04e2aefec136ecf60e346c2e61-Paper-Datasets_and_Benchmarks.pdf" class="" target="_none">SSL4EO-L - Datasets and Foundation Models for Landsat Imagery</a> by <a href="https://github.com/adamjstewart" target="_none">Adam J. Stewart, Nils Lehmann, Isaac A. Corley, Yi Wang, Yi-Chia Chang, Nassim Ait Ali Braham, Shradha Sehgal, Caleb Robinson, Arindam Banerjee</a></li></a>
          </li>
        </ul>
        <p><a href="/ssl4eo-multi-product-data/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="surface-pm2-5-v6gl02" class="dataset">
          <h3><a href="/surface-pm2-5-v6gl02/">SatPM2.5</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">health</span><span class="label label-info tag link-tag">netcdf</span></p>
          <p>Fine particulate matter (PM2.5) concentrations are estimated using information from satellite-, simulation- and monitor-based sources. Aerosol optical depth from multiple satellites (MODIS, VIIRS, MISR, SeaWiFS, and VIIRS) and their respective retrievals (Dark Target, Deep Blue, MAIAC) is combined with simulation (GEOS-Chem) based upon their relative uncertainties as determined using ground-based sun photometer (AERONET) observations to produce geophysical estimates that explain most of the variance in ground-based PM2.5 measurements. A subsequent statistical fusion incorporates additional inf...</p>
          <p><a href="/surface-pm2-5-v6gl02/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://pubs.acs.org/doi/full/10.1021/acsestair.3c00054" class="" target="_none">Enhancing Global Estimation of Fine Particulate Matter Concentrations by Including Geophysical a Priori Information in Deep Learning</a> by <a href="https://sites.wustl.edu/acag/" target="_none">Siyuan Shen, Chi Li, Aaron van Donkelaar, Nathan Jacobs, Chenguang Wang, and Randall V. Martin</a></li></a>
          </li>
          <li>
            <a href="https://sites.wustl.edu/acag/importing-and-plotting-the-v6-gl-02-02-dataset-into-matlab/" class="" target="_none">Importing and Plotting the V6.GL.02.02 dataset into Matlab</a> by <a href="https://sites.wustl.edu/acag/people/aaron-van-donkelaar/" target="_none">Aaron van Donkelaar</a></li></a>
          </li>
        </ul>
        <p><a href="/surface-pm2-5-v6gl02/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_mooring_satellite_altimetry_calibration_validation" class="dataset">
          <h3><a href="/aodn_mooring_satellite_altimetry_calibration_validation/">Satellite - Altimetry calibration and validation</a></h3>
          <p><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">ocean currents</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>High precision satellite altimeter missions including TOPEX/Poseidon (T/P), Jason-1 and now OSTM/Jason-2, have contributed fundamental advances in our understanding of regional and global ocean circulation and its role in the Earth&#39;s climate and regional applications. These altimeter satellites essentially observe the height of the global oceans – as such, they have become the tool of choice for scientists to measure sea level rise – both at regional and global scales as well as giving information about ocean currents and large- and small-scale variability. The determination of changes in ...</p>
          <p><a href="/aodn_mooring_satellite_altimetry_calibration_validation/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/mooring_satellite_altimetry_calibration_validation.ipynb" class="" target="_none">Accessing IMOS - SRS Satellite Altimetry Calibration and Validation Sub-Facility</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_mooring_satellite_altimetry_calibration_validation/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_chlorophylla_carder_1day_aqua" class="dataset">
          <h3><a href="/aodn_satellite_chlorophylla_carder_1day_aqua/">Satellite - Ocean Colour -  MODIS - 1 day - Chlorophyll-a concentration (Carder model)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Aqua satellite platform carries a MODIS sensor that observes sunlight reflected from within the ocean surface layer at multiple wavelengths. These multi-spectral measurements are used to infer the concentration of chlorophyll-a (Chl-a), most typically due to phytoplankton, present in the water.There are multiple retrieval algorithms for estimating Chl-a. These data use the Carder method implemented in the SeaDAS processing software l2gen and described in Carder K. L., Chen F. R., Lee Z. P., Hawes S. K. and Cannizzaro J. P. (2003), MODIS Ocean Science Team Algorithm Theoretical Basis Docume...</p>
          <p><a href="/aodn_satellite_chlorophylla_carder_1day_aqua/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_chlorophylla_carder_1day_aqua.ipynb" class="" target="_none">Accessing IMOS - SRS - MODIS - 01 day - Chlorophyll-a concentration (Carder model)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_chlorophylla_carder_1day_aqua/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_chlorophylla_gsm_1day_aqua" class="dataset">
          <h3><a href="/aodn_satellite_chlorophylla_gsm_1day_aqua/">Satellite - Ocean Colour -  MODIS - 1 day - Chlorophyll-a concentration (GSM model)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Aqua satellite platform carries a MODIS sensor that observes sunlight reflected from within the ocean surface layer at multiple wavelengths. These multi-spectral measurements are used to infer the concentration of chlorophyll-a (Chl-a), most typically due to phytoplankton, present in the water. There are multiple retrieval algorithms for estimating Chl-a. These data use the Garver-Siegel-Maritorena (GSM) method   implemented in the SeaDAS processing software l2gen and described in “Chapter 11, and references therein, of IOCCG Report 5, 2006, (<a href="http://ioccg.org/wp-content/uploads/2015/10/ioc...</p>
          <p><a href="/aodn_satellite_chlorophylla_gsm_1day_aqua/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_chlorophylla_gsm_1day_aqua.ipynb" class="" target="_none">Accessing IMOS - SRS - MODIS - 01 day - Chlorophyll-a concentration (GSM model)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_chlorophylla_gsm_1day_aqua/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_chlorophylla_oc3_1day_aqua" class="dataset">
          <h3><a href="/aodn_satellite_chlorophylla_oc3_1day_aqua/">Satellite - Ocean Colour -  MODIS - 1 day - Chlorophyll-a concentration (OC3 model)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Aqua satellite platform carries a MODIS sensor that observes sunlight reflected from within the ocean surface layer at multiple wavelengths. These multi-spectral measurements are used to infer the concentration of chlorophyll-a (Chl-a), most typically due to phytoplankton, present in the water. There are multiple retrieval algorithms for estimating Chl-a. These data use the OC3 method recommended by the NASA Ocean Biology Processing Group and implemented in the SeaDAS processing software l2gen. The OC3 algorithm is described at <a href="http://oceancolor.gsfc.nasa.gov/cms/atbd/chlor_a">http://oceancolor.gsfc.nasa.gov/cms/atbd/chlor_a</a> (and links th...</p>
          <p><a href="/aodn_satellite_chlorophylla_oc3_1day_aqua/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_chlorophylla_oc3_1day_aqua.ipynb" class="" target="_none">Accessing IMOS - SRS - MODIS - 01 day - Chlorophyll-a concentration (OC3 model)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_chlorophylla_oc3_1day_aqua/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_chlorophylla_oci_1day_aqua" class="dataset">
          <h3><a href="/aodn_satellite_chlorophylla_oci_1day_aqua/">Satellite - Ocean Colour -  MODIS - 1 day - Chlorophyll-a concentration (OCI model)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Aqua satellite platform carries a MODIS sensor that observes sunlight reflected from within the ocean surface layer at multiple wavelengths. These multi-spectral measurements are used to infer the concentration of chlorophyll-a (Chl-a), most typically due to phytoplankton, present in the water. There are multiple retrieval algorithms for estimating Chl-a. These data use the OCI method (Hu et al 2012, doi: 10.1029/2011jc007395) recommended by the NASA Ocean Biology Processing Group and implemented in the SeaDAS processing software l2gen. The OCI algorithm is described at <a href="https://oceancolor.gsfc.nasa.gov/atbd/chlor_a/">https://oceancolor.gsfc.nasa.gov/atbd/chlor_a/</a>  (and links therei...</p>
          <p><a href="/aodn_satellite_chlorophylla_oci_1day_aqua/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_chlorophylla_oci_1day_aqua.ipynb" class="" target="_none">Accessing IMOS - SRS - MODIS - 01 day - Chlorophyll-a concentration (OCI model)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_chlorophylla_oci_1day_aqua/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_diffuse_attenuation_coefficent_1day_aqua" class="dataset">
          <h3><a href="/aodn_satellite_diffuse_attenuation_coefficent_1day_aqua/">Satellite - Ocean Colour - MODIS - 1 day - Diffuse attenuation coefficient (k490)</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Aqua satellite platform carries a MODIS sensor that observes sunlight reflected from within the ocean surface layer at multiple wavelengths. These multi-spectral measurements are used to infer the diffuse attenuation coefficient (Kd) at 490nm wavelength which provides information on how light is attenuated in the water column. It is defined as the scaling length of the exponential decrease of the downwelling irradiance and has units (m^-1). The MODIS K490 product estimates Kd at 490nm wavelength, using a semi-empirical model based on the ratio of water leaving radiances at 490nm and 555nm....</p>
          <p><a href="/aodn_satellite_diffuse_attenuation_coefficent_1day_aqua/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_diffuse_attenuation_coefficent_1day_aqua.ipynb" class="" target="_none">Accessing IMOS - SRS - MODIS - 01 day - Diffuse attenuation coefficient (k490 )</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_diffuse_attenuation_coefficent_1day_aqua/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_nanoplankton_fraction_oc3_1day_aqua" class="dataset">
          <h3><a href="/aodn_satellite_nanoplankton_fraction_oc3_1day_aqua/">Satellite - Ocean Colour - MODIS - 1 day - Nanoplankton fraction (OC3 model and Brewin et al 2012 algorithm)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Aqua satellite platform carries a MODIS sensor that observes sunlight reflected from within the ocean surface layer at multiple wavelengths. These multi-spectral measurements are used to infer the concentration of chlorophyll-a (Chl-a), most typically due to phytoplankton, present in the water. An empirical relationship is then used to compute an estimate of the relative abundance of three phytoplankton size classes (micro, nano and picoplankton). The methods used to decompose chl_oc3 are described by Brewin et al in two papers in 2010 and 2012. The two methods, denoted Brewin2010at and Br...</p>
          <p><a href="/aodn_satellite_nanoplankton_fraction_oc3_1day_aqua/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_nanoplankton_fraction_oc3_1day_aqua.ipynb" class="" target="_none">Accessing IMOS - SRS - MODIS - 01 day - Nanoplankton fraction (OC3 model and Brewin et al 2012 algorithm)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_nanoplankton_fraction_oc3_1day_aqua/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_net_primary_productivity_gsm_1day_aqua" class="dataset">
          <h3><a href="/aodn_satellite_net_primary_productivity_gsm_1day_aqua/">Satellite - Ocean Colour - MODIS - 1 day - Net Primary Productivity (GSM model and Eppley-VGPM algorithm)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Aqua satellite platform carries a MODIS sensor that observes sunlight reflected from within the ocean surface layer at multiple wavelengths. These multi-spectral measurements are used to infer the concentration of chlorophyll-a (Chl-a), most typically due to phytoplankton, present in the water. Modelling is then used to compute an estimate of the Net Primary Productivity (NPP).The model used is based on the standard vertically generalised production model (VGPM).  The VGPM is a &quot;chlorophyll-based&quot; model that estimates net primary production from chlorophyll using a temperature-de...</p>
          <p><a href="/aodn_satellite_net_primary_productivity_gsm_1day_aqua/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_net_primary_productivity_gsm_1day_aqua.ipynb" class="" target="_none">Accessing IMOS - SRS - MODIS - 01 day - Net Primary Productivity (GSM model and Eppley-VGPM algorithm)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_net_primary_productivity_gsm_1day_aqua/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_net_primary_productivity_oc3_1day_aqua" class="dataset">
          <h3><a href="/aodn_satellite_net_primary_productivity_oc3_1day_aqua/">Satellite - Ocean Colour - MODIS - 1 day - Net Primary Productivity (OC3 model and Eppley-VGPM algorithm)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Aqua satellite platform carries a MODIS sensor that observes sunlight reflected from within the ocean surface layer at multiple wavelengths. These multi-spectral measurements are used to infer the concentration of chlorophyll-a (Chl-a), most typically due to phytoplankton, present in the water. Modelling is then used to compute an estimate of the Net Primary Productivity (NPP).The model used is based on the standard vertically generalised production model (VGPM).  The VGPM is a &quot;chlorophyll-based&quot; model that estimates net primary production from chlorophyll using a temperature-de...</p>
          <p><a href="/aodn_satellite_net_primary_productivity_oc3_1day_aqua/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_net_primary_productivity_oc3_1day_aqua.ipynb" class="" target="_none">Accessing IMOS - SRS - MODIS - 01 day - Net Primary Productivity (OC3 model and Eppley-VGPM algorithm)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_net_primary_productivity_oc3_1day_aqua/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_optical_water_type_1day_aqua" class="dataset">
          <h3><a href="/aodn_satellite_optical_water_type_1day_aqua/">Satellite - Ocean Colour - MODIS - 1 day - Optical Water Type (Moore et al 2009 algorithm)</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Aqua satellite platform carries a MODIS sensor that observes sunlight reflected from within the ocean surface layer at multiple wavelengths. These measurements at discrete wavelengths represent the spectrum of light leaving the water surface, and the shape of the spectrum is characteristic of the water optical properties.Moore et al. (2009) applied a clustering technique to spectra to identify 8 sets of discrete optical water types. This product &quot;owt_csiro&quot; is produced using a CSIRO implementation of the Moore et al. algorithm, and testing shows that it closely reproduces the res...</p>
          <p><a href="/aodn_satellite_optical_water_type_1day_aqua/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_optical_water_type_1day_aqua.ipynb" class="" target="_none">Accessing IMOS - SRS - MODIS - 01 day - Optical Water Type (Moore et al 2009 algorithm)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_optical_water_type_1day_aqua/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_picoplankton_fraction_oc3_1day_aqua" class="dataset">
          <h3><a href="/aodn_satellite_picoplankton_fraction_oc3_1day_aqua/">Satellite - Ocean Colour - MODIS - 1 day - Picoplankton fraction (OC3 model and Brewin et al 2012 algorithm)</a></h3>
          <p><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>The Aqua satellite platform carries a MODIS sensor that observes sunlight reflected from within the ocean surface layer at multiple wavelengths. These multi-spectral measurements are used to infer the concentration of chlorophyll-a (Chl-a), most typically due to phytoplankton, present in the water. An empirical relationship is then used to compute an estimate of the relative abundance of three phytoplankton size classes (micro, nano and picoplankton). The methods used to decompose chl_oc3 are described by Brewin et al in two papers in 2010 and 2012. The two methods, denoted Brewin2010at and Br...</p>
          <p><a href="/aodn_satellite_picoplankton_fraction_oc3_1day_aqua/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_picoplankton_fraction_oc3_1day_aqua.ipynb" class="" target="_none">Accessing IMOS - SRS - MODIS - 01 day - Picoplankton fraction (OC3 model and Brewin et al 2012 algorithm)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_picoplankton_fraction_oc3_1day_aqua/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_ghrsst_l3s_1day_daynighttime_single_sensor_australia" class="dataset">
          <h3><a href="/aodn_satellite_ghrsst_l3s_1day_daynighttime_single_sensor_australia/">Satellite - Sea surface temperature - Level 3 - Single sensor - 1 day - Day and night time</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>This is a single-sensor multi-satellite  SSTfnd product for a single 24 hour period, derived using observations from AVHRR instruments on all available NOAA polar-orbiting satellites.  It is provided as a 0.02deg x 0.02deg cylindrical equidistant projected map over the region 70°E to 170°W, 20°N to 70°S.  Each grid cell contains the 24 hour average of all the highest available quality SSTs that overlap with that cell, weighted by the area of overlap.  The diagram at <a href="https://help.aodn.org.au/satellite-data-product-information/">https://help.aodn.org.au/satellite-data-product-information/</a> indicates where this product fits within the GHRSST suite of NOAA/AVHRR products. The SSTfnd is derived by adding a constant 0.17 degC to the SSTskin observations after rejecting observations with low surface wind speeds (&lt;6m/s by day and &lt;2m/s at night) (see <a href="http://www.bom.gov.au/amoj/docs/2011/beggs.pdf">http://www.bom.gov.au/amoj/docs/2011/beggs.pdf</a>). Matchups with buoy SSTfnd observations indicate typical 2014 biases of &lt; 0.01 degC and standard deviations of 0.6 degC. Refer to the IMOS SST products web page at <a href="http://imos.org.au/sstproducts.html">http://imos.org.au/sstproducts.html</a> and Beggs et al. (2013) at <a href="http://imos.org.au/sstdata_references.html">http://imos.org.au/sstdata_references.html</a> for further informatio...</p>
          <p><a href="/aodn_satellite_ghrsst_l3s_1day_daynighttime_single_sensor_australia/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_ghrsst_l3s_1day_daynighttime_single_sensor_australia.ipynb" class="" target="_none">Accessing IMOS - SRS - SST - L3S - Single Sensor - 1 day - day and night time - Australia</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_ghrsst_l3s_1day_daynighttime_single_sensor_australia/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_ghrsst_l3s_1day_daynighttime_single_sensor_southernocean" class="dataset">
          <h3><a href="/aodn_satellite_ghrsst_l3s_1day_daynighttime_single_sensor_southernocean/">Satellite - Sea surface temperature - Level 3 - Single sensor - 1 day - Day and night time - Southern Ocean</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>This is a single-sensor SSTfnd product for a single 24 hour period, derived using observations from AVHRR instruments on all available NOAA polar-orbiting satellites.  It is provided as a 0.02deg x 0.02deg cylindrical equidistant projected map over the Southern Ocean region 3°E to 158°W, 27°S to 78°S.  Each grid cell contains the 24 hour average of all the highest available quality SSTs that overlap with that cell, weighted by the area of overlap.  The diagram at <a href="https://help.aodn.org.au/satellite-data-product-information/">https://help.aodn.org.au/satellite-data-product-information/</a> indicates where this product fits within the GHRSST suite of NOAA/AVHRR products. The SSTfnd is derived by adding a constant 0.17 degC to the SSTskin observations after rejecting observations with low surface wind speeds (&lt;6m/s by day and &lt;2m/s at night) (see <a href="http://www.bom.gov.au/amoj/docs/2011/beggs.pdf">http://www.bom.gov.au/amoj/docs/2011/beggs.pdf</a>). Matchups with buoy SSTfnd observations indicate typical 2014 biases of &lt; 0.01 degC and standard deviations of 0.6 degC. Refer to the IMOS SST products web page at <a href="http://imos.org.au/sstproducts.html">http://imos.org.au/sstproducts.html</a> and Beggs et al. (2013) at <a href="http://imos.org.au/sstdata...</p>
          <p><a href="/aodn_satellite_ghrsst_l3s_1day_daynighttime_single_sensor_southernocean/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_ghrsst_l3s_1day_daynighttime_single_sensor_southernocean.ipynb" class="" target="_none">Accessing IMOS - SRS - SST - L3S - Single Sensor - 1 day - day and night time - Southern Ocean</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_ghrsst_l3s_1day_daynighttime_single_sensor_southernocean/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_ghrsst_l3c_1day_nighttime_himawari8" class="dataset">
          <h3><a href="/aodn_satellite_ghrsst_l3c_1day_nighttime_himawari8/">Satellite - Sea surface temperature - Level 3 - Single sensor - Himawari-8 - 1 day - Night time</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>This is a regional GHRSST level 3 collated (L3C) dataset on 0.02-degree rectangular grid over the Australasian domain (70E to 190E, 70S to 20N) based on retrievals from the AHI imager on board Himawari-8 satellite. The Bureau of Meteorology (Bureau) produces Integrated Marine Observing System (IMOS) satellite SST products in the International Group for High Resolution SST (GHRSST) GDS2 file formats for Himawari-8 in real time and delayed mode. This product is composed of reprocessed multi-swath SSTskin retrievals obtained from compositing IMOS Himawari-8 hourly L3C files over the night (before dawn).<br>Every 10 minutes, the Himawari-8 full disk is processed to retrieve SSTs by using the Radiative Transfer Model (RTTOV12.3) and Bayesian cloud clearing method based on the ESA CCI SST code developed at the University of Reading. The hourly product on native grid is then produced by compositing over multiple swaths for the previous 1 hour. Further, L3C-01day SST fields on native grid are obtained by selecting the highest quality data from the hourly SST records with priority being given to the value the closest in time to the product nominal hour.  The L3C-1day night product on native grid is th</br>...</p>
          <p><a href="/aodn_satellite_ghrsst_l3c_1day_nighttime_himawari8/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_ghrsst_l3c_1day_nighttime_himawari8.ipynb" class="" target="_none">Accessing IMOS - SRS - SST - L3C - Himawari-8 - 1 day - night time - Australia</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_ghrsst_l3c_1day_nighttime_himawari8/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_ghrsst_l4_gamssa_1day_multi_sensor_world" class="dataset">
          <h3><a href="/aodn_satellite_ghrsst_l4_gamssa_1day_multi_sensor_world/">Satellite - Sea surface temperature - Level 4 - Multi sensor - Global Australian</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>An International Group for High-Resolution Sea Surface Temperature (GHRSST) Level 4 sea surface temperature analysis, produced daily on an operational basis at the Australian Bureau of Meteorology using optimal interpolation (OI) on a global 0.25 degree grid.  This Global Australian Multi-Sensor SST Analysis (GAMSSA) v1.0 system blends infra-red SST observations from the Advanced Very High Resolution Radiometer (AVHRR) on NOAA and METOP polar-orbiting satellites, microwave SST observations from the Advanced Microwave Scanning Radiometer-2 (AMSR-2) on GCOM-W, and in situ data from ships, and dr...</p>
          <p><a href="/aodn_satellite_ghrsst_l4_gamssa_1day_multi_sensor_world/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_ghrsst_l4_gamssa_1day_multi_sensor_world.ipynb" class="" target="_none">Accessing IMOS - SRS - SST - L4 - GAMSSA - World</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_ghrsst_l4_gamssa_1day_multi_sensor_world/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_satellite_ghrsst_l4_ramssa_1day_multi_sensor_australia" class="dataset">
          <h3><a href="/aodn_satellite_ghrsst_l4_ramssa_1day_multi_sensor_australia/">Satellite - Sea surface temperature - Level 4 - Multi sensor - Regional Australian</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">satellite imagery</span></p>
          <p>An International Group for High Resolution Sea Surface Temperature (GHRSST) Level 4 sea surface temperature analysis, produced daily on an operational basis at the Australian Bureau of Meteorology using optimal interpolation (OI) on a regional 1/12 degree grid over the Australian region (20N - 70S, 60E - 170W). This Regional Australian Multi-Sensor SST Analysis (RAMSSA) v1.0 system blends infra-red SST observations from the Advanced Very High Resolution Radiometer (AVHRR) on NOAA and METOP polar-orbiting satellites, microwave SST observations from the Advanced Microwave Scanning Radiometer-2 (...</p>
          <p><a href="/aodn_satellite_ghrsst_l4_ramssa_1day_multi_sensor_australia/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/satellite_ghrsst_l4_ramssa_1day_multi_sensor_australia.ipynb" class="" target="_none">Accessing IMOS - SRS - SST - L4 - RAMSSA - Australia</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_satellite_ghrsst_l4_ramssa_1day_multi_sensor_australia/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="dep-s1-annual-mosaics" class="dataset">
          <h3><a href="/dep-s1-annual-mosaics/">Sentinel-1 Mean and Median Annual Mosaic</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span></p>
          <p>Sentinel-1 carries a Synthetic Aperture RADAR (SAR) that
operates on the C-band. This platform offers SAR data
day and night and in all-weather conditions.</p>
          <p><a href="/dep-s1-annual-mosaics/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://digitalearthpacific.github.io/data-access/" class="" target="_none">Digital Earth Pacific Open Data Access Documentation</a> by <a href="https://github.com/digitalearthpacific" target="_none">digitalearthpacific</a></li></a>
          </li>
          <li>
            <a href="https://maps.digitalearthpacific.org/#share&#x3D;s-pzzMOVCCoPWNeWFs33Sg" class="" target="_none">Digital Earth Pacific Map</a> by Digital Earth Pacific Contributors</li></a>
          </li>
        </ul>
        <p><a href="/dep-s1-annual-mosaics/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="s1-orbits" class="dataset">
          <h3><a href="/s1-orbits/">Sentinel-1 Precise Orbit Determination (POD) Products</a></h3>
          <p><span class="label label-info tag link-tag">auxiliary data</span><span class="label label-info tag link-tag">disaster response</span><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">earthquakes</span><span class="label label-info tag link-tag">floods</span><span class="label label-info tag link-tag">geophysics</span><span class="label label-info tag link-tag">sentinel-1</span><span class="label label-info tag link-tag">synthetic aperture radar</span></p>
          <p>Sentinel-1 Precise Orbit Determination (POD) products contain auxiliary data on satellite position and velocity for the European Space Agency&#39;s (ESA) Sentinel-1 mission. Sentinel-1 is a C-band Synthetic Aperture Radar (SAR) satellite constellation first launched in 2014 as part of the European Union&#39;s Copernicus Earth Observation programme. POD products are a necessary auxiliary input for nearly all Sentinel-1 data processing workflows. <br/><br/> This dataset is a mirror of the <a href="https://documentation.dataspace.copernicus.eu/Data/ComplementaryData/Additional.html#sentinel-1-orbits">Sentinel-1 Orbits</a> dataset hosted in the Copernicus Data Space Ecosystem (CDSE). New files are added within 20 minutes of their publication to CDSE. This dataset includes two types of POD files: RESORB and POEORB. <ul> <li>RESORB POD files are restituted orbit files generated within 180 minutes from sensing time and published multiple times per day, with accuracy requirement of 10 cm in 2D RMS, but typically below 5 cm. RESORB products from the last 90 days are included in this dataset.</li> <br/> <li>POEORB POD files are precise orbit files generated with a timeliness of 20 days from sensing time ...</p>
          <p><a href="/s1-orbits/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://s1-orbits.asf.alaska.edu/ui/" class="" target="_none">REST API for downloading Sentinel-1 Orbit files from the Registry of Open Data on AWS</a> by <a href="https://asf.alaska.edu/" target="_none">Alaska Satellite Facility</a></li></a>
          </li>
          <li>
            <a href="https://pypi.org/project/s1-orbits/" class="" target="_none">s1-orbits, A Python utility for downloading Sentinel-1 Orbit files from the Registry of Open Data on AWS</a> by <a href="https://asf.alaska.edu/" target="_none">Alaska Satellite Facility</a></li></a>
          </li>
        </ul>
        <p><a href="/s1-orbits/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="dep-s2-geomads" class="dataset">
          <h3><a href="/dep-s2-geomads/">Sentinel-2 Geometric Median and Absolute Deviations (GeoMAD) over the Pacific</a></h3>
          <p><span class="label label-info tag link-tag">earth observation</span><span class="label label-info tag link-tag">geoscience</span><span class="label label-info tag link-tag">geospatial</span></p>
          <p>The Geometric Median and Absolute Deviations (GeoMAD) product is
a cloud-free annual mosaic that uses a more robust method of
determining the median observation than a simple median. Along
with the median observation, the GeoMAD produces three measures
of variance, or absolute deviations, which helps to understand
how the data over the time period changes. For example, some areas,
such as desert, will change very little. Whereas crop land will change
more. All ofthese values are useful in understand what is happening
in the area covered by the GeoMAD.</p>
          <p><a href="/dep-s2-geomads/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://maps.digitalearthpacific.org/" class="" target="_none">Digital Earth Pacific Map</a> by Digital Earth Pacific Contributors</li></a>
          </li>
          <li>
            <a href="https://digitalearthpacific.github.io/data-access/" class="" target="_none">Digital Earth Pacific Open Data Access Documentation</a> by <a href="https://github.com/digitalearthpacific" target="_none">digitalearthpacific</a></li></a>
          </li>
        </ul>
        <p><a href="/dep-s2-geomads/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_vessel_air_sea_flux_product_delayed" class="dataset">
          <h3><a href="/aodn_vessel_air_sea_flux_product_delayed/">Ships of Opportunity - Air-sea fluxes - Meteorological and flux - Real time</a></h3>
          <p><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">radiation</span></p>
          <p>Enhancement of Measurements on Ships of Opportunity (SOOP)-Air Sea Flux sub-facility collects underway meteorological and oceanographic observations during scientific and Antarctic resupply voyages in the oceans adjacent to Australia. Data product is quality controlled bulk air-sea fluxes and input observations. Research Vessel Real Time Air-Sea Fluxes, equips the Marine National Facility (MNF) (Research Vessels Southern Surveyor and Investigator), the Australian Antarctic Division (Research and Supply Vessels Aurora Australis and Nuyina), and Research Vessel Tangaroa with &quot;climate qualit...</p>
          <p><a href="/aodn_vessel_air_sea_flux_product_delayed/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/vessel_air_sea_flux_product_delayed.ipynb" class="" target="_none">Accessing IMOS-SOOP-Air Sea Flux: Flux product from the Research Vessel Tangaroa</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_vessel_air_sea_flux_product_delayed/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_vessel_air_sea_flux_sst_meteo_realtime" class="dataset">
          <h3><a href="/aodn_vessel_air_sea_flux_sst_meteo_realtime/">Ships of Opportunity - Air-sea fluxes - Meteorological and sea surface temperature - Real time</a></h3>
          <p><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">oceans</span><span class="label label-info tag link-tag">precipitation</span><span class="label label-info tag link-tag">radiation</span></p>
          <p>Enhancement of Measurements on Ships of Opportunity (SOOP)-Air Sea Flux sub-facility collects underway meteorological and oceanographic observations during scientific and Antarctic resupply voyages in the oceans adjacent to Australia. Data product is quality controlled observations. Research Vessel Real Time Air-Sea Fluxes, equips the Marine National Facility (MNF) (Research Vessels Southern Surveyor and Investigator), the Australian Antarctic Division (Research and Supply Vessels Aurora Australis and Nuyina), and Research Vessel Tangaroa with &quot;climate quality&quot; meteorological measure...</p>
          <p><a href="/aodn_vessel_air_sea_flux_sst_meteo_realtime/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/vessel_air_sea_flux_sst_meteo_realtime.ipynb" class="" target="_none">Accessing IMOS - SOOP-Air Sea Flux (ASF) sub-facility - Meteorological and SST Observations</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_vessel_air_sea_flux_sst_meteo_realtime/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_vessel_co2_delayed_qc" class="dataset">
          <h3><a href="/aodn_vessel_co2_delayed_qc/">Ships of Opportunity - Biogeochemical sensors - Delayed mode</a></h3>
          <p><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The IMOS Ship of Opportunity Underway CO2 Measurements group is a research and data collection project working within the IMOS Ship of Opportunity Multi-Disciplinary Underway Network sub-facility. The CO2 group sample critical regions of the Southern Ocean and the Australian shelf waters have a major impact on CO2 uptake by the ocean. These are regions where biogeochemical cycling is predicted to be particularly sensitive to a changing climate. The pCO2 Underway System measures the fugacity of carbon dioxide (fCO2) along with other variables such as sea surface salinity (SSS) and sea surface t...</p>
          <p><a href="/aodn_vessel_co2_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/vessel_co2_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - SOOP Underway CO2 Measurements Research Group - delayed mode data</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_vessel_co2_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_vessel_xbt_realtime_nonqc" class="dataset">
          <h3><a href="/aodn_vessel_xbt_realtime_nonqc/">Ships of Opportunity - Expendable bathythermographs - Real time</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span></p>
          <p>XBT real-time data is available through the IMOS portal. Data is acquired by technicians who ride the ships of opportunity in order to perform high density sampling along well established transit lines. The data acquisition system used is the Quoll developed by Turo Technology. Data collected and is stored in netcdf files, with real-time data messages (JJVV bathy messages) created on the ship and sent to shore by iridium sbd. This is inserted onto the GTS by our colleagues at the Australian Bureau of Meteorology. The full resolution data is collected from the ship and returned for processing t...</p>
          <p><a href="/aodn_vessel_xbt_realtime_nonqc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/vessel_xbt_realtime_nonqc.ipynb" class="" target="_none">Accessing IMOS - SOOP Expendable Bathythermographs (XBT) Research Group - XBT real-time data</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_vessel_xbt_realtime_nonqc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_vessel_fishsoop_realtime_qc" class="dataset">
          <h3><a href="/aodn_vessel_fishsoop_realtime_qc/">Ships of Opportunity - Fisheries vessels - Real time</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span></p>
          <p>Fisheries Vessels as Ships of Opportunities (FishSOOP) is an IMOS Sub-Facility working with fishers to collect real-time temperature and depth data by installing equipment on a network of commercial fishing vessels using a range of common fishing gear.Every day, fishing vessels operate broadly across the productive areas of Australia’s Exclusive Economic Zone where we have few subsurface ocean measurements. The FishSOOP Sub-Facility is utilising this observing opportunity to cost-effectively increase the spatial and temporal resolution of subsurface temperature data in Australia’s inshore, she...</p>
          <p><a href="/aodn_vessel_fishsoop_realtime_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/vessel_fishsoop_realtime_qc.ipynb" class="" target="_none">Accessing IMOS SOOP - Fisheries Vessels as Ships of Opportunity Sub-Facility - Real-time data</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_vessel_fishsoop_realtime_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_vessel_sst_delayed_qc" class="dataset">
          <h3><a href="/aodn_vessel_sst_delayed_qc/">Ships of Opportunity - Sea surface temperature - 1-minute average data products</a></h3>
          <p><span class="label label-info tag link-tag">air temperature</span><span class="label label-info tag link-tag">atmosphere</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Sea Surface Temperature (SST) sub-facility produces 1-minute average data products. Observed data are 1-minute median SST values and are retrieved from the vessel once an hour. High-resolution 1-minute median data are available in the delayed mode approximately every 3 months, these were produced in delayed mode using visual (manual) inspection of the quality flags after the data had been run through the automated quality control software at the Bureau of Meteorology The data products are produced from data from 3 ships: L&#39;Astrolabe, Xutra Buhm and Wana Buhm, within this sub-facility w...</p>
          <p><a href="/aodn_vessel_sst_delayed_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/vessel_sst_delayed_qc.ipynb" class="" target="_none">Accessing IMOS - SOOP Sea Surface Temperature - delayed mode data</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_vessel_sst_delayed_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_vessel_trv_realtime_qc" class="dataset">
          <h3><a href="/aodn_vessel_trv_realtime_qc/">Ships of Opportunity - Tropical research vessels - Real time</a></h3>
          <p><span class="label label-info tag link-tag">chemistry</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The research vessels (RV Cape Ferguson and RV Solander) of the Australian Institute of Marine Science (AIMS) routinely record along-track (underway) measurements of near-surface water temperature, salinity, chlorophyll (fluorescence) and turbidity (NTU) during scientific operations in the tropical waters of northern Australia, particularly the Great Barrier Reef (GBR). All data records include sampling time (UTC), position (Latitude, Longitude) and water depth (under keel). Data are recorded at 10 second intervals. Data are measured with a Seabird SBE38 thermometer, Seabird SBE21 thermosalinog...</p>
          <p><a href="/aodn_vessel_trv_realtime_qc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/vessel_trv_realtime_qc.ipynb" class="" target="_none">Accessing Sensors on Tropical Research Vessels: Enhanced Measurements from Ships of Opportunity (SOOP)</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_vessel_trv_realtime_qc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aodn_wave_buoy_realtime_nonqc" class="dataset">
          <h3><a href="/aodn_wave_buoy_realtime_nonqc/">Wave buoys observations - Real time</a></h3>
          <p><span class="label label-info tag link-tag">oceans</span></p>
          <p>Buoys provide integral wave parameters. Buoy data from the following organisations contribute to the National Wave Archive: Manly Hydraulics Laboratory (part of the NSW Department of Planning and Environment (DPE), which has assumed function of the former NSW Office of Environment and Heritage (OEH)); Bureau of Meteorology; Western Australia Department of Transport (DOT); the Queensland Department of Environment and Science (DES); the Integrated Marine Observing System (IMOS); Gippsland Ports; the NSW Nearshore Wave Data Program from the NSW Department of Planning and Environment (DPE); the Un...</p>
          <p><a href="/aodn_wave_buoy_realtime_nonqc/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/wave_buoy_realtime_nonqc.ipynb" class="" target="_none">Accessing Wave buoys Observations - Australia - near real-time</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
          <li>
            <a href="https://nbviewer.org/github/aodn/aodn_cloud_optimised/blob/main/notebooks/GetAodnData.ipynb" class="" target="_none">Accessing and search for any AODN dataset</a> by <a href="https://github.com/aodn/aodn_cloud_optimised" target="_none">Laurent Besnard</a></li></a>
          </li>
        </ul>
        <p><a href="/aodn_wave_buoy_realtime_nonqc/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="yt8m" class="dataset">
          <h3><a href="/yt8m/">YouTube 8 Million - Data Lakehouse Ready</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">labeled</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">parquet</span><span class="label label-info tag link-tag">video</span></p>
          <p>This both the original .tfrecords and a Parquet representation of the <a href="https://research.google.com/youtube8m/">YouTube 8 Million dataset</a>. YouTube-8M is a large-scale labeled video dataset that consists of millions of YouTube video IDs, with high-quality machine-generated annotations from a diverse vocabulary of 3,800+ visual entities. It comes with precomputed audio-visual features from billions of frames and audio segments, designed to fit on a single hard disk. This dataset also includes the YouTube-8M Segments data from June 2019. 
This dataset is &#39;Lakehouse Ready&#39;. Meaning, you can query this data in-place straight out of...</p>
          <p><a href="/yt8m/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/aws-samples/data-lake-as-code/blob/roda-ml/docs/roda_install.md" class="aws-link" target="_none">Data Lake as Code Deployment Guide</a> by AWS Industry Blueprints Team</li></a>
          </li>
          <li>
            <a href="https://research.google.com/youtube8m/" class="" target="_none">YouTube 8 Million</a> by Google Research</li></a>
          </li>
        </ul>
        <p><a href="/yt8m/#usageexamples">See 2 usage examples &rarr;</a></p>
        </div>
        <div id="aws-igenomes" class="dataset">
          <h3><a href="/aws-igenomes/">AWS iGenomes</a></h3>
          <p><span class="label label-info tag link-tag">agriculture</span><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">biology</span><span class="label label-info tag link-tag">Caenorhabditis elegans</span><span class="label label-info tag link-tag">Danio rerio</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">Homo sapiens</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">Rattus norvegicus</span><span class="label label-info tag link-tag">reference index</span></p>
          <p>Common reference genomes hosted on AWS S3. Can be used when aligning and analysing raw DNA sequencing data.</p>
          <p><a href="/aws-igenomes/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="http://nf-co.re/" class="" target="_none">nf-core analysis pipelines</a> by <a href="http://phil.ewels.co.uk/" target="_none">Phil Ewels</a></li></a>
          </li>
        </ul>
        <p><a href="/aws-igenomes/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="amazon-pqa" class="dataset">
          <h3><a href="/amazon-pqa/">Amazon-PQA</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Amazon product questions and their answers, along with the public product information.</p>
          <p><a href="/amazon-pqa/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.amazon.science/publications/answering-product-questions-by-utilizing-questions-from-other-contextually-similar-products" class="" target="_none">Answering Product-Questions by Utilizing Questions from Other Contextually Similar Products</a> by Ohad Rozen, David Carmel, Avihai Mejer, Vitaly Mirkis, and Yftah Ziser</li></a>
          </li>
        </ul>
        <p><a href="/amazon-pqa/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="answer-reformulation" class="dataset">
          <h3><a href="/answer-reformulation/">Answer Reformulation</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Original StackExchange answers and their voice-friendly Reformulation.</p>
          <p><a href="/answer-reformulation/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www2020.thewebconf.org/" class="" target="_none">Voice-based Reformulation of Community Answers</a> by Simone Filice, Nachshon Cohen &amp; David Carmel</li></a>
          </li>
        </ul>
        <p><a href="/answer-reformulation/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="asr-error-robustness" class="dataset">
          <h3><a href="/asr-error-robustness/">Automatic Speech Recognition (ASR) Error Robustness</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">speech recognition</span></p>
          <p>Sentence classification datatasets with ASR Errors.</p>
          <p><a href="/asr-error-robustness/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://assets.amazon.science/64/94/639ae0c44890837b0f1fbf11ef77/using-phoneme-representations-to-build-predictive-models-robust-to-asr-errors.pdf" class="" target="_none">Using Phoneme Representations to Build Predictive Models Robust to ASR Errors</a> by <a href="https://anjiefang.github.io" target="_none">Anjie Fang, Simone Filice, Nut Limsopatham and Oleg Rokhlenko</a></li></a>
          </li>
        </ul>
        <p><a href="/asr-error-robustness/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="bodym" class="dataset">
          <h3><a href="/bodym/">BodyM Dataset</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span></p>
          <p>The first large public body measurement dataset including 8978 frontal and lateral
silhouettes for 2505 real subjects, paired with height, weight and 14 body
measurements. The following artifacts are made available for each subject.<ul>
<li>Subject Height</li>
<li>Subject Weight</li>
<li>Subject Gender</li>
<li>Two black-and-white silhouette images of subject standing in frontal and side pose
  respectively with full body in view.</li>
<li>14 body measurements in cm - {ankle girth, arm-length, bicep girth, calf girth,
                        chest girth, forearm girth, height, hip girth, leg-length,
                        shoulder-breadth, shoulder-to-crotch length, thigh girth,
                        waist girth, wrist girth}</li>
</ul>
The data is split into 3 sets - Training, Test Set A, Test Set B. For the training and
Test-A sets, subjects are photographed and 3D-scanned by in a lab by technicians. For
the Test-B set, subjects ...</p>
          <p><a href="/bodym/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.amazon.science/publications/human-body-measurement-estimation-with-adversarial-augmentation" class="" target="_none">Human Body Measurement Estimation with Adversarial Augmentation</a> by Nataniel Ruiz, Miriam Bellver, Timo Bolkart, Ambuj Arora, Ming C. Lin, Javier Romero and Raja Bala</li></a>
          </li>
        </ul>
        <p><a href="/bodym/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="boltz1" class="dataset">
          <h3><a href="/boltz1/">Boltz-1 Training Data</a></h3>
          <p><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">molecular docking</span><span class="label label-info tag link-tag">open source software</span><span class="label label-info tag link-tag">protein folding</span></p>
          <p>This is the data used to train the Boltz-1 model. It contains the following datasets:<ul>
<li>Our pre-processed version of the Protein Data Bank</li>
<li>Our pre-processed version of the multiple sequence alignment data for each protein chain</li>
<li>The raw multiple sequence alginment data.</li>
<li>A pre-computed symmetry file for symmetry correction during training</li>
</ul>
</p>
          <p><a href="/boltz1/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2024.11.19.624167v1.full.pdf" class="" target="_none">Boltz-1: Democratizing Biomolecular Interaction Modeling</a> by J Wohlwend, G Corso, S Passaro, M Reveiz, K Leidal, W Swiderski, T Portnoi, I Chinn, J Silterra, T Jaakkola, R Barzilay</li></a>
          </li>
        </ul>
        <p><a href="/boltz1/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="dialoglue" class="dataset">
          <h3><a href="/dialoglue/">DialoGLUE: A Natural Language Understanding Benchmark for Task-Oriented Dialogue</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">conversation data</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This bucket contains the checkpoints used to reproduce the baseline results reported in the DialoGLUE benchmark hosted
on EvalAI (<a href="https://evalai.cloudcv.org/web/challenges/challenge-page/708/overview">https://evalai.cloudcv.org/web/challenges/challenge-page/708/overview</a>). The associated scripts for using the checkpoints are located here:
<a href="https://github.com/alexa/dialoglue">https://github.com/alexa/dialoglue</a>. The associated paper describing the benchmark and checkpoints is here: <a href="https://arxiv.org/abs/2009.13570">https://arxiv.org/abs/2009.13570</a>.
The provided checkpoints include the CONVBERT model, a BERT-esque model trained on a large open-domain conversational
dataset. It also includes the CONVBERT-DG and BERT-DG checkpoints descri...</p>
          <p><a href="/dialoglue/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2009.13570" class="" target="_none">DialoGLUE: A Natural Language Understanding Benchmark for Task-Oriented Dialogue</a> by Shikib Mehri, Mihail Eric, Dilek Hakkani-Tur</li></a>
          </li>
        </ul>
        <p><a href="/dialoglue/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="topical-chat-enriched" class="dataset">
          <h3><a href="/topical-chat-enriched/">Enriched Topical-Chat Dataset for Knowledge-Grounded Dialogue Systems</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">conversation data</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This dataset provides extra annotations on top of the publicly released
Topical-Chat dataset(<a href="https://github.com/alexa/Topical-Chat">https://github.com/alexa/Topical-Chat</a>) which will help in reproducing the results in our paper
&quot;Policy-Driven Neural Response Generation for Knowledge-Grounded Dialogue Systems&quot; (<a href="https://arxiv.org/abs/2005.12529?context=cs.CL">https://arxiv.org/abs/2005.12529?context=cs.CL</a>). 
The dataset contains 5 files: train.json, valid_freq.json, valid_rare.json, test_freq.json and test_rare.json. 
Each of these files will have additional annotations on top of the original Topical-Chat dataset.
These specific annotations are: dialogue act annotations a...</p>
          <p><a href="/topical-chat-enriched/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2005.12529?context&#x3D;cs.CL" class="" target="_none">Policy-Driven Neural Response Generation for Knowledge-Grounded Dialogue Systems</a> by Behnam Hedayatnia, Karthik Gopalakrishnan, Seokhwan Kim, Yang Liu, Mihail Eric &amp; Dilek Hakkani-Tur</li></a>
          </li>
        </ul>
        <p><a href="/topical-chat-enriched/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="gcbo-dataset" class="dataset">
          <h3><a href="/gcbo-dataset/">Global Carbon Budget Data</a></h3>
          <p><span class="label label-info tag link-tag">climate</span><span class="label label-info tag link-tag">land</span><span class="label label-info tag link-tag">oceans</span></p>
          <p>The Global Carbon Budget (GCB) is recognised globally as the most comprehensive report on global carbon emissions and sinks. This dataset, updated every year, includes estimates of land and ocean carbon fluxes from the suite of models used in the report.</p>
          <p><a href="/gcbo-dataset/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://essd.copernicus.org/articles/15/5301/2023/" class="" target="_none">Global Carbon Budget 2023</a> by Pierre Friedlingstein, Michael O’Sullivan, Matthew W. Jones, Robbie M. Andrew, Luke Gregor, Judith Hauck, Corinne Le Quéré, Ingrid T. Luijkx, Are Olsen, Glen P. Peters, Wouter Peters, Julia Pongratz, Clemens Schwingshackl, Stephen Sitch, Josep G. Canadell, Philippe Ciais, Rob B. Jackson,Simone Alin, Ramdane Alkama, Almut Arneth, Vivek K. Arora, Nicholas R. Bates, Meike Becker, Nicolas Bellouin, Henry C. Bittig, Laurent Bopp, Frédéric Chevallier, Louise P. Chini, Margot Cronin, Wiley Evans, Stefanie Falk, Richard A. Feely, Thomas Gasser, Marion Gehlen, Thanos Gkritzalis, Lucas Gloege, Giacomo Grassi, Nicolas Gruber, Özgür Gürses, Ian Harris, Matthew Hefner, Richard A. Houghton, George C. Hurtt, Yosuke Iida, Tatiana Ilyina, Atul K. Jain, Annika Jersild, Koji Kadono, Etsushi Kato, Daniel Kennedy, Kees Klein Goldewijk, Jürgen Knauer, Jan Ivar Korsbakken, Peter Landschützer, Nathalie Lefèvre, Keith Lindsay, Junjie Liu, Zhu Liu, Gregg Marland, Nicolas Mayot, Matthew J. McGrath, Nicolas Metzl, Natalie M. Monacci, David R. Munro, Shin-Ichiro Nakaoka, Yosuke Niwa, Kevin O´Brien, Tsuneo Ono, Paul I. Palmer, Naiqing Pan, Denis Pierrot, Katie Pocock, Benjamin Poulter, Laure Resplandy, Eddy Robertson, Christian Rödenbeck, Carmen Rodriguez, Thais M. Rosan, Jörg Schwinger, Roland Séférian, Jamie D. Shutler, Ingunn Skjelvan, Tobias Steinhoff, Qing Sun, Adrienne J. Sutton, Colm Sweeney, Shintaro Takao, Toste Tanhua, Pieter P. Tans, Xiangjun Tian, Hanqin Tian, Bronte Tilbrook, Hiroyuki Tsujino, Francesco Tubiello, Guido R. van der Werf, Anthony P. Walker, Rik Wanninkhof, Chris Whitehead, Anna Wranne, Rebecca Wright, Wenping Yuan, Chao Yue, Xu Yue, Sönke Zaehle, Jiye Zeng, Bo Zheng</li></a>
          </li>
        </ul>
        <p><a href="/gcbo-dataset/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="google-brain-genomics-public" class="dataset">
          <h3><a href="/google-brain-genomics-public/">Google Brain Genomics Sequencing Dataset for Benchmarking and Development</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">bioinformatics</span><span class="label label-info tag link-tag">fastq</span><span class="label label-info tag link-tag">genetic</span><span class="label label-info tag link-tag">genomic</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">long read sequencing</span><span class="label label-info tag link-tag">short read sequencing</span><span class="label label-info tag link-tag">whole exome sequencing</span><span class="label label-info tag link-tag">whole genome sequencing</span></p>
          <p>To facilitate benchmarking and development, the Google Brain group has sequenced 9 human samples covering the Genome in a Bottle truth sets on different sequencing instruments, sequencing modalities (Illumina short read and Pacific BioSciences long read), sample preparation protocols, and for whole genome and whole exome capture. The original source of these data are <a href="https://console.cloud.google.com/storage/browser/brain-genomics-public/research/sequencing;tab=objects?prefix=&amp;forceOnObjectsSortingFiltering=false">gs://google-brain-genomics-public</a>.</p>
          <p><a href="/google-brain-genomics-public/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.biorxiv.org/content/10.1101/2020.12.11.422022v1.full" class="" target="_none">An Extensive Sequence Dataset of Gold-Standard Samples for Benchmarking and Development</a> by Baid G., Nattestad M., Kolesnikov A., Goel S., Yang H., Chang P., and Carroll A (2020)</li></a>
          </li>
        </ul>
        <p><a href="/google-brain-genomics-public/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="helpful-sentences-from-reviews" class="dataset">
          <h3><a href="/helpful-sentences-from-reviews/">Helpful Sentences from Reviews</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">information retrieval</span><span class="label label-info tag link-tag">json</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">text analysis</span></p>
          <p>A collection of sentences extracted from customer reviews labeled with their helpfulness score.</p>
          <p><a href="/helpful-sentences-from-reviews/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2104.09792" class="" target="_none">Identifying Helpful Sentences in Product Reviews</a> by Iftah Gamzu et al (2021)</li></a>
          </li>
        </ul>
        <p><a href="/helpful-sentences-from-reviews/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="humor-detection" class="dataset">
          <h3><a href="/humor-detection/">Humor Detection from Product Question Answering Systems</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This dataset provides labeled humor detection from product question answering systems.
The dataset contains 3 csv files: <a href="https://humor-detection-pds.s3-us-west-2.amazonaws.com/Humorous.csv">Humorous.csv</a> 
containing the humorous product questions, 
<a href="https://humor-detection-pds.s3-us-west-2.amazonaws.com/Non-humorous-unbiased.csv">Non-humorous-unbiased.csv</a> containing 
the non-humorous prodcut questions from the same products as the humorous one, and, 
<a href="https://humor-detection-pds.s3-us-west-2.a...</p>
          <p><a href="/humor-detection/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://sigir.org/sigir2020/" class="" target="_none">Humor Detection in Product Question Answering Systems.</a> by Yftah Ziser, Elad Kravi &amp; David Carmel</li></a>
          </li>
        </ul>
        <p><a href="/humor-detection/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="humor-patterns" class="dataset">
          <h3><a href="/humor-patterns/">Humor patterns used for querying Alexa traffic</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">dialog</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Humor patterns used for querying Alexa traffic when creating the taxonomy described in the paper &quot;“Alexa, Do You Want to Build a Snowman?” Characterizing Playful Requests to Conversational Agents&quot;  by Shani C., Libov A., Tolmach S., Lewin-Eytan L., Maarek Y., and Shahaf D. (CHI LBW 2022). These patterns corrospond to the researchers&#39; hypotheses regarding what humor types are likely to appear in Alexa traffic. These patterns were used for querying Alexa traffic to evaluate these hypotheses.</p>
          <p><a href="/humor-patterns/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://camps.aptaracorp.com/ACM_PMS/PMS/ACM/CHI22EXTENDEDABSTRACTS/454/3754dba1-93c0-11ec-b613-166a08e17233/OUT/chi22extendedabstracts-454.html#fn3" class="" target="_none">“Alexa, Do You Want to Build a Snowman?” Characterizing Playful Requests to Conversational Agents</a> by Shani C., Libov A., Tolmach S., Lewin-Eytan L., Maarek Y., and Shahaf D.</li></a>
          </li>
        </ul>
        <p><a href="/humor-patterns/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="asset-data-igp-coal-plant" class="dataset">
          <h3><a href="/asset-data-igp-coal-plant/">IGP Coal Plant</a></h3>
          <p><span class="label label-info tag link-tag">air quality</span><span class="label label-info tag link-tag">energy</span><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span></p>
          <p>This dataset includes detailed information about coal power plants, their locations, capacities, emissions, and other relevant attributes around the Indian Gangetic Plain.</p>
          <p><a href="/asset-data-igp-coal-plant/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/APAD2024/APAD-Asset-Data/tree/main/Asset%20Data/Coal%20Plants" class="" target="_none">Analyzing Coal Plant Data and Emission Calculation</a> by APAD</li></a>
          </li>
        </ul>
        <p><a href="/asset-data-igp-coal-plant/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="ltrf-cqa-dataset" class="dataset">
          <h3><a href="/ltrf-cqa-dataset/">Learning to Rank and Filter - community question answering</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This dataset provides product related questions and answers, including answers&#39; quality labels, as as part of the paper &#39;IR Evaluation and Learning in the Presence of Forbidden Documents&#39;.</p>
          <p><a href="/ltrf-cqa-dataset/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://sigir.org/sigir2022/" class="" target="_none">IR Evaluation and Learning in the Presence of Forbidden Documents</a> by David Carmel, Nachshon Cohen, Amir Ingber &amp; Elad Kravi</li></a>
          </li>
        </ul>
        <p><a href="/ltrf-cqa-dataset/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="lowcontext-ner-gaz" class="dataset">
          <h3><a href="/lowcontext-ner-gaz/">Low Context Name Entity Recognition (NER) Datasets with Gazetteer</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>See <a href="https://lowcontext-ner-gaz.s3.amazonaws.com/readme.html">https://lowcontext-ner-gaz.s3.amazonaws.com/readme.html</a></p>
          <p><a href="/lowcontext-ner-gaz/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.amazon.science/publications/gemnet-effective-gated-gazetteer-representations-for-recognizing-complex-entities-in-low-context-input" class="" target="_none">GEMNET: Effective Gated Gazetteer Representations for Recognizing Complex Entities in Low-context Input</a> by Tao Meng, Anjie Fang, Oleg Rokhlenko and Shervin Malmasi</li></a>
          </li>
        </ul>
        <p><a href="/lowcontext-ner-gaz/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="multi-token-completion" class="dataset">
          <h3><a href="/multi-token-completion/">Multi Token Completion</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This dataset provides masked sentences and multi-token phrases that were masked-out of these sentences.
We offer 3 datasets: a general purpose dataset extracted from the Wikipedia and Books corpora, and 2 additional datasets extracted from pubmed abstracts.
As for the pubmed data, please be aware that the dataset does not reflect the most current/accurate data available from NLM (it is not being updated).
For these datasets, the columns provided for each datapoint are as follows:
text- the original sentence
span- the span (phrase) which is masked out
span_lower- the lowercase version of span
r...</p>
          <p><a href="/multi-token-completion/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://2023.eacl.org/" class="" target="_none">Simple and Effective Multi-Token Completion from Masked Language Models</a> by Oren Kalinsky, Guy Kushilevitz, Alex Libov &amp; Yoav Goldberg</li></a>
          </li>
        </ul>
        <p><a href="/multi-token-completion/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="code-mixed-ner" class="dataset">
          <h3><a href="/code-mixed-ner/">Multilingual Name Entity Recognition (NER) Datasets with Gazetteer</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Name Entity Recognition datasets containing short sentences and queries with low-context,
including LOWNER, MSQ-NER, ORCAS-NER and Gazetteers (1.67 million entities).
This release contains the multilingual versions of the datasets in <a href="https://registry.opendata.aws/lowcontext-ner-gaz/">Low Context Name Entity Recognition (NER) Datasets with Gazetteer</a>.</p>
          <p><a href="/code-mixed-ner/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.amazon.science/publications/gazetteer-enhanced-named-entity-recognition-for-code-mixed-web-queries" class="" target="_none">Gazetteer Enhanced Named Entity Recognition for Code-Mixed Web Queries</a> by Besnik Fetahu, Anjie Fang, Oleg Rokhlenko and Shervin Malmasi</li></a>
          </li>
        </ul>
        <p><a href="/code-mixed-ner/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="pass-summaries-fewsum" class="dataset">
          <h3><a href="/pass-summaries-fewsum/">PASS: Perturb-and-Select Summarizer for Product Reviews</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">text analysis</span></p>
          <p>A collection of product reviews summaries automatically generated by PASS for 32 Amazon products from the FewSum dataset</p>
          <p><a href="/pass-summaries-fewsum/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.amazon.science/publications/pass-perturb-and-select-summarizer-for-product-reviews" class="" target="_none">PASS: Perturb-and-Select Summarizer for Product Reviews</a> by Nadav Oved and Ran Levy (2021)</li></a>
          </li>
        </ul>
        <p><a href="/pass-summaries-fewsum/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="person-path-22" class="dataset">
          <h3><a href="/person-path-22/">PersonPath22</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">computer vision</span></p>
          <p>PersonPath22 is a large-scale multi-person tracking dataset containing 236 videos captured mostly from static-mounted cameras,  collected from sources where we were given the rights to redistribute the content and participants have given explicit consent. Each video has ground-truth annotations including both bounding boxes and tracklet-ids for all the persons in each frame.</p>
          <p><a href="/person-path-22/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://assets.amazon.science/6f/13/e48da0fa4cefb9c81adbad6d93ec/large-scale-real-world-multi-person-tracking.pdf" class="" target="_none">Large scale Real-world Multi-Person Tracking</a> by Bing Shuai, Alessandro Bergamo, Uta Buechler, Andrew Berneshawi, Alyssa Boden, Joseph Tighe</li></a>
          </li>
        </ul>
        <p><a href="/person-path-22/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="pcd" class="dataset">
          <h3><a href="/pcd/">Phrase Clustering Dataset (PCD)</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">json</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This dataset is part of the paper &quot;McPhraSy: Multi-Context Phrase Similarity and Clustering&quot; by DN Cohen et al (2022). The purpose of PCD is to evaluate the quality of semantic-based clustering of noun phrases. The phrases were collected from the [Amazon Review Dataset] (<a href="https://nijianmo.github.io/amazon/">https://nijianmo.github.io/amazon/</a>).</p>
          <p><a href="/pcd/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.amazon.science/publications/mcphrasy-multi-context-phrase-similarity-and-clustering" class="" target="_none">McPhraSy: Multi context phrase similarity and clustering</a> by Amir DN Cohen, Hila Gonen, Ori Shapira, Ran Levy, and Yoav Goldberg</li></a>
          </li>
        </ul>
        <p><a href="/pcd/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="tgs-opendata-poseidon" class="dataset">
          <h3><a href="/tgs-opendata-poseidon/">Poseidon 3D Seismic, Australia</a></h3>
          <p><span class="label label-info tag link-tag">exploration</span><span class="label label-info tag link-tag">geophysics</span><span class="label label-info tag link-tag">seismology</span></p>
          <p>Near, mid, far, full stack (with AGC) imaged 3D seismic data. We also include the decimated
stacking velocity field. The dataset is used in oil and gas exploration. Survey size is
approximately 2,900 km2.Coordinate system used is: GDA94 / MGA Zone 51 Petrel, 700004Datasets are converted to open-source MDIO format (v1 specification).Original SEG-Y files are licensed as CC BY 3.0 AU and are downloaded from
<a href="https://drive.google.com/drive/folders/0B7brcf-eGK8Cbk9ueHA0QUU4Zjg?resourcekey=0-njGYrj9ZlE751Oi1-89sOg">Google Drive</a>
accessed via <a href="https://wiki.seg.org/wiki/Open_data">SEG Open Data Wiki</a>. The datasets are available
courtesy of ConocoPhillips and Geoscience Australia. Raw data can be requested from Geoscience
Australia&#39;s <a href="https://nopims.dmp.wa.gov.au/Nopims/">NOPIMS</a> s...</p>
          <p><a href="/tgs-opendata-poseidon/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://mdio.dev" class="" target="_none">MDIO</a> by <a href="https://www.tgs.com" target="_none">TGS</a></li></a>
          </li>
        </ul>
        <p><a href="/tgs-opendata-poseidon/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="pre-post-purchase-questions" class="dataset">
          <h3><a href="/pre-post-purchase-questions/">Pre- and post-purchase product questions</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This dataset provides product related questions, including their textual content and gap, in hours, between purchase and posting time. 
Each question is also associated with related product details, including its id and title.</p>
          <p><a href="/pre-post-purchase-questions/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://sigir.org/sigir2021/" class="" target="_none">&quot;Did you buy it already?&quot;, Detecting Users Purchase-State From Their Product-Related Questions
</a> by Lital Kuchy, David Carmel, Thomas Huet &amp; Elad Kravi</li></a>
          </li>
        </ul>
        <p><a href="/pre-post-purchase-questions/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="prod-comp-shopping" class="dataset">
          <h3><a href="/prod-comp-shopping/">Product Comparison Dataset for Online Shopping</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span><span class="label label-info tag link-tag">online shopping</span><span class="label label-info tag link-tag">product comparison</span></p>
          <p>The Product Comparison dataset for online shopping is a new, manually annotated dataset with about 15K human generated sentences, which compare related products based on one or more of their attributes (the first such data we know of for product comparison). It covers ∼8K product sets, their selected attributes, and comparison texts.</p>
          <p><a href="/prod-comp-shopping/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://dl.acm.org/doi/pdf/10.1145/3539597.3570489" class="" target="_none">Generating Explainable Product Comparisons for Online Shopping</a> by Nikhita Vedula, Marcus Collins, Eugene Agichtein and Oleg Rokhlenko</li></a>
          </li>
        </ul>
        <p><a href="/prod-comp-shopping/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="pyenvs-and-callargs" class="dataset">
          <h3><a href="/pyenvs-and-callargs/">PyEnvs and CallArgs</a></h3>
          <p><span class="label label-info tag link-tag">code completion</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>PyEnvs is a collection of 2814 permissively licensed Python packages along with their isolated development environments. Paired with a program analyzer (e.g. Jedi Language Server), it supports querying for project-related information. CallArgs is a dataset built on top of PyEnvs for function call argument completion. It provides function definition, implementation, and usage information for each function call instance.</p>
          <p><a href="/pyenvs-and-callargs/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2306.03438" class="" target="_none">Better Context Makes Better Code Language Models: A Case Study on Function Call Argument Completion</a> by Hengzhi Pei, Jinman Zhao, Leonard Lausen, Sheng Zha, George Karypis</li></a>
          </li>
        </ul>
        <p><a href="/pyenvs-and-callargs/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="shopping-humor-generation" class="dataset">
          <h3><a href="/shopping-humor-generation/">Shopping Humor Generation</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">commerce</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This dataset provides a set of non-shoppable items, which are items that can&#39;t be purchased via a virtual assistant (love, vampires, etc). 
In addition, for each non-shoppable item, the dataset contains humorous responses generated by two different large language models, and a template-based generation solution that uses a commonsense knowledge graph called ConceptNet.
Finally, each row contains a score provided by human annotators that judge how funny each response is.
The columns provided for each datapoint are as follows:
question- purchase request of the non-shoppable item
answer- a ge...</p>
          <p><a href="/shopping-humor-generation/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://ecir2023.org/" class="" target="_none">Evaluating Humorous Response Generation to Playful Shopping Requests</a> by Natalie Shapira, Oren Kalinsky, Alex Libov, Chen Shani, Sofia Tolmach</li></a>
          </li>
        </ul>
        <p><a href="/shopping-humor-generation/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="visa" class="dataset">
          <h3><a href="/visa/">Visual Anomaly (VisA)</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">anomaly detection</span><span class="label label-info tag link-tag">classification</span><span class="label label-info tag link-tag">fewshot</span><span class="label label-info tag link-tag">industrial</span><span class="label label-info tag link-tag">segmentation</span></p>
          <p>Largest Visual Anomaly detection dataset containing objects from 12 classes in 3 domains across 10,821(9,621 normal and 1,200 anomaly) images. Both image and pixel level annotations are provided.</p>
          <p><a href="/visa/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://arxiv.org/abs/2207.14315" class="" target="_none">SPot-the-Difference Self-Supervised Pre-training for Anomaly Detection and Segmentation</a> by Yang Zou, Jongheon Jeong, Latha Pemula, Dongqing Zhang, and Onkar Dabeer</li></a>
          </li>
        </ul>
        <p><a href="/visa/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="amazon-conversational-product-search" class="dataset">
          <h3><a href="/amazon-conversational-product-search/">VoiSeR</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">information retrieval</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Voice-based refinements of product search</p>
          <p><a href="/amazon-conversational-product-search/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://www.aclweb.org/anthology/venues/eacl/" class="" target="_none">VoiSeR: A New Benchmark for Voice-Based Search Refinement</a> by Simone Filice, Giuseppe Castellucci, Marcus Collins, Eugene Agichtein &amp; Oleg Rokhlenko</li></a>
          </li>
        </ul>
        <p><a href="/amazon-conversational-product-search/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="wikisum" class="dataset">
          <h3><a href="/wikisum/">WikiSum: Coherent Summarization Dataset for Efficient Human-Evaluation</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>This dataset provides how-to articles from <a href="https://www.wikihow.com">wikihow.com</a> and their summaries,
written as a coherent paragraph.
The dataset itself is available at <a href="https://wikisum.s3.amazonaws.com/WikiSumDataset.zip">wikisum.zip</a>,
and contains the article, the summary, the wikihow url, and an official fold (train, val, or test).
In addition, human evaluation results are available at
<a href="https://wikisum.s3.amazonaws.com/HumanEvaluation.zip">wikisum-human-eval...</p>
          <p><a href="/wikisum/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://2021.aclweb.org/" class="" target="_none">WikiSum: Coherent Summarization Dataset for Efficient Human-Evaluation</a> by Nachshon Cohen, Oren Kalinsky, Yftah Ziser &amp; Alessandro Moschitti</li></a>
          </li>
        </ul>
        <p><a href="/wikisum/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="wizard-of-tasks" class="dataset">
          <h3><a href="/wizard-of-tasks/">Wizard of Tasks</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">conversation data</span><span class="label label-info tag link-tag">dialog</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>Wizard of Tasks (WoT) is a dataset containing conversations for Conversational Task Assistants (CTAs). A CTA is a conversational agent whose goal is to help humans to perform real-world tasks. A CTA can help in exploring available tasks, answering task-specific questions and guiding users through step-by-step instructions. WoT contains about 550 conversations with ~18,000 utterances in two domains, i.e., Cooking and Home Improvement.</p>
          <p><a href="/wizard-of-tasks/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://aclanthology.org/events/coling-2022/" class="" target="_none">Wizard of Tasks: A Novel Conversational Dataset for Solving Real-World Tasks in Conversational Settings</a> by Jason Ingyu Choi, Saar Kuzi, Nikhita Vedula, Jie Zhao, Giuseppe Castellucci, Marcus Collins, Shervin Malmasi, Oleg Rokhlenko and Eugene Agichtein</li></a>
          </li>
        </ul>
        <p><a href="/wizard-of-tasks/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="mirrulations" class="dataset">
          <h3><a href="/mirrulations/">mirrulations</a></h3>
          <p><span class="label label-info tag link-tag">government records</span></p>
          <p>The regulations.gov website allows users to view proposed rules and supporting documents for the federal rule-making process.  In addition, users can post and view comments about those proposed rules.  The site contains about 27 million pieces of text and binary data, but the API that provides access only allows a user to obtain one thousand items per hour.  As a result, it would take approximately 3 years to download all the data.
Mirrulations (MIRRor of regULATIONS.gov) is a system that uses a collection of donated API keys to create a mirror of the data.  In addition, for each pdf in the da...</p>
          <p><a href="/mirrulations/">Details &rarr;</a></p>
          <h4>Usage examples</h4>
          <ul class="dataatwork-list">
          <li>
            <a href="https://github.com/MoravianUniversity/mirrulations-data" class="" target="_none">Extracting Comment Sentiment from Public Comments</a> by <a href="https://www.moravian.edu/computer-science/faculty" target="_none">Ben Coleman</a></li></a>
          </li>
        </ul>
        <p><a href="/mirrulations/#usageexamples">See 1 usage example &rarr;</a></p>
        </div>
        <div id="aiwp" class="dataset">
          <h3><a href="/aiwp/">AI Weather Prediction (AIWP) Model Reforecasts</a></h3>
          <p><span class="label label-info tag link-tag">environmental</span><span class="label label-info tag link-tag">meteorological</span><span class="label label-info tag link-tag">weather</span></p>
          <p><br/>
<br/>

This is an archive of pure AI-based weather prediction reforecasts produced collaboratively between the Cooperative Institute for Research in the Atmosphere <a href="https://www.cira.colostate.edu/">(CIRA)</a> and the NOAA Global Systems Laboratory <a href="https://gsl.noaa.gov/">(NOAA-GSL)</a>.<br/>
<br/>

Currently, <a href="https://arxiv.org/abs/2306.03838">FourCastNetv2-small</a>, <a href="https://www.nature.com/articles/s41586-023-06185-3">Pangu-Weather</a>, and <a href="https://www.science.org/doi/10.1126/science.adi2336">GraphCast</a> are included, with more models to come. Each of these models has been initialized with both NOAA <a href="https://www.emc.ncep.noaa.gov/emc/pages/numerical_forecast_systems/gfs.php">GFS</a> (directories with no extension) and ECMWF <a href="https://www.ecmwf.int/en/forecasts/datasets/set-i">IFS</a> initial conditions (directories ending in &quot;_IFS&quot;). The datasets are updated with near-real-time data twice per day (00Z and 12Z initializations).<br/>
...</p>
          <p><a href="/aiwp/">Details &rarr;</a></p>
        </div>
        <div id="airborne-object-tracking" class="dataset">
          <h3><a href="/airborne-object-tracking/">Airborne Object Tracking Dataset</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>Airborne Object Tracking (AOT) is a collection of 4,943 flight sequences of around 120 seconds each, collected at 10 Hz in diverse conditions. There are 5.9M+ images and 3.3M+ 2D annotations of airborne objects in the sequences. There are 3,306,350 frames without labels as they contain no airborne objects. For images with labels, there are on average 1.3 labels per image. All airborne objects in the dataset are labelled.</p>
          <p><a href="/airborne-object-tracking/">Details &rarr;</a></p>
        </div>
        <div id="amazon-berkeley-objects" class="dataset">
          <h3><a href="/amazon-berkeley-objects/">Amazon Berkeley Objects Dataset</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">information retrieval</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">machine translation</span></p>
          <p>Amazon Berkeley Objects (ABO) is a collection of 147,702 product listings with multilingual metadata and 398,212 unique catalog images. 8,222 listings come with turntable photography (also referred as &quot;spin&quot; or &quot;360º-View&quot; images), as sequences of 24 or 72 images, for a total of 586,584 images in 8,209 unique sequences. For 7,953 products, the collection also provides high-quality 3d models, as glTF 2.0 files.</p>
          <p><a href="/amazon-berkeley-objects/">Details &rarr;</a></p>
        </div>
        <div id="amazon-seller-contact-intent-sequence" class="dataset">
          <h3><a href="/amazon-seller-contact-intent-sequence/">Amazon Seller Contact Intent Sequence</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">Hawkes Process</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">temporal point process</span></p>
          <p>When sellers need help from Amazon, such as how to create a listing, they often reach out to Amazon seller support through email, chat or phone. For each contact, we assign an intent so that we can manage the request more easily. The data we present in this release includes 548k contacts with 118 intents from 70k sellers sampled from recent years. There are 3 columns. 1. De-identified seller id - seller_id_anon; 2. Noisy inter-arrival time in the unit of hour between contacts - interarrival_time_hr_noisy; 3. An integer that represents the contact intent - contact_intent. Note that, to balance ...</p>
          <p><a href="/amazon-seller-contact-intent-sequence/">Details &rarr;</a></p>
        </div>
        <div id="bluebrain_opendata" class="dataset">
          <h3><a href="/bluebrain_opendata/">Blue Brain Open Data</a></h3>
          <p><span class="label label-info tag link-tag">brain images</span><span class="label label-info tag link-tag">brain models</span><span class="label label-info tag link-tag">electrophysiology</span><span class="label label-info tag link-tag">ion channels</span><span class="label label-info tag link-tag">life sciences</span><span class="label label-info tag link-tag">microcircuit modeling and simulation</span><span class="label label-info tag link-tag">morphological reconstructions</span><span class="label label-info tag link-tag">Mus musculus</span><span class="label label-info tag link-tag">neuroscience</span><span class="label label-info tag link-tag">simulation neuroscience</span><span class="label label-info tag link-tag">single neuron models</span></p>
          <p>The Blue Brain Open Data represents an extensive neuroscience dataset encompassing a diverse range of data types, including experimental, model, and simulation data, along with images and videos depicting reconstructed neurons and brain regions.</p>
          <p><a href="/bluebrain_opendata/">Details &rarr;</a></p>
        </div>
        <div id="fashionlocaltriplets" class="dataset">
          <h3><a href="/fashionlocaltriplets/">FashionLocalTriplets</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">computer vision</span><span class="label label-info tag link-tag">machine learning</span></p>
          <p>Fine-grained localized visual similarity and search for fashion.</p>
          <p><a href="/fashionlocaltriplets/">Details &rarr;</a></p>
        </div>
        <div id="google-ngrams" class="dataset">
          <h3><a href="/google-ngrams/">Google Books Ngrams</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">natural language processing</span></p>
          <p>N-grams are fixed size tuples of items. In this case the items are words extracted from the Google Books corpus. The n specifies the number of elements in the tuple, so a 5-gram contains five words or characters. The n-grams in this dataset were produced by passing a sliding window of the text of books and outputting a record for each new token.</p>
          <p><a href="/google-ngrams/">Details &rarr;</a></p>
        </div>
        <div id="mwis-vr-instances" class="dataset">
          <h3><a href="/mwis-vr-instances/">MWIS VR Instances</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">graph</span><span class="label label-info tag link-tag">traffic</span><span class="label label-info tag link-tag">transportation</span></p>
          <p>Large-scale node-weighted conflict graphs for maximum weight independent set solvers</p>
          <p><a href="/mwis-vr-instances/">Details &rarr;</a></p>
        </div>
        <div id="registry-open-data" class="dataset">
          <h3><a href="/registry-open-data/">Registry of Open Data on AWS</a></h3>
          <p><span class="label label-info tag link-tag">amazon.science</span><span class="label label-info tag link-tag">json</span><span class="label label-info tag link-tag">metadata</span></p>
          <p>The Registry of Open Data on AWS contains publicly available datasets that are available for access from AWS resources. Note that datasets in this registry are available via AWS resources, but they are not provided by AWS; these datasets are owned and maintained by a variety of government organizations, researchers, businesses, and individuals. This dataset contains derived forms of the data in <a href="https://github.com/awslabs/open-data-registry">https://github.com/awslabs/open-data-registry</a> that have been transformed for ease of use with machine interfaces. Curren...</p>
          <p><a href="/registry-open-data/">Details &rarr;</a></p>
        </div>
        <div id="spatiam-nlra-iss-experiments" class="dataset">
          <h3><a href="/spatiam-nlra-iss-experiments/">Spatiam Corporation National Lab Research Announcement International Space Station  Technology Demonstration</a></h3>
          <p><span class="label label-info tag link-tag">network traffic</span><span class="label label-info tag link-tag">telecommunications</span></p>
          <p>Experiment data for the Spatiam DTN Network Platform Technology demonstration carried to test a Delay and Disruption Tolerant Network between the ISS and Earth.</p>
          <p><a href="/spatiam-nlra-iss-experiments/">Details &rarr;</a></p>
        </div>
        <div id="tsbench" class="dataset">
          <h3><a href="/tsbench/">TSBench</a></h3>
          <p><span class="label label-info tag link-tag">benchmark</span><span class="label label-info tag link-tag">deep learning</span><span class="label label-info tag link-tag">machine learning</span><span class="label label-info tag link-tag">meta learning</span><span class="label label-info tag link-tag">time series forecasting</span></p>
          <p>TSBench comprises thousands of benchmark evaluations for time series forecasting methods. It provides various metrics (i.e. measures of accuracy, latency, number of model parameters, ...) of 13 time series forecasting methods across 44 heterogeneous datasets. Time series forecasting methods include both classical and deep learning methods while several hyperparameters settings are evaluated for the deep learning methods.In addition to the tabular data providing the metrics, TSBench includes the probabilistic forecasts of all evaluated methods for all 44 datasets. While the tabular data is smal...</p>
          <p><a href="/tsbench/">Details &rarr;</a></p>
        </div>
      </div>
    </div>


    <hr/>
  </div>  </body>
  <script src="/js/index.js"></script>
</html>
